<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Biomol Struct Dyn</span><span class="iso-abbrev" title="iso-abbrev">J. Biomol. Struct. Dyn</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Biomolecular Structure &amp;amp; Dynamics</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0739-1102</span><span class="issn-epub" tagx="issn" title="issn-epub">1538-0254</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor &amp;amp; Francis</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7441776">7441776</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1080/07391102.2020.1794974">10.1080/07391102.2020.1794974</a></span><span class="publisher-id" title="publisher-id">1794974</span><div class="unknown" title="article-version" tag="article-version">
Version of Record</div> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Virtual screening, molecular dynamics and structureâ€"activity relationship studies to identify potent approved drugs for Covid-19 treatment</div> <div class="alt-title" title="alt-title">
M. M. Rahman et al.</div> <div class="alt-title" title="alt-title">
Journal of Biomolecular Structure and Dynamics</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-6994-2598</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahman</span><span class="given-names" tagx="given-names" title="given-names">Md. Mahbubur</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saha</span><span class="given-names" tagx="given-names" title="given-names">Titon</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">Kazi Jahidul</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suman</span><span class="given-names" tagx="given-names" title="given-names">Rasel Hosen</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biswas</span><span class="given-names" tagx="given-names" title="given-names">Sourav</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahat</span><span class="given-names" tagx="given-names" title="given-names">Emon Uddin</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossen</span><span class="given-names" tagx="given-names" title="given-names">Md. Rubel</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">Rajib</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossain</span><span class="given-names" tagx="given-names" title="given-names">Md Nayeem</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mamun</span><span class="given-names" tagx="given-names" title="given-names">Abdulla Al</span></span><a href="#AF0002">b</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">Maksud</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">Md Ackas</span></span><a href="#AF0001">a</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-1698-7044</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halim</span><span class="given-names" tagx="given-names" title="given-names">Mohammad A.</span></span><a href="#AF0001">a</a><a href="#AF0003">c</a><a href="#AN0001" /></span><span class="citation_author_institution" id="AF0001">[a], <span class="institution" title="institution">Division of Infectious Diseases and Division of Computer-Aided Drug Design, The Red-Green Research Centre, BICCB</span><span class="city" tagx="city" title="city">Tejgaon</span><span class="state" tagx="state" title="state">Dhaka</span><span class="country" tagx="country" title="country">Bangladesh</span></span><span class="citation_author_institution" id="AF0002">[b], <span class="institution" title="institution">Key Laboratory of Soft Chemistry and Functional Materials of MOE, Department of Chemical Engineering, Nanjing University of Science and Technology</span><span class="city" tagx="city" title="city">Nanjing</span><span class="country" tagx="country" title="country">China</span></span><span class="citation_author_institution" id="AF0003">[c], <span class="institution" title="institution">Department of Physical Sciences, University of Arkansas-Fort Smith</span><span class="city" tagx="city" title="city">Fort Smith</span><span class="state" tagx="state" title="state">AR</span><span class="country" tagx="country" title="country">USA</span></span></div> <div class="author-notes" title="author-notes"> <div class="fn-type-" title=""> <p>Supplemental data for this article can be accessed online at https://doi.org/10.1080/07391102.2020.1794974</p> </div> <div class="corresp" title="corresp"><b>CONTACT</b> Mohammad A. Halim <span class="email" tagx="email" title="email">mohammad.halim@uafs.edu</span></div> </div><span class="pub-date-" title="pub-date-">: <span>2020-7-7</span></span><span class="pub-date-" title="pub-date-">: <span>2020</span></span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">11</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Informa UK Limited, trading as Taylor &amp;amp; Francis Group</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the COVID-19 pandemic or until permissions are revoked in writing. Upon expiration of these permissions, PMC is granted a perpetual license to make this article available via PMC and Europe PMC, consistent with existing copyright protections.</span></span></div><a class="self-uri" href="TBSD_0_1794974.pdf" title="self-uri">TBSD_0_1794974.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Computer-aided drug screening by molecular docking, molecular dynamics (MD) and structuralâ€"activity relationship (SAR) can offer an efficient approach to identify promising drug repurposing candidates for COVID-19 treatment. In this study, computational screening is performed by molecular docking of 1615 Food and Drug Administration (FDA) approved drugs against the main protease (Mpro) of SARS-CoV-2. Several promising approved drugs, including Simeprevir, Ergotamine, Bromocriptine and Tadalafil, stand out as the best candidates based on their binding energy, fitting score and noncovalent interactions at the binding sites of the receptor. All selected drugs interact with the key active site residues, including His41 and Cys145. Various noncovalent interactions including hydrogen bonding, hydrophobic interactions, piâ€"sulfur and piâ€"pi interactions appear to be dominant in drugâ€"Mpro complexes. MD simulations are applied for the most promising drugs. Structural stability and compactness are observed for the drugâ€"Mpro complexes. The protein shows low flexibility in both apo and holo form during MD simulations. The MM/PBSA binding free energies are also measured for the selected drugs. For pattern recognition, structural similarity and binding energy prediction, multiple linear regression (MLR) models are used for the quantitative structuralâ€"activity relationship. The binding energy predicted by MLR model shows an 82% accuracy with the binding energy determined by molecular docking. Our details results can facilitate rational drug design targeting the SARS-CoV-2 main protease.</p> <p>Communicated by Ramaswamy H. Sarma</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
COVID-19</div> <div class="kwd" title="kwd">
SARS-CoV-2</div> <div class="kwd" title="kwd">
FDA approved drug</div> <div class="kwd" title="kwd">
structuralâ€"activity relationship</div> <div class="kwd" title="kwd">
molecular docking</div> <div class="kwd" title="kwd">
molecular dynamics</div> <div class="kwd" title="kwd">
principal component analysis</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1.</span><div class="title" tagx="title" title="title">
Introduction</div> <p>A novel coronavirus was identified in Wuhan, China in December 2019 spreading an infectious pneumonia-like disease (Huang etÂ al., <a href="#CIT0020">2020</a>). The International Committee on Taxonomy of Viruses (ICTV) named the novel coronavirus (2019-nCoV) as SARS-CoV-2 (Gorbalenya etÂ al., <a href="#CIT0018">2020</a>). The World Health Organization (WHO) officially named the disease caused by SARS-CoV-2 as Coronavirus disease (COVID-19) on 11 February 2020. On the basis of â€˜alarming levels of spread and severity, and by the alarming levels of inactionâ€™, on 11 March 2020, the Director-General of WHO characterized the COVID-19 situation as a pandemic (Bedford etÂ al., <a href="#CIT0004">2020</a>). To date, the total numbers of reported cases have reached over 5 million and the number of death is more than 300 thousand (Worldometer, <a href="#CIT0046">2020</a>). Earlier, epidemics of severe acute respiratory syndrome coronavirus (SARS-CoV) in Guangdong, China in 2003 and Middle East respiratory syndrome coronavirus (MERS-CoV) in 2012 have shown a high fatality rate (Hilgenfeld &amp;amp; Peiris, <a href="#CIT0019">2013</a>; Ton etÂ al., <a href="#CIT0044">2020</a>). SARS-CoV-2 was found to be an enveloped positive-sense, single- strained RNA virus belonging to the genus betacoronavirus (Beta-CoV) (Chan etÂ al., <a href="#CIT0005">2020</a>; Lu etÂ al., <a href="#CIT0030">2020</a>). Phylogenetic analysis showed that SARS-CoV-2 is closely related to (with 88% identity) two bat-derived severe acute respiratory syndrome (SARS)-like coronaviruses, bat-SL-CoVZC45 and bat-SL- CoVZXC21, collected in 2018 in Zhoushan, Eastern China, but are more distant from SARS-CoV (about 79%) and MERS-CoV (about 50%) (Lu etÂ al., <a href="#CIT0030">2020</a>). CoVs have the largest known RNA virus genomes, ranging from 27 to 34â€‰kb (Sexton etÂ al., <a href="#CIT0039">2016</a>).</p> <p>Inside the host cell, coronavirus genomes are translated into two groups of proteins; structural proteins, such as Spike (S), Envelope (E), Matrix (M) and Nucleocapsid (N) and nonstructural proteins, such as 3C-Like Protease (3CL-PRO, nsp5), RNA Dependent RNA Polymerase (RdRp, nsp12) (Elfiky, <a href="#CIT0014">2020</a>). Coronavirus genome replication and transcription takes place at cytoplasmic membranes and involves a coordinated process of both continuous and discontinuous RNA synthesis that is mediated by the viral replicate (Sola etÂ al., <a href="#CIT0041">2015</a>). The main protease (Mpro) of SARS-CoV-2 plays an important role in cutting down polyproteins into functional pieces (Chen etÂ al., <a href="#CIT0007">2020</a>; Shaghaghi, <a href="#CIT0040">2020</a>). Thus, the main protease plays a vital role in viral replication. The main protease (also known as 3C-like protease) can be an attractive drug target to inhibit coronavirus. Inhibiting the activity of the main protease could block the replication of the virus inside infected cells.</p> <p>The crystallized structure of the main protease (Mpro) or Chymotrypsin-like protease (3CL-Pro) of SARS-CoV-2 was identified and repositioned at the RCSB Protein Data bank (PDB) (Jin etÂ al., <a href="#CIT0024">2020</a>). The crystallized structure (PDB ID: 6LU7) contains two chains, A and B, which form a homodimer. Chain A was used for macromolecule preparation. The protein chain is composed of 306 residues. His41 and Cys145 form an uncharged catalytic dyad (Paasche etÂ al., <a href="#CIT0036">2014</a>). The active site of the protease is activated by a protonation reaction in the catalytic site (Paasche etÂ al., <a href="#CIT0036">2014</a>). Inhibiting the activity of the catalytic dyad will ultimately help to inhibit the activity of the main protease.</p> <p>Currently, no effective drugs targeting SARS-CoV-2 are available. Drug repurposing, a strategy to identify new uses of approved drugs, could shorten the time and reduce the cost for identifying effective drugs against COVID-19 (Zhou etÂ al., <a href="#CIT0048">2020</a>). We have selected 1615 FDA approved drugs from the ZINC database (Irwin &amp;amp; Shoichet, <a href="#CIT0021">2005</a>). The focus of this study is to identify the binding affinities and molecular interactions of these drugs against the main protease using computational and statistical tools. Molecular docking, molecular dynamics (MD), principal components analysis (PCA) and quantitative structureâ€"activity relationships (QSAR) were executed to evaluate the performance of the drugs.</p> </div> <div class="computationalmethods" title="sec"><span class="label" tagx="label" title="label">2.</span><div class="title" tagx="title" title="title">
Computational methods</div> <div class="virtualscreeningandmoleculardockingprotocols" title="sec"><span class="label" tagx="label" title="label">2.1.</span><div class="title" tagx="title" title="title">
Virtual screening and molecular docking protocols</div> <p>The structures of 1615 FDA approved drugs are obtained from the ZINC database. The database was then checked for redundant molecules. The three-dimensional crystal structure of SARS-CoV-2 main protease (Mpro,3CL<sup>pro</sup>) was collected from the RCSB Protein Data Bank (PDB) database (PDB-ID:6LU7, Resolution = 2.16â€‰Ã…, Method: X-ray diffraction, Organism: Bat-SARS like coronavirus) (Zhang etÂ al., <a href="#CIT0047">2020</a>). The crystal structure of Mpro was prepared according to previously published methods for molecular docking (Islam etÂ al., <a href="#CIT0023">2020</a>). Potential active site residues were specified by their vicinity to the ligand, N3 (Sekhar, <a href="#CIT0038">2020</a>). PyRx Virtual Screening Tools incorporated with Autodock Vina was used for virtual screening (Dallakyan &amp;amp; Olson, <a href="#CIT0011">2015</a>). Structural optimization of the drugs was carried out with universal force field (UFF) using the steepest descent optimization algorithm, a total of 2000 minimization steps (Ahmed etÂ al., <a href="#CIT0001">2020</a>). These drugs were converted to AutoDock ligand (PDBQT format) for docking. The grid center points were set to <i>X</i> = âˆ’11.5693, <i>Y</i>â€‰=â€‰14.9663, <i>Z</i>â€‰=â€‰67.9041 and box dimensions (Angstrom) <i>X</i>â€‰=â€‰34.6857, <i>Y</i>â€‰=â€‰45.2264, <i>Z</i>â€‰=â€‰42.8980. Grid box center points and dimensions were set to target the substrate binding-binding pocket of the protein (Chen etÂ al., <a href="#CIT0006">2006</a>). The binding affinities of the drugs were measured in kcal/mol unit and sorted according to the higher negative values, which imply the best binding affinities (de Sousa etÂ al., <a href="#CIT0012">2020</a>). Based on calculated binding affinities, the 31 top-ranked potential drugs were chosen for further analysis. The top-ranked drugs were then optimized by Gaussian 09 software using semi-empirical PM6 method (Frisch, <a href="#CIT0600">2009</a>). These optimized drug structures are then docked again using AutoDock Vina software and GOLD (Jones etÂ al., <a href="#CIT0025">1997</a>) docking programs. The molecular interactions of the drugs as predicted by docking simulation were analyzed in BIOVIA Discovery Studio Visualizer (Studio, <a href="#CIT0042">2015</a>).</p> </div> <div class="mdsimulationprotocols" title="sec"><span class="label" tagx="label" title="label">2.2.</span><div class="title" tagx="title" title="title">
MD simulation protocols</div> <p>The MD simulations for the main protease were conducted (6LU7) in apo-form (protein without ligand) and in holo-form (proteinâ€"drug complex) to assess any probable conformational changes to and interactions with their structures over the 100 nanoseconds (ns). AMBER14 force field (Dickson etÂ al., <a href="#CIT0013">2014</a>) was used on YASARA Dynamics (Krieger etÂ al., <a href="#CIT0027">2013</a>) for the simulation. Water molecules were added and the system was neutralized by the addition of NaCl salt of 0.9% concentration at a temperature of 310â€‰K (Krieger etÂ al., <a href="#CIT0028">2006</a>). The particle-mesh Ewald method was used for calculating long-range electrostatic interactions. A periodic boundary condition was utilized to carry out the simulation. In all cases, the cell size was larger than the protein by 20â€‰Ã…. The temperature of the simulation was controlled by the Berendsen thermostat. The initial energy minimization process of each simulation was performed by the simulated annealing method, using the steepest gradient approach (5000 cycles). Normal simulation speed (1.25 fs time step) was maintained during all the simulations (Krieger &amp;amp; Vriend, <a href="#CIT0029">2015</a>). A snapshot was saved in every 100â€‰ps during simulations. After completion of 100â€‰ns MD simulation for all systems, the results were analyzed in YASARA. During the analysis of the simulations, time, energy, bond distances, bond angles, dihedral angles, solventâ€"accessible surface area (SASA), molecular surface area (MolSA), coulombic and Van der Waals interactions, root mean square fluctuation (RMSF) and root mean square deviation (RMSD) values for backbone, alpha carbon and heavy atoms were recorded. For more accuracy, the 100â€‰ns MD simulation was conducted twice for every complex and the average result was used for analysis.</p> <p>For binding free energy calculations, molecular mechanics/Poissonâ€"Boltzmann surface area (MM/PBSA) method was used in YASARA Dynamics software (Baker etÂ al., <a href="#CIT0003">2001</a>; Chirag etÂ al., <a href="#CIT0008">2020</a>; Massova &amp;amp; Kollman, <a href="#CIT0033">2000</a>). The AMBER14 force field was used during the binding free energy calculation. The default macro file was modified for the calculation. The following equation was used for the binding free energy calculation: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" position="float" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Î"</div> <div class="mi_UNKNOWN">
G</div> <div class="mtext_UNKNOWN">
bind</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
Î"</div> <div class="mi_UNKNOWN">
G</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
complex</div> </div> </div> <div class="mo_UNKNOWN">
(</div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
minimized</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
â€"</div> <div class="mi_UNKNOWN">
Â </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Î"</div> <div class="mi_UNKNOWN">
G</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
ligand</div> </div> </div> <div class="mo_UNKNOWN">
(</div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
minimized</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mo_UNKNOWN">
Î"</div> <div class="mi_UNKNOWN">
G</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
receptor</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mtext_UNKNOWN">
minimized</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></div> </div> <div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" position="float" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
Î"</div> </div> <div class="mi_UNKNOWN">
G</div> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
Î"</div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="mi_UNKNOWN">
M</div> </div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
Î"</div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
B</div> </div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
Î"</div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
Î"</div> <div class="mi_UNKNOWN">
S</div> </div> </div> </div></div></div> </div> <p /> <p>Here, Î"<i>G</i><sub>MM</sub> is the molecular mechanics interaction (sum of electrostatic and van der Waals interaction), Î"<i>G</i><sub>PB</sub> and Î"<i>G</i><sub>SA</sub> correspond to polar and nonpolar solvation energyies, respectively. <i>TÎ"S</i> is the entropic contribution. Due to the high computational time, the entropy contribution was not considered. The final 50â€‰ns data of MD simulation was used for binding free energy calculations.</p> <p>Principal component analysis (PCA) is applied to identify similarities and dissimilarities among the collected energy profiles of MD trajectory data (Martens &amp;amp; Naes, <a href="#CIT0032">1998</a>). The important components of a PCA model are highlighted in the following equation: <i>Xâ€‰=â€‰T<sub>k</sub>P<sub>k</sub><sup>T</sup> + E</i>, where the <i>X</i> matrix expresses a product of two new matrices, i.e. <i>T<sub>k</sub></i>, <i>P<sub>k</sub></i>, <i>T<sub>k</sub></i> is the matrix of scores that shows how samples relate to each other, <i>P<sub>k</sub></i> is the matrix of loadings, which carries information about how the variables relate to each other, <i>k</i> represents the number of factors involved in the model and <i>E</i> is the matrix of residuals.</p> <p>The selected four drugâ€"main protease complexes may have dissimilarities with the apoâ€"Mpro during MD simulation regarding the energy profile. These dissimilarities can be identified using the PCA algorithm (Islam et al., <a href="#CIT0022">2019</a>, <a href="#CIT0023">2020</a>). All calculations were carried out on R platform employing in-house developed codes and plots were produced using the package ggplot2 (Kassambara, <a href="#CIT0026">2016</a>). Data were preprocessed employing autoscale function before applying the PCA algorithm (Martens &amp;amp; Naes, <a href="#CIT0032">1998</a>). The last 40â€‰ns data of the MD trajectory was utilized for PCA.</p> </div> <div class="structureâ€" title="sec"><span class="label" tagx="label" title="label">2.3.</span><div class="title" tagx="title" title="title">
Structureâ€"activity relationship protocols</div> <p>Structureâ€"activity relationship (SAR) is a method for establishing computational or mathematical models that aim to detect a statistically significant correlation between structure and function employing chemometric technique (Verma etÂ al., <a href="#CIT0045">2010</a>). 31 initially selected drugs are explored in this study. Topological polar surface area (TPSA, Ã…<sup>2</sup>), molecular weight (MW, gmol<sup>âˆ’1</sup>), XLogP3-AA, hydrogen bond donor (HBD), hydrogen bond acceptor (HBA), number of the rotatable bonds (ROTB), number of H, C, O, Cl, F atoms, single bonds (SB) count, number of double bonds (DB) and number of benzene rings of the drug candidates were studied as variables. Along with calculated binding energies, these variables were applied to correlate with the SAR by multiple linear regression (MLR) (Fakayode etÂ al., <a href="#CIT0015">2009</a>; Mark &amp;amp; Workman, <a href="#CIT0031">2007</a>). MLR was carried out with XLSTAT (Adinsoft, <a href="#CIT0700">2010</a>).</p> </div> </div> <div class="resultsanddiscussions" title="sec"><span class="label" tagx="label" title="label">3.</span><div class="title" tagx="title" title="title">
Results and discussions</div> <div class="virtualscreeningandmoleculardocking" title="sec"><span class="label" tagx="label" title="label">3.1.</span><div class="title" tagx="title" title="title">
Virtual screening and molecular docking</div> <p>The structure of 1615 FDA approved drugs is considered for virtual screening using Autodock Vina. Considering âˆ’5.1â€‰kcal/mol as a cutoff value, the binding affinities of the drugs are distributed between âˆ’5.1 to âˆ’6.0, âˆ’6.1 to âˆ’7.0, âˆ’7.1 to âˆ’8.0, âˆ’8.1 to âˆ’9.0 and âˆ’9.1 to âˆ’10.4â€‰kcal/mol (<a href="#F0001">Figure 1</a>). Most drugs are found to have binding affinities between âˆ’6.1 to âˆ’7.0â€‰kcal/mol. The top 31 candidates are selected for further analysis according to their binding affinities. The binding affinity of two already reported drugs Remdesivir and Lopinavir were observed âˆ’7.5 and âˆ’7.4â€‰kcal/mol, respectively. After optimizing by PM6 level of theory, the selected drugs are screened again in AutoDock Vina. GOLD suit was also employed to understand the binding fitness using the CHEMPLP fitness score. Here, a higher fitness score suggests better docking interaction between protein and ligand. Along with 5 previously suggested FDA approved drugs (Contini, <a href="#CIT0010">2020</a>), the binding affinities and GOLD fitting scores of 31 drugs are summarized in <a href="#t0001">Table 1</a>. According to best binding affinities, Simeprevir, Ergotamine, Bromocriptine and Tadalafil are selected for further analysis (<a href="#F0002">Figure 2</a>). Remdesivir and Lopinavir are also considered as control drugs for MD simulations.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Frequency distribution of FDA approved drugs over the range of docking score (Cutoff value â€"5.1â€‰kcal/mol).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Two dimensional (2D) structures of the selected drugs.</p>   </div> <div class="table-wrap_UNKNOWN" id="t0001" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <p>AutoDock Vina predicted binding affinity (kcal/mol) and GOLD fitting score of 31 drugs and 5 already reported drugs with SARS-CoV-2 Mpro.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Drug</th> <th align="center">Binding affinity (kcal/mol)</th> <th align="center">GOLD fitting score</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">Simeprevir</td> <td rowspan="2" align="center">â€"10.3</td> <td rowspan="2" align="char" char=".">73.83</td> </tr> <tr> <td align="left">ZINC000164760756</td> </tr> <tr> <td align="left">Ergotamine</td> <td rowspan="2" align="center">â€"9.8</td> <td rowspan="2" align="char" char=".">81.8</td> </tr> <tr> <td align="left">ZINC000052955754</td> </tr> <tr> <td align="left">Bromocriptine</td> <td rowspan="2" align="center">â€"9.6</td> <td rowspan="2" align="char" char=".">70.66</td> </tr> <tr> <td align="left">ZINC000053683151</td> </tr> <tr> <td align="left">Tadalafil</td> <td rowspan="2" align="center">â€"9.5</td> <td rowspan="2" align="char" char=".">59.32</td> </tr> <tr> <td align="left">ZINC000003993855</td> </tr> <tr> <td align="left">Dihydroergotamine</td> <td rowspan="2" align="center">â€"9.2</td> <td rowspan="2" align="char" char=".">75.46</td> </tr> <tr> <td align="left">ZINC000003978005</td> </tr> <tr> <td align="left">Perampanel</td> <td rowspan="2" align="center">â€"9.2</td> <td rowspan="2" align="char" char=".">74.57</td> </tr> <tr> <td align="left">ZINC000030691797</td> </tr> <tr> <td align="left">Nilotinib</td> <td rowspan="2" align="center">â€"9.2</td> <td rowspan="2" align="char" char=".">84.77</td> </tr> <tr> <td align="left">ZINC000006716957</td> </tr> <tr> <td align="left">Rolapitant</td> <td rowspan="2" align="center">â€"9.1</td> <td rowspan="2" align="char" char=".">72.66</td> </tr> <tr> <td align="left">ZINC000003816514</td> </tr> <tr> <td align="left">Naldemedine</td> <td rowspan="2" align="center">â€"8.9</td> <td rowspan="2" align="char" char=".">70.68</td> </tr> <tr> <td align="left">ZINC000100378061</td> </tr> <tr> <td align="left">Irinotecan ZINC000001612996</td> <td align="center">â€"8.9</td> <td align="center">72.3</td> </tr> <tr> <td align="left">Raltegravir</td> <td rowspan="2" align="center">â€"8.9</td> <td rowspan="2" align="char" char=".">64.33</td> </tr> <tr> <td align="left">ZINC000013831130</td> </tr> <tr> <td align="left">Lumacaftor</td> <td rowspan="2" align="center">â€"8.8</td> <td rowspan="2" align="char" char=".">68.84</td> </tr> <tr> <td align="left">ZINC000064033452</td> </tr> <tr> <td align="left">Eltrombopag</td> <td rowspan="2" align="center">â€"8.8</td> <td rowspan="2" align="char" char=".">69.34</td> </tr> <tr> <td align="left">ZINC000011679756</td> </tr> <tr> <td align="left">Saquinavir</td> <td rowspan="2" align="center">â€"8.7</td> <td rowspan="2" align="char" char=".">87.79</td> </tr> <tr> <td align="left">ZINC000003914596</td> </tr> <tr> <td align="left">Sildenafil</td> <td rowspan="2" align="center">â€"8.7</td> <td rowspan="2" align="char" char=".">80.21</td> </tr> <tr> <td align="left">ZINC000019796168</td> </tr> <tr> <td align="left">Pimozide</td> <td rowspan="2" align="center">â€"8.6</td> <td rowspan="2" align="char" char=".">77.44</td> </tr> <tr> <td align="left">ZINC000004175630</td> </tr> <tr> <td align="left">Paliperidone</td> <td rowspan="2" align="center">â€"8.5</td> <td rowspan="2" align="char" char=".">62.18</td> </tr> <tr> <td align="left">ZINC000004214700</td> </tr> <tr> <td align="left">Suvorexant</td> <td rowspan="2" align="center">â€"8.5</td> <td rowspan="2" align="char" char=".">62.23</td> </tr> <tr> <td align="left">ZINC000049036447</td> </tr> <tr> <td align="left">Nintedanib</td> <td rowspan="2" align="center">â€"8.5</td> <td rowspan="2" align="char" char=".">79.48</td> </tr> <tr> <td align="left">ZINC000100014909</td> </tr> <tr> <td align="left">Maraviroc</td> <td rowspan="2" align="center">â€"8.5</td> <td rowspan="2" align="char" char=".">74.25</td> </tr> <tr> <td align="left">ZINC000100003902</td> </tr> <tr> <td align="left">Paliperidone</td> <td rowspan="2" align="center">â€"8.4</td> <td rowspan="2" align="char" char=".">58.75</td> </tr> <tr> <td align="left">ZINC000001481956</td> </tr> <tr> <td align="left">Conivaptan</td> <td rowspan="2" align="center">â€"8.4</td> <td rowspan="2" align="char" char=".">69.76</td> </tr> <tr> <td align="left">ZINC000012503187</td> </tr> <tr> <td align="left">Pazopanib</td> <td rowspan="2" align="center">â€"8.4</td> <td rowspan="2" align="char" char=".">66.93</td> </tr> <tr> <td align="left">ZINC000011617039</td> </tr> <tr> <td align="left">Ibrutinib</td> <td rowspan="2" align="center">â€"8.4</td> <td rowspan="2" align="char" char=".">76.69</td> </tr> <tr> <td align="left">ZINC000035328014</td> </tr> <tr> <td align="left">Tipranavir</td> <td rowspan="2" align="center">â€"8.4</td> <td rowspan="2" align="char" char=".">79.31</td> </tr> <tr> <td align="left">ZINC000100022637</td> </tr> <tr> <td align="left">Dabrafenib ZINC000068153186</td> <td align="center">â€"8.3</td> <td align="center">68.28</td> </tr> <tr> <td align="left">Telotristat</td> <td rowspan="2" align="center">â€"8.3</td> <td rowspan="2" align="char" char=".">74.89</td> </tr> <tr> <td align="left">ZINC000084758235</td> </tr> <tr> <td align="left">Teniposide</td> <td rowspan="2" align="center">â€"8.2</td> <td rowspan="2" align="char" char=".">58.82</td> </tr> <tr> <td align="left">ZINC000004099009</td> </tr> <tr> <td align="left">Apixaban</td> <td rowspan="2" align="center">â€"8.2</td> <td rowspan="2" align="char" char=".">72.74</td> </tr> <tr> <td align="left">ZINC000011677837</td> </tr> <tr> <td align="left">Rifaximin</td> <td rowspan="2" align="center">â€"7.7</td> <td rowspan="2" align="char" char=".">42.56</td> </tr> <tr> <td align="left">ZINC000169621200</td> </tr> <tr> <td align="left">Lifitegrast</td> <td rowspan="2" align="center">â€"7.5</td> <td rowspan="2" align="char" char=".">73.73</td> </tr> <tr> <td align="left">ZINC000084668739</td> </tr> <tr> <td align="left">Montelukast (Contini, <a href="#CIT0010">2020</a>)</td> <td align="center">â€"8.3</td> <td align="center">93.62</td> </tr> <tr> <td align="left">GHRP-2 (Contini, <a href="#CIT0010">2020</a>)</td> <td align="center">â€"8.1</td> <td align="center">95.49</td> </tr> <tr> <td align="left">Indinavir (Contini, <a href="#CIT0010">2020</a>)</td> <td align="center">â€"8</td> <td align="center">88.35</td> </tr> <tr> <td align="left">Cobicistat (Contini, <a href="#CIT0010">2020</a>)</td> <td align="center">â€"7.5</td> <td align="center">86.53</td> </tr> <tr> <td align="left">Angiotensin II (Contini, <a href="#CIT0010">2020</a>)</td> <td align="center">â€"6.9</td> <td align="center">89.85</td> </tr> </tbody> </table> </div> </div> <div class="molecularinteractionsoftheselecteddrugswiththemainprotease" title="sec"><span class="label" tagx="label" title="label">3.2.</span><div class="title" tagx="title" title="title">
Molecular interactions of the selected drugs with the main protease</div> <p>Noncovalent interactions of the selected drugs which are detected by Autodock Vina reveals that all of them interact with either both catalytic residues (His41 and Cys145) or at least one of them, as shown in <a href="#F0003">Figure 3</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>Nonbonding interactions of top drug candidates with the main protease of SARS-CoV-2 (pose predicted by AutoDock Vina). (a) Simeprevir, (b) Ergotamine, (c) Bromocriptine, and (d) Tadalafil.</p>   </div> <p>Simeprevir, Ergotamine, Bromocriptine and Tadalafil form a number of hydrogen bonds and hydrophobic interactions with the protein (noncovalent interactions detected by GOLD are reported in <b>(</b><a href="#t0002">Table S2</a>, supporting information). Among them, most of the interactions with the active site residues are categorized as hydrophobic (<a href="#t0002">Table 2</a>). Both Simeprevir and Ergotamine form a hydrogen bond with at least one of the catalytic residues.</p> <div class="table-wrap_UNKNOWN" id="t0002" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2.</span> <p>Noncovalent interactions of selected four drugs with main protease of SARS-CoV-2 (pose predicted by AutoDock Vina) where, Hâ€‰=â€‰Hydrogen bond, CHâ€‰=â€‰Conventional Hydrogen bond, Câ€‰=â€‰Carbon Hydrogen bond.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Interacting residue</th> <th align="center">Distance</th> <th align="center">Bond category</th> <th align="center">Bond type</th> </tr> </thead> <tbody valign="top"> <tr> <td colspan="4"><b>Simeprevir</b></td> </tr> <tr> <td align="left">CYS145</td> <td align="char" char=".">2.7202</td> <td align="left">H</td> <td align="left">CH</td> </tr> <tr> <td align="left">GLN189</td> <td align="char" char=".">2.29</td> <td align="left">H</td> <td align="left">CH</td> </tr> <tr> <td align="left">GLY143</td> <td align="char" char=".">2.19455</td> <td align="left">H</td> <td align="left">CH</td> </tr> <tr> <td align="left">THR190</td> <td align="char" char=".">3.81766</td> <td align="left">Hydrophobic</td> <td align="left">Amide-Pi Stacked</td> </tr> <tr> <td align="left">PRO168</td> <td align="char" char=".">4.48443</td> <td align="left">Hydrophobic</td> <td align="left">Alkyl</td> </tr> <tr> <td align="left">LEU50</td> <td align="char" char=".">5.06348</td> <td align="left">Hydrophobic</td> <td align="left">Alkyl</td> </tr> <tr> <td align="left">LEU50</td> <td align="char" char=".">4.94019</td> <td align="left">Hydrophobic</td> <td align="left">Alkyl</td> </tr> <tr> <td align="left">PRO168</td> <td align="char" char=".">4.58666</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Alkyl</td> </tr> <tr> <td align="left">ALA191</td> <td align="char" char=".">4.47775</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Alkyl</td> </tr> <tr> <td align="left">PRO168</td> <td align="char" char=".">3.91598</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Alkyl</td> </tr> <tr> <td colspan="4"><b>Ergotamine</b></td> </tr> <tr> <td align="left">HIS41</td> <td align="char" char=".">2.88425</td> <td align="left">H</td> <td align="left">C</td> </tr> <tr> <td align="left">MET165</td> <td align="char" char=".">1.89419</td> <td align="left">H</td> <td align="left">C</td> </tr> <tr> <td align="left">MET165</td> <td align="char" char=".">4.81695</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Alkyl</td> </tr> <tr> <td align="left">HIS41</td> <td align="char" char=".">4.81113</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Pi-T Shaped</td> </tr> <tr> <td align="left">MET165</td> <td align="char" char=".">5.65835</td> <td align="left">Other</td> <td align="left">Pi-sulfur</td> </tr> <tr> <td colspan="4"><b>Bromocriptine</b></td> </tr> <tr> <td align="left">GLY143</td> <td align="char" char=".">2.32271</td> <td align="left">H</td> <td align="left">CH</td> </tr> <tr> <td align="left">GLY143</td> <td align="char" char=".">1.86614</td> <td align="left">H</td> <td align="left">CH</td> </tr> <tr> <td align="left">ARG188</td> <td align="char" char=".">2.28318</td> <td align="left">H</td> <td align="left">CH</td> </tr> <tr> <td align="left">ASN142</td> <td align="char" char=".">2.55112</td> <td align="left">H</td> <td align="left">C</td> </tr> <tr> <td align="left">GLY143</td> <td align="char" char=".">2.91712</td> <td align="left">H</td> <td align="left">C</td> </tr> <tr> <td align="left">GLN189</td> <td align="char" char=".">1.99597</td> <td align="left">H</td> <td align="left">C</td> </tr> <tr> <td align="left">LEU27</td> <td align="char" char=".">3.98094</td> <td align="left">Hydrophobic</td> <td align="left">Alkyl</td> </tr> <tr> <td align="left">MET165</td> <td align="char" char=".">5.33935</td> <td align="left">Hydrophobic</td> <td align="left">Alkyl</td> </tr> <tr> <td align="left">LEU27</td> <td align="char" char=".">4.43789</td> <td align="left">Hydrophobic</td> <td align="left">Alkyl</td> </tr> <tr> <td align="left">CYS145</td> <td align="char" char=".">3.95427</td> <td align="left">Hydrophobic</td> <td align="left">Alkyl</td> </tr> <tr> <td align="left">MET49</td> <td align="char" char=".">4.83683</td> <td align="left">Hydrophobic</td> <td align="left">Alkyl</td> </tr> <tr> <td align="left">HIS41</td> <td align="char" char=".">5.27854</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Alkyl</td> </tr> <tr> <td align="left">MET165</td> <td align="char" char=".">4.15194</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Alkyl</td> </tr> <tr> <td align="left">MET165</td> <td align="char" char=".">4.68951</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Alkyl</td> </tr> <tr> <td align="left">GLN189</td> <td align="char" char=".">2.38092</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Sigma</td> </tr> <tr> <td align="left">HIS41</td> <td align="char" char=".">5.13653</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Pi-T Shaped</td> </tr> <tr> <td colspan="4"><b>Tadalafil</b></td> </tr> <tr> <td align="left">ASN142</td> <td align="char" char=".">2.16312</td> <td align="left">H</td> <td align="left">C</td> </tr> <tr> <td align="left">GLY143</td> <td align="char" char=".">2.66564</td> <td align="left">H</td> <td align="left">C</td> </tr> <tr> <td align="left">MET165</td> <td align="char" char=".">2.64728</td> <td align="left">H</td> <td align="left">C</td> </tr> <tr> <td align="left">CYS145</td> <td align="char" char=".">5.22621</td> <td align="left">Hydrophobic</td> <td align="left">Alkyl</td> </tr> <tr> <td align="left">MET49</td> <td align="char" char=".">5.50154</td> <td align="left">Other</td> <td align="left">Pi-Sulfur</td> </tr> <tr> <td align="left">MET49</td> <td align="char" char=".">5.03037</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Alkyl</td> </tr> <tr> <td align="left">CYS145</td> <td align="char" char=".">5.47403</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Alkyl</td> </tr> <tr> <td align="left">MET49</td> <td align="char" char=".">4.6017</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Alkyl</td> </tr> <tr> <td align="left">HIS41</td> <td align="char" char=".">5.18874</td> <td align="left">Hydrophobic</td> <td align="left">Pi-Alkyl</td> </tr> </tbody> </table> </div> <p>Simeprevir is observed to show three hydrophobic interactions with Pro168, two with Leu50 and one with each of the Thr190 and Ala191. Ergotamine exhibits one piâ€"alkyl and one piâ€"pi T-shaped hydrophobic interaction with Met165 and His41, respectively. Among all the proteinâ€"drug complexes, Bromocriptine demonstrates the most noncovalent interactions with the main protease. It shows two hydrophobic interactions with His41, two with Leu27, three with Met165 and one with each of the Met49, Cys145 and Gln189 residues. Tadalafil exhibits hydrophobic interactions with both catalytic residues. Both Ergotamine and Tadalafil show one piâ€"sulfur interaction with Met165 and Met49 residues, respectively.</p> </div> <div class="mdsimulation" title="sec"><span class="label" tagx="label" title="label">3.3.</span><div class="title" tagx="title" title="title">
MD simulation</div> <p>MD simulation for all complexes of the main protease (6LU7) with the top four selected drugs and apoâ€"form is performed for 100â€‰ns. RMSD of alpha carbon atoms, RMSF of all amino acid residues, solvent accessible surface area, radius of gyration and the number of hydrogen bonds of all the drugâ€"protein complexes and apoâ€"Mpro are investigated.</p> <p>The RMSD of Simeprevirâ€"Mpro and Ergotamineâ€"Mpro are stabilized after 20â€‰ns of simulation, with an average value of 1.81â€‰Â±â€‰0.30 Ã… and 1.90â€‰Â±â€‰0.32 Ã…, respectively (<a href="#t0003">Table 3</a>). <a href="#F0004">Figure 4a</a> reveals that Bromocriptineâ€"Mpro follows a similar trend in MD trajectory like apoâ€"Mpro. Tadalafilâ€"Mpro complex shows fluctuation until 30â€‰ns, while it reaches a peak of 3.01 Ã… at 26.8â€‰ns. It exhibits structural stability after 30â€‰ns of MD simulation. These drugâ€"Mpro complexes exhibit better stability in the RMSD trajectory than the control drugs. Unstable nature is observed for the RMSD of Remdesivirâ€"Mpro after 70â€‰ns of MD simulation, while the Lopinavirâ€"Mpro complex followed the similar trend like the Bromocriptineâ€"Mpro complex. <a href="#F0004">Figure 4b</a> displays root-mean-square fluctuation as a function of the residue number of the apoâ€"Mpro and drugâ€"Mpro complexes. This is explored to understand how the binding of drug molecules changes the behavior of the amino acid residues of the protein. A low RMSF is observed for the apoâ€"Mpro which suggests low flexibility of protein even without a ligand. It also shows flexibility at Ser46 to Asn53, Tyr154, Val303, Thr304, Phe305 and Gln306. The residues Arg188 to Asp197 show high flexibility in Tadalafilâ€"Mpro complex. RMSF of Simeprevirâ€"Mpro, Ergotamineâ€"Mpro, Bromocriptineâ€"Mpro, Remdesivirâ€"Mpro and Lopinavirâ€"Mpro complexes are almost similar in most areas.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>Analysis of RMSD, RMSF, SASA, <i>R</i><sub>g</sub> and total number of hydrogen bonds of apoâ€"Mpro and selected four drugâ€"protein complexes at 100â€‰ns. (a) Root-mean-square deviation (RMSD, Ã…) of the CÎ± atoms over the phase of 100â€‰ns, (b) RMSF values of the alpha carbon over the entire simulation, where the ordinate is RMSF (Ã…), (c) Solvent accessible surface area (SASA), (d) Radius of gyration (<i>R</i><sub>g</sub>) over the entire simulation, (e) Total H-bond count throughout the simulation and (f) Binding Free Energy during the last 50â€‰ns of simulation.</p>   </div> <p>The radius of gyration (<i>R</i><sub>g</sub>) is the root mean square distance of various atoms of a protein from the axis of rotation, and it illustrates structural compactness. A greater change in <i>R</i><sub>g</sub> value may occur due to the folding or conformational changes of protein structure, while a lower degree of fluctuation over the simulation period indicates rigidity and higher compactness of a structure. According to the observed <i>R</i><sub>g</sub> value, the apoâ€"Mpro and Simeprevirâ€"Mpro complex showed almost similar structural compactness over the phase of the 100â€‰ns simulation. The apoâ€"Mpro complex experienced a fluctuation in <i>R</i><sub>g</sub> value during 80â€"90â€‰ns. Ergotamine showed structural compactness all over the simulation time with an average of 22.32â€‰Â±â€‰0.11 Ã…. Bromocriptineâ€"Mpro followed a similar trend like the Remdesivirâ€"Mpro complex after 45â€‰ns. A loose packing is observed for Tadalafilâ€"Mpro during 60â€‰ns to 90â€‰ns. Except for this time frame, Tadalafilâ€"Mpro and Lopinavirâ€"Mpro followed a similar trend in MD trajectory. Based on the overall <i>R</i><sub>g</sub>, the average values are 22.35â€‰Â±â€‰0.14 Ã…, 22.39â€‰Â±â€‰0.14 Ã…, 22.20â€‰Â±â€‰0.13 Ã…, 22.57â€‰Â±â€‰0.24 Ã…, 22.22â€‰Â±â€‰0.12 Ã… and 22.42â€‰Â±â€‰0.11 Ã… for apoâ€"Mpro, Simeprevirâ€"Mpro, Bromocriptineâ€"Mpro, Tadalafilâ€"Mpro, Remdesivirâ€"Mpro and Lopinavirâ€"Mpro complexes, respectively. To anticipate the solvent accessibility of all complexes, the solvent-accessible surface area (SASA) of apoâ€"Mpro and complexes are calculated (<a href="#F0004">Figure 4(c)</a>). Binding of a drug to the protein may impact its structural properties, thus, SASA may also change. A higher SASA value suggests the expansion of protein structure. A low fluctuation of SASA value is expected during the simulation. All the examined complexes exhibit a similar trend until 40â€‰ns of MD trajectory. Simeprevirâ€"Mpro and Ergotamineâ€"Mpro showed lower SASA value than the control drugâ€"Mpro complexes. The overall average SASA value suggests Lopinavir could induce protein expansion, and thus, increase the solvent accessible surface of the protein (<a href="#t0003">Table 3</a>).</p> <div class="table-wrap_UNKNOWN" id="t0003" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3.</span> <p>Average RMSD, SASA, <i>R</i><sub>g</sub>, number of hydrogen bonds and binding free energy of the selected drugsâ€"Mpro complexes.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Complex</th> <th align="center">RMSD (Ã…)</th> <th align="center">SASA (Ã…<sup>2</sup>)</th> <th align="center">Radius of gyration (Ã…)</th> <th align="center">Number of hydrogen bonds</th> <th align="center">Binding free energy (kcal/mol)</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">Apoâ€"Mpro</td> <td align="char" char="Â±">2.07â€‰Â±â€‰0.32</td> <td align="char" char="Â±">14137.41â€‰Â±â€‰217.54</td> <td align="char" char="Â±">22.35â€‰Â±â€‰0.14</td> <td align="char" char="Â±">510.2â€‰Â±â€‰11.80</td> <td align="char" char="Â±">â€"</td> </tr> <tr> <td align="left">Simeprevirâ€"Mpro</td> <td align="char" char="Â±">1.81â€‰Â±â€‰0.30</td> <td align="char" char="Â±">13889.76â€‰Â±â€‰295.20</td> <td align="char" char="Â±">22.39â€‰Â±â€‰0.14</td> <td align="char" char="Â±">502â€‰Â±â€‰12.72</td> <td align="char" char="Â±">â€"77.44â€‰Â±â€‰2.43</td> </tr> <tr> <td align="left">Ergotamineâ€"Mpro</td> <td align="char" char="Â±">1.90â€‰Â±â€‰0.32</td> <td align="char" char="Â±">13800.03â€‰Â±â€‰243.83</td> <td align="char" char="Â±">22.32â€‰Â±â€‰0.11</td> <td align="char" char="Â±">503.36â€‰Â±â€‰11.36</td> <td align="char" char="Â±">â€"26.33â€‰Â±â€‰0.39</td> </tr> <tr> <td align="left">Bromocriptineâ€"Mpro</td> <td align="char" char="Â±">2.07â€‰Â±â€‰0.36</td> <td align="char" char="Â±">14035.29â€‰Â±â€‰208.71</td> <td align="char" char="Â±">22.20â€‰Â±â€‰0.13</td> <td align="char" char="Â±">512.50â€‰Â±â€‰12.33</td> <td align="char" char="Â±">â€"33.69â€‰Â±â€‰0.11</td> </tr> <tr> <td align="left">Tadalafilâ€"Mpro</td> <td align="char" char="Â±">2.24â€‰Â±â€‰0.41</td> <td align="char" char="Â±">14197.53â€‰Â±â€‰390.24</td> <td align="char" char="Â±">22.57â€‰Â±â€‰0.24</td> <td align="char" char="Â±">510.86â€‰Â±â€‰14.66</td> <td align="char" char="Â±">â€"14.46â€‰Â±â€‰0.63</td> </tr> <tr> <td align="left">Remdesivirâ€"Mpro</td> <td align="char" char="Â±">1.89â€‰Â±â€‰0.36</td> <td align="char" char="Â±">13892.51â€‰Â±â€‰252.63</td> <td align="char" char="Â±">22.22â€‰Â±â€‰0.12</td> <td align="char" char="Â±">507.96â€‰Â±â€‰12.04</td> <td align="char" char="Â±">â€"0.31â€‰Â±â€‰0.06</td> </tr> <tr> <td align="left">Lopinavirâ€"Mpro</td> <td align="char" char="Â±">2.03â€‰Â±â€‰0.41</td> <td align="char" char="Â±">14168.63â€‰Â±â€‰226.90</td> <td align="char" char="Â±">22.42â€‰Â±â€‰0.11</td> <td align="char" char="Â±">508.66â€‰Â±â€‰12.10</td> <td align="char" char="Â±">â€"27.89â€‰Â±â€‰0.35</td> </tr> </tbody> </table> </div> <p>Since intermolecular hydrogen bonds between proteinâ€"drug complexes have a significant contribution to the conformational stability of the complex, the total number of hydrogen bonds is calculated (<a href="#F0004">Figure 4(e)</a>). The average number of hydrogen bonds observed in apoâ€"Mpro is 510.2â€‰Â±â€‰11.80. The Bromocriptineâ€"Mpro complex has the most hydrogen bonds over the 100â€‰ns simulation time (<a href="#t0003">Table 3</a>).</p> <p>Binding free energy calculations for each complex were conducted by employing the MM/PBSA method. The Simeprevirâ€"Mpro shows the highest average binding free energy of âˆ’77.44â€‰Â±â€‰2.43â€‰kcal/mol. All our selected drugâ€"Mpro complexes demonstrate better binding free energy than the Remdesivirâ€"Mpro complex (<a href="#F0004">Figure 4(f)</a><b>)</b>. The binding free energy of the Remdesivirâ€"Mpro complex persisted as positive most of the time, which indicates loose binding between the drug and the protein. The binding free energy of Ergotamineâ€"Mpro, Bromocriptineâ€"Mpro, Tadalafilâ€"Mpro and Lopinavirâ€"Mpro complexes remained negative most of the time, which also indicates good binding (<a href="#F0002">Figure S2</a>, supporting information). The average binding free energy was observed âˆ’26.33â€‰Â±â€‰0.39, âˆ’33.69â€‰Â±â€‰0.11, âˆ’14.46â€‰Â±â€‰0.63, âˆ’0.31â€‰Â±â€‰0.06 and âˆ’27.89â€‰Â±â€‰0.35â€‰kcal/mol for Ergotamine, Bromocriptine, Tadalafil, Remdesivir and Lopinavir, respectively.</p> <p>The snapshots of Mproâ€"drug complexes are visualized at different time intervals to explore the drugs pose during MD simulation. All our selected drugs stayed at the binding pocket of the protein over the phase of 100â€‰ns MD simulation (<a href="#F0005">Figure 5</a>). Additionally, all drugs interact with at least one of the catalytic residues during most of the time of MD simulation (Table S4, supporting information).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>Binding pose of drugs during 100â€‰ns MD simulation. The crystal structure of Mpro is shown in beige color with (a) Simeprevir (cyan), (b) Ergotamine (green), (c) Bromocriptine (blue) and (d) Tadalafil (orange red).</p>   </div> </div> <div class="principalcomponentsanalysis" title="sec"><span class="label" tagx="label" title="label">3.4.</span><div class="title" tagx="title" title="title">
Principal components analysis</div> <p>PCA is applied to observe the structural properties and energy profiles of apoâ€"Mpro and the selected four drugâ€"Mpro complexes. A single PCA model consisting of five training sets (Mpro and four drugâ€"Mpro complexes) was constructed for analysis. The first two PCs explained 91.74% of the total variance, where PC1 contributed 75.07% and PC2 contributed 16.67% of the variance. It is observed from the scores plot (<a href="#F0006">Figure 6(a)</a>) that the apoâ€"Mpro (violet) and Simeprevirâ€"Mpro complex (cyan) overlap with each other, thus, indicating similarity in energy and structural profile. Moreover, Ergotamineâ€"Mpro (green) and Tadalafilâ€"Mpro (orangeâ€"red) shifted slightly toward the right in the PC1 direction and resided very close to apoâ€"Mpro and Simeprevirâ€"Mpro complex. Thus, by forming a complex with Ergotamine and Tadalafil, Mpro showed a similar structural profile compared to Mpro alone. However, the Bromocriptineâ€"Mpro complex (blue) shifted significantly toward the positive direction of PC1. This indicates greater dissimilarity of Bromocriptineâ€"Mpro with apoâ€"Mpro and other complexes. The loading plot (<a href="#F0006">Figure 6(b)</a>) revealed that the variables are separated in the PC1 direction, which explains different cluster formation in the PC1 direction. Therefore, it is evident from the PCA scoring plot that Simeprevir has the best binding stability with Mpro, while Ergotamine and Tadalafil also demonstrate almost similar behavior in MD simulation.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <p>(a) The score plot presented five data clusters in different color, where each dot represents one time point. The clustering is attributable as: apoâ€"Mpro (violet), Simeprevirâ€"Mpro complex (cyan), Ergotamineâ€"Mpro (green), Bromocriptineâ€"Mpro (blue), Tadalafilâ€"Mpro (orange red) (b) loading plot from principal components analysis of the energy and structural data.</p>   </div> </div> <div class="structureâ€" title="sec"><span class="label" tagx="label" title="label">3.5.</span><div class="title" tagx="title" title="title">
Structureâ€"activity relationship</div> <p>SAR has been used for decades by researchers as a cheaper way to develop relationships between physiochemical properties of chemicals and their bioactivities (Santos-Filho etÂ al., <a href="#CIT0037">2009</a>; Verma etÂ al., <a href="#CIT0045">2010</a>). It is a widely used tool in bioinformatics, drug discovery for clinical research, pharmaceutical industry, petrochemical and agrochemical sectors for modeling and predictive pattern analysis (Alam &amp;amp; Khan, <a href="#CIT0002">2017</a>; Fakayode etÂ al., <a href="#CIT0016">2014</a>; Funar-Timofei etÂ al., <a href="#CIT0017">2017</a>). For further analysis, MLR has been employed. For instance, TPSA (Ã…<sup>2</sup>), molecular weight, XLogP3, H-bond donor count, H-bond acceptor count of the drugs are the most significant variables on QSAR contributors to the MLR model (<a href="#t0003">Table S3</a>, supporting information<b>)</b>.</p> <p>The drugs with similar functional groups are found to stay side by side on the score plot <b>(</b><a href="#F0007">Figure 7</a>). For example, drugs (D2, D3 and D5) containing indole ring, amide group and alcohol group are clustered together on the fourth quadrant of the score plot. On the other hand, drugs (D23, D7, D16, D13, D22 and D24) with diazole ring attached to the benzene ring are clustered on the second and third quadrants. The grouping on the score plot was highly significant.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7.</span></a></div>   <p>Score plot of PCA analysis for quantitative structuralâ€"activity relationship of drugs.</p>   </div> <p>The MLR model is generated and employed to predict the binding energy obtained from molecular docking. Additionally, a test set is utilized for the validation of drug candidates. The observed results of validation reveal similarity in the binding energies (<a href="#t0004">Table 4</a>).</p> <div class="table-wrap_UNKNOWN" id="t0004" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 4.</span> <p>Predicted binding energy by the MLR model and actual binding energy from molecular docking.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Sample</th> <th align="center">Predicted binding energy (kcal/mol)</th> <th align="center">Actual binding energy(kcal/mol)</th> <th align="center">%RE</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">D2</td> <td align="center">â€"8.89</td> <td align="center">â€"9.80</td> <td align="center">â€"2.99</td> </tr> <tr> <td align="left">D5</td> <td align="center">â€"8.92</td> <td align="center">â€"9.20</td> <td align="center">â€"0.91</td> </tr> <tr> <td align="left">D8</td> <td align="center">â€"8.90</td> <td align="center">â€"9.10</td> <td align="center">â€"0.66</td> </tr> <tr> <td align="left">D12</td> <td align="center">â€"8.76</td> <td align="center">â€"8.80</td> <td align="center">â€"0.13</td> </tr> <tr> <td align="left">D15</td> <td align="center">â€"8.99</td> <td align="center">â€"8.70</td> <td align="center">0.98</td> </tr> <tr> <td align="left">D20</td> <td align="center">â€"9.03</td> <td align="center">â€"8.50</td> <td align="center">1.77</td> </tr> <tr> <td align="left">D24</td> <td align="center">â€"8.68</td> <td align="center">â€"8.40</td> <td align="center">0.92</td> </tr> <tr> <td align="left">D26</td> <td align="center">â€"8.62</td> <td align="center">â€"8.30</td> <td align="center">1.07</td> </tr> <tr> <td align="left">D28</td> <td align="center">â€"9.02</td> <td align="center">â€"8.20</td> <td align="center">2.72</td> </tr> <tr> <td align="left">D29</td> <td align="center">â€"8.77</td> <td align="center">â€"8.20</td> <td align="center">1.90</td> </tr> <tr> <td align="left">D31</td> <td align="center">â€"8.64</td> <td align="center">â€"7.50</td> <td align="center">3.79</td> </tr> <tr> <td align="left"><b><i>RMS%RE</i></b></td> <td align="center">Â </td> <td align="center">Â </td> <td align="center"><b><i>1.95</i></b></td> </tr> </tbody> </table> </div> <p>The ability of the MLR model to accurately anticipate the binding energy of the validated drugs is evaluated by the root-mean-square-relative percent-error (RMS%RE). The result obtained from MLR is 82% accurate while comparing to the binding energy. Yet, findings from this analysis can help researchers predict the drug performances against the main protease considering the binding energy, chemical behavior and pharmacological efficacy of upcoming drug candidates.</p> </div> </div> <div class="conclusion" title="sec"><span class="label" tagx="label" title="label">4.</span><div class="title" tagx="title" title="title">
Conclusion</div> <p>In this study, several computational and statistical tools such as molecular docking, MD simulation, SAR and PCA are used to detect the best existing drugs against the main protease of SARS-CoV-2. Among the 1615 studied drugs, Simeprevir, Ergotamine, Bromocriptine and Tadalafil are found to have the highest binding affinity. As these drugs are already certified for human use, they do not require long-term clinical trials. These drugs show a significant number of noncovalent interactions including hydrogen bond, hydrophobic interaction and electrostatic interaction with the binding site residues of Mpro. All the selected drugs interact either with both (His41 and Cys145) or at least one of the catalytic residues. Furthermore, MD simulations exhibit that Simeprevir, Ergotamine and Bromocriptine drugs with Mpro protein preserve the structural stability over the simulation period. Additionally, the MM/PBSA free energy profiles of these drugs suggest strong binding with the receptor. SAR pattern recognition reveals that drugs containing an indole ring, amide group and alcohol group are clustered together in the fourth quadrant and drugs containing an azole ring attached to benzene ring are assembled together in the second and third quadrant of the scoring plot. The predicted binding affinity values from MLR shows 82% accuracy compared to values obtained from molecular docking. It can be concluded that the selected drugs are promising and can be used to design effective drugs against the SARS-CoV-2.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="TBSD_A_1794974_SM5610.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We are grateful to our donors (<a href="http://grc-bd.org/donate/">http://grc-bd.org/donate/</a>) who supported the building of the computational platform in Bangladesh. The authors acknowledge the World Academy of Science (TWAS) for purchasing the High-Performance computer for the MD simulations.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Disclosure statement</div> <p>No potential conflict of interest was reported by the authors.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CIT0700" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: Adinsoft, S. (</span></span><span class="year" tagx="year" title="year">2010</span>). XLSTAT-software, version 10. Addinsoft.</span></li> <li tag="ref"><a name="CIT0001" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shahinozzaman</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fakayode</span><span class="given-names" tagx="given-names" title="given-names">S. O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Afrin</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halim</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Virtual screening, molecular dynamics, density functional theory and quantitative structure activity relationship studies to design peroxisome proliferator-activated receptor-Î³ agonists as anti-diabetic drugs</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">15</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1714482">10.1080/07391102.2020.1714482</a></span></span></li> <li tag="ref"><a name="CIT0002" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alam</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">3D-QSAR studies on maslinic acid analogs for anticancer activity against breast cancer cell line</span>. <span class="source" tagx="source" title="source"><i>Scientific Reports</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">13</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-06131-0">10.1038/s41598-017-06131-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28144038">28144038</a></span></span></li> <li tag="ref"><a name="CIT0003" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">N. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sept</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joseph</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holst</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCammon</span><span class="given-names" tagx="given-names" title="given-names">J. A.</span></span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Electrostatics of nanosystems: Application to microtubules and the ribosome</span>. <span class="source" tagx="source" title="source"><i>Proceedings of the National Academy of Sciences of the United States of America</i></span>, (<span class="issue" tagx="issue" title="issue">18</span>), <span class="fpage" tagx="fpage" title="fpage">10037</span>â€"<span class="lpage" tagx="lpage" title="lpage">10041</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.181342398">10.1073/pnas.181342398</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11517324">11517324</a></span></span></li> <li tag="ref"><a name="CIT0004" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enria</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giesecke</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heymann</span><span class="given-names" tagx="given-names" title="given-names">D. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ihekweazu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobinger</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lane</span><span class="given-names" tagx="given-names" title="given-names">H. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Memish</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oh</span><span class="given-names" tagx="given-names" title="given-names">M.-D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sall</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schuchat</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ungchusak</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wieler</span><span class="given-names" tagx="given-names" title="given-names">L. H.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">COVID-19: Towards controlling of a pandemic</span>. <span class="source" tagx="source" title="source"><i>The Lancet</i></span>, (<span class="issue" tagx="issue" title="issue">10229</span>), <span class="fpage" tagx="fpage" title="fpage">1015</span>â€"<span class="lpage" tagx="lpage" title="lpage">1018</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30673-5">10.1016/S0140-6736(20)30673-5</a></span></span></li> <li tag="ref"><a name="CIT0005" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">J. F.-W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kok</span><span class="given-names" tagx="given-names" title="given-names">K.-H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">To</span><span class="given-names" tagx="given-names" title="given-names">K. K.-W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">K.-Y.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan</span>. <span class="source" tagx="source" title="source"><i>Emerging Microbes &amp;amp; Infections</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">221</span>â€"<span class="lpage" tagx="lpage" title="lpage">236</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/22221751.2020.1719902">10.1080/22221751.2020.1719902</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31987001">31987001</a></span></span></li> <li tag="ref"><a name="CIT0006" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Only one protomer is active in the dimer of SARS 3C-like proteinase</span>. <span class="source" tagx="source" title="source"><i>The Journal of Biological Chemistry</i></span>, (<span class="issue" tagx="issue" title="issue">20</span>), <span class="fpage" tagx="fpage" title="fpage">13894</span>â€"<span class="lpage" tagx="lpage" title="lpage">13898</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M510745200">10.1074/jbc.M510745200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16565086">16565086</a></span></span></li> <li tag="ref"><a name="CIT0007" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Emerging coronaviruses: Genome structure, replication, and pathogenesis</span>. <span class="source" tagx="source" title="source"><i>Journal of Medical Virology</i></span>, (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">418</span>â€"<span class="lpage" tagx="lpage" title="lpage">423</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25681">10.1002/jmv.25681</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31967327">31967327</a></span></span></li> <li tag="ref"><a name="CIT0008" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chirag</span><span class="given-names" tagx="given-names" title="given-names">N. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prasanth Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Himanshu</span><span class="given-names" tagx="given-names" title="given-names">A. P.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rakesh</span><span class="given-names" tagx="given-names" title="given-names">M. R.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). Identification of potential binders of the SARS-Cov-2 spike protein via molecular docking, dynamics simulation and binding free energy calculation. <i>chemRxiv</i><span class="pub-id"><a href="https://dx.doi.org/10.26434/chemrxiv.12278588.v1">10.26434/chemrxiv.12278588.v1</a></span></span></li> <li tag="ref"><a name="CIT0010" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Contini</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). Virtual screening of an FDA approved drugs database on two COVID-19 coronavirus proteins. <a href="https://chemrxiv.org/articles/Virtual_Screening_of_an_FDA_Approved_Drugs_Database_on_Two_COVID-19_Coronavirus_Proteins/11847381/files/21713028.pdf">https://chemrxiv.org/articles/Virtual_Screening_of_an_FDA_Approved_Drugs_Database_on_Two_COVID-19_Coronavirus_Proteins/11847381/files/21713028.pdf</a></span></li> <li tag="ref"><a name="CIT0011" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dallakyan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olson</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Small-molecule library screening by docking with PyRx</span>. <span class="source" tagx="source" title="source"><i>Methods in Molecular Biology (Clifton, N.J.)</i></span>, , <span class="fpage" tagx="fpage" title="fpage">243</span>â€"<span class="lpage" tagx="lpage" title="lpage">250</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-4939-2269-7_19">10.1007/978-1-4939-2269-7_19</a></span></span></li> <li tag="ref"><a name="CIT0012" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Sousa</span><span class="given-names" tagx="given-names" title="given-names">A. C. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Combrinck</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maepa</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Egan</span><span class="given-names" tagx="given-names" title="given-names">T. J.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Virtual screening as a tool to discover new Î²-haematin inhibitors with activity against malaria parasites</span>. <span class="source" tagx="source" title="source"><i>Scientific Reports</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">3374</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-020-60221-0">10.1038/s41598-020-60221-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32099045">32099045</a></span></span></li> <li tag="ref"><a name="CIT0013" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dickson</span><span class="given-names" tagx="given-names" title="given-names">C. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madej</span><span class="given-names" tagx="given-names" title="given-names">B. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skjevik</span><span class="given-names" tagx="given-names" title="given-names">Ã…. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Betz</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teigen</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gould</span><span class="given-names" tagx="given-names" title="given-names">I. R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">R. C.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Lipid14: The amber lipid force field</span>. <span class="source" tagx="source" title="source"><i>Journal of Chemical Theory and Computation</i></span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">865</span>â€"<span class="lpage" tagx="lpage" title="lpage">879</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ct4010307">10.1021/ct4010307</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24803855">24803855</a></span></span></li> <li tag="ref"><a name="CIT0014" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Anti-HCV, nucleotide inhibitors, repurposing against COVID-19</span>. <span class="source" tagx="source" title="source"><i>Life Sciences</i></span>, , <span class="fpage" tagx="fpage" title="fpage">117477</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.lfs.2020.117477">10.1016/j.lfs.2020.117477</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32119961">32119961</a></span></span></li> <li tag="ref"><a name="CIT0015" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fakayode</span><span class="given-names" tagx="given-names" title="given-names">S. O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Busch</span><span class="given-names" tagx="given-names" title="given-names">K. W.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Busch</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Chemometric approach to chiral recognition using molecular spectroscopy</span>. <span class="source" tagx="source" title="source"><i>Encyclopedia of Analytical Chemistry</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/9780470027318.a9082">10.1002/9780470027318.a9082</a></span></span></li> <li tag="ref"><a name="CIT0016" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fakayode</span><span class="given-names" tagx="given-names" title="given-names">S. O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">B. S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pollard</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Determination of boiling point of petrochemicals by gas chromatography-mass spectrometry and multivariate regression analysis of structural activity relationship</span>. <span class="source" tagx="source" title="source"><i>Talanta</i></span>, , <span class="fpage" tagx="fpage" title="fpage">151</span>â€"<span class="lpage" tagx="lpage" title="lpage">156</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.talanta.2014.03.037">10.1016/j.talanta.2014.03.037</a></span></span></li> <li tag="ref"><a name="CIT0600" /><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: Frisch, M. J., Trucks, G. W., Schlegel, H. B., Scuseria, G. E., Robb, M. A., Cheeseman, J. R., Scalmani, G., Barone, V., Mennucci, B., &amp;amp; Petersson, G. A. (2009). Gaussian 09, A.02. Gaussian, Inc.</span></span></span></li> <li tag="ref"><a name="CIT0017" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Funar-Timofei</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borota</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crisan</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Combined molecular docking and QSAR study of fused heterocyclic herbicide inhibitors of D1 protein in photosystem II of plants</span>. <span class="source" tagx="source" title="source"><i>Molecular Diversity</i></span>, (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">437</span>â€"<span class="lpage" tagx="lpage" title="lpage">454</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11030-017-9735-x">10.1007/s11030-017-9735-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28299584">28299584</a></span></span></li> <li tag="ref"><a name="CIT0018" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorbalenya</span><span class="given-names" tagx="given-names" title="given-names">A. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">S. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baric</span><span class="given-names" tagx="given-names" title="given-names">R. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Groot</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gulyaeva</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haagmans</span><span class="given-names" tagx="given-names" title="given-names">B. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauber</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leontovich</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neuman</span><span class="given-names" tagx="given-names" title="given-names">B. W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Penzar</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">L. L. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samborskiy</span><span class="given-names" tagx="given-names" title="given-names">D. V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sidorov</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sola</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ziebuhr</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">The species Severe acute respiratory syndrome-related coronavirus: Classifying 2019-nCoV and naming it SARS-CoV-2</span>. <span class="source" tagx="source" title="source"><i>Nature Microbiology</i></span>, (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">536</span>â€"<span class="lpage" tagx="lpage" title="lpage">544</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41564-020-0695-z">10.1038/s41564-020-0695-z</a></span></span></li> <li tag="ref"><a name="CIT0019" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilgenfeld</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peiris</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">From SARS to MERS: 10 years of research on highly pathogenic human coronaviruses</span>. <span class="source" tagx="source" title="source"><i>Antiviral Research</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">286</span>â€"<span class="lpage" tagx="lpage" title="lpage">295</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2013.08.015">10.1016/j.antiviral.2013.08.015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24012996">24012996</a></span></span></li> <li tag="ref"><a name="CIT0020" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, â€¦ <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan</span>. <span class="source" tagx="source" title="source"><i>The <i>Lancet</i></i></span>, (<span class="issue" tagx="issue" title="issue">10223</span>), <span class="fpage" tagx="fpage" title="fpage">497</span>â€"<span class="lpage" tagx="lpage" title="lpage">506</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30183-5">10.1016/S0140-6736(20)30183-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32164906">32164906</a></span></span></li> <li tag="ref"><a name="CIT0021" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irwin</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shoichet</span><span class="given-names" tagx="given-names" title="given-names">B. K.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">ZINC-a free database of commercially available compounds for virtual screening</span>. <span class="source" tagx="source" title="source"><i>Journal of Chemical Information and Modeling</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">177</span>â€"<span class="lpage" tagx="lpage" title="lpage">182</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ci049714+">10.1021/ci049714+</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15667143">15667143</a></span></span></li> <li tag="ref"><a name="CIT0022" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parves</span><span class="given-names" tagx="given-names" title="given-names">M. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossain</span><span class="given-names" tagx="given-names" title="given-names">M. N.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halim</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Prediction of deleterious non-synonymous SNPs of human STK11 gene by combining algorithms, molecular docking, and molecular dynamics simulation</span>. <span class="source" tagx="source" title="source"><i>Scientific Reports</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">16426</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-019-52308-0">10.1038/s41598-019-52308-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31712642">31712642</a></span></span></li> <li tag="ref"><a name="CIT0023" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parves</span><span class="given-names" tagx="given-names" title="given-names">M. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">A. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uddin</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahman</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mamun</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>, Hossain, Md. <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Md. Ali</span><span class="given-names" tagx="given-names" title="given-names">N.A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halim</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">A molecular modeling approach to identify effective antiviral phytochemicals against the main protease of SARS-CoV-2</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">12</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1761883">10.1080/07391102.2020.1761883</a></span></span></li> <li tag="ref"><a name="CIT0024" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">You</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, â€¦ <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Structure of M<sup>pro</sup> from COVID-19 virus and discovery of its inhibitors</span>. <span class="source" tagx="source" title="source"><i>BioRxiv</i></span>, <span class="volume" tagx="volume" title="volume">2020</span>02.26.964882. <span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.02.26.964882">10.1101/2020.02.26.964882</a></span></span></li> <li tag="ref"><a name="CIT0025" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Willett</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glen</span><span class="given-names" tagx="given-names" title="given-names">R. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leach</span><span class="given-names" tagx="given-names" title="given-names">A. R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Development and validation of a genetic algorithm for flexible docking 1 1Edited by F. E. Cohen</span>. <span class="source" tagx="source" title="source"><i>Journal of Molecular Biology</i></span>, (<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">727</span>â€"<span class="lpage" tagx="lpage" title="lpage">748</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1006/jmbi.1996.0897">10.1006/jmbi.1996.0897</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9126849">9126849</a></span></span></li> <li tag="ref"><a name="CIT0026" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kassambara</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Practical guide to principal component methods in R. </span><span class="source" tagx="source" title="source"><i>STHDA</i></span>, , 1â€"170. CreateSpace Independent Publishing Platform.</span></li> <li tag="ref"><a name="CIT0027" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krieger</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elmar</span><span class="given-names" tagx="given-names" title="given-names">G. V.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spronk</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). YASARAâ€"Yet another scientific artificial reality application. <i>YASARA.Org</i>.</span></li> <li tag="ref"><a name="CIT0028" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krieger</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">J. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spronk</span><span class="given-names" tagx="given-names" title="given-names">C. A. E. M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vriend</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Fast empirical pKa prediction by Ewald summation</span><span class="source" tagx="source" title="source"><i>. <i>Journal of Molecular Graphics and Modelling</i></i></span>, (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">481</span>â€"<span class="lpage" tagx="lpage" title="lpage">486</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jmgm.2006.02.009">10.1016/j.jmgm.2006.02.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16644253">16644253</a></span></span></li> <li tag="ref"><a name="CIT0029" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krieger</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vriend</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">New ways to boost molecular dynamics simulations</span>. <span class="source" tagx="source" title="source"><i>Journal of Computational Chemistry</i></span>, (<span class="issue" tagx="issue" title="issue">13</span>), <span class="fpage" tagx="fpage" title="fpage">996</span>â€"<span class="lpage" tagx="lpage" title="lpage">1007</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.23899">10.1002/jcc.23899</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25824339">25824339</a></span></span></li> <li tag="ref"><a name="CIT0030" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhan</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, â€¦ <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: Implications for virus origins and receptor binding</span>. <span class="source" tagx="source" title="source"><i>The Lancet</i></span>, (<span class="issue" tagx="issue" title="issue">10224</span>), <span class="fpage" tagx="fpage" title="fpage">565</span>â€"<span class="lpage" tagx="lpage" title="lpage">574</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30251-8">10.1016/S0140-6736(20)30251-8</a></span></span></li> <li tag="ref"><a name="CIT0031" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mark</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Workman</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Chemometrics in spectroscopy</span> In <span class="source" tagx="source" title="source"><i>Chemometrics in spectroscopy</i></span>. Elsevier. <span class="pub-id"><a href="https://dx.doi.org/10.1016/B978-0-12-374024-3.X5000-4">10.1016/B978-0-12-374024-3.X5000-4</a></span></span></li> <li tag="ref"><a name="CIT0032" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martens</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naes</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">1998</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Multivariate calibration</span> In <span class="source" tagx="source" title="source"><i>Data handling in science and technology</i></span> (Vol. <span class="volume" tagx="volume" title="volume">20</span>). Wiley. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0922-3487(98)80046-4">10.1016/S0922-3487(98)80046-4</a></span></span></li> <li tag="ref"><a name="CIT0033" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Massova</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kollman</span><span class="given-names" tagx="given-names" title="given-names">P. A.</span></span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Combined molecular mechanical and continuum solvent approach (MM- PBSA/GBSA) to predict ligand binding</span>. <span class="source" tagx="source" title="source"><i>Perspectives in Drug Discovery and Design</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">113</span>â€"<span class="lpage" tagx="lpage" title="lpage">135</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1023/A:1008763014207">10.1023/A:1008763014207</a></span></span></li> <li tag="ref"><a name="CIT0036" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paasche</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zipper</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchÃ¤fer</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ziebuhr</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schirmeister</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Engels</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Evidence for substrate binding-induced zwitterion formation in the catalytic Cys-His dyad of the SARS-CoV main protease</span>. <span class="source" tagx="source" title="source"><i>Biochemistry</i></span>, (<span class="issue" tagx="issue" title="issue">37</span>), <span class="fpage" tagx="fpage" title="fpage">5930</span>â€"<span class="lpage" tagx="lpage" title="lpage">5946</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/bi400604t">10.1021/bi400604t</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25196915">25196915</a></span></span></li> <li tag="ref"><a name="CIT0037" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santos-Filho</span><span class="given-names" tagx="given-names" title="given-names">O. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hopfinger</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cherkasov</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Alencastro</span><span class="given-names" tagx="given-names" title="given-names">R. B.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">The receptor-dependent QSAR paradigm: An overview of the current state of the art</span>. <span class="source" tagx="source" title="source"><i>Medicinal Chemistry (Shariqah (United Arab Emirates))</i></span>, (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">359</span>â€"<span class="lpage" tagx="lpage" title="lpage">366</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2174/157340609788681458">10.2174/157340609788681458</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19689393">19689393</a></span></span></li> <li tag="ref"><a name="CIT0038" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sekhar</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <i>Virtual screening based prediction of potential drugs for COVID-19</i><span class="pub-id"><a href="https://dx.doi.org/10.20944/preprints202002.0418.v2">10.20944/preprints202002.0418.v2</a></span></span></li> <li tag="ref"><a name="CIT0039" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sexton</span><span class="given-names" tagx="given-names" title="given-names">N. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">E. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blanc</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vignuzzi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peersen</span><span class="given-names" tagx="given-names" title="given-names">O. B.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Denison</span><span class="given-names" tagx="given-names" title="given-names">M. R.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Homology-based identification of a mutation in the coronavirus RNA-dependent RNA polymerase that confers resistance to multiple mutagens</span>. <span class="source" tagx="source" title="source"><i>Journal of Virology</i></span>, (<span class="issue" tagx="issue" title="issue">16</span>), <span class="fpage" tagx="fpage" title="fpage">7415</span>â€"<span class="lpage" tagx="lpage" title="lpage">7428</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00080-16">10.1128/JVI.00080-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27279608">27279608</a></span></span></li> <li tag="ref"><a name="CIT0040" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaghaghi</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular docking study of novel COVID-19 protease with low risk terpenoides compounds of plants</span>. <span class="source" tagx="source" title="source"><i>ChemRxiv</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>). <span class="pub-id"><a href="https://dx.doi.org/10.26434/chemrxiv.11935722.v1">10.26434/chemrxiv.11935722.v1</a></span></span></li> <li tag="ref"><a name="CIT0041" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sola</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">AlmazÃ¡n</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ZÃºÃ±iga</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enjuanes</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Continuous and discontinuous RNA synthesis in coronaviruses</span>. <span class="source" tagx="source" title="source"><i>Annual Review of Virology</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">265</span>â€"<span class="lpage" tagx="lpage" title="lpage">288</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-virology-100114-055218">10.1146/annurev-virology-100114-055218</a></span></span></li> <li tag="ref"><a name="CIT0042" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Studio</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="source" tagx="source" title="source"><i>Dassault systemes BIOVIA, Discovery studio modelling environment, Release 4.5</i></span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Accelrys Software Inc</span>.</span></li> <li tag="ref"><a name="CIT0044" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ton</span><span class="given-names" tagx="given-names" title="given-names">A.-T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gentile</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsing</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ban</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cherkasov</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Rapid identification of potential inhibitors of SARS-CoV-2 main protease by deep docking of 1.3 billion compounds</span>. <span class="source" tagx="source" title="source"><i>Molecular Informatics</i></span><span class="pub-id"><a href="https://dx.doi.org/10.1002/minf.202000028">10.1002/minf.202000028</a></span></span></li> <li tag="ref"><a name="CIT0045" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verma</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khedkar</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coutinho</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">3D-QSAR in drug design-a review</span>. <span class="source" tagx="source" title="source"><i>Current Topics in Medicinal Chemistry</i></span>, (), <span class="fpage" tagx="fpage" title="fpage">95</span>â€"<span class="lpage" tagx="lpage" title="lpage">115</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2174/156802610790232260">10.2174/156802610790232260</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19929826">19929826</a></span></span></li> <li tag="ref"><a name="CIT0046" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: Worldometer.</span></span> (<span class="year" tagx="year" title="year">2020</span>). Coronavirus cases. <span class="person-group'"><span class="collab'">collab: Worldometer</span></span><a href="https://www.worldometers.info/coronavirus/coronavirus-cases/#daily-cases">https://www.worldometers.info/coronavirus/coronavirus-cases/#daily-cases</a></span></li> <li tag="ref"><a name="CIT0047" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Curth</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauerhering</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Becker</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rox</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilgenfeld</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved Î±-ketoamide inhibitors</span>. <span class="source" tagx="source" title="source"><i>Science</i>,</span><span class="volume" tagx="volume" title="volume">3405</span>(<span class="volume" tagx="volume" title="volume">March</span>), <span class="fpage" tagx="fpage" title="fpage">eabb3405</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.abb3405">10.1126/science.abb3405</a></span></span></li> <li tag="ref"><a name="CIT0048" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Network-based drug repurposing for novel coronavirus 2019-nCoV/SARS-CoV-2</span>. <span class="source" tagx="source" title="source"><i>Cell Discovery</i></span>, (<span class="issue" tagx="issue" title="issue">1</span>) <span class="fpage" tagx="fpage" title="fpage">14</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41421-020-0153-3">10.1038/s41421-020-0153-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32194980">32194980</a></span></span></li> </ul> </div> </div>  </body></html>