<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">North Clin Istanb</span><span class="iso-abbrev" title="iso-abbrev">North Clin Istanb</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Northern Clinics of Istanbul</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">2148-4902</span><span class="issn-epub" tagx="issn" title="issn-epub">2536-4553</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Kare Publishing</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Turkey</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7251277">7251277</a></span><span class="publisher-id" title="publisher-id">NCI-7-203</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.14744/nci.2020.90532">10.14744/nci.2020.90532</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
The origin of SARS-CoV-2 in Istanbul: Sequencing findings from the epicenter of the pandemic in Turkey</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karacan</span><span class="given-names" tagx="given-names" title="given-names">Ilker</span></span><a href="#aff1">1</a><a href="#aff2">2</a><a href="#fn1">â€ </a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akgun</span><span class="given-names" tagx="given-names" title="given-names">Tugba Kizilboga</span></span><a href="#aff1">1</a><a href="#fn1">â€ </a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agaoglu</span><span class="given-names" tagx="given-names" title="given-names">N. Bugra</span></span><a href="#aff1">1</a><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irvem</span><span class="given-names" tagx="given-names" title="given-names">Arzu</span></span><a href="#aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alkurt</span><span class="given-names" tagx="given-names" title="given-names">Gizem</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yildiz</span><span class="given-names" tagx="given-names" title="given-names">Jale</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kose</span><span class="given-names" tagx="given-names" title="given-names">Betsi</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ozel</span><span class="given-names" tagx="given-names" title="given-names">A. Serra</span></span><a href="#aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altunal</span><span class="given-names" tagx="given-names" title="given-names">L. Nilsun</span></span><a href="#aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Can</span><span class="given-names" tagx="given-names" title="given-names">Nisan Denizce</span></span><a href="#aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Demirkol</span><span class="given-names" tagx="given-names" title="given-names">Yasemin Kendir</span></span><a href="#aff1">1</a><a href="#aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aydin</span><span class="given-names" tagx="given-names" title="given-names">Mehtap</span></span><a href="#aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dogan</span><span class="given-names" tagx="given-names" title="given-names">Ozlem Akgun</span></span><a href="#aff1">1</a><a href="#aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doganay</span><span class="given-names" tagx="given-names" title="given-names">Levent</span></span><a href="#aff1">1</a><a href="#fn1">*</a><a href="#cor2" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doganay</span><span class="given-names" tagx="given-names" title="given-names">Gizem Dinler</span></span><a href="#aff6">6</a><a href="#fn1">*</a><a href="#cor1" /></span></div><span class="citation_author_institution" id="aff1">[1], </span><span class="citation_author_institution" id="aff2">[2], </span><span class="citation_author_institution" id="aff3">[3], </span><span class="citation_author_institution" id="aff4">[4], </span><span class="citation_author_institution" id="aff5">[5], </span><span class="citation_author_institution" id="aff6">[6], </span><span class="citation_author_institution" id="aff7">[7], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><b>Corresponding authors:</b> Gizem DINLER DOGANAY, PhD. Istanbul Teknik Universitesi, Molekuler Biyoloji ve Genetik Bolumu, Istanbul, Turkey. Tel: +90 212 285 72 50 e-mail: <span class="email" tagx="email" title="email">gddoganay@itu.edu.tr</span></div> <div class="corresp" title="corresp">
 Levent DOGANAY, MD. Saglik Bilimleri Universitesi, Umraniye Egitim ve Arastirma Hastanesi, Gastroenteroloji ve Hepatoloji Klinigi, Istanbul, Turkey. Tel: +90 216 632 18 18 - 1846 e-mail: <span class="email" tagx="email" title="email">levent.doganay@saglik.gov.tr</span></div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">â€ </span><p>These authors contributed equally to this work</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">*</span><p>These authors have equal correspondence.</p> </div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-5-5</span></span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">203</span><span class="lpage" tagx="lpage" title="lpage">209</span><span class="history" title="history"><span class="received" title="received">received: 2020-5-12</span><span class="accepted" title="accepted">accepted: 2020-5-15</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="objective:" title="sec"> <div class="title" tagx="title" title="title">
OBJECTIVE:</div> <p>Turkey is one of the latest countries that COVID-19 disease was reported, with the first case on March 11, 2020, and since then, Istanbul became the epicenter of the pandemic in Turkey. Here, we reveal sequences of the virus isolated from three different patients with various clinical presentations.</p> </div> <div class="methods:" title="sec"> <div class="title" tagx="title" title="title">
METHODS:</div> <p>Nasopharyngeal swab specimens of the patients were tested positive for the COVID-19 by qRT-PCR. Viral RNA extraction was performed from the same swab samples. Amplicon based libraries were prepared and sequenced using the Illumina NextSeq platform. Raw sequencing data were processed for variant calling and generating near-complete genome sequences. All three genomes were evaluated and compared with other worldwide isolates.</p> </div> <div class="results:" title="sec"> <div class="title" tagx="title" title="title">
RESULTS:</div> <p>The patients showed various clinics (an asymptomatic patient, patient with mild disease, and with severe pulmonary infiltration). Amplicon-based next-generation sequencing approach successfully applied to generate near-complete genomes with an average depth of 2.616. All three viral genomes carried the D614G variant (G clade according to GISAID classification) with implications for the origin of a spread first through China to Europe then to Istanbul.</p> </div> <div class="conclusion:" title="sec"> <div class="title" tagx="title" title="title">
CONCLUSION:</div> <p>Here, we report the viral genomes circulating in Istanbul for the first time. Further sequencing of the virus isolates may enable us to understand variations in disease presentation and association with viral factors if there is any. In addition, the sequencing of more viral genomes will delineate the spread of disease and will guide and ease the necessary measures taken to stem the spread of the novel coronavirus.</p> </div> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
COVID-19</div> <div class="kwd" title="kwd">
SARS-CoV-2</div> <div class="kwd" title="kwd">
whole-genome sequencing</div> </div> </div> </div> <div class="body" title="body"> <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), causing coronavirus disease 2019 (COVID-19), was first detected in Wuhan, China, around mid-December 2019. Following the emergence of the outbreak to other countries in a short period, the World Health Organization declared COVID-19 as a pandemic on 11 March 2020. As of May 2020, more than 3.5 million COVID-19 patients all around the world were reported, and nearly 250 thousand deaths arose from the SARS-CoV-2 pandemic [<a href="#ref1">1</a>].</p> <p>The novel SARS-CoV-2 belongs to the Betacoronavirus genus of the Coronaviridae family, which are single-stranded RNA viruses [<a href="#ref2">2</a>]. Genetic similarity analyses revealed that the bat coronavirus RaTG13 showed the highest similarity (96%) to SARS-CoV-2, and it was suggested that the zoonotic origin might include bats [<a href="#ref3">3</a>, <a href="#ref4">4</a>]. Coronaviruses were thought to result in seasonal mild respiratory illness in humans until the epidemics of the SARS-CoV in 2003 and MERS-CoV in 2012 [<a href="#ref5">5</a>, <a href="#ref6">6</a>].</p> <p>As the number of COVID-19 cases increases worldwide, new symptoms have been reported, such as nausea, diarrhea, skin rash, and loss of taste and/or smell [<a href="#ref7">7</a>, <a href="#ref8">8</a>]. In addition to the symptomatic patients, initial studies showed that a considerable amount of COVID-19 patients are asymptomatic (18â€"30%) or have mild symptoms [<a href="#ref9">9</a>â€"<a href="#ref11">11</a>], whereas hospitalization and mortality rates increase with patientâ€™s age [<a href="#ref12">12</a>]. The severity of the infection is related to the age of the patient and the presence of additional chronic diseases, such as hypertension, diabetes, and cancer [<a href="#ref13">13</a>]. Besides challenges on detecting evidence on virulence changes of such a pathogen that is spreading very fast, there is not any solid report highlighting a mutation that affects the biological features of the virus.</p> <p>The entrance of coronaviruses into the host cell is maintained by spike glycoprotein (encoded by S gene). The S1 subunit of SARS-CoV spike protein contains Receptor Binding Protein, which plays an essential role in the recognition of angiotensin-converting enzyme 2 (ACE2) [<a href="#ref14">14</a>]. The novel SARS-CoV-2 virus has a similar surface spike protein sharing 76% sequence identity with SARS-CoV [<a href="#ref15">15</a>]. The binding affinity of spike protein to ACE2 is important for virulence, and it has been shown that spike protein of the novel SARS-CoV-2 binds to ACE2 with a much higher affinity [<a href="#ref16">16</a>]. It can be hypothesized that a particular mutation in the spike protein may lead to conformational variations affecting the virulence.</p> <p>Due to the lack of polymerase proofreading activity, RNA viruses have a relatively high mutation rate and, thus capable to become resistant to drugs and escape from host immunity. As mutations accumulate, they may result in alterations in virulence, transmission capacity, infectivity, and pathogenicity of the viruses [<a href="#ref17">17</a>]. Although SARS-CoV-2 has a lower mutation rate than expected [<a href="#ref18">18</a>], real-time tracking of the virus isolates in populations may help epidemiological understanding of the disease and early detection of important mutational or recombination events.</p> <p>The first case of COVID-19 in Turkey was reported on 11 March 2020, much later than the virus had spread to European countries. As of April 1, the Ministry of Health of Turkey announced that COVID-19 had reached all over Turkey, exhibiting the highest spread in Istanbul. The first full-length SARS-CoV-2 genome in Turkey was isolated from a patient in Kayseri province and released on 13 April 2020. Herein, we analyzed full-length SARS-CoV-2 genomes from three patients in Istanbul together with their clinical findings.</p> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
MATERIALS AND METHODS</div> <p><b>Sample Collection:</b> Nasopharyngeal swabs were collected from unrelated patients and tested for SARS-CoV-2 presence as a standard care protocol for routine diagnosis in Umraniye Training and Research Hospital (UEAH), Istanbul. Three patients whose tests positive for SARS-CoV-2 with qRT-PCR testing were included in this study. This study was approved by the ethics committee of the Umraniye Training and Research Hospital (B.10.1.TKH.4.34.H.GP.0.01/95) and a written informed consent was obtained from all participating patients.</p> <p><b>Viral RNA Extraction:</b> Viral RNA was extracted from 200 Âµl of nasopharyngeal swab samples with High Pure Viral RNA Kit (Roche Life Science) according to the manufacturerâ€™s protocol. Extracted RNA was eluted in 45 Âµl of DNase/RNase-free water. Total RNA concentration was determined with Qubit 4.0 Fluorometer using Qubit RNA HS Assay Kit (Thermo Fisher Scientific Inc.) and RNA quality was evaluated at 260/280 nm and 260/230 nm ratios using Nanodrop 2000. RNA samples were stored at âˆ’80 Â°C until further processing.</p> <p><b>Viral Genome Sequencing and Data Analysis:</b> Sequencing libraries were generated using the CleanPlex SARS-CoV-2 Library Preparation Kit (Paragon Genomics Inc.) following the manufacturerâ€™s instructions. As input material, 50 ng of extracted RNA was used for each sample. Briefly, reverse transcription of RNA was performed; the viral genome was amplified with multiplex PCRs followed by indexing PCR to add adapters and sample-specific barcode sequences. DNA clean-up steps were performed with Agencourt AMPure XP beads (Beckman Coulter Inc.) when required to maximize the recovery of fragments. The quantity and quality of the final libraries were assessed using a Qubit dsDNA HS Assay Kit with Qubit 4.0 Fluorometer (Thermo Fisher Scientific Inc.) and an Agilent Bioanalyzer 2100 with High Sensitivity DNA Chips (Agilent Technologies Inc.) following manufacturersâ€™ protocols before sequencing. Sequencing was performed in the joint Genomic Laboratory (GLAB) of UEAH and Istanbul Technical University [<a href="#ref19">19</a>] using Illumina NextSeq500 instrument with paired-end 150 bp chemistry. Raw demultiplexed sequencing data were further processed to call variants and generate consensus genome sequences. First, the quality check of raw sequencing data was performed using the FASTQC program [<a href="#ref20">20</a>] and adapter sequences were trimmed from reads using cutadapt [<a href="#ref21">21</a>]. Processed reads were aligned to the reference SARS-CoV-2 genome (NC_045512.2) using bwa-mem [<a href="#ref22">22</a>]. Further, primer sequences were trimmed from aligned bam files. Variant calling and generating consensus sequences were performed using Samtools [<a href="#ref23">23</a>] and iVar [<a href="#ref24">24</a>]. Finally, all detected variants were checked manually to detect the presence of any sequencing errors, if any. Viral genome sequences were examined phylogenetically together with world-wide isolates using Nextstrain [<a href="#ref25">25</a>, <a href="#ref26">26</a>].</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
RESULTS</div> <div class="clinicalpresentationofthepatients" title="sec"> <div class="title" tagx="title" title="title">
Clinical Presentation of the Patients</div> <p><b>Patient COV-8:</b> COV-8 was a 51-year-old man, and he presented to the emergency department clinic in our hospital with a 2-day history of cough. He did not mention fever or dyspnea. He had no known contact with a COVID-19 positive patient. He was an ex-smoker with a 50 packs-year history and had a medical history of diabetes mellitus and hypertension. The physical examination in the emergency department revealed a body temperature of 36.8Â°C, blood pressure of 120/70 mm Hg, the pulse of 100 beats per minute, respiratory rate of 20 breaths per minute, and oxygen saturation of 97% while the patient was breathing ambient air. Lung auscultation was normal. The remaining physical examination findings were unremarkable or normal. Nasopharyngeal and oropharyngeal swab specimens were collected and sent for real-time PCR (RT-PCR) assay for SARS-CoV-2. The patient complied with the possible case definition of COVID-19, which was stated in the COVID-19 Guide of the Ministry of Health of Turkey [<a href="#ref27">27</a>]. Therefore, thorax computed tomography (CT), in line with COVID-19 Guide [<a href="#ref27">27</a>], was performed and reported as containing typical findings of COVID pneumonia with mild involvement. Hospitalization was decided by considering the thorax CT result and the patientâ€™s comorbid diseases. Azithromycin and hydroxychloroquine treatment combinations were started. On the first day of hospitalization, vital signs were in the normal range, and the general clinical condition was good. Viral RT-PCR analysis of the nasopharyngeal swab reported as positive. On the second and third day, subfebrile body temperature fluctuated between 37.7Â°C and 37.8Â°C were detected (<a href="#F1">Figure 1</a>). Besides, there was an increase in the frequency of cough, and rales were detected during lung auscultation. An increase in infiltrations was also detected on the chest radiograph, as well as the increase in acute phase reactants (<a href="#F2">Figure 2</a>). Close follow-up was continued by adding ceftriaxone to the treatment. On the 6th day, the subfebrile fever persisted, the patient reported fatigue. Oxygen saturation was &amp;lt;90% while inhaling in the room air. We observed that the serum acute phase reactants increased and lymphopenia appeared in the complete blood count test. In the thorax CT on the 6th day, we also detected progression compared to the previous CT scan. Due to the above-mentioned findings in favor of progression, favipiravir treatment was started. Twenty-four hours after the start of favipiravir, the patientâ€™s body temperature returned to normal range. Through 7 and 14 days of the hospitalization, the patientâ€™s clinical findings and laboratory values gradually improved. The physical examination revealed a body temperature of 36.5Â°C, blood pressure of 125/70 mm Hg, the pulse of 98 beats per minute, respiratory rate of 20 breaths per minute, and oxygen saturation of 98% while the patient was breathing ambient air. The remainder of the examination was normal. Viral RT-PCR analysis of the control nasopharyngeal swab on the 14th day reported as negative. He was discharged from the hospital to revisit for control two weeks later.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 1</span></a></div>   <p>Symptoms, maximum body temperatures, and laboratory findings of COV-8 according to the day of illness and day of hospitalization.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 2</span></a></div>   <p>Posteroanterior chest radiographs of COV-8 on the first <b>(A)</b> and the third <b>(B)</b> days of hospital stay (3<sup>rd</sup> and 5<sup>th</sup> day of illness).</p>   </div> <p><b>Patient COV-12:</b> COV-12 was a 49-year-old man and he presented to the COVID-19 clinic in our hospital with a history COVID-19 positive patient contact. The patient did not have any complaints. He was a healthy nonsmoker. The physical examination revealed a body temperature of 36.0Â°C, blood pressure of 130/70 mm Hg, the pulse of 100 beats per minute, respiratory rate of 18 breaths per minute, and oxygen saturation of 98% while the patient was breathing ambient air. Lung auscultation was normal, as well as the thorax CT. Complete blood count and serum acute phase reactants were in the normal range. Azithromycin and hydroxychloroquine treatment combinations were started, and the patient was isolated at home. The remaining physical examination findings were unremarkable or normal. The control viral RT-PCR test for SARS-CoV-2 was reported as negative.</p> <p><b>Patient COV-13:</b> COV-13 was a 29-year-old woman; she presented to the urgent care clinic with a 5-day history of cough, sore throat, fever, loss of taste and smell. She was a nonsmoker and reported no comorbid disease. The physical examination revealed a body temperature of 36.2Â°C, blood pressure of 140/75 mm Hg, the pulse of 96 beats per minute, respiratory rate of 20 breaths per minute, and oxygen saturation of 96%. The remaining physical examination findings were unremarkable or normal. Thorax CT revealed widespread patchy ground-glass opacities in both lungs. Complete blood count and serum acute phase reactants were in the normal range. She was hospitalized and started on azithromycin and hydroxychloroquine treatments. During the hospital stay, the patient did not have a fever and did not develop respiratory distress; oxygen saturation ranged between 96â€"97%. The treatment was completed in five days, and she was discharged to come for control one week later. After one week, the control examination of the patient was completely normal, for both vital signs and clinical findings. Control viral RT-PCR analysis was also reported as negative.</p> </div> <div class="viralgenomeanalysis" title="sec"> <div class="title" tagx="title" title="title">
Viral Genome Analysis</div> <p>Raw sequencing data consist of 600676, 390806 and 283036 paired-end reads for samples isolated from patients COV-8, COV-12 and COV-13, respectively. Nearly all reads were mapped to the 29.903 bp reference genome with a mean ratio of 99.02% (Â±0.85), resulting in an average 2.616Â±1.011 depth of coverage. Identified variants of three SARS-CoV-2 isolates are given in <a href="#T1">Table 1</a>. Since all three isolates have a D614G variant in spike glycoprotein, they belong to G clade based on GISAID classification. COV-8 and COV-12 had ten, and COV-13 had nine bp changes compared to the reference genome (NC_045512.2). Distinguishably, most of the single nucleotide variants were C to T conversion.</p> <div class="table-wrap_UNKNOWN" id="T1" position="float"><span class="label" tagx="label" title="label">TABLE 1</span> <p>Identified mutations in three SARS-CoV-2 isolates from Istanbul</p>  <table frame="hsides" rules="groups" width="100%"> <thead> <tr> <th align="left" rowspan="1" colspan="1">Sample</th> <th align="center" rowspan="1" colspan="1">Nucleotide position</th> <th align="center" rowspan="1" colspan="1">Nucleotide change (Ref/Alt)</th> <th align="center" rowspan="1" colspan="1">Variant type</th> <th align="center" rowspan="1" colspan="1">Amino acid position</th> <th align="center" rowspan="1" colspan="1">Amino acid change (Ref/Alt)</th> <th align="center" rowspan="1" colspan="1">Gene/Region</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">COV-8</td> <td align="center" rowspan="1" colspan="1">241</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">non-coding</td> <td align="center" rowspan="1" colspan="1">NA</td> <td align="center" rowspan="1" colspan="1">NA</td> <td align="center" rowspan="1" colspan="1">5â€™ UTR</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ2113</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">synonymous</td> <td align="center" rowspan="1" colspan="1">616</td> <td align="center" rowspan="1" colspan="1">I/I</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ2997</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">911</td> <td align="center" rowspan="1" colspan="1">S/F</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ3037</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">synonymous</td> <td align="center" rowspan="1" colspan="1">924</td> <td align="center" rowspan="1" colspan="1">F/F</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ7765</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">synonymous</td> <td align="center" rowspan="1" colspan="1">2500</td> <td align="center" rowspan="1" colspan="1">S/S</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ14408</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">314</td> <td align="center" rowspan="1" colspan="1">P/L</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ17690</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">1408</td> <td align="center" rowspan="1" colspan="1">S/L</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ18877</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">synonymous</td> <td align="center" rowspan="1" colspan="1">1804</td> <td align="center" rowspan="1" colspan="1">L/L</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ23403</td> <td align="center" rowspan="1" colspan="1">A/G</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">614</td> <td align="center" rowspan="1" colspan="1">D/G</td> <td align="center" rowspan="1" colspan="1">S</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ25563</td> <td align="center" rowspan="1" colspan="1">G/T</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">57</td> <td align="center" rowspan="1" colspan="1">Q/H</td> <td align="center" rowspan="1" colspan="1">orf3a</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">COV-12</td> <td align="center" rowspan="1" colspan="1">241</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">non-coding</td> <td align="center" rowspan="1" colspan="1">NA</td> <td align="center" rowspan="1" colspan="1">NA</td> <td align="center" rowspan="1" colspan="1">5â€™ UTR</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ2113</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">synonymous</td> <td align="center" rowspan="1" colspan="1">616</td> <td align="center" rowspan="1" colspan="1">I/I</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ3037</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">synonymous</td> <td align="center" rowspan="1" colspan="1">924</td> <td align="center" rowspan="1" colspan="1">F/F</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ7765</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">synonymous</td> <td align="center" rowspan="1" colspan="1">2500</td> <td align="center" rowspan="1" colspan="1">S/S</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ14408</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">314</td> <td align="center" rowspan="1" colspan="1">P/L</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ17690</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">1408</td> <td align="center" rowspan="1" colspan="1">S/L</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ18877</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">synonymous</td> <td align="center" rowspan="1" colspan="1">1804</td> <td align="center" rowspan="1" colspan="1">L/L</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ21452</td> <td align="center" rowspan="1" colspan="1">G/T</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">2662</td> <td align="center" rowspan="1" colspan="1">G/V</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ23403</td> <td align="center" rowspan="1" colspan="1">A/G</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">614</td> <td align="center" rowspan="1" colspan="1">D/G</td> <td align="center" rowspan="1" colspan="1">S</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ25563</td> <td align="center" rowspan="1" colspan="1">G/T</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">57</td> <td align="center" rowspan="1" colspan="1">Q/H</td> <td align="center" rowspan="1" colspan="1">orf3a</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">COV-13</td> <td align="center" rowspan="1" colspan="1">241</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">non-coding</td> <td align="center" rowspan="1" colspan="1">NA</td> <td align="center" rowspan="1" colspan="1">NA</td> <td align="center" rowspan="1" colspan="1">5â€™ UTR</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ3037</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">synonymous</td> <td align="center" rowspan="1" colspan="1">924</td> <td align="center" rowspan="1" colspan="1">F/F</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ11083</td> <td align="center" rowspan="1" colspan="1">G/T</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">3606</td> <td align="center" rowspan="1" colspan="1">L/F</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ12809</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">4182</td> <td align="center" rowspan="1" colspan="1">L/F</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ14408</td> <td align="center" rowspan="1" colspan="1">C/T</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">314</td> <td align="center" rowspan="1" colspan="1">P/L</td> <td align="center" rowspan="1" colspan="1">orf1ab</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ23403</td> <td align="center" rowspan="1" colspan="1">A/G</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">614</td> <td align="center" rowspan="1" colspan="1">D/G</td> <td align="center" rowspan="1" colspan="1">S</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ28881-28882</td> <td align="center" rowspan="1" colspan="1">GG/AA</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">203</td> <td align="center" rowspan="1" colspan="1">R/K</td> <td align="center" rowspan="1" colspan="1">N</td> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">â€ƒ28883</td> <td align="center" rowspan="1" colspan="1">G/C</td> <td align="center" rowspan="1" colspan="1">missense</td> <td align="center" rowspan="1" colspan="1">204</td> <td align="center" rowspan="1" colspan="1">G/R</td> <td align="center" rowspan="1" colspan="1">N</td> <td align="center" rowspan="1" colspan="1" /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>NA: Not applicable.</p> </div> </div> </div> <p>Phylogenetic analysis of the three isolates in this study showed that COV-8 and COV-12 were closely clustered with isolates from Belgium, Netherlands, and Latvia, whereas COV-13 was clustered with Sweden, Belgium and Wales isolates (<a href="#F3">Figure 3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 2</span></a></div>   <p>Inferred phylogeny and isolates in this study with their closely clustered sequences.</p> <p>Retrieved and adapted from <a href="http://www.nextstrain.org">www.nextstrain.org</a></p>   </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
DISCUSSION</div> <p>SARS-CoV-2 is a novel coronavirus that infected more than 3 million people leading to approximately 250,000 deaths globally as of May 2020 [<a href="#ref1">1</a>]. As of May 2020, more than 3,500 patients died in Turkey due to COVID-19, and most of the reported patients are located in Istanbul. Herein, we report three virus genomes isolated in Istanbul for the first time together with patientsâ€™ clinical findings. Patients with various clinical presentations (one asymptomatic, one moderate, and one with severe pulmonary infiltration) were included in this study.</p> <p>Since the first and most of the current cases in Turkey are located in Istanbul, the characterization of virus samples may help to predict the origins of the initial entry to Istanbul. The Nextstrain website (<a href="http://www.nextstrain.org">www.nextstrain.org</a>) provides real-time monitoring of viral isolates around the world, mainly based on publicly accessible GISAID data [<a href="#ref25">25</a>, <a href="#ref26">26</a>]. As of 1 May 2020, more than ten thousand genomes were uploaded to the GISAID database and nearly five thousand different genomes are available for analysis in Nextstrain online tool. Phylogenetic analysis in Nextstrain online tool showed that three isolates from Istanbul in this study were found to be closely clustered within samples isolated mostly in Belgium (<a href="#F3">Figure 3</a>). This close relationship with Belgium isolates gives clues of early viral entry, at least in Istanbul, may have originated from travelers from European countries.</p> <p>GISAID classified three large clades, namely S, G and V. Clades were named based on variants L84S in ORF8 (S clade), D614G in S gene (G clade), and G251V in ORF3a (V clade). Three isolates in this analysis carried the D614G variant in the S gene, indicating they are all in G clade, which was mostly detected in European countries. An increased number of patients should be analyzed to enlighten the effects of viral genetic changes on clinical outcomes.</p> <p>To conclude, we analyzed three SARS-CoV-2 positive individuals in Istanbul, where the epicenter of the pandemic in Turkey. All three viral isolates carried the D614G marker variant indicating the isolates belong to clade G, which encompasses mostly European countries according to GISAID classification. All three virus samples were located in clusters, including isolates from Belgium. As virus surveillance studies are ongoing worldwide, country-wide efforts would also support understanding the local spread of the disease but also evaluate the effectiveness of precautions, such as travel restrictions on disease spread in the country.</p> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data Availability</div> <p>Viral genome sequences in this research were deposited in the Global Initiative on Sharing All Influenza Data (GISAID; <a href="http://www.gisaid.org">www.gisaid.org</a>), with accession numbers EPI_ISL_427391, EPI_ISL_428346, and EPI_ISL_428368.</p> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We acknowledge the authors, originating and submitting laboratories of the sequence data shared through GISAIDâ€™s EpiCOV Database.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-other" title="other"> <p><b>Informed Consent:</b> Written informed consent was obtained from all participating patients.</p> </div> <div class="fn-type-other" title="other"> <p><b>Ethics Committee Approval:</b> This study was approved by the ethics committee of the Umraniye Training and Research Hospital (B.10.1.TKH.4.34.H.GP.0.01/95).</p> </div> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Conflict of Interest:</b> No conflict of interest was declared by the authors.</p> </div> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p><b>Financial Disclosure:</b> This work was supported by Health Institutes of Turkey (TUSEB) (Grant No: 2020TK02-8686/8799)</p> </div> <div class="fn-type-other" title="other"> <p><b>Authorship Contributions:</b> Concept â€" IK, LD, GDD; Design â€" IK, TKA, LD, GDD; Supervision â€" LD, GDD; Fundings â€" LD, GDD; Materials â€" TKA, NBA, AI, GA, JY, BK, ASO, LNA, YKD, MA, OAD, LD, GDD; Data collection and/or processing â€" IK, TKA, NBA, AI, GA, JY, BK, ASO, LNA, NDC, YKD, MA, OAD, LD, GDD; Analysis and/or interpretation â€" IK, TKA, NBA, AI, GA, JY, BK, ASO, LNA, NDC, YKD, MA, OAD, LD, GDD; Literature review â€" IK, NDC, TKA, GDD; Writing â€" IK, TKA, NDC, NBA, OAD, LD, GDD; Critical review â€" IK, TK, NBA, OAD, LD, GDD.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="ref1" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="collab'">collab: World Health Organization</span><span class="source" tagx="source" title="source">Coronavirus disease 2019 (COVID-19):situation report, 107</span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">Accessed May 15, 2020</span><span class="comment">Available at: <span class="uri" title="uri"><a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200506covid-19-sitrep-107.pdf?sfvrsn=159c3dc_2">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200506covid-19-sitrep-107.pdf?sfvrsn=159c3dc_2</a></span></span></span></li> <li tag="ref"><a name="ref2" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="collab'">collab: Coronaviridae Study Group of the International Committee on Taxonomy of Viruses</span><span class="mixed-article-title" title="mixed-article-title">The species Severe acute respiratory syndrome-related coronavirus:classifying 2019-nCoV and naming it SARS-CoV-2</span><span class="source" tagx="source" title="source">Nat Microbiol</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">536</span><span class="lpage" tagx="lpage" title="lpage">44</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32123347">32123347</a></span></span></li> <li tag="ref"><a name="ref3" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dallavilla</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertelli</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morresi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bushati</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuppia</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beccari</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Bioinformatic analysis indicates that SARS-CoV-2 is unrelated to known artificial coronaviruses</span><span class="source" tagx="source" title="source">Eur Rev Med Pharmacol Sci</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">4558</span><span class="lpage" tagx="lpage" title="lpage">64</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32373995">32373995</a></span></span></li> <li tag="ref"><a name="ref4" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">XL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">XG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">579</span><span class="fpage" tagx="fpage" title="fpage">270</span><span class="lpage" tagx="lpage" title="lpage">3</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32015507">32015507</a></span></span></li> <li tag="ref"><a name="ref5" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlman</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Netland</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronaviruses post-SARS:update on replication and pathogenesis</span><span class="source" tagx="source" title="source">Nat Rev Microbiol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">439</span><span class="lpage" tagx="lpage" title="lpage">50</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19430490">19430490</a></span></span></li> <li tag="ref"><a name="ref6" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zumla</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlman</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Middle East respiratory syndrome</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">386</span><span class="fpage" tagx="fpage" title="fpage">995</span><span class="lpage" tagx="lpage" title="lpage">1007</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26049252">26049252</a></span></span></li> <li tag="ref"><a name="ref7" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lian</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">YM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemiological, clinical and virological characteristics of 74 cases of coronavirus-infected disease 2019 (COVID-19) with gastrointestinal symptoms</span><span class="source" tagx="source" title="source">Gut</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">1002</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32213556">32213556</a></span></span></li> <li tag="ref"><a name="ref8" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eliezer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hautefort</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamel</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verillaud</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herman</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Houdart</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Sudden and Complete Olfactory Loss Function as a Possible Symptom of COVID-19</span><span class="source" tagx="source" title="source">JAMA Otolaryngol Head Neck Surg 2020 Apr 8 [Epub ahead of print]</span><span class="comment">doi:10.1001/jamaoto.2020.0832</span></span></li> <li tag="ref"><a name="ref9" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="collab'">collab: The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team</span><span class="mixed-article-title" title="mixed-article-title">The epidemiological characteristics of an outbreak of 2019 novel coronavirus diseases (COVID-19) - China, 2020</span><span class="source" tagx="source" title="source">China CDC Weekly</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">113</span><span class="lpage" tagx="lpage" title="lpage">22</span></span></li> <li tag="ref"><a name="ref10" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishiura</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobayashi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyama</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayashi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Estimation of the asymptomatic ratio of novel coronavirus infections (COVID-19)</span><span class="source" tagx="source" title="source">Int J Infect Dis</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">154</span><span class="lpage" tagx="lpage" title="lpage">5</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32179137">32179137</a></span></span></li> <li tag="ref"><a name="ref11" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mizumoto</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kagaya</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zarebski</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chowell</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Estimating the asymptomatic proportion of coronavirus disease 2019 (COVID-19) cases on board the Diamond Princess cruise ship, Yokohama, Japan, 2020</span><span class="source" tagx="source" title="source">Euro Surveill</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">2000180</span></span></li> <li tag="ref"><a name="ref12" /><span class="label" tagx="label" title="label">12</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garg</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whitaker</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Halloran</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cummings</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holstein</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Hospitalization Rates and Characteristics of Patients Hospitalized with Laboratory-Confirmed Coronavirus Disease 2019 - COVID-NET, 14 States, March 1-30, 2020</span><span class="source" tagx="source" title="source">MMWR Morb Mortal Wkly Rep</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">458</span><span class="lpage" tagx="lpage" title="lpage">64</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32298251">32298251</a></span></span></li> <li tag="ref"><a name="ref13" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China:a retrospective cohort study</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">395</span><span class="fpage" tagx="fpage" title="fpage">1054</span><span class="lpage" tagx="lpage" title="lpage">62</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32171076">32171076</a></span></span></li> <li tag="ref"><a name="ref14" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belouzard</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">VC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whittaker</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Activation of the SARS coronavirus spike protein via sequential proteolytic cleavage at two distinct sites</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">106</span><span class="fpage" tagx="fpage" title="fpage">5871</span><span class="lpage" tagx="lpage" title="lpage">6</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19321428">19321428</a></span></span></li> <li tag="ref"><a name="ref15" /><span class="label" tagx="label" title="label">15</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ou</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lei</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mi</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of spike glycoprotein of SARS-CoV-2 on virus entry and its immune cross-reactivity with SARS-CoV</span><span class="source" tagx="source" title="source">Nat Commun</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">1620</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32221306">32221306</a></span></span></li> <li tag="ref"><a name="ref16" /><span class="label" tagx="label" title="label">16</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wrapp</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corbett</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldsmith</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsieh</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abiona</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">367</span><span class="fpage" tagx="fpage" title="fpage">1260</span><span class="lpage" tagx="lpage" title="lpage">3</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32075877">32075877</a></span></span></li> <li tag="ref"><a name="ref17" /><span class="label" tagx="label" title="label">17</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berngruber</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Froissart</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choisy</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolution of virulence in emerging epidemics</span><span class="source" tagx="source" title="source">PLoS Pathog</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e1003209</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23516359">23516359</a></span></span></li> <li tag="ref"><a name="ref18" /><span class="label" tagx="label" title="label">18</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhan</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deverman</span><span class="given-names" tagx="given-names" title="given-names">BE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">YA</span></span></span><span class="mixed-article-title" title="mixed-article-title">SARS-CoV-2 is well adapted for humans. What does this mean for re-emergence?</span><span class="source" tagx="source" title="source">bioRxiv</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">28</span></span></li> <li tag="ref"><a name="ref19" /><span class="label" tagx="label" title="label">19</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doganay</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ozdil</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Memisoglu</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katrinli</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karakoc</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikerel</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Integrating personalized genomics into Turkish healthcare system:A cancer-oriented pilot activity of Istanbul Northern Anatolian Public Hospitals with GLAB</span><span class="source" tagx="source" title="source">North Clin Istanb</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">3</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28752136">28752136</a></span></span></li> <li tag="ref"><a name="ref20" /><span class="label" tagx="label" title="label">20</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andrews</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="source" tagx="source" title="source">FastQC:a quality control tool for high throughput sequence data</span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">Accessed May 15, 2020</span><span class="comment">Available at: <span class="uri" title="uri"><a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a></span></span></span></li> <li tag="ref"><a name="ref21" /><span class="label" tagx="label" title="label">21</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</span><span class="source" tagx="source" title="source">EMBnet Journal</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">10</span><span class="lpage" tagx="lpage" title="lpage">2</span></span></li> <li tag="ref"><a name="ref22" /><span class="label" tagx="label" title="label">22</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM</span><span class="source" tagx="source" title="source">arXiv</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">3</span></span></li> <li tag="ref"><a name="ref23" /><span class="label" tagx="label" title="label">23</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Handsaker</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wysoker</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fennell</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Homer</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="collab'">collab: 1000 Genome Project Data Processing Subgroup</span><span class="mixed-article-title" title="mixed-article-title">The Sequence Alignment/Map format and SAMtools</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">2078</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a></span></span></li> <li tag="ref"><a name="ref24" /><span class="label" tagx="label" title="label">24</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grubaugh</span><span class="given-names" tagx="given-names" title="given-names">ND</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gangavarapu</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quick</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matteson</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Jesus</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Main</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">An amplicon-based sequencing framework for accurately measuring intrahost virus diversity using PrimalSeq and iVar</span><span class="source" tagx="source" title="source">Genome Biol</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">8</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30621750">30621750</a></span></span></li> <li tag="ref"><a name="ref25" /><span class="label" tagx="label" title="label">25</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadfield</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Megill</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huddleston</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Potter</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callender</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Nextstrain:real-time tracking of pathogen evolution</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">4121</span><span class="lpage" tagx="lpage" title="lpage">3</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29790939">29790939</a></span></span></li> <li tag="ref"><a name="ref26" /><span class="label" tagx="label" title="label">26</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sagulenko</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puller</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neher</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="mixed-article-title" title="mixed-article-title">TreeTime:Maximum-likelihood phylodynamic analysis</span><span class="source" tagx="source" title="source">Virus Evolution</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">vex042</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29340210">29340210</a></span></span></li> <li tag="ref"><a name="ref27" /><span class="label" tagx="label" title="label">27</span><span class="element-citation'"><span class="collab'">collab: T.C. SaÄŸlÄ±k BakanlÄ±ÄŸÄ±</span><span class="source" tagx="source" title="source">COVID-19 (SARS-CoV-2 enfeksiyonu) rehberi</span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">Accessed May 15, 2020</span><span class="comment">Available at: <span class="uri" title="uri"><a href="https://covid19bilgi.saglik.gov.tr/depo/rehberler/COVID-19_Rehberi.pdf?type=file">https://covid19bilgi.saglik.gov.tr/depo/rehberler/COVID-19_Rehberi.pdf?type=file</a></span></span></span></li> </ul> </div> </div>  </body></html>