<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Netw Model Anal Health Inform Bioinform</span><span class="iso-abbrev" title="iso-abbrev">Netw Model Anal Health Inform Bioinform</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Network Modeling and Analysis in Health Informatics and Bioinformatics</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">2192-6662</span><span class="issn-epub" tagx="issn" title="issn-epub">2192-6670</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer Vienna</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Vienna</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7413836">7413836</a></span><span class="publisher-id" title="publisher-id">263</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1007/s13721-020-00263-6">10.1007/s13721-020-00263-6</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Molecular docking suggests repurposing of brincidofovir as a potential drug targeting SARS-CoV-2 ACE2 receptor and main protease</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-4803-167X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hussien</span><span class="given-names" tagx="given-names" title="given-names">Mostafa A.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">Maabdulaal@kau.edu.sa</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-1087-3521</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdelaziz</span><span class="given-names" tagx="given-names" title="given-names">Ahmed E. M.</span></span><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.412125.1</span><span class="institution-id" title="institution-id">0000 0001 0619 1117</span><span class="institution" title="institution">Department of Chemistry, Faculty of Science, </span><span class="institution" title="institution">King Abdulaziz University, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.440879.6</span><span class="institution-id" title="institution-id">0000 0004 0578 4430</span><span class="institution" title="institution">Faculty of Science, </span><span class="institution" title="institution">Port-Said University, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.22072.35</span><span class="institution-id" title="institution-id">0000 0004 1936 7697</span><span class="institution" title="institution">Department of Biological Sciences, </span><span class="institution" title="institution">University of Calgary, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-8-8</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020</span></span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">1</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">56</span><span class="history" title="history"><span class="received" title="received">received: 2020-6-9</span><span class="rev-recd" title="rev-recd">rev-recd: 2020-7-22</span><span class="accepted" title="accepted">accepted: 2020-7-27</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted research re-use and secondary analysis in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="abstract" title="sec"> <div class="title" tagx="title" title="title">
Abstract</div> <p id="Par1">The current outbreak of the highly transmittable and life-threatening severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has evolved rapidly and posed a global health emergency. Many clinical trials are now being conducted to test possible therapies. To assist this, virtual screening via molecular docking was performed on several FDA-approved drugs, previously used in epidemics, and the top ten compounds were selected. These ten well-characterized drugs, previously used to treat malaria and Ebola infections, were screened based on their interactions with the SARS-CoV-2 ACE2 receptor and 3C-like protease. Compared to the other nine medicines, brincidofovir, an ether lipid ester analog of cidofovir with potent antiviral activity, showed the highest docking scores and binding interactions. Therefore, brincidofovir is worth further investigations and clinical trials as a possible therapeutic agent for the COVID-19 disease caused by the novel SARS-CoV-2.</p> </div> <div class="graphicabstract" title="sec"> <div class="title" tagx="title" title="title">
Graphic abstract</div> <p id="Par2"> </p><div class="graphic" position="anchor" id="MO1" /> <p /> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p>The online version of this article (10.1007/s13721-020-00263-6) contains supplementary material, which is available to authorized users.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Novel coronavirus</div> <div class="kwd" title="kwd">
SARS-CoV-2</div> <div class="kwd" title="kwd">
Molecular docking</div> <div class="kwd" title="kwd">
ACE2 receptor</div> <div class="kwd" title="kwd">
3C-like protease</div> <div class="kwd" title="kwd">
Brincidofovir</div> <div class="kwd" title="kwd">
COVID-19</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© Springer-Verlag GmbH Austria, part of Springer Nature 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par3">Humankind has previously witnessed the outbreak of many life-threatening pathogens including Ebola, Zika, the Middle East respiratory syndrome (MERS) coronavirus, severe acute respiratory syndrome (SARS) coronavirus and, nowadays, the severe acute respiratory syndrome coronavirus 2Â (SARS-CoV-2) (Memish et al. <a href="#CR21">2013</a>; Organization; Sikka et al. <a href="#CR30">2016</a>; Stawicki et al. <a href="#CR30">2016</a>; Su et al. <a href="#CR31">2016</a>; Xiong et al. <a href="#CR36">2020</a>). The novel coronavirus has initially spread from China and propagated rapidly throughout the globe and has received worldwide attention due to its alarming levels of transmission and aggressive behavior in causing acute respiratory disease. The virus was then officially declared pandemic as a public health emergency of international concern by the World Health Organization (WHO). Researchers throughout the globe are working around the clock to develop potential vaccines and drugs to fight the new coronavirus, SARS-CoV-2, the causative agent of the COVID-19 disease. However, developing a new drug or vaccine usually takes a long time as it should be intensively tested and confirmed to be safe through clinical trials before they can be approved for human use (Control and Prevention <a href="#CR9">2014</a>; GSK files for approval of worldâ€™s first malaria vaccine <a href="#CR14">2014</a>). Therefore, repurposing FDA-approved drugs seems to be a quicker way to treat patients who otherwise have no option. The SARS-CoV-2 is a single-stranded positive-sense RNA virus that relies on its spike (S) protein to attach and enter the target cells (Chiang <a href="#CR7">2020</a>; Hoffmann et al. <a href="#CR15">2020</a>; Wu et al. <a href="#CR35">2020</a>). The virus S protein binds to the host cell angiotensin-converting enzyme 2 (ACE2) receptor, allowing the virus particles to enter the cells (Chiang <a href="#CR7">2020</a>; Hoffmann et al. <a href="#CR15">2020</a>; Wan et al. <a href="#CR34">2020</a>). Thus, blocking the ACE2 receptor reveals a potential therapeutic target for drug discovery to prevent SARS-CoV-2 transmissibility. Besides, two coronavirus proteases, designated 3-chymotrypsin-like protease (3CLpro) and a papain-like protease (PLpro), were previously considered vital targets to combat the SARS and MERS coronavirus epidemics (Wu et al. <a href="#CR35">2020</a>). These two proteases were shown to be highly conserved with the novel SARS-CoV-2, especially in the functional regions (Wu et al. <a href="#CR35">2020</a>). Viruses use their proteases to break down their viral peptides into functional units essential for replication and packaging inside the host cells, and thus considered antiviral drug targets. Molecular docking is a popular bioinformatic modeling tool broadly used in structure-based drug design (Kitchen et al. <a href="#CR17">2004</a>; Mubarak et al. <a href="#CR22">2020</a>; Naik <a href="#CR23">2020</a>; Naik and Pardasani <a href="#CR24">2017</a>, <a href="#CR25">2018</a>; Naik and Zu <a href="#CR26">2020</a>). It is an efficient way to predict the type of interaction, binding affinity and the appropriate target-binding sites between the drug and corresponding receptor using, for instance, scoring functions (Kitchen et al. <a href="#CR17">2004</a>; Lengauer and Rarey <a href="#CR19">1996</a>). Elucidating the binding behavior plays an important role inÂ the rational design of drugsÂ as well as to explicate fundamental biochemical processes (Kitchen et al. <a href="#CR17">2004</a>; Lengauer and Rarey <a href="#CR19">1996</a>). In this study, molecular docking was performed on dozens of FDA-approved drugs and the top ten hits, previously used in the treatment of malarial, fungal/bacterial and Ebola infections and FDA-approved/fast-tracked for human treatment, were selected. For the selected drugs used in this study, the MOE modeling program was used to predict the binding sites and their docking score.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="moleculardockingmethod" title="sec"> <div class="title" tagx="title" title="title">
Molecular docking method</div> <div class="softwareandmachineryused" title="sec"> <div class="title" tagx="title" title="title">
Software and machinery used</div> <p id="Par4"> All docking studies were performed and characterized by the MOE "Molecular Operating Environment" program. Drug preparation was compiled through ChemDraw, 3D structures were constructed using Chem 3D ultra 12.0 software (Molecular Modeling and Analysis; Cambridge Soft Corporation), then they were energetically minimized using MOPAC and saved as MDL MolFile (*.mol). All calculations were carried out on an Intel(R) Core(TM)i7, 3.8Â GHz-based machine running MS Windows 10 as the operating system(Mubarak et al. <a href="#CR22">2020</a>; Naik <a href="#CR23">2020</a>; Naik and Pardasani <a href="#CR24">2017</a>, <a href="#CR25">2018</a>; Naik and Zu <a href="#CR26">2020</a>).</p> </div> <div class="sars-cov-2proteaseandreceptorstructure" title="sec"> <div class="title" tagx="title" title="title">
SARS-CoV-2 protease and receptor structure</div> <p id="Par5">Generation of the protein structures and the crystal structure of the new COVID-19 protease (PDB codeâ€‰=â€‰1Q2W) and ACE2 receptor (PDB codeâ€‰=â€‰6M0J) were retrieved from the Protein Data Bank (<a href="http://www.rcsb.org/pdb/welcome.do">http://www.rcsb.org/pdb/welcome.do</a>) (Bonanno et al. <a href="#CR5">2003</a>). All bound waters, ligands and cofactors were removed from the proteins and then we added hydrogen atoms for optimization.</p> </div> <div class="moleculardockingprocedure" title="sec"> <div class="title" tagx="title" title="title">
Molecular docking procedure</div> <p id="Par6">The docking protocol was done against the SARS-CoV-2 ACE2 receptor (PDB codeâ€‰=â€‰6M0J), the SARS-CoV-2 3CL protease (PDB codeâ€‰=â€‰1Q2W) and its four active sites. The active sites were isolated and used as dummy atoms. The parameters and charges were assigned with the MMFF94x force field. After alpha-site spheres were generated using the site finder module of MOE, the structural model of the compounds were docked on the surface of the interior of the receptor using the DOCK module of MOE (Abdel-Rhman et al. <a href="#CR2">2019</a>). The Dock scoring in MOE software was done through the London dG scoring function and has been upgraded by two unrelated refinement methods. Auto rotatable bonds were allowed, and the best five binding poses were directed for analysis to achieve the best score. We used the database browsers to compare the docking poses to the ligand in the co-crystallized structure and to obtain RMSD of the docking pose. To rank the binding affinity of the synthesized compounds to the protein molecule, the binding free energy and hydrogen bonds between the compounds and amino acid in the receptor were calculated (Abdellattif et al. <a href="#CR1">2018</a>; Ketan et al. (<a href="#CR32">2020</a>). Evaluation of the hydrogen bonds was done by measuring the hydrogen bond length, which does not exceed 3.5 AÂ°. Also, the RMSD of the drug position compared to the docking pose was used in the ranking. RMSD, as well as the mode of interaction of the native ligand within the structure of the receptor, was used as a standard docked model (Abdel-Rhman et al. <a href="#CR2">2019</a>; Abdellattif et al. <a href="#CR1">2018</a>; Althagafi et al. <a href="#CR3">2019</a>; Mashat et al. <a href="#CR20">2019</a>; Ketan et al. <a href="#CR32">2020</a>).</p> </div> </div> </div> <div class="resultsanddiscussions" title="sec"> <div class="title" tagx="title" title="title">
Results and discussions</div> <p id="Par7">Molecular docking and other computer-related methods are efficient tools broadly used to understand the molecular aspects of proteinâ€"ligand interactions during drug discovery against many of previous emerging and fatal diseases including SARS coronavirus (Kitchen et al. <a href="#CR17">2004</a>; Lengauer and Rarey <a href="#CR19">1996</a>). In this study, virtual screening of several FDA-approved/fast-tracked drugs was performed against the SARS-CoV-2 ACE2 host receptor (PDB codeâ€‰=â€‰6M0J), the SARS-CoV-2 3CL protease (PDB codeâ€‰=â€‰1Q2W) and its four active sites, to find the most predicated drugâ€"ligand interactions. The presented parameters include the docking scores, ligand binding efficiency and hydrogen bonding interactions. The top ten ranked compounds were selected and are presented in TablesÂ <a href="#Tab1">1</a>, <a href="#Tab2">2</a>, <a href="#Tab3">3</a>, <a href="#Tab4">4</a>, <a href="#Tab5">5</a> and <a href="#Tab6">6</a> and Figs.Â <a href="#Fig1">1</a>, <a href="#Fig2">2</a>, <a href="#Fig3">3</a>, <a href="#Fig4">4</a> and <a href="#Fig5">5</a>. These ten drugs include four antivirals (<i>favipiravir, ribavirin, brincidofovir, and galidesivir</i>), four antimalarial (<i>chloroquine, mefloquine, primaquine, and tafenoquine</i>) and two antimicrobial agents (<i>doxycycline and atovaquone</i>). Whether we docked against the ACE2 receptor (PDB codeâ€‰=â€‰6M0J), the SARS-CoV-2 3CL protease (PDB codeâ€‰=â€‰1Q2W) or the four main active sites within the SARS-CoV-2 3CL protease, the docking scores of)<i>brincidofovir</i> or BCV) were shown to be the top hit (ranked #1) compared to the other nine drugs. The docking scores for the BCV were âˆ’Â 10.83, âˆ’Â 8.30 and âˆ’Â 9.02 toward the SARS-CoV-2 3CL protease active site 1 (PDB codeâ€‰=â€‰1Q2W), the SARS-CoV-2 3CL whole protease (PDB codeâ€‰=â€‰1Q2W) (TablesÂ <a href="#Tab1">1</a> and <a href="#Tab2">2</a> and Figs.Â <a href="#Fig1">1</a>, <a href="#Fig2">2</a>) and the ACE2 receptor (PDB codeâ€‰=â€‰6M0J) (TablesÂ <a href="#Tab3">3</a> and <a href="#Tab4">4</a> and Figs.Â <a href="#Fig3">3</a> and <a href="#Fig4">4</a>), respectively. The antimalarial drug <i>tafenoquine</i> comes second in the rank where it scored âˆ’Â 8.15 and âˆ’Â 7.76 with the AC2 receptor and the SARS-CoV-2 3CL protease active site 1, respectively (TablesÂ <a href="#Tab2">2</a> and <a href="#Tab4">4</a>). <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Docking score and energy of the malaria and Ebola drugs and 1Q2W of COVID-19 with site 1 of COVID-19 protease (PDB codeâ€‰=â€‰1Q2W)</p>  <div class="graphic" position="anchor" id="MO88" />  <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">TableÂ 2</span> <p>Interaction between malaria and Ebola drugs and 1Q2W of COVID-19 with site 1 of COVID-19 protease (PDB codeâ€‰=â€‰1Q2W)</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Drug name</th> <th align="left">Ligand</th> <th align="left">Receptor</th> <th align="left">Interaction</th> <th align="left">Distance</th> <th align="left">E (kcal/mol)</th> </tr> </thead> <tbody> <tr> <td align="left">Atovaquone</td> <td align="left">6-ring</td> <td align="left">CD PRO 122 (B)</td> <td align="left">pi-H</td> <td align="left">4.11</td> <td align="left">âˆ’Â 0.5</td> </tr> <tr> <td align="left" rowspan="3">Chloroquine</td> <td align="left">O 5</td> <td align="left">NZ LYS 5 (A)</td> <td align="left">H-acceptor</td> <td align="left">3.34</td> <td align="left">âˆ’Â 0.9</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">CB LYS 137 (A)</td> <td align="left">pi-H</td> <td align="left">4.16</td> <td align="left">âˆ’Â 0.6</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">CA GLY 2 (B)</td> <td align="left">pi-H</td> <td align="left">3.49</td> <td align="left">âˆ’Â 0.5</td> </tr> <tr> <td align="left">Doxycycline</td> <td align="left">N 6</td> <td align="left">N GLN 127 (A)</td> <td align="left">H-acceptor</td> <td align="left">3.27</td> <td align="left">âˆ’Â 3.2</td> </tr> <tr> <td align="left">Mefloquine</td> <td align="left">O 41</td> <td align="left">OG1 THR 285 (A)</td> <td align="left">H-donor</td> <td align="left">3.09</td> <td align="left">âˆ’Â 0.9</td> </tr> <tr> <td align="left" rowspan="2">Primaquine</td> <td align="left">F 1</td> <td align="left">N GLN 127 (B)</td> <td align="left">H-acceptor</td> <td align="left">3.05</td> <td align="left">âˆ’Â 0.6</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">CG LYS 5 (B)</td> <td align="left">pi-H</td> <td align="left">3.72</td> <td align="left">âˆ’Â 0.8</td> </tr> <tr> <td align="left" rowspan="2">Tafenoquine</td> <td align="left">N 27</td> <td align="left">NH1 ARG 4 (B)</td> <td align="left">H-acceptor</td> <td align="left">3.58</td> <td align="left">âˆ’Â 1.6</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">CD LYS 5 (A)</td> <td align="left">pi-H</td> <td align="left">4.49</td> <td align="left">âˆ’Â 0.7</td> </tr> <tr> <td align="left" rowspan="2">Favipiravir</td> <td align="left">N 13</td> <td align="left">O LYS 5 (A)</td> <td align="left">H-donor</td> <td align="left">3.16</td> <td align="left">âˆ’Â 1.6</td> </tr> <tr> <td align="left">N 9</td> <td align="left">N GLN 127 (B)</td> <td align="left">H-acceptor</td> <td align="left">3.32</td> <td align="left">âˆ’Â 2.3</td> </tr> <tr> <td align="left" rowspan="5">Ribavirin</td> <td align="left">O 1</td> <td align="left">O PHE 3 (B)</td> <td align="left">H-donor</td> <td align="left">2.98</td> <td align="left">âˆ’Â 0.8</td> </tr> <tr> <td align="left">O 15</td> <td align="left">NZ LYS 5 (A)</td> <td align="left">H-acceptor</td> <td align="left">3.26</td> <td align="left">âˆ’Â 1.2</td> </tr> <tr> <td align="left">O 26</td> <td align="left">N GLN 127 (B)</td> <td align="left">H-acceptor</td> <td align="left">3.14</td> <td align="left">âˆ’Â 3.2</td> </tr> <tr> <td align="left">N 27</td> <td align="left">N GLN 127 (A)</td> <td align="left">H-acceptor</td> <td align="left">3.32</td> <td align="left">âˆ’Â 2.1</td> </tr> <tr> <td align="left">5-ring</td> <td align="left">CB LYS 5 (B)</td> <td align="left">pi-H</td> <td align="left">3.99</td> <td align="left">âˆ’Â 0.7</td> </tr> <tr> <td align="left" rowspan="4">Galidesivir</td> <td align="left">O 33</td> <td align="left">O PHE 3 (B)</td> <td align="left">H-donor</td> <td align="left">3.00</td> <td align="left">âˆ’Â 1.2</td> </tr> <tr> <td align="left">N 9</td> <td align="left">NH1 ARG 4 (B)</td> <td align="left">H-acceptor</td> <td align="left">3.25</td> <td align="left">âˆ’Â 4.0</td> </tr> <tr> <td align="left">N 12</td> <td align="left">N GLN 127 (A)</td> <td align="left">H-acceptor</td> <td align="left">3.59</td> <td align="left">âˆ’Â 1.0</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">CD LYS 5 (A)</td> <td align="left">pi-H</td> <td align="left">4.39</td> <td align="left">âˆ’Â 0.7</td> </tr> <tr> <td align="left" rowspan="2">Brincidovir</td> <td align="left">O 63</td> <td align="left">O GLN 127 (B)</td> <td align="left">H-donor</td> <td align="left">3.02</td> <td align="left">âˆ’Â 2.9</td> </tr> <tr> <td align="left">O 68</td> <td align="left">NH1 ARG 4 (A)</td> <td align="left">H-acceptor</td> <td align="left">2.95</td> <td align="left">âˆ’Â 2.4</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> <p>Docking score and energy of the malaria and Ebola drugs with ACE-2 receptor (PDB codeâ€‰=â€‰6M0J)</p>  <div class="graphic" position="anchor" id="MO889" /> </div> <div class="table-wrap_UNKNOWN" id="Tab4"><span class="label" tagx="label" title="label">TableÂ 4</span> <p>Interaction between malaria and Ebola drugs with ACE-2 receptor (PDB codeâ€‰=â€‰6M0J)</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Drug</th> <th align="left">Ligand</th> <th align="left">Receptor</th> <th align="left">Interaction</th> <th align="left">Distance</th> <th align="left">E (kcal/mol)</th> </tr> </thead> <tbody> <tr> <td align="left">Atovaquone</td> <td align="left">6-ring</td> <td align="left">CA VAL 209 (A)</td> <td align="left">pi-H</td> <td align="left">3.90</td> <td align="left">âˆ’ 1.0</td> </tr> <tr> <td align="left" rowspan="5">Chloroquine</td> <td align="left">N 17</td> <td align="left">O GLU 208 (A)</td> <td align="left">H-donor</td> <td align="left">3.16</td> <td align="left">âˆ’ 0.6</td> </tr> <tr> <td align="left">CL 1</td> <td align="left">NZ LYS 94 (A)</td> <td align="left">H-acceptor</td> <td align="left">3.45</td> <td align="left">âˆ’ 0.9</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">CA VAL 209 (A)</td> <td align="left">pi-H</td> <td align="left">4.40</td> <td align="left">âˆ’ 0.5</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">CG1 VAL 209 (A)</td> <td align="left">pi-H</td> <td align="left">4.14</td> <td align="left">âˆ’ 0.6</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">N ASN 210 (A)</td> <td align="left">pi-H</td> <td align="left">3.62</td> <td align="left">âˆ’ 0.6</td> </tr> <tr> <td align="left" rowspan="2">Doxycycline</td> <td align="left">O 24</td> <td align="left">OE1 GLU 208 (A)</td> <td align="left">H-donor</td> <td align="left">3.01</td> <td align="left">âˆ’ 1.8</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">CG2 VAL 209 (A)</td> <td align="left">pi-H</td> <td align="left">4.28</td> <td align="left">âˆ’ 0.7</td> </tr> <tr> <td align="left" rowspan="4">Mefloquine</td> <td align="left">N 29</td> <td align="left">O ASN 210 (A)</td> <td align="left">H-donor</td> <td align="left">2.91</td> <td align="left">âˆ’ 0.7</td> </tr> <tr> <td align="left">N 29</td> <td align="left">N ASN 210 (A)</td> <td align="left">H-acceptor</td> <td align="left">3.33</td> <td align="left">âˆ’ 0.5</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">CB GLU 208 (A)</td> <td align="left">pi-H</td> <td align="left">4.42</td> <td align="left">âˆ’ 0.5</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">CG2 VAL 209 (A)</td> <td align="left">pi-H</td> <td align="left">4.46</td> <td align="left">âˆ’ 0.6</td> </tr> <tr> <td align="left" rowspan="2">Primaquine</td> <td align="left">6-ring</td> <td align="left">CG1 VAL 209 (A)</td> <td align="left">pi-H</td> <td align="left">4.24</td> <td align="left">âˆ’ 0.7</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">CG1 VAL 209 (A)</td> <td align="left">pi-H</td> <td align="left">4.52</td> <td align="left">âˆ’ 0.7</td> </tr> <tr> <td align="left">Tafenoquine</td> <td align="left" colspan="5">No measured interaction</td> </tr> <tr> <td align="left">Favipiravir</td> <td align="left">O 12</td> <td align="left">NZ LYS 94 (A)</td> <td align="left">H-acceptor</td> <td align="left">3.12</td> <td align="left">âˆ’ 3.5</td> </tr> <tr> <td align="left">Ribavirin</td> <td align="left">O 15</td> <td align="left">NZ LYS 562 (A)</td> <td align="left">H-acceptor</td> <td align="left">3.03</td> <td align="left">âˆ’ 3.6</td> </tr> <tr> <td align="left" rowspan="6">Galidesivir</td> <td align="left">N 14</td> <td align="left">O ASN 210 (A)</td> <td align="left">H-donor</td> <td align="left">3.05</td> <td align="left">âˆ’ 1.0</td> </tr> <tr> <td align="left">O 29</td> <td align="left">CE LYS 562 (A)</td> <td align="left">H-acceptor</td> <td align="left">3.16</td> <td align="left">âˆ’ 0.7</td> </tr> <tr> <td align="left">5-ring</td> <td align="left">CA VAL 209 (A)</td> <td align="left">pi-H</td> <td align="left">3.79</td> <td align="left">âˆ’ 2.1</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">CA VAL 209 (A)</td> <td align="left">pi-H</td> <td align="left">4.40</td> <td align="left">âˆ’ 0.5</td> </tr> <tr> <td align="left">5-ring</td> <td align="left">N ASN 210 (A)</td> <td align="left">pi-H</td> <td align="left">4.25</td> <td align="left">âˆ’ 2.7</td> </tr> <tr> <td align="left">6-ring</td> <td align="left">ND2 ASN 210 (A)</td> <td align="left">pi-H</td> <td align="left">4.58</td> <td align="left">âˆ’ 1.3</td> </tr> <tr> <td align="left" rowspan="2">Brincidovir</td> <td align="left">O 63</td> <td align="left">OE2 GLU 208 (A)</td> <td align="left">H-donor</td> <td align="left">2.79</td> <td align="left">âˆ’ 6.4</td> </tr> <tr> <td align="left">O 74</td> <td align="left">NE2 GLN 98 (A)</td> <td align="left">H-acceptor</td> <td align="left">3.01</td> <td align="left">âˆ’ 1.2</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="Tab5"><span class="label" tagx="label" title="label">Table 5</span> <p>Docking score and energy of ivermectin drug and 1Q2W of COVID-19 with site 1 of COVID-19 protease (PDB codeâ€‰=â€‰1Q2W)</p>  <div class="graphic" position="anchor" id="MO90" /> </div> <div class="table-wrap_UNKNOWN" id="Tab6"><span class="label" tagx="label" title="label">Table 6</span> <p>Docking score and energy of ivermectin drug with ACE-2 receptor (PDB codeâ€‰=â€‰6M0J)</p>  <div class="graphic" position="anchor" id="MO91" /> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 1</span></a></div>   <p>3d Docking of malaria and Ebola drugs and 1Q2W of COVID-19 with site 1 of COVID-19 protease (PDB codeâ€‰=â€‰1Q2W)</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 2</span></a></div>   <p>2d Docking of malaria and Ebola drugs and 1Q2W of COVID-19 with fixing the active site 1 of of COVID-19 protease (PDB codeâ€‰=â€‰1Q2W)</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 3</span></a></div>   <p>3d Docking of malaria and Ebola drugs with ACE-2 receptor (PDB codeâ€‰=â€‰6M0J)</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 4</span></a></div>   <p>2d Docking of malaria and Ebola drugs with ACE-2 RECEPTOR (PDB CODEâ€‰=â€‰6M0J)</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 5</span></a></div>   <p>Chemical structure of ivermectin</p>   </div> <p /> <p id="Par8">Brincidofovir (BCV) is an orally bioavailable, long-acting nucleotide analog broad-spectrum antiviral developedÂ by Chimerix Inc. of Durham, North Carolina, USA, for the treatment of double-stranded DNA (dsDNA) viruses (Lanier et al. <a href="#CR18">2010</a>). BCV is less toxic with an enhanced cellular penetration prodrug of cidofovir, wherein the cidofovir acyclic nucleoside monophosphate is conjugated through its phosphonate group to a lipid,Â 3-(hexadecyloxy)-1-propanol (Florescu and Keck <a href="#CR11">2014</a>). Being linked to a lipid particle, the compound ensures better and higher intracellular releases of cidofovir and lower plasma concentrations of the active drug, effectively increasing its antiviral activity. When intracellular, the released free cidofovir from the BCV is phosphorylated to its active metabolite cidofovir diphosphate, which, due to its structural similarity to the deoxycytidine triphosphate (dCTP) nucleotides, gets incorporated into the growing viral DNA strands (Parker et al. <a href="#CR28">2008a</a>, <a href="#CR29">b</a>). Once incorporated, it prevents further DNA polymerization and disrupts DNA replication of viruses. The drug received FDA Fast Track Designation and has been evaluated in healthy individuals in Phase I studies and in hematopoietic cell transplant recipients and other immunocompromised patients in Phase II/III clinical trials and revealed to be well tolerated and highly efficacious against adenoviruses, BK virus, herpes simplex viruses, and smallpox, but eventually somehow failed for cytomegalovirus (Chittick et al. <a href="#CR8">2017</a>; Hoiberg and Rapaka <a href="#CR16">1991</a>; Schepers). PreliminaryÂ in vitroÂ tests have also shown the drug potential for the treatment ofÂ Ebola virus disease, despite that Ebola is an RNA virus, albeit trials were eventually discontinued (Dunning et al. <a href="#CR10">2016</a>). Being acted on the Ebola RNA virus so, it is encouraging to act as well on the novel RNA SARS-CoV-2. In addition to its intracellular therapeutic strategy of arresting viral replication and packaging, our study shows here that it also interferes efficiently with the SARS-CoV-2 ACE2 receptor, revealing a different therapeutic mode of action through potentially blocking or inhibiting the virus entry to the host cell, thereby slowing the progression of the infection.</p> <p id="Par9">The second top-ranked drug is <i>tafenoquine</i> which is an orally active 8-aminoquinoline, a long-acting analog of primaquine, antimalarialÂ medicine developed by GlaxoSmithKline and 60 Degrees Pharmaceuticals (Baird <a href="#CR4">2018</a>; Frampton <a href="#CR12">2018</a>). The drug was FDA approved for the radical cure of <i>Plasmodium vivax</i> (<i>P. vivax</i>) malaria and the prophylaxis of malaria in 2018. The drug is active against pre-erythrocytic, erythrocytic forms and the gametocytes of Plasmodium species that include <i>P. falciparum</i> and <i>P. vivax</i> (Baird <a href="#CR4">2018</a>; Frampton <a href="#CR12">2018</a>). Clinical trials for this drug may be also recommended. Chloroquine, which is an antimalaria and immunosuppressive drug, has been recently shown to improve the outcomes in patients with the novel coronavirus pneumonia, which made the FDA issue an Emergency Use Authorization to be tested as a treatment for COVID-19, ranked at the fourth position in this study (Gao et al. <a href="#CR13">2020</a>).</p> <p id="Par10">Lastly, while we were working in this research, an Australian study showed that <i>ivermectin</i>, an antiparasitic drug, was effective against COVID-19 disease, although further clinical trials are underway to confirm this effectiveness (Caly et al. <a href="#CR6">2020</a>). We decided to do some investigations using molecular docking to check the binding interaction between ivermectin and the SARS-CoV-2 protease and receptor. We got comparable data to the antiviral <i>Brincidofovir</i> where the docking scores were -10.31 and -8.84 with the SARS-CoV-2 protease and ACE2 receptor (TablesÂ <a href="#Tab5">5</a> and <a href="#Tab6">6</a>), respectively. But overall, brincidofovir is better recommended because for its high lipophilicity â€œ5.54â€�, whereas for ivermectin it is â€œ2.01â€�.</p> <p id="Par11"> In conclusion, molecular modeling tools were used to screen for potential anti-SARS-CoV-2 therapeutic agents. After a virtual screening against SARS-CoV-2 protease and ACE2 receptor, a set of antivirals, antimalarials, and antimicrobials drugs showed a potent binding interaction, wherein biocidofovir was the top hit. Therefore, repurposing of biocidofovir against COVID-19 disease is suggested for clinical trials.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p>Below is the link to the electronic supplementary material. </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="13721_2020_263_MOESM1_ESM.docx">LINK</a> <p>Supplementary material 1 (DOCX 27413 kb)</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="title" tagx="title" title="title">
Compliance with ethical standards</div> <div class="title" tagx="title" title="title">
Conflict of interest</div> <p id="Par12">The authors declare no competing financial interest.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdellattif</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hussien</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alzahrani</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">New approaches of 4-aryl-2-hydrazinothiazole derivatives synthesis, molecular docking, and biological evaluations</span><span class="source" tagx="source" title="source">Int J Pharm Sci Res</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">5060</span><span class="lpage" tagx="lpage" title="lpage">5078</span></span></li> <li tag="ref"><a name="CR2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdel-Rhman</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hussien</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahmoud</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hosny</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Synthesis, characterization, molecular docking and cytotoxicity studies on <i>N</i>-benzyl-2-isonicotinoylhydrazine-1-carbothioamide and its metal complexes</span><span class="source" tagx="source" title="source">J Mol Struct</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">1196</span><span class="fpage" tagx="fpage" title="fpage">417</span><span class="lpage" tagx="lpage" title="lpage">428</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molstruc.2019.06.092">10.1016/j.molstruc.2019.06.092</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Althagafi</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El-Metwaly</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farghaly</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of new Pt(IV)â€"thiazole complexes: analytical, spectral, molecular modeling and molecular docking studies and applications in two opposing pathways</span><span class="source" tagx="source" title="source">Appl Organomet Chem</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">e5099</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/aoc.5099">10.1002/aoc.5099</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baird</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tafenoquine for travelersâ€™ malaria: evidence, rationale and recommendations</span><span class="source" tagx="source" title="source">J Travel Med</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">tay110</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jtm/tay110">10.1093/jtm/tay110</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonanno</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">X-ray crystal structure of the SARS coronavirus main protease</span><span class="source" tagx="source" title="source">Worldw Protein Data Bank</span><span class="year" tagx="year" title="year">2003</span><span class="pub-id"><a href="https://dx.doi.org/10.2210/pdb1q2w/pdb">10.2210/pdb1q2w/pdb</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caly</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Druce</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Catton</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jans</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wagstaff</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span></span><span class="mixed-article-title" title="mixed-article-title">The FDA-approved drug ivermectin inhibits the replication of SARS-CoV-2 in vitro</span><span class="source" tagx="source" title="source">Antivir Res</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">178</span><span class="fpage" tagx="fpage" title="fpage">104787</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2020.104787">10.1016/j.antiviral.2020.104787</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32251768">32251768</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiang</span><span class="given-names" tagx="given-names" title="given-names">C-M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Faculty opinions recommendation of SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor</span><span class="source" tagx="source" title="source">Fac Opin Ltd</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.3410/f.737494462.793573238">10.3410/f.737494462.793573238</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chittick</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morrison</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brundage</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nichols</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Short-term clinical safety profile of brincidofovir: a favorable benefitâ€"risk proposition in the treatment of smallpox</span><span class="source" tagx="source" title="source">Antivir Res</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">143</span><span class="fpage" tagx="fpage" title="fpage">269</span><span class="lpage" tagx="lpage" title="lpage">277</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2017.01.009">10.1016/j.antiviral.2017.01.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28093339">28093339</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Control CfD, Prevention (2014) Vaccine testing and the approval process. Webpage. Updated: May</span></li> <li tag="ref"><a name="CR10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunning</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Experimental treatment of Ebola virus disease with brincidofovir</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">e0162199</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0162199">10.1371/journal.pone.0162199</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27611077">27611077</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Florescu</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keck</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Development of CMX001 (Brincidofovir) for the treatment of serious diseases or conditions caused by dsDNA viruses</span><span class="source" tagx="source" title="source">Expert Rev Anti-infect Ther</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">1171</span><span class="lpage" tagx="lpage" title="lpage">1178</span><span class="pub-id"><a href="https://dx.doi.org/10.1586/14787210.2014.948847">10.1586/14787210.2014.948847</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25120093">25120093</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frampton</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tafenoquine: first global approval</span><span class="source" tagx="source" title="source">Drugs</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">1517</span><span class="lpage" tagx="lpage" title="lpage">1523</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s40265-018-0979-2">10.1007/s40265-018-0979-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30229442">30229442</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">Breakthrough: chloroquine phosphate has shown apparent efficacy in treatment of COVID-19 associated pneumonia in clinical studies</span><span class="source" tagx="source" title="source">BioSci Trends</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">72</span><span class="lpage" tagx="lpage" title="lpage">73</span><span class="pub-id"><a href="https://dx.doi.org/10.5582/bst.2020.01047">10.5582/bst.2020.01047</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32074550">32074550</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> GSK files for approval of worldâ€™s first malaria vaccine (2014) Pharm J. 10.1211/pj.2014.20065965</span></li> <li tag="ref"><a name="CR15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hoffmann M et al. (2020) SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor Cell</span></li> <li tag="ref"><a name="CR16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoiberg</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rapaka</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span></span><span class="mixed-article-title" title="mixed-article-title">FDA regulatory requirements for investigational new drugs and new drug applications: an update</span><span class="source" tagx="source" title="source">Am Psychol Assoc (APA)</span><span class="year" tagx="year" title="year">1991</span><span class="pub-id"><a href="https://dx.doi.org/10.1037/e496242006-019">10.1037/e496242006-019</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ketan ZA, Naser AW, AL-Shmgani HS (2020) Synthesis, design, docking and biological activity of new benzo hydrazide derivatives (2020) Int J Pharm Res. 10.31838/ijpr/2020.12.03.070</span></li> <li tag="ref"><a name="CR17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitchen</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Decornez</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furr</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bajorath</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Docking and scoring in virtual screening for drug discovery: methods and applications</span><span class="source" tagx="source" title="source">Nat Rev Drug Discov</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">935</span><span class="lpage" tagx="lpage" title="lpage">949</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrd1549">10.1038/nrd1549</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15520816">15520816</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanier</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Development of CMX001 for the treatment of poxvirus infections</span><span class="source" tagx="source" title="source">Viruses</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">2740</span><span class="lpage" tagx="lpage" title="lpage">2762</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v2122740">10.3390/v2122740</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21499452">21499452</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lengauer</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rarey</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Computational methods for biomolecular docking</span><span class="source" tagx="source" title="source">Curr Opin Struct Biol</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">402</span><span class="lpage" tagx="lpage" title="lpage">406</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/s0959-440x(96)80061-3">10.1016/s0959-440x(96)80061-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8804827">8804827</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mashat</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Babgi</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hussien</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arshad</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdellattif</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Synthesis, structures, DNA-binding and anticancer activities of some copper (I)-phosphine complexes</span><span class="source" tagx="source" title="source">Polyhedron</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">158</span><span class="fpage" tagx="fpage" title="fpage">164</span><span class="lpage" tagx="lpage" title="lpage">172</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.poly.2018.10.062">10.1016/j.poly.2018.10.062</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Memish</span><span class="given-names" tagx="given-names" title="given-names">ZA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Middle East respiratory syndrome coronavirus in bats</span><span class="source" tagx="source" title="source">Saudi Arabia Emerg Infect Dis</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">1819</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1911.131172">10.3201/eid1911.131172</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24206838">24206838</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mubarak</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khanday</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lone</span><span class="given-names" tagx="given-names" title="given-names">AUH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mathematical analysis based on eigenvalue approach to study liver metastasis disease with applied drug therapy</span><span class="source" tagx="source" title="source">Netw Model Anal Health Inf Bioinf</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s13721-020-00231-0">10.1007/s13721-020-00231-0</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naik</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Modeling the mechanics of calcium regulation in T lymphocyte: a finite element method approach</span><span class="source" tagx="source" title="source">Int J Biomath</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1142/s1793524520500382">10.1142/s1793524520500382</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naik</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pardasani</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Three-dimensional finite element model to study calcium distribution in oocytes</span><span class="source" tagx="source" title="source">Netw Model Anal Health Inf Bioinf</span><span class="year" tagx="year" title="year">2017</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s13721-017-0158-5">10.1007/s13721-017-0158-5</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naik</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pardasani</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span></span><span class="mixed-article-title" title="mixed-article-title">2D finite-element analysis of calcium distribution in oocytes</span><span class="source" tagx="source" title="source">Netw Model Anal Health Inf Bioinf</span><span class="year" tagx="year" title="year">2018</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s13721-018-0172-2">10.1007/s13721-018-0172-2</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naik</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Modeling and simulation of spatial-temporal calcium distribution in T lymphocyte cell by using a reaction-diffusion equation</span><span class="source" tagx="source" title="source">J Bioinf Comput Biol</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">2050013</span><span class="pub-id"><a href="https://dx.doi.org/10.1142/s0219720020500134">10.1142/s0219720020500134</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Organization WH Ebola virus disease. Fact sheet N 103. Updated September 2014. Fecha de consulta: 18 de octubre de 2014</span></li> <li tag="ref"><a name="CR28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oberle</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Touchette</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buller</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Efficacy of therapeutic intervention with an oral ether lipid analogue of cidofovir (CMX001) in a lethal mousepox model</span><span class="source" tagx="source" title="source">Antivir Res</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">A61</span><span class="lpage" tagx="lpage" title="lpage">A62</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2008.01.132">10.1016/j.antiviral.2008.01.132</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Efficacy of therapeutic intervention with an oral etherâ€"lipid analogue of cidofovir (CMX001) in a lethal mousepox model</span><span class="source" tagx="source" title="source">Antiviral Res</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">39</span><span class="lpage" tagx="lpage" title="lpage">49</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2007.08.003">10.1016/j.antiviral.2007.08.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17904231">17904231</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Schepers J. Chimerix announces emergency investigational new drug applications for brincidofovir authorized by FDA for patients with Ebola virus disease</span></li> <li tag="ref"><a name="CR30" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sikka</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The emergence of Zika virus as a global health security threat: a review and a consensus statement of the INDUSEM Joint Working Group (JWG)</span><span class="source" tagx="source" title="source">J Global Infect Dis</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">3</span><span class="pub-id"><a href="https://dx.doi.org/10.4103/0974-777X.176140">10.4103/0974-777X.176140</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemiology, genetic recombination, and pathogenesis of coronaviruses</span><span class="source" tagx="source" title="source">Trends Microbiol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">490</span><span class="lpage" tagx="lpage" title="lpage">502</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2016.03.003">10.1016/j.tim.2016.03.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27012512">27012512</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baric</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Receptor recognition by the novel coronavirus from Wuhan: an analysis based on decade-long structural studies of SARS coronavirus</span><span class="source" tagx="source" title="source">J Virol</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/jvi.00127-20">10.1128/jvi.00127-20</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of therapeutic targets for SARS-CoV-2 and discovery of potential drugs by computational methods</span><span class="source" tagx="source" title="source">Acta Pharmaceutica Sinica B</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">766</span><span class="lpage" tagx="lpage" title="lpage">788</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.apsb.2020.02.008">10.1016/j.apsb.2020.02.008</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiong</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="source" tagx="source" title="source">Spatial statistics and influencing factors of the novel coronavirus pneumonia 2019 epidemic in Hubei Province</span><span class="year" tagx="year" title="year">2020</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">China</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Res Square</span></span></li> </ul> </div> </div>  </body></html>