<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Biomol Struct Dyn</span><span class="iso-abbrev" title="iso-abbrev">J. Biomol. Struct. Dyn</span><span class="publisher-id" title="publisher-id">TBSD</span><span class="publisher-id" title="publisher-id">tbsd20</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Biomolecular Structure &amp;amp; Dynamics</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0739-1102</span><span class="issn-epub" tagx="issn" title="issn-epub">1538-0254</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor &amp;amp; Francis</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1080/07391102.2020.1763202">10.1080/07391102.2020.1763202</a></span><span class="publisher-id" title="publisher-id">1763202</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Express Communication</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Identification of new anti-nCoV drug chemical compounds from Indian spices exploiting SARS-CoV-2 main protease as target</div> <div class="alt-title" title="alt-title">
Umesh et al.</div> <div class="alt-title" title="alt-title">
Journal of Biomolecular Structure and Dynamics </div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">â€‰</span><span class="given-names" tagx="given-names" title="given-names">Umesh</span></span><a href="#AF0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kundu</span><span class="given-names" tagx="given-names" title="given-names">Debanjan</span></span><a href="#AF0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selvaraj</span><span class="given-names" tagx="given-names" title="given-names">Chandrabose</span></span><a href="#AF0002"><sup>b</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">Sanjeev Kumar</span></span><a href="#AF0002"><sup>b</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dubey</span><span class="given-names" tagx="given-names" title="given-names">Vikash Kumar</span></span><a href="#AF0001"><sup>a</sup></a><a href="#AN0001" /></span><span class="citation_author_institution" id="AF0001">[a], <span class="institution" title="institution">School of Biochemical Engineering, Indian Institute of Technology BHU</span><span class="country" tagx="country" title="country">India</span></span><span class="citation_author_institution" id="AF0002">[b], <span class="institution" title="institution">Computer Aided Drug Design and Molecular Modeling Lab, Department of Bioinformatics, Alagappa University</span><span class="country" tagx="country" title="country">India</span></span></div> <div class="author-notes" title="author-notes"> <div class="fn-type-" title=""> <p>Supplemental data for this article can be accessed online at <a href="https://doi.org/10.1080/07391102.2020.1763202">https://doi.org/10.1080/07391102.2020.1763202</a>.</p> </div> <div class="corresp" title="corresp"><b>CONTACT</b> Vikash Kumar Dubey <span class="email" tagx="email" title="email">vkdubey.bce@iitbhu.ac.in</span><span class="institution" title="institution">School of Biochemical Engineering, Indian Institute of Technology BHU</span>, Varanasi, UP 221005, <span class="country" tagx="country" title="country">India</span></div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-5-5</span></span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="history" title="history"><span class="received" title="received">received: 2020-4-06</span><span class="accepted" title="accepted">accepted: 2020-4-27</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Informa UK Limited, trading as Taylor &amp;amp; Francis Group</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the COVID-19 pandemic or until permissions are revoked in writing. Upon expiration of these permissions, PMC is granted a perpetual license to make this article available via PMC and Europe PMC, consistent with existing copyright protections.</span></span></div><a class="self-uri" href="TBSD_A_1763202.pdf" title="self-uri">TBSD_A_1763202.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>The 2019-novel coronavirus (nCoV) has caused a global health crisis by causing coronavirus disease-19 (COVID-19) pandemic in the human population. The unavailability of specific vaccines and anti-viral drug for nCoV, science demands sincere efforts in the field of drug design and discovery for COVID-19. The novel coronavirus main protease (SARS-CoV-2 Mpro) play a crucial role during the disease propagation, and hence SARS-CoV-2 Mpro represents as a drug target for the drug discovery. Herein, we have applied bioinformatics approach for screening of chemical compounds from Indian spices as potent inhibitors of SARS-CoV-2 main protease (PDBID: 6Y84). The structure files of Indian spices chemical compounds were taken from PubChem database or Zinc database and screened by molecular docking, by using AutoDock-4.2, MGLTools-1.5.6, Raccoon virtual screening tools. Top 04 hits based on their highest binding affinity were analyzed. Carnosol exhibited highest binding affinity -8.2â€‰Kcal/mol and strong and stable interactions with the amino acid residues present on the active site of SARS-CoV-2 Mpro. Arjunglucoside-I (-7.88â€‰Kcal/mol) and Rosmanol (-7.99â€‰Kcal/mol) also showed a strong and stable binding affinity with favourable ADME properties. These compounds on MD simulations for 50<b>â€‰</b>ns shows strong hydrogen-bonding interactions with the protein active site and remains stable inside the active site. Our virtual screening results suggest that these small chemical molecules can be used as potential inhibitors against SARS-CoV-2 Mpro and may have an anti-viral effect on nCoV. However, further validation and investigation of these inhibitors against SARS-CoV-2 main protease are needed to claim their candidacy for clinical trials.</p> <p>Communicated by Ramaswamy H. Sarma</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Novel coronavirus (nCoV)</div> <div class="kwd" title="kwd">
corona virus disease-2019 (COVID-19)</div> <div class="kwd" title="kwd">
novel coronavirus main protease (SARS-CoV-2 Mpro)</div> <div class="kwd" title="kwd">
molecular docking</div> <div class="kwd" title="kwd">
bioinformatics</div> <div class="kwd" title="kwd">
ADME</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">RUSA-Phase 2.0 Policy (TNmulti-Gen), Dept. of Edn, Govt. of India</span></div> </div> <div class="funding-statement" title="funding-statement">
The infrastructure facilities by IIT (BHU) Varanasi is acknowledged. Umesh is supported by CSIR project [27(344)/19-EMR-II] to VKD. DK acknowledges research fellowship from IIT (BHU) Varanasi. VKD also acknowledges SERB funding [CVD/2020/000031]. The Authors CS and SKS thankfully acknowledge RUSA-Phase 2.0 Policy (TNmulti-Gen), Dept. of Edn, Govt. of India (Grant No: F.24-51/2014-U). </div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Since December 2019, a outbreak of COVID-19 with massive global impact has started in Hubei and Wuhan city in China caused by a novel coronavirus, SARS-CoV-2 (Bhoopathi etÂ al., <a href="#CIT0004">2020</a>; Kumar &amp;amp; Rathi, <a href="#CIT0015">2020</a>; Wang etÂ al., <a href="#CIT0030">2020</a>). On January 2020, World health organization emergency committee declared a global health emergency based on the high rate of spreading of the infection with high fatality rate (Chhikara etÂ al., <a href="#CIT0006">2020</a>; Kumar &amp;amp; Rathi, <a href="#CIT0015">2020</a>). As of now on 4th April 2020, Europe and USA are new epicentres of COVID-19 Pandemic. Even in the past, different coronaviruses have caused multiple human diseases that resulted in global epidemics such as the Middle East respiratory syndrome (MERS), severe acute respiratory syndrome (SARS) and coronavirus disease 2019 (COVID-19). While the coronaviruses have a significant impact on human health, the general public has an inferior awareness of coronavirus pathogenesis and infection. COVID-19 was declared a pandemic in March 2020 as the worldwide human population is facing high risk from contracting the nCoV infection. Many countries, including India, has announced lockdown in the country and to maintain social distancing to avoid further spread as no drug or vaccine is available against SARS-CoV-2. Coronavirus has been classified into four subfamilies based on their shape and host. These subfamilies are alpha-coronavirus, beta-coronavirus, gamma-coronavirus and delta coronavirus (Paules etÂ al., <a href="#CIT0021">2020</a>). Alpha- coronavirus and beta-coronavirus are considered to have been originated from bats, while the gamma and delta coronaviruses are considered to be derived from birds and pigs (Banerjee etÂ al., <a href="#CIT0002">2019</a>). Coronaviruses contain a positive sense, single-strand RNA genome coding for viral polymerase, RNA synthesis materials, and large non-structural polypeptide. Coronavirus genome contains transcriptional modification, including 5â€™methylated cap and a 3â€™polyadenylated tail. Coronaviruses have very high rates of error in RNA replication due to constant errors by RNA dependent RNA polymerase (Banerjee etÂ al., <a href="#CIT0002">2019</a>; Lau etÂ al., <a href="#CIT0017">2018</a>).</p> <p>Only a few protein crystal structures of SARS-CoV-2 are available on protein databank. SARS-CoV-2 main protease, a potential drug target, crystal structure (PDB-ID: 6Y84) was available and used for docking simulation and identification of potential drug molecule form Indian spices. SARS-CoV-2 main protease has a vital role in the processing of polyprotein that is translated from viral RNA, and the protease is considered as key for viral survival and growth (Prajapat etÂ al., <a href="#CIT0022">2020</a>).</p> <p>Medicinal plants yielding biologically active compounds have always been of great interest to scientists as they play an essential role in preventing human diseases. In the entire world, India is recognized where spices have been traditionally used as a source of medicine. Many active pharmaceutical gradients have been identified and extracted having a wide range of physiological and pharmacological properties (Sachan etÂ al., <a href="#CIT0023">2018</a>). Apart from the regular uses of spices in culinary activities, they are widely used as indigenous medicines, nutraceuticals, in aromatherapy, as natural colouring agents, perfumes, cosmetics etc. In recent years there has been experimental evidence on physiological benefits that could be drawn in the context of various diseases like diabetes, cardiovascular issues and inflammatory disorders like arthritis and cancer. They have also been identified as preventive agents in certain conditions. Spices like red pepper, garlic, and fenugreek have been reported to have hypercholesterolemic activity, whereas fenugreek and garlic are also known to reduce and control blood sugar levels (Bhagya &amp;amp; Raveendra, <a href="#CIT0003">2017</a>). The main active ingredient of turmeric, Curcumin has been widely studied to have a broad spectrum of medicinal value ranging from anti-cancerous, anti-inflammatory and an anti-amyloidogenic activity (Bhagya &amp;amp; Raveendra, <a href="#CIT0003">2017</a>). Several computational studies related to drug or vaccine development against SARS-CoV-2 is recently published (Aanouz etÂ al., <a href="#CIT0001">2020</a>; Gupta etÂ al., <a href="#CIT0012">2020</a>; Joshi etÂ al., <a href="#CIT0013">2020</a>; Muralidharan etÂ al., <a href="#CIT0019">2020</a>; Sarma etÂ al., <a href="#CIT0024">2020</a>).</p> <p>In the current study, we have utilized the prior knowledge on the medicinal values and potential applications of Indian spices, we have tried to explore if they can be used as novel agents for controlling SARS-CoV-2. The data generated using computational approaches give very encouraging results.</p> </div> <div class="materialandmethods" title="sec"> <div class="title" tagx="title" title="title">
Material and methods</div> <div class="datasourcesandpreparationtargetprotein" title="sec"> <div class="title" tagx="title" title="title">
Data sources and preparation target protein</div> <p>In this study, prepared list of 45 chemical compounds from Indian spices and compound structure file was downloaded from PubChem database or Zinc database. Structure of protein SARS-CoV-2 Mpro was downloaded from RCSB protein database. (PDB ID: 6Y84).</p> <p>The atomic coordinates of active site were defined using report available in the literature and identification of catalytic His41, His164 and Cys145 (Khaerunnisa etÂ al., <a href="#CIT0014">2020</a>). Energy minimization of the protein molecule was done by Swiss PDB Viewer (Guex &amp;amp; Peitsch, <a href="#CIT0011">1997</a>). Before docking, the assignment of charge, solvation parameters and fragmental volumes to the protein was done using the Autodock Tool 4 (ADT). The protein 6Y84 PDB molecule was further optimized by using ADT for the molecular docking using established procedure.</p> <p>The structure data files of the all chemical compounds were downloaded from PubChem database and converted into mol2 structures by using Open Babel. In order to further simplify the analyses, ligands were fist optimized and converted mol2 to PDBQT format by using the graphical user interface version of Raccoon. MGLTools-1.5.6, Raccoon is preparing AutoDock virtual screening tool-python (Forli, Scripps Research Institute).</p> </div> <div class="compoundscreeningusingraccoon|autodockprogram" title="sec"> <div class="title" tagx="title" title="title">
Compound screening using raccoon |AutoDock program</div> <p>Molecular screening of the all chemical compound libraries was performed by using Raccoon and MGLTools-1.5.6 software by Autodock as the engine for docking (Morris etÂ al., <a href="#CIT0018">2009</a>). During the molecular docking period, the ligands were considered as a flexible molecule and the protein was considered as a rigid structure molecule. The configuration file for the grid parameters file and docking parameters file was generated by using Autodock. Autodock and Autogrid tools integrated with the Autodock4 were used to generate grid maps for each atom of the ligand. The grid boxes were made, such as to include one site at a time and perform docking. Grid X, Y and Z coordinates were 9.204, âˆ’4.557, and 19.602. We analyzed each ligand by setting default docking parameters except in the number of runs: We ran the LGA for 100 runs with each ligand with an initial population size of 150 random starting positions and conformation, 2.5 million number of energy evaluations.</p> <p>The application of grid parameters file was also used to predict the amino acid residue in the active site of the protein that interacts with the ligands. Positional root-mean-square deviation (RMSD) values were less than 1.0<b>â€‰</b>Ã… considered ideal and clustered together for finding the favourable binding. The highest negative binding energy was considered as the ligand molecule with maximum binding affinity.</p> </div> <div class="analysesandvisualization" title="sec"> <div class="title" tagx="title" title="title">
Analyses and visualization</div> <p>Visual analysis of the docking site was performed using Pymol-2.3.3 and the results were validated by using AutoDock Tools-1.5.6 (Morris etÂ al., 1998). Binding interaction analyses of identifying inhibitor and the SARS-CoV-2 Mpro (PDB ID: 6Y84) was done using an online program by using Ligplot analysis (Wallace etÂ al., <a href="#CIT0029">1995</a>).</p> <div class="mdsimulations" title="sec"> <div class="title" tagx="title" title="title">
MD simulations</div> <p>The classical molecular dynamics simulation is carried out for the prepared co-crystal structure and selected ligand-binding complex poses through Desmond Molecular dynamics package incorporated in Schrodinger suite (Chow etÂ al., <a href="#CIT0007">2008</a>). Both the apo and ligand complex is solvated in the TIP3P model (specifies a three-site rigid water molecule with charges), using the volume occupancy in an orthorhombic box with periodic boundary conditions. For neutralizing the system, the overall charge of apo and ligand complex is solvated with appropriate cation (Na+) or anion (Cl-) along with a salt concentration of 0.15<b>â€‰</b>mol/L. The prepared system is energy minimized for a convergence threshold of 1.0<b>â€‰</b>kcal/mol/Ã… by using the steepest descent method, and for minimization and relaxation of the system, the NPT ensemble is applied (Selvaraj etÂ al., <a href="#CIT0025">2015</a>). The standard temperature is kept constant at 300<b>â€‰</b>K and pressure at the level of 1.013<b>â€‰</b>bar for the total simulation, and each simulation is started for the total time scale of 50<b>â€‰</b>ns. For the analysis of MD trajectories, the RMSD, RMSF and Hydrogen bond interactions are analyzed using the trajectory analysis incorporated in the Desmond (Selvaraj &amp;amp; Singh, 2014).</p> </div> </div> <div class="admeanalyses" title="sec"> <div class="title" tagx="title" title="title">
ADME analyses</div> <p><i>The absorption</i>, <i>distribution</i>, <i>metabolism</i>, and excretion (ADME) properties of the studied top hits compound were calculated by using online SwissADME Program (Guex &amp;amp; Peitsch, <a href="#CIT0011">1997</a>). The significant parameters for ADME associated properties such as Lipinskiâ€™s rule of five, pharmacokinetic properties the solubilities of drug and drug likeness were considered.</p> </div> </div> <div class="resultanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Result and discussion</div> <p>We have created a database of 45 Indian spices compounds and their structures downloaded from PubChem and Zinc databases. Four small molecules were selected based on their binding affinity with SARS-CoV-2 Mpro as shown in the <a href="#t0001">Table 1</a>. The list of remaining compounds used of moleculer docking is submitted as <a href="https://doi.org/10.1080/07391102.2020.1763202">supplementary material</a> (<a href="#t0001">Table 1S</a>). The top three compounds include Carnosol, Rosmanol, and Arjunglucoside-I. The further insights into binding interactions of these compounds with SARS-CoV-2 Mpro were analysed using ligplot as shown in <a href="#F0001">Figures 1</a> and <a href="#F0002">2</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Ligplot images showing both hydrogen and hydrophobic interactions by (a) Î± Ketoamide and (b) Carnosol with SARS- CoV-2 main protease. Circled residues represent interaction with active site residues. Cysteine145 forms hydrogen bonding with both the compounds.</p>   </div> <div class="table-wrap_UNKNOWN" id="t0001" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <p>Details of various kinds of interaction shown between the amino acids near the active site of SARS- CoV-2 main protease along with their respective inhibitor constant (Ki) and biological source and binding energy.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Compound</th> <th align="center">Source of the compound</th> <th align="center">Binding affinity (kcal/mol)</th> <th align="center">Amino acid residues</th> <th align="center">Inhibitor Constant (Ki)</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">Alpha-ketoamide</td> <td align="center">Positive Control</td> <td align="center">âˆ’9.48</td> <td align="center">Â </td> <td align="center">112.59â€‰nM</td> </tr> <tr> <td align="left">Carnosol</td> <td align="center"><i>R. officinalis</i></td> <td align="center">âˆ’8.2</td> <td align="center"><b>Cys145, His164,</b> Glu166, Gln189, Met165, Arg188, Cys44, Met49, *<b>His41</b>, Thr45, Thr25, Thr26, Leu27, Gly143, Ser46, Asn142</td> <td align="center">969.58â€‰nM</td> </tr> <tr> <td align="left">Rosmanol</td> <td align="center"><i>R. Officinalis</i></td> <td align="center">âˆ’7.99</td> <td align="center"><b>Cys145,</b> Gly143, Thr25, <b>His41</b>, Met49, Asn142, His143, Glu166, Leu141, Ser144, Met165, Phe140</td> <td align="center">1.38 ÂµM</td> </tr> <tr> <td align="left">Arjunglucoside-I</td> <td align="center"><i>Terminalia chebula</i></td> <td align="center">âˆ’7.88</td> <td align="center"><b>Cys145</b>, Ser144, Leu141, Met165, Phe140, His163, Glu166, Asn142, <b>His41</b>, Gly143, Thr26, Leu27, Thr25</td> <td align="center">1.67 ÂµM</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>The active site residues are indicated in bold.</p> </div> </div> </div> <p>SARS-CoV-2 Mpro Protein (PDBID- 6Y84) active site defined but unliganded (Owen etÂ al., 2019) was used and the molecules where docked in the region where Î±â€"ketoamide was bound to protein (PDBID- 6Y2F) (SARS- CoV-2) main protease with bound with Î±â€"ketoamide (Zhang etÂ al., <a href="#CIT0031">2020</a>) involving His41,164 and Cys145 in the active site. Î±â€"ketoamide was hence used as a positive control for this study and resulting interactions were analytically compared with the other Indian spices. Î±â€"ketoamide showed interactions with Cys145 via hydrogen bonds and hydrophobic interactions His41. It was also seen to form hydrogen bonds with Thr26. Upon further analysis of the docking results, it was seen that our molecules had comparable binding energy as to Î±â€"ketoamide suggesting that these ingredients could indeed interact with same site (amino acids) as Î±â€"ketoamide. Carnasol (-8.2â€‰Kcal/mol) was seen to form hydrogen bonds with Leu141, Ser144 and Cys145. Hydrophobic interactions for Carnasol include His41, Thr25, Asn142, Phe140, Glu166, Met165 (<a href="#F0001">Figure 1</a>). Rosmanol, isolated from Rosemary (<i>Rosmarinus officinalis</i>), has been previously reported to have antioxidant activity, shows binding energy of -7.99â€‰Kcal/mol and forms hydrogen bonds with Leu141, Gly 143, Ser144 and Cys145. Further hydrophobic interactions are shown with Thr25, 26, His41, Phe140, His163, and Leu27 (<a href="#F0002">Figure 2a</a>). Arjunglucoside-I binds with hydrophobic interaction with His41, 164 and Cys145 additional to hydrogen bonding with Thr25, 26, His163, and Glu166 (<a href="#F0002">Figure 2b</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Ligplot images showing both hydrogen and hydrophobic interactions by (a) Rosmanol and (b) Arjunglucoside with SARS- CoV-2 main protease. Circled residues represent interaction with active site residues.</p>   </div> <div class="" title="sec"> <div class="title" tagx="title" title="title" /> <div class="mdsimulationsanalysis" title="sec"> <div class="title" tagx="title" title="title">
MD simulations analysis</div> <p>The results of MD simulation for both apo and ligand complex is analyzed for the 50<b>â€‰</b>ns of time scale to understand the dynamic behavior and stability. MD simulation is performed for the total of 50<b>â€‰</b>ns and the trajectories are for RMSD plot as shown in the <a href="#F0003">Figure 3</a>. The figure represents the colors with (a) Apo protein, complexed with (b) Alpha ketoamide, (c) Arjun glycoside, (d) Carnosol, and (e) Rosmanol for the 50<b>â€‰</b>ns of time scale. For the apo protein, observed the initial 45<b>â€‰</b>ns of MD simulation as stable, but after the 45<sup>th</sup> ns there is a sudden drift and that may occur due to the higher occupancy of flexible loops in the C-terminal region (182-304 residues). Even though the apo protein shows the value of âˆ¼2.0<b>â€‰</b>Ã… till 47<sup>th</sup> ns and remaining 3<b>â€‰</b>ns ends with âˆ¼3.4<b>â€‰</b>Ã…. For understanding the residue wise fluctuation between the apo and ligand complex, the RMSF values are plotted. The RMSD deviation for the apo and complex proteins clearly shows the dynamic movement that occurs in the loop regions. For understanding the residues participated in the causative for fluctuations, the RMSF plot is provided in the <a href="#F0004">Figure 4</a>. Here, the amino acid in each position is calculated for its deviation value with the calculated timescale of 50<b>â€‰</b>ns. For apo protein, the amino acid from 70-75<sup>th</sup>, 160-175<sup>th</sup> position shows the deviation up to âˆ¼4Ã…, apart from these regions other amino acids are deviating from âˆ¼1-3Ã…. The deviation that occurs in the 70-75<sup>th</sup>, 160-175<sup>th</sup> position may be functionally reason for the drift in apo protein RMSD at 45<sup>th</sup> ns. While comparing the apo RMSF values with the complex protein RMSF, notably Carnosol is showing high deviations than the other ligand complexes. The amino acid positions from 40-60<sup>th</sup>, 150-155<sup>th</sup>, 178-200<sup>th</sup> position is showing higher deviations, that ranges from âˆ¼4-6Ã… deviations. Due to these positional amino acid fluctuations, the RMSD of Carnosol deviates approximately 4<b>â€‰</b>Ã… after the 26<sup>th</sup> ns. Similarly, another compound Rosmanol also showing the fluctuations in C-terminal regions, that results in the âˆ¼3-3.5<b>â€‰</b>Ã… deviations in the RMSD values. The protein secondary structure is framed as 3 sheets, 7 beta hairpins, 9 beta bulges, 13 strands, 32 beta turns, 3 gamma turns from this 182-304 residues are occupied dominantly by loop regions. These 122 residues in the C-terminal functionally act in the MD simulation, and thus the sudden drift happens in the 45<sup>th</sup> ns. This values of RMSD for the apo protein, is compared with ligand molecules in the <a href="#F0003">Figure 3</a>, and for understanding the deviations, the 25<sup>th</sup> to 50<sup>th</sup> ns is focused. The results of ligand complex for the MD simulation of 50<b>â€‰</b>ns of timescale shows that the ligand complex Alpha ketoamide (Red color) and Arjun glycoside (Blue color) shows stable throughout the simulation. While comparing these two Alpha ketoamide and Arjun glycoside ligand complex with apo protein, the ligand complex is matched with apo protein for the timescale of 45<sup>th</sup> ns. After the 45<sup>th</sup> ns the apo protein is drifted upwards, but the Alpha ketoamide and Arjun glycoside ligand complex remains stable. This may be due to the strong interaction pattern seen in Alpha ketoamide and Arjun glycoside interactions with the protein. Both the Alpha ketoamide and Arjun glycoside ligand bound complex are stable and positioned in the range of âˆ¼2Ã…, which is close to the apo protein till 47<sup>th</sup> ns. The <a href="#F0005">Figures 5a and 5b</a> shows the hydrogen bond interactions for the Alpha ketoamide and Arjun glycoside, which clearly shows the minimum participation of 2-3 hydrogen bonds seen in between the Alpha ketoamide and protein, and minimum participation of 4-6 hydrogen bonds between the Arjun glycoside and protein. This active participation of hydrogen bonds between the Alpha ketoamide and Arjun glycoside with protein makes the ligand complex stable for 50<b>â€‰</b>ns of MD simulations. The ligand Carnosol shows stable movement in terms of RMSD values by showing a narrow graph, but while comparing with apo protein, the Carnosol ligand complex is deviated from the 14<sup>th</sup> ns and stays in the range bound of âˆ¼3.4<b>â€‰</b>Ã… till the end of the simulations. For attaining the stable MD simulation for the Carnosol ligand bound complex, 2-3 hydrogen bonds are actively contributed and makes the complex stable in the dynamic state. The ligand Rosmanol bound complex shows stability till the 26<sup>th</sup> ns with the range of âˆ¼2.5<b>â€‰</b>Ã…, but after that, the drift happens to make the RMSD value deviated in upward direction and fix the positional RMSD with the range of âˆ¼4<b>â€‰</b>Ã…. This may be due to the loss of hydrogen bonding interactions after the 27<sup>th</sup> ns seen in the <a href="#F0005">Figure 5d</a>. <a href="#F0005">Figure 5d</a> says that the initial 27<b>â€‰</b>ns shows the hydrogen bond interactions in the range of 2-3 between the protein and ligand, but after 27<sup>th</sup> ns the ligand losses the hydrogen bonding ability and shows the hydrogen bond interactions in the range of 1-2 between the protein and ligand. Overall, the apo protein shows a narrow range of stability and notable fluctuations are also shown, that indicates the participation of loop structures. Those fluctuations are arrested through the active interactions of ligand molecules that shows strong binding between the protein and ligand.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>RMSD analysis of Apo and ligand complex (C-Alpha) in molecular dynamics simulations for the time scale of 50<b>â€‰</b>ns. The colors represented in the figure (a) Apo protein, (b) Alpha ketoamide, (c) Arjunglucoside, (d) Carnosol, and (e) Rosmanol.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>RMSF analysis of Apo and ligand complex in molecular dynamics simulations for the time scale of 50<b>â€‰</b>ns shows the deviation of each amino acid positions.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>Hydrogen bonds interaction between the protein and the ligand molecules. (a) Alpha ketoamide, (b) Arjunglucoside, (c) Carnosol, and (d) Rosmanol.</p>   </div> <p>We have also analysed the molecules we report here for violation of Lipinskiâ€™s rule (<a href="#t0002">Table 2</a>). Rosmanol, Carnosol fits perfectly as within the defined parameters for non-violation of Lipinskiâ€™s rule. The molecules have Log P values ranging from 1.15 to 3.27 which imply that these can effectively have suitable cell membrane permeability.</p> <div class="table-wrap_UNKNOWN" id="t0002" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2.</span> <p>Lipinskiâ€™s parameters for drug likeliness and ADMET properties of chosen ligands along with the standard drugs approved.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Molecule</th> <th align="center">LogP</th> <th align="center">TPSA (Angstrom)</th> <th align="center">MW (Da) (g/mol)</th> <th align="center">H-Donor</th> <th align="center">H Acceptor</th> <th align="center">Log S</th> <th align="center">Violations</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">Arjunglucoside-I</td> <td align="char" char=".">1.15</td> <td align="char" char=".">197.37</td> <td align="char" char=".">666.84</td> <td align="char" char=".">8</td> <td align="char" char=".">11</td> <td align="char" char=".">âˆ’2.09</td> <td align="char" char=".">3</td> </tr> <tr> <td align="left">Carnosol</td> <td align="char" char=".">2.97</td> <td align="char" char=".">66.76</td> <td align="char" char=".">330.42</td> <td align="char" char=".">2</td> <td align="char" char=".">4</td> <td align="char" char=".">âˆ’4.77</td> <td align="char" char=".">0</td> </tr> <tr> <td align="left">Rosmanol</td> <td align="char" char=".">2.50</td> <td align="char" char=".">86.99</td> <td align="char" char=".">346.42</td> <td align="char" char=".">3</td> <td align="char" char=".">5</td> <td align="char" char=".">âˆ’3.64</td> <td align="char" char=".">0</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Log P value represents the lipophilicity of the molecule whereas Log S value represents the water solubility. TPSA is Total Polar Solvent Accessibility. BA is oral bioavailability of drug and BBB is blood brain barrier permeability. Violation value land between 0-5, in Lipinskiâ€™s rule of five, if value is 0, then Drug likeness shows yes and if violations &amp;gt;0 then shows No.</p> </div> </div> </div> <p>Their number of hydrogen bond donors as well acceptors are well within range for Carnosol and Rosmanol but Arjunglucoside-I show high number of hydrogen bond donors and acceptors. Arjunglucoside is a large molecule having high molecular weight and total polar solvent area. Despite these factors it might prove to be essential in terms of potential drug once preceded with advanced studies.</p> </div> </div> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Amidst the unforeseeable outbreak of CoVID-19 there has been a sudden rise in demand of drug development, vaccines and identification of potential bioactive molecules which could prove to be useful fulfilling the purpose of broadening treatment options. In the quest for finding novel treatment regimen for these kinds of viral outbreaks, screening of already known molecules could also prove to be vital. In this context, we report here some active pharmaceutical ingredients which are present in the commonly used spices in India could prove to be useful. Preliminary <i>in silico</i> investigations show that indeed some molecules like Carnosol and Rosmanol have the properties which can further exploited and investigated for drug candidate against SARS-CoV-2. Although it is imperative to understand that development of rigid and highly specific treatment options will require further expeimental studies.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Table_1S-revised.pdf</div>  <div class="media" title="media"><a href="TBSD_A_1763202_SM9565.pdf">LINK</a></div> </div> </div> </div> <div class="back" title="back"> <div class="disclosurestatement" title="sec"> <div class="title" tagx="title" title="title">
Disclosure statement</div> <p>No potential conflict of interest was reported by the author(s).</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CIT0001" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aanouz</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belhassan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khatabi</span><span class="given-names" tagx="given-names" title="given-names">E. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lakhlifi</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Idrissi</span><span class="given-names" tagx="given-names" title="given-names">M. E.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouachrine</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Moroccan medicinal plants as inhibitors of COVID-19: Computational investigations</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1758790">10.1080/07391102.2020.1758790</a></span></span></li> <li tag="ref"><a name="CIT0002" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banerjee</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulcsar</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Misra</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frieman</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mossman</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Bats and coronaviruses</span>. <span class="source" tagx="source" title="source"><i>Viruses</i></span>, <span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">41</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v11010041">10.3390/v11010041</a></span></span></li> <li tag="ref"><a name="CIT0003" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhagya</span><span class="given-names" tagx="given-names" title="given-names">H. P.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raveendra</span><span class="given-names" tagx="given-names" title="given-names">Y. C.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Mulibenificial uses of spices: A brief review</span>. <span class="source" tagx="source" title="source"><i>Acta Scientific Nutritional Health</i></span>, <span class="volume" tagx="volume" title="volume">1</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">03</span>â€"<span class="lpage" tagx="lpage" title="lpage">06</span>.</span></li> <li tag="ref"><a name="CIT0004" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhoopathi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poma</span><span class="given-names" tagx="given-names" title="given-names">B. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolandaivel</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Novel 2019 coronavirus structure, mechanism of action, antiviral drug promises and rule out against itstreatment</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1758788">10.1080/07391102.2020.1758788</a></span></span></li> <li tag="ref"><a name="CIT0006" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chhikara</span><span class="given-names" tagx="given-names" title="given-names">B. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rathi</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span> &amp;amp; Poonam</span>, (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Corona virus SARS-CoV-2 disease COVID-19: Infection, prevention and clinical advances of the prospective chemical drug therapeutics: A review on Corona Virus Disease COVID-19, epidemiology, prevention, and anticipated therapeutic advances</span>. <span class="source" tagx="source" title="source"><i>Chemical Biology Letters</i></span>, <span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">63</span>â€"<span class="lpage" tagx="lpage" title="lpage">72</span>.</span></li> <li tag="ref"><a name="CIT0007" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chow</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rendleman</span><span class="given-names" tagx="given-names" title="given-names">C. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowers</span><span class="given-names" tagx="given-names" title="given-names">K. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dror</span><span class="given-names" tagx="given-names" title="given-names">R. O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">D. H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gullingsrud</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sacerdoti</span><span class="given-names" tagx="given-names" title="given-names">F. D.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaw</span><span class="given-names" tagx="given-names" title="given-names">D. E.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>, July 28). <i>Desmond performance on a cluster of multicore processors</i>. DE Shaw Research Technical Report DESRES/TR.</span></li> <li tag="ref"><a name="CIT0011" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guex</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peitsch</span><span class="given-names" tagx="given-names" title="given-names">M. C.</span></span></span> (<span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">SWISS-MODEL and the Swiss-PdbViewer: An environment for comparative protein modeling</span>. <span class="source" tagx="source" title="source"><i>Electrophoresis</i></span>, <span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">15</span>), <span class="fpage" tagx="fpage" title="fpage">2714</span>â€"<span class="lpage" tagx="lpage" title="lpage">2723</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/elps.1150181505">10.1002/elps.1150181505</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9504803">9504803</a></span></span></li> <li tag="ref"><a name="CIT0012" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Gupta, M. K., Vemula, S., Donde, R., Gouda, G., Behera, L., &amp;amp; Vadde, R. (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">In-silico approaches to detect inhibitors of the human severe acute respiratory syndrome coronavirus envelope protein ion channel</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1751300">10.1080/07391102.2020.1751300</a></span></span></li> <li tag="ref"><a name="CIT0013" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joshi</span><span class="given-names" tagx="given-names" title="given-names">R. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jagdale</span><span class="given-names" tagx="given-names" title="given-names">S. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bansode</span><span class="given-names" tagx="given-names" title="given-names">S. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shankar</span><span class="given-names" tagx="given-names" title="given-names">S. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tellis</span><span class="given-names" tagx="given-names" title="given-names">M. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandya</span><span class="given-names" tagx="given-names" title="given-names">V. K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulkarni</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Discovery of potential multi-target-directed ligands by targeting host-specific SARS-CoV-2 structurally conserved main protease</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1760137">10.1080/07391102.2020.1760137</a></span></span></li> <li tag="ref"><a name="CIT0014" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khaerunnisa</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurniawan</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Awaluddin</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suhartati</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soetjipto</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Potential inhibitor of COVID-19 main protease (Mpro) from several medicinal plant compounds by molecular docking study</span>. <span class="source" tagx="source" title="source"><i>Preprints</i>,</span><span class="pub-id"><a href="https://dx.doi.org/10.20944/preprints202003.0226.v1">10.20944/preprints202003.0226.v1</a></span></span></li> <li tag="ref"><a name="CIT0015" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rathi</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Coronavirus disease COVID-19: A new threat to public health</span>. <span class="source" tagx="source" title="source"><i>Current Topics in Medicinal Chemistry</i></span>, <span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">8</span>), 599â€"600. <span class="pub-id"><a href="https://dx.doi.org/10.2174/1568026620999200305144319">10.2174/1568026620999200305144319</a></span></span></li> <li tag="ref"><a name="CIT0017" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lau, S. K. P., Wong, E. Y., Tsang, C. C., Ahmed, S. S., Au-Yeung, R. K., Yuen, K. Y., Wernery, U. &amp;amp; Woo, P. C. (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Discovery and sequence analysis of four deltacoronaviruses from birds in the Middle East reveal interspecies jumping with recombination as a potential mechanism for avian-to-avian and avian-to-mammalian transmission</span>. <span class="source" tagx="source" title="source"><i>Journal of Virology</i></span>, <span class="volume" tagx="volume" title="volume">92</span>(<span class="issue" tagx="issue" title="issue">15</span>).</span></li> <li tag="ref"><a name="CIT0018" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huey</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindstrom</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanner</span><span class="given-names" tagx="given-names" title="given-names">M. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belew</span><span class="given-names" tagx="given-names" title="given-names">R. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodsell</span><span class="given-names" tagx="given-names" title="given-names">D. S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olson</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Autodock4 and AutoDockTools4: Automated docking with selective receptor flexiblity</span>. <span class="source" tagx="source" title="source"><i>Journal of Computational Chemistry</i></span>, <span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">16</span>), <span class="fpage" tagx="fpage" title="fpage">2785</span>â€"<span class="lpage" tagx="lpage" title="lpage">2791</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.21256">10.1002/jcc.21256</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19399780">19399780</a></span></span></li> <li tag="ref"><a name="CIT0019" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muralidharan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakthivel</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Velmurugan</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gromiha</span><span class="given-names" tagx="given-names" title="given-names">M. M.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Computational studies of drug repurposing and synergism of lopinavir, oseltamivir andritonavir binding with SARS-CoV-2 Protease against COVID-19</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1752802">10.1080/07391102.2020.1752802</a></span></span></li> <li tag="ref"><a name="CIT0020" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Owen</span><span class="given-names" tagx="given-names" title="given-names">C. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lukacik</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strain-Damerell</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douangamath</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Powell</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fearon</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brandao-Neto</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crawshaw</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aragao</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flaig</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">D. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McAuley</span><span class="given-names" tagx="given-names" title="given-names">K. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazzorana</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuart</span><span class="given-names" tagx="given-names" title="given-names">D. I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Delft</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walsh</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). SARS-CoV-2 main protease with unliganded active site (2019-nCoV, coronavirus disease 2019, COVID-19). 25 March PDB Entry â€" 6Y84.</span></li> <li tag="ref"><a name="CIT0021" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paules</span><span class="given-names" tagx="given-names" title="given-names">C. I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marston</span><span class="given-names" tagx="given-names" title="given-names">H. D.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fauci</span><span class="given-names" tagx="given-names" title="given-names">A. S.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Coronavirus infections-more than Just the common cold</span>. <span class="source" tagx="source" title="source">JAMA</span>, <span class="volume" tagx="volume" title="volume">323</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">707</span>â€"<span class="lpage" tagx="lpage" title="lpage">708</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1001/jama.2020.0757">10.1001/jama.2020.0757</a></span></span></li> <li tag="ref"><a name="CIT0022" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prajapat</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarma</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shekhar</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avti</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sinha</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaur</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattacharyya</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bansal</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medhi</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Drug targets for corona virus: A systematic review</span>. <span class="source" tagx="source" title="source"><i>Indian Journal of Pharmacology</i></span>, <span class="volume" tagx="volume" title="volume">52</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">56</span>â€"<span class="lpage" tagx="lpage" title="lpage">65</span>. <span class="pub-id"><a href="https://dx.doi.org/10.4103/ijp.IJP_115_20">10.4103/ijp.IJP_115_20</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32201449">32201449</a></span></span></li> <li tag="ref"><a name="CIT0023" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sachan</span><span class="given-names" tagx="given-names" title="given-names">K. R. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumari</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Medicinal uses of our spices used in our traditional culture</span>. <span class="source" tagx="source" title="source"><i>Journal of Medicinal Plant Studies</i></span>, <span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">116</span>â€"<span class="lpage" tagx="lpage" title="lpage">122</span>.</span></li> <li tag="ref"><a name="CIT0024" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sarma, P., Sekhar, N., Prajapat, M., Avti, P., Kaur, H., Kumar, S., Singh, S., Kumar, H., Prakash, A., Dhibar, D. P., &amp;amp; Medhi, B. (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">In-silico homology assisted identification of inhibitor of RNA binding against 2019-nCoV N-protein (N terminal domain)</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1753580">10.1080/07391102.2020.1753580</a></span></span></li> <li tag="ref"><a name="CIT0025" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selvaraj</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Priya</span><span class="given-names" tagx="given-names" title="given-names">R. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J. K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Mechanistic insights of SrtAâ€"LPXTG blockers targeting the transpeptidase mechanism in <i>Streptococcus mutans</i></span>. <span class="source" tagx="source" title="source"><i>RSC Advances</i></span>, <span class="volume" tagx="volume" title="volume">5</span>(<span class="issue" tagx="issue" title="issue">122</span>), <span class="fpage" tagx="fpage" title="fpage">100498</span>â€"<span class="lpage" tagx="lpage" title="lpage">100510</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1039/C5RA12869B">10.1039/C5RA12869B</a></span></span></li> <li tag="ref"><a name="CIT0026" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selvaraj</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Validation of potential inhibitors for SrtA against Bacillus anthracis by combined approach of ligand-based and molecular dynamics simulation</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure &amp;amp; Dynamics</i></span>, <span class="volume" tagx="volume" title="volume">32</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">1333</span>â€"<span class="lpage" tagx="lpage" title="lpage">1349</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2013.818577">10.1080/07391102.2013.818577</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23869520">23869520</a></span></span></li> <li tag="ref"><a name="CIT0029" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallace</span><span class="given-names" tagx="given-names" title="given-names">A. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laskowski</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thornton</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span></span> (<span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">LIGPLOT: A program to generate schematic diagrams of protein-ligand interactions</span>. <span class="source" tagx="source" title="source"><i>Protein Engineering</i>, <i>Design and Selection</i></span>, <span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">127</span>â€"<span class="lpage" tagx="lpage" title="lpage">134</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/protein/8.2.127">10.1093/protein/8.2.127</a></span></span></li> <li tag="ref"><a name="CIT0030" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">S. H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yue</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C. X.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Review and prospect of pathological features of Corona virus disease</span>. <span class="source" tagx="source" title="source">Fa yi <i>Xue</i> za <i>Zhi</i></span>, <span class="volume" tagx="volume" title="volume">36</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">16</span>â€"<span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="pub-id"><a href="https://dx.doi.org/10.12116/j.issn.1004-5619.2020.01.004">10.12116/j.issn.1004-5619.2020.01.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32198986">32198986</a></span></span></li> <li tag="ref"><a name="CIT0031" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Curth</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauerhering</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Becker</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rox</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilgenfeld</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved Î±-ketoamide inhibitors</span>. <span class="source" tagx="source" title="source"><i>Science</i></span>, <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">eabb3405</span>.</span></li> </ul> </div> </div>  </body></html>