<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Comput Biol</span><span class="iso-abbrev" title="iso-abbrev">PLoS Comput Biol</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">ploscomp</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Computational Biology</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1553-734X</span><span class="issn-epub" tagx="issn" title="issn-epub">1553-7358</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7494101">7494101</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/32881984">32881984</a></span><span class="publisher-id" title="publisher-id">PCOMPBIOL-D-19-01410</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pcbi.1008122">10.1371/journal.pcbi.1008122</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span>: <span class="subject" title="subject">Phylogenetics</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span>: <span class="subject" title="subject">Phylogenetics</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Data Management</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span>: <span class="subject" title="subject">Phylogenetics</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and health sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epidemiology</span>: <span class="subject" title="subject">HIV epidemiology</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span>: <span class="subject" title="subject">Pathogens</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Engineering and Technology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Signal Processing</span>: <span class="subject" title="subject">Autocorrelation</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Mathematical and Statistical Techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Statistical Methods</span>: <span class="subject" title="subject">Autocorrelation</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physical Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Mathematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Statistics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Statistical Methods</span>: <span class="subject" title="subject">Autocorrelation</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epidemiology</span>: <span class="subject" title="subject">Infectious Disease Epidemiology</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Conditions</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span>: <span class="subject" title="subject">Infectious Disease Epidemiology</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physical Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Mathematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Applied Mathematics</span>: <span class="subject" title="subject">Algorithms</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Simulation and Modeling</span>: <span class="subject" title="subject">Algorithms</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span>: <span class="subject" title="subject">Public and Occupational Health</span></div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">People and places</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Geographical locations</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Europe</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">European Union</span>: <span class="subject" title="subject">Sweden</span></div> </div> </div> </div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Inferring transmission heterogeneity using virus genealogies: Estimation and targeted prevention</div> <div class="alt-title" title="alt-title">
Inferring transmission heterogeneity using virus genealogies</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9316-3276</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Yunjun</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Writing â€" original draft</span><span class="role" tagx="role" title="role">Writing â€" review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-8160-2588</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leitner</span><span class="given-names" tagx="given-names" title="given-names">Thomas</span></span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Writing â€" original draft</span><span class="role" tagx="role" title="role">Writing â€" review &amp;amp; editing</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-9020-0521</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albert</span><span class="given-names" tagx="given-names" title="given-names">Jan</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff004"><sup>4</sup></a><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9228-7357</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Britton</span><span class="given-names" tagx="given-names" title="given-names">Tom</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Writing â€" original draft</span><span class="role" tagx="role" title="role">Writing â€" review &amp;amp; editing</span><a href="#aff002"><sup>2</sup></a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Department of Biostatistics, School of Public Health, Peking University, Beijing, China</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Department of Mathematics, Stockholm University, Stockholm, Sweden</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">Theoretical Biology and Biophysics Group, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Department of Microbiology, Tumor and Cell Biology, Karolinska Institute, Stockholm, Sweden</span></span><span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">Department of Clinical Microbiology, Karolinska University Hospital, Stockholm, Sweden</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouyos</span><span class="given-names" tagx="given-names" title="given-names">Roger Dimitri</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">University of Zurich, SWITZERLAND</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">yunjun.zhang@pku.edu.cn</span></div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-9</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-9-9</span></span><span class="pub-date-ecorrected" title="pub-date-ecorrected">ecorrected: <span>2020-10-10</span></span><span class="volume" tagx="volume" title="volume">16</span><span class="issue" tagx="issue" title="issue">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1008122</span><span class="history" title="history"><span class="received" title="received">received: 2019-8-25</span><span class="accepted" title="accepted">accepted: 2020-7-2</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article, free of all copyright, and may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose. The work is made available under the <a href="https://creativecommons.org/publicdomain/zero/1.0/">Creative Commons CC0</a> public domain dedication.</span></span></div><a class="self-uri" href="pcbi.1008122.pdf" title="self-uri">pcbi.1008122.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Spread of HIV typically involves uneven transmission patterns where some individuals spread to a large number of individuals while others to only a few or none. Such transmission heterogeneity can impact how fast and how much an epidemic spreads. Further, more efficient interventions may be achieved by taking such transmission heterogeneity into account. To address these issues, we developed two phylogenetic methods based on virus sequence data: 1) to generally detect if significant transmission heterogeneity is present, and 2) to pinpoint where in a phylogeny high-level spread is occurring. We derive inference procedures to estimate model parameters, including the amount of transmission heterogeneity, in a sampled epidemic. We show that it is possible to detect transmission heterogeneity under a wide range of simulated situations, including incomplete sampling, varying levels of heterogeneity, and including within-host genetic diversity. When evaluating real HIV-1 data from different epidemic scenarios, we found a lower level of transmission heterogeneity in slowly spreading situations and a higher level of heterogeneity in data that included a rapid outbreak, while <i>R</i><sub>0</sub> and Sackinâ€™s index (overall tree shape statistic) were similar in the two scenarios, suggesting that our new method is able to detect transmission heterogeneity in real data. We then show by simulations that targeted prevention, where we pinpoint high-level spread using a coalescence measurement, is efficient when sequence data are collected in an ongoing surveillance system. Such phylogeny-guided prevention is efficient under both single-step contact tracing as well as iterative contact tracing as compared to random intervention.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author summary</div> <p>Detecting and preventing pathogen outbreaks in the background of steady and slow spread is difficult, yet highly desirable, because such transmission heterogeneity can be a main driver of an epidemic. Hence, detection of transmission heterogeneity may direct prevention efforts and reduce future infections. While incidence and prevalence estimates may give overall indications of an epidemicâ€™s progression, they typically cannot indicate episodic outbreaks or rapid spreads in subpopulations. Likewise, detailed and reliable information about dynamic social networks is rare and not generalizable to detect local outbreaks. HIV sequence data can be used to reconstruct HIV phylogenies, which due to HIVâ€™s high evolutionary rate contain information about both transmission networks and rates of spread. Here, we use HIV phylogenies to first design a general heterogeneity detection method that can signal that there is high-level spreading present. Secondly, we develop a phylogenetic method to pinpoint which individuals that may have been infected by a super-spreader or have been involved in an outbreak. We show that using such phylogeny-guided information to prevent future HIV spread is highly efficient under many epidemiological situations, especially in typical public health situations where samples are collected through time.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases (US)</span></div><span class="award-id" tagx="award-id" title="award-id">R01AI087520</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-8160-2588</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leitner</span><span class="given-names" tagx="given-names" title="given-names">Thomas</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">VetenskapsrÃ¥det (SE)</span></div><span class="award-id" tagx="award-id" title="award-id">340-2013-5003</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">VetenskapsrÃ¥det (SE)</span></div><span class="award-id" tagx="award-id" title="award-id">340-2013-5003</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9228-7357</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Britton</span><span class="given-names" tagx="given-names" title="given-names">Tom</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100004359</span><span class="institution" title="institution">VetenskapsrÃ¥det</span></div> </div><span class="award-id" tagx="award-id" title="award-id">340-2013-5003</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-9020-0521</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albert</span><span class="given-names" tagx="given-names" title="given-names">Jan</span></span></span></div> <div class="funding-statement" title="funding-statement">
Y.J.Z, J.A, and T.B acknowledge support from the Swedish research council (grant number 340-2013-5003). T.L is supported by National Institute of Allergy and Infectious Diseases/ National Institutes of Health (NIAID/NIH) under award number R01AI087520. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="PLOS Publication Stage" value="uncorrected-proof" /> <meta name="Data Availability" value="We used three published sequence datasets: ref 28 Esbjornsson et al. (JQ698667â€" /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>We used three published sequence datasets: ref 28 Esbjornsson et al. (JQ698667â€"JQ698874), ref 29 Skar et al. (EU010264-EU010360) and ref 30 Skar et al (GU222921 to GU223066). All sequences are publicly available in Genbank under the accession numbers provided.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Allocation of prevention resources to where they are needed most is important for effective disease control. Thus, identifying where disease spread has its highest intensity would allow for efficient resource allocation. The intensity of the spread is typically uneven in time and space, causing episodic and local outbreaks. One type of spread-heterogeneity comes from the situation when some individuals spread at a much higher rate than others, causing superspreading [<a href="#pcbi.1008122.ref001">1</a>], and invoking the 20-80 rule, where 20% of infected persons transmit to 80% of new infecteds [<a href="#pcbi.1008122.ref002">2</a>, <a href="#pcbi.1008122.ref003">3</a>]. Such transmission heterogeneity can have impact on how fast epidemics spread, and may affect the outcome of control efforts [<a href="#pcbi.1008122.ref004">4</a>].</p> <p>While epidemiological methods using incidence data can pick up the signal of increased number of cases, which may indicate an outbreak or be the result of increased surveillance, they can not identify individual-level transmission heterogeneity. To assess transmission heterogeneity on the individual-level transmissibility, contact tracing based on interviews, reviews of clinical records, and partner follow-up have traditionally been used. Such follow-up may be slow, expensive and inaccurate, however. Several studies have reported that interview-based information about sexual contacts where HIV-1 transmission might have taken place often was not in agreement with the phylogenetic history of the transmitted virus [<a href="#pcbi.1008122.ref005">5</a>, <a href="#pcbi.1008122.ref006">6</a>]. Early studies showed that virus phylogenies reflect their underlying transmission histories [<a href="#pcbi.1008122.ref007">7</a>], and more recent work has shown that phylogenies carry information about the underlying population structure [<a href="#pcbi.1008122.ref008">8</a>â€"<a href="#pcbi.1008122.ref010">10</a>] and the degree distributions of the sexual contact network [<a href="#pcbi.1008122.ref011">11</a>â€"<a href="#pcbi.1008122.ref013">13</a>] both of which contribute to the transmission heterogeneity.</p> <p>The Multi-state birth-death (MSBD) models have been used to model population structure and infer the related transmission heterogeneity [<a href="#pcbi.1008122.ref008">8</a>, <a href="#pcbi.1008122.ref014">14</a>]. The rationale of these models is to associate each cluster (or subpopulation) to a state in the MSBD model, with clusters differing in their transmission dynamics through time. An important limitation of the MSBD models is either requiring the correspondence between the tips and the states is known in advance [<a href="#pcbi.1008122.ref015">15</a>] or using fixed positions of state changes [<a href="#pcbi.1008122.ref010">10</a>]. An efficient method of model-based genetic clustering has recently been proposed in [<a href="#pcbi.1008122.ref009">9</a>], which is based on fitting a Markov-modulated Poisson process representing the evolution of transmission rates along the tree relating different infections. Though it performs well in clustering lineages of different risky groups, the method is not designed to infer basic reproduction number which is also of certain interest in epidemiloigcal practice.</p> <p>In addition, because both incidence and pathogen phylogenies carry information about the epidemic, several efforts have been made to combine these data into comprehensive heterogeneity inference [<a href="#pcbi.1008122.ref016">16</a>â€"<a href="#pcbi.1008122.ref018">18</a>]. Interestingly, Li et al [<a href="#pcbi.1008122.ref019">19</a>] recently showed that quantification of transmission heterogeneity was more accurately estimated using the pathogen phylogeny alone rather than in combination with incidence data. They describe the transmission heterogeneity as the variation among the offspring distributions, i.e., the number of secondary cases caused by each infected individual. The offspring distribution, however, is not only affected by the individual-level transmissibility but also by the length of the infectious period of each infected individual.</p> <p>In this study, we propose a heterogeneous birth-death model to describe the variation of the individual-level transmissibility, with birth rates corresponding to transmissibility (or transmission rates) and death rates to diagnosis rates (or removal rates). We model the transmission heterogeneity by letting each individual draw an independent random transmission rate from a continuous distribution, so the transmission heterogeneity is captured in a parameter quantifying the amount of variation in this distribution. Transmission heterogeneity may exist due to varying degrees of social activity as wells varying transmission risk upon contact, e.g., due to varying viral load in different disease stages. We then develop a general inference procedure to estimate model parameters and to test whether or not there is significant transmission heterogeneity, and thereafter we develop a method that identifies evolutionary lineages that likely have been involved in many transmission events of the epidemic. We use the second method to direct targeted prevention efforts, by means of (additional) contact tracing, towards individuals that have been involved in elevated spread as compared to random intervention. To address the realistic situation where not all persons have been sampled in an epidemic, we investigate the effects of sampling fractions, situations where samples are taken cross-sectionally and when samples are gathered in an ongoing surveillance system. We investigate the effects of within-host diversity in simulations on both the general heterogeneity test as well as the targeted prevention. We also test our estimate of transmission heterogeneity in facing of realistic complexities such as autocorrelation in transmission rates and varying transmission rate through time. We apply the heterogeneity model and estimate parameters on 3 real datasets sampled from different HIV-1 epidemics.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="atransmissionmodelwithheterogenoustransmissibility" title="sec"> <div class="title" tagx="title" title="title">
A transmission model with heterogenous transmissibility</div> <p>In order to estimate transmission heterogeneity in a population and, further, the effect of targeted prevention on individuals associated with high transmissibility, we develop a new transmission model, referred to as the heterogeneous birth-death (HBD) model, in which individuals in a community may have different rates of infecting new individuals. The difference in transmissibility could be caused by variable virus load leading to variable infection probability upon contact, and/or variable social contact rate. Here, we refer to any combination of these concepts as transmissibility.</p> <p>Specifically, in our model each infected individual <i>i</i> gets assigned a transmissibility rate Î»<sub><i>i</i></sub> by randomly drawing the rate from a <i>Gamma</i> distribution with mean <i>Î¼</i><sub>Î»</sub> and standard deviation <i>Ïƒ</i><sub>Î»</sub>. The gamma distribution is a common choice when modeling individual heterogeneity for a parameter. While infectious, this individual infects new individuals randomly in time, with rate Î»<sub><i>i</i></sub> per unit of time. Also we assume that the transmission rates of the infector and the infectee in a transmission event are independent. Later we will evaluate our model under the condition with autocorrelation of transmission rates.</p> <p>Infected individuals remain infectious until they are diagnosedâ€"it is then assumed that individuals are immediately treated such that they no longer are infectious and instead immune [<a href="#pcbi.1008122.ref020">20</a>]. The time to diagnosis, or more precisely to successful treatment, is assumed to be exponentially distributed with rate <i>Î³</i> common for all individuals (hence random and with no systematic heterogeneity). The diagnosed individuals are sampled for virus sequencing with probability <i>Ï�</i><sub><i>SD</i></sub> (referred to as the <i>sequencing ratio</i>). Consequently, there are three categories of infected people: a) the undiagnosed (who continue to spread the disease), b) the diagnosed/treated who have not been sampled for sequencing, and c) the sequenced diagnosed/treated individuals. The ratio of the sequenced diagnosed over all the infected (i.e. c/(a + b + c)) is referred to as the <i>sampling ratio</i> in this study. In contrast with the sequencing ratio (i.e. c/(b + c)) which is usually assumed to be known in advance, the sampling ratio is unknown in the analysis of virus sequence data.</p> <p>In our model, a transmission history is generated with our heterogeneous birth-death transmission model under a given set of parameters (Î˜ = (<i>Î¼</i><sub>Î»</sub>, <i>Ïƒ</i><sub>Î»</sub>, <i>Î³</i>)). The transmission history (denoted as <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e001g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
H</div></div></div>) contains all information about who-infected-whom in calendar time (<a href="#pcbi.1008122.g001">Fig 1</a>). The corresponding tree without information about who-infected-whom is called the transmission tree and denoted as <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e002g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
T</div></div></div></span>. In general, the transmission tree cannot be observed from virus sequence data. However, what we can infer is the tree that describes how the virus sequences from all the infected individuals are related, i.e. the virus genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e003g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></div></span>. If there is no within-host virus diversity, then the virus genealogy and transmission trees are identical: <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e004g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
T</div> </div></div></div></span>. If within-host diversity is taken into account (while still assuming only one strain is transferred at transmission), then the coalescence times of the sampled virus sequences may coalesce further back in time than the time of infection, with the effect that branch lengths and even tree topologies may differ from the underlying transmission tree [<a href="#pcbi.1008122.ref021">21</a>]. The estimation method in this study was first developed under the former situation (i.e. without within-host diversity), and was extended to the latter case in the simulation.<p /> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1008122.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
The connection between transmission history <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e005g" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
H</div></div></div></span> (a), complete genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e006g" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></div></span> (b), and sampled genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e007g" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> (c).</div> <p>In the model, a transmission history (a) is simulated with the heterogeneous birth-death transmission model under a given set of parameters (Î˜). The <i>i</i>-th (<i>i</i> = <i>I</i>, â‹¯, <i>V</i> in Roman numeral notation) individual was infected and stopped infection at <i>t</i><sub><i>i</i></sub> and <i>s</i><sub><i>i</i></sub> (<i>i</i> = 1, â‹¯, 5 in Arabic numerals) respectively. There are three categories of individuals at the end of the simulation: sampled individuals, i.e. diagnosed and sequenced (tips labeled with âˆ’*), diagnosed but unsampled individuals (tips labeled with âˆ’ â—‹), and undiagnosed individual (unlabeled tips). A complete virus genealogy <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e008g" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></div> (b) contains the genealogical lineages from all the infected individuals. In the case of no within host diversity it coincides with the transmission tree <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e009g" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
T</div></div></div></span> being the transmission history <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e010g" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
H</div></div></div></span> but without the direction of transmission. The sampled genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e011g" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> (c) is derived from the complete genealogy by removing edges from unsampled individuals.<p />   </div> <p>The input to our analysis is the pathogen (virus) genealogy (denoted as <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e012g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div>), i.e. the time-scaled phylogeny that can be inferred from the viral DNA sequences from sampled patients (i.e. the diagnosed and sequenced individuals) in an epidemic spanning from time 0 to <i>t</i>. The sampled virus genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e013g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> is obtained by removing the unobserved (undiagnosed and diagnosed but not sampled) individuals and the corresponding coalescent events from <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e014g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></div></span> (<a href="#pcbi.1008122.g001">Fig 1</a>). The parameters to be estimated in analysis are the mean <i>Î¼</i><sub>Î»</sub> and standard deviation <i>Ïƒ</i><sub>Î»</sub> of the transmissibility and the mean time to diagnosis/treatment <i>Î³</i><sup>âˆ’1</sup>, with the main focus on the degree of heterogeneity of transmissibility, either measured in absolute terms as <i>Ïƒ</i><sub>Î»</sub> or in relative terms as the coefficient of variation <i>CV</i><sub>Î»</sub> = <i>Ïƒ</i><sub>Î»</sub>/<i>Î¼</i><sub>Î»</sub>. The <i>CV</i> is a dimensionless quantity where standard deviation, as a measure of dispersion, is normalized by the mean that facilitates comparisons across data sets. When either <i>Ïƒ</i><sub>Î»</sub> or <i>CV</i><sub>Î»</sub> are large, then that implies that a few individuals will tend to infect at a much higher rate than others. We also infer the basic reproduction number <i>R</i><sub>0</sub>, which is the ratio of the mean transmissibility rate and the diagnosis rate, i.e. <i>R</i><sub>0</sub> = <i>Î¼</i><sub>Î»</sub>/<i>Î³</i>, being the average number of infections caused by an infected before diagnosis and treatment.<p /> <p>Our transmission model is an extension of the homogeneous birth-death model in [<a href="#pcbi.1008122.ref022">22</a>], which does not include transmission heterogeneity. Moreover, Leventhal et al [<a href="#pcbi.1008122.ref023">23</a>] explicitly take the finite population into account by considering depletion of susceptible. Here, we neglect this depletion but investigate this simplification in the sensitivity analysis. Recently, Li et al [<a href="#pcbi.1008122.ref019">19</a>] studied a related model where heterogeneity in the number of infections was described in a generation sense; while they assumed that all infections happen at the end of the infectious period, in our model infectious individuals infect new individuals at rate Î»<sub><i>i</i></sub><i>during</i> the infectious period. On the other hand, their model also allow for the possibility to have a highly concentrated distribution of number of infections, for example caused by a social network with most individuals having very similar number of contacts.</p> </div> <div class="inferringtransmissionheterogeneityfromsampledviralgenealogy" title="sec"> <div class="title" tagx="title" title="title">
Inferring transmission heterogeneity from sampled viral genealogy</div> <p>There are two difficulties of using the sampled genealogy <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e015g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div> to infer transmission heterogeneity: one is the lack of transmission direction (who-infected-whom) in <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e016g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span>, and the other that <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e017g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> has incomplete transmission chains [<a href="#pcbi.1008122.ref019">19</a>]. We deal with these two difficulties by using a coordinate-ascent algorithm to estimate both the parameters and the direction of transmission in a genealogy, and, additionally, a data imputation correction to handle the incomplete transmission history. We will first introduce the coordinate-ascent algorithm and consequently give a description about the analysis of the sampled genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e018g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span>.<p /> <div class="inferencewithcoordinate-ascentalgorithm" title="sec"> <div class="title" tagx="title" title="title">
Inference with coordinate-ascent algorithm</div> <p>The coordinate-ascent (CA) algorithm works with a complete genealogy as input, that is, the algorithm assumes there is no within-host diversity and that we observe and sequence all infected individuals up to a certain time <i>t</i>. Later, these assumptions have been relaxed to analyze the sampled genealogy.</p> <p>The CA algorithm has been developed on the basis of the likelihood function for a transmission history <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e019g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
H</div></div></div>. Under the assumptions of our HBD model, not only the transmission events caused by different individuals occur independently, but also the multiple transmission events caused by one individual are independent. So the likelihood for the whole transmission history can be evaluated in terms of individuals. Let <i>x</i><sub><i>i</i></sub> and <i>d</i><sub><i>i</i></sub> (<i>i</i> = 1, â‹¯, <i>n</i>) denote the number of infections and the duration of the infectious period for individual <i>i</i>, information which is contained in the transmission history <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e020g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
H</div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
}</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
n</div> </div> </div></div></div></span>. The likelihood function is then given by <div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e021g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
Î¸</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
H</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mtd_UNKNOWN" /> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ�</div> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ�</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
b</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
Î¸</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mtd_UNKNOWN" /> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ�</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
âˆ«</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
i</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="mo_UNKNOWN">
*</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mi_UNKNOWN">
G</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
Î³</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="mo_UNKNOWN">
*</div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î"</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
Î¸</div> <div class="mi_UNKNOWN">
k</div> </div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
i</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mfrac_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mi_UNKNOWN">
Î¸</div> </div> </div> </div> <div class="mi_UNKNOWN">
d</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="mtd_UNKNOWN" /> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ�</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î"</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î"</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
Î¸</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Î¸</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
k</div> </div> </div> </div> <div class="mo_UNKNOWN">
*</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mi_UNKNOWN">
G</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
Î³</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> </div> </div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(1)</span></div> where Î"(â‹…) is the gamma function, and the indicator <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e022g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mi_UNKNOWN">
G</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> in (<a href="#pcbi.1008122.e021">1</a>) indicates if the infectious period <i>d</i><sub><i>i</i></sub> has ended before the end of the observation period. The parameters of (<i>k</i>, <i>Î¸</i>) are parameterizations of <i>Gamma</i> distribution under which the mean and standard deviation can be represented as <i>Î¼</i><sub>Î»</sub> = <i>kÎ¸</i> and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e023g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> <div class="mi_UNKNOWN">
Î»</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msqrt_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> </div> <div class="mi_UNKNOWN">
Î¸</div> </div></div></div></span> respectively.<p /> <p>Consequently, the maximum likelihood estimates of the model parameters are obtained by equating the gradient of the log-likelihood function to zero. Particularly, the estimates of <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e024g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
^</div> </div></div></div> and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e025g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¸</div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span> are obtained by solving the following equations: <div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e026g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mfrac_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¸</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¸</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¸</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN">
and</div> <div class="mspace_UNKNOWN" /> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mo_UNKNOWN">
ln</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¸</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(2)</span></div> Moreover, the estimate of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e027g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span> is <div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e028g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mi_UNKNOWN">
G</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(3)</span></div> which is simply the observed number of recoveries divided by the total duration of the infectious periods in the transmission history <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e029g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
H</div></div></div></span>.<p /> <p>When we observe the virus genealogy <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e030g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></div> rather than the transmission history <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e031g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
H</div></div></div></span>, we can no longer estimate the parameters as described above since <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e032g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></div></span> does not contain information of transmission direction, i.e. who infected whom. In other words, we can then not determine <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e033g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
n</div> </div></div></div></span>, so the estimates in (<a href="#pcbi.1008122.e026">2</a>) are not available. However, the estimator <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e034g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span> based on (<a href="#pcbi.1008122.e028">3</a>) is still available without the information of transmission directions: even if each infectious period cannot be determined, the sum of infectious periods âˆ‘<sub><i>i</i></sub><i>d</i><sub><i>i</i></sub> is independent of transmission directions and equals the sum of branch lengths in <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e035g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></div></span> (denoted as <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e036g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
|</div> </div></div></div></span>). The estimator <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e037g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span> can hence be calculated before the CA algorithm is introduced below, and it is given by: <div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e038g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
#</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
|</div> </div> </div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(4)</span></div> where <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e039g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
#</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
)</div> </div></div></div></span> is the number of diagnosis events in <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e040g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></div></span>.<p /> <p>In the following, we focus on the estimation of parameter <i>k</i> and <i>Î¸</i>. We treat the transmission directions of the branching events in <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e041g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></div> as latent variables, and propose the CA algorithm which alternates between estimating the model parameters (Î˜ = {<i>k</i>, <i>Î¸</i>}) and reconstructing the transmission history <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e042g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
H</div></div></div></span>. Here the reconstruction is implemented in a form of labeling the branches of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e043g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></div></span> (as illustrated in <a href="#pcbi.1008122.g002">Fig 2</a>). The branching events following the labeled branches are the transmission events caused by the individuals with the corresponding labels. The CA algorithm traverse the genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e044g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></div></span><i>slice</i>-by-<i>slice</i> from the leaves up to the root (<a href="#pcbi.1008122.g002">Fig 2</a>). Each slice (illustrated in <a href="#pcbi.1008122.g002">Fig 2(a)</a>) consists of branches with the same <i>height</i>, defined as the number of edges on the longest downward path from this branch to a tip (i.e. similar to the definition of the height of a node). For example, the external branches have the height of 0, so they consist of a slice. The branches within a particular slice come from different infected individuals, and the transmission event following these branches are hence independent. Therefore, labeling these branches can be performed in parallel.<p /> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1008122.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Schematic representation of the coordinate-ascent algorithm.</div> <p>(a) The branches in the genealogy <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e045g" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></div> have been divided into 4 slices (labeled as â"ª to â‘¢). (b) The algorithm performs estimation of model parameters (denoted as Î˜) and reconstructing the transmission history <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e046g" /><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
H</div></div></div></span> (or labeling the branches (with different letters) of a genealogy) simultaneously under the likelihood framework. The algorithm starts from the external branches (labeled as â"ª) which come from different individuals, and proceeds backwards slice by slice. The internal branches with the same label represent the infectious period of a particular individual, who is the infector of the transmission events during that period. (c) Given the present estimate of model parameters and transmission history (<span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e047g" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î˜</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mn_UNKNOWN">
1</div> </div></div></div></span>,<span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e048g" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
H</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mn_UNKNOWN">
1</div> </div></div></div></span>), the transmission histories of individual <i>a</i> and <i>b</i> are (<i>d</i><sub><i>a</i></sub> = <i>l</i><sub>1</sub>, <i>e</i><sub><i>a</i></sub> = 0) and (<i>d</i><sub><i>b</i></sub> = <i>l</i><sub>2</sub> + <i>l</i><sub>3</sub>, <i>e</i><sub><i>b</i></sub> = 1) respectively. For the unlabeled branch <i>l</i><sub><i>ub</i></sub>, its labeling coefficient is evaluated as <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e049g" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
{</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mn_UNKNOWN">
3</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
|</div> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î˜</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
}</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
{</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mn_UNKNOWN">
3</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
|</div> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î˜</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
}</div> </div> </div> </div></div></div></span>.<p />   </div> <p>The inference process begins with the external branches which are assigned to different individuals, yielding the initial estimate of the transmission history <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e050g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
H</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mn_UNKNOWN">
0</div> </div></div></div>. Because of the independence assumption in the HBD model, the likelihood function in <a href="#pcbi.1008122.e021">Eq (1)</a> is also valid for the partially reconstructed transmission history <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e051g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
H</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mn_UNKNOWN">
0</div> </div></div></div></span>, and the initial estimate of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e052g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î˜</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mn_UNKNOWN">
0</div> </div></div></div></span> is hence obtained via (<a href="#pcbi.1008122.e026">2</a>). Further, for all unlabeled branches in a slice, the CA algorithm alternates between updating transmission history (i.e. labeling the branches based on the present estimate of model parameters) and updating the estimation of model parameter based on the updated transmission history, until convergence.<p /> <p>More precisely, we evaluate the <i>labeling coefficient</i> for an unlabeled branch as the ratio of the probabilities of labeling the branch to its left and right descendant branches. Suppose that the unlabeled branch has a length of <i>l</i><sub><i>u</i></sub>, and that its left and right descendant branches were already labeled as <i>a</i> and <i>b</i> respectively, that is, with reconstructed transmission history of (<i>d</i><sub><i>a</i></sub>, <i>e</i><sub><i>a</i></sub>) and (<i>d</i><sub><i>b</i></sub>, <i>e</i><sub><i>b</i></sub>), respectively. Labeling the new branch as <i>a</i> or <i>b</i> stands for updating the corresponding transmission history by prolonging for another <i>l</i><sub><i>u</i></sub> time unit and adding one more transmission event. Here the <i>labeling coefficient</i><i>Î²</i><sub><i>ab</i></sub> is calculated as (illustrated in <a href="#pcbi.1008122.g002">Fig 2(c)</a>): </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e053g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
{</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
a</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
u</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
a</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
|</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¸</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
}</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
{</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
a</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
a</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
u</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
|</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¸</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
}</div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(5)</span></div> where the probabilities such as <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e054g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
{</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
a</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
u</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
a</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
|</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¸</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
}</div> </div></div></div></span> is evaluated via (<a href="#pcbi.1008122.e021">1</a>) by plugging in these present estimates of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e055g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¸</div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span>, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e056g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span>, and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e057g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span>. Consequently, if <i>Î²</i><sub><i>ab</i></sub> &amp;gt; 1, then the new branch is labeled as its left descendant branch (<i>a</i> in this example); otherwise, as its right descendant branch (<i>b</i> in this example).<p /> </div> <div class="analysisofsampledvirusgenealogy" title="sec"> <div class="title" tagx="title" title="title">
Analysis of sampled virus genealogy</div> <p>In this part, we describe the analysis of the sampled genealogy <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e058g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div> based on the CA algorithm. Since <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e059g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> suffers from partial sampling, the CA algorithm cannot be applied directly. For a given tuning parameter <i>p</i> (0 &amp;lt; <i>p</i> &amp;lt; 1, often slightly smaller than 1), we focus on the part of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e060g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> up to time <i>t</i> âˆ’ <i>L</i><sub><i>p</i></sub> (denoted as <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e061g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span>), where <i>L</i><sub><i>p</i></sub> is the <i>p</i> Ã— 100 quantile of the lengths of the external branches of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e062g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span>. We set <i>p</i> to a high value (<i>p</i> = 0.9, 0.85, and 0.8) to get a local genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e063g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> which contains a relative large part of information about the epidemic dynamics up to time <i>t</i> âˆ’ <i>L</i><sub><i>p</i></sub>. Specifically, our analysis consists of three steps: firstly, we estimate the sampling ratio (i.e. the number of the diagnosed and sequenced divided by all the infected) of the local genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e064g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span>; secondly, we apply the CA algorithm to the local genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e065g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> by assuming that all individuals infected within [0, <i>t</i> âˆ’ <i>L</i><sub><i>p</i></sub>] are sampled; thirdly, we correct these parameter estimates by taking advantage of the estimated sampling ratio in the first step. Also, the results under different values of <i>p</i> are averaged out to generate the final estimation.<p /> <p>Firstly, the sampling ratio of the local genealogy <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e066g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div>, or in other words, the number of the unsampled individuals up to time <i>t</i> âˆ’ <i>L</i><sub><i>p</i></sub>, has been estimated. As defined above <i>L</i><sub><i>p</i></sub> is the <i>p</i> Ã— 100 percentile of the lengths of the external branches of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e067g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span>, so <i>L</i><sub><i>p</i></sub> provides a conservative estimate of the <i>p</i> Ã— 100 percentile for the distribution of infectious periods. Hence, we assume that a fraction <i>p</i> of the individuals infected within [0, <i>t</i> âˆ’ <i>L</i><sub><i>p</i></sub>] will be diagnosed before time <i>t</i>. In addition, a fraction <i>Ï�</i><sub><i>SD</i></sub> of the diagnosed individuals was selected to do sequencing (i.e. the sequencing ratio is <i>Ï�</i><sub><i>SD</i></sub>), so the sampling ratio of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e068g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> is <i>Ï�</i><sub><i>SD</i></sub> * <i>p</i>. Denoting <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e069g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mo_UNKNOWN">
(</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div></div></div></span> as the number of individuals collected in the local genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e070g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span>, so there are <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e071g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ï�</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
D</div> </div> </div> <div class="mo_UNKNOWN">
*</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> individuals which were infected before time <i>t</i> âˆ’ <i>L</i><sub><i>p</i></sub> but were not sampled.<p /> <p>Next, we apply the CA algorithm to the local genealogy <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e072g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div> and obtain the raw estimates of the mean (denoted as <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e073g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mn_UNKNOWN">
0</div> </div></div></div></span>) and the standard deviation (denoted as <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e074g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mn_UNKNOWN">
0</div> </div></div></div></span>) of the transmissibility rate. Furthermore, the raw estimate of the transmission heterogeneity is <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e075g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mi_UNKNOWN">
V</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></div></span>.<p /> <p>Last, we study the correction of the raw estimates by considering the unsampled individuals. This correction is performed on the basis of data imputation. For the sampled individuals, their estimated transmissibility rates shall be upscaled based on the sampling ratio by allowing for partial sampling. For the unsampled individuals, we assume that they have the same infectious period and the same transmissibility rate because of no further information about their difference. The corrected values for the sampled individuals together with the substituted values for the unsampled individuals are combined to yield the corrected estimation of model parameters as follows (Please see <a href="#pcbi.1008122.s010">S1 Text</a> for details about the derivation of (<a href="#pcbi.1008122.e076">6</a>)-(<a href="#pcbi.1008122.e080">8</a>)):</p> <p>The corrected estimation of the diagnosis rate is </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e076g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
p</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
#</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ï�</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
D</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> </div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ï�</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
D</div> </div> </div> <div class="mo_UNKNOWN">
*</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
*</div> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
Â¯</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
E</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(6)</span></div> where <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e077g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
Â¯</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
E</div> </div> </div></div></div></span> is the mean length of the external branches in <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e078g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span>, and is used as the infectious period of the unsampled individuals.<p /> <p>In addition, the corrected estimates of the mean and standard deviation of transmissibility rate are </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e079g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
p</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ï�</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
D</div> </div> </div> <div class="mo_UNKNOWN">
*</div> <div class="mi_UNKNOWN">
p</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
s</div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(7)</span></div> and <div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e080g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
p</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msqrt_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ï�</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
D</div> </div> </div> <div class="mo_UNKNOWN">
*</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ï�</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
D</div> </div> </div> <div class="mo_UNKNOWN">
*</div> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ï�</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
D</div> </div> </div> <div class="mo_UNKNOWN">
*</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
-</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ï�</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
D</div> </div> </div> <div class="mo_UNKNOWN">
*</div> <div class="mi_UNKNOWN">
p</div> <div class="mo_UNKNOWN">
*</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
s</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
,</div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(8)</span></div> where Î»<sub><i>us</i></sub> is the substituted transmissibility rate for the unsampled individuals. Furthermore, the corrected estimation of transmission heterogeneity is <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e081g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mi_UNKNOWN">
V</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
p</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
p</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
p</div> </div> </div></div></div></span>.<p /> <p>The value of Î»<sub><i>us</i></sub> shall vary with the level of heterogeneity. Under the homogeneous situation (i.e. <i>CV</i><sub>Î»</sub> = 0), all the infected individuals have the same transmissibility rates, so we set the transmissibility rates of the unsampled individuals to be the same as the average rate of the sampled individuals, that is, <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e082g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
s</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></div>. On the other hand, under the situation with extremely high level of heterogeneity, a small proportion of individuals have very high level of transmissibility rates, while a large proportion of individuals have small level of transmissibility rates. The former will cause most of the infections, while the latter will cause few or no infections. Note that the transmission events caused by an individual could be reconstructed by the information from its descendants. The sampled individuals in the genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e083g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> are more likely to be the descendants of the highly infectious individuals. Hence the coalescent events in the genealogy <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e084g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div></span> are more likely corresponding to the transmission events caused by these highly infectious individuals, even if some of them are not sampled while their descendants are sampled instead. Hence the transmissibility rates estimated from the sampled genealogy reflect the rates with which highly infectious individuals spread the disease. Based on this consideration, under the situation with high level of heterogeneity, we assume that unsampled individuals have low transmissibility rates and that they cause few or no infections before being diagnosed. In other words, we set the constant transmissibility rate Î»<sub><i>us</i></sub> for the unsampled individuals to <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e085g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
s</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div> </div></div></div></span>, where <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e086g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span> is the estimated recovery rate. For the general case we compromise between the two extreme situations on the basis of the level of heterogeneity as follows: <div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e087g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
s</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="mo_UNKNOWN">
exp</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
C</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mo_UNKNOWN">
exp</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
C</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
-</div> <div class="msubsup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
,</div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(9)</span></div> where <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e088g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mi_UNKNOWN">
V</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mn_UNKNOWN">
0</div> </div></div></div></span> is a raw estimation about the level of heterogeneity as defined above.<p /> </div> </div> <div class="targetedpreventionbasedonphylogeneticinformation" title="sec"> <div class="title" tagx="title" title="title">
Targeted prevention based on phylogenetic information</div> <p>An important application for heterogeneity detection is to optimize the epidemic control policy when transmission heterogeneity exists. As pointed out in [<a href="#pcbi.1008122.ref001">1</a>], focusing on the highly infectious individuals will greatly increase the efficiency of disease control policy. To facilitate such an application, we propose a phylogenetic indexâ€"the Number of Coalescent Events (NCE) which helps to identify potentially highly infectious individuals. We investigate the effectiveness of using this index with the intention of controlling the spread of disease in our heterogeneous birth-death model.</p> <p>Specifically, for each tip in the sampled genealogy <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e089g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div>, we calculate the <i>NCE</i> within a fixed time-period <i>t</i><sub><i>c</i></sub> prior to the time of sampling of the individual. The taxa (diagnosed individuals) with higher <i>NCE</i>-values are more likely to be phylogenetically linked with unsampled highly infectious individuals. Thus, performing contact tracing on the individuals with high <i>NCE</i>-values should identify undiagnosed and highly infectious individuals and/or their undiagnosed descendants, and therefore efficiently reduce further disease spread. We refer to contact tracing based on <i>NCE</i>-value as phylogeny-guided contact tracing. We evaluate different threshold values <i>m</i>, i.e. where individuals who have <i>NCE</i> â©¾ <i>m</i> within <i>t</i><sub><i>c</i></sub> prior to being sampled are selected for contact tracing. We also evaluate both single-step and iterative contact tracing schemes [<a href="#pcbi.1008122.ref024">24</a>] to show the full beneficial effects of phylogeny-guided contact tracing policies.<p /> </div> <div class="overviewofsimulationstudy" title="sec"> <div class="title" tagx="title" title="title">
Overview of simulation study</div> <p>To validate our general transmission heterogeneity detection method, we tested the performance of the inference algorithm on simulated data. As reported the average time between infection and diagnosis/treatment for HIV-1 in Sweden is around 2.5 years [<a href="#pcbi.1008122.ref011">11</a>, <a href="#pcbi.1008122.ref025">25</a>, <a href="#pcbi.1008122.ref026">26</a>], so we set the rate of diagnosis at <i>Î³</i> = 1/2.5 with year being our unit of time. We set <i>R</i><sub>0</sub> = 2.5 (implying that <i>Î¼</i><sub>Î»</sub> = 1) which is in agreement with the previous study on the transmission of HIV in Switzerland [<a href="#pcbi.1008122.ref022">22</a>]. We varied the level of transmission heterogeneity (i.e. <i>CV</i> = <i>Ïƒ</i><sub>Î»</sub>/<i>Î¼</i><sub>Î»</sub>) from 0 (no heterogeneity) to 5 (extremely high heterogeneity).</p> <p>Unless otherwise stated, each simulation began with one infection and was stopped when there were 100 diagnosed individuals (the average number of infected but not yet diagnosed were 130). Among these 100 diagnosed individuals, 90 diagnosed individuals were sampled for sequencing to reconstruct the viral genealogy, that is, the sequencing ratio was set to <i>Ï�</i><sub><i>SD</i></sub> = 0.9. In all, the average sampling ratio in our simulations was 90/230 â‰ˆ 0.39. We assume that there was no delay between the time of diagnosis and sampling. Please see <a href="#pcbi.1008122.s010">S1 Text</a> for details of simulation.</p> <p>All the codes for simulation and inference are available at github.com/yunPKU/HeteroInfer.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="estimationofheterogeneityandothermodelparametersonsimulateddatasets" title="sec"> <div class="title" tagx="title" title="title">
Estimation of heterogeneity and other model parameters on simulated datasets</div> <p>The epidemic parameters describing the recovery rate (<i>Î³</i>), average infectivity rate (<i>Î¼</i><sub>Î»</sub>), and the transmission heterogeneity (<i>Ïƒ</i><sub>Î»</sub> and <i>CV</i><sub>Î»</sub> = <i>Ïƒ</i><sub>Î»</sub>/<i>Î¼</i><sub>Î»</sub>) were estimated from our heterogeneous birth-death model and the described inference methodology (<a href="#pcbi.1008122.g003">Fig 3A, 3B, 3D and 3E</a>). While there was a slight bias trend over <i>CV</i><sub>Î»</sub> = <i>Ïƒ</i><sub>Î»</sub>/<i>Î¼</i><sub>Î»</sub>, the bias was well within the corresponding 95% confidence interval in each case. Hence, under a wide range of epidemic situations with different levels of heterogeneity (<i>CV</i><sub>Î»</sub>) the means of the epidemic parameter estimates were close to the corresponding true values. The inference of the basic reproduction number (<i>R</i><sub>0</sub>), showed slight upward biases under low level of heterogeneity and slight downward biases under high level of heterogeneity (<a href="#pcbi.1008122.g003">Fig 3B</a>). However, these biases were also covered by the 95% confidence intervals.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1008122.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
Estimates of the heterogeneity and other epidemiological parameters from simulated virus genealogy.</div> <p>The simulations were performed under various true levels of heterogeneity (<span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e090g" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> <div class="mi_UNKNOWN">
Î»</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mi_UNKNOWN">
Î»</div> </div> </div></div></div>). In each panel, the black line denotes the true value that was used to generate the simulated data, and the colored curve and the shaded area denote the mean and 95% confidence interval from 200 simulations, respectively. A estimate of recovery rate (<i>Î³</i>), B estimate of average infectivity rate (<i>Î¼</i><sub>Î»</sub>), C estimate of the basic reproduction number (<i>R</i><sub>0</sub>), D estimate of the standard deviation of infectivity rate (<i>Ïƒ</i><sub>Î»</sub>), and E estimate of the coefficient of variation (<i>CV</i><sub>Î»</sub>).<p />   </div> <p>The estimated <i>CV</i><sub>Î»</sub> showed non-negligible downward bias. Since the coefficient of variation <i>CV</i><sub>Î»</sub> is the inverse square-root of the shape parameter in the <i>Gamma</i> distribution, this observation agrees with earlier findings showing that the maximum likelihood estimate of the shape parameter tend to upward biased [<a href="#pcbi.1008122.ref027">27</a>], leading to <i>CV</i><sub>Î»</sub> being underestimated.</p> <p>The upward bias trend of <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e091g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div> over heterogeneity level was potentially due to our simulation process conditioning on reaching 100 tips. Given this condition, the average branch length of the simulated tree decreased with the heterogeneity level <i>CV</i><sub>Î»</sub> (as shown in <a href="#pcbi.1008122.s001">S1 Fig</a>), indicating the simulation datasets were biased in favor of â€˜shorterâ€™ trees under higher level of heterogeneity. Therefore, the estimate of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e092g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span> was upward biased under high <i>CV</i><sub>Î»</sub>. On the other hand, the estimate of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e093g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> </div></div></div></span> remained relatively stable. This is due to the fact that we used Î»<sub><i>us</i></sub> as the substituted transmissibility rate for the unsampled individuals. We explained in section of <i>Analysis of sampled virus genealogy</i> that, under the high level of heterogeneity, Î»<sub><i>us</i></sub> was close to the estimated recovery rate <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e094g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span> which is considerably smaller than the true value of <i>Î¼</i><sub>Î»</sub>. So this correction method reduced the upward bias of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e095g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> </div></div></div></span> under high heterogeneity situation, and hence resulted in a stable estimate. In all, the upward bias trend of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e096g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span> together with the stable estimate of <i>Î¼</i><sub>Î»</sub> over <i>CV</i><sub>Î»</sub> resulted in the downward bias trend of the estimated <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e097g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span> over heterogeneity levels.<p /> <p>In the simulation study we also vary time scales by fixing the average infectivity rate <i>Î¼</i><sub>Î»</sub> = 1 and gradually increasing the average length of infectious period <i>Î³</i><sup>âˆ’1</sup> from 1.5 to 3, corresponding to the basic reproduction number <i>R</i><sub>0</sub> increased from 1.5 to 3. As before the simulation began with one infection and was stopped when there were 100 diagnosed individuals. The results are summarized in <a href="#pcbi.1008122.s002">S2</a> and <a href="#pcbi.1008122.s003">S3</a> Figs.</p> <p>The estimates of transmission heterogeneity (<i>Ïƒ</i><sub>Î»</sub> and <i>CV</i><sub>Î»</sub>) were robust to differences in the average length of infectious period <i>Î³</i><sup>âˆ’1</sup> (<a href="#pcbi.1008122.s002">S2 Fig</a>). The effect on the estimated <i>CV</i><sub>Î»</sub> was small unless the true heterogeneity level was higher than 3 (<a href="#pcbi.1008122.s002">S2A Fig</a>). We also found that the estimated <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e098g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
^</div> </div></div></div> remained upward biased under the scenarios with shorter infectious period (<i>Î³</i><sup>âˆ’1</sup> = 1.5 and 2). As our simulation process being conditioning on reaching 100 tips, the simulated datasets were likely to be biased in favor of â€œfasterâ€� trees which resulted in the overestimated <i>R</i><sub>0</sub> [<a href="#pcbi.1008122.ref028">28</a>].<p /> <div class="inferenceofheterogeneitywhentransmissionratesareautocorrelated" title="sec"> <div class="title" tagx="title" title="title">
Inference of heterogeneity when transmission rates are autocorrelated</div> <p>Our new method was developed under the assumption that each infected individual has an independent transmission rate. In reality, however, risk factors of transmissibility such as behavior often exhibit a high degree of homophily, such that the transmission rate of a newly infected individual may be similar to that of the donor. We therefore evaluated our method when transmission rates are autocorrelated and made a comparison with the previously developed Markov-Modulated Poisson Process (MMPP) method [<a href="#pcbi.1008122.ref009">9</a>] which was developed by exploiting the autocorrelation of transmission rates to genetic clustering as well as to estimate different transmission rates of different subpopulations.</p> <p>We adopted the simulation scenario in [<a href="#pcbi.1008122.ref009">9</a>] to divide the whole susceptible population into two subpopulations with different transmission rates (Î»<sub>1</sub> and Î»<sub>2</sub> respectively) and simulated the outbreak in two ways, that is, with and without autocorrelation of transmission rates. In the former case, each newly infected individual has the same transmission rate as its infectee with the probability 1 âˆ’ <i>Ï€</i><sub><i>s</i></sub> and switch to another transmission rate with the probability <i>Ï€</i><sub><i>s</i></sub>. Here we set the transmission rates as Î»<sub>1</sub> = 0.9 and Î»<sub>2</sub> = 8.1 (9-fold faster than the other as in [<a href="#pcbi.1008122.ref009">9</a>]). The switching probability is <i>Ï€</i><sub><i>s</i></sub> = 0.2 or <i>Ï€</i><sub><i>s</i></sub> = 0.8, corresponding to high level of autocorrelation with low heterogeneity (<i>CV</i><sub>Î»</sub> = 0.46) and low level of autocorrelation with high heterogeneity (<i>CV</i><sub>Î»</sub> = 1.05) respectively. The evaluation under each condition is performed based on 100 simulated replicates.</p> <p>In the case without autocorrelation, each newly infected individual draws its transmission rate independently from a binary probabilistic distribution, that is, choosing Î»<sub>1</sub> with probability 1 âˆ’ <i>Ï€</i><sub><i>c</i></sub>, and choosing a larger transmission rate Î»<sub>2</sub> with probability <i>Ï€</i><sub><i>c</i></sub>. We set <i>Ï€</i><sub><i>c</i></sub> = 0.1, corresponding to the proportion of risky subpopulation in [<a href="#pcbi.1008122.ref009">9</a>]. We made the comparison for two settings, one with lower level of heterogeneity: Î»<sub>1</sub> = 2 and Î»<sub>2</sub> = 6 (corresponding to the mean transmission rate <i>Î¼</i><sub>Î»</sub> = 2.4 and the heterogeneity <i>CV</i><sub>Î»</sub> = 0.5); and the other with higher level of heterogeneity: Î»<sub>1</sub> = 1 and Î»<sub>2</sub> = 15, corresponding to the same <i>Î¼</i><sub>Î»</sub> = 2.4 but now <i>CV</i><sub>Î»</sub> = 1.75.</p> <p>Under the situation of no autocorrelation of transmission rates, it is clear that the new method generates more accurate estimates of the mean (<i>Î¼</i><sub>Î»</sub>) and the heterogeneity (<i>CV</i><sub>Î»</sub>) of two transmission rates (<a href="#pcbi.1008122.s009">S9A and S9B Fig</a>). For the low heterogeneity condition with true values of <i>Î¼</i><sub>Î»</sub> = 2.4 and <i>CV</i><sub>Î»</sub> = 0.5 (<a href="#pcbi.1008122.s009">S9A Fig</a>), the medians of the estimates for <i>Î¼</i><sub>Î»</sub> and <i>CV</i><sub>Î»</sub> are 2.32 and 0.7 from our new method and are 4.65 and 0.74 from the MMPP respectively. For the high heterogeneity condition with true values of <i>Î¼</i><sub>Î»</sub> = 2.4 and <i>CV</i><sub>Î»</sub> = 1.75 (<a href="#pcbi.1008122.s009">S9B Fig</a>), the medians are 2.19 and 1.56 from our new method and are 6.35 and 1.04 from the MMPP respectively.</p> <p>Under the situation with autocorrelation of transmission rates, the new method is comparable with the MMPP in terms of estimation accuracy (<a href="#pcbi.1008122.s009">S9C and S9D Fig</a>). For the low switching probability (<i>Ï€</i><sub><i>s</i></sub> = 0.2) condition with true values of <i>Î¼</i><sub>Î»</sub> = 6.49 and <i>CV</i><sub>Î»</sub> = 0.46 (<a href="#pcbi.1008122.s009">S9C Fig</a>), the medians of the estimates for <i>Î¼</i><sub>Î»</sub> and <i>CV</i><sub>Î»</sub> are 5.22 and 0.79 from the new method and are 8.73 and 0.99 from the MMPP respectively. For the high switching probability <i>Ï€</i><sub><i>s</i></sub> = 0.8 condition with true values of <i>Î¼</i><sub>Î»</sub> = 3.18 and <i>CV</i><sub>Î»</sub> = 1.05 (<a href="#pcbi.1008122.s009">S9D Fig</a>), the medians of the estimates are 2.64 and 1.27 from our new method and are 5.75 and 1.00 from the MMPP respectively.</p> </div> <div class="parameterestimatesremainaccurateunderfinitepopulationsizesandbiasedsequencingratioÏ�sd" title="sec"> <div class="title" tagx="title" title="title">
Parameter estimates remain accurate under finite population sizes and biased sequencing ratio <i>Ï�</i><sub><i>SD</i></sub></div> <p>The above results show good performance of the proposed inference method under the heterogeneous birth-death model assuming that there is no depletion of susceptibles (corresponding to an infinite population) and that the sequencing ratio <i>Ï�</i><sub><i>SD</i></sub> was known (but the sampling ratio is unknown). In real applications, however, these two assumptions do not hold. While the size of the susceptible population at risk is very difficult to know in real epidemics, it is never infinite. The sequencing ratio <i>Ï�</i><sub><i>SD</i></sub> is often known to some extent, but there can be uncertainty in the number of diagnosed but not sequenced, leading to uncertainty in <i>Ï�</i><sub><i>SD</i></sub>.</p> <p>To investigate if the susceptible population size and biased <i>Ï�</i><sub><i>SD</i></sub> affect the estimation of the model parameters, we tested the inference method under a relative realistic scenario, i.e. setting the population sizes <i>N</i> = 1000 and introducing 10 percent bias in <i>Ï�</i><sub><i>SD</i></sub> both upwards and downwards. As before we began with one infection and stopped when there were 100 diagnosed individuals. When the simulations were stopped, the average number of infected but not yet diagnosed was 120, that is, the average prevalence was 220/1000 = 22% which is a relatively higher level in reality [<a href="#pcbi.1008122.ref029">29</a>].</p> <p>The estimation of transmission heterogeneity was robust to finite populations at risk (<a href="#pcbi.1008122.g004">Fig 4</a>) and bias in <i>Ï�</i><sub><i>SD</i></sub>, with only small effects unless the true level of heterogeneity was very high (<i>CV</i><sub>Î»</sub> &amp;gt; 3). The effects were not significant as they was covered by the variability of the estimate under infinite population size. The same conclusion applied also to estimation of the basic reproduction number <i>R</i><sub>0</sub>. This is encouraging, as the susceptible population size is typically unknown, and may vary over time in real epidemics.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1008122.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
Performance of parameter estimation under finite population size and biased <i>Ï�</i><sub><i>SD</i></sub>.</div> <p>The estimates equipped with â€œhatâ€� (e.g.,<span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e099g" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div>) are based on the infinite population size and the exact value of <i>Ï�</i><sub><i>SD</i></sub>. The estimates equipped with â€œtildeâ€� (e.g.,<span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e100g" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
Ëœ</div> </div> <div class="mo_UNKNOWN">
+</div> </div></div></div></span> and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e101g" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
Ëœ</div> </div> <div class="mo_UNKNOWN">
-</div> </div></div></div></span>) are based on finite population size and 10% bias in <i>Ï�</i><sub><i>SD</i></sub> (<span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e102g" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
Ëœ</div> </div> <div class="mo_UNKNOWN">
+</div> </div></div></div></span> is on upwards bias and <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e103g" /><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
Ëœ</div> </div> <div class="mo_UNKNOWN">
-</div> </div></div></div></span> is with downwards bias in <i>Ï�</i><sub><i>SD</i></sub>). The colored curves in (A) and (B) are the mean of 200 simulations, while the shaded areas denote the 95% confidence intervals based on these simulation replicates.<p />   </div> </div> <div class="inferenceofheterogeneityisrobusttotransmissionheterogeneitythroughtime" title="sec"> <div class="title" tagx="title" title="title">
Inference of heterogeneity is robust to transmission heterogeneity through time</div> <p>In the above simulations, a constant transmissibility rate for each infected individual was assumed. However, it is well known that for HIV, infected individuals initially have a short high-transmission-risk phase (the acute infection phase when viral load is very high), followed by a much longer low-transmission-risk phase (the chronic phase during which viral load is relatively low), which, in the absence of treatment, is followed by another high-transmission-risk phase (progression to AIDS) [<a href="#pcbi.1008122.ref030">30</a>â€"<a href="#pcbi.1008122.ref032">32</a>]</p> <p>To investigate if this transmission-risk heterogeneity through time affect the inference of heterogeneity among individuals and other model parameters, we tested the proposed inference methodology under a more realistic simulation scenario by allowing for a time-varying transmissibility rate for each infected individual. As before, each individual <i>i</i> drew a random transmission rate <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e104g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
i</div> <div class="mn_UNKNOWN">
0</div> </div></div></div> from the <i>Gamma</i> distribution with mean <i>Î¼</i><sub>Î»</sub> and standard deviation <i>Ïƒ</i><sub>Î»</sub>, and a duration of the infectious period being <i>Exp</i>(<i>Î³</i>) distributed. The time-varying transmission rate was modeled by starting with transmission rate <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e105g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
i</div> <div class="mn_UNKNOWN">
0</div> </div></div></div></span> during acute phase and then dropping and rising again if not yet diagnosed. More specifically, we followed the four-phase model in [<a href="#pcbi.1008122.ref031">31</a>] to model the change of transmission rate over time since infection that is, <div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e106g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
{</div> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
i</div> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
0</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
â€"</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
6</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
months</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
i</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
*</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
.</div> <div class="mn_UNKNOWN">
5</div> <div class="mo_UNKNOWN">
/</div> <div class="mn_UNKNOWN">
8</div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
6</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
â€"</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
15</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
months</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
i</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
*</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
/</div> <div class="mn_UNKNOWN">
8</div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
16</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
â€"</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
36</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
months</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
Î»</div> <div class="mi_UNKNOWN">
i</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
*</div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
.</div> <div class="mn_UNKNOWN">
6</div> <div class="mo_UNKNOWN">
/</div> <div class="mn_UNKNOWN">
8</div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
after</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
37</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
months</div> </div> </div> </div> </div> <div class="mo_UNKNOWN" /> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(10)</span></div> where <i>t</i> was the time since infection for the <i>i</i>-th individual. As before each simulation was started with one infected individual and stopped when there were 100 diagnosed individuals. The results are summarized in <a href="#pcbi.1008122.s004">S4 Fig</a>.<p /> <p>Relevant to the estimate of <i>CV</i><sub>Î»</sub> (i.e. heterogeneity of individual-level transmissibility) which was shown in <a href="#pcbi.1008122.s004">S4B Fig</a>, we found that adding transmission-risk heterogeneity due to disease progression leads to a small overestimation when the true level of <i>CV</i><sub>Î»</sub> was low (<i>CV</i><sub>Î»</sub> &amp;lt; 1.5) and a slight underestimation when <i>CV</i><sub>Î»</sub> &amp;gt; 1.5. These effects, however, were not significant as they were covered by the 95% confidence interval. In addition, as shown in <a href="#pcbi.1008122.s004">S4A Fig</a>, adding heterogeneity through time tended to underestimate <i>R</i><sub>0</sub>, and the bias became significant under lower level of heterogeneity (<i>CV</i><sub>Î»</sub> &amp;lt; 1.0). This makes sense because adding heterogeneity through time reduced the average transmission rate over the whole infectious period for each individual, and hence resulted in a drop of the estimate of basic reproduction number.</p> </div> <div class="within-hostdiversitycausesunderestimationoftransmissionheterogeneity" title="sec"> <div class="title" tagx="title" title="title">
Within-host diversity causes underestimation of transmission heterogeneity</div> <p>Because HIV, and many other rapidly evolving pathogens, accumulate significant levels of within-host diversity, pathogen phylogenies may differ from the transmission history [<a href="#pcbi.1008122.ref021">21</a>]. Using a coalescent-based framework that allows for within-host pathogen diversification over time [<a href="#pcbi.1008122.ref011">11</a>, <a href="#pcbi.1008122.ref021">21</a>, <a href="#pcbi.1008122.ref033">33</a>], we investigated the effect of different diversification rates on the ability to detect transmission heterogeneity (please see <a href="#pcbi.1008122.s010">S1 Text</a> for details of simulation). We found that within-host diversity leads to underestimation of the level of transmission heterogeneity (<a href="#pcbi.1008122.g005">Fig 5A</a>), which becomes significant at very high heterogeneity levels (<i>CV</i><sub>Î»</sub> &amp;gt; 3). Within-host diversity adds an additional source of phylogenetic variation, and this variation partially overshadows the heterogeneity resulting from differences in transmission contacts. Nevertheless, even with within-host diversity, the trend of the inferred heterogeneity is monotonically increasing as the true level of heterogeneity increases, leading to a positive response of the inferred level of heterogeneity. Hence, when a pathogen generates significant levels of within-host diversity, we still detect increased levels of transmission heterogeneity if it occurs, but the true level of transmission heterogeneity may become underestimated.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1008122.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
Performance of parameter estimation in the presence of within-host diversity.</div> <p>Here <i>r</i> denotes the rate of within-host population size increases (per day). Four levels of within-host diversity have been calculated: <i>r</i> = 1(blue), 0.5(orange), 0.3(green), 0.15(red), and <i>r</i> = 0 (purple) corresponding to no heterogeneity. Results are the mean of 100 simulations. Also the estimates (purple) and the corresponding 95% confidence intervals (shaded area) under the condition without within-host diversity are also calculate for comparison.</p>   </div> <p>Increasing levels of within-host diversity also had an effect on the estimated <i>R</i><sub>0</sub> (<a href="#pcbi.1008122.g005">Fig 5B</a>). While increasing levels of within-host diversification tended to increase the estimated <i>R</i><sub>0</sub>, only at the highest level of within-host diversity (<i>r</i> = 1) did the overestimation become significantly higher than under no diversification. As within-host diversity pushes the phylogenetic node heights backwards in time, known as the pre-transmission interval [<a href="#pcbi.1008122.ref034">34</a>], the mean infectious period (<i>Î³</i><sup>âˆ’1</sup>) appears prolonged. However, the average time interval between neighboring transmission events, which corresponds to the mean transmissibility rate <i>Î¼</i><sub>Î»</sub>, is less affected by the within-host diversity. Therefore, the estimated <i>R</i><sub>0</sub> = <i>Î¼</i><sub>Î»</sub>/<i>Î³</i> in the presence of within-host diversity will increase with the level of diversification.</p> </div> <div class="thepowerofdetectingtransmissionheterogeneityincreaseswithheterogeneitylevelandsamplesize" title="sec"> <div class="title" tagx="title" title="title">
The power of detecting transmission heterogeneity increases with heterogeneity level and sample size</div> <p>For <i>Î¼</i><sub>Î»</sub> = 1 and <i>Î³</i> = 1/2.5, we estimated the expected power to detect heterogeneity for various sample sizes and heterogeneity levels. We reject homogeneity if the estimated heterogeneity <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e107g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mi_UNKNOWN">
V</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> </div></div></div> exceeds the upper-bound of the 95% confidence interval which was estimated under the homogenous situation by setting <i>CV</i><sub>Î»</sub> = 0. This upper-bound was calculated from 500 simulations.<p /> <p>The power to detect transmission heterogeneity depends on the sample size and true level of transmission heterogeneity (<a href="#pcbi.1008122.g006">Fig 6</a>). We investigated two different sampling size effects; 1) the sequencing ratio at the time of stopping simulation (when 100 persons have been diagnosed), and 2) with a fixed sequencing ratio <i>Ï�</i><sub><i>SD</i></sub> at a time when a certain number of persons have been diagnosed (from 20 to 100 persons). The effects on detection power of these two sampling scenarios were similar. For a given sample size (i.e. a particular colored line in <a href="#pcbi.1008122.g006">Fig 6</a>), the power naturally increased with the level of heterogeneity. From a public health action point of view, this is again encouraging as higher heterogeneity levels suggest that intervention may be more beneficial, which we address further below. Naturally, the power also increased with larger sample size (#(<i>D</i>) in panel A, and <i>Ï�</i><sub><i>SD</i></sub> in panel B).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1008122.g006"><span class="label" tagx="label" title="label">Fig 6</span></a></div>   <div class="title" tagx="title" title="title">
Comparison of power of detecting heterogeneity under different sample sizes.</div> <p>Here #(<i>D</i>) denotes the number of diagnosed individuals. A. The sequencing ratio is fixed as 1, while the simulation stopped when there were different number of diagnosed individuals. B. The simulation was stopped when there were 100 diagnosed individuals (#(<i>D</i>) = 100), while the sequencing ratio increases from <i>Ï�</i><sub><i>SD</i></sub> = 0.2 to 1, corresponding to a sample size going from 20 to 100 (colored curves). The power is estimated based on 200 simulations.</p>   </div> </div> <div class="computingtime" title="sec"> <div class="title" tagx="title" title="title">
Computing time</div> <p>There is a growing demand of analyzing large sequence databases where transmission heterogeneity may exists and be desirable to detect [<a href="#pcbi.1008122.ref035">35</a>]. Hence, we evaluated the computing time required to process trees with the number of tips (individuals) varied from 100 to 4,000. These results are summarized in supplementary <a href="#pcbi.1008122.s008">S8 Fig</a>. Our result indicated that the average computing time (over heterogeneity level <i>CV</i><sub>Î»</sub> varying from 1 to 5) scales linearly with the size of the tree. For instance, it required about 54.8 minutes (averaging over heterogeneity levels from 1 to 5) to process a tree with 4,000 tips. We also observed that the computing time for a tree with low level of heterogeneity (i.e. <i>CV</i><sub>Î»</sub> = 1) is much longer than that of analyzing a tree with high level of heterogeneity (i.e. <i>CV</i><sub>Î»</sub> â‰¥ 3). And the difference between these two computing times increases with the size of the tree. This is due to the fact that the proposed algorithm needs to reconstruct the transmission history while estimating the model parameters. This reconstruction is easier to proceed under the situation with high level of heterogeneity where few super-spreaders contribute to most of the transmission events. Hence, we expect that more computing time is needed to process a tree with heterogeneity smaller than 1 (<i>CV</i><sub>Î»</sub> &amp;lt; 1).</p> </div> </div> <div class="analysisofrealhivoutbreakdata" title="sec"> <div class="title" tagx="title" title="title">
Analysis of real HIV outbreak data</div> <p>We investigate HIV-1 DNA sequences from three local epidemics in Sweden ranging from fast explosive outbreak to more slow spread over longer time periods (<a href="#pcbi.1008122.g007">Fig 7</a>). The three corresponding genealogies were selected from a recent publication on HIV-1 spread in Sweden and neighboring Scandinavian countries [<a href="#pcbi.1008122.ref036">36</a>], but the datasets used here contain only Swedish sequences. The first genealogy, denoted IDU_AE, represents a rapid outbreak of CRF01_AE (circulating recombinant form number 1 of subtype A and E viruses) infections among intravenous drug users (IDUs) in Stockholm in 2006 and 2007 [<a href="#pcbi.1008122.ref037">37</a>]. The genealogy contains 83 taxa which were sampled from 2003 to 2010. The second genealogy, denoted IDU_B, represents slower local spread of subtype B infections among IDUs in Stockholm with sampling dates ranging from 2003 to 2010 [<a href="#pcbi.1008122.ref038">38</a>]. This genealogy contains 51 taxa which were sampled from 2002 to 2009. The last genealogy, denoted MSM_B, represents longstanding local spread of subtype B virus among men who have sex with men (MSM) with sampling dates ranging from 1994 to 2010. This genealogy contains 18 taxa which were sampled from 1994 to 2010. All genealogies were monophyletic relative to other sequences in the Swedish-Scandinavian study and BLASTâ€™ed international sequences [<a href="#pcbi.1008122.ref036">36</a>].</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1008122.g007"><span class="label" tagx="label" title="label">Fig 7</span></a></div>   <div class="title" tagx="title" title="title">
Estimation of heterogeneity and other epidemiological parameters for three HIV outbreaks in Sweden: IDU_AE (red), IDU_B(blue) and MSM_B (grey).</div> <p>On the top four panels, the distributions of the estimated parameters from a sample of 10000 posterior genealogies of each dataset are presented: the rate of diagnosis (A), the basic reproduction number (B), the Coefficient of Variation (C) and the <i>Sackin</i> Index normalized with the â€˜PDAâ€™ method (D). On the bottom, the time-scaled genealogies with the highest posterior probability are presented.</p>   </div> <p>For each data set, we analyzed a random sample of 10000 genealogies from the posterior distribution (generated using BEAST [<a href="#pcbi.1008122.ref039">39</a>]). <a href="#pcbi.1008122.g007">Fig 7</a> summarizes the results of the parameter estimates. Overall, the IDU_B data showed the lowest diagnosis rate (<span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e108g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div>), meaning the individuals infected in this outbreak had a longer transmission period before being diagnosed (and prevented to spread further). Interestingly, the IDU_B outbreak also showed the highest <i>R</i><sub>0</sub>, suggesting that the low value of <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e109g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div></span> may have driven up <i>R</i><sub>0</sub> = <i>Î¼</i><sub>Î»</sub>/<i>Î³</i>. <i>R</i><sub>0</sub> did not seem directly linked to transmission heterogeneity, however. Similarly, Sackinâ€™s index, which measures tree balance (high values indicate unbalanced trees where one side of a split carries more taxa than the other side), did not find any significant differences between the data that contained an outbreak (IDU_AE) and the data from more even spread (IDU_B). Instead, as one might have expected, <span class="inline-formula" title="inline-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e110g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mi_UNKNOWN">
V</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
Î»</div> </div></div></div></span> correlates with the speed of the different outbreaks, i.e. the highest heterogeneity was inferred in the data that contained the fast outbreak, which occurred in the IDU_AE epidemic, and the lowest heterogeneity was inferred in the slowest spreading epidemic (MSM_B). These observations agree with previous analyses of these local epidemics, where it was reported that there was a rapid outbreak, during a limited time period, among IDU infected with CRF01_AE in Sweden following a similar outbreak in Finland [<a href="#pcbi.1008122.ref038">38</a>]. This suggests that it took some time for HIV-1 CRF01_AE to reach one or several individuals with high contact numbers after it had entered the Swedish IDU community. Once the individuals with high contact rates were reached, a rapid outbreak followed, infecting many susceptibles. Finally, as shown in [<a href="#pcbi.1008122.ref038">38</a>], the outbreak subsequently slowed down to pre-outbreak levels, presumably because the recipients of the high-contact persons had been exhausted. Lower levels of transmission heterogeneity was observed in the slower spread that took place in the IDU_B community [<a href="#pcbi.1008122.ref037">37</a>], suggesting that the contact patterns in this community were more even among members.<p /> </div> <div class="simulationoftargetedcontacttracing:continuousmonitoringandcross-sectionalsampling" title="sec"> <div class="title" tagx="title" title="title">
Simulation of targeted contact tracing: Continuous monitoring and cross-sectional sampling</div> <p>Next, we evaluated phylogeny-guided contact tracing policies based on simulation. We again set the mean transmissibility rate at <i>Î¼</i><sub>Î»</sub> = 1 and the rate of diagnosis at <i>Î³</i> = 1/2.5, which resulted in the basic reproduction number as <i>R</i><sub>0</sub> = <i>Î¼</i><sub>Î»</sub>/<i>Î³</i> = 2.5. Here, one unit of time corresponded to one year as before, and we set the time period <i>t</i><sub><i>c</i></sub> in which we calculate the <i>NCE</i>-value to <i>t</i><sub><i>c</i></sub> = 1.25 (corresponding to half of the average infectious period). In addition, the heterogeneity varied from <i>CV</i><sub>Î»</sub> = 0 to 5.</p> <p>We define â€˜continuous monitoringâ€™ as the typical public health situation where samples (sequences of diagnosed cases) are continuously collected throughout an epidemic and where for each collected sample it is decided to perform (additional) contact tracing or not. The second situation is denoted â€˜cross-sectional samplingâ€™ which is the situation where all samples are collected (or re-analyzed) at some common time <i>t</i>, for instance when the epidemiological status is investigated in an otherwise unmonitored population or when additional preventive measures are decided upon or in retrospective contact tracing.</p> <div class="usingapathogenphylogenyimprovesdiseasepreventionundercontinuousmonitoring" title="sec"> <div class="title" tagx="title" title="title">
Using a pathogen phylogeny improves disease prevention under continuous monitoring</div> <p>When no transmission heterogeneity is present (<i>CV</i><sub>Î»</sub> = 0), targeting individuals with higher than average <i>NCE</i> for contact tracing has no effect on the resulting epidemic size (<a href="#pcbi.1008122.g008">Fig 8A</a>), i.e. randomly selecting the same number of persons for contact tracing has the same effect as a phylogenetically informed strategy (the dashed and solid lines are on top of each other in this case). The reason why preventing (i.e. contact tracing) persons with <i>NCE</i> â©¾ 1 has higher effect than <i>NCE</i> â©¾ 2 is simply because <i>NCE</i> â©¾ 1 involves a larger number of persons that are contact traced, shown in the sidebar â€˜Fraction of contact tracedâ€™. The â€˜no preventionâ€™ line shows the epidemic growth when no prevention (i.e. no additional contact tracing) was administered and is thus the maximum size of the simulated epidemic at any time point. Reduction from this level indicates the prevention effect.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1008122.g008"><span class="label" tagx="label" title="label">Fig 8</span></a></div>   <div class="title" tagx="title" title="title">
Comparison of <i>NCE</i>-based contact tracing and random contact tracing under the situation of continuous monitoring.</div> <p>The relative infection sizes (defined in <a href="#pcbi.1008122.s010">S1 Text</a>) of the <i>NCE</i> â©¾ <i>m</i> contact tracing (solid lines) and the corresponding random contact tracing (dashed lines) are compared for four threshold values: <i>m</i> = 1 (lines with â—�), <i>m</i> = 2 (lines with <i>triangledown</i>), <i>m</i> = 3 (lines with +), and <i>m</i> = 4 (lines with *). The subplots on the right side of all panels show the fraction of contact traced for all these prevention policies. Four panels show the comparison under four levels of heterogeneity respectively, i.e. <i>CV</i><sub>Î»</sub> = 0 (A), <i>CV</i><sub>Î»</sub> = 1 (B), <i>CV</i><sub>Î»</sub> = 3 (C), and <i>CV</i><sub>Î»</sub> = 5 (D). These results are the mean of 300 simulations.</p>   </div> <p>When transmission heterogeneity was present (i.e. <i>CV</i><sub>Î»</sub> &amp;gt; 0), the relative epidemic size under a phylogeny-guided prevention (based on <i>NCE</i>) was always smaller than that of random prevention at the same proportion of persons (<a href="#pcbi.1008122.g008">Fig 8Bâ€"8D</a>). At very high level of heterogeneity <i>CV</i><sub>Î»</sub> = 3, phylogeny-guided prevention typically reduced the epidemic by an additional 10% over random prevention at 2.5 years out, and approximately an additional 10% for each <i>NCE</i> step. Moreover, the advantage of a <i>NCE</i> â©¾ <i>m</i> strategy over the corresponding random prevention strategy increased with the level of heterogeneity. For instance, if extremely high heterogeneity occurred (<i>CV</i><sub>Î»</sub> = 5), by preventing spread at <i>NCE</i> â©¾ 4 one would only need to contact trace 20% of the population to get about 75% reduction in total number of infections 2.5 years later. This means that a phylogenetically informed prevention strategy (using the <i>NCE</i> concept) becomes more valuable the more heterogeneous transmission rates are.</p> <p>From a public health point of view, where resources always are limited, using the pathogen phylogeny to allocate resources becomes more efficient the more heterogeneity that exists in the epidemic, and the <i>NCE</i> guided prevention can pinpoint where contact tracing and prevention would make the largest impact in reducing future infections. This becomes clear when comparing the fraction of contact traced and the relative effect of <i>NCE</i>-guided prevention in the different <i>CV</i><sub>Î»</sub> and <i>NCE</i> cases, i.e. the fraction of contact traced in the <i>NCE</i> â©¾ <i>m</i> strategies decreased as the threshold value <i>m</i> increased while the relative effect over random prevention in the <i>NCE</i> â©¾ <i>m</i> strategies increased as the threshold value <i>m</i> increased (see the supplementary <a href="#pcbi.1008122.s005">S5 Fig</a>). Therefore, under low levels of heterogeneity (estimated using our general heterogeneity detection method), one may choose a small threshold <i>m</i> to gain more effect on reducing the epidemic size, while under high levels of heterogeneity, one may instead choose a high threshold <i>m</i>, involving less resources, to invoke efficient control.</p> <p>For pathogens that accumulate within-host diversity, such as HIV-1, we also evaluated the phylogeny-guided prevention strategy when significant within-host diversity accumulates. The effect of a <i>NCE</i>-strategy was, in fact, higher in the presence of within-host diversity (<i>r</i> = 1) than when no within-host diversity accumulated (<i>r</i> = 0) (<a href="#pcbi.1008122.g009">Fig 9</a>). This happens because large <i>NCE</i> values are associated with short branches, and the effect of within host diversity is to make short branches only slightly longer whereas longer branches are increased more due to within host diversity. As a consequence, within-host diversity has the effect of accentuating the difference between short and long branches thus leading to a better distinction between subtrees with a relatively large number of coalescent events (large <i>NCE</i>) compared to subtrees with lower number of coalescent events.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1008122.g009"><span class="label" tagx="label" title="label">Fig 9</span></a></div>   <div class="title" tagx="title" title="title">
Comparison of the performance of <i>NCE</i>-based policies under the situations of with/without within-host evolutionary dynamics.</div> <p>Here <i>r</i> is the diversification rate of within-host virus population. The solid lines (<i>r</i> = 0 lineage/day) and the dashed lines (<i>r</i> = 1 lineage/day) are the results with/without within-host diversity respectively. The performance of the <i>NCE</i> â©¾ <i>m</i> policy is measured by the relative effect over random prevention (please see text for definition). Three levels of threshold <i>m</i> are calculated: <i>m</i> = 1 (blue), <i>m</i> = 2 (orange), and <i>m</i> = 2 (green).</p>   </div> <p>Within-host diversity also has the effect of reducing the fraction of contact traced for given threshold <i>m</i> in <i>NCE</i> (<a href="#pcbi.1008122.s006">S6 Fig</a>). Because within-host diversity pushes the phylogenetic node heights backwards in time, known as the pre-transmission interval [<a href="#pcbi.1008122.ref034">34</a>], there will be fewer nodes at any given time-distance into the tree from the tips. Within-host diversity therefore reduces the <i>NCE</i>-count on all lineages, leading to a reduction in the fraction of contact traced. Hence, one may adjust the depth <i>t</i><sub><i>c</i></sub> at which <i>NCE</i> is calculated depending on how much diversity the pathogen under study typically accumulates.</p> <p>Finally, we evaluated the phylogeny-guided prevention under two different models of contact tracing: single-step tracing (where only direct recent contacts of the diagnosed are tested and prevented from further spread) or iterative tracing (where new cases from the single-step tracing are also contact traced, and so on) [<a href="#pcbi.1008122.ref024">24</a>, <a href="#pcbi.1008122.ref040">40</a>]. The relative effect of phylogeny-guided iterative contact tracing was more effective than single-step contact tracing at higher levels of the <i>NCE</i>-threshold (<i>m</i> = 2 and 3) in <a href="#pcbi.1008122.g010">Fig 10</a>. Contrary, at <i>NCE</i> â©¾ 1, single-step contact tracing did better at lower levels of transmission heterogeneity. Hence, iterative contact tracing is more advantageous when medium to high levels of transmission heterogeneity exist, where higher <i>NCE</i>-thresholds can pick up transmission chains where a super-spreader exists. At very high heterogeneity levels (<i>CV</i><sub>Î»</sub> â©¾ 3), the iterative contact tracing advantage gradually diminishes because the probability to find the super-spreaders increases after just a single-step contact tracing.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1008122.g010"><span class="label" tagx="label" title="label">Fig 10</span></a></div>   <div class="title" tagx="title" title="title">
Comparison of the performance of <i>NCE</i>-based policies under different modes of prevention.</div> <p>The solid lines/dashed lines are the results under the mode of single-step contact tracing/iterative contact tracing respectively. The performance of the <i>NCE</i> â©¾ <i>m</i> policy is measured by the relative effect over random prevention (please see text for definition). Three levels of threshold <i>m</i> are calculated: <i>m</i> = 1 (blue), <i>m</i> = 2 (orange), and <i>m</i> = 2 (green).</p>   </div> <p>As somewhat surprizing observation was that the fraction of contact traced under iterative contact tracing was actually found to besmaller than under the single-step contact tracing (<a href="#pcbi.1008122.s007">S7 Fig</a>). For each selected individual to contact trace there will of course be more (or at least not fewer) to contact trace under iterative contact tracing. However, iterative contact tracing has an even bigger negative effect on the spreading, such that the overall fraction (or number) being contact traced in the iterative setting is smaller than when applying single-step contact tracing.</p> </div> <div class="time-to-diagnosisisinformativeforcontacttracingundercross-sectionalsampling" title="sec"> <div class="title" tagx="title" title="title">
Time-to-diagnosis is informative for contact tracing under cross-sectional sampling</div> <p>When cross-sectional samples have been collected, phylogeny-guided prevention has no advantage over a random prevention strategy (<a href="#pcbi.1008122.g011">Fig 11</a>). Thus, to take advantage of the epidemic information captured by the pathogen phylogeny, continuous monitoring provides better public health prevention capacity. If only a cross-sectional sample is available, however, we find that targeting the most recently diagnosed individuals (<i>MRD</i>) for additional contact tracing performs better in preventing future spread than a random selection of the same number of individuals (<a href="#pcbi.1008122.g011">Fig 11</a>) or by selecting individuals to contact trace based on the <i>NCE</i>-value. At prevention fractions <i>p</i> = 0.2 and <i>p</i> = 0.5 (20% and 50% being contact traced, respectively), the relative effect of the <i>MRD</i>-strategy was constantly larger than random selection (of which the relative effect was set to 1) as well as that of the <i>NCE</i>-based strategy.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1008122.g011"><span class="label" tagx="label" title="label">Fig 11</span></a></div>   <div class="title" tagx="title" title="title">
Relative effect of <i>NCE</i>-based strategy (blue lines) and MRD strategy (red lines) over random prevention under the situation of Cross-sectional prevention.</div> <p>The fraction of contact traced is fixed as <i>p</i> = 0.2 (lines with â–½) and <i>p</i> = 0.5 (lines with â‹†).</p>   </div> </div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>In this study we have shown that when transmission heterogeneity exists, a phylogeny-guided prevention strategy is more efficient than randomly selecting cases for contact tracing in order to reduce the number of future infections. We first developed a general method to identify the level of transmission heterogeneity in a population, i.e. how much variation in the rate of transmission contacts that exist in a human population where a pathogen spreads. We then developed a phylogenetic measure, the <i>NCE</i>â€"the Number of Coalescent Events over a fixed time-interval ending at time of diagnosisâ€"to identify which infected individuals to target for (additional) contact tracing. It was shown that, under the typical situation where pathogen samples are continuously collected (continuous monitoring), a phylogeny-guided prevention approach was clearly superior in effectively allocating public health resources to reduce future disease spread.</p> <p>We show that our heterogeneity detection method and phylogeny-guided approach work under a variety of realistic conditions. The estimates of heterogeneity and basic reproduction number are robust to assumptions about the susceptible population size and exact sampling fraction (the fraction of diagnosed individuals that are sequenced), both of which may be difficult to know exactly in a real epidemic. Also when changing the recovery rate of the infected individual, allowing autocorrelation in transmission rates, or adding heterogeneity through time to individual-level transmissibility, the estimate of heterogeneity is still less affected. Of particular importance is that our method also works when a pathogen accumulates genetic variation within each host. Such within-host diversity is significant in e.g. HIV infections. If not taken into account, tree measurements may significantly mislead epidemiological estimates by implicitly assuming identity between transmission history and pathogen phylogeny. It is well established that the between-host pathogen phylogeny will display nodes corresponding to the transmitted lineages biased backwards in time, as described by the pre-transmission interval [<a href="#pcbi.1008122.ref034">34</a>], as well as potential lineage disordering relative to the transmission history [<a href="#pcbi.1008122.ref021">21</a>]. While previously often ignored, the within-host diversity has recently been shown to have significant impact on inferences about pathogen epidemics, sometimes rendering results completely as compared to when within-host diversity was ignored [<a href="#pcbi.1008122.ref011">11</a>, <a href="#pcbi.1008122.ref021">21</a>, <a href="#pcbi.1008122.ref041">41</a>, <a href="#pcbi.1008122.ref042">42</a>]. Here, we show that taking within-host diversity into account may, in fact, improve the prevention effect (<a href="#pcbi.1008122.g009">Fig 9</a>).</p> <p>When applying our method for detecting possible spread heterogeneity to three separate sub-epidemics from the larger Swedish HIV-1 epidemic it was found that the highest heterogeneity was inferred from the data that included the episodic IDU_AE outbreak. Interestingly, this outbreak did not show significant differences from the other sub-epidemics when analyzing Sackinâ€™s index or <i>R</i><sub>0</sub> (which is also affected by the estimated diagnosis rate <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1008122.e111g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
Î³</div> <div class="mo_UNKNOWN">
^</div> </div></div></div>). This suggests that our methodology is useful for identifying heterogeneous spread when typical tree balance or well-known epidemiological parameters fail.<p /> <p>In some countries and settings where HIV transmission is criminalized there may be ethical limitations to the use of phylogenetics and other ways to identify clusters with active HIV transmission. We acknowledge this complication, but show that our approach could limit HIV transmissions in settings where such limitations do not exist.</p> <p>Our model treats heterogeneity as a single feature, while in real life there are several factors that may contribute to heterogeneous spread rates. It is possible that modeling one or several of these factors separately may be valuable. One such factor relates to social network structure, modeling heterogeneities in terms of contacts. Secondly, the current model assumes that the transmissibility of different individuals are independent. A (more complicated) alternative could be to assume that transmissibility of infectors and their cases are positively correlated, which would imply assortativity in sexual activity between sex-partners. Finally, individuals may also differ in terms of testing (i.e. diagnosis) rates; in our model all individuals were assumed to have equal rates of testing. For example, among MSM, individuals with risky behavior (high contact rates) may also test themselves more often. This simplifying assumption may lead to a systematic loss of sensitivity as sampling may not be complete and be biased towards some group of infected persons. Again, the heterogeneity in our model may be interpreted as a combination of all these and other aspects.</p> <p>In conclusion, with the increase of pathogen genetic data in public health databases, phylogeny-guided prevention will benefit public health efforts. As relevant epidemic information about differences in number of contacts is effectively â€œrecordedâ€� in a phylogeny by the evolutionary process of the pathogen, it only makes sense to use this information to the public health benefit. Hence, phylogeny-guided prevention results in more efficient epidemic control, which should reduce disease burden and costs to society.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title">
Average branch length of the simulated tree under different conditions.</div> <p>Average branch length of the simulated tree under varied basic reproduction number (x-axis) and varied levels of heterogeneity (y-axis). Under each condition, the simulation started from one infection and stopped when there were 100 recovered individuals. The average branch length was calculated as the total branch length of the reconstructed tree over the number of tips. The results were the average over 1000 simulations.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1008122.s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Fig</span> <div class="title" tagx="title" title="title">
Inference of heterogeneity under various lengths of mean infectious period <i>Î³</i><sup>âˆ’1</sup>.</div> <p>In each panel, the black line denotes the true value that was used to generate the simulated data. The colored curves are the means of the estimates under different levels of <i>Î³</i><sup>âˆ’1</sup>. The shaded area denotes 95% confidence interval estimated when <i>Î³</i><sup>âˆ’1</sup> = 2.5. These results are obtained from 100 simulation replicates where the average transmissibility rate <i>Î¼</i><sub>Î»</sub> was fixed as 1, the sequencing ratio <i>Ï�</i> = 0.9, and the simulation stopped when there were 100 diagnosed individuals. The panel A is coefficient of variation (<i>CV</i><sub>Î»</sub>) and the panel B is standard deviation of infectivity rate (<i>Ïƒ</i><sub>Î»</sub>).</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1008122.s002.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Fig</span> <div class="title" tagx="title" title="title">
Performance of parameter estimation under various lengths of mean infectious period <i>Î³</i><sup>âˆ’1</sup>.</div> <p>In each panel, the colored curves are the means of relative biases (the bias over the true value) under different levels of <i>Î³</i><sup>âˆ’1</sup>. The shaded area denotes the relative bias of the 95% confidence interval estimated when <i>Î³</i><sup>âˆ’1</sup> = 2.5. These results are obtained from 100 simulation replicates where the average transmissibility rate <i>Î¼</i><sub>Î»</sub> was fixed as 1, the sequencing ratio <i>Ï�</i> = 0.9, and the simulation stopped when there were 100 diagnosed individuals. A. basic reproduction number (<i>R</i><sub>0</sub>), B. average infectivity rate (<i>Î¼</i><sub>Î»</sub>), and C. recovery rate (<i>Î³</i>).</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1008122.s003.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S4 Fig</span> <div class="title" tagx="title" title="title">
Comparison of parameter estimation under constant/time-varying transmissibility.</div> <p>The lines with circles â€œâ—‹â€� are the estimate under the situation of constant transmissibility, and the shaded area denote the 95% confidence interval from 100 simulations. The lines with triangles â€œâ–½â€� are the estimates under the situation with time-varying transmissibility (TVT). The comparison is performed under the situation of <i>R</i><sub>0</sub> = 2.5 and the heterogeneity (<i>CV</i><sub>Î»</sub>) varies from 0 to 5. Sample size n = 100 and simulation runs = 100.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1008122.s004.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S5 Fig</span> <div class="title" tagx="title" title="title">
Relative effect over random prevention of <i>NCE</i><sub><i>m</i></sub> strategy under the continuous monitoring scenario.</div> <p>Four levels of threshold <i>c</i> have been calculated: <i>m</i> = 1 (blue), <i>m</i> = 2 (orange), <i>m</i> = 3 (green), and <i>m</i> = 4 (red). Results in (a) and (b) are the mean of 300 simulations.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1008122.s005.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S6 Fig</span> <div class="title" tagx="title" title="title">
Comparison of fraction of contact traced of <i>NCE</i><sub><i>m</i></sub> strategy under the situation of with/without within-host diversity (solid/dash lines respectively).</div> <p>Three levels of threshold <i>m</i> have been calculated: <i>m</i> = 1 (blue), <i>m</i> = 2 (orange), and <i>m</i> = 3 (green). Results are the mean of 300 simulations.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1008122.s006.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S7 Fig</span> <div class="title" tagx="title" title="title">
Comparison of fraction of contact traced of <i>NCE</i><sub><i>m</i></sub> strategy under the situation of single-step/iterative contact tracing (solid/dash lines respectively).</div> <p>Three levels of threshold <i>m</i> have been calculated: <i>m</i> = 1 (blue), <i>m</i> = 2 (orange), and <i>m</i> = 3 (green). Results are the mean of 300 simulations.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1008122.s007.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S8 Fig</span> <div class="title" tagx="title" title="title">
Computing times required for inferring heterogeneity from trees with varying numbers of tips.</div> <p>Computing times (y-axis) required for processing trees with varying numbers of tips (each tip represents a diagnosed individual). For each number of tips (x-axis), we simulated trees with different levels of heterogeneity (i.e., <i>CV</i><sub>Î»</sub> = 1, 2, â‹¯, 5). Each point represents the average computing time based on 10 simulations with given heterogeneity level and the size of tree. The number of tips was varied along the following sequence: 100, 200, 400, 1000, 2000, 4000. The dashed line represents the average computing time over all levels of heterogeneity. All runs were executed on an Intel Core i7 processor (Mac mini 2018).</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1008122.s008.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S9 Fig</span> <div class="title" tagx="title" title="title">
Performance of new method and the Markov-modulated Poisson process (MMPP) based genetic clustering method on simulated data.</div> <p>Virus genealogies were simulated under two scenarios: without autocorrelation in transmission rates (A and B) and with autocorrelation (C and D). In the former cases, each infected individual independently draws its transmission rate from a binomial distribution, i.e., choosing the slow rate Î»<sub>1</sub> with probability 1 âˆ’ <i>Ï€</i><sub><i>c</i></sub> = 0.9 and choosing the fast rate Î»<sub>2</sub> with probability <i>Ï€</i><sub><i>c</i></sub> = 0.1. In (A), Î»<sub>1</sub> = 2 and Î»<sub>2</sub> = 6, corresponding to mean rate of <i>Î¼</i><sub>Î»</sub> = 2.4 and a low level of heterogeneity (<i>CV</i><sub>Î»</sub> = 0.5). In (B)Î»<sub>1</sub> = 1 and Î»<sub>2</sub> = 15, corresponding to mean rate of <i>Î¼</i><sub>Î»</sub> = 2.4 and a high level of heterogeneity (<i>CV</i><sub>Î»</sub> = 1.75). In the cases with autocorrelation in transmission rates, each newly infected individual switch its transmission rate with probability <i>Ï€</i><sub><i>s</i></sub> and remains its infecteeâ€™s rate with probability 1 âˆ’ <i>Ï€</i><sub><i>s</i></sub>. In both (C) and (D), Î»<sub>1</sub> = 0.9 and Î»<sub>2</sub> = 8.1 with different switching probability, i.e., <i>Ï€</i><sub><i>s</i></sub> = 0.2 in (C), corresponding to <i>Î¼</i><sub>Î»</sub> = 5.22 and low level heterogeneity <i>CV</i><sub>Î»</sub> = 0.79, and <i>Ï€</i><sub><i>s</i></sub> = 0.8 in (D) corresponding to <i>Î¼</i><sub>Î»</sub> = 2.64 and low level heterogeneity <i>CV</i><sub>Î»</sub> = 1.27. And the diagnosis rate is fixed as <i>Î³</i> = 1. The x- and y-axes correspond to the estimated <i>Î¼</i><sub>Î»</sub> and the estimated <i>CV</i><sub>Î»</sub> respectively. Each point represents the outcome when analyzing one of 100 replicates.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1008122.s009.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Text</span> <div class="title" tagx="title" title="title">
Supporting information on models, simulation methods and the phylogenetic analysis of HIV data in Sweden.</div> <p>Includes details on simulating the virus genealogy, how the correction for incomplete transmission chain were derived, the preprocess of the real epidemiological data from Sweden, how the evaluation of control measures was performed based on simulation, and the pseudo-code of the proposed algorithm.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1008122.s010.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pcbi.1008122.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lloyd-Smith</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schreiber</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kopp</span><span class="given-names" tagx="given-names" title="given-names">PE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Getz</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span>. <span class="mixed-article-title" title="mixed-article-title">Superspreading and the effect of individual variation on disease emergence</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">438</span>(<span class="issue" tagx="issue" title="issue">7066</span>):<span class="fpage" tagx="fpage" title="fpage">355</span>â€"<span class="lpage" tagx="lpage" title="lpage">359</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature04153">10.1038/nature04153</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16292310">16292310</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>. <span class="mixed-article-title" title="mixed-article-title">Transmission dynamics of HIV infection</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">1987</span>;<span class="volume" tagx="volume" title="volume">326</span>(<span class="issue" tagx="issue" title="issue">6109</span>):<span class="fpage" tagx="fpage" title="fpage">137</span>â€"<span class="lpage" tagx="lpage" title="lpage">142</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/326137a0">10.1038/326137a0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3821890">3821890</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woolhouse</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dye</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Etard</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charlwood</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garnett</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Heterogeneities in the transmission of infectious agents: implications for the design of control programs</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span>. <span class="year" tagx="year" title="year">1997</span>;<span class="volume" tagx="volume" title="volume">94</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">338</span>â€"<span class="lpage" tagx="lpage" title="lpage">342</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.94.1.338">10.1073/pnas.94.1.338</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garske</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rhodes</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="mixed-article-title" title="mixed-article-title">The effect of superspreading on epidemic outbreak size distributions</span>. <span class="source" tagx="source" title="source">Journal of Theoretical Biology</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">253</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">228</span>â€"<span class="lpage" tagx="lpage" title="lpage">237</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2008.02.038">10.1016/j.jtbi.2008.02.038</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18423673">18423673</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wertheim</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pond</span><span class="given-names" tagx="given-names" title="given-names">SLK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forgione</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mehta</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murrell</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Social and genetic networks of HIV-1 transmission in New York City</span>. <span class="source" tagx="source" title="source">PLoS pathogens</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">e1006000</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1006000">10.1371/journal.ppat.1006000</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28068413">28068413</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Resik</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ping</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouri</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joanes</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PÃ©rez</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Limitations to contact tracing and phylogenetic analysis in establishing HIV type 1 transmission networks in Cuba</span>. <span class="source" tagx="source" title="source">AIDS research and human retroviruses</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">23</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">347</span>â€"<span class="lpage" tagx="lpage" title="lpage">356</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1089/aid.2006.0158">10.1089/aid.2006.0158</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17411367">17411367</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leitner</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Escanilla</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franzen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uhlen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albert</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">Accurate reconstruction of a known HIV-1 transmission history by phylogenetic tree analysis</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span>. <span class="year" tagx="year" title="year">1996</span>;<span class="volume" tagx="volume" title="volume">93</span>(<span class="issue" tagx="issue" title="issue">20</span>):<span class="fpage" tagx="fpage" title="fpage">10864</span>â€"<span class="lpage" tagx="lpage" title="lpage">10869</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.93.20.10864">10.1073/pnas.93.20.10864</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonhoeffer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Uncovering epidemiological dynamics in heterogeneous host populations using phylogenetic methods</span>. <span class="source" tagx="source" title="source">Philosophical Transactions of the Royal Society of London B: Biological Sciences</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">368</span>(<span class="issue" tagx="issue" title="issue">1614</span>):<span class="fpage" tagx="fpage" title="fpage">20120198</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2012.0198">10.1098/rstb.2012.0198</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382421">23382421</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCloskey</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">AFY</span></span>. <span class="mixed-article-title" title="mixed-article-title">A model-based clustering method to detect infectious disease transmission outbreaks from sequence variation</span>. <span class="source" tagx="source" title="source">PLOS Computational Biology</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">11</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1005868">10.1371/journal.pcbi.1005868</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29131825">29131825</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barido-Sottani</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaughan</span><span class="given-names" tagx="given-names" title="given-names">TG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Detection of HIV transmission clusters from phylogenetic trees using a multi-state birthâ€"death model</span>. <span class="source" tagx="source" title="source">Journal of the Royal Society Interface</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">15</span>(<span class="issue" tagx="issue" title="issue">146</span>):<span class="fpage" tagx="fpage" title="fpage">20180512</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsif.2018.0512">10.1098/rsif.2018.0512</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giardina</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero-Severson</span><span class="given-names" tagx="given-names" title="given-names">EO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albert</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Britton</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leitner</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span>. <span class="mixed-article-title" title="mixed-article-title">Inference of Transmission Network Structure from HIV Phylogenetic Trees</span>. <span class="source" tagx="source" title="source">PLOS Computational Biology</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">59865</span>.</span></li> <li tag="ref"><a name="pcbi.1008122.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasmussen</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouyos</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ¼nthard</span><span class="given-names" tagx="given-names" title="given-names">HF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phylodynamics on local sexual contact networks</span>. <span class="source" tagx="source" title="source">PLoS computational biology</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">e1005448</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1005448">10.1371/journal.pcbi.1005448</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28350852">28350852</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leventhal</span><span class="given-names" tagx="given-names" title="given-names">GE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouyos</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Von Wyl</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yerly</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BÃ¶ni</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Inferring epidemic contact structure from phylogenetic trees</span>. <span class="source" tagx="source" title="source">PLoS computational biology</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">e1002413</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1002413">10.1371/journal.pcbi.1002413</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22412361">22412361</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maddison</span><span class="given-names" tagx="given-names" title="given-names">WP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Midford</span><span class="given-names" tagx="given-names" title="given-names">PE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Otto</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span>. <span class="mixed-article-title" title="mixed-article-title">Estimating a Binary Characterâ€™s Effect on Speciation and Extinction</span>. <span class="source" tagx="source" title="source">Systematic Biology</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">56</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">701</span>â€"<span class="lpage" tagx="lpage" title="lpage">710</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/10635150701607033">10.1080/10635150701607033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17849325">17849325</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KÃ¼hnert</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouyos</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shirreff</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">PeÄ�erska</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scherrer</span><span class="given-names" tagx="given-names" title="given-names">AU</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BÃ¶ni</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Quantifying the fitness cost of HIV-1 drug resistance mutations through phylodynamics</span>. <span class="source" tagx="source" title="source">PLOS Pathogens</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">14</span>(<span class="issue" tagx="issue" title="issue">2</span>).</span></li> <li tag="ref"><a name="pcbi.1008122.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magiorkinis</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sypsa</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magiorkinis</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paraskevis</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katsoulidou</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belshaw</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Integrating phylodynamics and epidemiology to estimate transmission diversity in viral epidemics</span>. <span class="source" tagx="source" title="source">PLoS computational biology</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">e1002876</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1002876">10.1371/journal.pcbi.1002876</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382662">23382662</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volz</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pond</span><span class="given-names" tagx="given-names" title="given-names">SLK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ward</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">AJL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frost</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phylodynamics of infectious disease epidemics</span>. <span class="source" tagx="source" title="source">Genetics</span>. <span class="year" tagx="year" title="year">2009</span>;. <span class="pub-id"><a href="https://dx.doi.org/10.1534/genetics.109.106021">10.1534/genetics.109.106021</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19797047">19797047</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koelle</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasmussen</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Rates of coalescence for common epidemiological models at equilibrium</span>. <span class="source" tagx="source" title="source">Journal of the Royal Society Interface</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">70</span>):<span class="fpage" tagx="fpage" title="fpage">997</span>â€"<span class="lpage" tagx="lpage" title="lpage">1007</span>.</span></li> <li tag="ref"><a name="pcbi.1008122.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grassly</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraser</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="mixed-article-title" title="mixed-article-title">Quantifying transmission heterogeneity using both pathogen phylogenies and incidence time series</span>. <span class="source" tagx="source" title="source">Molecular biology and evolution</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">34</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">2982</span>â€"<span class="lpage" tagx="lpage" title="lpage">2995</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msx195">10.1093/molbev/msx195</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28981709">28981709</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eshleman</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">XC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ou</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piwowar-Manning</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eron</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Virologic outcomes in early antiretroviral treatment: HPTN 052</span>. <span class="source" tagx="source" title="source">HIV clinical trials</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">100</span>â€"<span class="lpage" tagx="lpage" title="lpage">109</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/15284336.2017.1311056">10.1080/15284336.2017.1311056</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28385131">28385131</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero-Severson</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skar</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bulla</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albert</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leitner</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Timing and order of transmission events is not directly reflected in a pathogen phylogeny</span>. <span class="source" tagx="source" title="source">Molecular biology and evolution</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">31</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">2472</span>â€"<span class="lpage" tagx="lpage" title="lpage">2482</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msu179">10.1093/molbev/msu179</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24874208">24874208</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouyos</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Wyl</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yerly</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BÃ¶ni</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BÃ¼rgisser</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Estimating the basic reproductive number from viral sequence data</span>. <span class="source" tagx="source" title="source">Molecular biology and evolution</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">347</span>â€"<span class="lpage" tagx="lpage" title="lpage">357</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msr217">10.1093/molbev/msr217</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21890480">21890480</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leventhal</span><span class="given-names" tagx="given-names" title="given-names">GE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ¼nthard</span><span class="given-names" tagx="given-names" title="given-names">HF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonhoeffer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Using an epidemiological model for phylogenetic inference reveals density dependence in HIV transmission</span>. <span class="source" tagx="source" title="source">Molecular biology and evolution</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">31</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">6</span>â€"<span class="lpage" tagx="lpage" title="lpage">17</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24085839">24085839</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eames</span><span class="given-names" tagx="given-names" title="given-names">KTD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Contact tracing strategies in heterogeneous populations</span>. <span class="source" tagx="source" title="source">Epidemiology and Infection</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">135</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">443</span>â€"<span class="lpage" tagx="lpage" title="lpage">454</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1017/S0950268806006923">10.1017/S0950268806006923</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16848925">16848925</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero-Severson</span><span class="given-names" tagx="given-names" title="given-names">EO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee Petrie</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ionides</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albert</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leitner</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Trends of HIV-1 incidence with credible intervals in Sweden 2002â€"09 reconstructed using a dynamic model of within-patient IgG growth</span>. <span class="source" tagx="source" title="source">International journal of epidemiology</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">44</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">998</span>â€"<span class="lpage" tagx="lpage" title="lpage">1006</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/ije/dyv034">10.1093/ije/dyv034</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26163684">26163684</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sommen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Commenges</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vu</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alioum</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Estimation of the distribution of infection times using longitudinal serological markers of HIV: implications for the estimation of HIV incidence</span>. <span class="source" tagx="source" title="source">Biometrics</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">67</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">467</span>â€"<span class="lpage" tagx="lpage" title="lpage">475</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1541-0420.2010.01473.x">10.1111/j.1541-0420.2010.01473.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20731647">20731647</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kliche</span><span class="given-names" tagx="given-names" title="given-names">DV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Maximum likelihood estimation of gamma parameters for coarsely binned and truncated raindrop size data</span>. <span class="source" tagx="source" title="source">Quarterly Journal of the Royal Meteorological Society</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">140</span>(<span class="issue" tagx="issue" title="issue">681</span>):<span class="fpage" tagx="fpage" title="fpage">1245</span>â€"<span class="lpage" tagx="lpage" title="lpage">1256</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/qj.2209">10.1002/qj.2209</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">HOW CAN WE IMPROVE ACCURACY OF MACROEVOLUTIONARY RATE ESTIMATES</span>. <span class="source" tagx="source" title="source">Systematic Biology</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">62</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">321</span>â€"<span class="lpage" tagx="lpage" title="lpage">329</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/sys073">10.1093/sysbio/sys073</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22962038">22962038</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dwyer-Lindgren</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cork</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sligar</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steuben</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">KF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Provost</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Mapping HIV prevalence in sub-Saharan Africa between 2000 and 2017</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">570</span>(<span class="issue" tagx="issue" title="issue">7760</span>):<span class="fpage" tagx="fpage" title="fpage">189</span>â€"<span class="lpage" tagx="lpage" title="lpage">193</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-019-1200-9">10.1038/s41586-019-1200-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31092927">31092927</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quinn</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wawer</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sewankambo</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serwadda</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wabwire-Mangen</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">VIRAL LOAD AND HETEROSEXUAL TRANSMISSION OF HUMAN IMMUNODEFICIENCY VIRUS TYPE 1</span>. <span class="source" tagx="source" title="source">The New England Journal of Medicine</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">342</span>(<span class="issue" tagx="issue" title="issue">13</span>):<span class="fpage" tagx="fpage" title="fpage">921</span>â€"<span class="lpage" tagx="lpage" title="lpage">929</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJM200003303421303">10.1056/NEJM200003303421303</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10738050">10738050</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wawer</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sewankambo</span><span class="given-names" tagx="given-names" title="given-names">NK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serwadda</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laeyendecker</span><span class="given-names" tagx="given-names" title="given-names">OB</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Rates of HIV-1 Transmission per Coital Act, by Stage of HIV-1 Infection, in Rakai, Uganda</span>. <span class="source" tagx="source" title="source">The Journal of Infectious Diseases</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">191</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">1403</span>â€"<span class="lpage" tagx="lpage" title="lpage">1409</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1086/429411">10.1086/429411</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15809897">15809897</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cohen</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaw</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMichael</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haynes</span><span class="given-names" tagx="given-names" title="given-names">BF</span></span>. <span class="mixed-article-title" title="mixed-article-title">Acute HIV-1 Infection</span>. <span class="source" tagx="source" title="source">The New England Journal of Medicine</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">364</span>(<span class="issue" tagx="issue" title="issue">20</span>):<span class="fpage" tagx="fpage" title="fpage">1943</span>â€"<span class="lpage" tagx="lpage" title="lpage">1954</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMra1011874">10.1056/NEJMra1011874</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21591946">21591946</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero-Severson</span><span class="given-names" tagx="given-names" title="given-names">EO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bulla</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leitner</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phylogenetically resolving epidemiologic linkage</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span>. <span class="year" tagx="year" title="year">2016</span>; p. <span class="fpage" tagx="fpage" title="fpage">201522930</span>.</span></li> <li tag="ref"><a name="pcbi.1008122.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leitner</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albert</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">The molecular clock of HIV-1 unveiled through analysis of a known transmission history</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span>. <span class="year" tagx="year" title="year">1999</span>;<span class="volume" tagx="volume" title="volume">96</span>(<span class="issue" tagx="issue" title="issue">19</span>):<span class="fpage" tagx="fpage" title="fpage">10752</span>â€"<span class="lpage" tagx="lpage" title="lpage">10757</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.96.19.10752">10.1073/pnas.96.19.10752</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">AFY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joy</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woods</span><span class="given-names" tagx="given-names" title="given-names">CK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shurgold</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colley</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brumme</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The Impact of Clinical, Demographic and Risk Factors on Rates of HIV Transmission: A Population-based Phylogenetic Analysis in British Columbia, Canada</span>. <span class="source" tagx="source" title="source">The Journal of Infectious Diseases</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">211</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">926</span>â€"<span class="lpage" tagx="lpage" title="lpage">935</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiu560">10.1093/infdis/jiu560</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25312037">25312037</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">EsbjÃ¶rnsson</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mild</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Audelin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fonager</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skar</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bruun JÃ¸rgensen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">HIV-1 transmission between MSM and heterosexuals, and increasing proportions of circulating recombinant forms in the Nordic Countries</span>. <span class="source" tagx="source" title="source">Virus evolution</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">2</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">vew010</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/ve/vew010">10.1093/ve/vew010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774303">27774303</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skar</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sylvan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansson</span><span class="given-names" tagx="given-names" title="given-names">HB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gustavsson</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boman</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albert</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Multiple HIV-1 introductions into the Swedish intravenous drug user population</span>. <span class="source" tagx="source" title="source">Infection, Genetics and Evolution</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">545</span>â€"<span class="lpage" tagx="lpage" title="lpage">552</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2008.03.004">10.1016/j.meegid.2008.03.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18472306">18472306</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skar</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Axelsson</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berggren</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thalme</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gyllensten</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liitsola</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Dynamics of two separate but linked HIV-1 CRF01_AE outbreaks among injection drug users in Stockholm, Sweden, and Helsinki, Finland</span>. <span class="source" tagx="source" title="source">Journal of virology</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">85</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">510</span>â€"<span class="lpage" tagx="lpage" title="lpage">518</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01413-10">10.1128/JVI.01413-10</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20962100">20962100</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">jie Xie</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Bayesian Phylogenetics with BEAUti and the BEAST 1.7</span>. <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">1969</span>â€"<span class="lpage" tagx="lpage" title="lpage">1973</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mss075">10.1093/molbev/mss075</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22367748">22367748</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klinkenberg</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraser</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heesterbeek</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>. <span class="mixed-article-title" title="mixed-article-title">The effectiveness of contact tracing in emerging epidemics</span>. <span class="source" tagx="source" title="source">PloS one</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">1</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">e12</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0000012">10.1371/journal.pone.0000012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17183638">17183638</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Didelot</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gardy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colijn</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="mixed-article-title" title="mixed-article-title">Bayesian Inference of Infectious Disease Transmission from Whole-Genome Sequence Data</span>. <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">31</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">1869</span>â€"<span class="lpage" tagx="lpage" title="lpage">1879</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msu121">10.1093/molbev/msu121</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24714079">24714079</a></span></span></li> <li tag="ref"><a name="pcbi.1008122.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volz</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero-Severson</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leitner</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phylodynamic Inference across Epidemic Scales</span>. <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">34</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">1276</span>â€"<span class="lpage" tagx="lpage" title="lpage">1288</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msx077">10.1093/molbev/msx077</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28204593">28204593</a></span></span></li> </ul> </div> </div>  </body></html>