<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Biol Proced Online</span><span class="iso-abbrev" title="iso-abbrev">Biol Proced Online</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Biological Procedures Online</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1480-9222</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7462115">7462115</a></span><span class="publisher-id" title="publisher-id">134</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12575-020-00134-4">10.1186/s12575-020-00134-4</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Review</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Potential Diagnostic Systems for Coronavirus Detection: a Critical Review</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ekrami</span><span class="given-names" tagx="given-names" title="given-names">Elena</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pouresmaieli</span><span class="given-names" tagx="given-names" title="given-names">Mahdi</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barati</span><span class="given-names" tagx="given-names" title="given-names">Fatemeh</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asghari</span><span class="given-names" tagx="given-names" title="given-names">Sahar</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ziarani</span><span class="given-names" tagx="given-names" title="given-names">Farzad Ramezani</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shariati</span><span class="given-names" tagx="given-names" title="given-names">Parvin</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mamoudifard</span><span class="given-names" tagx="given-names" title="given-names">Matin</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">matinmahmodifard@yahoo.com</span><span class="email" tagx="email" title="email">m_mahmodifard@nigeb.ac.ir</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.419420.a</span><span class="institution-id" title="institution-id">0000 0000 8676 7464</span><span class="institution" title="institution">Department of Industrial and Environmental Biotechnology, </span><span class="institution" title="institution">National Institute of Genetic Engineering and Biotechnology (NIGEB), </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.411747.0</span><span class="institution-id" title="institution-id">0000 0004 0418 0096</span><span class="institution" title="institution">Department of Microbiology, School of Medicine, </span><span class="institution" title="institution">Golestan University of Medical Sciences, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-9-9</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">22</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">21</span><span class="history" title="history"><span class="received" title="received">received: 2020-6-22</span><span class="accepted" title="accepted">accepted: 2020-8-23</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="abstract" title="sec"> <div class="title" tagx="title" title="title">
Abstract</div> <p id="Par1">Currently there are no effective anti-viral drugs for SARS-CoV-2, so the primary line of defense is to detect infected cases as soon as possible. The high rate of contagion for this virus and the highly nonspecific symptoms of the disease (Coronovirus disease 2019, (Covid-19)) that it causes, such as respiratory symptoms, cough, dyspnea, fever, and viral pneumonia, require the urgent establishment of precise and fast diagnostic tests to verify suspected cases, screen patients, and conduct virus surveillance. Nowadays, several virus detection methods are available for viral diseases, which act on specific properties of each virus or virus family, therefore, further investigations and trials are needed to find a highly efficient and accurate detection method to detect and prevent the outcomes of the disease. Hence, there is an urgent need for more and precise studies in this field. In this review, we discussed the properties of a new generation of coronaviruses (SARS-CoV-2) following routine virus detection methods and proposed new strategies and the use of potential samples for SARS-CoV-2 detection.</p> </div> <div class="graphicalabstract" title="sec"> <div class="title" tagx="title" title="title">
Graphical Abstract</div> <p id="Par2"> </p><div class="graphic" position="anchor" id="MO1" /> <p /> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
SARS-CoV-2</div> <div class="kwd" title="kwd">
Detection</div> <div class="kwd" title="kwd">
Biosensor</div> <div class="kwd" title="kwd">
RT-PCR</div> <div class="kwd" title="kwd">
Immunoassay</div> <div class="kwd" title="kwd">
Nano-materials</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par34">On 30th January 2020 the WHO declared a global â€œpublic health emergency of international concernâ€� (PHEIC) regarding the epidemic caused by the 2019 novel coronavirus (2019-nCoV), which started in a Wuhan seafood market, Hubei province [<a href="#CR1">1</a>]. This virus is genetically very similar to the bat SARS-CoV in the subgenus <i>Sarbecovirus</i> [<a href="#CR2">2</a>]. The WHO has currently named the disease, which has spread so rapidly throughout the world, as coronavirus disease 2019 (COVID-19) [<a href="#CR3">3</a>]. The 2019 novel coronavirus (2019-nCoV) has also been renamed as severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Coronaviruses got their name from their morphology when observed under a microscope. The virus consists of a core of genetic material surrounded by an envelope with protein spikes. This gives it the appearance of a crown. The word Corona means â€œcrownâ€� in Latin. Coronaviruses are zoonotic [<a href="#CR4">4</a>] meaning that the viruses are transmitted between animals and humans. Coronaviruses can be classified into four genera (Î±, Î², Î³, and Î´), and these viruses are detected in a very wide selection of animal species, including humans [<a href="#CR5">5</a>]. The virus that causes the COVID-19 disease belongs to the Î²-coronavirus genus. Since the beginning of the twenty-first century, three kinds of coronaviruses have crossed the species barrier, causing deadly pneumonia in humans. These include the severe acute respiratory syndrome SARS-CoV [<a href="#CR6">6</a>], Middle-East respiratory syndrome (MERS-CoV) [<a href="#CR7">7</a>], and SARS-CoV-2 [<a href="#CR8">8</a>] coronaviruses. Unlike previous coronaviruses that caused large-scale epidemics such as the Middle East Respiratory Syndrome (MERS) and Severe Acute Respiratory Syndrome (SARS), the transmission rate for SARS-CoV-2 is much higher, with an average of two to three people becoming infected for every already infected person [<a href="#CR9">9</a>]. SARS-CoV-2 is mainly transmitted to healthy persons by exposure to the respiratory droplets of one infected person through close interactions. After entering the body, this virus is first observed in the nasal mucosa and mouth, and is then transferred to the lungs of the exposed individuals. People infected with SARS-CoV2 will experience weak symptoms, but in susceptible persons with certain underlying diseases like diabetes, hypertension, heart and autoimmune diseases, severe respiratory and lethal sickness or damage to some other vital organs such as the gastrointestinal tract, liver, kidney or the central nervous systems will take place [<a href="#CR10">10</a>].</p> <p id="Par35">After entrance to the body, the virus finds its target cells through its receptor, namely angiotensin-converting enzyme 2 (ACE2). ACE2 is mainly expressed on the epithelial cells of healthy lung tissue. But, it is reported that other types of cells may also present ACE2 thus making them vulnerable to the virus [<a href="#CR11">11</a>]. Besides, it has been also reported that SARS-CoV-2 is indirectly harmful to immune cells, mostly T cells, and macrophages, eliciting their destruction. In this regard, there are some COVID-19 immunopathogenesis factors including granulocyte colony-stimulating factor (G-CSF), tumor necrosis factor-Î± (TNF-Î±), macrophage inflammatory protein, (MIP-1A), monocyte chemoattractant protein-1 (MCP-1), interferon gamma-induced protein-10 (IP-10), interleukin-1 (IL-1), IL-2, IL-6, IL-7, and IL-10 that can be used as detection markers in COVID-19 patients [<a href="#CR12">12</a>].</p> <p id="Par36">Because of the high rate of contagion and the highly nonspecific symptoms of the COVID-19 disease, such as respiratory symptoms, cough, dyspnea, fever and viral pneumonia [<a href="#CR8">8</a>], the establishment of precise, fast and inexpensive diagnostic tests methods that can lower the risk of spreading infection, alleviate the strain on the healthcare system, and mitigate the cost of care for both individuals and the government, can improve the health care system to combat this and similar diseases. Generally, for the detection of viral diseases like COVID-19, different kinds of diagnostic strategies will be purposed including: inspection of physical symptoms, clinical evidence, radiological images and para-clinical findings. Para-clinical detection methods are also categorized as nucleic acid amplification tests (NAATs), viral sequencing, serological assays, and viral culture and real-time polymerase chain reaction (RT-PCR) assays.</p> <p id="Par37">The main goal of the current review is to provide a comprehensive overview on the newly developed methods and also potential techniques for the detection of viral diseases especially corona ones to help researchers around the world assess the advantages and disadvantages of each method and select the right one to efficiently control such diseases. Before starting, a brief overview of viruses and their types and properties is presented.</p> </div> <div class="virusstructureandfunction" title="sec"> <div class="title" tagx="title" title="title">
Virus Structure and Function</div> <p id="Par38">Viruses are small infectious agents and obligatory intracellular parasites that use the host cellâ€™s biosynthetic and metabolic machinery to survive and replicate their genomes, and in doing so, they cause damage and subsequent destruction of host cells, which ultimately leads to different types of diseases in humans.</p> <p id="Par39">Viruses have different types of structures that include genomes comprised of single or double-stranded DNA or RNA. The genome is encircled by a protective cover made of a protein known as the capsid, which is encoded by the viral genome. Such proteins that are structurally associated with the nucleic acid are referred to as nucleoproteins [<a href="#CR13">13</a>]. Viruses infect all types of life forms [<a href="#CR14">14</a>], and can use viral digestive enzymes to dissolve the host cell membrane, thereby gaining entry into the cell. However, they do not contain various enzymatic systems apart from digestive enzymes which are used to enter the cell. So, because of their structure and their enzyme system as mentioned above, antibiotics are not effective for viral infections [<a href="#CR15">15</a>]. There are different kinds of viruses that can infect humans, and until now only 219 viral species have been detected. The first type of virus that was discovered in 1901 was the yellow fever virus. Every year new types are found, showing a rise in the number of known viruses that cause various diseases [<a href="#CR16">16</a>]. An example of this is the current novel infectious coronavirus, SARS-CoV-2, which has quickly become widespread throughout the world.</p> <div class="thehistoryofthesars-cov-2virus" title="sec"> <div class="title" tagx="title" title="title">
The History of the SARS-CoV-2 Virus</div> <p id="Par40">The <i>Nidovirales</i> order, includes <i>Coronaviridae, Roniviridae,</i> families and Coronaviruses (CoVs) are the greatest groups of viruses that relate to this order and the <i>Coronaviridae</i> family [<a href="#CR17">17</a>]. Coronaviruses have a single-stranded RNA genome, 26 to 32 kilobases in length [<a href="#CR18">18</a>]. They usually have a wide range of hosts to infect, such as amphibians and mammals, but their main hosts for infection are avian [<a href="#CR19">19</a>]. Mammalian hosts include mice, bats, masked palm civets, camels, cats, and dogs [<a href="#CR20">20</a>], but lately, the number of new mammalian hosts infected by coronaviruses is growing at an alarming rate [<a href="#CR18">18</a>]. For example, fatal severe diarrhea syndrome in pigs, associated with an HKU2-related coronavirus of bat origin was reported in 2018 [<a href="#CR21">21</a>]. There are different coronaviruses, but the majority of them are associated with mild clinical symptoms [<a href="#CR18">18</a>]. SARS-CoV-2 is a virus belonging to the <i>Coronaviridae</i> family, which causes infection in humans; SARS-CoV, MERS-CoV, and SARS-CoV-2 can cause acute disease, while NL63, HKU1, OC43, and 229E cause mild symptoms [<a href="#CR22">22</a>]. The first time that severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV) appeared, was during November 2002 in Guangdong, southern China, and the first time the Middle East respiratory syndrome (MERS) coronavirus (MERS-CoV) emerged, was in 2012 in Saudi Arabia [<a href="#CR7">7</a>, <a href="#CR23">23</a>, <a href="#CR24">24</a>]. During 2002â€"03, more than 8000 infected cases by SARS-CoV with more than 774 deaths in 37 countries were confirmed. The MERS-CoV outbreak in September 2012, caused 2494 confirmed cases of infection and 858 mortalities, including 38 fatalities in South Korea [<a href="#CR25">25</a>, <a href="#CR26">26</a>]. In late December 2019, many infected patients with viral pneumonia were reported, but notably, all of them were associated with the Huanan seafood wholesale market [<a href="#CR27">27</a>]. Next-generation sequencing confirmed the presence of a new virus which infected humans, and was provisionally named the 2019 novel coronavirus (2019-nCoV).</p> <p id="Par41">From Mar 21, 2020, China has reported 80,967 confirmed cases across the country, with 3248 fatalities [<a href="#CR28">28</a>]. Infections in hospital workers and doctors and families were also revealed, thus indicating that human-to-human transmission of the new virus was possible [<a href="#CR29">29</a>]. Further observation showed that this virus outbreak happens quickly by human-to-human transmission [<a href="#CR30">30</a>]. It started from Wuhan and then spread to other areas, causing more than 44,000 cases of COVID-19 in China until Feb 12, 2020, based on statistical data. Besides, outside China including Hong Kong, Macao, and Taiwan, more than 400 cases were reported in Thailand, Japan, Turkey, Italy, Singapore, France, Australia, and Canada [<a href="#CR31">31</a>]. The number of reported cases was growing appreciably. Clinical manifestations related to this virus infection ranged from no symptom to fatal pneumonia [<a href="#CR32">32</a>]. Most of the patients had a high fever and some had dyspnea, with chest radiographs revealing incursive lesions in the lungs [<a href="#CR8">8</a>].</p> <p id="Par42">Reported cases with SARS-CoV-2 are now more prevalent, and as of 10 August 2020, 19.9â€‰M cases have been reported in 203 countries, and the number of death stands at 49,242 in the world [<a href="#CR28">28</a>] at 14:57 GMT. At present, available data shows that SARS-CoV-2 which infected the human population originated from a bat source, at present, available data shows that SARS-CoV-2 which infected the human population originated from a bat source, Pangolins have been cited as possible intermediate between humans and bats.</p> <p id="Par43">Full-genome sequencing of 2019-nCoV shows that this virus is genetically similar to the bat-SL-CoVZC45 (sequence identity 87ãƒ»99%; query coverage 99%) and SARS-like betacoronavirus of bat origin, bat-SL-CoVZXC21 (accession number MG772934;23 87ãƒ»23%; query coverage 98%) <b>(</b>Fig.Â <a href="#Fig1">1</a><b>).</b></p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Phylogenetic analysis of full-length genomes of Sars-CoV-2 and representative viruses of the genus beta-coronaviruses (reprint with permission from ref. [<a href="#CR33">33</a>]</p>   </div> <p /> <p id="Par44">Genetic-wise, the SARS-CoV-2 strains are less similar to SARS-CoV and MERS-CoV (about 79 and 52%, respectively). Sequencing of samples from China especially Wuhan indicated that this virus comes from the subgenus <i>Sarbecovirus</i>. Also, it shows that it is more analogous to the bat-SL-CoVZC45 and bat-SL-CoVZXC21 viruses, both of which are bat-derived coronavirus strains, than to other recognized human-infecting coronaviruses, which caused the SARS outbreak of 2003 [<a href="#CR34">34</a>]. Coronaviruses have an evolutionary rate of 10<sup>âˆ’â€‰4</sup> nucleotide substitutions per site per year [<a href="#CR18">18</a>]. Their genetic sequence endures mutations at every replication cycle. The facts and information available so far indicate that SARS-CoV-2 emerged from one source instantaneously. However, as the virus is transmitted to more individuals, arising mutations should be investigated. All the evidence indicates that there is a relationship between bats and the novel coronaviruses, but other observations also show that another animal may act as an intermediate.</p> <p id="Par45">First, primary reported cases of coronavirus were in late December 2019, but at this time most of the bats in Wuhan were in hibernation. Second, as mentioned before, the first reported cases were from the Huanan seafood market, but facts show that there are not any bats at this market. However, other non-aquatic animals (like mammals) were accessible for sale. Third, genetic-wise, the detected sequence of 2019-nCoV has less than 90% similarity to the bat-SL-CoVZXC21 and bat-SL-CoVZC45 viruses. This data indicates that 2019-nCoVâ€™s direct progenitor is not bat-SL-CoVZC45 and bat-SL-CoVZXC21 [<a href="#CR35">35</a>]. Fourth, other animals such as the masked palm civet for SARS-CoV and dromedary camels for MERS-CoV were an intermediate host for both SARS-CoV and MERS-CoV, and in both SARS-CoV and MERS-CoV, bats were acting as natural source and humans were final hosts [<a href="#CR36">36</a>, <a href="#CR37">37</a>] So, with these facts and information, it shows that the SARS-CoV-2 virus causing the Wuhan outbreak might have also started by bats, and it might have been transmitted to humans through another unfamiliar animal(s) which was accessible and available for sale at the Huanan seafood wholesale market.</p> </div> </div> <div class="methodsforvirusdetection" title="sec"> <div class="title" tagx="title" title="title">
Methods for Virus Detection</div> <p id="Par46">There are many techniques for the detection of viruses, the functions of which depend on virus type and virus particle properties. Herein, we review the most important types.</p> <div class="immunofluorescencemethods" title="sec"> <div class="title" tagx="title" title="title">
Immunofluorescence Methods</div> <p id="Par47">Immunofluorescence (IF) staining including the â€œindirectâ€� fluorescent-antibody assay (IFA) are classified as fast methods for the detection of viral antigens, which typically take at least 2â€‰h or more to complete. These techniques have ideal specificity and sensitivity [<a href="#CR38">38</a>].</p> <p id="Par48">In other words, they are based on enzyme immunoassays (EIAs) techniques like the enzyme-linked immunosorbent assay (ELISA) and make use of similar protocols and materials, such as monoclonal antibodies. Therefore, ELISA has become the most useful assay for simple and sensitive virus testing due to its speed, ease of operation, and easy interpretation of results [<a href="#CR39">39</a>].</p> <p id="Par49">Despite the many benefits of this method, there are also disadvantages, which limit its use to detect all viruses. The most important limitation of ELISA is the time-consuming antisera production, which must be conducted in specialized laboratories. The main differences between ELISA and nucleic acid-based detection methods are accessibility and low costs [<a href="#CR40">40</a>].</p> <p id="Par50">On the other hand, the ELISA method can detect viral antigens and proteins of the recombinant virus using antisera. Generally, this strategy is efficient and sensitive, but in many situations, for example, where identification of specific viral species/strains is required, ELISA is often not appropriate [<a href="#CR41">41</a>].</p> <p id="Par51">Therefore, ELISA does not have sufficient flexibility and compatibility, which is an intrinsic property in some molecular methods.</p> </div> <div class="nucleicacidamplificationtests(naats)" title="sec"> <div class="title" tagx="title" title="title">
Nucleic Acid Amplification Tests (NAATs)</div> <p id="Par52">Since the early twentieth century, the detection of viruses in clinical samples via molecular methods has become widespread. Diagnosis of viruses can be done directly using clinical samples and cell culture supernatants via specific nucleic acid probes, which can find their complementary target viral RNA or DNA sequences or by using NAATs [<a href="#CR42">42</a>]. Also, purification and separation of nucleic acids from antibodies is easy, which is not always the case for viral antigens, whose detection may be inhibited by antigen/antibody complexes [<a href="#CR43">43</a>].</p> <p id="Par53">While methods based on nucleic acids are functional for most viroids and some viruses, the technique that has been most successfully exploited is based on the Polymerase Chain Reaction (PCR).</p> <div class="polymerasechainreaction(pcr)" title="sec"> <div class="title" tagx="title" title="title">
Polymerase Chain Reaction (PCR)</div> <p id="Par54">The diagnosis of viruses through PCR methods was established in the early 1990s [<a href="#CR44">44</a>]. In this technique, many copies of one DNA template are made using primers, enzymes, and variable temperatures [<a href="#CR45">45</a>]. Unfortunately, the small amounts of DNA liberated into the laboratory environment through the opening of the tubes after thermal cycling could eventually be detected by the PCR method, resulting in reporting false-positive results. To solve these problems, closed-tube and homogeneous PCR assays-commonly known as real-time PCR or quantitative PCR (q-PCR) are used [<a href="#CR46">46</a>]. This technique is quickly performed for diagnostic applications due to the generated fluorescent signal, which could be detected at a time (â€˜real-timeâ€™) or the end of the process (â€˜endpointâ€™) without the contamination risk. Initially, Real-time PCR for virus detection was developed to increase sensitivity [<a href="#CR33">33</a>], thus playing a critical role where antibody methods could not be used [<a href="#CR47">47</a>].</p> <p id="Par55">Real-time PCR methods have been established for three important targets. 1. Real-time PCR is quicker than ELISA to establish, considering the development of antibodies for new viruses, 2. This method is more functional, especially in routine laboratories, 3. Real-time PCR is more cost-effective due as compared to the expensive antibody production for immunosorbent assays. Thus, it seems that real-time PCR enables high-throughput testing at a relatively low per-sample cost and great speed to set up a new assay [<a href="#CR48">48</a>].</p> </div> <div class="isothermalamplification" title="sec"> <div class="title" tagx="title" title="title">
Isothermal Amplification</div> <p id="Par56">The main theory of isothermal amplification of DNA is separating two strands of the template through non-thermal ways, such as helicase dependent amplification (HDA) [<a href="#CR49">49</a>] and Recombinase Polymerase Amplification (RPA) [<a href="#CR50">50</a>].</p> <p id="Par57">Helicase dependent amplification (HDA) uses a helicase to separate the strands of double-stranded DNA leading to primer annealing and extension of the supplementary strand via DNA polymerase enzyme at 65â€‰Â°C. Reaction times for HDA are generally in the range 30â€"90â€‰min. The function of this method relates to the short products of nearly 70â€"120â€‰bp [<a href="#CR49">49</a>].</p> <p id="Par58">Recombinase Polymerase Amplification (RPA) uses a recombinase to produce a complex with primers resulting in extension without thermal denaturation. The reaction is carried at a low reaction temperature (between 37 and 42â€‰Â°C), which can easily be provided via a low power source. However, the production of non-specific amplification pieces is common due to the low reaction temperature. Nevertheless, RPA has a short reaction time, which is typically <div class="â€‰30â€‰min_UNKNOWN" href="#CR50">
50].</div></p> <p id="Par59">An alternative isothermal amplification approach is to design primers such that the extension products contain single-stranded primer binding sites [<a href="#CR51">51</a>]. Accordingly, loop-mediated isothermal amplification (LAMP) is the most ordinarily used method, using three pairs of primers (internal, external and loop primers), to get an amplification product which has single-stranded loop regions to which primers can bind without template denaturation [<a href="#CR52">52</a>] at a reaction temperature of around 65â€‰Â°C. The supplementary sequences of internal primers can stick to themselves to produce loop structures, while the extension of the external primers leads to the transposition of the extension products of the internal primers. The products of LAMP reactions are interchangeably oriented repeats of the target sequence. The addition of loop primers increases amplification due to the priming role of loops. These regions can raise sensitivity and reduce reaction times, which are desired outcomes.</p> <p id="Par60">Loop-mediated isothermal amplification (LAMP) can be modified to detect RNA targets by the addition of reverse transcriptase enzyme to the reaction. Besides, in RT-LAMP, reverse transcription and amplification of cDNA continue temporarily at a single temperature of around 65â€‰Â°C.</p> <p id="Par61">Now, there are detection kits produced via this approach for a wide range of pathogens including bacteria and viruses. The mix of LAMP and innovative microfluidic equipment including Lab-on-a-chip can lead to design genetic checkpoint testing systems [<a href="#CR51">51</a>, <a href="#CR53">53</a>].</p> <p id="Par62">As an example, this technique has been used for the diagnosis of the Zika virus with high sensitivity. This cost-effectiveness system can be freeze-dried for distribution and replace reverse transcription-quantitative polymerase chain reaction (RT-qPCR) [<a href="#CR54">54</a>].</p> </div> <div class="next-generationsequencing(ngs)" title="sec"> <div class="title" tagx="title" title="title">
Next-Generation Sequencing (NGS)</div> <p id="Par63">Next-generation sequencing (NGS) has contributed to the field of virology through the identification of new viruses via popular platforms such as pyrosequencing [<a href="#CR55">55</a>].</p> <p id="Par64">NCBI BLAST and PLAN are the most popular Blast tools but they have a limitation, and that is the acceptance of a limited number of sequences in the flat fasta format. On the other hand, Galaxy is more flexible with NGS data but not for novel virus discovery purposes. In general, map NGS reads the human genome and eliminates host reads after the Blast steps. The remaining data are analyzed to classify into non-human, microbial, or viral integrated sequences. This approach can make a difference in costs, ease of sequence assembly, and identification, the key distinctions lie in the nucleic acid purification techniques employed [<a href="#CR56">56</a>]. In one study, Czotter et al. [<a href="#CR57">57</a>] used virus-derived small RNAs. Through their deep sequencing, they can determine the viromes of vineyards in Hungary. NGS of these RNAs, enabled them to diagnose routine viruses and new ones, which had never been described in Hungary before.</p> </div> </div> <div class="electronmicroscopy(em)" title="sec"> <div class="title" tagx="title" title="title">
Electron Microscopy (EM)</div> <p id="Par65">Electron microscopy (EM) is one of the primary virus detection methods, especially in unsuspected and unknown agents. This tool can capture live images of cells and tissues with high-resolution. In most cases, the observed morphology can lead to immediate detection at the family level based on particle size, shape, and stability. Human viruses include two major morphological categories, naked or enveloped. Naked viruses are icosahedral; with rigid capsids that can endure drying processes and keep their spherical structure in negative stains. Enveloped viruses have an extra outer cover that is usually derived from host membranes. These viruses (except for poxviruses) may show various shapes [<a href="#CR58">58</a>].</p> <p id="Par66">Transmission electron microscopy (TEM) is a good primary step in virus diagnosis, which targets proteins within the virus structure and is harmless to RNA or DNA genomes. Immuno-electron microscopy (IEM) is also based on the same serological principles as ELISA and can be a useful approach for more virus identification [<a href="#CR59">59</a>].</p> </div> <div class="cellculture" title="sec"> <div class="title" tagx="title" title="title">
Cell Culture</div> <p id="Par67">The use of human cells instead of embryonated eggs and laboratory animals, for in vitro isolation of viruses was first carried out in the 1900s. It has been improved since the 1970s due to the availability of highly purified reagents and commercially-prepared cell lines [<a href="#CR60">60</a>]. The progress in virus isolation using cultured cells has occurred since the addition of antibiotics to the cell culture media, development of chemically defined culture media, and the use of cell-dispensing equipment for preparing replicate cultures [<a href="#CR61">61</a>].</p> <p id="Par68">Cell cultures provide large numbers of cells as hosts for the virus and help decrease the use of experimental animals, and hence a lower risk of contamination. Therefore, viruses reach high titers in the cells which are accessible for microscopic examinations. Also, cell cultures are less expensive than eggs and animals [<a href="#CR42">42</a>].</p> </div> <div class="nanoelectromechanicaldevices" title="sec"> <div class="title" tagx="title" title="title">
Nanoelectromechanical Devices</div> <p id="Par69">High-frequency nanoelectromechanical systems (NEMS) [<a href="#CR62">62</a>] are being considered as new sensors and devices. It has been proved that the selective molecular binding to the surface of nanomechanical oscillators may lead to detecting pathogen viral binding through observing their effects on the natural frequency shift of NEMS devices [<a href="#CR63">63</a>].</p> <p id="Par70">This method for virus detection is still in its infancy and needs further development to be more efficient. Additional information will be mentioned in the next chapter.</p> </div> <div class="useofelectrochemicalandnanobiosensorstodetectvirus-infections" title="sec"> <div class="title" tagx="title" title="title">
Use of Electrochemical and Nanobiosensors to Detect Virus-Infections</div> <div class="electrochemicalstudies" title="sec"> <div class="title" tagx="title" title="title">
Electrochemical Studies</div> <p id="Par71">In typical methods for diagnosing viruses, instruments and staff are needed, also, the detection of viruses is time-consuming.</p> <p id="Par72">Applying nanoparticles (NPs) in combination with electrochemical detection is promising in detecting viruses. A biosensor is an analytical tool applied for the detection of analytes that combine a biological component with a physicochemical detector [<a href="#CR64">64</a>].</p> <p id="Par73">Electrochemical nanosensors have advantages that include being quick, precise, selective, practical, and economical when compared to traditional techniques. The NP-based biosensor is better for detecting pathogenic microorganisms in clinical samples because they are user-friendly and have high specificity, and low costs [<a href="#CR65">65</a>â€"<a href="#CR67">67</a>].</p> </div> <div class="nanobiosensor" title="sec"> <div class="title" tagx="title" title="title">
NanoBiosensor</div> <p id="Par74">Analyses of biosensor-based researches have been very significant in the last three decades, due to their advantages that include low costs, quick reactions and responses, easy to use, and being user- friendly [<a href="#CR68">68</a>]. These devices detect the existence or concentration of a biological analyte, like a biological system or a microorganism, or a biomolecule [<a href="#CR69">69</a>]. Typically biosensors consist of three components: a part which recognizes the analyte or biological identification part, a signal transducer, and an amplifier or a part which is known as the reader device [<a href="#CR70">70</a>]. A new achievement in biosensor technology is the immobilization of biologically sensitive substances on the surface of a biosensor. Bioreceptor bases are commonly considered as biomarkers, nucleic acids, enzymes, microorganisms, tissues, viruses, bacteria, and antigens. Electrochemical, various field-effect transistor-based methods, and optical methods are defined as the most popular traditional types of biosensors [<a href="#CR71">71</a>].</p> <p id="Par75">Nanomaterials (NMs) developed new possibilities for the expansion of electrochemical biosensors [<a href="#CR72">72</a>, <a href="#CR73">73</a>]. Nanomaterials (NMs) in combination with new advances in biosensor structures help to provide developed novel electrochemical assays. These materials have great advantages because of promoting electron transfer reactions, electrical conductivity, great surface area, mechanical robustness, and good chemical stability [<a href="#CR74">74</a>]. The electrochemical nanobiosensors have been applied to cancer diagnostics and detection of infectious microorganisms, such as viruses, etc. [<a href="#CR75">75</a>].</p> <p id="Par76">Production of biosensors, applying the techniques of electrochemical impedance spectroscopy (EIS) CV, and square wave voltammetry (SWV), helps fast biosensing for several kinds of analytes. Electrochemical impedance spectroscopy (EIS) is applied at surface area and interaction modification agents and the electrode surface [<a href="#CR76">76</a>]. Due to the occurrence of an electrochemical reaction at the electrode surface over interaction with the exact molecule, impedance biosensors have been used more in environmental monitoring of disruptive drugs and chemicals, the interaction between antibody and antigen and DNA strains. The EIS provides data regarding surface adsorption, ion exchange, charge transfer, and diffusion [<a href="#CR77">77</a>].</p> <p id="Par77">In other electrochemical techniques, quantitative analysis applying SWV is one of the most promising mechanisms in the production of biosensors, because of their ability to provide more sensitive answers for fast biosensing as compared to differential pulse voltammetry (DPV) techniques. In pulse methods, the systems depend on the usage of pulse changes of potential, and the usual response is measured at a suitable time relative to the time of the pulse [<a href="#CR78">78</a>].</p> <p id="Par78">Siuzdak et al. [<a href="#CR79">79</a>], worked on different methods for the diagnosis of pathogens. They applied a nanocrystalline boron-doped diamond-based electrode (B: NCD) as a platform for the biosensor.</p> <p id="Par79">Detection of avian influenza virus, (H5N1), is determined by different methods such as the immunochromatography, reverse-transcription PCR (RT-PCR), serological methods, ELISA [<a href="#CR80">80</a>], and fluorescence.</p> <p id="Par80">Moreover, the electrochemical diagnosis method gained much attention because it offered low costs, small specimen volume without an amplification step, and user-friendly interface and portability [<a href="#CR81">81</a>].</p> <p id="Par81">The hepatitis B e-antigen (HBeAg) immunosensor was also developed by applying electrochemical methods. Cocatalysis of nanoporous gold and horseradish peroxidase (HRP) were applied as modifier factors. The developed immunosensor demonstrated a linear realtionship between peak current and concentration of HBeAg (1â€‰pg/mL to 1â€‰ng/mL as well as 0.064â€‰pg/mL of LoD (limit of detection)).</p> <p id="Par82">The electrochemical DNA biosensing instrument is an effective device due to properties, such as fast response time, user-friendly, high specificity, and sensitivity. Electrochemical paper analytical device (ePADs) creates a perfect contribution to the sensor, because of its use of the inexpensive paper substrate. Singhal et al. [<a href="#CR82">82</a>] reported the manufacturing of ePADs, by detecting the target DNA of the Chikungunya virus (CHIKV).</p> <p id="Par83">The most notable applied techniques include voltammetry, amperometry, and impedance spectroscopy methods. The most developed virus biosensors can be applied with integrated substrates for clinical, environmental, and industrial applications [<a href="#CR83">83</a>]. Some of the selected usages are listed in TableÂ <a href="#Tab1">1</a>. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Some selected studies on the detection of viruses by electrochemical methods</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Target</th> <th>Biosensor type</th> <th>Nanomaterial</th> <th>Sample type</th> <th>Reference</th> </tr> </thead> <tbody> <tr> <td>Influenza virus M1 protein</td> <td>Electrochemical impedance</td> <td>Nanocrystalline boron-doped diamond</td> <td>Saliva</td> <td>[<a href="#CR84">84</a>]</td> </tr> <tr> <td>Hepatitis C virus DNA</td> <td>Electrochemical impedance</td> <td>MB@SiNPs</td> <td>Real patient specimen</td> <td>[<a href="#CR85">85</a>]</td> </tr> <tr> <td>Dengue virus DNA</td> <td>Voltammetric</td> <td> <p>ZnO/Pt_Pd</p> <p>nanocomposite</p> </td> <td>â€"</td> <td>[<a href="#CR85">85</a>]</td> </tr> <tr> <td>HBV DNA</td> <td>Electrochemical impedance</td> <td>AuNPs</td> <td>Real patient specimen</td> <td>[<a href="#CR86">86</a>]</td> </tr> <tr> <td>Avian influenza virus H5N1 gene</td> <td>Voltammetric</td> <td>MWCNTs_AuNPs</td> <td>â€"</td> <td>[<a href="#CR87">87</a>]</td> </tr> <tr> <td>Dengue type 2 virus</td> <td>Voltammetric</td> <td>Nanoporous alumina</td> <td> <p>Infected Aedesaegypti</p> <p>Mosquito sample</p> </td> <td>[<a href="#CR88">88</a>]</td> </tr> <tr> <td>Influenza virus</td> <td>Voltammetric</td> <td>CdS QDs</td> <td>Real patient samples</td> <td>[<a href="#CR89">89</a>]</td> </tr> <tr> <td>Chikungunya virus DNA</td> <td>Electrochemical paper analytical device</td> <td>Gold shells_coated magnetic nanocubes</td> <td>Serum</td> <td>[<a href="#CR90">90</a>]</td> </tr> <tr> <td>Influenza virus H1N1</td> <td>Chronoamperometric</td> <td>rGO</td> <td>â€"</td> <td>[<a href="#CR91">91</a>]</td> </tr> </tbody> </table>  <p /> </div> <div class="aptamer-baseddetection" title="sec"> <div class="title" tagx="title" title="title">
Aptamer-Based Detection</div> <p id="Par84">Aptamers are artificial single-stranded DNA or RNA oligonucleotides that have a high affinity towards a target, which they are designed to bind to [<a href="#CR92">92</a>]. Aptamer-based detection of viruses may be able to overcome barriers generated by the use of other methods such as costs and false-negative or false-positive results [<a href="#CR93">93</a>]. Aptamer-based biosensors or aptasensors, are fabricated by using aptamer as bioreceptors (capturing aptamer/probe) or transducers (signal aptamer/probe) [<a href="#CR94">94</a>]. In a comprehensive review by Zou et al. [<a href="#CR95">95</a>] on the application of aptamers in virus detection and antiviral therapy, a list of virus-specific aptamers that have been assessed but are yet to be used in the detection of viruses and treatment of viral infectious diseases, is introduced for the first time. Based on the transducer type, aptasensors are either electrical or optical.</p> <div class="opticalaptasensors" title="sec"> <div class="title" tagx="title" title="title">
Optical Aptasensors</div> <p id="Par85"> </p><ol> <li> <p id="Par86">Surface plasmon resonance or SPR aptasensors, which by evaluating the change of refractivity of a material bound on a surface, measures the resonance of free electrons in the metal films [<a href="#CR96">96</a>]. Typically, the capturing aptamer is immobilized on a metal surface. As the virus binds to the aptamer, the thickness of the metal surface changes, resulting in the alteration of the refractive index. Quantification can be carried out by monitoring the difference in the angle or intensity of light after the virus is bound to the aptamer. The advantages of this method are miniaturization, automation, and absence of labelling [<a href="#CR97">97</a>]. This method has been used for the detection of avian influenza virus (AIV) H5N1 [<a href="#CR98">98</a>, <a href="#CR99">99</a>], and HIV-1 Tat protein [<a href="#CR100">100</a>].</p> </li> <li> <p id="Par87">Calorimetric-based aptasensors use a shift in color to detect a virus. This shift can be seen by the naked eye, or it will require a spectrophotometer. These types of aptasensors are cheap, simple, usually portable, and have been used in many applications [<a href="#CR101">101</a>]. They fall into different groups, for example, nanomaterial-based calorimetric aptasensors [<a href="#CR102">102</a>, <a href="#CR103">103</a>] can use a nanomaterial as support for the aptamer or as a part of the transducer to create the signal with their assistance. Another type is the enzyme-linked aptamer assays (ELAA) which have the same base as ELISA, but use aptamers instead of antibodies as the bio-receptor or the transducer [<a href="#CR104">104</a>]. This type has been used to detect H5N1 [<a href="#CR104">104</a>], Zika virus [<a href="#CR105">105</a>], and others [<a href="#CR106">106</a>]. Lastly, aptamers have been used to modify lateral flow assay (LFA) for virus detection [<a href="#CR107">107</a>].</p> </li> <li> <p id="Par88">Fluorescent aptasensors, which can be categorized into aptasesnors that respond with fluorescent intensity [<a href="#CR108">108</a>], or the ones that respond with fluorescence polarization [<a href="#CR109">109</a>].</p> </li> <li> <p id="Par89">Others such as SERS based [<a href="#CR110">110</a>] and CL aptasensors [<a href="#CR111">111</a>].</p> </li> </ol> <p /> </div> <div class="electricalaptasensors" title="sec"> <div class="title" tagx="title" title="title">
Electrical Aptasensors</div> <p id="Par90">In this type, by binding of the aptamer to the target, production or change of an electrical signal occurs. Based on their detection mechanism they fall into two groups. </p><ol> <li> <p id="Par91"><i>Electrochemical aptasensors</i></p> </li> </ol> <p /> <p id="Par92">Usually in this type of sensor, the aptamer is immobilized on an electrode. In some applications, no enzyme is employed and binding of the immobilized aptamer to the target changes the impedance directly [<a href="#CR112">112</a>]. But in some others, the electrical signal change is assisted by employing enzymatic reactions [<a href="#CR113">113</a>]. The latter is a field-effect transistor (FET) which is a type of voltage-controlled semiconductor device that regulates electrical behavior by using an electric field and has been used to detect the Tat protein of HIV-1 [<a href="#CR114">114</a>]. </p><ul> <li><span class="label" tagx="label" title="label">2.</span><p id="Par93"><i>Piezoelectric Transducers</i></p> </li> </ul> <p /> <p id="Par94">Some materials such as the quartz crystal microbalance (QCM) have a Piezoelectric effect which is the ability to generate an electric charge in response to applied mechanical stress. In QCM aptasensors, the aptamer is fixated on the quartz crystal electrode so it can capture the target. The binding of the aptamer to the target changes the quality of the pole, which is subsequently converted into detectable frequency changes [<a href="#CR115">115</a>].</p> </div> </div> </div> <div class="nanomechanical" title="sec"> <div class="title" tagx="title" title="title">
Nanomechanical</div> <p id="Par95">B. Ilic et al. [<a href="#CR116">116</a>] used a resonating nanomechanical cantilever made from polycrystalline silicon to detect the immunospecific binding of viruses. Arrays of AcV1 antibody-coated polycrystalline silicon nanomechanical cantilever beams were used to detect binding at different concentrations of baculoviruses in a buffer solution. They calculated the mass of single-virus particles bound to the cantilever through a viable technique for sensitive detection of bound mass. Therefore, using this nanoelectromechanical device enables the selective detection of the virus through virus - antibody interaction (TableÂ <a href="#Tab2">2</a>). <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>Summary of the mentioned virus detection methods</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Method</th> <th>Advantages</th> <th>limitations</th> </tr> </thead> <tbody> <tr> <td><b>IF methods</b></td> <td>Inexpensive, accessibility, ease of operation, easy interpretation of results</td> <td>Antisera concentration, the inability of the detection of new strains/species, instability of antibodies</td> </tr> <tr> <td><b>NAATs</b></td> <td>Ease of purification and separation of nucleic acids from antibodies, Excellent sensitivity, and specificity</td> <td>Expensive, the technical expertise required</td> </tr> <tr> <td><b>Electron microscopy</b></td> <td>Accessible</td> <td>The inability of the specific diagnosis of viruses</td> </tr> <tr> <td><b>Cell culture</b></td> <td>Capacity to isolate a wide variety of viruses, ease of operation</td> <td>The long incubation period for some viruses,</td> </tr> <tr> <td><b>Aptamer- based</b></td> <td>Easy synthesize and modification of aptamers, a broad range of targets, inexpensive</td> <td>Some methods need expert operators</td> </tr> </tbody> </table>  <p /> </div> </div> <div class="applicationsofviraldiagnostictechniquesfordifferenttypesofviruses" title="sec"> <div class="title" tagx="title" title="title">
Applications of Viral Diagnostic Techniques for Different Types of Viruses</div> <p id="Par96">No single approach is optimal for detecting all viruses in all clinical situations. Therefore, it is vital to combine culture and nonculture methods to optimize viral disease diagnosis, yielding medically useful, cost-effective, and laborsaving viral testing results. In determining appropriate testing algorithms for the laboratory, one must consider a broad range of factors, including the patient population (i.e., age, immune level, etc.), clinical expressions, physicianâ€™s diagnosis, and time of year. However, with global travel and changing epidemiology of viral diseases, it isnâ€™t possible to prepare a virology laboratory for unexpected epidemics or contamination [<a href="#CR42">42</a>].</p> <div class="testingforrespiratoryviruses" title="sec"> <div class="title" tagx="title" title="title">
Testing for Respiratory Viruses</div> <p id="Par97">During periods of low virus activity, OIAs, or lateral-flow antigen assays may be used, but it is better to check all positive results via IF, viral culture, or RT-PCR [<a href="#CR117">117</a>]. Most of these assays have good specificity, which leads to a decrease in false-positives, hence achieving trustworthy results.</p> <p id="Par98">NAATs are another approach for respiratory virus detection. NAATs may be the only test, a supplemental test, or an additional test for specimen that obtain negative results by other test methods. The most reference laboratories perform NAAT testing for several virus diagnoses. Depending on the testing format, the costs will vary. In general, NAAT testing will be more costly than viral culture methods [<a href="#CR118">118</a>].</p> </div> <div class="viraltestingofvesicularlesions" title="sec"> <div class="title" tagx="title" title="title">
Viral Testing of Vesicular Lesions</div> <p id="Par99">At first, direct fluorescent antibody (DFA) testing for VZV, HSV-1, and HSV-2 is carried out on direct smears of vesicular lesions. If the DFA is positive, virus isolation in cell culture is usually not useful, in contrast, if DFA is negative, cell culture-based methods should be used [<a href="#CR42">42</a>].</p> <p id="Par100">If NAAT testing is available, it should be considered due to its great sensitivity [<a href="#CR119">119</a>].</p> </div> <div class="viraltestingofcsf(cerebrospinalfluid)" title="sec"> <div class="title" tagx="title" title="title">
Viral Testing of CSF (Cerebrospinal Fluid)</div> <p id="Par101">NAAT is more sensitive than culture techniques in detecting viral CNS pathogens [<a href="#CR120">120</a>]. The enteroviruses, the arboviruses, JC viruses, and the viruses in the <i>Herpesviridae</i> family (VZV, HSV-1, HSV-2, EBV, CMV, and HHV-6) are most commonly associated with CNS infections.</p> </div> <div class="enterovirusesincsf" title="sec"> <div class="title" tagx="title" title="title">
Enteroviruses in CSF</div> <p id="Par102">Isolation of CSF enterovirus in cell culture is 75% less sensitive than NAATs. Therefore, NAATs are the most reliable method for the diagnosis of enterovirus infection in CSF [<a href="#CR119">119</a>]. 5_ UTR pan- enteroviral primers used in these approaches can detect a broad spectrum of enteroviruses, containing the nonculturable coxsackievirus strains. However, for the detection of parechoviruses, other probes are required [<a href="#CR121">121</a>].</p> </div> <div class="viraltestingofothertypesofsamples" title="sec"> <div class="title" tagx="title" title="title">
Viral Testing of Other Types of Samples</div> <div class="fecalsamplesforthedetectionofentericviruses" title="sec"> <div class="title" tagx="title" title="title">
Fecal Samples for the Detection of Enteric Viruses</div> <p id="Par103">Detection of CMV, HSV, and enteroviruses can be carried out using cell culture methods. Adenovirus and rotavirus antigens can be recognized via EIA-based assays. Besides, NAATs or electron microscopy can detect noroviruses, astroviruses, and caliciviruses accurately [<a href="#CR42">42</a>].</p> </div> <div class="peripheralbloodsamples" title="sec"> <div class="title" tagx="title" title="title">
Peripheral Blood Samples</div> <p id="Par104">Only a few types of viruses including CMV are isolated from peripheral blood. Rapid shell vial cultures of fibroblasts or H &amp;amp; V Mix yield sensitive results within 24 to 48â€‰h or sooner. Although the detection of CMV in blood samples through shell vial culture shows less sensitivity when compared to CMV isolation in traditional tube cell cultures. However, itâ€™s better to use tube cell cultures for samples with very low levels of virus.</p> <p id="Par105">Also, EBV, HHV-6, and/or BK viruses should be tested by NAATs. HSV and adenoviruses may also be isolated from blood cell cultures, rarely. NAATs also can be used for the detection of HSV and enteroviruses in blood from infants with neonatal sepsis [<a href="#CR42">42</a>].</p> </div> <div class="urinesamples" title="sec"> <div class="title" tagx="title" title="title">
Urine Samples</div> <p id="Par106">Detection of CMV and adenoviruses can be performed through shell vial and traditional tube cultures using a urine sample. In renal transplant patients, quantitation of the BK virus should be checked using NAATs [<a href="#CR42">42</a>].</p> </div> </div> </div> <div class="methodsdevelopedtodetectsars-cov-2" title="sec"> <div class="title" tagx="title" title="title">
Methods Developed to Detect SARS-CoV-2</div> <p id="Par107">With the improvement in molecular biological techniques, nucleic acid detection methods have developed at a fast rate and are now considered as one of the best methods for virus detection [<a href="#CR122">122</a>â€"<a href="#CR124">124</a>]. At present, a suitable method for the diagnosis of new pneumonia in clinics and laboratories is chest radiographic findings, but the most accurate approach depends on nucleic acid-based assays [<a href="#CR125">125</a>]. However, the sample for nucleic acid assays from suspected patients of SARS-CoV-2 is mainly throat swabs [<a href="#CR80">80</a>]. This sample is not standardized and easily fails the detection process. The collection process of such samples is hazardous for medical personnel. The other method is RT-PCR, which is used for detecting and analyzing SARS-CoV-2 [<a href="#CR126">126</a>]. Polymerase chain reaction (PCR) has numerous advantages, such as fast diagnosis, high specificity, sensitivity, and this ordinary quantitative assay has been considered as the â€œgold standardâ€� for virus detection [<a href="#CR96">96</a>]. The number of SARS-CoV-2-infected patients is increasing quickly, however, laboratory and clinical detection methods are limited [<a href="#CR125">125</a>].</p> <p id="Par108">Here below are a number of methods that researchers used for detection of the SARS-CoV-2 virus.</p> <div class="pcr-basedmethods" title="sec"> <div class="title" tagx="title" title="title">
PCR-Based Methods</div> <p id="Par109">As described above, PCR is an enzymatic method that creates multiple copies of a gene [<a href="#CR69">69</a>]. In general, SARS-CoV-2 involves mRNA converted into cDNA by reverse transcription, then PCR occurs and the resulting products undergo gel electrophoresis and sequencing to detect the SARS-CoV-2 virus [<a href="#CR70">70</a>, <a href="#CR71">71</a>].</p> <p id="Par110">Corman et al. [<a href="#CR72">72</a>], used respiratory specimens from infected cases at CharitÃ© Medical Center during 2019. They obtained cell culture supernatants from SARS-CoV-2 in their clinical laboratories, and extracted their RNA for PCR purposes.</p> <p id="Par111">They used all complete sequences of SARS-related viruses that were available at GenBank by January 1st, 2020 to create oligonucleotides and carry out in-silico evaluation. Then, they used these sequences for alignment and assay design.</p> <p id="Par112">In the end, they performed sequence alignments with those from the Wuhan cluster. All of the sequences that they found matched the amplicons (Fig.Â <a href="#Fig2">2</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Partial alignments of oligonucleotide binding regions. Panels show six available sequences of the Wuhan-CoV, aligned to the corresponding partial sequences of SARS-CoV strain Frankfurt 1, which can be used as a positive control for all three RT-PCR assays. The alignment also contains the most closely-related bat virus (Bat SARS-related CoV isolate, bat-SL-CoVZC45, GenBank Acc.No. MG772933.1) as well as the most distant member within the SARS-related bat CoV clade, detected in Bulgaria (GenBank Acc. No. NC_014470). Dots represent identical nucleotides compared to Wuhan-Hu 1. Substitutions are specified. More comprehensive alignments in the Appendix. Reprinted with permission from ref. [<a href="#CR127">127</a>]</p>   </div> <p /> <p id="Par113">In a study by Corman et al. [<a href="#CR127">127</a>], they indicate the most closely-related bat virus and the distant member within the SARS-related bat CoV clade to Wuhan-Hu 1. They show that in all assays researchers can use SARS-CoV genomic RNA as a positive control.</p> <p id="Par114">In another study, real-time reverse-transcription PCR assays for detecting the different parts (ORF1b and N) of the viral genome were carried out. Accordingly, the probe and primer were designed in such a way that they could react to this new SARS-CoV-2 and other SARS-CoV viruses including SARS and MERS [<a href="#CR71">71</a>].</p> <p id="Par115">They used two different kits; for RNA extractions, they used the viral RNA purification kit (QIAamp Viral RNA Mini Kit, Qiagen), and for DNA extractions they used the DNA plasmid purification kit (QIAprep Spin Miniprep Kit, Qiagen).</p> <p id="Par116">Moreover, they obtained respiratory samples from two SARS-CoV-2-infected cases. For the first patient, a sputum sample was obtained on day 5 after the first appearance of symptoms. For the second patient, a throat swab sample was obtained on day 3 after the symptoms appeared. The samples were then stored in standard virus transport media.</p> <p id="Par117">They utilized RNA from cells that were infected with SARS-CoV-2 as a positive control. Furthermore, after RT-PCR, they cloned the products of the SARS-CoV-2 formed by the ORF1b and N gene assays into plasmids.</p> <p id="Par118">In this study, researchers used two assays with different viruses, due to insufficient information about this novel SARS-CoV-2 virus in humans and animals.</p> <p id="Par119">They achieved some interesting data, which showed that, when RNA from infected cells is used as a positive control, assays indicate a range of seven orders of (2*10<sup>âˆ’â€‰4</sup>-2000 TCID50/reaction), and when DNA plasmid was used as a positive control, then the diagnoses of the assays were below 10 copies for each reaction. The specimen from the infected cases were positive in the test.</p> <p id="Par120">They claimed that these assays can quickly detect this novel SARS-CoV-2 in human specimens, and this will help to identify patients as soon as possible.</p> </div> <div class="enzyme-linkedimmunosorbentassay(elisa)" title="sec"> <div class="title" tagx="title" title="title">
Enzyme-Linked Immunosorbent Assay (ELISA)</div> <p id="Par121">Xiang et al. [<a href="#CR123">123</a>] confirmed two new kits for detecting the novel SARS-CoV-2 virus. These kits have acquired significant results, but they are not available on the market. In the above study, all the SARS-CoV-2-infected cases at the Jinyintan Hospital, starting from January 1 to January 28, 2020, were investigated.</p> <p id="Par122">The new IgG/IgM antibody enzyme-linked immunosorbent assay (ELISA) kit was obtained from the Zhuhai Livzon Diagnostics Inc., China. Serum specimens for the ELISA IgG and IgM antibodies tests were obtained from infected cases on February 2, 2020. The 98 serum specimens were tested in accordance with the manufacturerâ€™s instructions.</p> <p id="Par123">Results of the ELISA test for the 63 samples of SARS-CoV-2-infected cases indicates that 28 IgM antibodies were positive, the accuracy of the test was 82.54% (52/63) with specificity of 100% (35/35) and 52 IgG antibodies were positive, the accuracy of the test was 88.8% (87/98) with the specificity of 100% (35/35), and the sensitivity for the detection of the combined IgM and IgG was 55/63 (87.3%) and the healthy controls was negative.</p> <div class="colloidalgold-immunochromatographicassay(gica)" title="sec"> <div class="title" tagx="title" title="title">
Colloidal Gold-Immunochromatographic Assay (GICA)</div> <p id="Par124">A new IgG/IgM antibody colloidal gold-immunochromatographic assay (GICA) kit for detecting SARS-CoV-2 was obtained from the Zhuhai Livzon Diagnostics Inc., China. Ninety one plasma specimens for the GICA were obtained from infected cases on February 3â€"4, 2020. The 126 plasma specimens were tested as per the manufacturerâ€™s instructions.</p> <p id="Par125">Result of the ELISA test for the 91 samples of COVID-19-infected cases indicated that 52 IgM antibodies were positive, the accuracy of the test was 69.0% (87/126) with the specificity of 100% (35/35), and 74 IgG antibodies were positive, the accuracy of the test was 86.5% (109/126) with the specificity of 100% (35/35. The sensitivity for the detection of combined IgM and IgG was 82.4% (75/91) and the healthy controls was negative [<a href="#CR126">126</a>].</p> </div> <div class="qrt-pcrassayforsars-cov-2" title="sec"> <div class="title" tagx="title" title="title">
qRT-PCR Assay for SARS-CoV-2</div> <p id="Par126">Three different samples including a sputum specimen, throat swab specimen, and alveolar lavage fluid samples were obtained from suspected cases of SARS-CoV-2 infection. The goal was to extract the RNA from these samples using the nucleic acid extraction kit (QIAamp viral RNA mini kit). The qRT-PCR was carried out using the ABI 7500 Real-Time PCR system. Two target genes, including the open reading frame 1ab (ORF1ab) and nucleocapsid protein (N), were setup together and tested during the qRT-PCR assay. This procedure used a SARS-CoV-2 nucleic acid detection kit, as per the manufacturerâ€™s instructions (Shanghai ZJ Bio-Tech Co., Ltd). The result of the qRT-PCR test for 81 samples of SARS-COV-2-infected cases indicated that 42 cases were positive and the sensitivity was 51.9% (42/81). They compared the sensitivity of the ELISA IgM+ IgG, GICA IgMâ€‰+â€‰IgG, and qRT-PCR methods and found significant variances between these methods (<i>P</i>â€‰<div class="â€‰0.001)._UNKNOWN">
Pâ€‰=â€‰0.411 [<a href="#CR126">126</a>].</div></p> <p id="Par127">In another study, Amanat et al. [<a href="#CR74">74</a>] found a new way of detecting SARS-COV-2 by using recombinant antigens extracted from the spike protein of SARS-CoV-2. The method depends on reactivity to the immunogenic S protein of the virus. These assays were formed with a negative control specimen showing pre-SARS-CoV-2 background immunity in the general population and samples obtained from the SARS-COV-2 infected cases.</p> <p id="Par128">They produced two different versions of the spike protein. The first structure declares a complete trimeric and stabilized version of the spike protein and the second one expresses only the much smaller receptor-binding domain (RBD). The sequence applied for both proteins depends on the genomic sequence of the first virus isolate, Wuhan-Hu-1, which was extracted on January 10th, 2020.</p> <p id="Par129">The plasma/sera sample were obtained from infected cases at day 20 (SARS-CoV2#1), on day 4 (SARS-CoV-2 #2), days 2 and 6 (SARS-CoV-2 #3A and B), after the symptoms appeared.</p> <p id="Par130">They applied a panel of 59 banked human serum specimens obtained from candidates who had previously-confirmed viral infections (e.g., hantavirus, dengue virus, SARS-CoV-2 NL63 â€" specimen collected 30â€‰days after the first symptom appeared) to establish an ELISA with these proteins.</p> <p id="Par131">ELISAs were conducted by doing serial dilutions of the individual serum specimens. All COVID-19 plasma/serum specimens reacted strongly to both RBD and the full-length spike protein, as the reactivity of the other serum specimens only yielded background reactivity.</p> <p id="Par132">Generally, the reactivity of SARS-CoV-2 sera was better against the full-length S protein than against the RBD. This may happen due to the higher number of epitopes existing on the much larger spike protein [<a href="#CR126">126</a>].</p> </div> <div class="antibodyisotypingandsubtyping" title="sec"> <div class="title" tagx="title" title="title">
Antibody Isotyping and Subtyping</div> <p id="Par133">For the four COVID-19-infected plasma/sera cases, Amanat et al. [<a href="#CR74">74</a>] also performed an isotyping and subtyping ELISA, which applied the mammalian cell and insect cell-expressed S proteins. Reactivity for all specimens for IgG3, IgM, and IgA was strong.</p> <p id="Par134">An IgG1 signal was also discovered for three out of the four specimens, while one sample indicated no reactivity. There was not any signal for IgG4 and reagents for IgG2 were not available.</p> <p id="Par135">The results indicate that these assays are sensitive and specific, allowing for diagnosis of SARS-COV-2, by applying human plasma/serum as soon as 3â€‰days after the symptoms appear. Most importantly, these methods do not need handling of infectious viruses and they can be modified to detect different kinds of antibodies. They claimed that their ELISA method will play a key role in determining the real rate of attack and rate of infection fatality in different human populations, and also in designing the kinetics of the antibody response to SARS-CoV-2 [<a href="#CR127">127</a>].</p> </div> </div> <div class="microarray-basedmethod" title="sec"> <div class="title" tagx="title" title="title">
Microarray-Based Method</div> <p id="Par136">The microarray-based method is a quick diagnostic procedure. In this method, at first RNA from the SARS-CoV-2 virus produces cDNA which is labeled with special probes during the reverse transcription stage. Afterward, these labeled cDNAs will be loaded onto the solid phase microarrays. Oligonucleotides are fixed on it, which will then go through a series of washing steps, and the free DNAs will be subsequently deleted.</p> <p id="Par137">In the end, with the special diagnosis probes, SARS-CoV-2 RNA will be detected. Because of its advantages, this method is one of the popular techniques for the diagnosis of COVID-19 [<a href="#CR75">75</a>].</p> <p id="Par138">In one study, Shi et al. [<a href="#CR76">76</a>] detected the SARS-CoV-2 virus in a clinical specimen by developing a 60-mer oligonucleotide microarray, based on the TOR2 sequence.</p> <p id="Par139">Guo et al. [<a href="#CR77">77</a>] designed a microarray to diagnose 24 single nucleotide polymorphism (SNP) mutations across the spike (S) gene of SARS-CoV, and the accuracy of the method was 100% with regard to the detection of the SARS-CoV-infected specimen.</p> <p id="Par140">In another study, Luna et al. [<a href="#CR78">78</a>] developed a nonfluorescent oligonucleotide array for diagnosis of the entire SARS-CoV genus. The advantage of this microarray was its low costs and high sensitivity when compared to real-time RT-PCR.</p> </div> <div class="newlydevelopedmethods" title="sec"> <div class="title" tagx="title" title="title">
Newly Developed Methods</div> <p id="Par141">One of the novel methods for detecting RNA is the RNA-targeting clustered regularly interspaced short palindromic repeats (CRISPR) method which makes use of the enzyme, Cas13 [<a href="#CR81">81</a>, <a href="#CR82">82</a>].</p> <p id="Par142">Zhang et al. [<a href="#CR128">128</a>], indicated that Cas13 can be used to target and destroy the different mammalian single-stranded RNA viruses. They produced a platform named SHERLOCK, which uses isothermal pre-amplification along with Cas13 to detect DNA or RNA. Their new protocol for SARS-COV-2, named â€œA protocol for detection of SARS-COV-2 using CRISPR diagnosticsâ€�, has been introduced on the website (<a href="https://broad.io/sherlockprotocol">https://broad.io/sherlockprotocol</a>), this protocol may help researchers to detect nucleic acids.</p> </div> <div class="immunosensor" title="sec"> <div class="title" tagx="title" title="title">
Immunosensor</div> <p id="Par143">Layqah and Eissa managed to fabricate an Immunosensor for detecting the SARS-CoV-2 virus, based on the array of gold nanoparticle-modified carbon electrodes [<a href="#CR129">129</a>]. The carbon electrode chip endured surface modification using AuNPs, so that the electron transfer rate and surface area would increase. In this work, AuNPs were electrodeposited on carbon disposable array electrodes; this was then used for electrode preparation. Such an immunosensor can be used for spiked nasal samples to simultaneously detect different types of CoV virus. This device was made out of eight electrodes, attached to the same chip. These electrodes were covered with human SARS-CoV-2 and MERS-CoV antigens. Two of the electrodes were bound to BSA instead of antigens for comparative purposes and to be used as controls. Also for each antigen, two electrodes were modified so the results could be more reliable.</p> <p id="Par144">The principle of the method is that if an antibody is present in the sample, it binds to the antigens immobilized on the surface of the electrodes. When power current is running through the sample, if an antibody is bound on the electrodes, the current will decrease. This is because the antibody is bulky and will prevent electrons from moving further. A fixed concentration of antibody was applied, so as to be compared to the results of the specimens. This device showed high sensitivity, as the detection range for MERS-CoV and SARS-CoV-2 were 0.001â€"100 and 0.01â€"10,000â€‰ng/ml<sup>âˆ’â€‰1</sup>, respectively. Also, it showed to be highly selective to MERS-CoV and SARS-CoV-2 antibodies among others such as Flu A and Flu B. The applications of gold nanoparticles for virus detection have been discussed fully in a review by Draz et al. [<a href="#CR130">130</a>].</p> </div> </div> <div class="samples" title="sec"> <div class="title" tagx="title" title="title">
Samples</div> <p id="Par145">Rapid collection and testing of appropriate samples collected from suspicious patients is a necessary step for management and control of a SARS-CoV-2. Suspected cases should be screened for the genome of the virus using nucleic acid amplification tests (NAAT), such as RT-PCR.</p> <div class="categoriesofspecimen[131]" title="sec"> <div class="title" tagx="title" title="title">
Categories of Specimen [<a href="#CR131">131</a>]</div> <p id="Par146"> </p><ol> <li> <p id="Par147">Upper respiratory tract specimens: These include nasopharyngeal and pharyngeal swabs.</p> </li> <li> <p id="Par148">Lower respiratory tract specimens: These include deep-cough sputum, alveolar lavage fluids, bronchial lavage fluid, and respiratory tract extracts.</p> </li> <li> <p id="Par149">Fecal specimens: Fecal samples are about 10â€‰g in weight.</p> </li> <li> <p id="Par150">Blood specimens: One should, as much as possible, collect anticoagulated blood in the acute phase within 7â€‰days after the onset of the disease. A 5â€‰ml sample of blood is required for each collection.</p> </li> </ol> <p /> <p id="Par151">Vacuum tubes containing EDTA anticoagulants are recommended for blood collection. </p><ul> <li><span class="label" tagx="label" title="label">5)</span><p id="Par152">Serum specimens: Both acute-phase and convalescent serum specimens should be collected as much as possible. The first serum specimen should be collected as soon as possible (preferably within 7â€‰days after the onset of illness), and the second specimen should be collected during 3â€"4â€‰weeks after the onset of illness. A 5â€‰ml sample of blood is required for each specimen. Serum specimens are mainly used for measuring antibodies, rather than nucleic acid testing.</p> </li> </ul> <p /> </div> <div class="methodsofspecimencollectionandprocessing[132,133]" title="sec"> <div class="title" tagx="title" title="title">
Methods of Specimen Collection and Processing [<a href="#CR132">132</a>, <a href="#CR133">133</a>]</div> <p id="Par153"> </p><ol> <li> <p id="Par154">Nasopharyngeal swab: The sampler gently holds the personâ€™s head with one hand, the swab in</p> <p id="Par155">another, insert the swab via nostril to enter, slowly get deep along the bottom of the lower nasal</p> <p id="Par156">canal. Because the nasal canal is curved, do not force too hard to avoid traumatic bleeding. When</p> <p id="Par157">the tip of the swab reaches the posterior wall of the nasopharyngeal cavity, rotate gently once, then slowly remove the swab and dip the swab tip into a tube containing 2-3â€‰ml virus preservation solution (or isotonic saline solution, tissue culture solution or phosphate buffer), discard the tail, and tighten the cap.</p> </li> <li> <p id="Par158">Pharyngeal swab: The sampled person first gargles with normal saline, the sampler immerses the swabs in sterile saline (virus preservation solution is not allowed to avoid antibiotic allergies), holds the head of the sampled person up slightly, with oneâ€™ s mouth wide open, making a sound â€œahâ€� to expose the lateral pharyngeal tonsils, insert the swabs, stick across the tongue roots, and wipe both sides of the pharyngeal tonsils with pressure at least 3 times, then wipe on the upper and lower walls of the pharynx for at least 3 times, and dip the swabs in a tube containing 2â€"3â€‰ml storage solution (or isotonic saline solution, tissue culture solution or phosphate buffer solution), discard the tail and tighten the cap. The pharyngeal swabs can also be placed in the same tube together with the nasopharyngeal swab.</p> </li> <li> <p id="Par159">Nasopharyngeal or respiratory tract extract: Extract mucus from the nasopharynx or extract</p> <p id="Par160">respiratory secretions from the trachea with a collector connected to a negative-pressure pump;</p> <p id="Par161">insert the head of the collector into the nasal cavity or trachea, turn on the negative pressure, rotate and slowly withdraw the head of the collector, collect the extracted mucus, and rinse the collector once with 3â€‰ml of sampling solution.</p> </li> <li> <p id="Par162">Deep cough sputum: Ask the patient to cough deeply, and collect the sputum coughed up in a 50â€‰ml screw-capped plastic tube containing 3â€‰ml of sampling solution. If the sputum is not collected in the sampling solution, 2â€"3â€‰ml of the sampling solution can be added into the tube before testing, or add sputum digestive reagents of an equal volume of sputum.</p> </li> <li> <p id="Par163">Bronchial lavage fluid: Insert the head of the collector into the trachea (about 30â€‰cm deep) from the nostril or the tracheal insertion part, inject 5â€‰ml of physiological saline, turn on the negative pressure, rotate the head of the collector and slowly withdraw it. Collect the extracted mucus and rinse the collector once with the sampling solution (a pediatric catheter connected to a 50-ml syringe may be used as an alternative to the collector).</p> </li> <li> <p id="Par164">Alveolar lavage fluid: After local anesthesia, insert a bronchoscope through the mouth or nose, pass through the pharynx into the branch of the right middle lobe or the lingular segment of the left lung, and insert the tip into the bronchial branch opening; slowly add sterilized physiological saline through the biopsy hole of the bronchoscope, with 30â€"50â€‰ml of saline each time, 100â€"250â€‰ml in total, 300â€‰ml at most.</p> </li> <li> <p id="Par165">Fecal specimen: Take 1â€‰ml sample treatment solution, pick up a little sample about the size of soybean and add it into the tube, gently blow for 3â€"5 times, set aside at room temperature for 10â€‰min, centrifuged at 8000â€‰rpm for 5â€‰min, absorb the supernatant for detection. Fecal specimen treatment solution can be prepared in-house by the laboratory: 1.211â€‰g tris, 8.5â€‰g sodium chloride, 1.1â€‰g calcium chloride anhydrous or 1.47â€‰g calcium chloride-containing crystalline water, dissolved into 800â€‰ml of deionized water, with the pH adjusted to 7.5 with concentrated hydrochloric acid and replenishing with deionized water to 1000â€‰ml. Stool suspensions can also be prepared using HANKâ€™s solution or other isotonic saline solution, tissue culture solution, or phosphate buffer solution. If the patient has diarrhea symptoms, collect 3â€"5â€‰ml of stool specimen, gently blow and mix, centrifuge it at 8000â€‰rpm for 5â€‰min, absorb the supernatant to reserve for use.</p> </li> <li> <p id="Par166">Anal swab: Gently insert the disinfectant cotton swab into the anus for 3-5â€‰cm in-depth, then gently rotate and pull out, immediately put the swab into a 15-ml screw-capped sampling tube containing 3â€"5â€‰ml of virus preservation solution, discard the tail and tighten the tube cover.</p> </li> <li> <p id="Par167">Blood samples: It is recommended to use vacuum blood vessels containing the EDTA anticoagulant to collect 5â€‰ml of blood samples. Nucleic acid extraction should be performed on whole blood or plasma according to the type of nucleic acid extraction reagent selected. For plasma separation, the whole blood should be centrifuged at 1500 to 2000â€‰rpm for 10â€‰min, and the supernatant will be collected in sterile plastic tubes with a screw cap.</p> </li> <li> <p id="Par168">Serum specimen: Collect a 5-ml blood specimen with a vacuum negative-pressure blood collection tube. Keep the specimen at room temperature for 30â€‰min, centrifuge it at 1500â€"2000â€‰rpm for 10â€‰min, and collect the serum in a sterile plastic tube with a screw cap (<a href="http://www.chinacdc.cn">www.chinacdc.cn</a>).</p> </li> </ol> <p /> <p id="Par169">Other samples that can be collected for detecting SARS-CoV-2 are blood and stool [<a href="#CR134">134</a>â€"<a href="#CR138">138</a>] (TableÂ <a href="#Tab3">3</a>). <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>Types of specimen, collection material, and storage</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Specimen type</th> <th>Collection materials</th> <th>Storage temperature until testing in-country laboratory</th> <th>Recommended temperature for shipment according to expected shipment time</th> </tr> </thead> <tbody> <tr> <td>Nasopharyngeal and oropharyngeal swab</td> <td>Dacron or polyester flockedswabs</td> <td>2â€"8â€‰Â°C</td> <td> <p>2â€"8â€‰Â°C if â‰¤5â€‰days</p> <p>â€"70â€‰Â°C (dry ice) if &amp;gt;â€‰5â€‰days</p> </td> </tr> <tr> <td>Bronchoalveolar lavage</td> <td>Sterile container</td> <td>2â€"8â€‰Â°C</td> <td> <p>2â€"8â€‰Â°C if â‰¤2â€‰days</p> <p>â€"70â€‰Â°C (dry ice) if &amp;gt;â€‰2â€‰days</p> </td> </tr> <tr> <td>(Endo) tracheal aspirate, nasopharyngeal or nasal wash/aspirate</td> <td>Sterile container</td> <td>2â€"8â€‰Â°C</td> <td> <p>2â€"8â€‰Â°C if â‰¤2â€‰days</p> <p>â€"70â€‰Â°C (dry ice) if &amp;gt;â€‰2â€‰days</p> </td> </tr> <tr> <td>Sputum</td> <td>Sterile container</td> <td>2â€"8â€‰Â°C</td> <td> <p>2â€"8â€‰Â°C if â‰¤2â€‰days</p> <p>â€"70â€‰Â°C (dry ice) if &amp;gt;â€‰2â€‰days</p> </td> </tr> <tr> <td>Tissue from biopsy or autopsy including from lung.</td> <td>Sterile container with saline orVTM</td> <td>2â€"8â€‰Â°C</td> <td> <p>2â€"8â€‰Â°C if â‰¤2â€‰days</p> <p>â€"70â€‰Â°C (dry ice) if &amp;gt;â€‰2â€‰days</p> </td> </tr> <tr> <td>Serum</td> <td>Serum separator tubes (adults:collect 3â€"5â€‰ml whole blood).</td> <td>2â€"8â€‰Â°C</td> <td> <p>2â€"8â€‰Â°C if â‰¤2â€‰days</p> <p>â€"70â€‰Â°C (dry ice) if &amp;gt;â€‰2â€‰days</p> </td> </tr> <tr> <td>Whole blood</td> <td>Collection tube</td> <td>2â€"8â€‰Â°C</td> <td> <p>2â€"8â€‰Â°C if â‰¤2â€‰days</p> <p>â€"70â€‰Â°C (dry ice) if &amp;gt;â€‰2â€‰days</p> </td> </tr> <tr> <td>Stool</td> <td>Stool container</td> <td>2â€"8â€‰Â°C</td> <td> <p>2â€"8â€‰Â°C if â‰¤2â€‰days</p> <p>â€"70â€‰Â°C (dry ice) if &amp;gt;â€‰2â€‰days</p> </td> </tr> <tr> <td>Urine</td> <td>Urine collection container</td> <td>2â€"8â€‰Â°C</td> <td>2â€"8â€‰Â°C if â‰¤2â€‰daysâ€"70â€‰Â°C (dry ice) if &amp;gt;â€‰2â€‰days</td> </tr> </tbody> </table>  <p /> </div> </div> <div class="futureofdiagnosticmethodsforviralinfections" title="sec"> <div class="title" tagx="title" title="title">
Future of Diagnostic Methods for Viral Infections</div> <p id="Par170">Tissue culture is considered as the gold standard for viral diagnosis [<a href="#CR139">139</a>], but in the case of epidemics such a SARS-CoV-2, this method has the disadvantage of being time-consuming. Hence when rapid diagnosis is required, the tissue culture technique is rarely considered. However, when it comes to research, this method is the best one for detection and characterization of viruses isolated from cells and investigating the response of each host to viral infection [<a href="#CR139">139</a>].</p> <p id="Par171">Recently, Kaya et al. [<a href="#CR83">83</a>] reviewed the progress in electrochemical methods for the diagnosis of viral infections. These methods can be designed and specified for different viruses. They are fast, not expensive, and user-friendly. However, they must be modified to be suitable for a medical diagnostic laboratory, so as to compete with the currently most used methods, like RT-PCR. Most importantly, they should be as sensitive as molecular methods, such as RT-PCR.</p> <p id="Par172">A systematic review carried out by Pang et al. [<a href="#CR140">140</a>] indicated that molecular tests such as RT-PCR show higher sensitivity and specificity than serological ones, such as ELISA. However, Xiang et al. [<a href="#CR141">141</a>] in a recent study on COVID-19 patients showed that while using ELISA and GICA kits designed for IgG/IgM antibodies and serum samples, sensitivity was high enough to compete with the qRT-PCR method. This can be promising with regard to modifying serological methods for viral infection diagnosis, as they can get sensitive enough to replace PCR-based methods. Moreover, they are less time-consuming than PCR and donâ€™t have the risk of showing false results, because of being contaminated [<a href="#CR83">83</a>]. Other methods based on antibody recognition, which were also discussed previously in this paper, can be adapted for different viruses if they could show enough sensitivity as well.</p> <p id="Par173">Microarray-based methods including the ones discussed here, have shown adequate sensitivity, almost as high as the RT-PCR method, and considering that they are cheaper and less time-consuming than RT-PCR, they can be very promising in the development of a new clinical diagnostic method.</p> <p id="Par174">As for SARS-CoV-2, WHO advises laboratories that while encountering a suspected case, nucleic acid amplification tests (NAAT), such as RT-PCR, should be carried out for virus detection. The world health Organization (WHO) has also presented several instructions regarding specimen collection. In fact in a recent investigation by Gu et al. [<a href="#CR142">142</a>], stool specimens were confirmed to contain the virus, even in patients that were discharged. So the WHO advises that stool specimens should be taken from suspects, as well as respiratory samples [<a href="#CR1">1</a>]. Nucleic acid amplification tests (NAAT), are the only methods confirmed by WHO for diagnosing SARS-CoV-2 infections, so any proposed method should show sensitivity as high as that of the RT-PCR method. Furthermore, it also be more affordable and faster than the current available molecular methods.</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par175">COVID-19 is an ongoing pandemic disease, and since the progress of infection by the SARS-CoV-2 virus can lead to severe permanent respiratory problems and possible death, there is an urgent need to provide various diagnostic strategies for early detection of the disease. SARS-CoV-2 is a novel coronavirus, therefore, it is not yet possible to determine with certainty what method is the best for reliable and accurate diagnosis of the disease.</p> <p id="Par176">Although the clinical symptoms of COVID-19, such as fever, cough, dyspnea, acute kidney injury, liver damage may help healthcare workers make a primary diagnosis, but the final detection of the COVID-19 in patients is confirmed by one of the approved methods that include lung CT scans, molecular assays, and serological tests. Moreover, nowadays, several other virus detection methods are available for viral diseases, the use of which depend on specific properties of each virus or virus family, however, they are not routine and specific for COVID-19 detection, which requires further research for the development of more accurate tests. Besides the type of sample and sampling procedures, clinical specimens are also of critical importance in acquiring the most reliable test results for COVID-19. In conclusion, the understanding of the advantages and disadvantages of various virus detection techniques and their proper implementation could prevent further virus outbreaks, provide efficient control of the current pandemic situation, and prevent medical staff fatigue, all of which will ultimately help in the promotion of public health and safety.</p> <p id="Par177">Hence, researchers should focus more on different approaches, in order to ultimately find a highly efficient and accurate method for the rapid and precise detection of viral diseases like COVID-19. In this way, the devastating outcomes of the disease could be prevented.</p> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
SARS-CoV-2</div> <div class="def" title="def"> <p id="Par3">Severe acute respiratory syndrome coronavirus 2</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PHEIC</div> <div class="def" title="def"> <p id="Par4">Public health emergency of international concern</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MERS</div> <div class="def" title="def"> <p id="Par5">Middle East Respiratory Syndrome</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SARS</div> <div class="def" title="def"> <p id="Par6">Severe Acute Respiratory Syndrome</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RT-PCR</div> <div class="def" title="def"> <p id="Par7">Reverse-transcription Polymerase Chain Reaction</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
rRT-PCR</div> <div class="def" title="def"> <p id="Par8">Real-time RT-PCR</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CT</div> <div class="def" title="def"> <p id="Par9">Computed tomography</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RNA</div> <div class="def" title="def"> <p id="Par10">Ribonucleic Acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DNA</div> <div class="def" title="def"> <p id="Par11">Deoxyribonucleic Acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
IFA</div> <div class="def" title="def"> <p id="Par12">Indirect Fluorescent-Antibody</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
EIAs</div> <div class="def" title="def"> <p id="Par13">Enzyme Immunoassays</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ELISA</div> <div class="def" title="def"> <p id="Par14">Enzyme-Linked Immunosorbent Assay</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NAATs</div> <div class="def" title="def"> <p id="Par15">Nucleic Acid Amplification Tests</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PCR</div> <div class="def" title="def"> <p id="Par16">Polymerase Chain Reaction</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HDA</div> <div class="def" title="def"> <p id="Par17">Helicase Dependent Amplification</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RPA</div> <div class="def" title="def"> <p id="Par18">Recombinase Polymerase Amplification</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LAMP</div> <div class="def" title="def"> <p id="Par19">Loop-Mediated Isothermal Amplification</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NGS</div> <div class="def" title="def"> <p id="Par20">Next-Generation Sequencing</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NEMS</div> <div class="def" title="def"> <p id="Par21">Nanoelectromechanical Systems</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NPs</div> <div class="def" title="def"> <p id="Par22">Nanoparticles</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NMs</div> <div class="def" title="def"> <p id="Par23">Nanomaterials</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
EIS</div> <div class="def" title="def"> <p id="Par24">Electrochemical Impedance Spectroscopy</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SWV</div> <div class="def" title="def"> <p id="Par25">Square Wave Voltammetry</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DPV</div> <div class="def" title="def"> <p id="Par26">Differential Pulse Voltammetry</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
B: NCD</div> <div class="def" title="def"> <p id="Par27">Boron-doped Nanocrystalline Diamond-based</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HBeAg</div> <div class="def" title="def"> <p id="Par28">Hepatitis B e Antigen</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CHIKV</div> <div class="def" title="def"> <p id="Par29">Chikungunya Virus</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ELAA</div> <div class="def" title="def"> <p id="Par30">Enzyme-linked Aptamer Assays</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
FET</div> <div class="def" title="def"> <p id="Par31">Field-effect Transistor</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
QCM</div> <div class="def" title="def"> <p id="Par32">Quartz Crystal Microbalance</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
WHO</div> <div class="def" title="def"> <p id="Par33">World Health Organization</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> <div class="fn-type-" title=""> <p>Elena Ekrami, Mahdi Pouresmaieli, Fatemeh Barati, Sahar Asghari and Farzad Ramezani Ziarani contributed equally to this work.</p> </div> </div> <div class="ack" title="ack"> <p>This study was made possible by a grant from the National Institute of Genetic Engineering and Biotechnology (NIGEB), Tehran, Iran.</p> </div> <div class="title" tagx="title" title="title">
Authorsâ€™ Contributions</div> <p>Matin Mahmoudifard has a role in the design and collection of data. All authors contributed equally in drafting the first version of the manuscript. All authors participated in writing modified versions and read and approved the final manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>To perform this study, Matin Mahmoudifard received financial support (Grant No.: 750) from the National Institute of Genetic Engineering and Biotechnology (NIGEB), Tehran, The Islamic Republic of Iran.</p> <div class="title" tagx="title" title="title">
Availability of Data and Materials</div> <p>Not applicable for this study.</p> <div class="title" tagx="title" title="title">
Ethics Approval and Consent to Participate</div> <p id="Par178">Not applicable for this study.</p> <div class="title" tagx="title" title="title">
Consent for Publication</div> <p id="Par179">All authors have read and approved the final version of the manuscript.</p> <div class="title" tagx="title" title="title">
Competing Interests</div> <p id="Par180">The authors declare that they have no conflict of interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: Organization, W.H</span></span><span class="source" tagx="source" title="source">Novel coronavirus (2019-nCoV) situation report 23</span><span class="year" tagx="year" title="year">2020</span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Zhou P, et al. Discovery of a novel coronavirus associated with the recent pneumonia outbreak in humans and its potential bat origin. bioRxiv. 2020:2020.01.22.914952.</span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Novel Coronavirus ( 2019-nCoV ) Situation Report 48.</span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Middle East respiratory syndrome coronavirus: another zoonotic betacoronavirus causing SARS-like disease</span><span class="source" tagx="source" title="source">Clin Microbiol Rev</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">28</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">465</span><span class="lpage" tagx="lpage" title="lpage">522</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25810418">25810418</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">ZL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Origin and evolution of pathogenic coronaviruses</span><span class="source" tagx="source" title="source">Nat Rev Microbiol</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">181</span><span class="lpage" tagx="lpage" title="lpage">192</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30531947">30531947</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of a novel coronavirus in patients with severe acute respiratory syndrome</span><span class="source" tagx="source" title="source">N Engl J Med</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">348</span><span class="issue" tagx="issue" title="issue">20</span><span class="fpage" tagx="fpage" title="fpage">1967</span><span class="lpage" tagx="lpage" title="lpage">1976</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12690091">12690091</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaki</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia</span><span class="source" tagx="source" title="source">N Engl J Med</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">367</span><span class="issue" tagx="issue" title="issue">19</span><span class="fpage" tagx="fpage" title="fpage">1814</span><span class="lpage" tagx="lpage" title="lpage">1820</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23075143">23075143</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">395</span><span class="issue" tagx="issue" title="issue">10223</span><span class="fpage" tagx="fpage" title="fpage">497</span><span class="lpage" tagx="lpage" title="lpage">506</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31986264">31986264</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riou</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Althaus</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pattern of early human-to-human transmission of Wuhan 2019 novel coronavirus (2019-nCoV), December 2019 to January 2020</span><span class="source" tagx="source" title="source">Eurosurveillance</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">2000058</span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wunderink</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span></span><span class="mixed-article-title" title="mixed-article-title">MERS, SARS and other coronaviruses as causes of pneumonia</span><span class="source" tagx="source" title="source">Respirology</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">23</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">130</span><span class="lpage" tagx="lpage" title="lpage">137</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29052924">29052924</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Prevalence and impact of cardiovascular metabolic diseases on COVID-19 in China</span><span class="source" tagx="source" title="source">Clin Res Cardiol</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">109</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">531</span><span class="lpage" tagx="lpage" title="lpage">538</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32161990">32161990</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Wan Y, et al. Receptor recognition by the novel coronavirus from Wuhan: an analysis based on decade-long structural studies of SARS coronavirus. J Virol. 2020;94(7):1â€"9.</span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">H. Appleton, Viruses, in Food Technol., Elsevier, 2003, pp. 88_95.</span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oldstone</span><span class="given-names" tagx="given-names" title="given-names">MBA</span></span></span><span class="source" tagx="source" title="source">History of virology, in Encyclopedia of Microbiology</span><span class="year" tagx="year" title="year">2014</span><span class="edition" tagx="edition" title="edition">4</span><span class="fpage" tagx="fpage" title="fpage">608</span><span class="lpage" tagx="lpage" title="lpage">6012</span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levine</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enquist</span><span class="given-names" tagx="given-names" title="given-names">LW</span></span></span><span class="source" tagx="source" title="source">History of virology. In fields virology (eds B. N. Fields, D. M. Knipe &amp;amp; P. M. Howley), pp. 565â€"604</span><span class="year" tagx="year" title="year">2007</span><span class="edition" tagx="edition" title="edition">5</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Philadelphia</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Lippincott Williams &amp;amp; Wilkins</span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuart</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Viruses</span><span class="source" tagx="source" title="source">Curr Opin Struct Biol</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">167</span><span class="lpage" tagx="lpage" title="lpage">174</span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fehr</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlman</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronaviruses: an overview of their replication and pathogenesis</span><span class="source" tagx="source" title="source">Methods Mol Biol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">1282</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">23</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25720466">25720466</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemiology, genetic recombination, and pathogenesis of coronaviruses</span><span class="source" tagx="source" title="source">Trends Microbiol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">24</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">490</span><span class="lpage" tagx="lpage" title="lpage">502</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27012512">27012512</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cavanagh</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronavirus avian infectious bronchitis virus</span><span class="source" tagx="source" title="source">Vet Res</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">38</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">281</span><span class="lpage" tagx="lpage" title="lpage">297</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17296157">17296157</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ismail</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">AY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saif</span><span class="given-names" tagx="given-names" title="given-names">YM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pathogenicity of Turkey coronavirus in turkeys and chickens</span><span class="source" tagx="source" title="source">Avian Dis</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">47</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">515</span><span class="lpage" tagx="lpage" title="lpage">522</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14562877">14562877</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Fatal swine acute diarrhoea syndrome caused by an HKU2-related coronavirus of bat origin</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">556</span><span class="issue" tagx="issue" title="issue">7700</span><span class="fpage" tagx="fpage" title="fpage">255</span><span class="lpage" tagx="lpage" title="lpage">258</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29618817">29618817</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corman</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Hosts and sources of endemic human coronaviruses</span><span class="source" tagx="source" title="source">Adv Virus Res</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">163</span><span class="lpage" tagx="lpage" title="lpage">188</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29551135">29551135</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peiris</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">KY</span></span></span><span class="mixed-article-title" title="mixed-article-title">Severe acute respiratory syndrome</span><span class="source" tagx="source" title="source">Nat Med</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">12 Suppl</span><span class="fpage" tagx="fpage" title="fpage">S88</span><span class="lpage" tagx="lpage" title="lpage">S97</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15577937">15577937</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan-Yeung</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span></span><span class="mixed-article-title" title="mixed-article-title">SARS: epidemiology</span><span class="source" tagx="source" title="source">Respirology</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">Suppl</span><span class="fpage" tagx="fpage" title="fpage">S9</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15018127">15018127</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chowell</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">A dynamic compartmental model for the Middle East respiratory syndrome outbreak in the Republic of Korea: a retrospective analysis on control interventions and superspreading events</span><span class="source" tagx="source" title="source">J Theor Biol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">408</span><span class="fpage" tagx="fpage" title="fpage">118</span><span class="lpage" tagx="lpage" title="lpage">126</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27521523">27521523</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The clinical and virological features of the first imported case causing MERS-CoV outbreak in South Korea, 2015</span><span class="source" tagx="source" title="source">BMC Infect Dis</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">498</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28709419">28709419</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A novel coronavirus from patients with pneumonia in China, 2019</span><span class="source" tagx="source" title="source">N Engl J Med</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">382</span><span class="issue" tagx="issue" title="issue">8</span><span class="fpage" tagx="fpage" title="fpage">727</span><span class="lpage" tagx="lpage" title="lpage">733</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31978945">31978945</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><a href="https://www.worldometers.info/coronavirus/">https://www.worldometers.info/coronavirus/</a> Date retrieved: 3/21/2020.</span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">395</span><span class="issue" tagx="issue" title="issue">10223</span><span class="fpage" tagx="fpage" title="fpage">514</span><span class="lpage" tagx="lpage" title="lpage">523</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31986261">31986261</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Li Q, et al. Early transmission dynamics in Wuhan, China, of novel coronavirusâ€"infected pneumonia. N Engl J Med. 2020;382:1199â€"207.</span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO. Coronavirus disease 2019 (COVID-19) Situation Report â€" 23.</span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Cai, X., et al., A Peptide-based Magnetic Chemiluminescence Enzyme Immunoassay for Serological Diagnosis of Corona Virus Disease 2019 (COVID-19). medRxiv. 2020;222:2020.02.22.20026617.</span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Elden</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Simultaneous detection of influenza viruses a and B using real-time quantitative PCR</span><span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">39</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">196</span><span class="lpage" tagx="lpage" title="lpage">200</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11136770">11136770</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic characterization and infectivity of a novel SARS-like coronavirus in Chinese bats</span><span class="source" tagx="source" title="source">Emerg Microbes Infect</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">154</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30209269">30209269</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">395</span><span class="issue" tagx="issue" title="issue">10224</span><span class="fpage" tagx="fpage" title="fpage">565</span><span class="lpage" tagx="lpage" title="lpage">574</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32007145">32007145</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of viruses related to the SARS coronavirus from animals in southern China</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">302</span><span class="issue" tagx="issue" title="issue">5643</span><span class="fpage" tagx="fpage" title="fpage">276</span><span class="lpage" tagx="lpage" title="lpage">278</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12958366">12958366</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alagaili</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Middle East respiratory syndrome coronavirus infection in dromedary camels in Saudi Arabia</span><span class="source" tagx="source" title="source">mBio</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">e00884</span><span class="lpage" tagx="lpage" title="lpage">e00814</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24570370">24570370</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ray</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minnich</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Efficiency of immunofluorescence for rapid detection of common respiratory viruses</span><span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="year" tagx="year" title="year">1987</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">355</span><span class="lpage" tagx="lpage" title="lpage">357</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3029168">3029168</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madeley</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peiris</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Methods in virus diagnosis: immunofluorescence revisited</span><span class="source" tagx="source" title="source">J Clin Virol</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">121</span><span class="lpage" tagx="lpage" title="lpage">134</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12367646">12367646</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boonham</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Methods in virus diagnostics: from ELISA to next generation sequencing</span><span class="source" tagx="source" title="source">Virus Res</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">186</span><span class="fpage" tagx="fpage" title="fpage">20</span><span class="lpage" tagx="lpage" title="lpage">31</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24361981">24361981</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification and genomic characterization of the first isolate of bluetongue virus serotype 5 detected in Australia</span><span class="source" tagx="source" title="source">Vet Med Sci</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">129</span><span class="lpage" tagx="lpage" title="lpage">145</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30747479">30747479</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leland</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ginocchio</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Role of cell culture for virus detection in the age of technology</span><span class="source" tagx="source" title="source">Clin Microbiol Rev</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">20</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">49</span><span class="lpage" tagx="lpage" title="lpage">78</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17223623">17223623</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yolken</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coutlee</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viscidi</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span></span><span class="mixed-article-title" title="mixed-article-title">New prospects for the diagnosis of viral infections</span><span class="source" tagx="source" title="source">Yale J Biol Med</span><span class="year" tagx="year" title="year">1989</span><span class="volume" tagx="volume" title="volume">62</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">131</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2672618">2672618</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vunsh</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosner</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stein</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">The use of the polymerase chain reaction (PCR) for the detection of bean yellow mosaic virus in gladiolus</span><span class="source" tagx="source" title="source">Ann Appl Biol</span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">117</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">561</span><span class="lpage" tagx="lpage" title="lpage">569</span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dorado</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="source" tagx="source" title="source">Polymerase chain reaction (PCR)</span><span class="year" tagx="year" title="year">2019</span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mackay</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arden</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nitsche</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Real-time PCR in virology</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">30</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">1292</span><span class="lpage" tagx="lpage" title="lpage">1305</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11884626">11884626</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boonham</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Development of a real-time RT-PCR assay for the detection of potato spindle tuber viroid</span><span class="source" tagx="source" title="source">J Virol Methods</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">116</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">139</span><span class="lpage" tagx="lpage" title="lpage">146</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14738980">14738980</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mumford</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Advances in molecular phytodiagnosticsâ€"new solutions for old problems</span><span class="source" tagx="source" title="source">Eur J Plant Pathol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">116</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">19</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32214677">32214677</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Helicase-dependent isothermal DNA amplification</span><span class="source" tagx="source" title="source">EMBO Rep</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">8</span><span class="fpage" tagx="fpage" title="fpage">795</span><span class="lpage" tagx="lpage" title="lpage">800</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15247927">15247927</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Piepenburg O, et al. DNA detection using recombination proteins. PLoS Biol. 2006;4(7):1115â€"21.</span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mori</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Notomi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Loop-mediated isothermal amplification (LAMP): a rapid, accurate, and cost-effective diagnostic method for infectious diseases</span><span class="source" tagx="source" title="source">J Infect Chemother</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">15</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">62</span><span class="lpage" tagx="lpage" title="lpage">69</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19396514">19396514</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Notomi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Loop-mediated isothermal amplification of DNA</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">28</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">e63</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10871386">10871386</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">YP</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Loop-mediated isothermal amplification (LAMP): a versatile technique for detection of micro-organisms</span><span class="source" tagx="source" title="source">J Appl Microbiol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">124</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">626</span><span class="lpage" tagx="lpage" title="lpage">643</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29165905">29165905</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Silva, S.J.R.d., K. Pardee, and L. Pena, Loop-mediated isothermal amplification (LAMP) for the diagnosis of Zika virus: A Review<i>.</i> Viruses, 2020. 12(1): p. 19.</span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al Rwahnih</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Deep sequencing evidence from single grapevine plants reveals a virome dominated by mycoviruses</span><span class="source" tagx="source" title="source">Arch Virol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">156</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">397</span><span class="lpage" tagx="lpage" title="lpage">403</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21140178">21140178</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tzanetakis</span><span class="given-names" tagx="given-names" title="given-names">IE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Development of a virus detection and discovery pipeline using next generation sequencing</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">471</span><span class="fpage" tagx="fpage" title="fpage">54</span><span class="lpage" tagx="lpage" title="lpage">60</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25461531">25461531</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Czotter</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">NGS of virus-derived small RNAs as a diagnostic method used to determine viromes of Hungarian vineyards</span><span class="source" tagx="source" title="source">Front Microbiol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">122</span></span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldsmith</span><span class="given-names" tagx="given-names" title="given-names">CS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Modern uses of electron microscopy for detection of viruses</span><span class="source" tagx="source" title="source">Clin Microbiol Rev</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">22</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">552</span><span class="lpage" tagx="lpage" title="lpage">563</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19822888">19822888</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richert-PÃ¶ggeler</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Electron microscopy methods for virus diagnosis and high resolution analysis of viruses</span><span class="source" tagx="source" title="source">Front Microbiol</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">3255</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30666247">30666247</a></span></span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henrickson</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Advances in the laboratory diagnosis of viral respiratory disease</span><span class="source" tagx="source" title="source">Pediatr Infect Dis J</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">23</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">S6</span><span class="lpage" tagx="lpage" title="lpage">S10</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14730264">14730264</a></span></span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="source" tagx="source" title="source">Tissue culture technics for diagnostic virology. Diagnostic procedures for viral and rickettsial infections</span><span class="year" tagx="year" title="year">1969</span><span class="edition" tagx="edition" title="edition">4</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">American Public Health Association, Inc.</span><span class="fpage" tagx="fpage" title="fpage">81</span><span class="lpage" tagx="lpage" title="lpage">178</span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bisht R, Mandal A, Mitra AK. Micro-and Nanotechnology-Based Implantable Devices and Bionics, in Emerging Nanotechnologies for Diagnostics, Drug Delivery and Medical Devices. 2017, Elsevier. p. 249â€"290.</span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ilic</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Attogram detection using nanoelectromechanical oscillators</span><span class="source" tagx="source" title="source">J Appl Phys</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">95</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">3694</span><span class="lpage" tagx="lpage" title="lpage">3703</span></span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hafaiedh</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Supported protein G on gold electrode: characterization and immunosensor application</span><span class="source" tagx="source" title="source">Talanta</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">116</span><span class="fpage" tagx="fpage" title="fpage">84</span><span class="lpage" tagx="lpage" title="lpage">90</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24148377">24148377</a></span></span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tepeli</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ÃœlkÃ¼</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Electrochemical biosensors for influenza virus a detection: the potential of adaptation of these devices to POC systems</span><span class="source" tagx="source" title="source">Sensors Actuators B Chem</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">254</span><span class="fpage" tagx="fpage" title="fpage">377</span><span class="lpage" tagx="lpage" title="lpage">384</span></span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anik</span><span class="given-names" tagx="given-names" title="given-names">Ãœ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Towards the electrochemical diagnostic of influenza virus: development of a grapheneâ€"au hybrid nanocomposite modified influenza virus biosensor based on neuraminidase activity</span><span class="source" tagx="source" title="source">Analyst</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">143</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">150</span><span class="lpage" tagx="lpage" title="lpage">156</span></span></li> <li tag="ref"><a name="CR67" /><span class="label" tagx="label" title="label">67.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krejcova</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Electrochemical sensors and biosensors for influenza detectionâ€"literature survey 2012-2013</span><span class="source" tagx="source" title="source">Int J Electrochem Sci</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">3440</span><span class="lpage" tagx="lpage" title="lpage">3448</span></span></li> <li tag="ref"><a name="CR68" /><span class="label" tagx="label" title="label">68.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perumal</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hashim</span><span class="given-names" tagx="given-names" title="given-names">U</span></span></span><span class="mixed-article-title" title="mixed-article-title">Advances in biosensors: principle, architecture and applications</span><span class="source" tagx="source" title="source">J Appl Biomed</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">15</span></span></li> <li tag="ref"><a name="CR69" /><span class="label" tagx="label" title="label">69.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Balboni A, et al. A real-time PCR assay for bat SARS-like coronavirus detection and its application to Italian greater horseshoe bat faecal sample surveys. Sci World J. 2012;2012:1â€"8.</span></li> <li tag="ref"><a name="CR70" /><span class="label" tagx="label" title="label">70.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adachi</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Comprehensive detection and identification of human coronaviruses, including the SARS-associated coronavirus, with a single RT-PCR assay</span><span class="source" tagx="source" title="source">J Virol Methods</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">122</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">29</span><span class="lpage" tagx="lpage" title="lpage">36</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15488617">15488617</a></span></span></li> <li tag="ref"><a name="CR71" /><span class="label" tagx="label" title="label">71.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Setianingsih</span><span class="given-names" tagx="given-names" title="given-names">TY</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Detection of multiple viral sequences in the respiratory tract samples of suspected Middle East respiratory syndrome coronavirus patients in Jakarta, Indonesia 2015â€"2016</span><span class="source" tagx="source" title="source">Int J Infect Dis</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">102</span><span class="lpage" tagx="lpage" title="lpage">107</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31238156">31238156</a></span></span></li> <li tag="ref"><a name="CR72" /><span class="label" tagx="label" title="label">72.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corman</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="source" tagx="source" title="source">Diagnostic detection of Wuhan coronavirus 2019 by real-time RT-PCR</span><span class="year" tagx="year" title="year">2020</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Geneva</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">World Health Organization</span><span class="fpage" tagx="fpage" title="fpage">13</span></span></li> <li tag="ref"><a name="CR73" /><span class="label" tagx="label" title="label">73.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Chu DK, et al. Molecular diagnosis of a novel coronavirus (2019-nCoV) causing an outbreak of pneumonia. Clin Chem. 2020;66:549â€"55.</span></li> <li tag="ref"><a name="CR74" /><span class="label" tagx="label" title="label">74.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Amanat F, et al. A serological assay to detect SARS-CoV-2 seroconversion in humans. Nat Med. 2020;26:1033â€"6.</span></li> <li tag="ref"><a name="CR75" /><span class="label" tagx="label" title="label">75.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Electrochemical sensing in paper-based microfluidic devices</span><span class="source" tagx="source" title="source">Lab Chip</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">477</span><span class="lpage" tagx="lpage" title="lpage">483</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20126688">20126688</a></span></span></li> <li tag="ref"><a name="CR76" /><span class="label" tagx="label" title="label">76.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Design and application of 60mer oligonucleotide microarray in SARS coronavirus detection</span><span class="source" tagx="source" title="source">Chin Sci Bull</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">48</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">1165</span><span class="lpage" tagx="lpage" title="lpage">1169</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32214700">32214700</a></span></span></li> <li tag="ref"><a name="CR77" /><span class="label" tagx="label" title="label">77.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Development of a single nucleotide polymorphism DNA microarray for the detection and genotyping of the SARS coronavirus</span><span class="source" tagx="source" title="source">J Microbiol Biotechnol</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">24</span><span class="issue" tagx="issue" title="issue">10</span><span class="fpage" tagx="fpage" title="fpage">1445</span><span class="lpage" tagx="lpage" title="lpage">1454</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24950883">24950883</a></span></span></li> <li tag="ref"><a name="CR78" /><span class="label" tagx="label" title="label">78.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Souza Luna</span><span class="given-names" tagx="given-names" title="given-names">LK</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Generic detection of coronaviruses and differentiation at the prototype strain level by reverse transcription-PCR and nonfluorescent low-density microarray</span><span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">45</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">1049</span><span class="lpage" tagx="lpage" title="lpage">1052</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17229859">17229859</a></span></span></li> <li tag="ref"><a name="CR79" /><span class="label" tagx="label" title="label">79.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">NuÃ±ez</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doudna</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biology and applications of CRISPR systems: harnessing natureâ€™s toolbox for genome engineering</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">164</span><span class="issue" tagx="issue" title="issue">1â€"2</span><span class="fpage" tagx="fpage" title="fpage">29</span><span class="lpage" tagx="lpage" title="lpage">44</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26771484">26771484</a></span></span></li> <li tag="ref"><a name="CR80" /><span class="label" tagx="label" title="label">80.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gootenberg</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Nucleic acid detection with CRISPR-Cas13a/C2c2</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">356</span><span class="issue" tagx="issue" title="issue">6336</span><span class="fpage" tagx="fpage" title="fpage">438</span><span class="lpage" tagx="lpage" title="lpage">442</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28408723">28408723</a></span></span></li> <li tag="ref"><a name="CR81" /><span class="label" tagx="label" title="label">81.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freije</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Programmable inhibition and detection of RNA viruses using Cas13</span><span class="source" tagx="source" title="source">Mol Cell</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">76</span><span class="issue" tagx="issue" title="issue">5</span><span class="fpage" tagx="fpage" title="fpage">826</span><span class="lpage" tagx="lpage" title="lpage">837. e11</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31607545">31607545</a></span></span></li> <li tag="ref"><a name="CR82" /><span class="label" tagx="label" title="label">82.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gootenberg</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Multiplexed and portable nucleic acid detection platform with Cas13, Cas12a, and Csm6</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">360</span><span class="issue" tagx="issue" title="issue">6387</span><span class="fpage" tagx="fpage" title="fpage">439</span><span class="lpage" tagx="lpage" title="lpage">444</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29449508">29449508</a></span></span></li> <li tag="ref"><a name="CR83" /><span class="label" tagx="label" title="label">83.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kaya, S.I., et al., Electrochemical virus detections with nanobiosensors, in Nanosensors for Smart Cities. 2020, Elsevier. p. 303â€"326.</span></li> <li tag="ref"><a name="CR84" /><span class="label" tagx="label" title="label">84.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siuzdak</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biomolecular influenza virus detection based on the electrochemical impedance spectroscopy using the nanocrystalline boron-doped diamond electrodes with covalently bound antibodies</span><span class="source" tagx="source" title="source">Sensors Actuators B Chem</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">280</span><span class="fpage" tagx="fpage" title="fpage">263</span><span class="lpage" tagx="lpage" title="lpage">271</span></span></li> <li tag="ref"><a name="CR85" /><span class="label" tagx="label" title="label">85.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singhal</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pundir</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Narang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">A genosensor for detection of consensus DNA sequence of dengue virus using ZnO/Pt-Pd nanocomposites</span><span class="source" tagx="source" title="source">Biosens Bioelectron</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">97</span><span class="fpage" tagx="fpage" title="fpage">75</span><span class="lpage" tagx="lpage" title="lpage">82</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28577500">28577500</a></span></span></li> <li tag="ref"><a name="CR86" /><span class="label" tagx="label" title="label">86.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C-C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Polymerase chain reaction-free detection of hepatitis B virus DNA using a nanostructured impedance biosensor</span><span class="source" tagx="source" title="source">Biosens Bioelectron</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">603</span><span class="lpage" tagx="lpage" title="lpage">608</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26479905">26479905</a></span></span></li> <li tag="ref"><a name="CR87" /><span class="label" tagx="label" title="label">87.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Electrochemical detection of avian influenza virus H5N1 gene sequence using a DNA aptamer immobilized onto a hybrid nanomaterial-modified electrode</span><span class="source" tagx="source" title="source">Electrochim Acta</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">56</span><span class="issue" tagx="issue" title="issue">18</span><span class="fpage" tagx="fpage" title="fpage">6266</span><span class="lpage" tagx="lpage" title="lpage">6270</span></span></li> <li tag="ref"><a name="CR88" /><span class="label" tagx="label" title="label">88.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Development of an electrochemical membrane-based nanobiosensor for ultrasensitive detection of dengue virus</span><span class="source" tagx="source" title="source">Anal Chim Acta</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">725</span><span class="fpage" tagx="fpage" title="fpage">74</span><span class="lpage" tagx="lpage" title="lpage">80</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22502614">22502614</a></span></span></li> <li tag="ref"><a name="CR89" /><span class="label" tagx="label" title="label">89.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krejcova</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">3D printed chip for electrochemical detection of influenza virus labeled with CdS quantum dots</span><span class="source" tagx="source" title="source">Biosens Bioelectron</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">54</span><span class="fpage" tagx="fpage" title="fpage">421</span><span class="lpage" tagx="lpage" title="lpage">427</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24296063">24296063</a></span></span></li> <li tag="ref"><a name="CR90" /><span class="label" tagx="label" title="label">90.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jarocka</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Impedimetric immunosensor for detection of plum pox virus in plant extracts</span><span class="source" tagx="source" title="source">Electroanalysis</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">23</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">2197</span><span class="lpage" tagx="lpage" title="lpage">2204</span></span></li> <li tag="ref"><a name="CR91" /><span class="label" tagx="label" title="label">91.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Label-free detection of influenza viruses using a reduced graphene oxide-based electrochemical immunosensor integrated with a microfluidic platform</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">42771</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28198459">28198459</a></span></span></li> <li tag="ref"><a name="CR92" /><span class="label" tagx="label" title="label">92.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Teixeira, M., et al., Advances in antibiotic nanotherapy: Overcoming antimicrobial resistance, in Emerging Nanotechnologies in Immunology. 2018, Elsevier. p. 233â€"259.</span></li> <li tag="ref"><a name="CR93" /><span class="label" tagx="label" title="label">93.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Development and characterization of aptamer-based enzyme-linked apta-sorbent assay for the detection of Singapore grouper iridovirus infection</span><span class="source" tagx="source" title="source">J Appl Microbiol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">121</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">634</span><span class="lpage" tagx="lpage" title="lpage">643</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27124762">27124762</a></span></span></li> <li tag="ref"><a name="CR94" /><span class="label" tagx="label" title="label">94.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sen</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">H-Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">Design and testing of aptamer-based electrochemical biosensors for proteins and small molecules</span><span class="source" tagx="source" title="source">Bioelectrochemistry</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">77</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19473883">19473883</a></span></span></li> <li tag="ref"><a name="CR95" /><span class="label" tagx="label" title="label">95.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Application of aptamers in virus detection and antiviral therapy</span><span class="source" tagx="source" title="source">Front Microbiol</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1462</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31333603">31333603</a></span></span></li> <li tag="ref"><a name="CR96" /><span class="label" tagx="label" title="label">96.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adamczyk</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Surface plasmon resonance (SPR) as a tool for antibody conjugate analysis</span><span class="source" tagx="source" title="source">Bioconjug Chem</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">1032</span><span class="lpage" tagx="lpage" title="lpage">1037</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10563772">10563772</a></span></span></li> <li tag="ref"><a name="CR97" /><span class="label" tagx="label" title="label">97.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skottrup</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicolaisen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Justesen</span><span class="given-names" tagx="given-names" title="given-names">AF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Towards on-site pathogen detection using antibody-based sensors</span><span class="source" tagx="source" title="source">Biosens Bioelectron</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">24</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">339</span><span class="lpage" tagx="lpage" title="lpage">348</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18675543">18675543</a></span></span></li> <li tag="ref"><a name="CR98" /><span class="label" tagx="label" title="label">98.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bai</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A SPR aptasensor for detection of avian influenza virus H5N1</span><span class="source" tagx="source" title="source">Sensors</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">12506</span><span class="lpage" tagx="lpage" title="lpage">12518</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23112728">23112728</a></span></span></li> <li tag="ref"><a name="CR99" /><span class="label" tagx="label" title="label">99.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">V-T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Highly sensitive sandwich-type SPR based detection of whole H5Nx viruses using a pair of aptamers</span><span class="source" tagx="source" title="source">Biosens Bioelectron</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">293</span><span class="lpage" tagx="lpage" title="lpage">300</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27387259">27387259</a></span></span></li> <li tag="ref"><a name="CR100" /><span class="label" tagx="label" title="label">100.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tombelli</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Aptamer-based biosensors for the detection of HIV-1 tat protein</span><span class="source" tagx="source" title="source">Bioelectrochemistry</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">67</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">135</span><span class="lpage" tagx="lpage" title="lpage">141</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16027048">16027048</a></span></span></li> <li tag="ref"><a name="CR101" /><span class="label" tagx="label" title="label">101.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Optical aptasensors for adenosine triphosphate</span><span class="source" tagx="source" title="source">Theranostics</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">10</span><span class="fpage" tagx="fpage" title="fpage">1683</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27446501">27446501</a></span></span></li> <li tag="ref"><a name="CR102" /><span class="label" tagx="label" title="label">102.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Functionalized magnetic microparticle-based colorimetric platform for influenza a virus detection</span><span class="source" tagx="source" title="source">Nanotechnology</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">27</span><span class="issue" tagx="issue" title="issue">43</span><span class="fpage" tagx="fpage" title="fpage">435102</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27655150">27655150</a></span></span></li> <li tag="ref"><a name="CR103" /><span class="label" tagx="label" title="label">103.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Interface engineering catalytic graphene for smart colorimetric biosensing</span><span class="source" tagx="source" title="source">ACS Nano</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">3142</span><span class="lpage" tagx="lpage" title="lpage">3151</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22443302">22443302</a></span></span></li> <li tag="ref"><a name="CR104" /><span class="label" tagx="label" title="label">104.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nie</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A self-assemble aptamer fragment/target complex based high-throughput colorimetric aptasensor using enzyme linked aptamer assay</span><span class="source" tagx="source" title="source">Talanta</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">106</span><span class="fpage" tagx="fpage" title="fpage">309</span><span class="lpage" tagx="lpage" title="lpage">314</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23598133">23598133</a></span></span></li> <li tag="ref"><a name="CR105" /><span class="label" tagx="label" title="label">105.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Aptamer-based ELISA assay for highly specific and sensitive detection of Zika NS1 protein</span><span class="source" tagx="source" title="source">Anal Chem</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">89</span><span class="issue" tagx="issue" title="issue">23</span><span class="fpage" tagx="fpage" title="fpage">12743</span><span class="lpage" tagx="lpage" title="lpage">12748</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29120623">29120623</a></span></span></li> <li tag="ref"><a name="CR106" /><span class="label" tagx="label" title="label">106.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Escudero-Abarca</span><span class="given-names" tagx="given-names" title="given-names">BI</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Selection, characterization and application of nucleic acid aptamers for the capture and detection of human norovirus strains</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">e106805</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25192421">25192421</a></span></span></li> <li tag="ref"><a name="CR107" /><span class="label" tagx="label" title="label">107.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Dual recognition element lateral flow assay toward multiplex strain specific influenza virus detection</span><span class="source" tagx="source" title="source">Anal Chem</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">89</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">6781</span><span class="lpage" tagx="lpage" title="lpage">6786</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28558471">28558471</a></span></span></li> <li tag="ref"><a name="CR108" /><span class="label" tagx="label" title="label">108.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Development and evaluation of a novel in situ target-capture approach for aptamer selection of human noroviruses</span><span class="source" tagx="source" title="source">Talanta</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">193</span><span class="fpage" tagx="fpage" title="fpage">199</span><span class="lpage" tagx="lpage" title="lpage">205</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30368291">30368291</a></span></span></li> <li tag="ref"><a name="CR109" /><span class="label" tagx="label" title="label">109.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SzakÃ¡cs</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Selective counting and sizing of single virus particles using fluorescent aptamer-based nanoparticle tracking analysis</span><span class="source" tagx="source" title="source">Nanoscale</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">29</span><span class="fpage" tagx="fpage" title="fpage">13942</span><span class="lpage" tagx="lpage" title="lpage">13948</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29845157">29845157</a></span></span></li> <li tag="ref"><a name="CR110" /><span class="label" tagx="label" title="label">110.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Negri</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Direct optical detection of viral nucleoprotein binding to an anti-influenza aptamer</span><span class="source" tagx="source" title="source">Anal Chem</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">84</span><span class="issue" tagx="issue" title="issue">13</span><span class="fpage" tagx="fpage" title="fpage">5501</span><span class="lpage" tagx="lpage" title="lpage">5508</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22687054">22687054</a></span></span></li> <li tag="ref"><a name="CR111" /><span class="label" tagx="label" title="label">111.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xi</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Selection of HBsAg-specific DNA aptamers based on carboxylated magnetic nanoparticles and their application in the rapid and simple detection of hepatitis B virus infection</span><span class="source" tagx="source" title="source">ACS Appl Mater Interfaces</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">21</span><span class="fpage" tagx="fpage" title="fpage">11215</span><span class="lpage" tagx="lpage" title="lpage">11223</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25970703">25970703</a></span></span></li> <li tag="ref"><a name="CR112" /><span class="label" tagx="label" title="label">112.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kirkegaard, J. and N. Rozlosnik, Screen-printed all-polymer aptasensor for impedance based detection of influenza A virus, in Biosensors and Biodetection. 2017, Springer. p. 55â€"70.</span></li> <li tag="ref"><a name="CR113" /><span class="label" tagx="label" title="label">113.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bai</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Aptamer selection and application in multivalent binding-based electrical impedance detection of inactivated H1N1 virus</span><span class="source" tagx="source" title="source">Biosens Bioelectron</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">110</span><span class="fpage" tagx="fpage" title="fpage">162</span><span class="lpage" tagx="lpage" title="lpage">167</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29609164">29609164</a></span></span></li> <li tag="ref"><a name="CR114" /><span class="label" tagx="label" title="label">114.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruslinda</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Effects of diamond-FET-based RNA aptamer sensing for detection of real sample of HIV-1 tat protein</span><span class="source" tagx="source" title="source">Biosens Bioelectron</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">40</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">277</span><span class="lpage" tagx="lpage" title="lpage">282</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22975093">22975093</a></span></span></li> <li tag="ref"><a name="CR115" /><span class="label" tagx="label" title="label">115.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hydrogel based QCM aptasensor for detection of avian influenzavirus</span><span class="source" tagx="source" title="source">Biosens Bioelectron</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">148</span><span class="lpage" tagx="lpage" title="lpage">155</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23202345">23202345</a></span></span></li> <li tag="ref"><a name="CR116" /><span class="label" tagx="label" title="label">116.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ilic</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Craighead</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Virus detection using nanoelectromechanical devices</span><span class="source" tagx="source" title="source">Appl Phys Lett</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">85</span><span class="issue" tagx="issue" title="issue">13</span><span class="fpage" tagx="fpage" title="fpage">2604</span><span class="lpage" tagx="lpage" title="lpage">2606</span></span></li> <li tag="ref"><a name="CR117" /><span class="label" tagx="label" title="label">117.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ginocchio</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Detection of respiratory viruses using non-molecular based methods</span><span class="source" tagx="source" title="source">J Clin Virol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">S11</span><span class="lpage" tagx="lpage" title="lpage">S14</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18162248">18162248</a></span></span></li> <li tag="ref"><a name="CR118" /><span class="label" tagx="label" title="label">118.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somerville</span><span class="given-names" tagx="given-names" title="given-names">LK</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular diagnosis of respiratory viruses</span><span class="source" tagx="source" title="source">Pathology</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">47</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">243</span><span class="lpage" tagx="lpage" title="lpage">249</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25764205">25764205</a></span></span></li> <li tag="ref"><a name="CR119" /><span class="label" tagx="label" title="label">119.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Young, T.P., LABORATORY TESTING STRATEGIES<i>.</i> Fundamentals of HIV Medicine 2017, 2017.</span></li> <li tag="ref"><a name="CR120" /><span class="label" tagx="label" title="label">120.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ginocchio, C.C., Viral infections of the central nervous system, in Molecular Pathology in Clinical Practice. 2007, Springer. p. 437â€"446.</span></li> <li tag="ref"><a name="CR121" /><span class="label" tagx="label" title="label">121.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Landry</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garner</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rapid enterovirus RNA detection in clinical specimens by using nucleic acid sequence-based amplification</span><span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">41</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">346</span><span class="lpage" tagx="lpage" title="lpage">350</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12517871">12517871</a></span></span></li> <li tag="ref"><a name="CR122" /><span class="label" tagx="label" title="label">122.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Shen M, et al. Recent advances and perspectives of nucleic acid detection for coronavirus. J Pharm Anal. 2020;10:97â€"101.</span></li> <li tag="ref"><a name="CR123" /><span class="label" tagx="label" title="label">123.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Xiang, J., et al., Evaluation of Enzyme-Linked Immunoassay and Colloidal Gold-Immunochromatographic Assay Kit for Detection of Novel Coronavirus (SARS-Cov-2) Causing an Outbreak of Pneumonia (COVID-19). medRxiv, 2020.</span></li> <li tag="ref"><a name="CR124" /><span class="label" tagx="label" title="label">124.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Rapid diagnosis of a coronavirus associated with severe acute respiratory syndrome (SARS)</span><span class="source" tagx="source" title="source">Clin Chem</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">49</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">953</span><span class="lpage" tagx="lpage" title="lpage">955</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12765993">12765993</a></span></span></li> <li tag="ref"><a name="CR125" /><span class="label" tagx="label" title="label">125.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Amanat F, et al. A serological assay to detect SARS-CoV-2 seroconversion in humans. Nat Med. 2020;26:1â€"4.</span></li> <li tag="ref"><a name="CR126" /><span class="label" tagx="label" title="label">126.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Snijder, E., E. Decroly, and J. Ziebuhr, The nonstructural proteins directing coronavirus RNA synthesis and processing, in Advances in virus research. 2016, Elsevier. p. 59â€"126.</span></li> <li tag="ref"><a name="CR127" /><span class="label" tagx="label" title="label">127.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corman</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span><span class="source" tagx="source" title="source">Eurosurveillance</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">2000045</span></span></li> <li tag="ref"><a name="CR128" /><span class="label" tagx="label" title="label">128.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kellner</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">SHERLOCK: nucleic acid detection with CRISPR nucleases</span><span class="source" tagx="source" title="source">Nat Protoc</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">10</span><span class="fpage" tagx="fpage" title="fpage">2986</span><span class="lpage" tagx="lpage" title="lpage">3012</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31548639">31548639</a></span></span></li> <li tag="ref"><a name="CR129" /><span class="label" tagx="label" title="label">129.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Layqah</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eissa</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">An electrochemical immunosensor for the corona virus associated with the Middle East respiratory syndrome using an array of gold nanoparticle-modified carbon electrodes</span><span class="source" tagx="source" title="source">Microchim Acta</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">186</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">224</span></span></li> <li tag="ref"><a name="CR130" /><span class="label" tagx="label" title="label">130.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Draz</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shafiee</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Applications of gold nanoparticles in virus detection</span><span class="source" tagx="source" title="source">Theranostics</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">1985</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29556369">29556369</a></span></span></li> <li tag="ref"><a name="CR131" /><span class="label" tagx="label" title="label">131.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">COVID-19: Laboratory Testing Guidelines, Annex 4 of the Prevention and Control Plan Coronavirus Disease 2019 ed. F. edition.</span></li> <li tag="ref"><a name="CR132" /><span class="label" tagx="label" title="label">132.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Organization, W.H., Laboratory testing for coronavirus disease 2019 (COVID-19) in suspected human cases: interim guidance, 2 March 2020. 2020, World Health Organization.</span></li> <li tag="ref"><a name="CR133" /><span class="label" tagx="label" title="label">133.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Organization, W.H., Laboratory testing for coronavirus disease ( COVID-19) in suspected human cases: interim guidance, 19 March 2020. 2020, World Health Organization.</span></li> <li tag="ref"><a name="CR134" /><span class="label" tagx="label" title="label">134.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linton</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Incubation period and other epidemiological characteristics of 2019 novel coronavirus infections with right truncation: a statistical analysis of publicly available case data</span><span class="source" tagx="source" title="source">J Clin Med</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">538</span></span></li> <li tag="ref"><a name="CR135" /><span class="label" tagx="label" title="label">135.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular and serological investigation of 2019-nCoV infected patients: implication of multiple shedding routes</span><span class="source" tagx="source" title="source">Emerg Microbes Infect</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">386</span><span class="lpage" tagx="lpage" title="lpage">389</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32065057">32065057</a></span></span></li> <li tag="ref"><a name="CR136" /><span class="label" tagx="label" title="label">136.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Severe acute respiratory syndrome associated coronavirus is detected in intestinal tissues of fatal cases</span><span class="source" tagx="source" title="source">Am J Gastroenterol</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">100</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">169</span><span class="lpage" tagx="lpage" title="lpage">176</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15654797">15654797</a></span></span></li> <li tag="ref"><a name="CR137" /><span class="label" tagx="label" title="label">137.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Human intestinal tract serves as an alternative infection route for Middle East respiratory syndrome coronavirus</span><span class="source" tagx="source" title="source">Sci Adv</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">11</span><span class="fpage" tagx="fpage" title="fpage">eaao4966</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29152574">29152574</a></span></span></li> <li tag="ref"><a name="CR138" /><span class="label" tagx="label" title="label">138.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Organ distribution of severe acute respiratory syndrome (SARS) associated coronavirus (SARS-CoV) in SARS patients: implications for pathogenesis and virus transmission pathways</span><span class="source" tagx="source" title="source">J Pathol</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">203</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">622</span><span class="lpage" tagx="lpage" title="lpage">630</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15141376">15141376</a></span></span></li> <li tag="ref"><a name="CR139" /><span class="label" tagx="label" title="label">139.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Beck ET, Henrickson KJ. Molecular diagnosis of respiratory viruses. Future Microbiol. 2010;5(6):901â€"16.</span></li> <li tag="ref"><a name="CR140" /><span class="label" tagx="label" title="label">140.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Pang J, et al. Potential rapid diagnostics, vaccine and therapeutics for 2019 novel Coronavirus (2019-ncoV): a systematic review. J Clin Med. 2020;9(3):623.</span></li> <li tag="ref"><a name="CR141" /><span class="label" tagx="label" title="label">141.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="source" tagx="source" title="source">Evaluation of enzyme-linked immunoassay and colloidal gold-Immunochromatographic assay kit for detection of novel coronavirus (SARS-Cov-2) causing an outbreak of pneumonia (COVID-19)</span><span class="year" tagx="year" title="year">2020</span></span></li> <li tag="ref"><a name="CR142" /><span class="label" tagx="label" title="label">142.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">JJG</span></span></span><span class="source" tagx="source" title="source">COVID-19: gastrointestinal manifestations and potential fecal-oral transmission</span><span class="year" tagx="year" title="year">2020</span></span></li> </ul> </div> </div>  </body></html>