<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Cell Res</span><span class="iso-abbrev" title="iso-abbrev">Cell Res</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Cell Research</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1001-0602</span><span class="issn-epub" tagx="issn" title="issn-epub">1748-7838</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer Singapore</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Singapore</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7294525">7294525</a></span><span class="publisher-id" title="publisher-id">356</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41422-020-0356-z">10.1038/s41422-020-0356-z</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Boceprevir, GC-376, and calpain inhibitors II, XII inhibit SARS-CoV-2 viral replication by targeting the viral main protease</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-5766-8189</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">Chunlong</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sacco</span><span class="given-names" tagx="given-names" title="given-names">Michael Dominic</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurst</span><span class="given-names" tagx="given-names" title="given-names">Brett</span></span><a href="#Aff3">3</a><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Townsend</span><span class="given-names" tagx="given-names" title="given-names">Julia Alma</span></span><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Yanmei</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szeto</span><span class="given-names" tagx="given-names" title="given-names">Tommy</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Xiujun</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tarbet</span><span class="given-names" tagx="given-names" title="given-names">Bart</span></span><a href="#Aff3">3</a><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marty</span><span class="given-names" tagx="given-names" title="given-names">Michael Thomas</span></span><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Yu</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">ychen1@usf.edu</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Jun</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">junwang@pharmacy.arizona.edu</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2168 186X</span><span class="institution-id" title="institution-id">grid.134563.6</span><span class="institution" title="institution">Department of Pharmacology and Toxicology, College of Pharmacy, </span><span class="institution" title="institution">The University of Arizona, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2353 285X</span><span class="institution-id" title="institution-id">grid.170693.a</span><span class="institution" title="institution">Department of Molecular Medicine, Morsani College of Medicine, </span><span class="institution" title="institution">University of South Florida, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2185 8768</span><span class="institution-id" title="institution-id">grid.53857.3c</span><span class="institution" title="institution">Institute for Antiviral Research, </span><span class="institution" title="institution">Utah State University, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2185 8768</span><span class="institution-id" title="institution-id">grid.53857.3c</span><span class="institution" title="institution">Department of Animal, Dairy and Veterinary Sciences, </span><span class="institution" title="institution">Utah State University, </span></div></span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2168 186X</span><span class="institution-id" title="institution-id">grid.134563.6</span><span class="institution" title="institution">Department of Chemistry and Biochemistry, </span><span class="institution" title="institution">The University of Arizona, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-6-6</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020-8</span></span><span class="volume" tagx="volume" title="volume">30</span><span class="issue" tagx="issue" title="issue">8</span><span class="fpage" tagx="fpage" title="fpage">678</span><span class="lpage" tagx="lpage" title="lpage">692</span><span class="history" title="history"><span class="received" title="received">received: 2020-5-7</span><span class="accepted" title="accepted">accepted: 2020-5-29</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">A new coronavirus SARS-CoV-2, also called novel coronavirus 2019 (2019-nCoV), started to circulate among humans around December 2019, and it is now widespread as a global pandemic. The disease caused by SARS-CoV-2 virus is called COVID-19, which is highly contagious and has an overall mortality rate of 6.35% as of May 26, 2020. There is no vaccine or antiviral available for SARS-CoV-2. In this study, we report our discovery of inhibitors targeting the SARS-CoV-2 main protease (M<sup>pro</sup>). Using the FRET-based enzymatic assay, several inhibitors including boceprevir, GC-376, and calpain inhibitors II, and XII were identified to have potent activity with single-digit to submicromolar IC<sub>50</sub> values in the enzymatic assay. The mechanism of action of the hits was further characterized using enzyme kinetic studies, thermal shift binding assays, and native mass spectrometry. Significantly, four compounds (boceprevir, GC-376, calpain inhibitors II and XII) inhibit SARS-CoV-2 viral replication in cell culture with EC<sub>50</sub> values ranging from 0.49 to 3.37â€‰ÂµM. Notably, boceprevir, calpain inhibitors II and XII represent novel chemotypes that are distinct from known substrate-based peptidomimetic M<sup>pro</sup> inhibitors. A complex crystal structure of SARS-CoV-2 M<sup>pro</sup> with GC-376, determined at 2.15â€‰Ã… resolution with three protomers per asymmetric unit, revealed two unique binding configurations, shedding light on the molecular interactions and protein conformational flexibility underlying substrate and inhibitor binding by M<sup>pro</sup>. Overall, the compounds identified herein provide promising starting points for the further development of SARS-CoV-2 therapeutics.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Structural biology</div> <div class="kwd" title="kwd">
Molecular biology</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/100006492</span><span class="institution" title="institution">Division of Intramural Research, National Institute of Allergy and Infectious Diseases (Division of Intramural Research of the NIAID)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">AI147325</span><span class="award-id" tagx="award-id" title="award-id">N01-AI-30048</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tarbet</span><span class="given-names" tagx="given-names" title="given-names">Bart</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Jun</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/100008335</span><span class="institution" title="institution">ADHS | Arizona Biomedical Research Commission (ABRC)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">ADHS18-198859</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Jun</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">https://doi.org/10.13039/100006133</span><span class="institution" title="institution">DOE | Advanced Research Projects Agency - Energy (Advanced Research Projects Agency - Energy - U.S. Department of Energy)</span></div> </div><span class="award-id" tagx="award-id" title="award-id">DE-AC02-06CH11357</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Yu</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© Center for Excellence in Molecular Cell Science, CAS 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">An emerging respiratory disease COVID-19 started to circulate among human in December 2019. Since its first outbreak in China from an unknown origin, it quickly became a global pandemic. As of May 26, 2020, there are 343,562 deaths among 5,406,282 confirmed cases in 215 countries (<a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019">https://www.who.int/emergencies/diseases/novel-coronavirus-2019</a>). The etiological pathogen of COVID-19 is a new coronavirus, the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), also called novel coronavirus (nCoV-2019). As the name indicates, SARS-CoV-2 is similar to SARS-CoV, the virus that causes severe acute respiratory symptoms in human and killed 774 people among 8098 infected worldwide in 2003.<sup><a href="#CR1">1</a></sup> SARS-CoV-2 shares ~82% of sequence identity with SARS-CoV and to a less extent with Middle East respiratory syndrome coronavirus (MERS-CoV) (~50%).<sup><a href="#CR2">2</a>,<a href="#CR3">3</a></sup> SARS-CoV-2 is an enveloped, positive-sense, single-stranded RNA virus that belongs to the Î²-lineage of the coronavirus.<sup><a href="#CR4">4</a></sup> The Î²-lineage also contains two other important human pathogens, the SARS-CoV and MERS-CoV. The mortality rate of COVID-19 is around 6.35% as of May 26, 2020, which is lower than that of SARS (~10%) and MERS (~34%).<sup><a href="#CR1">1</a></sup> However, current data indicate that COVID-19 is more contagious and has a larger R0 value than SARS and MERS<sup><a href="#CR5">5</a></sup>, resulting in higher overall death tolls than SARS and MERS. The SARS-CoV-2 virus is currently spreading at an alarming speed in Europe and the United States.</p> <p id="Par3">There is currently no antiviral or vaccine for SARS-CoV-2. The SARS-CoV-2 viral genome encodes a number of structural proteins (e.g., capsid spike glycoprotein), non-structural proteins (e.g., 3-chymotrypsin-like protease (3CL or main protease), papain-like protease, helicase, and RNA-dependent RNA polymerase), and accessary proteins. Compounds that target any of these viral proteins might be potential antiviral drug candidates.<sup><a href="#CR6">6</a>,<a href="#CR7">7</a></sup> In this study, we focus on the viral 3CL protease, also called the main protease (M<sup>pro</sup>), and aim to develop potent M<sup>pro</sup> inhibitors as SAR-CoV-2 antivirals. The M<sup>pro</sup> plays an essential role in coronavirus replication by digesting the viral polyproteins at more than 11 sites, and it appears like a high-profile target for antiviral drug discovery.<sup><a href="#CR8">8</a>â€"<a href="#CR11">11</a></sup> The M<sup>pro</sup> has a unique substrate preference for glutamine at the P1 site (Leu-Glnâ†"(Ser, Ala, Gly)), a feature that is absent in closely related host proteases, suggesting that it is feasible to achieve high selectivity by targeting viral M<sup>pro</sup>. As such, we developed the fluorescence resonance energy transfer (FRET)-based enzymatic assay for the SARS-CoV-2 M<sup>pro</sup> and applied it to screen a focused library of protease inhibitors. Here we report our findings of several novel hits targeting SARS-CoV-2 M<sup>pro</sup> and their mechanism of action. The in vitro antiviral activity of the hits was also evaluated in cell culture using infectious SARS-CoV-2 virus. Overall, our study provides a list of drug candidates for SARS-CoV-2 with a confirmed mechanism of action, and the results might help speed up the drug discovery efforts in combating COVID-19. The compounds identified herein represent some of the most potent and selective SARS-CoV-2 M<sup>pro</sup> inhibitors so far with both enzymatic inhibition and cellular antiviral activity.<sup><a href="#CR8">8</a>,<a href="#CR10">10</a>,<a href="#CR11">11</a></sup> The X-ray crystal structure of SARS-CoV-2 M<sup>pro</sup> with GC-376 showed that the compound can adopt two configurations R and S, offering a molecular explanation of the high binding affinity of the aldehyde-containing inhibitors. Significantly, the discovery of calpain II and XII inhibitors as potent SARS-CoV-2 antivirals suggests that it might be feasible to design dual inhibitors against the viral M<sup>pro</sup> and the host calpains/cathepsins, both of which are important for viral replication.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="establishingthefret-basedassayforthesars-cov-2mpro" title="sec"> <div class="title" tagx="title" title="title">
Establishing the FRET-based assay for the SARS-CoV-2 M<sup>pro</sup></div> <p id="Par4">The M<sup>pro</sup> gene from SARS-CoV-2 strain BetaCoV/Wuhan/WIV04/2019 was inserted into pET-29a(+) vector and expressed in BL21(DE3) <i>Escherichia Coli</i>. with a His-tag in its C-terminus. The M<sup>pro</sup> protein was purified with Ni-NTA column to high purity (Fig.Â <a href="#Fig1">1a</a>). The determined molecular weight of M<sup>pro</sup> is 34,861â€‰Da from native mass spectrometry, which matches the calculated molecular weight of M<sup>pro</sup> without the N-terminal methionine (Supplementary information, Fig.Â <a href="#MOESM1">S1</a>). The first methionine might be cleaved by the methionine aminopeptidase as reported before.<sup><a href="#CR12">12</a></sup> To establish the FRET assay condition, we designed a FRET based substrate using the sequence between viral polypeptide NSP4-NSP5 junction from SARS-CoV-2: Dabcyl-KTSAVLQ/SGFRKME(Edans).<sup><a href="#CR8">8</a></sup> We then tested the M<sup>pro</sup> proteolytic activity in buffers with different pH. We found that M<sup>pro</sup> displays the highest activity in pH 6.5 buffer (Fig.Â <a href="#Fig1">1b</a>), which contains 20â€‰mM HEPES, 120â€‰mM NaCl, 0.4â€‰mM EDTA, 4â€‰mM 1,4-dithiothreitol (DTT) and 20% glycerol. As such, all the following proteolytic assay was conducted using this pH 6.5 buffer. A standard curve was generated to convert the relative fluorescence unit (RFU) to the amount of the cleaved substrate (nM) such that k<sub>cat</sub> can be calculated (Fig.Â <a href="#Fig1">1c</a>). Next, we characterized the enzymatic activity of this SARS-CoV-2 M<sup>pro</sup> by measuring the <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub> values. When 200â€‰nM M<sup>pro</sup> was mixed with various concentration of FRET substrate (0â€"100â€‰ÂµM), the initial velocity was measured and plotted against substrate concentration. Curve fitting with Michaelisâ€"Menten equation gave the best-fit values of <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub> as 28.2â€‰Â±â€‰3.4â€‰ÂµM and 31.7â€‰Â±â€‰1.6 RFU/s, respectively (Fig.Â <a href="#Fig1">1d</a>). The calculated <i>k</i><sub>cat</sub>/<i>K</i><sub>m</sub> was 5,624â€‰s<sup>âˆ’1</sup>â€‰M<sup>âˆ’1</sup>, which is similar to the previously reported value of 3,426.1â€‰s<sup>âˆ’1</sup>â€‰M<sup>âˆ’1</sup> by Hilgenfeld et al.<sup><a href="#CR8">8</a></sup>, suggesting that the C-terminal His-tag M<sup>pro</sup> is as active as the native M<sup>pro</sup>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <div class="title" tagx="title" title="title">
SARS-CoV-2 M<sup>pro</sup> expression and characterization.</div> <p><b>a</b> SDS-PAGE of His-tagged M<sup>pro</sup> (lane 1); Lane M, protein ladder; the calculated molecular weight of the His-tagged M<sup>pro</sup> is 34,861â€‰Da. <b>b</b> Reaction buffer optimization: 250â€‰nM His-tagged M<sup>pro</sup> was diluted into three reaction buffers with different pH values. <b>c</b> An Edans standard curve was generated to convert RFU to the amount of the cleaved substrate (nM). <b>d</b> Michaelisâ€"Menten plot of 200â€‰nM His-tagged M<sup>pro</sup> with various concentrations of FRET substrate in pH 6.5 reaction buffer. The best-fit <i>V</i><sub>max</sub>â€‰=â€‰31.7â€‰Â±â€‰1.6â€‰nM/s; <i>K</i><sub>m</sub>â€‰=â€‰28.2â€‰Â±â€‰3.4â€‰ÂµM, and the calculated <i>k</i><sub>cat</sub>/<i>K</i><sub>m</sub>â€‰=â€‰5,624â€‰s<sup>âˆ’1</sup>â€‰M<sup>âˆ’1</sup>.</p>   </div> <p /> </div> <div class="primaryscreeningofafocusedproteaseinhibitorlibraryagainstthesars-cov-2mpro" title="sec"> <div class="title" tagx="title" title="title">
Primary screening of a focused protease inhibitor library against the SARS-CoV-2 M<sup>pro</sup></div> <p id="Par5">With the established FRET assay condition, we screened a collection of protease inhibitors from the Selleckchem bioactive compound library to identify potential SARS-CoV-2 M<sup>pro</sup> inhibitors (TableÂ <a href="#Tab1">1</a>). The protease inhibitors are grouped based on their targets and mechanism of action, including proteasome inhibitors (<b>1</b>â€"<b>8</b>), HIV protease inhibitors (<b>9</b>â€"<b>14</b>), Î³-secretase inhibitors (<b>15</b>â€"<b>22</b>), HCV NS3â€"4A protease inhibitors (<b>23</b>â€"<b>29</b>), DPP-4 inhibitors (<b>30</b>â€"<b>35</b>), miscellaneous serine protease inhibitors (<b>36</b>â€"<b>39</b>), cathepsin and calpain protease inhibitors (<b>40</b>â€"<b>43</b>), miscellaneous cysteine protease inhibitors (<b>44</b>â€"<b>48</b>), matrix metalloprotease inhibitors (<b>49</b>â€"<b>51</b>), and miscellaneous protease inhibitors (<b>52</b>â€"<b>55</b>). The inhibitors were pre-incubated with 100â€‰nM of M<sup>pro</sup> at 30â€‰Â°C for 30â€‰min in the presence of 4â€‰mM DTT before the addition of 10â€‰ÂµM FRET substrate. The addition of DTT was to quench non-specific thiol reactive compounds and to ensure that M<sup>pro</sup> is in the reducing condition. All compounds were tested at 20â€‰ÂµM, except compound <b>26</b>, which was tested at 2â€‰ÂµM due to its fluorescent background. Encouragingly, four inhibitors (<b>24</b>, <b>28</b>, <b>29</b> and <b>43</b>) showed more than 60% inhibition against M<sup>pro</sup> at 20â€‰ÂµM (Fig.Â <a href="#Fig2">2</a>). Among the hits, simeprevir (<b>24</b>), boceprevir (<b>28</b>), and narlaprevir (<b>29</b>) are HCV NS3â€"4A serine protease inhibitors, and compound MG-132 (<b>43</b>) inhibits both proteasome and calpain. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>List of protease inhibitors tested against SARS-CoV-2 M<sup>pro</sup> in the primary FRET assay.</p>  <table frame="hsides" rules="groups"> <tbody> <tr> <td colspan="2"><span class="inline-graphic" title="inline-graphic" /></td> </tr> <tr> <td colspan="2"><span class="inline-graphic" title="inline-graphic" /></td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <div class="title" tagx="title" title="title">
Screening of known protease inhibitors against SARS-CoV-2 M<sup>pro</sup> using the FRET assay.</div> <p>20â€‰ÂµM of compounds (<b>26</b> was tested at 2â€‰ÂµM) was pre-incubated with 100â€‰nM of SARS-CoV-2 M<sup>pro</sup> for 30â€‰min at 30â€‰Â°C, and then 10â€‰ÂµM FRET substrate was added to reaction mixture to initiate the reaction. The reaction was monitored for 2â€‰h. The initial velocity was calculated by linear regression using the data points from the first 15â€‰min of the reaction. The calculated initial velocity with each compound was normalized to DMSO control. The results are averageâ€‰Â±â€‰standard deviation of two repeats.</p>   </div> <p /> </div> <div class="secondaryscreeningofafocusedlibraryofcalpain/cathepsininhibitorsandknownviral3clproinhibitors" title="sec"> <div class="title" tagx="title" title="title">
Secondary screening of a focused library of calpain/cathepsin inhibitors and known viral 3CL<sup>pro</sup> inhibitors</div> <p id="Par6">Given the encouraging results from the primary screening, we then further characterized the four hits (<b>24</b>, <b>28</b>, <b>29</b>, and <b>43</b>) in a combination of assays including dose-response titration, thermal shift binding assay (TSA), and counter screening assays with two other viral cysteine proteases, the enterovirus A71 (EV-A71) 2A and 3C proteases, both of which are cysteine proteases (TableÂ <a href="#Tab2">2</a>). The HCV NS3â€"4A protease inhibitors boceprevir (<b>28</b>) and narlaprevir (<b>29</b>) inhibited M<sup>pro</sup> with 50% inhibitory concentration (IC<sub>50</sub>) values of 4.13 and 4.73â€‰ÂµM, respectively (TableÂ <a href="#Tab2">2</a>), more potent than simeprevir (<b>24</b>) (IC<sub>50</sub>â€‰=â€‰13.74â€‰ÂµM). Both compounds (<b>28</b> and <b>29</b>)Â also showed strong binding towards M<sup>pro</sup> and shifted the melting temperature of the protein (Î"T<sub>m</sub>) by 6.67â€‰Â°C and 5.18â€‰Â°C, respectively, at 40â€‰ÂµM. Despite their potent inhibition against the HCV NS3â€"4A serine protease and the SARS-CoV-2 cysteine M<sup>pro</sup>, boceprevir (<b>28</b>) and narlaprevir (<b>29</b>) did not inhibit the EV-A71 2A and 3C proteases (IC<sub>50</sub>â€‰&amp;gt;â€‰20â€‰ÂµM), suggesting that they are not non-specific cysteine protease inhibitors. The calpain inhibitor MG-132 (<b>43</b>) had an IC<sub>50</sub> value of 3.90â€‰ÂµM against M<sup>pro</sup>, and was not active against the EV-A71 2A and 3C proteases (IC<sub>50</sub>â€‰&amp;gt;â€‰20â€‰ÂµM). The binding of MG-132 (<b>43</b>) to M<sup>pro</sup> was also confirmed in the TSA assay with a Î"T<sub>m</sub> of 4.02â€‰Â°C. <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>Characterization of HCV and calpain/cathepsin protease inhibitors against SARS-CoV-2 M<sup>pro</sup> using a consortium of secondary assays<sup>a</sup>.</p>  <table frame="hsides" rules="groups"> <tbody> <tr> <td colspan="2"><span class="inline-graphic" title="inline-graphic" /></td> </tr> <tr> <td colspan="2"><span class="inline-graphic" title="inline-graphic" /></td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup>Valueâ€‰=â€‰meansâ€‰Â±â€‰SE from three independent experiments.</p> <p><sup>b</sup>N.T.â€‰=â€‰not tested.</p> <p><sup>c</sup>The IC<sub>50</sub> of PSI (<b>64</b>) on SARS CoV-2 M<sup>pro</sup> was calculated by end point reading of 1â€‰h digestion, instead of the initial velocity.</p> </div>  <p /> <p id="Par7">In light of the promising results of the calpain inhibitor MG-132 (<b>43</b>), we then pursued testing for other calpain and cathepsin inhibitors that are commercially available (<b>56</b>â€"<b>63</b>) (TableÂ <a href="#Tab2">2</a>). These compounds were not included in the initial library because they have not been advanced to clinical studies. Among this series of analogs, calpain inhibitor II (<b>61</b>) and XII (<b>62</b>) are the most potent M<sup>pro</sup> inhibitors with IC<sub>50</sub> values of 0.97 and 0.45â€‰ÂµM, respectively. Binding of compounds (<b>61</b> and <b>62</b>)Â to M<sup>pro</sup> shifted the melting curve of the protein by 6.65 and 7.86â€‰Â°C, respectively. Encouragingly, both compounds (<b>61</b> and <b>62</b>)Â did not inhibit the EV-A71 2A and 3C proteases (IC<sub>50</sub>â€‰&amp;gt;â€‰20â€‰ÂµM). Calpain inhibitor I (<b>59</b>) and MG-115 (<b>60</b>) also showed potent inhibition against M<sup>pro</sup> with IC<sub>50</sub> values of 8.60 and 3.14â€‰ÂµM, respectively. Calpeptin (<b>56</b>) and PSI (<b>63</b>) had moderate activity against M<sup>pro</sup> with IC<sub>50</sub> values of 10.69 and 10.38â€‰ÂµM, respectively. In contrast, calpain inhibitors III (<b>57</b>) and VI (<b>58</b>) were not active (IC<sub>50</sub>â€‰&amp;gt;â€‰20â€‰ÂµM).</p> <p id="Par8">We also included two well-known viral 3CL protease inhibitors GC-376 (<b>64</b>) and rupintrivir (<b>65</b>) in the secondary screening. GC-376 (<b>64</b>) is an investigational veterinary drug that is being developed for feline infectious peritonitis (FIP).<sup><a href="#CR13">13</a>,<a href="#CR14">14</a></sup> GC-376 (<b>64</b>) was designed to target the viral 3CL protease and had potent antiviral activity against multiple viruses including MERS-CoV, FIPV, and norovirus.<sup><a href="#CR13">13</a>,<a href="#CR15">15</a></sup> Rupintrivir (<b>65</b>) was developed as a rhinovirus antiviral by targeting the viral 3CL protease, but it was discontinued in clinical trials due to side effects.<sup><a href="#CR16">16</a></sup> In our study, we found that GC-376 (<b>64</b>) was the most potent M<sup>pro</sup> inhibitor with an IC<sub>50</sub> value of 0.03â€‰ÂµM. It shifted the melting curve of M<sup>pro</sup> by 18.30â€‰Â°C upon binding. In contrast, rupintrivir (<b>65</b>) was not active against M<sup>pro</sup> (IC<sub>50</sub>â€‰&amp;gt;â€‰20â€‰ÂµM). Previous report also showed that rupintrivir was not active against the SARS-CoV 3CL<sup>pro</sup> (M<sup>pro</sup>) (IC<sub>50</sub>â€‰&amp;gt;â€‰100â€‰ÂµM).<sup><a href="#CR17">17</a></sup> Both compounds (<b>64</b> and <b>65</b>)Â were inactive against the EV-A71 2A protease, but showed potent inhibition against the EV-A71 3C protease, which is consistent with previously reported results.<sup><a href="#CR15">15</a>,<a href="#CR18">18</a>,<a href="#CR19">19</a></sup></p> <p id="Par9">When plotting the IC<sub>50</sub> values (log scale) of the inhibitors against M<sup>pro</sup> from the FRET enzymatic assay with the melting temperature shifts (Î"T<sub>m</sub>) from TSA (Fig.Â <a href="#Fig3">3a</a>), a linear correlation was observed, and the r<sup>2</sup> of the linear regression fitting is 0.94. This suggests that there is a direct correlation between the enzymatic inhibition and protein binding: a more potent enzyme inhibitor also binds to the protein with higher affinity. The stabilization of the M<sup>pro</sup> against thermal denaturation was also compound concentration dependent (Fig.Â <a href="#Fig3">3b</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <div class="title" tagx="title" title="title">
Binding of inhibitors to SARS-CoV-2 M<sup>pro</sup> using thermal shift binding assay and native mass spectrometry.</div> <p><b>a</b> Correlation of inhibition efficacy (IC<sub>50</sub>) with Î"T<sub>m</sub> from thermal shift binding assay. Data in TableÂ <a href="#Tab2">2</a> were used for the plot. The r<sup>2</sup> of fitting is 0.94. <b>b</b> Dose-dependent melting temperature (T<sub>m</sub>) shift. Native MS reveals binding of SARS-CoV-2 M<sup>pro</sup> to GC-376 (<b>64</b>) (<b>c</b>), calpain inhibitor II (<b>61</b>) (<b>d</b>), calpain inhibitor XII (<b>62</b>) (<b>e</b>), and boceprevir (<b>28</b>) (<b>f</b>). All ligand concentrations are 12.5â€‰ÂµM except (<b>e</b>), which is 25â€‰ÂµM. Peaks are annotated for dimer (blue circle), dimer with one bound ligand (yellow down triangle), and dimer with two bound ligands (red up triangle). Other minor signals are truncated dimers, which bind ligands at the same ratios. Charge states are annotated in <b>c</b>, and insets show the deconvolved zero-charge mass distribution.</p>   </div> <p /> <p id="Par10">The binding of the four most potent inhibitors boceprevir (<b>28</b>), calpain inhibitors II (<b>61</b>), XII (<b>62</b>), and GC-376 (<b>64</b>) to SARS-CoV-2 M<sup>pro</sup> was further characterized by native mass spectrometry (MS) (Fig.Â <a href="#Fig3">3câ€"f</a>). Native MS analysis showed that M<sup>pro</sup> formed a dimer complex with a mass of 69,722â€‰Da, indicating that the extra N-terminal methionine in our M<sup>pro</sup> construct was cleaved, yielding the native M<sup>pro</sup> N-terminus with a serine residue (Supplementary information, Fig.Â <a href="#MOESM1">S1</a>). A small amount of protomer and dimer with a single C-terminal truncation of the His-tag was also observed, but the intact dimer was the predominant signal (Supplementary information, Fig.Â <a href="#MOESM1">S1</a>). Addition of all four ligands tested, boceprevir (<b>28</b>), calpain inhibitors II (<b>61</b>), XII (<b>62</b>), and GC-376 (<b>64</b>), showed binding of up to two ligands per dimer (Fig.Â <a href="#Fig3">3câ€"f</a>), suggesting a binding stoichiometry of one drug per protomer.</p> </div> <div class="mechanismofactionofhits" title="sec"> <div class="title" tagx="title" title="title">
Mechanism of action of hits</div> <p id="Par11">To elucidate the mechanism of action of hits against SARS-CoV-2 M<sup>pro</sup>, we focus on five most potent compounds prioritized from the primary and secondary screenings including boceprevir (<b>28</b>), MG-132 (<b>43</b>), calpain inhibitor II (<b>61</b>), calpain inhibitor XII (<b>62</b>), and GC-376 (<b>64</b>). For this, we performed enzyme kinetic studies with different concentrations of inhibitors (Fig.Â <a href="#Fig4">4</a>). A biphasic enzymatic progression curve in the presence but not absence of inhibitor is typically a hallmark for a slow covalent binding inhibitor. In Fig.Â <a href="#Fig4">4</a>, the left column shows the progression curves up to 4â€‰h. Biphasic progression curves were observed for all 5 inhibitors at high drug concentrations. Significant substrate depletion was observed when the proteolytic reaction proceeded beyond 90â€‰min. We therefore chose the first 90â€‰min of the progression curves for curve fitting (Fig.Â <a href="#Fig4">4</a>, middle column). We fit the progression curves in the presence of different concentrations of GC-376 (<b>64</b>) with the two-step Morrison equation (Eq. (<a href="#Equ3">3</a>) in â€œMaterials and methodsâ€� section). GC-376 (<b>64</b>) binds to SARS-CoV-2 M<sup>pro</sup> with an equilibrium dissociation constant for the inhibitor (K<sub>I</sub>) of 59.9â€‰Â±â€‰21.7â€‰nM in the first step. After initial binding, a covalent bond is formed at a slower velocity between GC-376 (<b>64</b>) and M<sup>pro</sup> with the second reaction rate constant (k<sub>2</sub>) being 0.00245â€‰Â±â€‰0.00047â€‰s<sup>âˆ’1</sup>, resulting in an overall k<sub>2</sub>/K<sub>I</sub> value of 4.08â€‰Ã—â€‰10<sup>4</sup>â€‰M<sup>âˆ’1</sup>â€‰s<sup>âˆ’1</sup> (Fig.Â <a href="#Fig4">4a</a>). However, when we tried to fit the proteolytic progression curves for boceprevir (<b>28</b>), MG-132 (<b>43</b>), calpain inhibitors II (<b>61</b>) and XII (<b>62</b>) using the same two-step reaction mechanism, we could not obtain accurate values for the second rate constant k<sub>2</sub>. This is presumably due to significant substrate depletion before the equilibrium between EI and EI*, leading to very small values of k<sub>2</sub>. Accordingly, for these four inhibitors (<b>28</b>, <b>43</b>, <b>61</b>, and <b>62</b>),Â only the dissociation constant K<sub>I</sub> values from the first step were determined (Fig.Â <a href="#Fig4">4bâ€"e</a>). The inhibition constants (K<sub>I</sub>) for boceprevir (<b>28</b>), MG-132 (<b>43</b>), calpain inhibitors II (<b>61</b>) and XII (<b>62</b>) are 1.18â€‰Â±â€‰0.10â€‰ÂµM, 1.57â€‰Â±â€‰0.13â€‰ÂµM, 0.40â€‰Â±â€‰0.02â€‰ÂµM, and 0.13â€‰Â±â€‰0.02â€‰ÂµM, respectively. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <div class="title" tagx="title" title="title">
Proteolytic reaction progression curves of M<sup>pro</sup> in the presence or the absence of compounds.</div> <p><b>aâ€"e</b> In the kinetic studies, 5â€‰nM M<sup>pro</sup> was added to a solution containing various concentrations of protease inhibitors and 20â€‰ÂµM FRET substrate to initiate the reaction. The reaction was then monitored for 4â€‰h. The left column shows the reaction progression up to 4â€‰h; middle column shows the progression curves for the first 90â€‰min, which were used for curve fitting to generate the plot shown in the right column. Detailed methods were described in â€œMaterials and methodsâ€� section. GC-376 (<b>64</b>) (<b>a</b>); boceprevir (<b>28</b>) (<b>b</b>); MG-132 (<b>43</b>) (<b>c</b>); calpain inhibitor II (<b>61</b>) (<b>d</b>); calpain inhibitor XII (<b>62</b>) (<b>e</b>).</p>   </div> <p /> </div> <div class="cellularantiviralactivityandcytotoxicityofhits" title="sec"> <div class="title" tagx="title" title="title">
Cellular antiviral activity and cytotoxicity of hits</div> <p id="Par12">To test the hypothesis that inhibiting the enzymatic activity of M<sup>pro</sup> will lead to the inhibition of SARS-CoV-2 viral replication, we performed cellular antiviral assays for the five promising hits (<b>64</b>, <b>28</b>, <b>43</b>, <b>61</b>, and <b>62</b>)Â against SARS-CoV-2. For this, we first tested the cellular cytotoxicity of these compounds in multiple cell lines (Supplementary information, TableÂ <a href="#MOESM6">S1</a>). GC-376 (<b>64</b>), boceprevir (<b>28</b>), and calpain inhibitor II (<b>61</b>) were well tolerated and had CC<sub>50</sub> values of over 100â€‰ÂµM for all the cell lines tested. MG-132 (<b>43</b>) was cytotoxic to all the cells with CC<sub>50</sub> values less than 1â€‰ÂµM except A549 cells. Calpain inhibitor XII (<b>62</b>) had acceptable cellular cytotoxicity with CC<sub>50</sub> values above 50â€‰ÂµM for all the cell lines tested (Supplementary information, TableÂ <a href="#MOESM6">S1</a>).</p> <p id="Par13">Next, we chose four compounds boceprevir (<b>28</b>), calpain inhibitors II (<b>61</b>), XII (<b>62</b>), and GC-376 (<b>64</b>) for the antiviral assay with infectious SARS-CoV-2. MG-132 (<b>43</b>) was not included due to its cytotoxicity. Gratifyingly, all four compounds showed potent antiviral activity against SARS-CoV-2 in the primary viral cytopathic effect (CPE) assay with 50% effective concentration (EC<sub>50</sub>) values ranging from 0.49 to 3.37â€‰ÂµM (Fig.Â <a href="#Fig5">5aâ€"d</a> and TableÂ <a href="#Tab3">3</a>). Their antiviral activity was further confirmed in the secondary viral yield reduction (VYR) assay (Fig.Â <a href="#Fig5">5eâ€"h</a> and TableÂ <a href="#Tab3">3</a>). The most potent compound was calpain inhibitor XII (<b>62</b>), which showed an EC<sub>50</sub> of 0.49â€‰ÂµM in the primary CPE assay and an EC<sub>50</sub> of 0.78â€‰ÂµM in the secondary VYR assay (TableÂ <a href="#Tab3">3</a>; Supplementary information, Fig.Â <a href="#MOESM2">S2</a>). In comparison, remdesivir was reported to inhibit SARS-CoV-2 in the VYR assay with an EC<sub>50</sub> of 0.77â€‰ÂµM.<sup><a href="#CR20">20</a></sup> None of the compounds inhibited the unrelated influenza virus A/California/07/2009 (H1N1) virus (EC<sub>50</sub>â€‰&amp;gt;â€‰20â€‰ÂµM) (Supplementary information, TableÂ <a href="#MOESM6">S1</a>), suggesting that the antiviral activity of the four compounds (boceprevir, calpain inhibitors II, XII, and GC-376) against SARS-CoV-2 is specific. In comparison with recently reported SARS-CoV-2 M<sup>pro</sup> inhibitors (TableÂ <a href="#Tab3">3</a>), the hits identified herein represent some of the most potent and selective drug candidates with broad chemical diversity. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <div class="title" tagx="title" title="title">
Antiviral potency of boceprevir (28), calpain inhibitors II (61), XII (62), and GC-376 (64), in CPE and VYR assays.</div> <p><b>a</b>â€"<b>d</b> Primary CPE assay results of boceprevir (<b>28</b>), calpain inhibitors II (<b>61</b>), XII (<b>62</b>), and GC-376 (<b>64</b>). <b>e</b>â€"<b>h</b> Secondary VYR assay results of boceprevir (<b>28</b>), calpain inhibitors II (<b>61</b>), XII (<b>62</b>), and GC-376 (<b>64</b>). Each dot represents the average of three repeats from one experiment, and each assay was repeated in three different experiments. The curve fittings were shown in Supplementary information, Fig. <a href="#MOESM2">S2</a>.</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> <p>Antiviral activity of hits against SARS-CoV-2 and the comparison with recently reported M<sup>pro</sup> inhibitors.</p>  <table frame="hsides" rules="groups"> <tbody> <tr> <td colspan="2"><span class="inline-graphic" title="inline-graphic" /></td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><i>N.A.</i> not available.</p> <p><sup>a</sup>CPE EC<sub>50</sub>, VYR EC<sub>90</sub>, and cytotoxicity CC<sub>50</sub> values are meansâ€‰Â±â€‰SD of 3 independent experiments.</p> <p><sup>b</sup>Results were retrieved from recent publications.<sup><a href="#CR8">8</a>,<a href="#CR10">10</a>,<a href="#CR11">11</a></sup></p> <p><sup>c</sup>The antiviral activity of ebselen was determined in plaque assay.</p> </div> </div> <p /> </div> <div class="complexcrystalstructureofsars-cov-2mprowithgc-376(64)" title="sec"> <div class="title" tagx="title" title="title">
Complex crystal structure of SARS-CoV-2 M<sup>pro</sup> with GC-376 (64)</div> <p id="Par14">The crystal structure of the SARS-CoV-2 M<sup>pro</sup> in complex with GC-376 (<b>64</b>) was solved in the P3<sub>2</sub>21 space group at 2.15â€‰Ã… resolution (PDB: 6WTT) (Supplementary information, TableÂ <a href="#MOESM7">S2</a>). There are three protomers per asymmetric unit (ASU) with two constituting a biological dimer and the third forming a dimer with a crystallographic symmetry related neighboring protomers (Supplementary information, Fig.Â <a href="#MOESM3">S3</a>). The presence of three protomers in our crystal structure allowed us to capture different binding configurations of GC-376 (<b>64</b>) (Fig.Â <a href="#Fig6">6</a>), a unique feature that was not observed in previous X-ray crystal structures.<sup><a href="#CR8">8</a>,<a href="#CR10">10</a>,<a href="#CR11">11</a></sup> The pairwise r.m.s.d among the protomer backbone CÎ± atoms ranges from 0.435 to 0.564â€‰Ã…. Previously, SARS-CoV M<sup>pro</sup> and SARS-CoV-2 M<sup>pro</sup> crystal structures have been solved most frequently as one protomer per ASU, and occasionally a dimer.<sup><a href="#CR8">8</a>,<a href="#CR10">10</a>,<a href="#CR11">11</a>,<a href="#CR21">21</a>,<a href="#CR22">22</a></sup> In its native state, M<sup>pro</sup> requires dimerization to become catalytically active<sup><a href="#CR23">23</a>,<a href="#CR24">24</a></sup>, which is supported by our native MS data (Supplementary information, Fig.Â <a href="#MOESM1">S1</a>). In our crystal structures, all three protomers appear catalytically competent, with the third protomer activated by the N-finger from an adjacent ASU. The well-defined electron density also clearly shows a serine at the protease N-terminus, again indicating that the first methionine in the protease construct was cleaved (Supplementary information, Fig.Â <a href="#MOESM3">S3b</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <div class="title" tagx="title" title="title">
Molecular recognition of GC-376 (64) by SARS-CoV-2 M<sup>pro</sup>.</div> <p>Complex of SARS-CoV-2 M<sup>pro</sup> and GC-376 (<b>64</b>) with protomer A (<b>a</b>, <b>b</b>) and protomer C (<b>c</b>, <b>d</b>). Unbiased F<sub>o</sub>â€"F<sub>c</sub> map, shown in gray, is contoured at 2Ïƒ. Hydrogen bonds are shown as red dashed lines. <b>e</b> Surface representation of SARS-CoV-2 M<sup>pro</sup>â€‰+â€‰GC-376 (<b>64</b>) (white) superimposed with the SARS-CoV M<sup>pro</sup> natural, N-terminal substrate (PDB ID: 2Q6G, with residues P1â€²â€"P4 in different colors). The SARS-CoV-2 M<sup>pro</sup> cleaves between the P1â€² and P1 residues. <b>f</b> Superimposition of the three protomers in the asymmetric subunit of SARS-CoV-2 M<sup>pro</sup> with GC-376 (<b>64</b>). Significant conformational flexibility is observed, particularly in the TSEDMLN loop.</p>   </div> <p /> <p id="Par15">GC-376 (<b>64</b>) forms an extensive network of hydrogen bonds with the active site while also exhibiting excellent geometric complementarity (Fig.Â <a href="#Fig6">6</a>). These interactions are coupled with the thermodynamic payoff of covalent adduct formation between the aldehyde bisulfite warhead and Cys145, making GC-376 (<b>64</b>) one of the most potent SARS-CoV-2 M<sup>pro</sup> inhibitors in vitro with an IC<sub>50</sub> of 0.030â€‰Â±â€‰0.008â€‰ÂµM (TableÂ <a href="#Tab3">3</a>). Along with other known M<sup>pro</sup> inhibitors (<b>N3</b>, <b>13b</b>, <b>11a</b> and <b>11b</b>)Â (TableÂ <a href="#Tab3">3</a>), GC-376 (<b>64</b>) mimics the peptide substrate that is cleaved by this enzyme (Fig.Â <a href="#Fig6">6e</a>).<sup><a href="#CR8">8</a>,<a href="#CR22">22</a>,<a href="#CR25">25</a>,<a href="#CR26">26</a></sup> The glutamine surrogate Î³-lactam ring is a cyclized derivative of the P1 glutamine side chain that normally occupies the S1 site; here it forms hydrogen bonds with the His163 and Glu166 side chains and the main chain of Phe140 (Fig.Â <a href="#Fig6">6a, c</a>). An amide bond connects the Î³-lactam side chain to an isobutyl moiety that embeds itself in the hydrophobic S2 site formed by His41, Met49, and Met169. Normally, this S2 site in SARS-CoV-2 M<sup>pro</sup> can accommodate a variety of hydrophobic substitutions such as isobutyl in GC-376 (<b>64</b>) and <b>N3</b>),Â cyclopropyl in (<b>13b</b>),Â cyclohexyl in (<b>11a</b>),Â and 3-fluorophenyl in (<b>11b</b>)Â (TableÂ <a href="#Tab3">3</a>).<sup><a href="#CR27">27</a>,<a href="#CR28">28</a></sup> A carbamate bond in GC-376 (<b>64</b>), which forms hydrogen bonds with the main chain of Glu166 and the side chain of Gln189, connects the P2 isobutyl group to a phenylmethyl ester that interacts with the aliphatic S4 site. Compared with previous inhibitors, the phenylmethyl ester of GC-376 (<b>64</b>) exhibits high complementarity with the S4 site, and the extensive non-polar interactions may contribute significantly to the potency of this compound (Fig.Â <a href="#Fig6">6e</a>).</p> <p id="Par16">Three copies of GC-376 (<b>64</b>) were found in the crystal structure, one in each protomer active site (Fig.Â <a href="#Fig6">6a, c</a>; Supplementary information, Fig.Â <a href="#MOESM4">S4</a>). The configurations of GC-376 (<b>64</b>) were consistent in protomers A and B, where the thioacetal hydroxide is positioned in the â€œoxyanion holeâ€� formed by the backbone amides of Gly143, Ser144, and Cys145 (Fig.Â <a href="#Fig6">6a</a>; Supplementary information, Fig.Â <a href="#MOESM4">S4</a>), resulting in the (S)-configuration. It is noted that aldehydes (<b>11a</b> and <b>11b</b>)Â also bind the active site of SARS-CoV-2 M<sup>pro</sup> in the (S)-configuration (PDB: 6M0K and 6LZE) (Supplementary information, Fig.Â <a href="#MOESM5">S5a</a>).<sup><a href="#CR11">11</a></sup> In protomer C, however, the same hydroxide group orients outwards from the oxyanion hole, forming hydrogen bonds with His41 (Fig.Â <a href="#Fig6">6c</a>), which gives the (R)-configuration. This (R)-configuration is consistent with the binding mode of Î±-ketoamide (<b>13b</b>)Â in the active site of SARS-CoV-2 M<sup>pro</sup> (PDB: 6Y2F) (Supplementary information, Fig.Â <a href="#MOESM5">S5b</a>).<sup><a href="#CR8">8</a></sup> These two unique configurations R and S might be a result of the Cys145 thiol nucleophilic attacking the aldehyde of GC-376 (<b>64</b>) from two different faces (Fig.Â <a href="#Fig6">6b, d</a>). The fact that GC-376 (<b>64</b>) can adopt two different configurations R and S upon binding to the active site might explain its high binding affinity towards the target.</p> <p id="Par17">An additional difference between the configurations of GC-376 (<b>64</b>) in protomers A, B and C is observed in the orientation of the phenylmethyl ester. In protomer C, the CH<sub>2</sub> of the phenylmethyl points toward the main chain of Leu167 in a â€œ<i>cis</i>â€� conformation (Fig.Â <a href="#Fig6">6c</a>), whereas in protomers A and B this same CH<sub>2</sub> points downwards in a â€œ<i>trans</i>â€� conformation (Fig.Â <a href="#Fig6">6a</a>; Supplementary information, Fig.Â <a href="#MOESM4">S4</a>). Consequently, this influences the rotameric configuration of the Leu167 isobutyl moiety, where a rotational adjustment of 180Â° occurs at its â€œÎ²â€� carbon. Furthermore, large rearrangements are observed in the flexible loop that consists of residues 45â€"51 (TSEDMLN in SARS-CoV-2 M<sup>pro</sup>) and forms the S2 and S3â€² subsites (Fig.Â <a href="#Fig6">6f</a>), explaining the broad substrate scope in the P2 site (TableÂ <a href="#Tab3">3</a>). The loop conformations in protomers B and C may be influenced by crystal packing interactions with protomers from adjacent ASUs and resemble the conformations in previously determined structures.<sup><a href="#CR10">10</a>,<a href="#CR11">11</a></sup> Meanwhile, the conformation of protomer A is less restrained and exhibits the most significant conformational divergence. The different loop conformations offer a glimpse of the protein plasticity that allows M<sup>pro</sup> to accommodate peptides with differing amino acid composition, and underscores the importance of considering this flexibility when analyzing and modeling protein-ligand interactions for M<sup>pro</sup>.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par18">Coronaviruses have caused three epidemics/pandemics in the past 20 years including SARS, MERS, and COVID-19. With the ongoing pandemic of COVID-19, scientists and researchers around the globe are racing to find effective vaccines and antiviral drugs.<sup><a href="#CR6">6</a></sup> The viral polymerase inhibitor remdesivir holds the greatest promise and it is currently being evaluated in several clinical trials.<sup><a href="#CR29">29</a>,<a href="#CR30">30</a></sup> Remdesivir was recently granted emergency use authorization by FDA. The HIV drug combination lopinavir and ritonavir recently failed in a clinical trial for COVID-19 as no significant therapeutic efficacy was observed.<sup><a href="#CR31">31</a></sup> To address this unmet medical need, we initiated a drug repurposing screening to identify potent inhibitors against the SARS-CoV-2 M<sup>pro</sup> from a collection of FDA-approved protease inhibitors. The M<sup>pro</sup> has been shown to be a validated antiviral drug target for SARS-CoV and MERS-CoV.<sup><a href="#CR32">32</a></sup> As the SARS-CoV-2 M<sup>pro</sup> shares a high sequence similarity with SARS-CoV and to a less extent with MERS-CoV, we reasoned that inhibiting the enzymatic activity of SARS-CoV-2 M<sup>pro</sup> will similarly prevent viral replication.<sup><a href="#CR8">8</a>,<a href="#CR10">10</a></sup></p> <p id="Par19">Noticeable findings from our study include: (1) Boceprevir (<b>28</b>), an FDA-approved HCV drug, inhibits the enzymatic activity of M<sup>pro</sup> with IC<sub>50</sub> of 4.13â€‰ÂµM, and has an EC<sub>50</sub> of 1.90â€‰ÂµM against the SARS-CoV-2 virus in the cellular viral CPE assay. The therapeutic potential of boceprevir (<b>28</b>) should be further evaluated in relevant animal models and human clinical trials. Since boceprevir (<b>28</b>) is an FDA-approved drug, the dose, toxicity, formulation and pharmacokinetic properties are already known, which will greatly speed up the design of follow-up studies; (2) GC-376 (<b>64</b>), an investigational veterinary drug, showed promising antiviral activity against the SARS-CoV-2 virus (EC<sub>50</sub>â€‰=â€‰3.37â€‰ÂµM). It has the highest enzymatic inhibition against the M<sup>pro</sup> with an IC<sub>50</sub> value of 0.03â€‰ÂµM. This compound has promising in vivo efficacy in treating cats infected with FIP, and has favorable in vivo pharmacokinetic properties. Therefore, GC-376 (<b>64</b>) is ready to be tested in relevant animal models of SARS-CoV-2 when available. Importantly, the X-ray crystal structure of SARS-CoV-2 M<sup>pro</sup> in complex with GC-376 (<b>64</b>) provides a molecular explanation of the high binding affinity of aldehyde-containing compounds as they can adopt two configurations R and S. The conformational flexibility at the TSEDMLN loop explains the broad substrate scope at the P2 position of M<sup>pro</sup> inhibitors; (3) Three calpain/cathepsin inhibitors, MG-132 (<b>43</b>), calpain inhibitors II (<b>61</b>) and XII (<b>62</b>), are potent inhibitors of M<sup>pro</sup> with single-digit to submicromolar efficacy in the enzymatic assay. Calpain inhibitors II (<b>61</b>) and XII (<b>62</b>) also inhibit SARS-CoV-2 in the CPE assay with EC<sub>50</sub> values of 2.07 and 0.49â€‰ÂµM, respectively. This result suggests that calpain/cathepsin inhibitors are rich sources of drug candidates for SARS-CoV-2. Indeed, previous studies have shown that calpain II and cathepsin L are required for the proteolytic processing of the coronavirus S protein, a step that is essential for the viral fusion and genome release during the early stage of viral replication.<sup><a href="#CR33">33</a></sup> Calpain and cathepsin inhibitors such as MDL28170 (calpain inhibitor III),<sup><a href="#CR33">33</a></sup> MG-132,<sup><a href="#CR34">34</a></sup> calpain inhibitor VI<sup><a href="#CR35">35</a></sup> have been shown to inhibit SARS-CoV replication in cell culture. Other than the increased potency of targeting both M<sup>pro</sup> and calpain/cathepsin, an additional benefit of such dual inhibitors might be their high genetic barrier to drug resistance. A significant number of calpain/cathepsin inhibitors have been developed over the years for various diseases including cancer, neurodegeneration disease, kidney diseases, and ischemia/reperfusion injury.<sup><a href="#CR36">36</a></sup> Given our promising results of calpain inhibitors II (<b>61</b>) and XII (<b>62</b>) in inhibiting the SARS-CoV-2 M<sup>pro</sup> and their potent antiviral activity in cell culture, it might be worthwhile to repurpose them as antivirals for SARS-CoV-2.</p> <p id="Par20">All potent SARS-CoV-2 M<sup>pro</sup> inhibitors contain reactive warheads such as Î±-ketoamide (boceprevir (<b>28</b>), calpain inhibitor XII (<b>62</b>)) or aldehyde (MG-132 (<b>43</b>), calpain inhibitor II (<b>61</b>)) or aldehyde prodrug, the bisulfite (GC-376 (<b>64</b>)). This result suggests that reactive warheads might be essential for SARS-CoV-2 M<sup>pro</sup> inhibition. The compounds identified in this study represent some of the most potent and selective hits reported so far, and have similar activity to the recently reported SARS-CoV-2 M<sup>pro</sup> inhibitors (<b>ebselen</b>, <b>N3</b>, and <b>13b</b>)Â (TableÂ <a href="#Tab3">3</a>). Calpain inhibitor XII (<b>62</b>) had similar potency to the recently disclosed compounds (<b>11a</b> and <b>11b</b>)Â (TableÂ <a href="#Tab3">3</a>).<sup><a href="#CR11">11</a></sup> Notably, boceprevir (<b>28</b>), calpain inhibitor II (<b>61</b>) and XII (<b>62</b>) have different chemical scaffolds than the substrate mimetics GC-376 (<b>64</b>), (<b>N3</b>, <b>13b</b>, <b>11a</b>, and <b>11b</b>),Â therefore providing new opportunities for designing more potent and selective SARS-CoV-2 M<sup>pro</sup> inhibitors.</p> <p id="Par21">Aside from the above positive results, we also showed that ritonavir (<b>9</b>) and lopinavir (<b>10</b>) failed to inhibit the SARS-CoV-2 M<sup>pro</sup> (IC<sub>50</sub>â€‰&amp;gt;â€‰20â€‰ÂµM, Fig.Â <a href="#Fig2">2</a>), which might explain why they lack efficacy in clinical trials for COVID-19.<sup><a href="#CR31">31</a></sup> Camostat (<b>39</b>) was recently proposed to inhibit SARS-CoV-2 entry through inhibiting the host TMPRSS2, a host serine protease that is important for viral S protein priming.<sup><a href="#CR37">37</a></sup> In our study, we found that camostat (<b>39</b>) has no inhibition against the SARS-CoV-2 M<sup>pro</sup> (IC<sub>50</sub>â€‰&amp;gt;â€‰20â€‰ÂµM).</p> <p id="Par22">In summary, this study identified several potent SARS-CoV-2 M<sup>pro</sup> inhibitors with potent enzymatic inhibition as well as cellular antiviral activity. Further development based on these hits might lead to clinically useful COVID-19 antivirals. They can be used either alone or in combination with polymerase inhibitors such as remdesivir as a means to achieve potential synergistic antiviral effect as well as to suppress drug resistance.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="celllinesandviruses" title="sec"> <div class="title" tagx="title" title="title">
Cell lines and viruses</div> <p id="Par23">Human rhabdomyosarcoma RD, A549, MDCK, Caco-2, and Vero cells were maintained in Dulbeccoâ€™s modified Eagleâ€™s medium (DMEM); BEAS2B and HCT-8 cells were maintained in RPMI 1640 medium. Both medium was supplemented with 10% fetal bovine serum (FBS) and 1% penicillinâ€"streptomycin antibiotics. Cells were kept at 37â€‰Â°C in a 5% CO<sub>2</sub> atmosphere. The USA-WA1/2020 strain of SARS-CoV-2 was obtained from the World Reference Center for Emerging Viruses and Arboviruses (WRCEVA).</p> </div> <div class="proteinexpressionandpurification" title="sec"> <div class="title" tagx="title" title="title">
Protein expression and purification</div> <p id="Par24">SARS-CoV-2 main protease (M<sup>pro</sup> or 3CL) gene from strain BetaCoV/Wuhan/WIV04/2019 was ordered from GenScript (Piscataway, NJ, USA) in the pET29a(+) vector with <i>E. coli</i> codon optimization. pET29a(+) plasmids with SARS-CoV-2 M<sup>pro</sup> was transformed into competent <i>E. coli</i> BL21(DE3) cells, and a single colony was picked to inoculate 10â€‰mL of LB supplemented with 50â€‰g/mL kanamycin and grown at 37â€‰Â°C with shaking at 250â€‰rpm. The 10-mL inoculum was added to 1â€‰L of LB with 50â€‰g/mL kanamycin and grown to an optical density at 600â€‰nm of 0.8, and then induced using 1.0â€‰mM IPTG. Induced cultures were incubated at 37â€‰Â°C for an additional 3â€‰h and then harvested, resuspended in lysis buffer (25â€‰mM Tris, pH 7.5, 750â€‰mM NaCl, 2â€‰mM DTT with 0.5â€‰mg/mL lysozyme, 0.5â€‰mM phenylmethylsulfonyl fluoride (PMSF), 0.02â€‰mg/mL DNase I), and lysed by alternating sonication and French press cycles. The cell debris were removed by centrifugation at 12,000Ã—â€‰<i>g</i> for 45â€‰min (20% amplitude, 1â€‰s on/1â€‰s off). The supernatant was incubated with Ni-NTA resin for over 2â€‰h at 4â€‰Â°C on a rotator. The Ni-NTA resin was thoroughly washed with 30â€‰mM imidazole in wash buffer (50â€‰mM Tris, pH 7.0, 150â€‰mM NaCl, 2â€‰mM DTT); and eluted with 100â€‰mM imidazole in elution buffer (50â€‰mM Tris, pH 7.0, 150â€‰mM NaCl, 2â€‰mM DTT). The imidazole was removed via dialysis or on a 10,000-molecular-weight-cutoff centrifugal concentrator spin column. The purity of the protein was confirmed with SDS-PAGE. The protein concentration was determined via 260â€‰nm absorbance with Îµ 32890. EV-A71 2A<sup>pro</sup> and 3C<sup>pro</sup> were expressed in the pET28b(+) vector as previously described.<sup><a href="#CR18">18</a>,<a href="#CR38">38</a>,<a href="#CR39">39</a></sup></p> <p id="Par25">The protein sequence of the SARS-CoV-2 M<sup>pro</sup> used in the enzymatic assay and X-ray crystallization is listed below:</p> <p id="Par26">SGFRKMAFPS GKVEGCMVQV TCGTTTLNGL WLDDVVYCPR HVICTSEDML NPNYEDLLIR KSNHNFLVQA GNVQLRVIGH SMQNCVLKLK VDTANPKTPK YKFVRIQPGQ TFSVLACYNG SPSGVYQCAM RPNFTIKGSF LNGSCGSVGF NIDYDCVSFC YMHHMELPTG VHAGTDLEGN FYGPFVDRQT AQAAGTDTTI TVNVLAWLYA AVINGDRWFL NRFTTTLNDF NLVAMKYNYE PLTQDHVDIL GPLSAQTGIA VLDMCASLKE LLQNGMNGRT ILGSALLEDE FTPFDVVRQC SGVTFQ LEHHHHHH.</p> </div> <div class="peptidesynthesis" title="sec"> <div class="title" tagx="title" title="title">
Peptide synthesis</div> <p id="Par27">The SARS-CoV-2 M<sup>pro</sup> FRET substrate Dabcyl-KTSAVLQ/SGFRKME (Edans) was synthesized by solid-phase synthesis through iterative cycles of coupling and deprotection using the previously optimized procedure.<sup><a href="#CR40">40</a></sup> Specifically, chemmatrix rink-amide resin was used. Typical coupling condition was 5 equiv of amino acid, 5 equiv of HATU, and 10 equiv of DIEA in DMF for 5â€‰min at 80â€‰Â°C. For deprotection, 5% piperazine plus 0.1â€‰M HOBt were used and the mixture was heated at 80â€‰Â°C for 5â€‰min. The peptide was cleaved from the resin using 95% TFA, 2.5% Tris and 2.5% H<sub>2</sub>O, and the crude peptide was precipitated from ether after removal of TFA. The final peptide was purified by preparative HPLC. The purify and identity of the peptide were confirmed by analytical HPLC (&amp;gt;Â 98% purity) and mass spectrometry. [Mâ€‰+â€‰3]<sup>3+</sup> was calculated as 694.15 and detected as 694.90; [Mâ€‰+â€‰4]<sup>4+</sup> was calculated as 520.86 and detected as 521.35.</p> </div> <div class="edansstandardcurve" title="sec"> <div class="title" tagx="title" title="title">
Edans standard curve</div> <p id="Par28">Edans standard curve was generated as described below: 200â€‰nM SARS-CoV-2 M<sup>pro</sup> was incubated with varying concentrations of FRET substrate (0.5â€"40â€‰ÂµM) and the reaction progress was monitored until the fluorescence signals reached plateau, at which point we deemed all the FRET substrate was digested by M<sup>pro</sup>. The endpoint fluorescence signal was plotted against FRET substrate concentration with a linear regression function in Prism 5.</p> </div> <div class="nativemassspectrometry" title="sec"> <div class="title" tagx="title" title="title">
Native mass spectrometry</div> <p id="Par29">Prior to analysis, the protein was buffer exchanged into 0.2â€‰M ammonium acetate (pH 6.8) and diluted to 10â€‰Î¼M. DTT was dissolved in water and prepared at a 400â€‰mM stock. Each ligand was dissolved in ethanol and diluted to 10Ã— stock concentrations. The final mixture was prepared by adding 4â€‰Î¼L protein, 0.5â€‰Î¼L DTT stock, and 0.5â€‰Î¼L ligand stock for final concentration of 4â€‰mM DTT and 8â€‰Î¼M protein. Final ligand concentrations were used as annotated. The mixtures were then incubated for 10â€‰min at room temperature prior to analysis. Each sample was mixed and analyzed in triplicate.</p> <p id="Par30">Native MS was performed using a Q-Exactive HF quadrupole-Orbitrap mass spectrometer with the Ultra-High Mass Range research modifications (Thermo Fisher Scientific). Samples were ionized using nano-electrospray ionization in positive ion mode using 1.0â€‰kV capillary voltage at a 150â€‰Â°C capillary temperature. The samples were all analyzed with a 1000â€"25,000â€‰m/z range, the resolution set to 30,000, and a trapping gas pressure set to 3. Source fragmentation (10â€"50â€‰V) was applied to all samples to aid in desolvation. Data were deconvolved and analyzed with UniDec.<sup><a href="#CR41">41</a></sup></p> </div> <div class="enzymaticassays" title="sec"> <div class="title" tagx="title" title="title">
Enzymatic assays</div> <p id="Par31">For reaction condition optimization, 200â€‰ÂµM SARS-CoV-2 M<sup>pro</sup> was used. pH 6.0 buffer contains 20â€‰mM MES, pH 6.0, 120â€‰mM NaCl, 0.4â€‰mM EDTA, 4â€‰mM DTT and 20% glycerol; pH 6.5 buffer contains 20â€‰mM HEPES, pH 6.5, 120â€‰mM NaCl, 0.4â€‰mM EDTA, 4â€‰mM DTT and 20% glycerol; pH 7.0 buffer contains 20â€‰mM HEPES, pH 7.0, 120â€‰mM NaCl, 0.4â€‰mM EDTA, 4â€‰mM DTT and 20% glycerol. Upon addition of 20â€‰ÂµM FRET substrate, the reaction progress was monitored for 1â€‰h. The first 15â€‰min of reaction was used to calculate initial velocity via linear regression in Prism 5. M<sup>pro</sup> displays the highest proteolytic activity in pH 6.5 buffer. All the following enzymatic assays were carried out in pH 6.5 buffer.</p> <p id="Par32">For the measurements of <i>K</i><sub>m</sub>/<i>V</i><sub>max</sub>, screening of the protease inhibitor library, as well as IC<sub>50</sub> measurements, proteolytic reaction with 100â€‰nM M<sup>pro</sup> in 100â€‰ÂµL of pH 6.5 reaction buffer was carried out at 30â€‰Â°C in a Cytation 5 imaging reader (Thermo Fisher Scientific) with filters for excitation at 360/40â€‰nm and emission at 460/40â€‰nm. Reactions were monitored every 90â€‰s. For <i>K</i><sub>m</sub>/<i>V</i><sub>max</sub> measurements, a FRET substrate concentration ranging from 0 to 200â€‰ÂµM was applied. The initial velocity of the proteolytic activity was calculated by linear regression for the first 15â€‰min of the kinetic progress curves. The initial velocity was plotted against the FRET concentration with the classic Michaelisâ€"Menten equation in Prism 5 software. For the screening of protease inhibitor library and IC<sub>50</sub> measurements, 100â€‰nM M<sup>pro</sup> was incubated with protease inhibitor at 30â€‰Â°C for 30â€‰min in reaction buffer, then the reaction was initiated by adding 10â€‰ÂµM FRET substrate, the reaction was monitored for 1â€‰h, and the initial velocity was calculated for the first 15â€‰min by linear regression. The IC<sub>50</sub> was calculated by plotting the initial velocity against various concentrations of protease inhibitors by use of a dose-response curve in Prism 5 software. Proteolytic reaction progress curve kinetics measurements with GC376, MG132, boceprevir, calpain inhibitor II, and calpain inhibitor XII used for curve fitting, were carried out as follows: 5â€‰nM M<sup>pro</sup> protein was added to 20â€‰ÂµM FRET substrate with various concentrations of testing inhibitor in 200â€‰ÂµL of reaction buffer at 30â€‰Â°C to initiate the proteolytic reaction. The reaction was monitored for 4â€‰h. The progress curves were fit to a slow binding Morrison equation (Eq. (<a href="#Equ3">3</a>)) as described previously<sup><a href="#CR18">18</a>,<a href="#CR42">42</a></sup>: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">1</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$E + I \,\,{\mathop {\leftrightarrow}\limits _{k_1} ^{K_2}}\,\, EI \,\,{\mathop {\leftrightarrow}\limits _{k_1} ^{K_2}}\,\, EI \,\,^{\ast}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
I</div> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="munderover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
â†"</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
E</div> <div class="mi_UNKNOWN">
I</div> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="munderover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
â†"</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
E</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mspace_UNKNOWN" /> <div class="mspace_UNKNOWN" /> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
*</div> </div> </div></div><div class="graphic" position="anchor" /> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">2</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\mathrm{K}_{\mathrm{I}} = \mathrm{k}_{ - 1}/\mathrm{k}_1$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div></div><div class="graphic" position="anchor" /> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">3</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\mathrm{P}\left( \mathrm{t} \right) = \mathrm{P}_0 + \mathrm{V}_{\mathrm{s}}{\mathrm{t}}-\left( {\mathrm{Vs}-\mathrm{V}_0} \right)\left( {1-e^{ - \mathrm{kt}}} \right)/\mathrm{k}$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> </div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Vs</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
kt</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
k</div></div><div class="graphic" position="anchor" /> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">4</span><div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M7">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\mathrm{k} = \mathrm{k}_2\left[ \mathrm{I} \right]/\left( {\mathrm{K}_{\mathrm{I}} + \left[ \mathrm{I} \right]} \right)$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> </div> </div> </div> </div></div><div class="graphic" position="anchor" /> </div> </div>where P(t) is the fluorescence signal at time t, P<sub>0</sub> is the background signal at time 0, V<sub>0</sub>, V<sub>s</sub>, and k represent, respectively, the initial velocity, the final steady-state velocity and the apparent first-order rate constant for the establishment of the equilibrium between EI and EI*.<sup><a href="#CR42">42</a></sup> k<sub>2</sub>/K<sub>I</sub> is commonly used to evaluate the efficacy for covalent inhibitor. We observed substrate depletion when proteolytic reactions progress longer than 90â€‰min; therefore only first 90â€‰min of the progress curves were used in the curve fitting (Fig.Â <a href="#Fig6">6</a>, middle column). In this study, we could not accurately determine the k<sub>2</sub> for the protease inhibitors: calpain inhibitor II, MG132, boceprevir, and calpain inhibitor XII, due to the very slow k<sub>2</sub> in these cases: significant substrate depletion before the establishment of the equilibrium between EI and EI*. In these cases, K<sub>I</sub> was determined with Morrison equation in Prism 5.<p /> </div> <div class="differentialscanningfluorimetry" title="sec"> <div class="title" tagx="title" title="title">
Differential scanning fluorimetry</div> <p id="Par33">The binding of protease inhibitors to M<sup>pro</sup> was monitored by differential scanning fluorimetry (DSF) using a Thermo Fisher QuantStudio<sup>TM</sup> 5 Real-Time PCR System. TSA plates were prepared by mixing M<sup>pro</sup> protein (final concentration of 3â€‰Î¼M) with inhibitors, and incubated at 30â€‰Â°C for 30â€‰min. 1Ã— SYPRO orange (Thermo Fisher Scientific) were added and the fluorescence of the plates were taken under a temperature gradient ranging from 20 to 90â€‰Â°C (incremental steps of 0.05â€‰Â°C/s). The melting temperature (T<sub>m</sub>) was calculated as the mid-log of the transition phase from the native to the denatured protein using a Boltzmann model (Protein Thermal Shift Software v1.3). Thermal shift that was represented as Î"T<sub>m</sub> was calculated by subtracting reference melting temperature of proteins in DMSO from the T<sub>m</sub> in the presence of compound.</p> </div> <div class="cytotoxicitymeasurement" title="sec"> <div class="title" tagx="title" title="title">
Cytotoxicity measurement</div> <p id="Par34">A549, MDCK, HCT-8, Caco-2, Vero, and BEAS2B cells for cytotoxicity CPE assays were seeded and grown overnight at 37â€‰Â°C in a 5% CO<sub>2</sub> atmosphere to âˆ¼90% confluence on the next day. Cells were washed with phosphate-buffered saline (PBS) and 200â€‰ÂµL DMEM containing 2% FBS and 1% penicillinâ€"streptomycin, and various concentrations of protease inhibitors were added to each well. 48â€‰h after addition of the protease inhibitors, cells were stained with 66â€‰Î¼g/mL neutral red for 2â€‰h, and neutral red uptake was measured at an absorbance at 540â€‰nm using a Multiskan FC microplate photometer (Thermo Fisher Scientific). The CC<sub>50</sub> values were calculated from best-fit dose-response curves using GraphPad Prism 5 software.</p> </div> <div class="sars-cov-2cpeassay" title="sec"> <div class="title" tagx="title" title="title">
SARS-CoV-2 CPE assay</div> <p id="Par35">Antiviral activities of test compounds were determined in nearly confluent cultures of Vero 76 cells. The assays were performed in 96-well Corning microplates. Cells were infected with approximately 60-foldÂ of 50% cell culture infectious doses (CCID<sub>50</sub>) of SARS-CoV-2 and 50%Â effective concentrations (EC<sub>50</sub>)Â were calculated based on virus-induced CPE quantified by neutral red dye uptake after 5 days of incubation. USA-WA1/2020 strain of SARS-CoV-2 obtained from the WRCEVA was used for both CPE and VYR assays. Three microwells at each concentration of compound were infected. Two uninfected microwells served as toxicity controls. Cells were stained for viability with neutral red (0.11% final concentration) for 2â€‰h. Excess dye was rinsed from the cells with PBS. The absorbed dye was eluted from the cells with 0.1â€‰mL of 50% SÃ¶rensenâ€™s citrate buffer (pH 4.2)-50% ethanol. Plates were read for optical density determination at 540â€‰nm. Readings were converted to the percentage of the results for the uninfected control using an Excel spreadsheet developed for this purpose. EC<sub>50</sub> values were determined by plotting percent CPE versus log<sub>10</sub> inhibitor concentration. Toxicity at each concentration was determined in uninfected wells in the same microplates by measuring dye uptake.</p> </div> <div class="sars-cov-2vyrassay" title="sec"> <div class="title" tagx="title" title="title">
SARS-CoV-2 VYR assay</div> <p id="Par36">VYR assays were conducted by first replicating the viruses in the presence of test compound. Supernatant was harvested 3 days post infection from each concentration of test compound and the virus yield was determined by endpoint dilution method. Briefly, supernatant virus was serially diluted in log<sub>10</sub> increments and then plated onto quadruplicate wells of 96-well plates seeded with Vero 76 cells. The presence or absence of CPE for determining a viral endpoint was evaluated by microscopic examination of cells 6 days post infection. From these data, 90% virus inhibitory concentrations (EC<sub>90</sub>) were determined by regression analysis.</p> </div> <div class="influenzaavirusa/california/07/2009(h1n1)plaquereductionassay" title="sec"> <div class="title" tagx="title" title="title">
Influenza A virus A/California/07/2009 (H1N1) plaque reduction assay</div> <p id="Par37">The plaque assay was performed according to previously published procedures.<sup><a href="#CR43">43</a></sup></p> </div> <div class="mprocrystallizationandstructuredetermination" title="sec"> <div class="title" tagx="title" title="title">
M<sup>pro</sup> crystallization and structure determination</div> <p id="Par38">10â€‰mg/mL of SARS-CoV-2 M<sup>pro</sup> was incubated with 2â€‰mM GC376 at 4â€‰Â°C overnight. The protein was diluted to 2.5â€‰mg/mL the following day in protein buffer (50â€‰mM Tris, pH 7.0, 150â€‰mM NaCl, 4â€‰mM DTT). Since GC3760 is water soluble, no precipitation was observed, and centrifugation was not necessary. Crystals were grown by mixing 2â€‰Î¼L of the protein solution with 1â€‰Î¼L of the precipitant solution (15% PEG 2â€‰K, 10% 1,6-hexanediol, and 0.2â€‰M NaCl) in a hanging-drop vapor-diffusion apparatus. Crystals were cryoprotected by transferring to a cryoprotectant solution (20% PEG 2â€‰K, 10% 1,6-hexanediol, 20% glycerol) and flash-frozen in liquid nitrogen.</p> <p id="Par39">X-ray diffraction data for the SARS-CoV-2 M<sup>pro</sup>â€"GC376 complex structure was collected on the SBC 19-ID beamline at the Advanced Photon Source (APS) in Argonne, IL, and processed with the HKL3000 software suite.<sup><a href="#CR44">44</a></sup> The CCP4 versions of MOLREP was used for molecular replacement using a previously solved SARS-CoV-2 M<sup>pro</sup> structure (PDB ID: 5RGG) as a reference model.<sup><a href="#CR45">45</a></sup> Rigid and restrained refinements were performed using REFMAC and model building was performed with COOT.<sup><a href="#CR46">46</a>,<a href="#CR47">47</a></sup> Protein structure figures were made using PyMOL (SchrÃ¶dinger, LLC).</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41422_2020_356_MOESM1_ESM.pdf">LINK</a> <p>Supplementary information Figure S1</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41422_2020_356_MOESM2_ESM.pdf">LINK</a> <p>Supplementary information Figure S2</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41422_2020_356_MOESM3_ESM.pdf">LINK</a> <p>Supplementary information Figure S3</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41422_2020_356_MOESM4_ESM.pdf">LINK</a> <p>Supplementary information Figure S4</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41422_2020_356_MOESM5_ESM.pdf">LINK</a> <p>Supplementary information Figure S5</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41422_2020_356_MOESM6_ESM.pdf">LINK</a> <p>Supplementary information, Table S1</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41422_2020_356_MOESM7_ESM.pdf">LINK</a> <p>Supplementary information, Table S2</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p><b>Supplementary information</b> accompanies this paper at 10.1038/s41422-020-0356-z.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This research was partially supported by the National Institutes of Health (NIH) (Grant AI147325) and the Arizona Biomedical Research Centre Young Investigator grant (ADHS18-198859) to J.W., B.H. and B.T. thanks for the support from the Respiratory Diseases Branch, National Institute of Allergy and Infectious Diseases, NIH, USA (Contract N01-AI-30048). J.A.T. and M.T.M. were funded by the National Institute of General Medical Sciences, NIH (Grant R35 GM128624 to M.T.M.). We thank Michael Kemp for assistance with crystallization and X-ray diffraction data collection. We also thank the staff members of the Advanced Photon Source of Argonne National Laboratory, particularly those at the Structural Biology Center (SBC), with X-ray diffraction data collection. SBC-CAT is operated by UChicago Argonne, LLC, for the U.S. Department of Energy, Office of Biological and Environmental Research under contract DE-AC02-06CH11357.</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>J.W. and C.M. conceived and designed the study; C.M. expressed the M<sup>pro</sup> with the assistance of T.S.; C.M. performed the primary screening, secondary IC<sub>50</sub> determination, thermal shift binding assay, and enzymatic kinetic studies; M.D.S. carried out M<sup>pro</sup> crystallization and structure determination with the assistance of X.Z., and analyzed the data with Y.C.; B.H. and B.T. performed the SARS-CoV-2 CPE and VYR assays; J.A.T. performed the native mass spectrometry experiments with the guidance from M.T.M.; Y.H. performed the plaque reduction assay with influenza A/California/07/2009 (H1N1) virus; J.W. and Y.C. secured funding and supervised the study; J.W., Y.C., and C.M. wrote the paper with the input from others.</p> <div class="title" tagx="title" title="title">
Data availability</div> <p>The structure for SARS-CoV-2 M<sup>pro</sup> has been deposited in the Protein Data Bank with accession number 6WTT.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par40">J.W. and C.M. are inventors of a pending patent that claims the use of the identified compounds for COVID-19.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahase</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronavirus: covid-19 has killed more people than SARS and MERS combined, despite lower case fatality rate</span><span class="source" tagx="source" title="source">BMJ</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">368</span><span class="fpage" tagx="fpage" title="fpage">m641</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32071063">32071063</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">395</span><span class="fpage" tagx="fpage" title="fpage">565</span><span class="lpage" tagx="lpage" title="lpage">574</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32007145">32007145</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genome composition and divergence of the novel coronavirus (2019-nCoV) originating in China</span><span class="source" tagx="source" title="source">Cell Host Microbe</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">325</span><span class="lpage" tagx="lpage" title="lpage">328</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32035028">32035028</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Coronaviridae Study Group of the International Committee on Taxonomy of Viruses. The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2. <i>Nat. Microbiol</i>. <b>5</b>, 536â€"544 (2020).</span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">An updated estimation of the risk of transmission of the novel coronavirus (2019-nCov)</span><span class="source" tagx="source" title="source">Infect. Dis. Model.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">248</span><span class="lpage" tagx="lpage" title="lpage">255</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32099934">32099934</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Clercq</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Therapeutic options for the 2019 novel coronavirus (2019-nCoV)</span><span class="source" tagx="source" title="source">Nat. Rev. Drug Discov.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">149</span><span class="lpage" tagx="lpage" title="lpage">150</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32127666">32127666</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of SARS-CoV-2 (previously 2019-nCoV) infection by a highly potent pan-coronavirus fusion inhibitor targeting its spike protein that harbors a high capacity to mediate membrane fusion</span><span class="source" tagx="source" title="source">Cell Res.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">343</span><span class="lpage" tagx="lpage" title="lpage">355</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32231345">32231345</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved alpha-ketoamide inhibitors</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">368</span><span class="fpage" tagx="fpage" title="fpage">409</span><span class="lpage" tagx="lpage" title="lpage">412</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32198291">32198291</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anand</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ziebuhr</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wadhwani</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mesters</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilgenfeld</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronavirus main proteinase (3CLpro) structure: basis for design of anti-SARS drugs</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">300</span><span class="fpage" tagx="fpage" title="fpage">1763</span><span class="lpage" tagx="lpage" title="lpage">1767</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12746549">12746549</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure of M(pro) from SARS-CoV-2 and discovery of its inhibitors</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">582</span><span class="fpage" tagx="fpage" title="fpage">289</span><span class="lpage" tagx="lpage" title="lpage">293</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32272481">32272481</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dai</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.abb4489">10.1126/science.abb4489</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32554628">32554628</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wingfield</span><span class="given-names" tagx="given-names" title="given-names">PT</span></span></span><span class="mixed-article-title" title="mixed-article-title">N-terminal methionine processing</span><span class="source" tagx="source" title="source">Curr. Protoc. Protein Sci.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">88</span><span class="fpage" tagx="fpage" title="fpage">6.14.1</span><span class="lpage" tagx="lpage" title="lpage">6.14.3</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28369664">28369664</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pedersen</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Efficacy of a 3C-like protease inhibitor in treating various forms of acquired feline infectious peritonitis</span><span class="source" tagx="source" title="source">J. Feline Med. Surg.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">378</span><span class="lpage" tagx="lpage" title="lpage">392</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28901812">28901812</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Reversal of the progression of fatal coronavirus infection in cats by a broad-spectrum coronavirus protease inhibitor</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">e1005531</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27027316">27027316</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">11754</span><span class="lpage" tagx="lpage" title="lpage">11762</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22915796">22915796</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayden</span><span class="given-names" tagx="given-names" title="given-names">FG</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Phase II, randomized, double-blind, placebo-controlled studies of ruprintrivir nasal spray 2-percent suspension for prevention and treatment of experimentally induced rhinovirus colds in healthy volunteers</span><span class="source" tagx="source" title="source">Antimicrob. Agents Chemother.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">3907</span><span class="lpage" tagx="lpage" title="lpage">3916</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14638501">14638501</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shie</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of the severe acute respiratory syndrome 3CL protease by peptidomimetic alpha,beta-unsaturated esters</span><span class="source" tagx="source" title="source">Bioorg. Med. Chem.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">5240</span><span class="lpage" tagx="lpage" title="lpage">5252</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15994085">15994085</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Musharrafieh</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Validating enterovirus D68-2A(pro) as an antiviral drug target and the discovery of telaprevir as a potent D68-2A(pro) inhibitor</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">93</span><span class="fpage" tagx="fpage" title="fpage">e0221-18</span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuo</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Design, synthesis, and evaluation of 3C protease inhibitors as anti-enterovirus 71 agents</span><span class="source" tagx="source" title="source">Bioorg. Med. Chem.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">7388</span><span class="lpage" tagx="lpage" title="lpage">7398</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18583140">18583140</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</span><span class="source" tagx="source" title="source">Cell Res.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">269</span><span class="lpage" tagx="lpage" title="lpage">271</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32020029">32020029</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The crystal structures of severe acute respiratory syndrome virus main protease and its complex with an inhibitor</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">13190</span><span class="lpage" tagx="lpage" title="lpage">13195</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14585926">14585926</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muramatsu</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">SARS-CoV 3CL protease cleaves its C-terminal autoprocessing site by novel subsite cooperativity</span><span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">113</span><span class="fpage" tagx="fpage" title="fpage">12997</span><span class="lpage" tagx="lpage" title="lpage">13002</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27799534">27799534</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Maturation mechanism of severe acute respiratory syndrome (SARS) coronavirus 3C-like proteinase</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">285</span><span class="fpage" tagx="fpage" title="fpage">28134</span><span class="lpage" tagx="lpage" title="lpage">28140</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20489209">20489209</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anand</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra alpha-helical domain</span><span class="source" tagx="source" title="source">EMBO J.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">3213</span><span class="lpage" tagx="lpage" title="lpage">3224</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12093723">12093723</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structures of two coronavirus main proteases: implications for substrate binding and antiviral drug design</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">82</span><span class="fpage" tagx="fpage" title="fpage">2515</span><span class="lpage" tagx="lpage" title="lpage">2527</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18094151">18094151</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Alpha-ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication: structure-based design, synthesis, and activity assessment</span><span class="source" tagx="source" title="source">J. Med. Chem.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">63</span><span class="fpage" tagx="fpage" title="fpage">4562</span><span class="lpage" tagx="lpage" title="lpage">4578</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32045235">32045235</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chuck</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Profiling of substrate specificity of SARS-CoV 3CL</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">e13197</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20949131">20949131</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hegyi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ziebuhr</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Conservation of substrate specificities among coronavirus main proteases</span><span class="source" tagx="source" title="source">J. Gen. Virol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">595</span><span class="lpage" tagx="lpage" title="lpage">599</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11842254">11842254</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grein</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Compassionate use of remdesivir for patients with severe Covid-19</span><span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">382</span><span class="fpage" tagx="fpage" title="fpage">2327</span><span class="lpage" tagx="lpage" title="lpage">2336</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32275812">32275812</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holshue</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">First case of 2019 novel coronavirus in the United States</span><span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">382</span><span class="fpage" tagx="fpage" title="fpage">929</span><span class="lpage" tagx="lpage" title="lpage">936</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32004427">32004427</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A trial of lopinavirâ€"ritonavir in adults hospitalized with severe Covid-19</span><span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">382</span><span class="fpage" tagx="fpage" title="fpage">1787</span><span class="lpage" tagx="lpage" title="lpage">1799</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32187464">32187464</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zumla</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azhar</span><span class="given-names" tagx="given-names" title="given-names">EI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">KY</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronavirusesâ€"drug discovery and therapeutic options</span><span class="source" tagx="source" title="source">Nat. Rev. Drug Discov.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">327</span><span class="lpage" tagx="lpage" title="lpage">347</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26868298">26868298</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibitors of cathepsin L prevent severe acute respiratory syndrome coronavirus entry</span><span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">102</span><span class="fpage" tagx="fpage" title="fpage">11876</span><span class="lpage" tagx="lpage" title="lpage">11881</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081529">16081529</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schneider</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Severe acute respiratory syndrome coronavirus replication is severely impaired by MG132 due to proteasome-independent inhibition of M-calpain</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">10112</span><span class="lpage" tagx="lpage" title="lpage">10122</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22787216">22787216</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barnard</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of severe acute respiratory syndrome-associated coronavirus (SARSCoV) by calpain inhibitors and beta-D-N4-hydroxycytidine</span><span class="source" tagx="source" title="source">Antivir. Chem. Chemother.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">22</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15074711">15074711</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ono</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saido</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sorimachi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Calpain research for drug discovery: challenges and potential</span><span class="source" tagx="source" title="source">Nat. Rev. Drug Discov.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">854</span><span class="lpage" tagx="lpage" title="lpage">876</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27833121">27833121</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoffmann</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">181</span><span class="fpage" tagx="fpage" title="fpage">271</span><span class="lpage" tagx="lpage" title="lpage">280</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32142651">32142651</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biochemical characterization of recombinant Enterovirus 71 3C protease with fluorogenic model peptide substrates and development of a biochemical assay</span><span class="source" tagx="source" title="source">Antimicrob. Agents Chemother.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">1827</span><span class="lpage" tagx="lpage" title="lpage">1836</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25421478">25421478</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Conformational plasticity of the 2A proteinase from enterovirus 71</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">7348</span><span class="lpage" tagx="lpage" title="lpage">7356</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23616646">23616646</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cady</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeGrado</span><span class="given-names" tagx="given-names" title="given-names">WF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Specific binding of adamantane drugs and direction of their polar amines in the pore of the influenza M2 transmembrane domain in lipid bilayers and dodecylphosphocholine micelles determined by NMR spectroscopy</span><span class="source" tagx="source" title="source">J. Am. Chem. Soc.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">133</span><span class="fpage" tagx="fpage" title="fpage">4274</span><span class="lpage" tagx="lpage" title="lpage">4284</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21381693">21381693</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marty</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Bayesian deconvolution of mass and ion mobility spectra: from binary interactions to polydisperse ensembles</span><span class="source" tagx="source" title="source">Anal. Chem.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">4370</span><span class="lpage" tagx="lpage" title="lpage">4376</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25799115">25799115</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morrison</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walsh</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span></span><span class="mixed-article-title" title="mixed-article-title">The behavior and significance of slow-binding enzyme inhibitors</span><span class="source" tagx="source" title="source">Adv. Enzymol. Relat. Areas Mol. Biol.</span><span class="year" tagx="year" title="year">1988</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">201</span><span class="lpage" tagx="lpage" title="lpage">301</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3281418">3281418</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pharmacological characterization of the spectrum of antiviral activity and genetic barrier to drug resistance of M2-S31N channel blockers</span><span class="source" tagx="source" title="source">Mol. Pharmacol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">188</span><span class="lpage" tagx="lpage" title="lpage">198</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27385729">27385729</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minor</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cymborowski</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Otwinowski</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chruszcz</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">HKL-3000: the integration of data reduction and structure solutionâ€"from diffraction images to an initial model in minutes</span><span class="source" tagx="source" title="source">Acta Crystallogr. D Biol. Crystallogr.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">62</span><span class="fpage" tagx="fpage" title="fpage">859</span><span class="lpage" tagx="lpage" title="lpage">866</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16855301">16855301</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vagin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teplyakov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular replacement with MOLREP</span><span class="source" tagx="source" title="source">Acta Crystallogr. D Biol. Crystallogr.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">22</span><span class="lpage" tagx="lpage" title="lpage">25</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20057045">20057045</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murshudov</span><span class="given-names" tagx="given-names" title="given-names">GN</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">REFMAC5 for the refinement of macromolecular crystal structures</span><span class="source" tagx="source" title="source">Acta Crystallogr. D Biol. Crystallogr.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">355</span><span class="lpage" tagx="lpage" title="lpage">367</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21460454">21460454</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emsley</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cowtan</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coot: model-building tools for molecular graphics</span><span class="source" tagx="source" title="source">Acta Crystallogr. D Biol. Crystallogr.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">2126</span><span class="lpage" tagx="lpage" title="lpage">2132</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15572765">15572765</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonuccelli</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Proteasome inhibitor (MG-132) treatment of mdx mice rescues the expression and membrane localization of dystrophin and dystrophin-associated proteins</span><span class="source" tagx="source" title="source">Am. J. Pathol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">163</span><span class="fpage" tagx="fpage" title="fpage">1663</span><span class="lpage" tagx="lpage" title="lpage">1675</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14507673">14507673</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mani</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">In vivo administration of calpeptin attenuates calpain activation and cardiomyocyte loss in pressure-overloaded feline myocardium</span><span class="source" tagx="source" title="source">Am. J. Physiol. Heart Circ. Physiol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">295</span><span class="fpage" tagx="fpage" title="fpage">H314</span><span class="lpage" tagx="lpage" title="lpage">326</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18487434">18487434</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span></span><span class="mixed-article-title" title="mixed-article-title">The protective effects and potential mechanism of Calpain inhibitor Calpeptin against focal cerebral ischemia-reperfusion injury in rats</span><span class="source" tagx="source" title="source">Mol. Biol. Rep.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">905</span><span class="lpage" tagx="lpage" title="lpage">912</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20473717">20473717</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akdemir</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Therapeutic efficacy of SJA6017, a calpain inhibitor, in rat spinal cord injury</span><span class="source" tagx="source" title="source">J. Clin. Neurosci.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">1130</span><span class="lpage" tagx="lpage" title="lpage">1136</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18656362">18656362</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">SZ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">ALLN hinders HCT116 tumor growth through Bax-dependent apoptosis</span><span class="source" tagx="source" title="source">Biochem. Biophys. Res. Commun.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">437</span><span class="fpage" tagx="fpage" title="fpage">325</span><span class="lpage" tagx="lpage" title="lpage">330</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23831472">23831472</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delli Pizzi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Morphological and metabolic changes in the nigro-striatal pathway of synthetic proteasome inhibitor (PSI)-treated rats: a MRI and MRS study</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e56501</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23431380">23431380</a></span></span></li> </ul> </div> </div>  </body></html>