<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Infect Genet Evol</span><span class="iso-abbrev" title="iso-abbrev">Infect. Genet. Evol</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Infection, Genetics and Evolution</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1567-1348</span><span class="issn-epub" tagx="issn" title="issn-epub">1567-7257</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier B.V.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7462568">7462568</a></span><span class="publisher-id" title="publisher-id">S1567-1348(20)30348-8</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1016/j.meegid.2020.104517">10.1016/j.meegid.2020.104517</a></span><span class="publisher-id" title="publisher-id">104517</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Genome based evolutionary lineage of SARS-CoV-2 towards the development of novel chimeric vaccine</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akhand</span><span class="given-names" tagx="given-names" title="given-names">Mst Rubaiat Nazneen</span></span><a href="#af0005">a</a><a href="#af0010">b</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azim</span><span class="given-names" tagx="given-names" title="given-names">Kazi Faizul</span></span><a href="#af0005">a</a><a href="#af0015">c</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoque</span><span class="given-names" tagx="given-names" title="given-names">Syeda Farjana</span></span><a href="#af0005">a</a><a href="#af0020">d</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moli</span><span class="given-names" tagx="given-names" title="given-names">Mahmuda Akther</span></span><a href="#af0005">a</a><a href="#af0020">d</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joy</span><span class="given-names" tagx="given-names" title="given-names">Bijit Das</span></span><a href="#af0005">a</a><a href="#af0010">b</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akter</span><span class="given-names" tagx="given-names" title="given-names">Hafsa</span></span><a href="#af0005">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Afif</span><span class="given-names" tagx="given-names" title="given-names">Ibrahim Khalil</span></span><a href="#af0025">e</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">Nadim</span></span><a href="#af0005">a</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasan</span><span class="given-names" tagx="given-names" title="given-names">Mahmudul</span></span><a href="#af0005">a</a><a href="#af0020">d</a><a href="#cr0005">⁎</a></span><span class="citation_author_institution" id="af0005">[a], </span><span class="citation_author_institution" id="af0010">[b], </span><span class="citation_author_institution" id="af0015">[c], </span><span class="citation_author_institution" id="af0020">[d], </span><span class="citation_author_institution" id="af0025">[e], </span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">⁎</span>Corresponding author at: Department of Pharmaceuticals and Industrial Biotechnology, Faculty of Biotechnology and Genetic Engineering, Sylhet Agricultural University, Sylhet 3100, Bangladesh.</div> </div><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-9-9</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020-11</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-9-9</span></span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">104517</span><span class="lpage" tagx="lpage" title="lpage">104517</span><span class="history" title="history"><span class="received" title="received">received: 2020-6-8</span><span class="rev-recd" title="rev-recd">rev-recd: 2020-7-28</span><span class="accepted" title="accepted">accepted: 2020-8-25</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Elsevier B.V.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Since January 2020 Elsevier has created a COVID-19 resource centre with free information in English and Mandarin on the novel coronavirus COVID-19. The COVID-19 resource centre is hosted on Elsevier Connect, the company's public news and information website. Elsevier hereby grants permission to make all its COVID-19-related research that is available on the COVID-19 resource centre - including this research content - immediately available in PubMed Central and other publicly funded repositories, such as the WHO COVID database with rights for unrestricted research re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for free by Elsevier for as long as the COVID-19 resource centre remains active.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The present study aimed to predict a novel chimeric vaccine by simultaneously targeting four major structural proteins via the establishment of ancestral relationship among different strains of coronaviruses. Conserved regions from the homologous protein sets of spike glycoprotein, membrane protein, envelope protein and nucleocapsid protein were identified through multiple sequence alignment. The phylogeny analyses of whole genome stated that four proteins reflected the close ancestral relation of SARS-CoV-2 to SARS-COV-1 and bat coronavirus. Numerous immunogenic epitopes (both T cell and B cell) were generated from the common fragments which were further ranked on the basis of antigenicity, transmembrane topology, conservancy level, toxicity and allergenicity pattern and population coverage analysis. Top putative epitopes were combined with appropriate adjuvants and linkers to construct a novel multiepitope subunit vaccine against COVID-19. The designed constructs were characterized based on physicochemical properties, allergenicity, antigenicity and solubility which revealed the superiority of construct V3 in terms safety and efficacy. Essential molecular dynamics and normal mode analysis confirmed minimal deformability of the refined model at molecular level. In addition, disulfide engineering was investigated to accelerate the stability of the protein. Molecular docking study ensured high binding affinity between construct V3 and HLA cells, as well as with different host receptors. Microbial expression and translational efficacy of the constructs were checked using pET28a(+) vector of <i>E. coli</i> strain K12. However, the <i>in vivo</i> and <i>in vitro</i> validation of suggested vaccine molecule might be ensured with wet lab trials using model animals for the implementation of the presented data.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Graphical abstract</div> <p> </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/" /></div>  </div> <p /> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Highlights</div> <p> </p><ul> <li><span class="label" tagx="label" title="label">•</span><p id="p0005">Phylogeny analysis reflected the close ancestral relation of SARS-CoV-2 and SARS-COV-1.</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="p0010">Conserved epitopes of S, M, E and N proteins of SARS-COV-2 were employed to construct a novel chimeric vaccine.</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="p0015">Antigenicity, conservancy, toxicity and allergenicity of designed vaccine were experimented.</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="p0020">Minimal deformability of the refined vaccine model at molecular level was determined.</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="p0025">Microbial expression and translational efficacy were checked using pET28a(+) vector.</p> </li> </ul> <p /> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
SARS-CoV-2</div> <div class="kwd" title="kwd">
COVID-19</div> <div class="kwd" title="kwd">
Chimeric vaccine</div> <div class="kwd" title="kwd">
Evolutionary relationship</div> <div class="kwd" title="kwd">
Normal mode analysis</div> <div class="kwd" title="kwd">
Molecular docking</div> <div class="kwd" title="kwd">
Restriction cloning</div> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
Introduction</div> <p id="p0030">Novel coronavirus named SARS-CoV-2/2019-nCoV was identified at the end of 2019 in Wuhan, a city in the Hubei province of China, causing severe pneumonia that leads to huge death cases (<a href="#bb0465">Wang et al., 2020</a>). Gradually this virus emerged as a new threat to the whole world and affecting almost all parts of the world. To date, the pathogen has affected 198 countries, and thus becoming a global public health emergency. Global public health concern with pandemic notion of COVID-19 was declared on January 30th, 2020 by the <a href="#bb0015">World Health Organization (WHO), 2020a</a>, <a href="#bb0035">World Health Organization (WHO), 2020b</a>. Again, an adverse situation has also been announced on 13 March 2020 for increasing the infections of COVID-19 (<a href="#bb0255">Kunz and Minder, 2020</a>). Till April 10, 2020, total virus affected people around the world exceeded 1,633,272 and more than 97,601 committed death, while 366,610 people fully recovered from the infection <a href="#bb0015">World Health Organization (WHO), 2020a</a>, <a href="#bb0035">World Health Organization (WHO), 2020b</a>). The alarming situation is that the number of confirmed cases worldwide has exceeded one million by this time. It took more than three months to reach the first 10,000 confirmed cases, while required only 12 days to detect the next 100,000 cases. The situation is getting worse in European region. Total death cases in Italy, Spain, USA, France, United Kingdom was 14,681,11,744,7847,6507 and 4313 respectively (till April 4, 2020) and this number is exacerbating day by day (<a href="#bb0015">World Health Organization (WHO), 2020a</a>, <a href="#bb0035">World Health Organization (WHO), 2020b</a>).</p> <p id="p0035">Some common clinical manifestations of COVID-19 are fever, sputum production and shortness in breath, cough, fatigue, sore throat and headache which leads to severe cases of pneumonia. A few patients also have gastrointestinal symptoms with diarrhea and vomiting (<a href="#bb0180">Guan et al., 2020</a>). Though several early studies showed that the mortality rate for SARS-CoV-2 is not as high (2–3%), the latest global death rate for COVID-19 is 3.4% which indicates the increasing trends (<a href="#bb0495">Wu et al., 2020</a>). The investigation of Chinese Center for Diseases Control and Prevention (2020) revealed that the prevalence of COVID-19 is more apparent in the people ages 50 years rather than the lower age groups (<a href="#bb0535">Jeong-ho et al., 2020</a>). High fever and Lymphocytopenia were found more common in Covid-19, though the frequency of the patient without fever condition is also higher than in the earlier outbreaks caused by SARS-CoV (1%) and MERS-CoV (2%) (<a href="#bb0210">Huang et al., 2020</a>; <a href="#bb0085">Chen et al., 2020</a>).</p> <p id="p0040">SARS-CoV-2 is a beta-coronavirus that has a positive sense, 26–32 kb in length, single stranded RNA molecule as its genetic material and belongs to the family Coronaviridae, order Nidovirales (<a href="#bb0215">Hui et al., 2020</a>). It shares genome similarity with SARS-CoV (79.5%) and bat coronavirus (96%) (<a href="#bb0545">Zhou et al., 2020a</a>, <a href="#bb0550">Zhou et al., 2020b</a>; <a href="#bb0555">Zhu et al., 2020</a>). However, there are still obscured hypothesis regarding the vector or carrier of SARS-CoV-2, though its detection was primarily linked to Wuhan’s Huanan Seafood wholesale market (<a href="#bb0290">Lu et al., 2020</a>; <a href="#bb0015">World Health Organization (WHO), 2020a</a>, <a href="#bb0035">World Health Organization (WHO), 2020b</a>). Though the species of SARS-CoV-1 and bat coronavirus shares sufficient sequence similarities with the COVID-19, the known way mechanism of infection to the host, and the death rate is quite different in case of the novel coronavirus. In addition, there is an evolutionary distance between SARS-CoV-1 and bat coronavirus as well as the COVID-19 (<a href="#bb0205">Hu et al., 2018</a>; <a href="#bb0495">Wu et al., 2020</a>; <a href="#bb0490">Wu 2020</a>). Because of high sequence variability of the pathogen, many of the efforts that have been undertaken to develop vaccine against SARS-CoV remain unsuccessful (<a href="#bb0170">Graham et al. 2013</a>). Therefore, there is an urgent need to develop vaccines for treatment of SARS-CoV-2 based on the understanding of actual evolutionary ancestral relationship. While some natural metabolites and traditional medication may come up with comfort and take the edge off few symptoms of COVID-19, there is no proof that existing treatment procedures can effectively combat against the diseased condition (<a href="#bb0015">World Health Organization (WHO), 2020a</a>, <a href="#bb0035">World Health Organization (WHO), 2020b</a>). However, inactivated or live-attenuated forms of pathogenic organisms are usually recommended for the initiation of antigen-specific responses that alleviate or reduce the possibility of host experience with secondary infections (<a href="#bb0455">Thompson and Staats 2011</a>). Moreover, all of the proteins are not usually targeted for protective immunity, whereas only a few numbers of proteins are necessary depending on the microbes (<a href="#bb0450">Tesh et al., 2002</a>, <a href="#bb0275">Li et al. 2014</a>). Depending on sufficient antigen expression from experimental assays, traditional vaccine could take 15 years to develop, while sometimes can lead to undesirable consequences (<a href="#bb0345">Purcell et al. 2007</a>; <a href="#bb0335">Petrovsky and Aguilar 2004</a>).</p> <p id="p0045">Reverse vaccinology approach, on the other hand, is an effective way to develop vaccine against COVID-19. In this method, computation analysis towards genomic architecture of pathogenic candidate could predict the antigens of pathogens without the prerequisite to culture the pathogens in lab condition. Although, few pathogens that challenge to develop effective vaccines so far may become possible through such approach (<a href="#bb0355">Rappuoli 2000</a>) which initiates a huge move in the development of vaccine against the deadly pathogens. The strategy included the comprehensive utilization of bioinformatics algorithm or tools to develop epitope based vaccine molecules, though further validation and experimental procedures are also needed (<a href="#bb0295">Moxon et al., 2019</a>). In addition, peptide based subunit vaccines are biologically safer due to the absence of continuous <i>in vitro</i> culture during the production period, and also implies an appropriate activation of immune responses (<a href="#bb0345">Purcell et al. 2007</a>; <a href="#bb0005">Dudek et al., 2010</a>). Such immunoinformatic approaches have already been employed by the researchers to design vaccines against a number of deadly pathogens including Ebola virus (<a href="#bb0240">Khan et al. 2015</a>), HIV (<a href="#bb0315">Pandey et al. 2018</a>), Areanaviruses (<a href="#bb0055">Azim et al. 2019a</a>), Marburgvirus (<a href="#bb0190">Hasan et al. 2019a</a>), Norwalk virus (<a href="#bb0060">Azim et al. 2019b</a>), Nipah virus (<a href="#bb0375">Saha et al. 2017</a>), Influenza virus (<a href="#bb0195">Hasan et al. 2019b</a>) and so on. At present, a suitable peptide vaccine to combat COVID-19 is urgently necessary that could efficiently generate enough immune response to destroy the virus. Hence, the study was designed to develop a chimeric recombinant vaccine against SARS-CoV-2 by targeting four major structural proteins of the pathogen, while revealing the evolutionary history of different species of coronavirus based on whole genome and protein domain-based phylogeny.</p> </div> <div class="materialsandmethods" title="sec"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
Materials and methods</div> <div class="dataacquisition" title="sec"><span class="label" tagx="label" title="label">2.1</span><div class="title" tagx="title" title="title">
Data acquisition</div> <p id="p0050">Complete Genomes of the COVID-19 and other coronaviruses were retrieved from the NCBI (<a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a>), using the keyword ‘coronavirus’ and the search option ‘nucleotide’. A total 61 complete genomes were retrieved, with unique identity (<a href="#ec0045">File S1</a>). Protein sequence of the spike, envelope, membrane and nucleocapsid were also retrieved from the corresponding genome sequences found in NCBI (<a href="#ec0045">File S1</a>).</p> </div> <div class="phylogenyconstructionandvisualization" title="sec"><span class="label" tagx="label" title="label">2.2</span><div class="title" tagx="title" title="title">
Phylogeny construction and visualization</div> <p id="p0060">The complete genome sequences of coronaviruses and the sequences of viral proteins, such as spike glycoprotein, envelope, membrane and nucleocapsid were employed to construct different phylogenetic trees. Multiple sequence alignment (MSA) of the complete genome and protein sequences were performed using MAFFT v7.310 (<a href="#bb0235">Katoh and Standley 2013</a>) tool. For the whole genome alignment, we used MAFFT Auto algorithm, while for the protein sequences alignment, MAFFT G-INS-I algorithm was used using default parameters. Next, alignment was visualized using the JalView-2.11 (<a href="#bb0475">Waterhouse et al. 2009</a>). Alignment position with more than 50% gaps was pruned from coronavirus genome using Phyutility 2.2.6 program (<a href="#bb0405">Smith and Dunn 2008</a>). Again, more than 20% gaps from the spike protein alignment were removed. PartitionFinder-2.1.1 (<a href="#bb0260">Lanfear et al., 2016</a>) indicated the best fit substitution model of the completed genome sequences and the protein sequences. The phylogeny of the whole genome sequences of coronavirus was constructed using both the Maximum Likelihood Method and Bayesian Method. RAxML version 8.2.11 (<a href="#bb0415">Stamatakis 2014</a>) with the substitution model GTRGAMMAI was used using 1000 rapid bootstrap replicates. MrBayes version 3.2.6 (<a href="#bb0370">Ronquist et al. 2012</a>) with INVGAMMA model was used for the corona virus genomes. Phylogenetic analyses of four different protein sequences were performed by using RAxML-8.2.11 tool. For spike and nucleocapsid proteins, we found PROTGAMMAIWAG and PROTGAMMAIWAG as the best fit model, respectively. Again, PROTGAMMAWAG was the best fit model of evolution for both the membrane and envelope proteins. For the retrieval of the domain sequences of the stated protein sequences, InterPro database (<a href="https://www.ebi.ac.uk/interpro/">https://www.ebi.ac.uk/interpro/</a>) was utilized. Finally, the Interactive Tree of Life (iTOL; EMBL, Heidelberg, Germany) was used for the visualization of the phylogenetic trees. All the trees were rooted in the midpoint.</p> </div> <div class="identificationofconservedregionsasvaccinetarget" title="sec"><span class="label" tagx="label" title="label">2.3</span><div class="title" tagx="title" title="title">
Identification of conserved regions as vaccine target</div> <p id="p0065">In the present study, reverse vaccinology technique was utilized to model a novel multiepitope subunit vaccine against 2019-nCoV. The scheme in <a href="#f0005">Fig. 1</a> represents the complete methodology that has been adopted to develop the final vaccine construct. Among 496 proteins (available in the NCBI database) from different strains of novel corona virus, four structural proteins, i.e. spike glycoprotein, membrane glycoprotein, envelope protein and nucleocapsid protein, were prioritized for further investigation (<a href="#ec0050">File S2</a>). After sequence retrieval from NCBI, the sequences were subjected to BLASTp analysis to find out the homologous protein sequences. Multiple sequence alignment was done by using Clustal Omega to identify the conserved regions (<a href="#bb0395">Sievers and Higgins 2014</a>).The topology of each conserved regions were predicted by TMHMM Server v.2.0 (<a href="http://www.cbs.dtu.dk/services/TMHMM/">http://www.cbs.dtu.dk/services/TMHMM/</a>), while the antigenicity of the conserved regions was determined by VaxiJen v2.0 (<a href="#bb0110">Doytchinova and Flower 2007a</a>).</p> </div> <div class="t-cellepitopeprediction,transmembranetopologyscreeningandantigenicityanalysis" title="sec"><span class="label" tagx="label" title="label">2.4</span><div class="title" tagx="title" title="title">
T-cell epitope prediction, transmembrane topology screening and antigenicity analysis</div> <p id="p0075"> Only the common fragments were used for T-Cell epitopes enumeration via T-Cell epitope prediction server of IEDB (<a href="http://tools.iedb.org/main/tcell/">http://tools.iedb.org/main/tcell/</a>) (<a href="#bb0460">Vita et al., 2015</a>). Again, TMHMM server was utilized for the prediction of transmembrane topology of predicted MHC-I and MHC-II binding peptides followed by antigenicity scoring via VaxiJen v2.0 server (<a href="#bb0250">Krogh et al. 2001</a>; <a href="#bb0115">Doytchinova and Flower 2007b</a>). The epitopes which have antigenic potency were picked and used for preceding analysis. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Flow chart summarizing the protocol of multi-epitope subunit vaccine design against SARS-CoV-2 through reverse vaccinology approach.</p>   </div> <p /> </div> <div class="conservancyanalysisandtoxicityprofilingofthepredictedepitopes" title="sec"><span class="label" tagx="label" title="label">2.5</span><div class="title" tagx="title" title="title">
Conservancy analysis and toxicity profiling of the predicted epitopes</div> <p id="p0080">The level of conservancy scrutinizes the ability of epitope candidates to impart capacious spectrum immunity. Homologous sequence sets of the chosen antigenic proteins were retrieved from the NCBI database by utilizing BLASTp tool. Later, conservancy analysis tool (<a href="http://tools.iedb.org/conservancy/">http://tools.iedb.org/conservancy/</a>) in IEDB was used to demonstrate the conservancy level of the predicted epitopes among different viral strains. The toxicity of non-allergenic epitopes was enumerated by using ToxinPred server (<a href="#bb0185">Gupta, 2013</a>).</p> </div> <div class="populationcoverageandallergenicitypatternofputativeepitopes" title="sec"><span class="label" tagx="label" title="label">2.6</span><div class="title" tagx="title" title="title">
Population coverage and allergenicity pattern of putative epitopes</div> <p id="p0085">Among different ethnic societies and geographic spaces, the HLA distribution varies around the world. Population coverage study was conducted by using IEDB population coverage calculation server (<a href="#bb0460">Vita et al., 2015</a>). To check the allergenicity of the proposed epitopes, four distinct servers i.e. AllergenFP (<a href="#bb0105">Dimitrov et al. 2014b</a>), AllerTOP (<a href="#bb0025">Dimitrov et al., 2014a</a>, <a href="#bb0105">Dimitrov et al., 2014b</a>), Allermatch (Fiers et al., and Allergen Online (<a href="http://www.allergenonline.org/">http://www.allergenonline.org/</a>) servers were utilized.</p> </div> <div class="identificationofb-cellepitopesandconservancyanalysis" title="sec"><span class="label" tagx="label" title="label">2.7</span><div class="title" tagx="title" title="title">
Identification of B-cell epitopes and conservancy analysis</div> <p id="p0090">Three different algorithms i.e. Bepipred Linear Epitope Prediction 2.0 (<a href="#bb0225">Jespersen et al. 2017</a>), Emini surface accessibility prediction (<a href="#bb0130">Emini et al. 1985</a>) and Kolaskar and Tongaonkar antigenicity scale (<a href="#bb0245">Kolaskar and Tongaonkar 1990</a>) from IEDB predicted the potential B-Cell epitopes within conserved fragments of the chosen viral proteins. To cross check the epitope conservancy, we used 30 strains from 30 different countries using World Health Organization’s most infected countries’ list. We demonstrated the conservancy for both B-cell and T-cell epitopes used to design the final vaccine construct in this study.</p> </div> <div class="constructionofvaccinemoleculesandpredictionofallergenicity,antigenicityandsolubilityoftheconstructs" title="sec"><span class="label" tagx="label" title="label">2.8</span><div class="title" tagx="title" title="title">
Construction of vaccine molecules and prediction of allergenicity, antigenicity and solubility of the constructs</div> <p id="p0095">Top CTL, HTL and B cell epitopes were compiled to design the final vaccine constructs in the study. Each vaccine constructs commenced with an adjuvant followed by top CTL epitopes, HTL epitopes and BCL epitopes respectively. For construction of novel corona vaccine, the chosen adjuvants i.e. L7/L12 ribosomal protein, beta defensin (a 45 mer peptide) and HABA protein (<i>M. tuberculosis</i>, accession number: AGV15514.1) were used (Rana and Akhter, 2016). Several linkers such as EAAAK, GGGS, GPGPG and KK in association with PADRE sequence were incorporated to construct fruitful vaccine sequences against COVID-19. The constructed vaccines were then analyzed whether they are non-allergenic by utilizing the following tool named Algpred (<a href="#bb0055">Azim et al., 2019a</a>, <a href="#bb0060">Azim et al., 2019b</a>). The most potential vaccine among the three constructs was then determined by assessing the antigenicity and solubility of the vaccines via VaxiJen v2.0 (<a href="#bb0115">Doytchinova and Flower 2007b</a>) and Proso II server (<a href="#bb0400">Smialowski et al., 2007</a>), respectively.</p> </div> <div class="physicochemicalcharacterizationandsecondarystructureanalysis" title="sec"><span class="label" tagx="label" title="label">2.9</span><div class="title" tagx="title" title="title">
Physicochemical characterization and secondary structure analysis</div> <p id="p0100">ProtParam tool (<a href="https://web.expasy.org/protparam/">https://web.expasy.org/protparam/</a>), provided by ExPASy server (<a href="#bb0200">Hasan et al. 2019c</a>) was used to functionally characterize (<a href="#bb0150">Gasteiger et al., 2005</a>) the vaccine constructs. The studied functional properties were isoelectric pH, molecular weight, aliphatic index, instability index, hydropathicity, estimated half-life, GRAVY values and other physicochemical characteristics. Alpha helix, beta sheet and coil structures of the vaccine constructs were analyzed through GOR4 secondary structure prediction method using Prabi (<a href="https://npsa-prabi.ibcp.fr/">https://npsa-prabi.ibcp.fr/</a>). In addition, Espript 3.0 (<a href="#bb0365">Robert and Gouet 2014</a>) was also used to predict the secondary structure of the stated protein sequences.</p> </div> <div class="homologymodeling,structurerefinement,validationanddisulfideengineering" title="sec"><span class="label" tagx="label" title="label">2.10</span><div class="title" tagx="title" title="title">
Homology modeling, structure refinement, validation and disulfide engineering</div> <p id="p0105">Vaccine 3D model was generated on the basis of percentage similarity between target protein and available template structures from PDB by using I-TASSER (<a href="#bb0330">Peng and Xu 2011</a>). The modelled structures were further refined via FG-MD refinement server. Structure validation was performed by Ramachandran plot assessment in RAMPAGE (<a href="#bb0195">Hasan et al. 2019b</a>). By utilizing DbD2 server, probable disulfide bonds were designed for the anticipated vaccine constructs (<a href="#bb0095">Craig and Dombkowski, 2013</a>). The value of energy was considered &amp;lt;2.5, while the chi3 value for the residue screening was chosen between −87 to +97 for the operation (<a href="#bb0195">Hasan et al. 2019b</a>).</p> </div> <div class="conformationalb-cellandifn-αinducingepitopesprediction" title="sec"><span class="label" tagx="label" title="label">2.11</span><div class="title" tagx="title" title="title">
Conformational B-cell and IFN-α inducing epitopes prediction</div> <p id="p0110">The B-cell epitopes of putative vaccine molecules were predicted via ElliPro server (<a href="http://tools.iedb.org/ellipro/">http://tools.iedb.org/ellipro/</a>) with minimum score 0.5 and maximum distance of 7 Å (Ponomarenko et al., 2004). Moreover, IFN- inducing epitopes within the vaccine were predicted using IFNepitope with motif and SVM hybrid detection strategy (Hajighahramani et al., 2017).</p> </div> <div class="moleculardynamicsandnormalmodeanalysis(nma)" title="sec"><span class="label" tagx="label" title="label">2.12</span><div class="title" tagx="title" title="title">
Molecular dynamics and normal mode analysis (NMA)</div> <p id="p0115">Normal mode analysis (NMA) was performed to predict the stability and large scale mobility of the vaccine protein. The iMod server determined the stability of construct V3 by comparing the essential dynamics to the normal modes of protein (<a href="#bb0040">Aalten et al., 1997</a>; <a href="#bb0500">Wüthrich et al., 1980</a>). It is a recommended alternative to costly atomistic simulation (<a href="#bb0445">Tama and Brooks 2006</a>; <a href="#bb0100">Cui and Bahar 2007</a>) and shows much quicker and efficient assessments than the typical molecular dynamics (MD) simulations tools (<a href="#bb0340">Prabhakar et al. 2016</a>; <a href="#bb0050">Awan et al. 2017</a>). The main-chain deformability was also predicted by measuring the efficacy of target molecule to deform at each of its residues. The motion stiffness was represented via eigenvalue, while the covariance matrix and elastic network model was also analyzed.</p> </div> <div class="protein-proteininteractionstudy" title="sec"><span class="label" tagx="label" title="label">2.13</span><div class="title" tagx="title" title="title">
Protein-protein interaction study</div> <p id="p0120">Patchdock server was prioritized for docking between different HLA alleles and the putative vaccine molecules (<a href="#bb0065">Azim et al. 2020</a>). In addition, the superior construct was also docked with different human immune receptors such as ACE 3, APN, DPP4 and TLR-8. The 3D structures of these receptors were retrieved from RCSB protein data bank. Detection of highest binding affinity between the putative vaccine molecules and the receptor was experimented based on the lowest interaction energy of the docked structure.</p> </div> <div class="codonadaptationandinsilicocloning" title="sec"><span class="label" tagx="label" title="label">2.14</span><div class="title" tagx="title" title="title">
Codon adaptation and in silico cloning</div> <p id="p0125">JCAT tool was utilized for codon adaptation in order to fasten the expression of vaccine construct V3 in <i>E. coli</i> strain K12. For this, some restriction enzymes (i.e. BglI and BglII), Rho independent transcription termination and prokaryote ribosome-binding site were put away from the work (<a href="#bb0175">Grote et al., 2005</a>). After that, the mRNA sequence of constructed V3 vaccine was ligated within BglI (401) and BglII (2187) restriction site at the C-terminal and N-terminal sites respectively. SnapGene tool was utilized for in silico restriction cloning (<a href="#bb0410">Solanki and Tiwari 2018</a>).</p> </div> </div> <div class="results" title="sec"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
Results</div> <div class="covid-19exhibitscloseancestralrelationtosars-cov-1andbat-coronavirus" title="sec"><span class="label" tagx="label" title="label">3.1</span><div class="title" tagx="title" title="title">
COVID-19 exhibits close ancestral relation to SARS-CoV-1 and bat-coronavirus</div> <p id="p0130">In the phylogenetic analysis, we introduced different coronavirus from three different genera: alpha-coronavirus, beta-coronavirus and gamma-coronavirus. Total 61 species of the corona virus covered 21 sub-genera (<a href="#ec0005">Table S1</a> and <a href="#f0010">Fig. 2</a>). These 61 species of corona viruses included 7 pathogenic species (<a href="#f0010">Fig. 2</a>), which are: COVID-19 or SARS-CoV-2, SARS-CoV-1 virus, MERS virus, HCoV-HKU1, HCoV-OC43, HCoV-NL63, HCoV-229E (<a href="#bb0140">Forni et al., 2017</a>; <a href="#bb0545">Zhou et al., 2020a</a>, <a href="#bb0550">Zhou et al., 2020b</a>; <a href="#bb0560">Zumla et al. 2016</a>). Among these, the first five species belong to the beta-coronavirus genera, while the last two belongs to the alpha-genera. Here, none of the members of Gammacoronavirus studied in this study were found to use human as potential host. Apart from the human coronaviruses, we introduced other coronaviruses which choose different species of bats, whale, turkey, rat, mink, ferret, swine, camel, rabbit, cow and others as host (<a href="#ec0005">Table S1</a>).</p> <p id="p0140">The phylogeny of these species clearly revealed two broad clades (<a href="#f0010">Fig. 2</a>), where first large clade contains Gammacoronavirus and Alpha-coronavirus genera, while the other belongs to the beta-coronavirus. Within the beta-coronavirus clade, we found three clear divisions. HCoV-HKU1 and HCoV-OC43 have been placed in the first clade, while in the second clade we found MERS coronavirus. COVID-19 or SARS-COV-2 formed clade with SARS-COV-1 and bat beta-coronaviruses (<a href="#f0010">Fig. 2</a>), which is consistent with the previous finding (<a href="#bb0075">Ceraolo and Giorgi 2020</a>). Though SARS-CoV-1 belongs to the same sub-genus as the COVID-19, the bat coronaviruses belong to two different sub-genera including Hibecovirus and Nobecovirus (<a href="#ec0045">File S1</a>).</p> </div> <div class="evolutionofspikeproteinsbasedondomain" title="sec"><span class="label" tagx="label" title="label">3.2</span><div class="title" tagx="title" title="title">
Evolution of spike proteins based on domain</div> <p id="p0150">Domain analysis of spike protein of coronaviruses reveals that they contain mainly one signature domains namely, coronavirus S2 glycoprotein (IPR002552), which is present in all the candidates. All other beta-coronavirus contains spike receptor binding protein (IPR018548), coronavirus spike glycoprotein hapted receptor 2 domain (IPR027400) and spike receptor binding domain superfamily (IPR036326). SARS-CoV-1 contains an extra domain, namely spike glycoprotein N-terminal domain (IPR032500), which is also present in some the sub-genera (Embecovirus) of beta-coronavirus, but not in COVID-19. One important finding in our study is that the COVID-19 candidates do not contain the domain spike glycoprotein (IPR042578), which is present in the SARS-CoV-1 (<a href="#f0015">Fig. 3</a>). The secondary structure prediction study shows a large numbers of cysteine residues which contribute to the formation of disulfide bonds within the spike protein. Most of them fall within the S1 spike protein, which is 654 amino acid long in SARS-CoV-1, while 672 amino acids long in COVID-19. The RGD motif which is conserved within the COVID-19 is present in the vicinity of the S1 protein. It exists as KGD that clearly demonstrates the mutation over the short time period. Again, the receptor binding domain and receptor binding motif analyses disclose variations within several region between the COVID19 and SARS-CoV-1 (<a href="#ec0050">File S2</a>). The domain-based phylogenetic analysis reflects two main divisions, where the all the novel beta-coronavirus i.e., COVID19 form clade with the SARS-CoV-1; while other beta-coronavirus fall in another clade which further divide to give rise different sub-genera. This clearly shows that the COVID-19 exerts specific ancestral connection to the SARS-CoV-1 in terms of spike glycoproteins. Interestingly, our study also revealed close relatedness of both the SARS-CoV-1 and COVID-19 to the bat beta-coronavirus that belongs to the Hibecovirus sub-genus. However, in our study, the bat coronaviruses of Nobecovirus sub-genus did not fall into the same clade of novel coronaviruses. The phylogenetic study and MSA also revealed that, the functional portion of the spike glycoprotein domain and spike glycoprotein N-terminal domain might be lost from the COVID-19 during the course of evolution.</p> </div> <div class="domainarchitectureandancestralstateofenvelopeproteins" title="sec"><span class="label" tagx="label" title="label">3.3</span><div class="title" tagx="title" title="title">
Domain architecture and ancestral state of envelope proteins</div> <p id="p0160">The envelope proteins of both beta-coronavirus and Alpha-coronavirus contain only one protein domain (IPR003873) namely, Nonstructural protein NS3 or small envelope protein E (NS3/E). This domain is well conserved in coronavirus and also found in murine hepatitis virus. On the other hands, the gamma coronavirus shows the exception, which possess (IPR005296) IBV3C protein domain, which thought to be expressed from the <i>ORF3C</i> gene of infectious bronchitis virus (<a href="#bb0230">Jia and Naqi 1997</a>) (<a href="#ec0045">File S1</a>, <a href="#ec0050">File S2</a>). The length of the domains for the COVID-19, SARS-CoV and MARS virus are 75, 76 and 82 amino acids, respectively. on the contrary, the length of the gamma coronavirus candidate in our study which utilizes turkey as host is 99 amino acids.</p> <p id="p0170">The average length of the COVID-19 envelope proteins is 75 amino acids long. The NS3/E protein domains span the whole length of the protein and possess mainly one transmembrane domain, one non-cytoplasmic domain and a cytoplasmic domain. However, some species from the sub-genera of Embecovirus shows two transmembrane domains (<a href="#f0020">Fig. 4</a>, <a href="#ec0045">File S1</a>, <a href="#ec0050">File S2</a>). While in our <i>in-silico</i> study, we found 2 transmembrane domains in SARS-CoV-1 and 2–3 transmembrane domains in MERS virus, previous experiments proved that both contains only one α-helical transmembrane domains (<a href="#bb0305">Nieto-Torres et al. 2011</a>; <a href="#bb0440">Surya et al. 2015</a>). Though the computational analysis of CoVID-19 envelope protein secondary structure shows 2 transmembrane domains, our domain analysis shows only one such domain in their structures. Again, the Turkey corona viruses also possess the same transmembrane, non-cytoplasmic domain and cytoplasmic domain, with a variation in the orientation. The domain-based phylogeny of the envelope proteins of novel corona viruses reveals close ancestral relationship with the SARS-CoV-1 and bat coronaviruses (<a href="#f0020">Fig. 4</a>). In spite to the previous findings, where it was found that the envelope proteins of the MERS virus and SARS-CoV-1 exerted close proximity in terms of secondary structure and functions (<a href="#bb0440">Surya et al. 2015</a>). Unlike to earlier finding, we got that gamma-coronavirus candidate in our study shows close connection with both SARS-CoV-1 and COVID-19 in terms of envelope proteins.</p> </div> <div class="domainarchitectureandphylogenyofmembraneproteins" title="sec"><span class="label" tagx="label" title="label">3.4</span><div class="title" tagx="title" title="title">
Domain architecture and phylogeny of membrane proteins</div> <p id="p0180">Membrane proteins of all the coronavirus mainly contain coronavirus M matrix/glycoprotein (IPR002574) domain family. However, the candidates of alpha- and gamma-coronaviruses contain M matrix/glycoprotein: Alpha-coronavirus (IPR042551) and M matrix/glycoprotein: gamma-coronavirus (IPR042550) domains, respectively. The next two domains belong to the M matrix/glycoprotein domain family. The membrane proteins of coronaviruses range from 221 to 230 amino acid long. Computational analysis of secondary structures shows some variations of COVID-19 with SARS-CoV-1 and MERS virus (<a href="#ec0055">File S3</a>). SARS-CoV-2 possesses alpha helical structure in their structure, while the other two completely devoid of this structure. Again, both SARS-CoV-1 and MARS contain parallel beta sheets, while it is completely absent in the novel corona virus (<a href="#ec0055">File S3</a>). The phylogenetic analysis of the membrane supports that the novel coronavirus is closely connected to the SARS-CoV-1 virus membrane protein. As well as it produced connections with bat coronaviruses of Hibecovirus and Nobecovirus sub-genus (<a href="#f0025">Fig. 5</a>).</p> </div> <div class="domain-basedphylogenyofnucleocapsidproteins" title="sec"><span class="label" tagx="label" title="label">3.5</span><div class="title" tagx="title" title="title">
Domain-based phylogeny of nucleocapsid proteins</div> <p id="p0190"> The length of nucleocapsid proteins of beta-coronavirus genus ranges from 410 to 450 amino acids. Three signature domains are mainly present in the nucleocapsid proteins, which are: Coronavirus Nucleocapsid protein (IPR001218), Nucleocapsid Proteins C-terminal (IPR037179) and Nucleocapsid Proteins N-terminal (IPR037195). However, in our experiment, we didn't find these domains in HCoV-HKU1 (<a href="#f0030">Fig. 6</a> ); this virus belongs to Embecovirus sub-genus and contains only Coronavirus Nucleocapsid I (IPR004876) domain. The candidates of alpha- and gamma-coronavirus have their special domains. According to our domain-based phylogeny study of nucleocapsid proteins, we found the close approximation of the COVID-19 with the SARS-CoV-1, which is consistent with the findings of phylogeny of whole genome. Strikingly, unlike spike and membrane proteins, the closest homologs of COVID-19 are not only the SARS-COV-1 and bat coronaviruses (Sub-genus: Hibecovirus and Nobecovirus), but it also includes the MERS viruses and other proteins which is from the Merbecovirus sub-genus. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Phylogeny of 61 species of coronaviruses. Seven pathogenic human coronaviruses have been represented by blue star and the IDs have been made bold. COVID-19 clade has been shown with red colour. SARS-CoV-1 and MERS virus have been represented by orange and blue colors, respectively. The genera have been represented on the left coloured labels. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Phylogeny of spike glycoprotein of coronavirus. The sub-genera have been labeled in the left table. The filled and unfilled circles show the presence and absence of the domains labeled on the top.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Phylogeny of envelope proteins of coronaviruses. The sub-genera under three different genera have been shown on the left labels. The star signs represent the COVID-19 virus. 5 different pathogenic human corona viruses have been shown in bold form, including the MERS virus (blue) and SARS-CoV-1 (yellow). (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>Phylogeny of membrane proteins of coronavirus. The sub-genera under three different genera have been shown on the left labels. The blue and orange star represent the SARS-CoV1 and MERS virus, respectively. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>Phylogeny of nucleocapsid proteins of coronavirus. The sub-genera under three different genera have been shown on the left labels. The filled and unfilled circles show the presence and absence of the domains labeled on the top. COVID-19, SARS-CoV1 and MERS viruses are clades are labeled with blue, green and red colors, respectively. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p>   </div> <p /> </div> <div class="identificationofconservedregionsasvaccinetarget" title="sec"><span class="label" tagx="label" title="label">3.6</span><div class="title" tagx="title" title="title">
Identification of conserved regions as vaccine target</div> <p id="p0195">A total 31, 24, 29 and 29 sequences of spike glycoprotein, membrane protein, envelope protein and nucleocapsid protein were retrieved from the NCBI database belonging to different strains of SARS-CoV-2. Following by BLASTp analysis and Multiple Sequence Alignment, two conserved regions were detected for membrane glycoprotein and nucleocapsid, while single fragments were identified for both spike glycoprotein and envelope protein (<a href="#t0005">Table 1</a> ).Results showed that, all the conserved sequences except one from membrane glycoprotein met the criteria of default threshold standard in VaxiJen. Again, transmembrane topology scrutinizing showed that among the immunogenic conserved sequences from the corresponding proteins except spike glycoprotein met the criteria of desired exomembrane characteristics (<a href="#t0005">Table 1</a>). <div class="table-wrap_UNKNOWN" position="float" id="t0005"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Identified conserved regions among different homologous protein sets.</p>  <div tag="alt-text">
Table 1</div> <table frame="hsides" rules="groups"> <thead> <tr> <th>Protein</th> <th>Conserved region</th> <th>Vaxijen score</th> <th>Topology</th> </tr> </thead> <tbody> <tr> <td>Spike glycoprotein</td> <td>NVYADSFVIRGDEVRQIAPGQTGKIADYNYKLPDD</td> <td>0.5471</td> <td>Inside</td> </tr> <tr> <td rowspan="2">Membrane protein</td> <td>LACFVLAAVYRINWITGGIAIAMACLVGLMWLSYFIASFRLFARTRSMWSFNPETNILL</td> <td>0.7352</td> <td>Outside</td> </tr> <tr> <td>NVPLHGTILTRPLLESELVIGAVILRGHLRIAGHHLGRCDIKDLPKEI</td> <td>0.2688</td> <td>Outside</td> </tr> <tr> <td>Envelope protein</td> <td>MYSFVSEETGTLIVNSVLLFLAFVVFLLVTLAILTALRLCAYCCNIVNVSLVKPSFYVYSRVKNLNSSRVPDLLV</td> <td>0.6025</td> <td>Outside</td> </tr> <tr> <td rowspan="2">Nucleocapsid protein</td> <td>MSDNGPQNQRNAPRITFGGPSDSTGSNQNGERSGARSKQRRPQGLPNNTASWFTALTQHGKEDLKFPRGQGVPINTNSSPDDQIGYYRRATRRIRGGDGKMKDLSPRWYFYYLGTGPEAGLPYGANKDGIIWVATEGALNTPKDHIGTRNPANNAAIVLQLPQGTTLPKGFYAEGSRGGSQASSRSSSRSRNS</td> <td>0.3985</td> <td>Outside</td> </tr> <tr> <td>RNSTPGSSRGTSPARMAGNGGDAALALLLLDRLNQLESKMSGKGQQQQGQTVTKKSAAEASKKPRQKRTATKAYNVTQAFGRRGPEQTQGNFGDQELIRQGTDYKHWPQIAQFAPSASAFFGMSRIGMEVTPSGTWLTYTGAIKLDDKDPNFKDQVILLNKHIDAYKTFPPTEPKKDKKKKADETQALPQRQKKQQTVTLLPAADLDDFSKQLQQSMSSADSTQA</td> <td>0.5965</td> <td>Outside</td> </tr> </tbody> </table>  <p /> </div> <div class="t-cellepitopeprediction,transmembranetopologyscreeningandantigenicityanalysis" title="sec"><span class="label" tagx="label" title="label">3.7</span><div class="title" tagx="title" title="title">
T-cell epitope prediction, transmembrane topology screening and antigenicity analysis</div> <p id="p0200">A plethora of immunogenic epitopes were generated from the conserved sequences that were able to bind with most noteworthy number of HLA cells (<a href="#ec0005">Table S1</a>, <a href="#ec0010">Table S2</a>, <a href="#ec0015">Table S3</a>, <a href="#ec0020">Table S4</a>). Top epitopes with exomembrane characteristics were ranked for each individual protein after investigating their antigenicity score and transmembrane topology (<a href="#t0010">Table 2</a>).</p> </div> <div class="conservancyanalysis,toxicityprofiling,populationcoverageandallergenicitypatternofthepredictedepitopes" title="sec"><span class="label" tagx="label" title="label">3.8</span><div class="title" tagx="title" title="title">
Conservancy analysis, toxicity profiling, population coverage and allergenicity pattern of the predicted epitopes</div> <p id="p0210">Epitopes from each protein showed high level of conservancy up to 100% (<a href="#t0010">Table 2</a>). ToxinPred server predicted the relative toxicity of each epitope which indicated that the top epitopes were non-toxin in nature (<a href="#ec0025">Table S5</a>). Population coverage of four structures proteins were also done for the predicted CTL and HTL epitopes. From the screening, results showed that population of the various geographic regions could be covered by the predicted T-cell epitopes (<a href="#f0035">Fig. 7</a>).Finally, the allergenic epitopes were excluded from the list based on the evaluation of four allergenicity prediction server (<a href="#ec0025">Table S5</a>).</p> </div> <div class="identificationofbcellepitopesandconservancyanalysis" title="sec"><span class="label" tagx="label" title="label">3.9</span><div class="title" tagx="title" title="title">
Identification of B cell epitopes and conservancy analysis</div> <p id="p0220"> Top B-cell epitopes were predicted for Spike glycoprotein, membrane protein, envelope protein and nucleocapsid protein using 3 distinct algorithms (i.e. Bepipred Linear Epitope prediction, Emini Surface Accessibility, Kolaskar and Tongaonkar Antigenicity prediction) from IEDB. Epitopes were also allowed to analyze their vaxijen scoring and allergenicity (<a href="#t0015">Table 3</a> ). Results revealed that top B-cell and T-cell epitopes were highly conserved (up to100%) among different strains of SARS-CoV-2 (<a href="#t0020">Table 4</a> ), while 19 out of final 20 epitopes showed conservancy &amp;gt; 90%. Therefore, the designed vaccine construct is expected to confer broad range immunity in the host. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 7</span></a></div>   <p>Population coverage analysis of spike protein (A), envelope protein (B), membrane protein (C) and nucleocapsid proteins (D).</p>   </div> <div class="table-wrap_UNKNOWN" position="float" id="t0010"><span class="label" tagx="label" title="label">Table 2</span> <p>Predicted T-cell (CTL and HTL) epitopes of Spike glycoprotein, membrane protein, envelope protein and nucleocapsid protein.</p>  <div tag="alt-text">
Table 2</div> <table frame="hsides" rules="groups"> <thead> <tr> <th>Types</th> <th>Proteins</th> <th>Epitope</th> <th>Start</th> <th>End</th> <th>Vaxijen score</th> <th>No. of HLAs</th> <th>Conservancy</th> </tr> </thead> <tbody> <tr> <td rowspan="31">CTL epitopes</td> <td>Spike glycoprotein</td> <td>ADYNYKLPD</td> <td>26</td> <td>34</td> <td>1.3382</td> <td>81</td> <td>100% (47/47)</td> </tr> <tr> <td rowspan="10">Membrane protein</td> <td>GIAIAMACL</td> <td>18</td> <td>26</td> <td>1.2059</td> <td>54</td> <td>10.6% (5/47)</td> </tr> <tr> <td>LACFVLAAV</td> <td>1</td> <td>9</td> <td>1.1825</td> <td>27</td> <td>93.6% (44/47)</td> </tr> <tr> <td>LVGLMWLSY</td> <td>26</td> <td>34</td> <td>1.0633</td> <td>54</td> <td>8.51% (4/47)</td> </tr> <tr> <td>LACFVLAAVY</td> <td>1</td> <td>10</td> <td>1.0354</td> <td>27</td> <td>91.5% (43/47)</td> </tr> <tr> <td>CLVGLMWLSY</td> <td>25</td> <td>34</td> <td>1.0255</td> <td>27</td> <td>8.51% (4/47)</td> </tr> <tr> <td>LVIGAVILR</td> <td>18</td> <td>26</td> <td>1.1027</td> <td>54</td> <td>8.51% (4/47)</td> </tr> <tr> <td>ELVIGAVILR</td> <td>17</td> <td>26</td> <td>0.9998</td> <td>27</td> <td>8.51% (4/47)</td> </tr> <tr> <td>ESELVIGAV</td> <td>15</td> <td>23</td> <td>0.9872</td> <td>54</td> <td>100% (47/47)</td> </tr> <tr> <td>IGAVILRGH</td> <td>20</td> <td>28</td> <td>0.9127</td> <td>27</td> <td>8.51% (4/47)</td> </tr> <tr> <td>LESELVIGA</td> <td>14</td> <td>22</td> <td>0.8597</td> <td>27</td> <td>23.4% (11/47)</td> </tr> <tr> <td rowspan="10">Envelope protein</td> <td>VKPSFYVYS</td> <td>52</td> <td>60</td> <td>1.0547</td> <td>27</td> <td>8.33% (3/36)</td> </tr> <tr> <td>FLLVTLAIL</td> <td>26</td> <td>34</td> <td>0.9645</td> <td>81</td> <td>88.9% (32/36)</td> </tr> <tr> <td>VVFLLVTLA</td> <td>24</td> <td>32</td> <td>0.9374</td> <td>27</td> <td>83.3% (30/36)</td> </tr> <tr> <td>LAILTALRL</td> <td>31</td> <td>39</td> <td>0.8872</td> <td>54</td> <td>91.7% (33/36)</td> </tr> <tr> <td>LNSSRVPDL</td> <td>65</td> <td>73</td> <td>0.8553</td> <td>54</td> <td>5.56% (2/36)</td> </tr> <tr> <td>LLFLAFVVF</td> <td>18</td> <td>26</td> <td>0.8144</td> <td>81</td> <td>63.9% (23/36)</td> </tr> <tr> <td>VFLLVTLAI</td> <td>25</td> <td>33</td> <td>0.8134</td> <td>54</td> <td>88.9% (32/36)</td> </tr> <tr> <td>LAFVVFLLV</td> <td>21</td> <td>29</td> <td>0.7976</td> <td>54</td> <td>72.2% (26/36)</td> </tr> <tr> <td>VSLVKPSFY</td> <td>49</td> <td>57</td> <td>0.7476</td> <td>81</td> <td>11.11% (4/36)</td> </tr> <tr> <td>FVVFLLVTL</td> <td>23</td> <td>31</td> <td>0.7403</td> <td>81</td> <td>83.3% (30/36)</td> </tr> <tr> <td>Nucleocapsid</td> <td>RSGARSKQR</td> <td>32</td> <td>40</td> <td>1.7874</td> <td>81</td> <td>6.41% (5/78)</td> </tr> <tr> <td>Protein</td> <td>DLSPRWYFY</td> <td>103</td> <td>111</td> <td>1.7645</td> <td>81</td> <td>7.69% (6/78)</td> </tr> <tr> <td /> <td>DGKMKDLSP</td> <td>98</td> <td>106</td> <td>1.7554</td> <td>27</td> <td>7.69% (6/78)</td> </tr> <tr> <td /> <td>KMKDLSPRW</td> <td>100</td> <td>108</td> <td>1.7462</td> <td>54</td> <td>7.69% (6/78)</td> </tr> <tr> <td /> <td>TQHGKEDLKF</td> <td>57</td> <td>66</td> <td>1.646</td> <td>54</td> <td>7.69% (6/78)</td> </tr> <tr> <td /> <td>KLDDKDPNF</td> <td>144</td> <td>152</td> <td>2.6591</td> <td>27</td> <td>93.6% (73/78)</td> </tr> <tr> <td /> <td>AIKLDDKDP</td> <td>142</td> <td>150</td> <td>2.167</td> <td>27</td> <td>8.97% (7/78)</td> </tr> <tr> <td /> <td>LDDKDPNFK</td> <td>145</td> <td>153</td> <td>1.9433</td> <td>54</td> <td>93.6% (73/78)</td> </tr> <tr> <td /> <td>GAIKLDDKDP</td> <td>141</td> <td>150</td> <td>1.9075</td> <td>27</td> <td>39.7% (31/78)</td> </tr> <tr> <td /> <td>TQGNFGDQE</td> <td>88</td> <td>96</td> <td>1.8694</td> <td>27</td> <td>8.97% (7/78)</td> </tr> <tr> <td rowspan="32" /> <td rowspan="2">Spike glycoprotein</td> <td>SFVIRGDEVRQIAPG</td> <td>6</td> <td>20</td> <td>0.5882</td> <td>27</td> <td>8.70% (8/92)</td> </tr> <tr> <td>DSFVIRGDEVRQIAP</td> <td>5</td> <td>19</td> <td>0.1792</td> <td /> <td>8.70% (8/92)</td> </tr> <tr> <td rowspan="10">Membrane glycoprotein</td> <td>LACFVLAAVYRINWI</td> <td>1</td> <td>15</td> <td>1.2905</td> <td>27</td> <td>8.51% (4/47)</td> </tr> <tr> <td>FVLAAVYRINWITGG</td> <td>4</td> <td>18</td> <td>1.0230</td> <td>27</td> <td>8.51% (4/47)</td> </tr> <tr> <td>AIAMACLVGLMWLSY</td> <td>20</td> <td>34</td> <td>0.9526</td> <td>27</td> <td>8.51% (4/47)</td> </tr> <tr> <td>IAIAMACLVGLMWLS</td> <td>19</td> <td>33</td> <td>0.9464</td> <td>27</td> <td>8.51% (4/47)</td> </tr> <tr> <td>ITGGIAIAMACLVGL</td> <td>15</td> <td>29</td> <td>0.9310</td> <td>27</td> <td>6.38% (3/47)</td> </tr> <tr> <td>LVIGAVILRGHLRIA</td> <td>18</td> <td>32</td> <td>0.8769</td> <td>27</td> <td>8.51% (4/47)</td> </tr> <tr> <td>ELVIGAVILRGHLRI</td> <td>17</td> <td>31</td> <td>0.7972</td> <td>27</td> <td>8.51% (4/47)</td> </tr> <tr> <td>PLLESELVIGAVILR</td> <td>12</td> <td>26</td> <td>0.7261</td> <td>27</td> <td>8.51% (4/47)</td> </tr> <tr> <td>SELVIGAVILRGHLR</td> <td>16</td> <td>30</td> <td>0.6768</td> <td>27</td> <td>8.51% (4/47)</td> </tr> <tr> <td>LESELVIGAVILRGH</td> <td>14</td> <td>28</td> <td>0.6528</td> <td>27</td> <td>8.51% (4/47)</td> </tr> <tr> <td rowspan="10">Envelope protein</td> <td>VTLAILTALRLCAYC</td> <td>29</td> <td>43</td> <td>0.8599</td> <td>27</td> <td>75.0% (27/36)</td> </tr> <tr> <td>LAFVVFLLVTLAILT</td> <td>21</td> <td>35</td> <td>0.8229</td> <td>27</td> <td>63.9% (23/36)</td> </tr> <tr> <td>LLFLAFVVFLLVTLA</td> <td>18</td> <td>32</td> <td>0.8122</td> <td>27</td> <td>58.3% (21/36)</td> </tr> <tr> <td>VSLVKPSFYVYSRVK</td> <td>49</td> <td>63</td> <td>0.7974</td> <td>27</td> <td>8.33% (3/36)</td> </tr> <tr> <td>VVFLLVTLAILTALR</td> <td>24</td> <td>38</td> <td>0.7559</td> <td>27</td> <td>77.8% (28/36)</td> </tr> <tr> <td>VNVSLVKPSFYVYSR</td> <td>47</td> <td>61</td> <td>0.7513</td> <td>27</td> <td>8.33% (3/36)</td> </tr> <tr> <td>FLAFVVFLLVTLAIL</td> <td>20</td> <td>34</td> <td>0.7476</td> <td>27</td> <td>63.9% (23/36)</td> </tr> <tr> <td>LFLAFVVFLLVTLAI</td> <td>19</td> <td>33</td> <td>0.7471</td> <td>27</td> <td>61.1% (22/36)</td> </tr> <tr> <td>ILTALRLCAYCCNIV</td> <td>33</td> <td>47</td> <td>0.7427</td> <td>27</td> <td>83.3% (30/36)</td> </tr> <tr> <td>LVKPSFYVYSRVKNL</td> <td>51</td> <td>65</td> <td>0.7311</td> <td>27</td> <td>8.33% (3/36)</td> </tr> <tr> <td rowspan="10">Nucleocapsid protein</td> <td>DLSPRWYFYYLGTGP</td> <td>103</td> <td>117</td> <td>1.5180</td> <td>27</td> <td>7.69% (6/78)</td> </tr> <tr> <td>LSPRWYFYYLGTGPE</td> <td>104</td> <td>118</td> <td>1.3086</td> <td>27</td> <td>100% (78/78)</td> </tr> <tr> <td>KDLSPRWYFYYLGTG</td> <td>102</td> <td>116</td> <td>1.2051</td> <td>27</td> <td>7.69% (6/78)</td> </tr> <tr> <td>GGDGKMKDLSPRWYF</td> <td>96</td> <td>110</td> <td>1.169</td> <td>27</td> <td>7.69% (6/78)</td> </tr> <tr> <td>GDGKMKDLSPRWYFY</td> <td>97</td> <td>11</td> <td>1.1013</td> <td>27</td> <td>7.69% (6/78)</td> </tr> <tr> <td>LDDKDPNFKDQVILL</td> <td>145</td> <td>159</td> <td>1.4829</td> <td>27</td> <td>8.97% (7/78)</td> </tr> <tr> <td>WLTYTGAIKLDDKDP</td> <td>136</td> <td>150</td> <td>1.2787</td> <td>27</td> <td>6.41% (5/78)</td> </tr> <tr> <td>DDKDPNFKDQVILLN</td> <td>146</td> <td>160</td> <td>1.2508</td> <td>27</td> <td>8.97% (7/78)</td> </tr> <tr> <td>AFFGMSRIGMEVTPS</td> <td>119</td> <td>133</td> <td>1.1085</td> <td>27</td> <td>83.3% (65/78)</td> </tr> <tr> <td>DPNFKDQVILLNKHI</td> <td>149</td> <td>163</td> <td>1.1072</td> <td>27</td> <td>8.97% (7/78)</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" position="float" id="t0015"><span class="label" tagx="label" title="label">Table 3</span> <p>Allergenicity and antigenicity assessment of predicted B-cell epitopes.</p>  <div tag="alt-text">
Table 3</div> <table frame="hsides" rules="groups"> <thead> <tr> <th>Protein</th> <th>Algorithms</th> <th>Top peptide sequence</th> <th>Allergenicity</th> <th>Vaxijen score</th> </tr> </thead> <tbody> <tr> <td rowspan="3">Spike glycoprotein</td> <td>Bepipred Linear Epitope Prediction 2.0</td> <td>IRGDEVRQIAPGQTGKIADYNYK</td> <td>Non-allergen</td> <td>1.0547</td> </tr> <tr> <td>Emini surface accessibility prediction</td> <td>GDEVRQ</td> <td>Non-allergen</td> <td>0.6701</td> </tr> <tr> <td>Kolaskar and Tongaonkar antigenicity</td> <td>VRQIAPG</td> <td>Non-allergen</td> <td>1.2611</td> </tr> <tr> <td rowspan="3">Membrane glycoprotein</td> <td>Bepipred Linear Epitope Prediction 2.0</td> <td>MWSFNPETN</td> <td>Non-allergen</td> <td>0.5509</td> </tr> <tr> <td>Emini surface accessibility prediction</td> <td>LFARTRSMWSFNPET</td> <td>Non-allergen</td> <td>0.9033</td> </tr> <tr> <td>Kolaskar and Tongaonkar antigenicity</td> <td>AMACLVGLM</td> <td>Non-allergen</td> <td>0.6251</td> </tr> <tr> <td rowspan="3">Envelope protein</td> <td>Bepipred Linear Epitope Prediction 2.0</td> <td>YVYSRVKNLNSSRVP</td> <td>Non-allergen</td> <td>0.4492</td> </tr> <tr> <td>Emini surface accessibility prediction</td> <td>PSFYVYSRVKNLNSSRVP</td> <td>Non-allergen</td> <td>0.5796</td> </tr> <tr> <td>Kolaskar and Tongaonkar antigenicity</td> <td>VNSVLLFLAFVVFLLVTLA</td> <td>Non-allergen</td> <td>0.5893</td> </tr> <tr> <td rowspan="3">Nucleocapsid protein</td> <td>Bepipred Linear Epitope Prediction 2.0</td> <td>PGSSRGTSPARMAGNGG</td> <td>Non-allergen</td> <td>0.3854</td> </tr> <tr> <td>Emini surface accessibility prediction</td> <td>TEPKKDKKKKAD</td> <td>Non-allergen</td> <td>0.2378</td> </tr> <tr> <td>Kolaskar and Tongaonkar antigenicity</td> <td>HWPQIAQFAPSASAF</td> <td>Non-allergen</td> <td>0.4320</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" position="float" id="t0020"><span class="label" tagx="label" title="label">Table 4</span> <p>Conservancy of the final epitopes among different SARS-CoV-2 strains (retrieved from the list of World Health Organization for top 30 infected regions).</p>  <div tag="alt-text">
Table 4</div> <table frame="hsides" rules="groups"> <thead> <tr> <th>Protein</th> <th>Epitope</th> <th>Conservancy</th> </tr> </thead> <tbody> <tr> <td rowspan="5">Spike glycoprotein</td> <td>ADYNYKLPD</td> <td>100.00% (31/31)</td> </tr> <tr> <td>SFVIRGDEVRQIAPG</td> <td>100.00% (31/31)</td> </tr> <tr> <td>IRGDEVRQIAPGQTGKIADYNYK</td> <td>100.00% (31/31)</td> </tr> <tr> <td>GDEVRQ</td> <td>100.00% (31/31)</td> </tr> <tr> <td>VRQIAPG</td> <td>100.00% (31/31)</td> </tr> <tr> <td rowspan="5">Envelope protein</td> <td>FVVFLLVTL</td> <td>96.30% (26/27)</td> </tr> <tr> <td>LCAYCCNIV</td> <td>92.59% (25/27)</td> </tr> <tr> <td>VTLAILTALRLCAYC</td> <td>96.30% (26/27)</td> </tr> <tr> <td>YVYSRVKNLNSSRVP</td> <td>92.59% (25/27)</td> </tr> <tr> <td>PSFYVYSRVKNLNSSRVP</td> <td>92.59% (25/27)</td> </tr> <tr> <td rowspan="5">Membrane glycoprotein</td> <td>LVIGAVILR</td> <td>96.67% (29/30)</td> </tr> <tr> <td>LACFVLAAVYRINWI</td> <td>100.00% (30/30)</td> </tr> <tr> <td>MWSFNPETN</td> <td>100.00% (30/30)</td> </tr> <tr> <td>LFARTRSMWSFNPET</td> <td>100.00% (30/30)</td> </tr> <tr> <td>AMACLVGLM</td> <td>100.00% (30/30)</td> </tr> <tr> <td rowspan="5">Nucleocapsid protein</td> <td>AIKLDDKDP</td> <td>93.33% (28/30)</td> </tr> <tr> <td>TEPKKDKKKKAD</td> <td>93.33% (28/30)</td> </tr> <tr> <td>HWPQIAQFAPSASAF</td> <td>93.33% (28/30)</td> </tr> <tr> <td>DLSPRWYFYYLGTGP</td> <td>93.33% (28/30)</td> </tr> <tr> <td>PGSSRGTSPARMAGNGG</td> <td>76.67% (23/30)</td> </tr> </tbody> </table> </div> <p /> </div> <div class="constructionofvaccinemoleculesandpredictionofallergenicity,antigenicityandsolubilityoftheconstructs" title="sec"><span class="label" tagx="label" title="label">3.10</span><div class="title" tagx="title" title="title">
Construction of vaccine molecules and prediction of allergenicity, antigenicity and solubility of the constructs</div> <p id="p0225">Three putative vaccine molecules (i.e. V1, V2 and V3) were constructed, each comprising a protein adjuvant, eight T-cell epitopes, twelve B-cell epitopes and respective linkers (<a href="#ec0030">Table S6</a>). PADRE sequence was included to extend the efficacy and potency of the constructed vaccine. The putative vaccine constructs, V1, V2 and V3 were 397, 481 and 510 residues long respectively. However, allergenicity score of V3 (−0.89886723) revealed that it was superior among the three constructs in terms safety and efficacy. V3 also had a solubility score (0.60) (<a href="#f0040">Fig. 8</a>E) and antigenicity (0.58) over threshold value (<a href="#t0025">Table 5</a>).</p> </div> <div class="physicochemicalcharacterizationandsecondarystructureanalysisoftheconstruct" title="sec"><span class="label" tagx="label" title="label">3.11</span><div class="title" tagx="title" title="title">
Physicochemical characterization and secondary structure analysis of the construct</div> <p id="p0235">ProtParam tool was employed to analyze the physicochemical properties of V3. Molecular weight of 3 was scored as 55.181 kDa. The extinction coefficient of V3 was calculated as 63,830 at 0.1% absorption. It had been found that the protein would have net negative charge which was higher than the recommended pI 9.81. Aliphatic index and GRAVY value were found 77.80 and −0.383 respectively, which could express the thermostability and hydrophilic status of the V3 vaccine construct. Around sixty minutes <i>in vitro</i> half-life stability in mammalian reticulocytes was predicted for V3. The computed instability index (II) 36.98 classified the protein as a stable one. In contrast, Secondary structure of V3 exhibited to have 46.47% alpha helix, 15.00% sheet and 38.63% coil structure (<a href="#f0070">Fig. S1</a>).</p> </div> <div class="homologymodeling,structurerefinement,validationanddisulfideengineering" title="sec"><span class="label" tagx="label" title="label">3.12</span><div class="title" tagx="title" title="title">
Homology modeling, structure refinement, validation and disulfide engineering</div> <p id="p0245">Tertiary structure of the putative vaccine construct V3 was generated using I-TASSER server (<a href="#f0040">Fig. 8</a>A and B). The server used 10 best templates with highest significant (measured via Z-score) from the LOMETS threading program to model the 3D structure. After refinement, Ramachandran plot analysis revealed that 92.7% and 5.7% residues were in the favored and allowed regions respectively, while only 8 residues (1.6%) occupied in the outlier region (<a href="#f0040">Fig. 8</a>C). The overall quality factor determined by ERRAT server was 91.56% (<a href="#f0040">Fig. 8</a>D). 3D modelled structure of V1 and V2 are shown in <a href="#f0075">Fig. S2</a>. DbD2 server recognized 33 pairs of amino acid residue with the potentiality to create disulfide bond between them. After analysis chi3 and B-factor parameter of residue pairs on the basis of energy, only 2 pairs (PRO 277-THR 329 and LEU 425-CYS 435) met the criteria for disulfide bond formation which were changed with cysteine (<a href="#f0080">Fig. S3</a>).</p> </div> <div class="conformationalb-cellandifn-inducingepitopesprediction" title="sec"><span class="label" tagx="label" title="label">3.13</span><div class="title" tagx="title" title="title">
Conformational B-cell and IFN- inducing epitopes prediction</div> <p id="p0255">Ellipro server predicted a total 6 conformational B-cell epitopes from the 3D structure of the construct V3. Epitopes No. 1 were considered as the broadest conformational B cell epitopes with 25 amino acid residues (<a href="#f0045">Fig. 9</a> and <a href="#ec0035">Table S7</a>). Results also revealed that predicted linear epitopes from 76 to 101, 131–143 and 48–55 were included in the conformational B-cell epitopes. Moreover, the sequence of the final vaccine was scanned for 15-mer IFN-inducing epitopes. Results showed that there were 292 positive IFN- inducing epitopes from which 20 had a score ≥ 5 (<a href="#ec0040">Table S8</a>). Residues of 194–209 regions (GGGSLVIGAVILRGG) in the vaccine showed highest score of 17 (<a href="#ec0040">Table S8</a>).</p> </div> <div class="moleculardynamicsandnormalmodeanalysis" title="sec"><span class="label" tagx="label" title="label">3.14</span><div class="title" tagx="title" title="title">
Molecular dynamics and normal mode analysis</div> <p id="p0265"> Stability of the vaccine construct V3 was investigated through mobility analysis (<a href="#f0050">Fig. 10</a>A and B), B-factor, eigenvalue and deformability analysis, covariance map and recommended elastic network model. Results revealed that the placements of hinges in the chain was insignificant (<a href="#f0050">Fig. 10</a>C) and the B-factor column gave an averaged RMS (<a href="#f0050">Fig. 10</a>D). The estimated higher eigenvalue 6.341333e<sup>−06</sup> (<a href="#f0050">Fig. 10</a>E) indicated low chance of deformation of vaccine protein V3. The correlation matrix and elasticity of the construct have been shown in <a href="#f0050">Fig. 10</a>G and <a href="#f0050">Fig. 10</a>H, respectively. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 8</span></a></div>   <p>Homology modeling, structure validation and solubility prediction of construct V3 (A: Cartoon structure, B: Surface structure, C: Ramachandran Plot analysis, D: Quality factor analysis, E: Solubility analysis).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 9</span></a></div>   <p>Predicted conformational epitopes (A and B) and linear epitopes (C and D) within construct V3.</p>   </div> <div class="table-wrap_UNKNOWN" position="float" id="t0025"><span class="label" tagx="label" title="label">Table 5</span> <p>Allergenicity, antigenicity and solubility prediction of designed vaccine constructs.</p>  <div tag="alt-text">
Table 5</div> <table frame="hsides" rules="groups"> <thead> <tr> <th>Vaccine constructs</th> <th>Composition</th> <th>Complete sequence of vaccine construct</th> <th>Allergenecity (Threshold −0.4)</th> <th>Antigenicit (Threshold 0.4)</th> <th>Solubility (Threshold 0.45)</th> </tr> </thead> <tbody> <tr> <td>V1</td> <td>Predicted CTL,HTL and BCL epitopes with defensin adjuvant and PADRE sequence</td> <td><b>EAAAK</b>GIINTLQKYYCRVRGGRCAVLSCLPKEEQIGKCSTRGRKCCRRKK<b>EAAAK</b>AKFVAAWTLKAAA<b>GGGS</b>ADYNYKLPD<b>GGGSLVIGAVILRGGGS</b>FVVFLLVTL<b>GGGSAIKLDDKDPGPGPG</b>SFVIRGDEVRQIAPG<b>GPGPGLACFVLAAVYRINWIGPGPG</b>VTLAILTALRLCAYC<b>GPGPG</b>DLSPRWYFYYLGTGP<b>KK</b>IRGDEVRQIAPGQTGKIADYNYK<b>KK</b>GDEVRQ<b>KK</b>VRQIAPG<b>KK</b>MWSFNPETN<b>KK</b>LFARTRSMWSFNPET<b>KK</b>AMACLVGLM<b>KK</b>YVYSRVKNLNSSRVP<b>KK</b>PSFYVYSRVKNLNSSRVP<b>KK</b>LCAYCCNIV<b>KK</b>PGSSRGTSPARMAGGG<b>KK</b>TEPKKDKKKKAD<b>KK</b>HWPQIAQFAPSASAF<b>KK</b>AKFVAAWTLKAAA<b>GGGS</b></td> <td>−0.17698947</td> <td>0.60</td> <td>0.61</td> </tr> <tr> <td>V2</td> <td>Predicted CTL, HTL and BCL epitopes with L7/L12 ribosomal protein adjuvant and PADRE sequence</td> <td><b>EAAK</b>MAKLSTDELLDAFKEMTLLELSDFVKKFEETFEVTAAAPVAVAAAGAAPAGAAVEAAEEQSEFDVILEAAGDKKIGVIKVVREIVSGLGLKEAKDLVDGAPKPLLEKVAKEAADEAKAKLEAAGATVTVK<b>EAAAK</b>AKFVAAWTLKAAA<b>GGGS</b>ADYNYKLPD<b>GGGSLVIGAVILRGGGS</b>FVVFLLVTL<b>GGGSAIKLDDKDPGPGPG</b>SFVIRGDEVRQIAPG<b>GPGPGLACFVLAAVYRINWIGPGPG</b>VTLAILTALRLCAYC<b>GPGPG</b>DLSPRWYFYYLGTGP<b>KK</b>IRGDEVRQIAPGQTGKIADYNYK<b>KK</b>GDEVRQ<b>KK</b>VRQIAPG<b>KK</b>MWSFNPETN<b>KK</b>LFARTRSMWSFNPET<b>KK</b>AMACLVGLM<b>KK</b>YVYSRVKNLNSSRVP<b>KK</b>PSFYVYSRVKNLNSSRVP<b>KK</b>LCAYCCNIV<b>KK</b>PGSSRGTSPARMAGGG<b>KK</b>TEPKKDKKKKAD<b>KK</b>HWPQIAQFAPSASAF<b>KK</b>AKFVAAWTLKAAA<b>GGGS</b></td> <td>0.13269078</td> <td>0.55</td> <td>0.64</td> </tr> <tr> <td>V3</td> <td>Predicted CTL, HTL and BCL epitopes with HABA adjuvant and PADRE sequence</td> <td><b>EAAK</b>MAENPNIDDLPAPLLAALGAADLALATVNDLIANLRERAEETRAETRTRVEERRARLTKFQEDLPEQFIELRDKFTTEELRKAAEGYLEAATNRYNELVERGEAALQRLRSQTAFEDASARAEGYVDQAVELTQEALGTVASQTRAVGERAAKLVGIEL<b>EAAAK</b>AKFVAAWTLKAAA<b>GGGS</b>ADYNYKLPD<b>GGGSLVIGAVILRGGGS</b>FVVFLLVTL<b>GGGSAIKLDDKDPGPGPG</b>SFVIRGDEVRQIAPG<b>GPGPGLACFVLAAVYRINWIGPGPG</b>VTLAILTALRLCAYC<b>GPGPG</b>DLSPRWYFYYLGTGP<b>KK</b>IRGDEVRQIAPGQTGKIADYNYK<b>KK</b>GDEVRQ<b>KK</b>VRQIAPG<b>KK</b>MWSFNPETN<b>KK</b>LFARTRSMWSFNPET<b>KK</b>AMACLVGLM<b>KK</b>YVYSRVKNLNSSRVP<b>KK</b>PSFYVYSRVKNLNSSRVP<b>KK</b>LCAYCCNIV<b>KK</b>PGSSRGTSPARMAGGG<b>KK</b>TEPKKDKKKKAD<b>KK</b>HWPQIAQFAPSASAF<b>KK</b>AKFVAAWTLKAAA<b>GGGS</b></td> <td>−0.89886723</td> <td>0.58</td> <td>0.60</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>The bold sequences are linker sequences</p> </div> </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 10</span></a></div>   <p>Normal Mode Analysis (NMA) of vaccine protein V3. The directions of each residues are given by arrows and the length of the line represented the degree of mobility in the 3D model (A and B). The main-chain deformability derived from high deformability regions indicated by hinges in the chain which are negligible (C). The experimental B-factor was taken from the corresponding PDB field and calculated from NMA (D). The eigenvalue represents the motion stiffness and directly related to the energy required to deform the structure (E). The variance associated to each normal mode is inversely related to the eigenvalue. Coloured bars show the individual (red) and cummulative (green) variances (F). The covariance matrix indicates coupling between pairs of residues, where they may be associated with correlated, uncorrelated or anti-correlated motions, indicated by red, white and blue colors respectively (G). The elastic network model identifies the pairs of atoms connected via springs. Each dot in the diagram is coloured based on extent of stiffness between the corresponding pair of atoms. The darker the greys, the stiffer the springs (H). (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p>   </div> <p /> </div> <div class="protein-proteindocking" title="sec"><span class="label" tagx="label" title="label">3.15</span><div class="title" tagx="title" title="title">
Protein-protein docking</div> <p id="p0270">The structural interaction between HLA alleles and the designed vaccines were investigated by molecular docking approach. The server detected the complexed structure by focusing on complementary score, ACE (Atomic Contact Energy) and estimated interface area of the compound (<a href="#t0030">Table 6</a> ). The molecular affinity between the putative vaccine molecules V3 and several immune receptors were also experimented. The result showed that construct V3 interacted with each receptor with significantly lower binding energy (<a href="#f0055">Fig. 11</a> ). <div class="table-wrap_UNKNOWN" position="float" id="t0030"><span class="label" tagx="label" title="label">Table 6</span> </div></p><p>Docking score of vaccine construct V3 with different HLA alleles and human immune receptors.</p>  <div tag="alt-text">
Table 6</div> <table frame="hsides" rules="groups"> <thead> <tr> <th>Vaccine construct</th> <th>PDB ID's HLAs</th> <th>Global energy</th> <th>Hydrogen bond energy</th> <th>ACE</th> <th>Score</th> <th>Area</th> </tr> </thead> <tbody> <tr> <td rowspan="6">V1</td> <td><a href="pdb:1a6a">1a6a</a></td> <td>−11.07</td> <td>−2.61</td> <td>8.85</td> <td>17,640</td> <td>2211.50</td> </tr> <tr> <td><a href="pdb:1h15">1h15</a></td> <td>−17.51</td> <td>−1.51</td> <td>4.35</td> <td>15,862</td> <td>2302.20</td> </tr> <tr> <td><a href="pdb:2fse">2fse</a></td> <td>−5.38</td> <td>0.00</td> <td>5.61</td> <td>16,720</td> <td>2493.50</td> </tr> <tr> <td><a href="pdb:2q6w">2q6w</a></td> <td>−10.48</td> <td>−0.39</td> <td>−1.45</td> <td>15,348</td> <td>2289.00</td> </tr> <tr> <td><a href="pdb:2seb">2seb</a></td> <td>9.03</td> <td>0.00</td> <td>0.20</td> <td>17,640</td> <td>2903.00</td> </tr> <tr> <td><a href="pdb:3c5j">3c5j</a></td> <td>−25.18</td> <td>−4.60</td> <td>8.20</td> <td>16,804</td> <td>3022.80</td> </tr> <tr> <td rowspan="6">V2</td> <td><a href="pdb:1a6a">1a6a</a></td> <td>−37.24</td> <td>−3.11</td> <td>−9.54</td> <td>19,666</td> <td>2703.90</td> </tr> <tr> <td><a href="pdb:1h15">1h15</a></td> <td>−9.33</td> <td>0.00</td> <td>4.33</td> <td>16,132</td> <td>2312.70</td> </tr> <tr> <td><a href="pdb:2fse">2fse</a></td> <td>−0.33</td> <td>0.00</td> <td>−0.60</td> <td>16,702</td> <td>2956.10</td> </tr> <tr> <td><a href="pdb:2q6w">2q6w</a></td> <td>−10.85</td> <td>−3.06</td> <td>1.20</td> <td>15,062</td> <td>1924.40</td> </tr> <tr> <td><a href="pdb:2seb">2seb</a></td> <td>1.38</td> <td>0.00</td> <td>−0.79</td> <td>18,008</td> <td>3034.30</td> </tr> <tr> <td><a href="pdb:3c5j">3c5j</a></td> <td>10.58</td> <td>−1.56</td> <td>5.54</td> <td>16,158</td> <td>2385.30</td> </tr> <tr> <td rowspan="6">V3</td> <td><a href="pdb:1a6a">1a6a</a></td> <td>−4.14</td> <td>−1.49</td> <td>12.24</td> <td>14,358</td> <td>2208.30</td> </tr> <tr> <td><a href="pdb:1h15">1h15</a></td> <td>−3.15</td> <td>−4.45</td> <td>10.93</td> <td>13,908</td> <td>1637.70</td> </tr> <tr> <td><a href="pdb:2fse">2fse</a></td> <td>−8.68</td> <td>−2.76</td> <td>9.98</td> <td>18,340</td> <td>3024.40</td> </tr> <tr> <td><a href="pdb:2q6w">2q6w</a></td> <td>−21.02</td> <td>−2.41</td> <td>5.05</td> <td>17,060</td> <td>2626.90</td> </tr> <tr> <td><a href="pdb:2seb">2seb</a></td> <td>−17.54</td> <td>−5.91</td> <td>3.87</td> <td>16,094</td> <td>2946.80</td> </tr> <tr> <td><a href="pdb:3c5j">3c5j</a></td> <td>−2.30</td> <td>0.00</td> <td>−2.30</td> <td>18,550</td> <td>3041.00</td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 11</span></a></div>   <p>Docked complex of vaccine construct V3 with human ACE 2 (A), TLR- (B), DPP4 (C) and APN (D).</p>   </div> <p /> </div> <div class="codonadaptationandinsilicocloning" title="sec"><span class="label" tagx="label" title="label">3.16</span><div class="title" tagx="title" title="title">
Codon adaptation and in silico cloning</div> <p id="p0275">The Codon Adaptation Index (CAI) and GC content for the predicted codons of the putative vaccine constructs V1 were demonstrated as 1.0 and 51.56% respectively. An insert of 1542 bp was found which lacked the restriction sites for BglI and BglII, thus providing comfort zone for cloning. The codons were inserted into pET28a(+) vector alongside two restriction sites (BglI and BglII) and a clone of 5125 base pair was generated (<a href="#f0060">Fig. 12</a> ). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 12</span></a></div>   <p>In silico restriction cloning of the gene sequence of final vaccine construct V3 into pET28a(+) expression vector (A: Restriction digestion of the vector pET28a(+) and construct V3 with BglII and BglI, B: Inserted desired fragment (V3 Construct) between BglII (401) and BglI (1943) indicated in red colour. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
Discussion</div> <p id="p0280">In December 2019, a new coronavirus prevalence flourished in Wuhan, China, causing clutter among the medical community, as well as to the rest of the world (<a href="#bb0430">Sun et al., 2020a</a>, <a href="#bb0435">Sun et al., 2020b</a>). The new species has been renamed as 2019-nCoV or, SARS-CoV-2, already causing considerable number infections and deaths in China, Italy, Spain, Iran, USA and to a growing degree throughout the world. The major outbreak and spread of SARS-CoV-2 in 2020 forced the scientific community to make considerable investment and research activity for developing a vaccine against the pathogen. However, owing to high infectivity and pathogenicity, the culture of SARS-CoV-2 needs biosafety level 3 conditions, which may obstructed the rapid development of any vaccine or therapeutics. It had been found that about 35 companies and academic institutions are engaged in such works (<a href="#bb0010">Spinney et al., 2020</a>, <a href="#bb0020">Ziady, 2020</a>). Among the potential SARS-CoV-2 vaccines in the pipeline, four have nucleic acid based designs, four involve non-replicating viruses or protein constructs, two contain live attenuated virus and one involves a viral vector (<a href="#bb0325">Pang et al. 2020</a>), while only one, called mRNA-1273 (developed by NIAID collaboration with Moderna, Inc.), has confirmed to start phase-1 trial (<a href="#bb0030">NIH News Event, 2020</a>). However, in this study we emphasized on a different approaches by prioritizing the advantages of different genome and proteome database using the immunoinformatic approach. Computational vaccine predictions were adopted by the researchers to design vaccines against both MERS-CoV (<a href="#bb0425">Sudhakar et al. 2013</a>; <a href="#bb0135">Fernando et al., 2013</a>) and SARS-CoV-1 (; <a href="#bb0310">Oany et al., 2014</a>), targeting the outer membrane or functional proteins (<a href="#bb0385">Sharmin and Islam 2014</a>). Several in silico strategies have also been employed to predict potential T cell and B cell epitopes against SARS-CoV-2, either emphasizing on spike glycoprotein or envelope proteins (<a href="#bb0070">Behbahani 2020</a>; <a href="#bb0360">Rasheed et al., 2020</a>). None of the studies, however, focused on other structural proteins. Moreover, random genetic changes and mutations in the protein sequences (<a href="#bb0520">Yin, 2020</a>) may obstruct the development of effective vaccines and therapeutics against human coronavirus in the future. Hence, the present study was employed to identify the similarity and divergence among the close relatives of the target pathogen and develop a novel chimeric recombinant vaccine considering all major structural proteins i.e. spike glycoprotein, membrane glycoprotein, envelope protein and nucleocapsid protein simultaneously.</p> <p id="p0285">Structural proteins are often chosen as attractive therapeutic targets by the researchers due to their surface exposure. Spike (S) protein is currently the most promising antigen formulation for SARS-CoV-2 vaccine research which is able to be directly recognized by host immune system (<a href="#bb0485">Wrapp et al. 2020</a>) and already been used for vaccine development to prevent Severe Acute Respiratory Syndrome (SARS) and Middle-East Respiratory Syndrome (MERS) (<a href="#bb0540">Zhou et al. 2018</a>; <a href="#bb0120">Du et al. 2009</a>). However, Membrane (M) protein, the most abundant one on the virus surface (<a href="#bb0300">Neuman et al. 2011</a>), was also reported to stimulate efficient neutralizing antibodies in SARS patients upon immunization (<a href="#bb0320">Pang et al. 2004</a>). Moreover, structural investigation revealed the existence of T-cell epitope cluster on the trans-membrane domain of M protein with the ability to induce a strong cellular immune response (<a href="#bb0280">Liu et al. 2010</a>). Thus, it can be utilized as a candidate antigen for developing SARS-CoV-2 vaccine. Researchers have already considered Envelope (E) protein for vaccine candidacy against COVID-19 due to high antigenicity (<a href="#bb0045">Abdelmageed et al. 2020</a>). It was also investigated against SARS-CoV in 2003 and more recently, against MERS-CoV (<a href="#bb0380">Schoeman and Fielding 2019</a>). Nucleocapsid (N) protein, on the other hand, is believed to be more conserved than spike or membrane glycoprotein with the potential to elicit broad-based cellular responses (<a href="#bb0530">Zhao et al. 2005</a>). Previously, cellular response against the N protein of some animal coronaviruses enhanced recovery from viral infection (<a href="#bb0160">Glansbeek et al. 2002</a>; <a href="#bb0470">Wasmoen et al., 1995</a>; <a href="#bb0480">Wesseling et al. 1993</a>). A study showed that N protein of SARS-CoV act as an important B cell immunogen, indicating its potential value in vaccine development against SARS (<a href="#bb0530">Zhao et al. 2005</a>). All these data suggest that S, E, M and N protein can induce both humoral and cellular immune responses against SARS-CoV-2.</p> <p id="p0290">Phylogenetic analysis is used for fundamental as well as applied issues of virus research and it includes the development of antiviral drugs and vaccines (<a href="#bb0165">Gorbalenya and Lauber 2017</a>). Here, we used phylogeny data to perform similarity analysis between different coronavirus subfamiles to finalize which structural protein to be used as vaccine target in this study. We wanted to target the proteins which are most common in each subfamily and exhibit significant similarities to design a broad and universally effective vaccine candidate. The results revealed that each of the four major structural proteins possess highly conserved domains among alpha-, beta- and gamma-coronaviruses. Therefore we considered all major structural proteins to design a multiepitope subunit vaccine which is expected to stimulate more effective and broad response in the host. The topology of the phylogenetic trees of the whole genome and the stated four proteins sequences from different species of coronaviruses reveal that SARS-CoV-1 and bat coronaviruses are the closest homologs of the novel coronaviruses. Our results infer a significant level of similarities within the COVID-19 and SARS-CoV-1 which was also aligned with the previous findings (<a href="#bb0220">Jaimes et al., 2020</a>; <a href="#bb0430">Sun et al., 2020a</a>, <a href="#bb0435">Sun et al., 2020b</a>; <a href="#bb0490">Wu 2020</a>). The sequence similarities between the SARS-CoV, bat coronaviruses and the COVID-19 from the reported studies (<a href="#bb0205">Hu et al. 2018</a>; <a href="#bb0495">Wu et al. 2020</a>; <a href="#bb0490">Wu 2020</a>) suggests that those are distantly related, in spite those are capable of infecting the humans and therefore possess the adaptive convergent evolution. Interestingly, the COVID-19 envelope proteins form clade with the Turkey coronavirus which belongs to gamma-coronavirus genus. So, in terms of envelope proteins, the envelope gene of turkey coronavirus might contribute to the convergence process, which needs further analysis. In addition, from the domain-based phylogeny of nucleocapsid proteins, it can be deduced that this protein might have originated in bats and was transmitted to camels and then later on choose human as the potential host. Overall, the COVID-19 might go through complex adaptation strategies in order to be transmitted into the human via different animals.</p> <p id="p0295">The homologous protein sets for four structural proteins of Coronavirus were sorted to identify conserved regions through BLASTp analysis and MSA. Only the conserved sequences were utilized to identify potential B-cell and T-cell epitopes for each individual protein (<a href="#t0005">Table 1</a>). Thus, our constructs are expected to stimulate a broad-spectrum immunity in host upon administration. Cytotoxic CD8 + T lymphocytes (CTL) play a crucial role to control the spread of pathogens by recognizing and killing diseased cells or by means of antiviral cytokine secretion (<a href="#bb0145">Garcia et al. 1999</a>). Thus, T cell epitope-based vaccination is a unique process to confer defensive response against pathogenic candidates (<a href="#bb0390">Shrestha and Diamond, 2004</a>). Approximately 800 MHC-I peptides (CTL epitopes) and 600 MHC-II peptides (HTL epitopes) were predicted via IEDB server, from which we screened the top ones through analyzing the antigenicity score, trans-membrane topology, conservancy level and other important physiochemical parameters employing a number of bioinformatics tools (<a href="#t0010">Table 2</a>). The top 10 epitopes from each protein was further assessed by investigating the toxicity profile and allergenicity pattern. Different servers rely on different parameters to predict the allergenic nature of small peptides. Therefore, we used 4 distinct servers for such assessment and the epitopes predicted as non-allergen at least via 3 servers were retained for further analysis (<a href="#ec0025">Table S5</a>). Vaccine initiates the generation of effective antibodies that are usually produced by B cells and plays effector functions by targeting specifically to a foreign particles (<a href="#bb0090">Cooper and Nemerow 1984</a>). The potential B cell epitopes were generated by three different algorithms (Bepipred linear epitope prediction 2.0, Kolaskar and Tongaonkar antigenicity prediction and Emini surface accessibility prediction) from IEDB database (<a href="#t0015">Table 3</a>).</p> <p id="p0305">Suitable linkers and adjuvants were used to combine top finalized epitopes from each protein that led to develop multiepitope vaccine molecules (<a href="#ec0030">Table S6</a>). As PADRE sequence was usually recommended to lessen the polymorphism of HLA molecules in the population (<a href="#bb0155">Ghaffari-Nazari et al. 2015</a>), it was also considered to construct the final vaccine molecule. Here, adjuvants would enhance the immunogenicity of the vaccine constructs and appropriate separation of epitopes in the host environment would be ensured by the linker (<a href="#bb0515">Yang et al. 2015</a>). Allergenicity, physiochemical properties, antigenicity and three-dimensional structure of vaccine constructs were characterized, and it was concluded that V3 was superior to V1 and V2 vaccine constructs due to its highly negative allergenicity score (−0.89886723). Therefore, it is expected to be less likely to cause adverse side effects or hypersensitivity. V3 also had a solubility score (0.60) and antigenicity (0.58) over threshold value (<a href="#t0025">Table 5</a>). Hence, construct V3 was proposed as the most suitable and safe vaccine protein for further analysis.</p> <p id="p0315">The final construct also occupied by several interferon-α producing epitopes (<a href="#ec0040">Table S8</a>). The vaccine protein (V3) was subjected to di-sulfide engineering to enhance its stability. Analysis of the normal modes in internal coordinates by iMODS was employed to investigate the collective motion of vaccine molecules (<a href="#bb0285">López-Blanco et al., 2014</a>). Negligible chance of deformability at molecular level was analyzed for the putative vaccine construct V3, thereby strengthening our prediction. Moreover, molecular docking was investigated to analyze the molecular affinity of the vaccine with different HLA molecules i.e. DRB1*0101, DRB5*0101, DRB3*0202, DRB1*0401, DRB3*0101 and DRB1*0301 (<a href="#t0030">Table 6</a>). It had been reported that a specific receptor-binding domain of CoV spike protein usually recognizes its host receptor ACE2 (angiotensin-converting enzyme 2) (<a href="#bb0270">Li et al., 2003</a>; <a href="#bb0265">Li 2015</a>). Previous studies also identified dipeptidyl peptidase 4 (DPP4) as a functional receptor for human coronavirus (<a href="#bb0350">Raj et al. 2013</a>). Therefore, we performed another docking study prioritizing these immune receptors to strengthen our prediction (<a href="#f0055">Fig. 11</a>). Results showed that the designed construct bound with the selected receptors with minimum binding energy which was biologically significant. Finally, <i>in-silico</i> restriction cloning was adopted to check the suitability of construct V3 for entry into pET28a (+) vector and expression in <i>E. coli</i> strain K12 (<a href="#f0060">Fig. 12</a>).</p> <p id="p0325">Traditional ways to vaccine development are time consuming and laborious. Moreover, the result may not be always as expected or fruitful (<a href="#bb0420">Stratton et al., 2003</a>; <a href="#bb0190">Hasan et al., 2019a</a>, <a href="#bb0195">Hasan et al., 2019b</a>). In silico prediction and prescreening methods, on the contrary, offer some advantages while saving time and cost for production. Therefore, the present study may aid in the development of preventive strategies and novel vaccines to combat infections caused by 2019-nCoV. However, further wet lab trials involving model organism needs to be experimented for validating our findings.</p> <p id="p0330">The following are the supplementary data related to this article </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. S1</span></a></div>   <p>Secondary structure prediction of constructed vaccine protein V3.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. S2</span></a></div>   <p>3D modelled structure of vaccine protein V1 and V2.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. S3</span></a></div>   <p>Disulfide engineering of vaccine protein V3 (A: Initial form, B: Mutant form).</p>   </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Table S1</div> <p>Predicted CTL and HTL epitopes of spike glycoprotein.</p>  <div class="media" title="media"><a href="mmc4.docx">LINK</a><div tag="alt-text">
Table S1</div> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Table S2</div> <p>Predicted CTL and HTL epitopes of membrane.</p>  <div class="media" title="media"><a href="mmc5.docx">LINK</a><div tag="alt-text">
Table S2</div> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Table S3</div> <p>Predicted CTL and HTL epitopes of envelope protein</p>  <div class="media" title="media"><a href="mmc6.docx">LINK</a><div tag="alt-text">
Table S3</div> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Table S4</div> <p>Predicted CTL and HTL epitopes of nucleocapsid protein.</p>  <div class="media" title="media"><a href="mmc7.docx">LINK</a><div tag="alt-text">
Table S4</div> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Table S5</div> <p>Allergenicity pattern and toxicity profile of top T cell epitopes.</p>  <div class="media" title="media"><a href="mmc8.docx">LINK</a><div tag="alt-text">
Table S5</div> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Table S6</div> <p>Proposed CTL and HTL epitopes for vaccine construction.</p>  <div class="media" title="media"><a href="mmc9.docx">LINK</a><div tag="alt-text">
Table S6</div> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Table S7</div> <p>Predicted conformational epitopes within construct V3.</p>  <div class="media" title="media"><a href="mmc10.docx">LINK</a><div tag="alt-text">
Table S7</div> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Table S8</div> <p>Predicted IFN alpha producing epitopes in the vaccine construct V3.</p>  <div class="media" title="media"><a href="mmc11.docx">LINK</a><div tag="alt-text">
Table S8</div> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
File S1</div> <p>NCBI IDs of the complete genome, spike glycoprotein, envelope protein, membrane protein and nucleocapsid protein of coronavirus with Genera and Sub-Genera.</p>  <div class="media" title="media"><a href="mmc12.docx">LINK</a><div tag="alt-text">
File S1</div> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
File S2</div> <p>Retrieved protein sequences of major structural proteins of COVID-19.</p>  <div class="media" title="media"><a href="mmc13.docx">LINK</a><div tag="alt-text">
File S2</div> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
File S3</div> <p>Secondary structure and domain analysis of spike glycoprotein, envelope protein, membrane protein and nucleocapsid proteins.</p>  <div class="media" title="media"><a href="mmc14.docx">LINK</a><div tag="alt-text">
File S3</div> </div> </div> <p /> </div> <div class="creditauthorstatement" title="sec"> <div class="title" tagx="title" title="title">
Credit author statement</div> <p id="p0380"><b>Mst Rubaiat Nazneen Akhand:</b> Conceptualization, Methodology, Data curation, Formal analysis, Software and Writing - original draft. <b>Kazi Faizul Azim:</b> Conceptualization, Methodology, Software, Writing - original draft <b>Syeda Farjana Hoque:</b> Software, Data Curation and Writing - original draft <b>Mahmuda Akther Moli:</b> Software, Data Curation and Writing - original draft <b>Bijit Das Joy:</b> Software, Validation, and Visualization <b>Hafsa Akter:</b> Software, Validation and Visualization <b>Ibrahim Khalil Afif:</b> Software and Validation <b>Nadim Ahmed:</b> Software and Validation <b>Mahmudul Hasan:</b> Conceptualization, Supervision, Project administration, Resources and Writing - review &amp;amp; editing.</p> </div> <div class="fundinginformation" title="sec"> <div class="title" tagx="title" title="title">
Funding information</div> <p id="p0385">This research did not receive any specific grant from funding agencies in the public, commercial, or not-for-profit sectors.</p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Declaration of Competing Interest</div> <p id="p0410">Authors declare that they have no conflict of interests.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bb0005" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudek</span><span class="given-names" tagx="given-names" title="given-names">N.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlmutter</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aguilar</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Croft</span><span class="given-names" tagx="given-names" title="given-names">N.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purcell</span><span class="given-names" tagx="given-names" title="given-names">A.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Epitope discovery and their use in peptide based vaccines</span><span class="source" tagx="source" title="source">Curr. Pharm. Des.</span><span class="volume" tagx="volume" title="volume">16</span><span class="issue" tagx="issue" title="issue">28</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">3149</span><span class="lpage" tagx="lpage" title="lpage">3157</span><span class="pub-id"><a href="https://dx.doi.org/10.2174/138161210793292447">10.2174/138161210793292447</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20687873">20687873</a></span></span></li> <li tag="ref"><a name="bb0010" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spinney</span><span class="given-names" tagx="given-names" title="given-names">Laura</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">When Will A Coronavirus Vaccine Be Ready?</span><span class="year" tagx="year" title="year">2020</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">The Guardian</span><span class="comment">(Retrieved 18 March 2020)</span></span></li> <li tag="ref"><a name="bb0015" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">World Health Organization (WHO)</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Infection Prevention and Control During Health Care When COVID-19 Is Suspected: Interim Guidance</span><span class="year" tagx="year" title="year">2020</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">World Health Organization</span><span class="fpage" tagx="fpage" title="fpage">2020</span></span></li> <li tag="ref"><a name="bb0020" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ziady</span><span class="given-names" tagx="given-names" title="given-names">Hanna</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Biotech Company Moderna Says Its Coronavirus Vaccine is Ready For First Tests</span><span class="year" tagx="year" title="year">2020</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">CNN</span><span class="comment">Archived from the original on 28 February 2020. (Retrieved 2 March 2020)</span></span></li> <li tag="ref"><a name="bb0025" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimitrov</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bangov</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flower</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doytchinova</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span><span class="mixed-article-title" title="mixed-article-title">AllerTOP v.2 - a server for in silico prediction of allergens</span><span class="source" tagx="source" title="source">J. Mol. Model.</span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">6</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">S4</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00894-014-2278-5">10.1007/s00894-014-2278-5</a></span></span></li> <li tag="ref"><a name="bb0030" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">NIH News Event</span></span></span><span class="mixed-article-title" title="mixed-article-title">NIH Clinical Trial of Investigational Vaccine For COVID-19 Begins. Study Enrolling Seattle-Based Healthy Adult Volunteers</span><span class="comment">(Accessed on 1st April 2020)</span><a href="https://www.nih.gov/news-events/news-releases/nih-clinical-trial-investigational-vaccine-covid-19-begins">https://www.nih.gov/news-events/news-releases/nih-clinical-trial-investigational-vaccine-covid-19-begins</a><span class="year" tagx="year" title="year">2020</span></span></li> <li tag="ref"><a name="bb0035" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">World Health Organization (WHO)</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronavirus Disease (COVID-19) Outbreak</span><a href="https://www.who.int">https://www.who.int</a><span class="year" tagx="year" title="year">2020</span><span class="comment">(opens in new tab)</span></span></li> <li tag="ref"><a name="bb0040" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aalten</span><span class="given-names" tagx="given-names" title="given-names">D.M.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Groot</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Findlay</span><span class="given-names" tagx="given-names" title="given-names">J.B.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berendsen</span><span class="given-names" tagx="given-names" title="given-names">H.J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amadei</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A comparison of techniques for calculating protein essential dynamics</span><span class="source" tagx="source" title="source">J. Comput. Chem.</span><span class="volume" tagx="volume" title="volume">18</span><span class="issue" tagx="issue" title="issue">2</span><span class="year" tagx="year" title="year">1997</span><span class="fpage" tagx="fpage" title="fpage">169</span><span class="lpage" tagx="lpage" title="lpage">181</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/(SICI)1096-987X">10.1002/(SICI)1096-987X</a></span></span></li> <li tag="ref"><a name="bb0045" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdelmageed</span><span class="given-names" tagx="given-names" title="given-names">M.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdelmoneim</span><span class="given-names" tagx="given-names" title="given-names">A.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mustafa</span><span class="given-names" tagx="given-names" title="given-names">M.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfadol</span><span class="given-names" tagx="given-names" title="given-names">N.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murshed</span><span class="given-names" tagx="given-names" title="given-names">N.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shantier</span><span class="given-names" tagx="given-names" title="given-names">S.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makhawi</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Design of a multiepitope-based peptide vaccine against the E protein of human COVID-19: an immunoinformatics approach</span><span class="source" tagx="source" title="source">Biomed. Res. Int.</span><span class="volume" tagx="volume" title="volume">2020</span><span class="year" tagx="year" title="year">2020</span><span class="comment">(Jan 1)</span></span></li> <li tag="ref"><a name="bb0050" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Awan</span><span class="given-names" tagx="given-names" title="given-names">F.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obaid</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ikram</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janjua</span><span class="given-names" tagx="given-names" title="given-names">H.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mutation-structure function relationship 893 based integrated strategy reveals the potential impact of deleterious missense mutations in 894 autophagy related proteins on hepatocellular carcinoma (HCC): a comprehensive 895 informatics approach</span><span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span><span class="volume" tagx="volume" title="volume">18</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">139</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms18010139">10.3390/ijms18010139</a></span></span></li> <li tag="ref"><a name="bb0055" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azim</span><span class="given-names" tagx="given-names" title="given-names">K.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lasker</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akter</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hia</span><span class="given-names" tagx="given-names" title="given-names">M.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhuiyan OF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Conglomeration of highly antigenic nucleoproteins to inaugurate a heterosubtypic next generation vaccine candidate against Arenaviridae family</span><span class="source" tagx="source" title="source">bioRxiv</span><span class="year" tagx="year" title="year">2019</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/2019.12.29.885731">10.1101/2019.12.29.885731</a></span><span class="comment">(Jan 1)</span></span></li> <li tag="ref"><a name="bb0060" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azim</span><span class="given-names" tagx="given-names" title="given-names">K.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossain</span><span class="given-names" tagx="given-names" title="given-names">M.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somana</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoque</span><span class="given-names" tagx="given-names" title="given-names">S.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bappy</span><span class="given-names" tagx="given-names" title="given-names">M.N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Immunoinformatics approaches for designing a novel multi epitope peptide vaccine against human norovirus (Norwalk virus)</span><span class="source" tagx="source" title="source">Infect. Genet. Evol.</span><span class="volume" tagx="volume" title="volume">74</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">103936</span><span class="comment">(Oct 1)</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2019.103936">10.1016/j.meegid.2019.103936</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31233780">31233780</a></span></span></li> <li tag="ref"><a name="bb0065" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azim</span><span class="given-names" tagx="given-names" title="given-names">K.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banik</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">M.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deb</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somana</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Screening and druggability analysis of some plant metabolites against SARS-CoV-2: an integrative computational approach</span><span class="source" tagx="source" title="source">Inform. Med. Unlocked</span><span class="volume" tagx="volume" title="volume">20</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">100367</span><span class="comment">(Jun 9)</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32537482">32537482</a></span></span></li> <li tag="ref"><a name="bb0070" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Behbahani</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">In Silico Design of Novel Multi-Epitope Recombinant Vaccine Based on Coronavirus Spike Glycoprotein</span><span class="year" tagx="year" title="year">2020</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">bioRxiv</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.03.10.985499">10.1101/2020.03.10.985499</a></span></span></li> <li tag="ref"><a name="bb0075" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ceraolo</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giorgi</span><span class="given-names" tagx="given-names" title="given-names">F.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic variance of the 2019-nCoV coronavirus</span><span class="source" tagx="source" title="source">J. Med. Virol.</span><span class="volume" tagx="volume" title="volume">92</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">522</span><span class="lpage" tagx="lpage" title="lpage">528</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25700">10.1002/jmv.25700</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32027036">32027036</a></span></span></li> <li tag="ref"><a name="bb0085" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study</span><span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">395</span><span class="issue" tagx="issue" title="issue">10223</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">507</span><span class="lpage" tagx="lpage" title="lpage">513</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30211-7">10.1016/S0140-6736(20)30211-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32007143">32007143</a></span></span></li> <li tag="ref"><a name="bb0090" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooper</span><span class="given-names" tagx="given-names" title="given-names">N.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nemerow</span><span class="given-names" tagx="given-names" title="given-names">G.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The role of antibody and complement in the control of viral infections</span><span class="source" tagx="source" title="source">J. Invest. Dermatol.</span><span class="volume" tagx="volume" title="volume">83</span><span class="issue" tagx="issue" title="issue">1 Suppl</span><span class="year" tagx="year" title="year">1984</span><span class="fpage" tagx="fpage" title="fpage">121</span><span class="lpage" tagx="lpage" title="lpage">127</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/jid.1984.33">10.1038/jid.1984.33</a></span></span></li> <li tag="ref"><a name="bb0095" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Craig</span><span class="given-names" tagx="given-names" title="given-names">D.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dombkowski</span><span class="given-names" tagx="given-names" title="given-names">A.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Disulfide by Design 2.0: a web-based tool for disulfide engineering in proteins</span><span class="source" tagx="source" title="source">BMC Bioinformatics</span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">346</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-14-346">10.1186/1471-2105-14-346</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24289175">24289175</a></span></span></li> <li tag="ref"><a name="bb0100" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bahar</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Normal mode analysis theoretical and applications to biological and chemical systems</span><span class="source" tagx="source" title="source">Brief. Bioinform.</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">378</span><span class="lpage" tagx="lpage" title="lpage">379</span><span class="pub-id"><a href="https://dx.doi.org/10.1201/9781420035070">10.1201/9781420035070</a></span></span></li> <li tag="ref"><a name="bb0105" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimitrov</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naneva</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doytchinova</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bangov</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span><span class="mixed-article-title" title="mixed-article-title">AllergenFP: allergenicity prediction by descriptor fingerprints</span><span class="source" tagx="source" title="source">Bioinformatics.</span><span class="volume" tagx="volume" title="volume">30</span><span class="issue" tagx="issue" title="issue">6</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">846</span><span class="lpage" tagx="lpage" title="lpage">851</span><span class="comment">(Mar 15)</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btt619">10.1093/bioinformatics/btt619</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24167156">24167156</a></span></span></li> <li tag="ref"><a name="bb0110" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doytchinova</span><span class="given-names" tagx="given-names" title="given-names">I.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flower</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identifying candidate subunit vaccines using an alignment-independent method based on principal amino acid properties</span><span class="source" tagx="source" title="source">Vaccine.</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">856</span><span class="lpage" tagx="lpage" title="lpage">866</span><span class="comment">(Jan 15)</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vaccine.2006.09.032">10.1016/j.vaccine.2006.09.032</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17045707">17045707</a></span></span></li> <li tag="ref"><a name="bb0115" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doytchinova</span><span class="given-names" tagx="given-names" title="given-names">I.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flower</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines</span><span class="source" tagx="source" title="source">BMC Bioinformatics</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">4</span><span class="comment">(Dec 1)</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-8-4">10.1186/1471-2105-8-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17207271">17207271</a></span></span></li> <li tag="ref"><a name="bb0120" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">B.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The spike protein of SARS-CoV—a target for vaccine and therapeutic development</span><span class="source" tagx="source" title="source">Nat. Rev. Microbiol.</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">226</span><span class="lpage" tagx="lpage" title="lpage">236</span><span class="comment">(Mar)</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19198616">19198616</a></span></span></li> <li tag="ref"><a name="bb0130" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emini</span><span class="given-names" tagx="given-names" title="given-names">E.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">J.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlow</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boger</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Induction of hepatitis a virus-neutralizing antibody by a virus-specific synthetic peptide</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="volume" tagx="volume" title="volume">55</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">1985</span><span class="fpage" tagx="fpage" title="fpage">836</span><span class="lpage" tagx="lpage" title="lpage">839</span><span class="comment">(Sep 1)</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/jvi.55.3.836-839.1985">10.1128/jvi.55.3.836-839.1985</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2991600">2991600</a></span></span></li> <li tag="ref"><a name="bb0135" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernando</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marta</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Isabel</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sonia</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jose</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silvia</span><span class="given-names" tagx="given-names" title="given-names">L.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Engineering a replication-competent, propagation defective middle east respiratory syndrome coronavirus as a vaccine candidate</span><span class="source" tagx="source" title="source">mBio.</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2013</span>e00650-13<span class="pub-id"><a href="https://dx.doi.org/10.1128/mbio.00650-13">10.1128/mbio.00650-13</a></span></span></li> <li tag="ref"><a name="bb0140" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forni</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cagliani</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clerici</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sironi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular evolution of human coronavirus genomes</span><span class="source" tagx="source" title="source">Trends Microbiol.</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">35</span><span class="lpage" tagx="lpage" title="lpage">48</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2016.09.001">10.1016/j.tim.2016.09.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27743750">27743750</a></span></span></li> <li tag="ref"><a name="bb0145" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia</span><span class="given-names" tagx="given-names" title="given-names">K.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teyton</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">I.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural basis of T cell recognition</span><span class="source" tagx="source" title="source">Annu. Rev. Immunol.</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">1999</span><span class="fpage" tagx="fpage" title="fpage">369</span><span class="lpage" tagx="lpage" title="lpage">397</span><span class="comment">(Apr)</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.immunol.17.1.369">10.1146/annurev.immunol.17.1.369</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10358763">10358763</a></span></span></li> <li tag="ref"><a name="bb0150" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gasteiger</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoogland</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gattiker</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilkins</span><span class="given-names" tagx="given-names" title="given-names">M.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Appel</span><span class="given-names" tagx="given-names" title="given-names">R.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bairoch</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Protein identification and analysis tools on the ExPASy server</span><span class="source" tagx="source" title="source">The Proteomics Protocols Handbook</span><span class="year" tagx="year" title="year">2005</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Humana Press</span><span class="fpage" tagx="fpage" title="fpage">571</span><span class="lpage" tagx="lpage" title="lpage">607</span><span class="pub-id"><a href="https://dx.doi.org/10.1385/1592598900">10.1385/1592598900</a></span></span></li> <li tag="ref"><a name="bb0155" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghaffari-Nazari</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tavakkol-Afshari</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaafari</span><span class="given-names" tagx="given-names" title="given-names">M.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tahaghoghi-Hajghorbani</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masoumi</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jalali</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Improving multi-epitope long peptide vaccine potency by using a strategy that enhances CD4+ T help in BALB/c mice</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">11</span><span class="year" tagx="year" title="year">2015</span>e0142563<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0142563">10.1371/journal.pone.0142563</a></span></span></li> <li tag="ref"><a name="bb0160" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glansbeek</span><span class="given-names" tagx="given-names" title="given-names">H.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haagmans</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">te Lintelo</span><span class="given-names" tagx="given-names" title="given-names">E.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Egberink</span><span class="given-names" tagx="given-names" title="given-names">H.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duquesne</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aubert</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horzinek</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rottier</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Adverse effects of feline IL-12 during DNA vaccination against feline infectious peritonitis virus</span><span class="source" tagx="source" title="source">J. Gen. Virol.</span><span class="volume" tagx="volume" title="volume">83</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2002</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">10</span><span class="comment">(Jan 1)</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11752695">11752695</a></span></span></li> <li tag="ref"><a name="bb0165" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorbalenya</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauber</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Phylogeny of viruses</span><span class="source" tagx="source" title="source">Reference Module in Biomedical Sciences</span><span class="year" tagx="year" title="year">2017</span></span></li> <li tag="ref"><a name="bb0170" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">R.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donaldson</span><span class="given-names" tagx="given-names" title="given-names">E.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baric</span><span class="given-names" tagx="given-names" title="given-names">R.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A decade after SARS: strategies for controlling emerging coronaviruses</span><span class="source" tagx="source" title="source">Nat. Rev. Microbiol.</span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">12</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">836</span><span class="lpage" tagx="lpage" title="lpage">848</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro3143">10.1038/nrmicro3143</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24217413">24217413</a></span></span></li> <li tag="ref"><a name="bb0175" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grote</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiller</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scheer</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Münch</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nörtemann</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hempel</span><span class="given-names" tagx="given-names" title="given-names">D.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">JCat: a novel tool to adapt codon usage of a target gene to its potential expression host</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="volume" tagx="volume" title="volume">33</span><span class="issue" tagx="issue" title="issue">Suppl. 2</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">W526</span><span class="lpage" tagx="lpage" title="lpage">W531</span><span class="comment">(Jul 1)</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gki376">10.1093/nar/gki376</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980527">15980527</a></span></span></li> <li tag="ref"><a name="bb0180" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">W.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ni</span><span class="given-names" tagx="given-names" title="given-names">Z.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">W.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ou</span><span class="given-names" tagx="given-names" title="given-names">C.Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">J.X.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Clinical characteristics of coronavirus disease 2019 in China</span><span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMoa2002032">10.1056/NEJMoa2002032</a></span></span></li> <li tag="ref"><a name="bb0185" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">In silico approach for predicting toxicity of peptides and proteins</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">9</span><span class="year" tagx="year" title="year">2013</span>e73957<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0073957">10.1371/journal.pone.0073957</a></span></span></li> <li tag="ref"><a name="bb0190" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azim</span><span class="given-names" tagx="given-names" title="given-names">K.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Begum</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">N.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shammi</span><span class="given-names" tagx="given-names" title="given-names">T.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imran</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vaccinomics strategy for developing a unique multi-epitope monovalent vaccine against <i>Marburg marburg</i> virus</span><span class="source" tagx="source" title="source">Infect. Genet. Evol.</span><span class="volume" tagx="volume" title="volume">70</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">140</span><span class="lpage" tagx="lpage" title="lpage">157</span><span class="comment">(Jun 1)</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2019.03.003">10.1016/j.meegid.2019.03.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30849525">30849525</a></span></span></li> <li tag="ref"><a name="bb0195" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghosh</span><span class="given-names" tagx="given-names" title="given-names">P.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azim</span><span class="given-names" tagx="given-names" title="given-names">K.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mukta</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abir</span><span class="given-names" tagx="given-names" title="given-names">R.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nahar</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Reverse vaccinology approach to design a novel multi-epitope subunit vaccine against avian influenza A (H7N9) virus</span><span class="source" tagx="source" title="source">Microb. Pathog.</span><span class="volume" tagx="volume" title="volume">130</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">19</span><span class="lpage" tagx="lpage" title="lpage">37</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.micpath.2019.02.023">10.1016/j.micpath.2019.02.023</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30822457">30822457</a></span></span></li> <li tag="ref"><a name="bb0200" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chakraborty</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mustafa</span><span class="given-names" tagx="given-names" title="given-names">A.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azim</span><span class="given-names" tagx="given-names" title="given-names">K.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joy</span><span class="given-names" tagx="given-names" title="given-names">Z.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Contriving a chimeric polyvalent vaccine to prevent infections caused by herpes simplex virus (Type-1 and Type-2): an exploratory immunoinformatic approach</span><span class="source" tagx="source" title="source">J. Biomol. Struct. Dyn.</span><span class="volume" tagx="volume" title="volume">9</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">8</span><span class="comment">(Aug)</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2019.1647286">10.1080/07391102.2019.1647286</a></span></span></li> <li tag="ref"><a name="bb0205" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ai</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic characterization and infectivity of a novel SARS-like coronavirus in Chinese bats</span><span class="source" tagx="source" title="source">Emerg. Microbes Infect.</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">10</span><span class="comment">(Dec 1)</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41426-018-0155-5">10.1038/s41426-018-0155-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29323102">29323102</a></span></span></li> <li tag="ref"><a name="bb0210" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span><span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">395</span><span class="issue" tagx="issue" title="issue">10223</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">497</span><span class="lpage" tagx="lpage" title="lpage">506</span><span class="comment">(Feb 15)</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/s0140-6736(20)30183-5">10.1016/s0140-6736(20)30183-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31986264">31986264</a></span></span></li> <li tag="ref"><a name="bb0215" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">I Azhar E, Madani TA, Ntoumi F, Kock R, Dar O, et al.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The continuing 2019-nCoV epidemic threat of novel coronaviruses to global health - the latest 2019 novel coronavirus outbreak in Wuhan, China</span><span class="source" tagx="source" title="source">Int. J. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">91</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">264</span><span class="lpage" tagx="lpage" title="lpage">266</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijid.2020.01.009">10.1016/j.ijid.2020.01.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31953166">31953166</a></span></span></li> <li tag="ref"><a name="bb0220" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaimes</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">André</span><span class="given-names" tagx="given-names" title="given-names">N.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Millet</span><span class="given-names" tagx="given-names" title="given-names">J.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whittaker</span><span class="given-names" tagx="given-names" title="given-names">G.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural modeling of 2019-novel coronavirus (nCoV) spike protein reveals a proteolytically-sensitive activation loop as a distinguishing feature compared to SARS-CoV and related SARS-like coronaviruses</span><span class="source" tagx="source" title="source">ArXiv</span><span class="volume" tagx="volume" title="volume">2002</span><span class="issue" tagx="issue" title="issue">06196</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.02.10.942185">10.1101/2020.02.10.942185</a></span></span></li> <li tag="ref"><a name="bb0225" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jespersen</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peters</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marcatili</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">BepiPred-2.0: improving sequence-based B-cell epitope prediction using conformational epitopes</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="volume" tagx="volume" title="volume">45</span><span class="issue" tagx="issue" title="issue">W1</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">W24</span><span class="lpage" tagx="lpage" title="lpage">W29</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkx346">10.1093/nar/gkx346</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28472356">28472356</a></span></span></li> <li tag="ref"><a name="bb0230" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jia</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naqi</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Sequence analysis of gene 3, gene 4 and gene 5 of avian infectious bronchitis virus strain CU-T2</span><span class="source" tagx="source" title="source">Gene.</span><span class="volume" tagx="volume" title="volume">189</span><span class="issue" tagx="issue" title="issue">2</span><span class="year" tagx="year" title="year">1997</span><span class="fpage" tagx="fpage" title="fpage">189</span><span class="lpage" tagx="lpage" title="lpage">193</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/s0378-1119(96)00847-5">10.1016/s0378-1119(96)00847-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9168126">9168126</a></span></span></li> <li tag="ref"><a name="bb0235" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katoh</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Standley</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">30</span><span class="issue" tagx="issue" title="issue">4</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">772</span><span class="lpage" tagx="lpage" title="lpage">780</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mst010">10.1093/molbev/mst010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690">23329690</a></span></span></li> <li tag="ref"><a name="bb0240" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossain</span><span class="given-names" tagx="given-names" title="given-names">M.U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rakib-Uz-Zaman</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morshed</span><span class="given-names" tagx="given-names" title="given-names">M.N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Epitope-based peptide vaccine design and target site depiction against Ebola viruses: an immunoinformatics study</span><span class="source" tagx="source" title="source">Scand. J. Immunol.</span><span class="volume" tagx="volume" title="volume">82</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">25</span><span class="lpage" tagx="lpage" title="lpage">34</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/sji.12302">10.1111/sji.12302</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25857850">25857850</a></span></span></li> <li tag="ref"><a name="bb0245" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolaskar</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tongaonkar</span><span class="given-names" tagx="given-names" title="given-names">P.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A semi-empirical method for prediction of antigenic determinants on protein antigens</span><span class="source" tagx="source" title="source">FEBS Lett.</span><span class="volume" tagx="volume" title="volume">276</span><span class="issue" tagx="issue" title="issue">1–2</span><span class="year" tagx="year" title="year">1990</span><span class="fpage" tagx="fpage" title="fpage">172</span><span class="lpage" tagx="lpage" title="lpage">174</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0014-5793(90)80535-q">10.1016/0014-5793(90)80535-q</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1702393">1702393</a></span></span></li> <li tag="ref"><a name="bb0250" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krogh</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsson</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Heijne</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sonnhammer</span><span class="given-names" tagx="given-names" title="given-names">E.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes</span><span class="source" tagx="source" title="source">J. Mol. Biol.</span><span class="volume" tagx="volume" title="volume">305</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">2001</span><span class="fpage" tagx="fpage" title="fpage">567</span><span class="lpage" tagx="lpage" title="lpage">580</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/jmbi.2000.4315">10.1006/jmbi.2000.4315</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11152613">11152613</a></span></span></li> <li tag="ref"><a name="bb0255" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kunz</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minder</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">COVID-19 pandemic: palliative care for elderly and frail patients at home and in residential and nursing homes</span><span class="source" tagx="source" title="source">Swiss Med. Wkly.</span><span class="volume" tagx="volume" title="volume">150</span><span class="issue" tagx="issue" title="issue">1314</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.4414/smw.2020.20235">10.4414/smw.2020.20235</a></span></span></li> <li tag="ref"><a name="bb0260" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanfear</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frandsen</span><span class="given-names" tagx="given-names" title="given-names">P.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Senfeld</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calcott</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Partitionfinder 2: new methods for selecting partitioned models of evolution for molecular and morphological phylogenetic analyses</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">34</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">772</span><span class="lpage" tagx="lpage" title="lpage">773</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msw260">10.1093/molbev/msw260</a></span></span></li> <li tag="ref"><a name="bb0265" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Receptor recognition mechanisms of coronaviruses: a decade of structural studies</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="volume" tagx="volume" title="volume">89</span><span class="issue" tagx="issue" title="issue">4</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">1954</span><span class="lpage" tagx="lpage" title="lpage">1964</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/jvi.02615-14">10.1128/jvi.02615-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25428871">25428871</a></span></span></li> <li tag="ref"><a name="bb0270" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasilieva</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sui</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berne</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span><span class="source" tagx="source" title="source">Nature.</span><span class="volume" tagx="volume" title="volume">426</span><span class="issue" tagx="issue" title="issue">6965</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">450</span><span class="lpage" tagx="lpage" title="lpage">454</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature02145">10.1038/nature02145</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14647384">14647384</a></span></span></li> <li tag="ref"><a name="bb0275" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joshi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singhania</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramsey</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murthy</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Peptide vaccine: progress and challenges</span><span class="source" tagx="source" title="source">Vaccines.</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">515</span><span class="lpage" tagx="lpage" title="lpage">536</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/vaccines2030515">10.3390/vaccines2030515</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26344743">26344743</a></span></span></li> <li tag="ref"><a name="bb0280" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">G.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The membrane protein of severe acute respiratory syndrome coronavirus acts as a dominant immunogen revealed by a clustering region of novel functionally and structurally defined cytotoxic T-lymphocyte epitopes</span><span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">202</span><span class="issue" tagx="issue" title="issue">8</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">1171</span><span class="lpage" tagx="lpage" title="lpage">1180</span><span class="comment">(Oct 15)</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20831383">20831383</a></span></span></li> <li tag="ref"><a name="bb0285" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">López-Blanco</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aliaga</span><span class="given-names" tagx="given-names" title="given-names">J.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quintana-Ortí</span><span class="given-names" tagx="given-names" title="given-names">E.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chacón</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">iMODS: internal coordinates normal mode analysis server</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="volume" tagx="volume" title="volume">42</span><span class="issue" tagx="issue" title="issue">W1</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">W271</span><span class="lpage" tagx="lpage" title="lpage">W276</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gku339">10.1093/nar/gku339</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24771341">24771341</a></span></span></li> <li tag="ref"><a name="bb0290" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stratton</span><span class="given-names" tagx="given-names" title="given-names">C.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Outbreak of pneumonia of unknown etiology in Wuhan China: the mystery and the miracle</span><span class="source" tagx="source" title="source">J. Med. Virol.</span><span class="volume" tagx="volume" title="volume">92</span><span class="issue" tagx="issue" title="issue">4</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">401</span><span class="lpage" tagx="lpage" title="lpage">402</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25678">10.1002/jmv.25678</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31950516">31950516</a></span></span></li> <li tag="ref"><a name="bb0295" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moxon</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reche</span><span class="given-names" tagx="given-names" title="given-names">P.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rappuoli</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Editorial: reverse vaccinology</span><span class="source" tagx="source" title="source">Front. Immunol.</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">2776</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fimmu.2019.02776">10.3389/fimmu.2019.02776</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31849959">31849959</a></span></span></li> <li tag="ref"><a name="bb0300" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neuman</span><span class="given-names" tagx="given-names" title="given-names">B.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiss</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kunding</span><span class="given-names" tagx="given-names" title="given-names">A.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhella</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baksh</span><span class="given-names" tagx="given-names" title="given-names">M.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Connelly</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Droese</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klaus</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makino</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sawicki</span><span class="given-names" tagx="given-names" title="given-names">S.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siddell</span><span class="given-names" tagx="given-names" title="given-names">S.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A structural analysis of M protein in coronavirus assembly and morphology</span><span class="source" tagx="source" title="source">J. Struct. Biol.</span><span class="volume" tagx="volume" title="volume">174</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="lpage" tagx="lpage" title="lpage">22</span><span class="comment">(Apr 1)</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21130884">21130884</a></span></span></li> <li tag="ref"><a name="bb0305" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nieto-Torres</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeDiego</span><span class="given-names" tagx="given-names" title="given-names">M.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Álvarez</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiménez-Guardeño</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Regla-Nava</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Llorente</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Subcellular location and topology of severe acute respiratory syndrome coronavirus envelope protein</span><span class="source" tagx="source" title="source">Virology.</span><span class="volume" tagx="volume" title="volume">415</span><span class="issue" tagx="issue" title="issue">2</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">69</span><span class="lpage" tagx="lpage" title="lpage">82</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2011.03.029">10.1016/j.virol.2011.03.029</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21524776">21524776</a></span></span></li> <li tag="ref"><a name="bb0310" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oany</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emran</span><span class="given-names" tagx="given-names" title="given-names">A.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jyoti</span><span class="given-names" tagx="given-names" title="given-names">T.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Design of an epitope-based peptide vaccine against spike protein of human coronavirus: an in silico approach</span><span class="source" tagx="source" title="source">Drug Des. Devel. Ther.</span><span class="volume" tagx="volume" title="volume">8</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">1139</span><span class="pub-id"><a href="https://dx.doi.org/10.2147/dddt.s67861">10.2147/dddt.s67861</a></span></span></li> <li tag="ref"><a name="bb0315" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandey</span><span class="given-names" tagx="given-names" title="given-names">R.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ojha</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aathmanathan</span><span class="given-names" tagx="given-names" title="given-names">V.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krishnan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prajapati</span><span class="given-names" tagx="given-names" title="given-names">V.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Immunoinformatics approaches to design a novel multi-epitope subunit vaccine against HIV infection</span><span class="source" tagx="source" title="source">Vaccine.</span><span class="volume" tagx="volume" title="volume">36</span><span class="issue" tagx="issue" title="issue">17</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">2262</span><span class="lpage" tagx="lpage" title="lpage">2272</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vaccine.2018.03.042">10.1016/j.vaccine.2018.03.042</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29571972">29571972</a></span></span></li> <li tag="ref"><a name="bb0320" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartlam</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Protective humoral responses to severe acute respiratory syndrome-associated coronavirus: implications for the design of an effective protein-based vaccine</span><span class="source" tagx="source" title="source">J. Gen. Virol.</span><span class="volume" tagx="volume" title="volume">85</span><span class="issue" tagx="issue" title="issue">10</span><span class="year" tagx="year" title="year">2004</span><span class="fpage" tagx="fpage" title="fpage">3109</span><span class="lpage" tagx="lpage" title="lpage">3113</span><span class="comment">(Oct 1)</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15448374">15448374</a></span></span></li> <li tag="ref"><a name="bb0325" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M.X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ang</span><span class="given-names" tagx="given-names" title="given-names">I.Y.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">S.H.X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">R.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.I.-P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Potential rapid diagnostics, vaccine and therapeutics for 2019 novel coronavirus (2019-nCoV): a systematic review</span><span class="source" tagx="source" title="source">J. Clin. Med.</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">623</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/jcm9030623">10.3390/jcm9030623</a></span></span></li> <li tag="ref"><a name="bb0330" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Raptorx: exploiting structure information for protein alignment by statistical inference</span><span class="source" tagx="source" title="source">Proteins</span><span class="volume" tagx="volume" title="volume">79</span><span class="issue" tagx="issue" title="issue">S10</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">161</span><span class="lpage" tagx="lpage" title="lpage">171</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/prot.23175">10.1002/prot.23175</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21987485">21987485</a></span></span></li> <li tag="ref"><a name="bb0335" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrovsky</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aguilar</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vaccine adjuvants: current state and future trends</span><span class="source" tagx="source" title="source">Immunol. Cell Biol.</span><span class="volume" tagx="volume" title="volume">82</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2004</span><span class="fpage" tagx="fpage" title="fpage">488</span><span class="lpage" tagx="lpage" title="lpage">496</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.0818-9641.2004.01272.x">10.1111/j.0818-9641.2004.01272.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15479434">15479434</a></span></span></li> <li tag="ref"><a name="bb0340" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prabhakar</span><span class="given-names" tagx="given-names" title="given-names">P.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srivastava</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">K.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balaji</span><span class="given-names" tagx="given-names" title="given-names">P.V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Monomerization alters the dynamics 897 of the lid region in <i>campylobacter jejuni</i> CstII: an MD simulation study</span><span class="source" tagx="source" title="source">J. Biomol. Struct. Dyn.</span><span class="volume" tagx="volume" title="volume">34</span><span class="issue" tagx="issue" title="issue">4</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">778</span><span class="lpage" tagx="lpage" title="lpage">779</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2015.1054430">10.1080/07391102.2015.1054430</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26208676">26208676</a></span></span></li> <li tag="ref"><a name="bb0345" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purcell</span><span class="given-names" tagx="given-names" title="given-names">A.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCluskey</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossjohn</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">More than one reason to rethink the use of peptides in vaccine design</span><span class="source" tagx="source" title="source">Nat. Rev. Drug Discov.</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">404</span><span class="lpage" tagx="lpage" title="lpage">414</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrd2224">10.1038/nrd2224</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17473845">17473845</a></span></span></li> <li tag="ref"><a name="bb0350" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raj</span><span class="given-names" tagx="given-names" title="given-names">V.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mou</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smits</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dekkers</span><span class="given-names" tagx="given-names" title="given-names">D.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Müller</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dijkman</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC</span><span class="source" tagx="source" title="source">Nature.</span><span class="volume" tagx="volume" title="volume">495</span><span class="issue" tagx="issue" title="issue">7440</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">251</span><span class="lpage" tagx="lpage" title="lpage">254</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature12005">10.1038/nature12005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23486063">23486063</a></span></span></li> <li tag="ref"><a name="bb0355" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rappuoli</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Reverse vaccinology</span><span class="source" tagx="source" title="source">Curr. Opin. Microbiol.</span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2000</span><span class="fpage" tagx="fpage" title="fpage">445</span><span class="lpage" tagx="lpage" title="lpage">450</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/s1369-5274(00)00119-3">10.1016/s1369-5274(00)00119-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11050440">11050440</a></span></span></li> <li tag="ref"><a name="bb0360" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasheed</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raza</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zohaib</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yaqub</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rabbani</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riaz</span><span class="given-names" tagx="given-names" title="given-names">M.I.</span></span></span><span class="mixed-article-title" title="mixed-article-title">In silico identification of novel B cell and T cell epitopes of Wuhan coronavirus (2019-nCoV) for effective multi epitope-based peptide vaccine production</span><span class="source" tagx="source" title="source">Preprints</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.20944/preprints202002.0359.v1">10.20944/preprints202002.0359.v1</a></span></span></li> <li tag="ref"><a name="bb0365" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robert</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouet</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Deciphering key features in protein structures with the new ENDscript server</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="volume" tagx="volume" title="volume">42</span><span class="issue" tagx="issue" title="issue">W1</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">320</span><span class="lpage" tagx="lpage" title="lpage">324</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gku316">10.1093/nar/gku316</a></span></span></li> <li tag="ref"><a name="bb0370" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teslenko</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Der Mark</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ayres</span><span class="given-names" tagx="given-names" title="given-names">D.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darling</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Höhna</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larget</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huelsenbeck</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space</span><span class="source" tagx="source" title="source">Syst. Biol.</span><span class="volume" tagx="volume" title="volume">61</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">539</span><span class="lpage" tagx="lpage" title="lpage">542</span><span class="comment">(May 1)</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/sys029">10.1093/sysbio/sys029</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22357727">22357727</a></span></span></li> <li tag="ref"><a name="bb0375" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saha</span><span class="given-names" tagx="given-names" title="given-names">C.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasan</span><span class="given-names" tagx="given-names" title="given-names">M.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossain</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jahan</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azad</span><span class="given-names" tagx="given-names" title="given-names">A.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">In silico identification and characterization of common epitope-based peptide vaccine for Nipah and Hendra viruses</span><span class="source" tagx="source" title="source">Asian Pac J Trop Med</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">6</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">529</span><span class="lpage" tagx="lpage" title="lpage">538</span><span class="comment">(Jun 1)</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.apjtm.2017.06.016">10.1016/j.apjtm.2017.06.016</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28756915">28756915</a></span></span></li> <li tag="ref"><a name="bb0380" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schoeman</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fielding</span><span class="given-names" tagx="given-names" title="given-names">B.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronavirus envelope protein: current knowledge</span><span class="source" tagx="source" title="source">Virol. J.</span><span class="volume" tagx="volume" title="volume">16</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">22</span><span class="comment">(Dec)</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30606229">30606229</a></span></span></li> <li tag="ref"><a name="bb0385" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharmin</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">A.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A highly conserved WDYPKCDRA epitope in the RNA directed RNA polymerase of human coronaviruses can be used as epitope-based universal vaccine design</span><span class="source" tagx="source" title="source">BMC Bioinformatics.</span><span class="volume" tagx="volume" title="volume">15</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">161</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-15-161">10.1186/1471-2105-15-161</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24884408">24884408</a></span></span></li> <li tag="ref"><a name="bb0390" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shrestha</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diamond</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Role of CD8+ T cells in control of West Nile virus infection</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="volume" tagx="volume" title="volume">78</span><span class="issue" tagx="issue" title="issue">15</span><span class="year" tagx="year" title="year">2004</span><span class="fpage" tagx="fpage" title="fpage">8312</span><span class="lpage" tagx="lpage" title="lpage">8321</span><span class="comment">(Aug 1)</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/jvi.78.15.8312-8321.2004">10.1128/jvi.78.15.8312-8321.2004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15254203">15254203</a></span></span></li> <li tag="ref"><a name="bb0395" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sievers</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgins</span><span class="given-names" tagx="given-names" title="given-names">D.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Clustal omega, accurate alignment of very large numbers of sequences</span><span class="source" tagx="source" title="source">Methods Mol. Biol.</span><span class="volume" tagx="volume" title="volume">1079</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">105</span><span class="lpage" tagx="lpage" title="lpage">116</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-62703-646-7_6">10.1007/978-1-62703-646-7_6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24170397">24170397</a></span></span></li> <li tag="ref"><a name="bb0400" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smialowski</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin-Galiano</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mikolajka</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girschick</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holak</span><span class="given-names" tagx="given-names" title="given-names">T.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frishman</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Protein solubility: sequence based prediction and experimental verification</span><span class="source" tagx="source" title="source">Bioinformatics.</span><span class="volume" tagx="volume" title="volume">23</span><span class="issue" tagx="issue" title="issue">19</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">2536</span><span class="lpage" tagx="lpage" title="lpage">2542</span><span class="comment">(Oct 1)</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btl623">10.1093/bioinformatics/btl623</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17150993">17150993</a></span></span></li> <li tag="ref"><a name="bb0405" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunn</span><span class="given-names" tagx="given-names" title="given-names">C.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phyutility: a phyloinformatics tool for trees, alignments and molecular data</span><span class="source" tagx="source" title="source">Bioinformatics.</span><span class="volume" tagx="volume" title="volume">24</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">715</span><span class="lpage" tagx="lpage" title="lpage">716</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btm619">10.1093/bioinformatics/btm619</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18227120">18227120</a></span></span></li> <li tag="ref"><a name="bb0410" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solanki</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tiwari</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Subtractive proteomics to identify novel drug targets and reverse vaccinology for the development of chimeric vaccine against <i>Acinetobacter baumannii</i></span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="comment">(Jun 13)</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-26689-7">10.1038/s41598-018-26689-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29311619">29311619</a></span></span></li> <li tag="ref"><a name="bb0415" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span><span class="source" tagx="source" title="source">Bioinformatics.</span><span class="volume" tagx="volume" title="volume">30</span><span class="issue" tagx="issue" title="issue">9</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">1312</span><span class="lpage" tagx="lpage" title="lpage">1313</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu033">10.1093/bioinformatics/btu033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623">24451623</a></span></span></li> <li tag="ref"><a name="bb0420" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stratton</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almario</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wizemann</span><span class="given-names" tagx="given-names" title="given-names">T.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCormick</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Immunization Safety Review: Vaccinations And Sudden Unexpected Death In Infancy, Institute of Medicine (US)</span><span class="year" tagx="year" title="year">2003</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">National Academies Press</span><span class="pub-id"><a href="https://dx.doi.org/10.17226/10649">10.17226/10649</a></span></span></li> <li tag="ref"><a name="bb0425" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sudhakar</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robin</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boyd</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agnihothram</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gopal</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yount</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donaldson</span><span class="given-names" tagx="given-names" title="given-names">E.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Platform strategies for rapid response against emerging coronaviruses: MERS-CoV serologic and antigenic relationships in vaccine design</span><span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">209</span><span class="issue" tagx="issue" title="issue">7</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">995</span><span class="lpage" tagx="lpage" title="lpage">1006</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jit609">10.1093/infdis/jit609</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24253287">24253287</a></span></span></li> <li tag="ref"><a name="bb0430" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">W.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhai</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span></span><span class="mixed-article-title" title="mixed-article-title">COVID-19: epidemiology, evolution, and cross-disciplinary perspectives</span><span class="source" tagx="source" title="source">Trends Mol. Med.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molmed.2020.02.008">10.1016/j.molmed.2020.02.008</a></span><span class="comment">(Mar 21)</span></span></li> <li tag="ref"><a name="bb0435" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thilakavathy</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">S.V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Potential factors influencing repeated SARS outbreaks in China</span><span class="source" tagx="source" title="source">Int. J. Environ. Res. Public Health</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">1633</span><span class="comment">(Jan)</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijerph17051633">10.3390/ijerph17051633</a></span></span></li> <li tag="ref"><a name="bb0440" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Surya</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verdià-Bàguena</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aguilella</span><span class="given-names" tagx="given-names" title="given-names">V.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torres</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MERS coronavirus envelope protein has a single transmembrane domain that forms pentameric ion channels</span><span class="source" tagx="source" title="source">Virus Res.</span><span class="volume" tagx="volume" title="volume">201</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">61</span><span class="lpage" tagx="lpage" title="lpage">66</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2015.02.023">10.1016/j.virusres.2015.02.023</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25733052">25733052</a></span></span></li> <li tag="ref"><a name="bb0445" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tama</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brooks</span><span class="given-names" tagx="given-names" title="given-names">C.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Symmetry, form, and shape: guiding principles for robustness in macromolecular machines</span><span class="source" tagx="source" title="source">Annu. Rev. Biophys. Biomol. Struct.</span><span class="volume" tagx="volume" title="volume">35</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2006</span><span class="fpage" tagx="fpage" title="fpage">115</span><span class="lpage" tagx="lpage" title="lpage">133</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.biophys.35.040405.102010">10.1146/annurev.biophys.35.040405.102010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16689630">16689630</a></span></span></li> <li tag="ref"><a name="bb0450" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tesh</span><span class="given-names" tagx="given-names" title="given-names">R.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arroyo</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">da Rosa</span><span class="given-names" tagx="given-names" title="given-names">A.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guzman</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">S.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monath</span><span class="given-names" tagx="given-names" title="given-names">T.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Efficacy of killed virus vaccine, live attenuated chimeric virus vaccine, and passive immunization for prevention of West Nile virus encephalitis in hamster model</span><span class="source" tagx="source" title="source">Emerg. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">12</span><span class="year" tagx="year" title="year">2002</span><span class="fpage" tagx="fpage" title="fpage">1392</span><span class="lpage" tagx="lpage" title="lpage">1397</span><span class="comment">(Dec)</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid0812.020229">10.3201/eid0812.020229</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12498653">12498653</a></span></span></li> <li tag="ref"><a name="bb0455" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Staats</span><span class="given-names" tagx="given-names" title="given-names">H.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cytokines: the future of intranasal vaccine adjuvants</span><span class="source" tagx="source" title="source">Clin. Dev. Immunol.</span><span class="volume" tagx="volume" title="volume">2011</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="comment">(Jul 31)</span><span class="pub-id"><a href="https://dx.doi.org/10.1155/2011/289597">10.1155/2011/289597</a></span></span></li> <li tag="ref"><a name="bb0460" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vita</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Overton</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenbaum</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ponomarenko</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">J.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cantrell</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wheeler</span><span class="given-names" tagx="given-names" title="given-names">D.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gabbard</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hix</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sette</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peters</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The immune epitope database (IEDB) 3.0</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="volume" tagx="volume" title="volume">43</span><span class="issue" tagx="issue" title="issue">D1</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">D405</span><span class="lpage" tagx="lpage" title="lpage">D412</span><span class="comment">(Jan 28)</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkq888">10.1093/nar/gkq888</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25300482">25300482</a></span></span></li> <li tag="ref"><a name="bb0465" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horby</span><span class="given-names" tagx="given-names" title="given-names">P.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayden</span><span class="given-names" tagx="given-names" title="given-names">F.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">G.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A novel coronavirus outbreak of global health concern</span><span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">395</span><span class="issue" tagx="issue" title="issue">10223</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">470</span><span class="lpage" tagx="lpage" title="lpage">473</span><span class="comment">(Internet] Elsevier BV. (Feb)</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/s0140-6736(20)30185-9">10.1016/s0140-6736(20)30185-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31986257">31986257</a></span></span></li> <li tag="ref"><a name="bb0470" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wasmoen</span><span class="given-names" tagx="given-names" title="given-names">T.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kadakia</span><span class="given-names" tagx="given-names" title="given-names">N.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Unfer</span><span class="given-names" tagx="given-names" title="given-names">R.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fickbohm</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cook</span><span class="given-names" tagx="given-names" title="given-names">C.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">H.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Acree</span><span class="given-names" tagx="given-names" title="given-names">W.M.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Protection of cats from infectious peritonitis by vaccination with a recombinant raccoon poxvirus expressing the nucleocapsid gene of feline infectious peritonitis virus</span><span class="source" tagx="source" title="source">Corona And Related Viruses</span><span class="year" tagx="year" title="year">1995</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Boston, MA</span><span class="fpage" tagx="fpage" title="fpage">221</span><span class="lpage" tagx="lpage" title="lpage">228</span></span></li> <li tag="ref"><a name="bb0475" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterhouse</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Procter</span><span class="given-names" tagx="given-names" title="given-names">J.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">D.M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clamp</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barton</span><span class="given-names" tagx="given-names" title="given-names">G.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Jalview version 2-a multiple sequence alignment editor and analysis workbench</span><span class="source" tagx="source" title="source">Bioinformatics.</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">9</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">1189</span><span class="lpage" tagx="lpage" title="lpage">1191</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp033">10.1093/bioinformatics/btp033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19151095">19151095</a></span></span></li> <li tag="ref"><a name="bb0480" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wesseling</span><span class="given-names" tagx="given-names" title="given-names">J.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Godeke</span><span class="given-names" tagx="given-names" title="given-names">G.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schijns</span><span class="given-names" tagx="given-names" title="given-names">V.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prevec</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">F.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horzinek</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rottier</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mouse hepatitis virus spike and nucleocapsid proteins expressed by adenovirus vectors protect mice against a lethal infection</span><span class="source" tagx="source" title="source">J. Gen. Virol.</span><span class="volume" tagx="volume" title="volume">74</span><span class="issue" tagx="issue" title="issue">10</span><span class="year" tagx="year" title="year">1993</span><span class="fpage" tagx="fpage" title="fpage">2061</span><span class="lpage" tagx="lpage" title="lpage">2069</span><span class="comment">(Oct 1)</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8409930">8409930</a></span></span></li> <li tag="ref"><a name="bb0485" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wrapp</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corbett</span><span class="given-names" tagx="given-names" title="given-names">K.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldsmith</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsieh</span><span class="given-names" tagx="given-names" title="given-names">C.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abiona</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">B.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McLellan</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span><span class="source" tagx="source" title="source">Science.</span><span class="volume" tagx="volume" title="volume">367</span><span class="issue" tagx="issue" title="issue">6483</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">1260</span><span class="lpage" tagx="lpage" title="lpage">1263</span><span class="comment">(Mar 13)</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32075877">32075877</a></span></span></li> <li tag="ref"><a name="bb0490" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses</span><span class="source" tagx="source" title="source">bioRxiv.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.03.04.975995">10.1101/2020.03.04.975995</a></span></span></li> <li tag="ref"><a name="bb0495" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y.-M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Z.-G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A new coronavirus associated with human respiratory disease in China</span><span class="source" tagx="source" title="source">Nature.</span><span class="volume" tagx="volume" title="volume">579</span><span class="issue" tagx="issue" title="issue">7798</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">265</span><span class="lpage" tagx="lpage" title="lpage">269</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-020-2008-3">10.1038/s41586-020-2008-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32015508">32015508</a></span></span></li> <li tag="ref"><a name="bb0500" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wüthrich</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wagner</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richarz</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Braun</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Correlations between internal mobility and stability of globular proteins</span><span class="source" tagx="source" title="source">Biophys. J.</span><span class="volume" tagx="volume" title="volume">32</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">1980</span><span class="fpage" tagx="fpage" title="fpage">549</span><span class="lpage" tagx="lpage" title="lpage">560</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/s0006-3495(80)84989-7">10.1016/s0006-3495(80)84989-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7248460">7248460</a></span></span></li> <li tag="ref"><a name="bb0515" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">In silico design of a DNA-based HIV-1 multi-epitope vaccine for Chinese populations</span><span class="source" tagx="source" title="source">Hum. Vaccin. Immunother.</span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">795</span><span class="lpage" tagx="lpage" title="lpage">805</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/21645515.2015.1012017">10.1080/21645515.2015.1012017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25839222">25839222</a></span></span></li> <li tag="ref"><a name="bb0520" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genotyping coronavirus SARS-CoV-2: methods and implications</span><span class="source" tagx="source" title="source">ArXiv</span><span class="volume" tagx="volume" title="volume">2003</span><span class="issue" tagx="issue" title="issue">10965</span><span class="year" tagx="year" title="year">2020</span></span></li> <li tag="ref"><a name="bb0530" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">L.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">Z.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ke</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">J.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">Z.T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Immune responses against SARS-coronavirus nucleocapsid protein induced by DNA vaccine</span><span class="source" tagx="source" title="source">Virology.</span><span class="volume" tagx="volume" title="volume">331</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">128</span><span class="lpage" tagx="lpage" title="lpage">135</span><span class="comment">(Jan 5)</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15582659">15582659</a></span></span></li> <li tag="ref"><a name="bb0535" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeong-ho</span><span class="given-names" tagx="given-names" title="given-names">Lee</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">William</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Laura</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Chinese Scientists Race To Develop Vaccine As Coronavirus Death Toll Jumps</span><span class="year" tagx="year" title="year">2020</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">South China Morning Post</span><span class="comment">Archived from the original on 26 January 2020. (Retrieved 28 January 2020)</span></span></li> <li tag="ref"><a name="bb0540" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Prospects for a MERS-CoV spike vaccine</span><span class="source" tagx="source" title="source">Expert Rev. Vaccines</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">8</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">677</span><span class="lpage" tagx="lpage" title="lpage">686</span><span class="comment">(Aug 3)</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30058403">30058403</a></span></span></li> <li tag="ref"><a name="bb0545" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Network-based drug repurposing for novel coronavirus 2019-nCoV/SARS-CoV-2</span><span class="source" tagx="source" title="source">Cell Discov.</span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">18</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41421-020-0153-3">10.1038/s41421-020-0153-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31934347">31934347</a></span></span></li> <li tag="ref"><a name="bb0550" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span><span class="source" tagx="source" title="source">Nature.</span><span class="volume" tagx="volume" title="volume">579</span><span class="issue" tagx="issue" title="issue">7798</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">270</span><span class="lpage" tagx="lpage" title="lpage">273</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-020-2012-7">10.1038/s41586-020-2012-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32015507">32015507</a></span></span></li> <li tag="ref"><a name="bb0555" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A novel coronavirus from patients with pneumonia in China, 2019</span><span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="volume" tagx="volume" title="volume">382</span><span class="issue" tagx="issue" title="issue">8</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">727</span><span class="lpage" tagx="lpage" title="lpage">733</span><span class="pub-id"><a href="https://dx.doi.org/10.1056/nejmoa2001017">10.1056/nejmoa2001017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31978945">31978945</a></span></span></li> <li tag="ref"><a name="bb0560" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zumla</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">J.F.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azhar</span><span class="given-names" tagx="given-names" title="given-names">E.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">D.S.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">K.-Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronaviruses-drug discovery and therapeutic options</span><span class="source" tagx="source" title="source">Nat. Rev. Drug Discov.</span><span class="volume" tagx="volume" title="volume">15</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">327</span><span class="lpage" tagx="lpage" title="lpage">347</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrd.2015.37">10.1038/nrd.2015.37</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26868298">26868298</a></span></span></li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Authors would like to acknowledge the Department of Biochemistry and Chemistry and Department of Pharmaceuticals and Industrial Biotechnology of Sylhet Agricultural University for the technical support of the project.</p> </div> </div>  </body></html>