<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Viruses</span><span class="iso-abbrev" title="iso-abbrev">Viruses</span><span class="publisher-id" title="publisher-id">viruses</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Viruses</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1999-4915</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7551987">7551987</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/v12091055">10.3390/v12091055</a></span><span class="publisher-id" title="publisher-id">viruses-12-01055</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
The Epidemiological Signature of Pathogen Populations That Vary in the Relationship between Free-Living Parasite Survival and Virulence</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-2184-5622</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomez</span><span class="given-names" tagx="given-names" title="given-names">Lourdes M.</span></span><a href="#af1-viruses-12-01055">1</a><a href="#af2-viruses-12-01055">2</a><a href="#fn1-viruses-12-01055">â€ </a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meszaros</span><span class="given-names" tagx="given-names" title="given-names">Victor A.</span></span><a href="#af2-viruses-12-01055">2</a><a href="#fn1-viruses-12-01055">â€ </a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-0302-1646</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">Wendy C.</span></span><a href="#af3-viruses-12-01055">3</a><a href="#fn2-viruses-12-01055">â€¡</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-1581-8345</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogbunugafor</span><span class="given-names" tagx="given-names" title="given-names">C. Brandon</span></span><a href="#af1-viruses-12-01055">1</a><a href="#af2-viruses-12-01055">2</a><a href="#c1-viruses-12-01055">*</a></span></div><span class="citation_author_institution" id="af1-viruses-12-01055">[1], <span class="email" tagx="email" title="email">lourdes.gomez@yale.edu</span></span><span class="citation_author_institution" id="af2-viruses-12-01055">[2], <span class="email" tagx="email" title="email">victor_meszaros@brown.edu</span></span><span class="citation_author_institution" id="af3-viruses-12-01055">[3], <span class="email" tagx="email" title="email">wcturner@albany.edu</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">brandon.ogbunu@yale.edu</span></div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">â€ </span><p>These authors contributed equally to this work.</p> </div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">â€¡</span><p>Current address: U.S. Geological Survey, Wisconsin Cooperative Wildlife Research Unit, Department of Forest and Wildlife Ecology, University of Wisconsin-Madison, Madison, WI 53706, USA.</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-9</span></span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">1055</span><span class="history" title="history"><span class="received" title="received">received: 2020-8-01</span><span class="accepted" title="accepted">accepted: 2020-9-17</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The relationship between parasite virulence and transmission is a pillar of evolutionary theory that has implications for public health. Part of this canon involves the idea that virulence and free-living survival (a key component of transmission) may have different relationships in different hostâ€"parasite systems. Most examinations of the evolution of virulence-transmission relationshipsâ€"Theoretical or empirical in natureâ€"Tend to focus on the evolution of virulence, with transmission being a secondary consideration. Even within transmission studies, the focus on free-living survival is a smaller subset, though recent studies have examined its importance in the ecology of infectious diseases. Few studies have examined the epidemic-scale consequences of variation in survival across different virulenceâ€"survival relationships. In this study, we utilize a mathematical model motivated by aspects of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) natural history to investigate how evolutionary changes in survival may influence several aspects of disease dynamics at the epidemiological scale. Across virulenceâ€"survival relationships (where these traits are either positively or negatively correlated), we found that small changes (5% above and below the nominal value) in survival can have a meaningful effect on certain outbreak features, including <i>R</i><sub>0</sub>, and on the size of the infectious peak in the population. These results highlight the importance of properly understanding the mechanistic relationship between virulence and parasite survival, as the evolution of increased survival across different relationships with virulence may have considerably different epidemiological signatures.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
virulence</div> <div class="kwd" title="kwd">
free-living survival</div> <div class="kwd" title="kwd">
disease dynamics</div> <div class="kwd" title="kwd">
evolution of infectious disease</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Interactions between the life history of a pathogen and the environment in which it is embedded drive the evolution of virulence. These interactions thus dictate both the experience of disease at the individual host level and the shape of disease dynamics in host populations [<a href="#B1-viruses-12-01055">1</a>,<a href="#B2-viruses-12-01055">2</a>]. The nature of the interaction between virulence and transmission has been the object of both theoretical and empirical examination [<a href="#B2-viruses-12-01055">2</a>,<a href="#B3-viruses-12-01055">3</a>,<a href="#B4-viruses-12-01055">4</a>,<a href="#B5-viruses-12-01055">5</a>,<a href="#B6-viruses-12-01055">6</a>,<a href="#B7-viruses-12-01055">7</a>,<a href="#B8-viruses-12-01055">8</a>]. Free-living survival, here defined as the ability of a pathogen to persist outside of its host, is one of many transmission life-history traits associated with virulence. The relationship between the two varies between hostâ€"pathogen types and different environments [<a href="#B4-viruses-12-01055">4</a>,<a href="#B8-viruses-12-01055">8</a>,<a href="#B9-viruses-12-01055">9</a>,<a href="#B10-viruses-12-01055">10</a>].</p> <p>Several hypotheses serve as the canon in the evolution of virulence, theorizing its relationship with transmission traits. The Curse of the Pharaoh hypothesisâ€"Named after a tale about a mythical curse that torments individuals who dig up tombs of Egyptian pharaohs [<a href="#B11-viruses-12-01055">11</a>]â€"Suggests that, if a parasite has high free-living survival, then it is far less dependent on its host for transmission and, consequently, will have no evolutionary incentive to decrease virulence [<a href="#B2-viruses-12-01055">2</a>,<a href="#B4-viruses-12-01055">4</a>,<a href="#B12-viruses-12-01055">12</a>]. The potential negative fitness consequences of killing hosts rapidly (being highly virulent) can be counteracted by persisting in the environment until the arrival of new susceptible hosts. Any presumptive selection on beneficence may be relaxed: parasites can detrimentally affect the health of hosts at no cost to transmission because most of their life cycle is spent outside of a host. Previous studies support a positive correlation between free-living survival and mortality per infection (a common proxy for virulence) [<a href="#B13-viruses-12-01055">13</a>].</p> <p>Alternatively, the â€œtradeoffâ€� hypothesis suggests that there is some intermediate level of parasite virulence [<a href="#B3-viruses-12-01055">3</a>,<a href="#B6-viruses-12-01055">6</a>,<a href="#B14-viruses-12-01055">14</a>] that is optimal for a given setting. In this scenario, too high a virulence kills the host and parasite and too low a virulence leads to failure to transmit. Applying this hypothesis specifically to free-living survival would suggest that selection for increased free-living survival should come at the expense of virulence (producing a pathogen that is less harmful to the host). Mechanistically, as a consequence of increased adaptation to a nonhost environment, a virus may be less fit to replicate inside a host [<a href="#B9-viruses-12-01055">9</a>,<a href="#B15-viruses-12-01055">15</a>]. For example, a more robust viral capsid may help to survive harsh environmental conditions but may make it more difficult to package RNA/DNA [<a href="#B15-viruses-12-01055">15</a>]. More generally, the tradeoff hypothesis can be framed in the context of a life-history tradeoff: investment in certain parts of the life cycle often comes at the expense of others [<a href="#B2-viruses-12-01055">2</a>,<a href="#B16-viruses-12-01055">16</a>].</p> <p>Theoretical studies have explored varying evolutionary relationships between heightened virulence and extreme pathogen longevity [<a href="#B4-viruses-12-01055">4</a>,<a href="#B5-viruses-12-01055">5</a>,<a href="#B12-viruses-12-01055">12</a>,<a href="#B17-viruses-12-01055">17</a>,<a href="#B18-viruses-12-01055">18</a>,<a href="#B19-viruses-12-01055">19</a>]. One critical component of these studies revolves around whether virulence evolves independently of free-living survival. For example, some models have argued [<a href="#B4-viruses-12-01055">4</a>] that pathogen virulence is independent of survival under a set of conditions: when the hostâ€"pathogen system is at an equilibrium (evolutionary and ecological), if host density fluctuates around an equilibrium, or if turnover of the infected host population is fast relative to the pathogen in the environment. However, if the hostâ€"pathogen system is at disequilibrium and if the dynamics of propagules in the environment are fast compared to the dynamics of infected hosts, then virulence is, as hypothesized, an increasing function of propagule survival [<a href="#B4-viruses-12-01055">4</a>]. Kamo and Boots [<a href="#B17-viruses-12-01055">17</a>] examined this hypothesis by incorporating a spatial structure in the environment using a cellular, automata model and found that, if virulence evolution is independent of transmission, then long-lived infective stages select for higher virulence. However, if there is a tradeoff between virulence and transmission, there is no evidence for the Curse of the Pharaoh hypothesis, and in fact, higher virulence may be selected for by shorter rather than long-lived infectious stages. Further, the evolution of high virulence does not have to occur solely through a transmissionâ€"virulence tradeoff. Day [<a href="#B18-viruses-12-01055">18</a>] demonstrated how pathogens can evolve high virulence and even select for traits to kill the host (e.g., toxins) if pathogen transmission and reproductive success are decoupled. These studies emphasized the context-dependence of virulenceâ€"survival relationships. Understanding where in the relationship between virulence and survival a given pathogen population exists may allow one to understand how virus evolution will manifest at the level of epidemics.</p> <p>In this study, we examine the epidemic consequences of different virulenceâ€"survival relationshipsâ€"Positive and negative correlationâ€"In a viral disease with an environmental transmission component. In order to measure how pathogen survival influences disease dynamics, we included an environmental compartment in our model, which represents contaminated environments that act as a reservoir for persisting pathogens, causing disease spread when they come in contact with susceptible individuals (infection via â€œfomitesâ€�) [<a href="#B20-viruses-12-01055">20</a>,<a href="#B21-viruses-12-01055">21</a>].</p> <p>We find that the identity of the virulenceâ€"free-living survival relationship (e.g., positive vs. negative) has distinct implications for how an epidemic will unfold. Some, but not all, features of an outbreak are dramatically influenced by the nature of the underlying virulenceâ€"survival relationship. This indicates that signatures for evolution (adaptive or other) in a pathogen population will manifest more conspicuously in certain features of an outbreak. We reflect on these findings in light of their theoretical implications on the evolution and ecology of infectious disease and for their potential utility in public health interventions.</p> </div> <div class="2.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
2. Materials and Methods</div> <div class="2.1.modelmotivationandapplication" title="sec"> <div class="title" tagx="title" title="title">
2.1. Model Motivation and Application</div> <p>The mathematical model explored in this study is adapted from a recent one developed to investigate environmental transmission of SARS-CoV-2 during the early-stage outbreak dynamics of coronavirus disease 2019 (COVID-19), with parameter values based on fits to actual country outbreak data [<a href="#B22-viruses-12-01055">22</a>]. In this study, we utilize this model to examine questions about the evolution of free-living survival. While the phenomenon we examine is a very relevant one that manifests in the real world, we want to emphasize that none of the methods or results in this study are intended to be applied to the current COVID-19 pandemic (as of September, 2020). This study is an attempt at responsible theoretical biology, with data-informed models and inferences that are germane to the natural world. However, neither do we support the extrapolation of these findings to any particular aspect of COVID-19 nor should they inform a policy or intervention. The model applies to a number of scenarios that include outbreaks in a naÃ¯ve host population. This describes situations such as the evolution of novel viral lineages, viral spillover events, or host shifts, where a virus with a preexisting relationship between virulence and survival emerges in a population of new hosts. Another such scenario where this model applies is one where a virus has already emerged but evolves in a subpopulation in the novel hosts before a migration event of some kind introduces the evolved virus population to a fully susceptible population of hosts.</p> </div> <div class="2.2.modeldescription" title="sec"> <div class="title" tagx="title" title="title">
2.2. Model Description</div> <p>The model is implemented via a set of ordinary differential equations, defined by Equations (1)â€"(6). It implements viral free-living survival via the â€œWaterborne Abiotic or other Indirect Transmission (WAIT)â€� modelling framework, coupling individuals and the pathogen within the environment [<a href="#B23-viruses-12-01055">23</a>,<a href="#B24-viruses-12-01055">24</a>].</p> <p>Within the model, the <i>Î²<sub>w</sub></i> term allows for individuals to become infected via viral pathogen deposited in the environment and terms <i>ð�œŽ<sub>A</sub></i> and <i>ð�œŽ<sub>I</sub></i> allow asymptomatic and symptomatic individuals to deposit pathogens into the environment, respectively. Adapted from the more traditional SEIR (susceptible-exposed-infected-recovered) model, the SEAIR-W (susceptible-exposed-asymptomatic-infected-recovered-WAIT) model interrogates the consequences of the two hypotheses outlined above while representing the dynamics of a very relevant disease system (SARS-CoV-2) that includes an asymptomatic infectious population. While the importance of asymptomatic transmission was debated early in the pandemic, many studies have affirmed its role in the spread of disease [<a href="#B25-viruses-12-01055">25</a>,<a href="#B26-viruses-12-01055">26</a>,<a href="#B27-viruses-12-01055">27</a>]. Though environmental transmission of SARS-CoV-2 remains a controversial topic, it is plausible that asymptomatic individuals may spread disease through frequent contact with the environment, thus increasing the proportion of virus that is free-living [<a href="#B28-viruses-12-01055">28</a>]. We acknowledge that mathematical models of epidemics can be limited by â€œidentifiability,â€� which can obfuscate the relative importance of some routes of transmission. In models that have both indirect and direct routes of transmission, it can be very difficult to conclude that one route is predominant [<a href="#B29-viruses-12-01055">29</a>,<a href="#B30-viruses-12-01055">30</a>,<a href="#B31-viruses-12-01055">31</a>]. </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
S</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
Î¼</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
S</div> </div> </div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mi_UNKNOWN">
N</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mi_UNKNOWN">
W</div> </div> <div class="mi_UNKNOWN">
W</div> </div> </div> <div class="mi_UNKNOWN">
S</div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(2)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mi_UNKNOWN">
N</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mi_UNKNOWN">
W</div> </div> <div class="mi_UNKNOWN">
W</div> </div> </div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Îµ</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Î¼</div> </div> </div> <div class="mi_UNKNOWN">
E</div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(3)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
Îµ</div> <div class="mi_UNKNOWN">
E</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Ï‰</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Î¼</div> </div> </div> <div class="mi_UNKNOWN">
A</div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(4)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mi_UNKNOWN">
Ï‰</div> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mi_UNKNOWN">
I</div> </div> </div> </div> <div class="mi_UNKNOWN">
I</div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(5)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
R</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
Ï‰</div> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
Î¼</div> <div class="mi_UNKNOWN">
R</div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(6)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
W</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mi_UNKNOWN">
N</div> </div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
W</div> </div> </div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
W</div> </div> </div></div></div> <p /> <p><a href="#viruses-12-01055-f001">Figure 1</a> depicts the compartmental diagram for the model. The direction of the arrows corresponds to the flow of the individuals and the pathogen through the system. Note that individuals can move directly from the asymptomatically infected compartment to the recovered compartment (bypassing the symptomatic compartment) via what we call a â€œmild trackâ€�. The dashed arrows represent WAIT coupling to the environment. The model is inspired by one developed to interrogate environmental transmission of SARS-CoV-2 [<a href="#B22-viruses-12-01055">22</a>].</p> </div> <div class="2.3.simulationsofoutbreaks" title="sec"> <div class="title" tagx="title" title="title">
2.3. Simulations of Outbreaks</div> <p>The system was numerically integrated using the â€œodeintâ€� solver in the Scipy 1.4â€"Python scientific computation suite [<a href="#B32-viruses-12-01055">32</a>]. The simulations track the populations for each of the compartments listed in <a href="#viruses-12-01055-f001">Figure 1</a>. Each model run occurred over 250 days, which amounts to over 8 months of the epidemic or 5Ã— the peak of the outbreak. This length of time is consistent with the antecedent SARS-CoV-2 model [<a href="#B22-viruses-12-01055">22</a>], long enough for the dynamics of the system to manifest. Note however that, for this study, we are especially interested in the early window of an outbreak: the first 30 days. We focus on this window because this is the time frame that best captures the underlying physics of an epidemic, as 30 days is often before populations are able to adjust their individual behaviors. The code constructed for the analysis in this study is publicly available on github: <span class="uri" title="uri"><a href="https://github.com/OgPlexus/Pharaohlocks">https://github.com/OgPlexus/Pharaohlocks</a></span>.</p> </div> <div class="2.4.populationdefinitionsandparametervalues" title="sec"> <div class="title" tagx="title" title="title">
2.4. Population Definitions and Parameter Values</div> <p><a href="#viruses-12-01055-t001">Table 1</a> outlines the definitions of each population and provides the initial population values used for all simulations conducted in this study. The nominal parameter values used are defined in <a href="#viruses-12-01055-t002">Table 2</a>. The initial values are drawn from the aforementioned COVID-19 outbreak study, derived from empirical findings and country-level outbreak data [<a href="#B22-viruses-12-01055">22</a>].</p> </div> <div class="2.5.virulencedefinition" title="sec"> <div class="title" tagx="title" title="title">
2.5. Virulence Definition</div> <p>In this study, we define virulence as the capacity to cause a disease. In order to measure it, we utilize a set of parameters that uniformly increase the rate or probability of causing symptomatic disease or the severity of those symptoms (including death). Our definition is more comprehensive than many other models of parasite virulence (e.g., [<a href="#B4-viruses-12-01055">4</a>,<a href="#B13-viruses-12-01055">13</a>]), which tend to focus on a single aspect of the natural history of disease associated with harm to a host (e.g., the fitness consequences of an infection on the host population or the case fatality rate). Instead of having to justify a definition built around a single term (e.g., the term associated with fatality), we took a collective approach to defining virulence through all terms that foment the viral-induced onset of symptomatic disease and death. This definition allows for the reality of pleiotropic effects in viral pathogens, where adaptations can have multiple effects on the natural history of disease [<a href="#B2-viruses-12-01055">2</a>,<a href="#B33-viruses-12-01055">33</a>]. Our definition of virulence emphasizes terms that influence host wellness and/or are symptoms of disease. The iteration of virulence used in this study also undermines the potential for overly weighting only one or a small number of parameters under a large umbrella of virulence. Because so many varying definitions exist for virulence, we have also performed calculations according to a different definition of virulence, one that exclusively considers terms that have a detrimental direct effect on the host and neither of the terms that reflect symptoms of severe disease (<i>ð�œŽ<sub>a</sub></i> and <i>ð�œŽ<sub>I</sub></i>). These calculations can be found in the <a href="#app1-viruses-12-01055">Supplementary Materials</a>.</p> <p>The collection of parameters that we use to define virulence are as follows: the infected population death rate (<i>ð�œ‡<sub>I</sub></i>), the incubation period of SARS-CoV-2 (<i>ð�œ‚</i>), the rate of transfer from asymptomatic to symptomatic (1/<i>â�µ</i>), the infected population recovery rate (<i>Î½</i>), the percent of individuals that move from the asymptomatic to the recovered compartment without showing symptoms (the â€œmildâ€� recovery track, <i>p</i>), the contact rate of people with people Ã— the transmission probability of people to people by an asymptomatic individual (<i>Î²<sub>A</sub></i>), the contact rate of people with people Ã— the transmission probability of people to people by an asymptomatically infected person (<i>Î²<sub>I</sub></i>), the contact rate of people with the environment Ã— the probability of shedding by an asymptomatic individual to the environmental (<i>ð�œŽ<sub>A</sub></i>), the contact rate of people with the environment Ã— the probability of symptomatically infected individuals shedding in the environment (<i>ð�œŽ<sub>I</sub></i>), and the average number of days before infection (1/<i>Îµ</i>).</p> <p><a href="#viruses-12-01055-t003">Table 3</a> outlines the direction in which each of the virulence-associated parameters are modulated as virulence decreases or increases. An up arrow (â†‘) indicates the parameter increases (by an equivalent percent) when the percent virulence is changed. A down arrow (â†") indicates the parameter decreases (by an equivalent percent) when the percent change in virulence is applied. Changes in virulence are then defined, in this study, as an equivalent uniform (percent) change in each of the parameters listed above. For the purposes of our study, we modify virulence by changing all parameters associated with virulence by 5%. One could also disambiguate virulence into changes in individual subcomponents; however, that is not the focus of this current study.</p> </div> <div class="2.6.survivaldefinition" title="sec"> <div class="title" tagx="title" title="title">
2.6. Survival Definition</div> <p>Survival is defined as the set of parameters that, when uniformly modulated, increases the pathogenâ€™s probability of surviving the outside environment and successfully infecting a new host [<a href="#B2-viruses-12-01055">2</a>]. In our model, this includes both the waning virus rate in the environment (<i>k</i>) and the contact rate of an individual with the environment Ã— the transmission probability of the environment to people (<i>Î²<sub>w</sub></i>). <a href="#viruses-12-01055-t004">Table 4</a> outlines the direction (increasing or decreasing) in which these parameters are modulated when survival is decreased or increased. Within both models, a (percent) change in survival is defined as an equivalent uniform (percent) change in the survival parameters.</p> <p>Throughout this study, the impact of changes in virulence and survival (and the relationship between these traits) are assessed with respect to the following four epidemic metrics: the number of infected individuals (asymptomatic and symptomatic) at the maximum (when the outbreak is at its most severe), the rate at which the peak infected population is reached (t<sub>max</sub><sup>âˆ’1</sup>), the total infected population after 30 days, and the basic reproductive ratio (<i>R</i><sub>0</sub>). Importantly, among these signatures, the basic reproductive ratio is the most frequently used in epidemiology and benefits from familiarity and mathematical formalism (see <a href="#sec2dot7-viruses-12-01055">Section 2.7</a>). The other signatures are determined through simulations of an epidemic for a given set of parameter values. Nonetheless, this studyâ€™s inclusion of multiple features of the epidemic allows us to examine how variation in virus life-history traits may influence different aspects of an epidemic in peculiar ways.</p> </div> <div class="2.7.basicreproductiveratio" title="sec"> <div class="title" tagx="title" title="title">
2.7. Basic Reproductive Ratio</div> <p>Equations (7)â€"(9) give the analytic expression of the basic reproductive ratio (<i>R</i><sub>0</sub>) for the model used in this study. This expression for <i>R</i><sub>0</sub> can be deconstructed into two components. Equation (8) only contains parameters associated with person to person transmission (<i>R<sub>p</sub></i>), while Equation (9) solely contains parameters associated with transmission from the environment (<i>R<sub>e</sub></i>). In the <a href="#app1-viruses-12-01055">Supplementary Materials</a>, we provide additional information on these terms and their derivations. Applying the parameters values in <a href="#viruses-12-01055-t002">Table 2</a>, the numerical value of the basic reproductive ratio is given as <i>R</i><sub>0</sub> ~ 2.82. </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(7)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mi_UNKNOWN">
p</div> </div> <div class="msqrt_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mi_UNKNOWN">
p</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
4</div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mi_UNKNOWN">
e</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(8)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mi_UNKNOWN">
p</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Îµ</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
v</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mi_UNKNOWN">
Ï‰</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Îµ</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Ï‰</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
v</div> </div> </div> </div> </div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(9)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mi_UNKNOWN">
e</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Îµ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î²</div> <div class="mi_UNKNOWN">
W</div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> <div class="mi_UNKNOWN">
A</div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
v</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Ïƒ</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
âˆ’</div> <div class="mi_UNKNOWN">
p</div> </div> </div> <div class="mi_UNKNOWN">
Ï‰</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Îµ</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
Ï‰</div> </div> </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
Î¼</div> <div class="mi_UNKNOWN">
I</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
v</div> </div> </div> </div> </div> </div> </div></div></div> <p /> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
3. Results</div> <div class="3.1.modelsensitivityanalysis" title="sec"> <div class="title" tagx="title" title="title">
3.1. Model Sensitivity Analysis</div> <p><a href="#viruses-12-01055-f002">Figure 2</a> depicts a tornado plot that communicates the sensitivity of the model to permutations in parameters. Across features, the model is most sensitive to parameters that are considered virulence-associated (<a href="#viruses-12-01055-t003">Table 3</a>) and is relatively less sensitive to survival-associated parameters (<a href="#viruses-12-01055-t004">Table 4</a>). Similar to other features, <i>R</i><sub>0</sub> (<a href="#viruses-12-01055-f002">Figure 2</a>D) of the model is most sensitive to the parameters <i>â�µ</i>, <i>Î²<sub>A</sub></i>, and <i>Î½</i>. The sensitivity of <i>R</i><sub>0</sub> to changes in <i>â�µ</i> reflects the importance of the rate of conversion to the symptomatic state on model dynamics. In addition, <i>Î²<sub>A</sub></i> has a very important influence on the model, consistent with other findings for COVID-19 that have emphasized the importance of asymptomatic transmission in disease spread [<a href="#B25-viruses-12-01055">25</a>,<a href="#B26-viruses-12-01055">26</a>,<a href="#B27-viruses-12-01055">27</a>].</p> </div> <div class="3.2.illustrativedynamicsofmodelsystem" title="sec"> <div class="title" tagx="title" title="title">
3.2. Illustrative Dynamics of Model System</div> <p>Based on the parameter values in <a href="#viruses-12-01055-t002">Table 2</a>, <a href="#viruses-12-01055-f003">Figure 3</a>A demonstrates the base dynamics of the model playing out over the first 100 days while <a href="#viruses-12-01055-f003">Figure 3</a>B shows the dynamics within the environment over the course of 250 days. In these dynamics, the population begins to be fixed for susceptible hosts. The disease dynamics manifest in the shapes of the curves corresponding to exposed, asymptomatic, and symptomatic individuals. Note the long tail of the curve corresponding to contamination by the environment. The environment remains infectious even after the infected populations have declined in number. The length and shape of this tail are influenced by the free-living survival of the virus.</p> </div> <div class="3.3.theepidemicconsequencesofvaryingvirulenceandsurvival" title="sec"> <div class="title" tagx="title" title="title">
3.3. The Epidemic Consequences of Varying Virulence and Survival</div> <p>In the next analysis, we examine the epidemic consequences of varying traits associated with survival and virulence. One can consider this as a scenario where we compare the endpoints of evolution of different virus populations (corresponding to combinations of values of survival and virulence) and calculating how these evolved populations manifest in epidemic terms.</p> <p>In <a href="#viruses-12-01055-f004">Figure 4</a>, we observe how dynamics of the outbreak are influenced across a space of combinations of traits altering virulence (see <a href="#viruses-12-01055-t003">Table 3</a> for a list of virulence-associated parameters) and survival (see <a href="#viruses-12-01055-t004">Table 4</a> for a list of survival-associated parameters), changed by Â±5% (10% overall). In <a href="#viruses-12-01055-f004">Figure 4</a>D, we demonstrate how changes in virulence and free-living survival traits influence <i>R</i><sub>0</sub>, with variation in virulence-related traits having the largest effect on <i>R</i><sub>0</sub>. Of note is how the range in <i>R</i><sub>0</sub> values varies widely across virulenceâ€"survival values, from nearly 2.0 to 3.7 (<a href="#viruses-12-01055-f004">Figure 4</a>D).</p> </div> <div class="3.4.implicationsofvirulenceâ€" title="sec"> <div class="title" tagx="title" title="title">
3.4. Implications of Virulenceâ€"Survival Relationships at Their Relative Extremes</div> <p>Having observed how outbreak dynamics are influenced by variation in traits that alter virulenceâ€"survival phenotypes, we then examined how each outbreak metric is influenced by the extreme (Â±5%) values of the trait combinations considered. Specifically, we assess how a change in pathogen survival affects outbreak dynamics, based on two expected relationships between survival and virulence traits.</p> </div> <div class="3.5.positivecorrelationbetweensurvivalandvirulence" title="sec"> <div class="title" tagx="title" title="title">
3.5. Positive Correlation Between Survival and Virulence</div> <p>In a positive correlation scenario, high values for survival would be associated with high values for virulence [<a href="#B4-viruses-12-01055">4</a>,<a href="#B13-viruses-12-01055">13</a>]. Because the correlations we observe are often not exactly linear, we utilize quadrants to express a trend, allowing for some variance around the expected â€œlineâ€�. In <a href="#viruses-12-01055-f005">Figure 5</a>, the positive correlation scenario can be represented by combinations of virulence and survival residing in quadrants I and III.</p> <p>If hostâ€"pathogen evolution proceeds according to a positive correlation scenario, all outbreak metrics would show an increase in severity as both survival and virulence increase. Across the range of variation in virulence and survival traits considered (5% above and below the nominal value), the peak number of infected individuals increases by approximately 35%, the rate at which the peak is reached increases by approximately 16%, the total number of infected individuals after 30 days increases by approximately 98%, and <i>R</i><sub>0</sub> increases by approximately 94% (<a href="#viruses-12-01055-f006">Figure 6</a> and <a href="#viruses-12-01055-t005">Table 5</a>).</p> </div> <div class="3.6.negativecorrelationbetweensurvivalandvirulence" title="sec"> <div class="title" tagx="title" title="title">
3.6. Negative Correlation Between Survival and Virulence</div> <p>In a negative correlation scenario, high values for survival would be associated with low values for virulence [<a href="#B2-viruses-12-01055">2</a>,<a href="#B9-viruses-12-01055">9</a>,<a href="#B15-viruses-12-01055">15</a>] and a low peak in total infected population. Pathogens with a life history that exhibits negative virulenceâ€"survival associations would likely appear in quadrants II and IV in <a href="#viruses-12-01055-f005">Figure 5</a>.</p> <p>Under negative correlation, outbreak severity decreases as survival increases. Across the measured range of variation in virulenceâ€"survival traits, the peak number of infected individuals decreases by approximately 23%, the rate at which the epidemic peak is reached decreases by 0.15%, the total number of infected individuals decreases by 3%, and <i>R</i><sub>0</sub> decreases by approximately 84% (<a href="#viruses-12-01055-f006">Figure 6</a> and <a href="#viruses-12-01055-t006">Table 6</a>). Across all metrics considered, the effects of increased viral survival on outbreak dynamics is more extreme under the positive correlation than the negative correlation scenarios (<a href="#viruses-12-01055-f006">Figure 6</a>).</p> </div> <div class="3.7.dynamicsofepidemicsatextremevaluesforvirusfree-livingsurvival" title="sec"> <div class="title" tagx="title" title="title">
3.7. Dynamics of Epidemics at Extreme Values for Virus Free-Living Survival</div> <p>In <a href="#viruses-12-01055-f007">Figure 7</a>, we observe the disease dynamics at extreme values for survival and the dynamics corresponding to the fraction of the environment that is contaminated with infectious virus. Consistent with the data represented in <a href="#viruses-12-01055-f006">Figure 6</a>, we observe that minimum and maximum simulations differ more substantially for extreme survival scenarios in the positive correlation scenario than for the negative correlation scenario.</p> <p>The feature of different outbreaks that varies most ostensibly between the correlation scenarios is the time needed to reach the peak number of infected individuals. In positive correlation simulations, one can observe that the low virulence, low survival scenario (<a href="#viruses-12-01055-f007">Figure 7</a>A,B) takes longer to reach the peak number of infected individuals. Most notably, however, the low virulence, low survival setting has a far smaller peak of environmental contamination and shorter tail relative to its high virulence, high survival counterpart (<a href="#viruses-12-01055-f007">Figure 7</a>D). Similarly, intriguing findings exist in the comparison between the simulation sets corresponding to extremes in the negative correlation setting (<a href="#viruses-12-01055-f007">Figure 7</a>Eâ€"H). Especially notable is the difference in the length of the tail of the environmental contamination for the high virulence, low survival combination (<a href="#viruses-12-01055-f007">Figure 7</a>F) vs. the low virulence, low survival combination variant (<a href="#viruses-12-01055-f007">Figure 7</a>H). The explanation is that, in this model, higher virulence influences (among many other things) the rate at which the virus is shed into the environment from either the asymptomatic (<i>ð�œŽ<sub>A</sub></i>) or symptomatic (<i>ð�œŽ<sub>I</sub></i>) host. We observe how the high virulence, low survival simulation (<a href="#viruses-12-01055-f007">Figure 7</a>E) features a symptomatic peak that is larger in size and is prolonged relative to the lower virulence counterpart (<a href="#viruses-12-01055-f007">Figure 7</a>G). This relatively large symptomatic population sheds infectious virus into the environment for a longer period of time, contributing to the long tail of contaminated environments observed in <a href="#viruses-12-01055-f005">Figure 5</a>F.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
4. Discussion</div> <p>The virulenceâ€"survival relationship drives the consequences of virus evolution on the trajectory of an outbreak. In this study, we examined how different virulenceâ€"survival relationships may dictate different features of outbreaks at the endpoints of evolution (according to the positive or negative correlation scenarios). When the parameter space for virulence and survival is mapped, we find that certain outbreak metrics are more sensitive to change in free-living survival and virulence than others and that the nature of this sensitivity differs depending on whether survival and virulence are positively or negatively correlated.</p> <p>For the positive correlation scenario, when free-living survival varies between 5% below and above the nominal value, we observed a dramatic change in the total number of infected individuals in the first 30 days (98% increase from minimum survival to maximum survival; <a href="#viruses-12-01055-t005">Table 5</a>), and <i>R</i><sub>0</sub> nearly doubles (94% increase; <a href="#viruses-12-01055-t005">Table 5</a>). These two traits are, of course, connected: the theoretical construction of the <i>R</i><sub>0</sub> metric specifically applies to settings where a pathogen spreads in a population of susceptible hosts [<a href="#B34-viruses-12-01055">34</a>,<a href="#B35-viruses-12-01055">35</a>], an early window that is captured in the first 30 days.</p> <p>When survival and virulence are negatively correlated, different outbreak dynamics emerge: while the <i>R</i><sub>0</sub> difference between minimum and maximum survival is significant (approximately 84% decrease), the total number of infected individuals only changes by roughly 3% (<a href="#viruses-12-01055-t006">Table 6</a>). This large difference between <i>R</i><sub>0</sub> at higher and lower survival values also does not translate to a difference in the total number of infected individuals in the first 30 days of an infection (the early outbreak window). In a scenario where survival and virulence are negatively correlated, a highly virulent and less virulent virus population can have similar signatures on a population with respect to the number of infected individuals in the first month. Thus, simply measuring the number of infected individuals in the first month of an outbreak is unlikely to reveal whether a pathogen population has undergone adaptive evolution or has evolved in a manner that meaningfully influences the natural history of disease.</p> <p>Notably, for scenarios where survival and virulence are both positively and negatively correlated, the time that it takes for an epidemic to reach its maximum number of infected individuals changes little across extreme values of survival (12% in the positive correlation scenario; 0.15% in the negative correlation scenario; see <a href="#viruses-12-01055-t005">Table 5</a> and <a href="#viruses-12-01055-t006">Table 6</a>). That is, the time that it takes for an epidemic to reach its peak (however high) is not especially sensitive to evolution in virulence or survival.</p> <div class="practicalimplicationsfortheunderstandingofoutbreakscausedbyemergingviruses" title="sec"> <div class="title" tagx="title" title="title">
Practical Implications for the Understanding of Outbreaks Caused by Emerging Viruses</div> <p>That different features of an outbreak are differentially influenced by the endpoints of viral life-history evolution highlights how epidemiology should continue to consider principles in the evolution and ecology of infectious disease in its analyses and predictions. As not all features of an epidemic are going to be equally reliable signatures of virus evolution, we should carefully consider the data on how the dynamics of an epidemic change when making inferences about whether a pathogen population is essentially different from prior iterations (e.g., prior outbreaks of the same virus type). The results of this study suggest that carefully constructed, mechanistically sound models of epidemics are important, both for capturing the dynamics of an outbreak and for abetting our efforts to understand how evolution of survival and virulence influences disease dynamics.</p> <p>For example, the potential for adaptive evolution of SARS-CoV-2 has emerged as a possible explanation for different COVID-19 dynamics in different countries. We suggest that such interpretations should be considered with caution and that they require very specific types of evidence to support them. As of 1 July 2020, any conclusion that widespread SARS-CoV-2 evolution is an explanation for variation in disease patterns across settings (space and/or time) is premature.</p> <p>The practical process of interpreting the evolutionary consequences of signals of virus evolution should encompass several discrete steps. Firstly, we should determine whether molecular signatures exist for adaptive evolution. Adaptive evolution would manifest in observable differences in genotype and phenotype and, perhaps, in the natural history of disease. Secondly, we should aim to attain knowledge of the underlying mechanistic relationship between survival and virulence. This knowledge is not necessarily easy to attain (it requires extensive laboratory studies) but would allow added biological insight: we may be able to extrapolate how changes in some traits (e.g., those that compose survival) influence others (e.g., those that influence virulence).</p> <p>More generally, our findings suggest that the ability to detect the consequences of virus evolution would depend on which feature of an outbreak an epidemiologist measures: from our analysis, <i>R</i><sub>0</sub> is most impacted by changes in virulence and survival. In addition, the total number of infected individuals in the early window and the size of the infected â€œpeakâ€� would each be impacted most readily by changes in virulenceâ€"survival traits. The rate at which the epidemic peak was reached, on the other hand, showed relatively little change as survival increased or between the two correlation scenarios. Consequently, it would not serve as a useful proxy for virus evolution.</p> <p>While the stochastic, sometimes entropic nature of epidemics renders them very challenging to predict [<a href="#B36-viruses-12-01055">36</a>], we suggest that canons such as life-history theory and the evolution of virulence provide useful lenses that can aid in our ability to interpret how life-history changes in virus populations will manifest at the epidemiological scale. We propose that, in an age of accumulating genomic and phenotypic data in many pathogenâ€"host systems, we continue to responsibly apply or modify existing theory in order to collate said data into an organized picture for how different components of the hostâ€"parasite interaction influence the shape of viral outbreaks of various kinds.</p> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>The authors would like to thank members of the OGPlexus for helpful discussions and input on all aspects of the manuscript. In particular, we thank M.M.D. and A.L.M. for especially helpful interactions.</p> </div> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <p>The following are available online at <span class="uri" title="uri"><a href="https://www.mdpi.com/1999-4915/12/9/1055/s1">https://www.mdpi.com/1999-4915/12/9/1055/s1</a></span>. Table S1. Fixed parameters, and their references. Table S2. Alternative virulence definition containing only items that directly affect host well-being (fitness). Figure S1. Alternative virulence definition: The impact of varying virus virulence and survival assessed on four key epidemic metrics. Figure S2. Alternative virulence definition: The percent change in SEAIR-W outbreak metrics as survival increases from âˆ’5% to +5% for different virulence-survival relationships (positive correlation and negative correlation). Figure S3. Alternative virulence definition: Virus outbreak dynamics for the extreme values of virulence and free-living survival considered for the two hypotheses. Table S3. Alternative virulence definition: Comparing epidemic metrics under low survival/low virulence versus high survival/high virulence scenarios. Table S4. Alternative virulence definition: Comparing epidemic metrics under low survival/low virulence versus high survival/high virulence scenarios (as in the negative correlation scenario). Figure S4. Side by side comparison of the main text virulence definition (A1 â†’ D1) and alternative definition of virulence (A2 â†’ D2). Figure S5. Side by side comparison of the main text virulence definition (left) and alternative definition of virulence (right). Figure S6. Side by side comparison of the main text definition (A<sub>1</sub> â†’ H<sub>1</sub>) and supplemental definition of virulence (A<sub>2</sub> â†’ H<sub>2</sub>).</p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="viruses-12-01055-s001.zip">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>Conceptualization, L.M.G., W.C.T. and C.B.O.; methodology, V.A.M. and C.B.O.; software, V.A.M.; validation, V.A.M. and C.B.O.; formal analysis, L.M.G., V.A.M., W.C.T. and C.B.O.; investigation, L.M.G., V.A.M., W.C.T. and C.B.O.; resources, C.B.O.; data curation, L.M.G and V.A.M.; writingâ€"Original draft preparation, L.M.G., V.A.M., W.C.T. and C.B.O.; writingâ€"Review and editing, L.M.G., V.A.M., W.C.T. and C.B.O.; visualization, V.A.M.; supervision, C.B.O. and W.C.T.; project administration, C.B.O.; funding acquisition, N/A. All authors have read and agreed to the published version of the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This research received no external funding.</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-viruses-12-01055" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ewald</span><span class="given-names" tagx="given-names" title="given-names">P.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Host-parasite relations, vectors, and the evolution of disease severity</span><span class="source" tagx="source" title="source">Annu. Rev. Ecol. Syst.</span><span class="year" tagx="year" title="year">1983</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">465</span><span class="lpage" tagx="lpage" title="lpage">485</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.es.14.110183.002341">10.1146/annurev.es.14.110183.002341</a></span></span></li> <li tag="ref"><a name="B2-viruses-12-01055" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldhill</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">P.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The evolution of life history trade-offs in viruses</span><span class="source" tagx="source" title="source">Curr. Opin. Virol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">79</span><span class="lpage" tagx="lpage" title="lpage">84</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.coviro.2014.07.005">10.1016/j.coviro.2014.07.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25087040">25087040</a></span></span></li> <li tag="ref"><a name="B3-viruses-12-01055" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robert</span><span class="given-names" tagx="given-names" title="given-names">R.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Trends and perspectives: Coevolution of hosts and parasites</span><span class="source" tagx="source" title="source">Parasitology</span><span class="year" tagx="year" title="year">1982</span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">411</span><span class="lpage" tagx="lpage" title="lpage">426</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0031182000055360">10.1017/S0031182000055360</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6755367">6755367</a></span></span></li> <li tag="ref"><a name="B4-viruses-12-01055" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonhoeffer</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lenski</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ebert</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The curse of the pharaoh: The evolution of virulence in pathogens with long living propagules</span><span class="source" tagx="source" title="source">Proc. R. Soc. B Biol. Sci.</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">263</span><span class="fpage" tagx="fpage" title="fpage">715</span><span class="lpage" tagx="lpage" title="lpage">721</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.1996.0107">10.1098/rspb.1996.0107</a></span></span></li> <li tag="ref"><a name="B5-viruses-12-01055" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gandon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The curse of the pharoah hypothesis</span><span class="source" tagx="source" title="source">Proc. R. Soc. B Biol. Sci.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">265</span><span class="fpage" tagx="fpage" title="fpage">1545</span><span class="lpage" tagx="lpage" title="lpage">1552</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.1998.0470">10.1098/rspb.1998.0470</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9744106">9744106</a></span></span></li> <li tag="ref"><a name="B6-viruses-12-01055" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alizon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurford</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mideo</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Baalen</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Virulence evolution and the trade-off hypothesis: History, current state of affairs and the future</span><span class="source" tagx="source" title="source">J. Evol. Biol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">245</span><span class="lpage" tagx="lpage" title="lpage">259</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1420-9101.2008.01658.x">10.1111/j.1420-9101.2008.01658.x</a></span></span></li> <li tag="ref"><a name="B7-viruses-12-01055" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cressler</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McLeod</span><span class="given-names" tagx="given-names" title="given-names">D.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rozins</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoogen</span><span class="given-names" tagx="given-names" title="given-names">J.V.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The adaptive evolution of virulence: A review of theoretical predictions and empirical tests</span><span class="source" tagx="source" title="source">Parasitology</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">143</span><span class="fpage" tagx="fpage" title="fpage">915</span><span class="lpage" tagx="lpage" title="lpage">930</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S003118201500092X">10.1017/S003118201500092X</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26302775">26302775</a></span></span></li> <li tag="ref"><a name="B8-viruses-12-01055" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rafaluk-Mohr</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The relationship between parasite virulence and environmental persistence: A meta-analysis</span><span class="source" tagx="source" title="source">Parasitology</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">146</span><span class="fpage" tagx="fpage" title="fpage">897</span><span class="lpage" tagx="lpage" title="lpage">902</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0031182019000015">10.1017/S0031182019000015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30777585">30777585</a></span></span></li> <li tag="ref"><a name="B9-viruses-12-01055" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogbunugafor</span><span class="given-names" tagx="given-names" title="given-names">C.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alto</span><span class="given-names" tagx="given-names" title="given-names">B.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Overton</span><span class="given-names" tagx="given-names" title="given-names">T.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhushan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morales</span><span class="given-names" tagx="given-names" title="given-names">N.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">P.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolution of increased survival in RNA viruses specialized on cancer-derived cells</span><span class="source" tagx="source" title="source">Am. Nat.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">181</span><span class="fpage" tagx="fpage" title="fpage">585</span><span class="lpage" tagx="lpage" title="lpage">595</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/670052">10.1086/670052</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23594543">23594543</a></span></span></li> <li tag="ref"><a name="B10-viruses-12-01055" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rafaluk</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jansen</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulenburg</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joop</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">When experimental selection for virulence leads to loss of virulence</span><span class="source" tagx="source" title="source">Trends Parasitol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">426</span><span class="lpage" tagx="lpage" title="lpage">434</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.pt.2015.06.002">10.1016/j.pt.2015.06.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26166750">26166750</a></span></span></li> <li tag="ref"><a name="B11-viruses-12-01055" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corelli</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="source" tagx="source" title="source">The Curse of Osiris</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Daily News</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London, UK</span><span class="year" tagx="year" title="year">1923</span><span class="fpage" tagx="fpage" title="fpage">1</span></span></li> <li tag="ref"><a name="B12-viruses-12-01055" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roche</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drake</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohani</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The curse of the Pharaoh revisited: Evolutionary bi-stability in environmentally transmitted pathogens</span><span class="source" tagx="source" title="source">Ecol. Lett.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">569</span><span class="lpage" tagx="lpage" title="lpage">575</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1461-0248.2011.01619.x">10.1111/j.1461-0248.2011.01619.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21496194">21496194</a></span></span></li> <li tag="ref"><a name="B13-viruses-12-01055" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walther</span><span class="given-names" tagx="given-names" title="given-names">B.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ewald</span><span class="given-names" tagx="given-names" title="given-names">P.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pathogen survival in the external environment and the evolution of virulence</span><span class="source" tagx="source" title="source">Biol. Rev.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">79</span><span class="fpage" tagx="fpage" title="fpage">849</span><span class="lpage" tagx="lpage" title="lpage">869</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S1464793104006475">10.1017/S1464793104006475</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15682873">15682873</a></span></span></li> <li tag="ref"><a name="B14-viruses-12-01055" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frank</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Models of parasite virulence</span><span class="source" tagx="source" title="source">Q. Rev. Biol.</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">37</span><span class="lpage" tagx="lpage" title="lpage">78</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/419267">10.1086/419267</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8919665">8919665</a></span></span></li> <li tag="ref"><a name="B15-viruses-12-01055" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Paepe</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taddei</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Virusesâ€™ life history: Towards a mechanistic basis of a trade-off between survival and reproduction among phages</span><span class="source" tagx="source" title="source">PLoS Biol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">1248</span><span class="lpage" tagx="lpage" title="lpage">1256</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.0040193">10.1371/journal.pbio.0040193</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16756387">16756387</a></span></span></li> <li tag="ref"><a name="B16-viruses-12-01055" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stearns</span><span class="given-names" tagx="given-names" title="given-names">S.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Trade-offs in life-history evolution</span><span class="source" tagx="source" title="source">Funct. Ecol.</span><span class="year" tagx="year" title="year">1989</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">259</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/2389364">10.2307/2389364</a></span></span></li> <li tag="ref"><a name="B17-viruses-12-01055" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamo</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boots</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The curse of the pharaoh in space: Free-living infectious stages and the evolution of virulence in spatially explicit populations</span><span class="source" tagx="source" title="source">J. Theor. Biol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">231</span><span class="fpage" tagx="fpage" title="fpage">435</span><span class="lpage" tagx="lpage" title="lpage">441</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2004.07.005">10.1016/j.jtbi.2004.07.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15501473">15501473</a></span></span></li> <li tag="ref"><a name="B18-viruses-12-01055" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Virulence evolution via host exploitation and toxin production in spore-producing pathogens</span><span class="source" tagx="source" title="source">Ecol. Lett.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">471</span><span class="lpage" tagx="lpage" title="lpage">476</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1461-0248.2002.00342.x">10.1046/j.1461-0248.2002.00342.x</a></span></span></li> <li tag="ref"><a name="B19-viruses-12-01055" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caraco</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">I.-N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Free-living pathogens: Life-history constraints and strain competition</span><span class="source" tagx="source" title="source">J. Theor. Biol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">250</span><span class="fpage" tagx="fpage" title="fpage">569</span><span class="lpage" tagx="lpage" title="lpage">579</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2007.10.029">10.1016/j.jtbi.2007.10.029</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18062992">18062992</a></span></span></li> <li tag="ref"><a name="B20-viruses-12-01055" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barker</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevens</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloomfield</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Spread and prevention of some common viral infections in community facilities and domestic homes</span><span class="source" tagx="source" title="source">J. Appl. Microbiol.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">7</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-2672.2001.01364.x">10.1046/j.1365-2672.2001.01364.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11442709">11442709</a></span></span></li> <li tag="ref"><a name="B21-viruses-12-01055" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boone</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerba</span><span class="given-names" tagx="given-names" title="given-names">C.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Significance of fomites in the spread of respiratory and enteric viral disease</span><span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">73</span><span class="fpage" tagx="fpage" title="fpage">1687</span><span class="lpage" tagx="lpage" title="lpage">1696</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.02051-06">10.1128/AEM.02051-06</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17220247">17220247</a></span></span></li> <li tag="ref"><a name="B22-viruses-12-01055" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogbunugafor</span><span class="given-names" tagx="given-names" title="given-names">C.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller-Dickson</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meszaros</span><span class="given-names" tagx="given-names" title="given-names">V.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomez</span><span class="given-names" tagx="given-names" title="given-names">L.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murillo</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scarpino</span><span class="given-names" tagx="given-names" title="given-names">S.V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The intensity of COVID-19 outbreaks is modulated by SARS-CoV-2 free-living survival and environmental transmission</span><span class="source" tagx="source" title="source">medRxiv</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.05.04.20090092">10.1101/2020.05.04.20090092</a></span></span></li> <li tag="ref"><a name="B23-viruses-12-01055" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller-Dickson</span><span class="given-names" tagx="given-names" title="given-names">M.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meszaros</span><span class="given-names" tagx="given-names" title="given-names">V.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almagro-Moreno</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogbunugafor</span><span class="given-names" tagx="given-names" title="given-names">C.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis C virus modelled as an indirectly transmitted infection highlights the centrality of injection drug equipment in disease dynamics</span><span class="source" tagx="source" title="source">J. R. Soc. Interface</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">16</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">20190334</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsif.2019.0334">10.1098/rsif.2019.0334</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31480919">31480919</a></span></span></li> <li tag="ref"><a name="B24-viruses-12-01055" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meszaros</span><span class="given-names" tagx="given-names" title="given-names">V.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller-Dickson</span><span class="given-names" tagx="given-names" title="given-names">M.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baffour-Awuah</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almagro-Moreno</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogbunugafor</span><span class="given-names" tagx="given-names" title="given-names">C.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Direct transmission via households informs models of disease and intervention dynamics in cholera</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">15</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0229837</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0229837">10.1371/journal.pone.0229837</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32163436">32163436</a></span></span></li> <li tag="ref"><a name="B25-viruses-12-01055" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bai</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">D.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Presumed asymptomatic carrier transmission of COVID-19</span><span class="source" tagx="source" title="source">JAMA</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">323</span><span class="fpage" tagx="fpage" title="fpage">1406</span><span class="pub-id"><a href="https://dx.doi.org/10.1001/jama.2020.2565">10.1001/jama.2020.2565</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32083643">32083643</a></span></span></li> <li tag="ref"><a name="B26-viruses-12-01055" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">S.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cornforth</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dushoff</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weitz</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The time scale of asymptomatic transmission affects estimates of epidemic potential in the COVID-19 outbreak</span><span class="source" tagx="source" title="source">Epidemics</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">31</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">100392</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.epidem.2020.100392">10.1016/j.epidem.2020.100392</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32446187">32446187</a></span></span></li> <li tag="ref"><a name="B27-viruses-12-01055" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huo</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">High incidence of asymptomatic SARS-CoV-2 infection, Chongqing, China</span><span class="source" tagx="source" title="source">SSRN Electron. J.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.2139/ssrn.3559583">10.2139/ssrn.3559583</a></span></span></li> <li tag="ref"><a name="B28-viruses-12-01055" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santarpia</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rivera</span><span class="given-names" tagx="given-names" title="given-names">D.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrera</span><span class="given-names" tagx="given-names" title="given-names">V.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morwitzer</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Creager</span><span class="given-names" tagx="given-names" title="given-names">H.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santarpia</span><span class="given-names" tagx="given-names" title="given-names">G.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crown</span><span class="given-names" tagx="given-names" title="given-names">K.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brett-Major</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnaubelt</span><span class="given-names" tagx="given-names" title="given-names">E.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broadhurst</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Aerosol and surface contamination of SARS-CoV-2 observed in quarantine and isolation care</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">8</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-020-69286-3">10.1038/s41598-020-69286-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31913322">31913322</a></span></span></li> <li tag="ref"><a name="B29-viruses-12-01055" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eisenberg</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robertson</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tien</span><span class="given-names" tagx="given-names" title="given-names">J.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identifiability and estimation of multiple transmission pathways in cholera and waterborne disease</span><span class="source" tagx="source" title="source">J. Theor. Biol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">324</span><span class="fpage" tagx="fpage" title="fpage">84</span><span class="lpage" tagx="lpage" title="lpage">102</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jtbi.2012.12.021">10.1016/j.jtbi.2012.12.021</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23333764">23333764</a></span></span></li> <li tag="ref"><a name="B30-viruses-12-01055" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cortez</span><span class="given-names" tagx="given-names" title="given-names">M.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weitz</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Distinguishing between indirect and direct modes of transmission using epidemiological time series</span><span class="source" tagx="source" title="source">Am. Nat.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">181</span><span class="fpage" tagx="fpage" title="fpage">E43</span><span class="lpage" tagx="lpage" title="lpage">E52</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/668826">10.1086/668826</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23348785">23348785</a></span></span></li> <li tag="ref"><a name="B31-viruses-12-01055" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roosa</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chowell</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Assessing parameter identifiability in compartmental dynamic models using a computational approach: Application to infectious disease transmission models</span><span class="source" tagx="source" title="source">Theor. Biol. Med Model.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12976-018-0097-6">10.1186/s12976-018-0097-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30642334">30642334</a></span></span></li> <li tag="ref"><a name="B32-viruses-12-01055" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Virtanen</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gommers</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliphant</span><span class="given-names" tagx="given-names" title="given-names">T.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haberland</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reddy</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cournapeau</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burovsk</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petersom</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wekesser</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bright</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">SciPy 1.0: Fundamental algorithms for scientific computing in Python</span><span class="source" tagx="source" title="source">Nat. Methods</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">261</span><span class="lpage" tagx="lpage" title="lpage">272</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41592-019-0686-2">10.1038/s41592-019-0686-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32015543">32015543</a></span></span></li> <li tag="ref"><a name="B33-viruses-12-01055" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duffy</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">P.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burch</span><span class="given-names" tagx="given-names" title="given-names">C.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pleiotropic costs of niche expansion in the RNA bacteriophage Î¦ 6</span><span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">172</span><span class="fpage" tagx="fpage" title="fpage">751</span><span class="lpage" tagx="lpage" title="lpage">757</span><span class="pub-id"><a href="https://dx.doi.org/10.1534/genetics.105.051136">10.1534/genetics.105.051136</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16299384">16299384</a></span></span></li> <li tag="ref"><a name="B34-viruses-12-01055" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">J.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Notes on R0</span><span class="source" tagx="source" title="source">Calif. Dep. Anthropol. Sci.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">323</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">19</span></span></li> <li tag="ref"><a name="B35-viruses-12-01055" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diekmann</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heesterbeek</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roberts</span><span class="given-names" tagx="given-names" title="given-names">M.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The construction of next-generation matrices for compartmental epidemic models</span><span class="source" tagx="source" title="source">J. R. Soc. Interface</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">873</span><span class="lpage" tagx="lpage" title="lpage">885</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsif.2009.0386">10.1098/rsif.2009.0386</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19892718">19892718</a></span></span></li> <li tag="ref"><a name="B36-viruses-12-01055" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scarpino</span><span class="given-names" tagx="given-names" title="given-names">S.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petri</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">On the predictability of infectious disease outbreaks</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">898</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-019-08616-0">10.1038/s41467-019-08616-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30796206">30796206</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Compartmental diagram of the SEAIR-W (susceptible-exposed-asymptomatic-infected-recovered) version of a-WAIT (Waterborne Abiotic or other Indirect Transmission)) model: this is based on a previously developed mathematical model used to interrogate environmental transmission of SARS-CoV-2 (see [<a href="#B22-viruses-12-01055">22</a>]).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Tornado plot showing the sensitivity of epidemic properties to individual parameter changes: (<b>A</b>) the number of infected individuals (asymptomatic and symptomatic) at the epidemic peak; (<b>B</b>) the rate at which the epidemic peak is reached, t<sub>max</sub><sup>âˆ’1</sup>; (<b>C</b>) the total infected population after 30 days; and (<b>D</b>) the basic reproductive ratio (<i>R</i><sub>0</sub>). Filled bars indicate the value of the epidemic feature when the associated parameter is increased by 5.0% from its nominal value. White bars indicate the value of a feature when the associated parameter is decreased by 5.0%. Blue coloring with checkered patterning indicates a parameter associated with survival, and orange coloring with lined patterning indicates a parameter associated with virulence.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Sample dynamics for the model system: (<b>A</b>) the dynamics for all host compartments within the model and (<b>B</b>) the fraction of environmental reservoirs in a setting that are contaminated with infectious virus.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Heatmap describing the impact of varying virus virulence and survival trait values assessed across four key epidemic metrics: these heatmaps express the change in (<b>A</b>) the number of infected individuals (asymptomatic and symptomatic) at the epidemic peak; (<b>B</b>) the rate at which the epidemic peak is reached, t<sub>max</sub><sup>âˆ’1</sup>; (<b>C</b>) the total infected population after 30 days; and (<b>D</b>) the basic reproductive ratio (<i>R</i><sub>0</sub>) when virulence and survival are modulated by Â±5% within the model. Contour lines are available for clarity.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>The expected effect of increasing survival on virulence for the two correlation models considered: here, we present a schematic of how the different hypotheses for the relationship between virulence and survival manifest on a map with a structure similar to the heat maps shown in <a href="#viruses-12-01055-f004">Figure 4</a>. The directions of the arrows depict how increasing survival would affect virulence under the two hypotheses: the blue arrow indicates the flow of an increasing positive correlation dynamic, while the direction of the orange arrow indicates an increasing negative correlation dynamic.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>The percent change in SEAIR-W outbreak metrics as survival increases from âˆ’5% to +5% for different virulenceâ€"survival relationships (positive correlation and negative correlation). For each metric analyzed, we present the percent difference between the minimum and maximum survival values given the two hypotheses tested: (i) positive correlation between survival and virulence (comparing low virulence/low survival to high virulence/high survival) and (ii) negative correlation (high virulence/low survival to high survival/low virulence). The bars here correspond to the values (percent) in the third columns of <a href="#viruses-12-01055-t005">Table 5</a> and <a href="#viruses-12-01055-t006">Table 6</a>, which denote the differences between the minimum and maximum values.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>Virus outbreak dynamics for the extreme values of virulence and free-living survival considered for the two different relationships (positive or negative) between virulence and survival traits: these plots are similar to the illustrative dynamics in <a href="#viruses-12-01055-f001">Figure 1</a>. Here, we observe the dynamics of disease corresponding to the extreme values presented in <a href="#viruses-12-01055-t005">Table 5</a> and <a href="#viruses-12-01055-t006">Table 6</a>. Subfigures (<b>A</b>,<b>C</b>,<b>E</b>,<b>G</b>) depict disease dynamics, and (<b>B</b>,<b>D</b>,<b>F</b>,<b>H</b>) depict the dynamics of contaminated environments. (<b>A</b>â€"<b>D</b>) correspond to the parameter values considered for the positive correlation scenario, while (<b>E</b>â€"<b>H</b>) correspond to the negative correlation scenario.</p>   </div> <div class="table-wrap_UNKNOWN" id="viruses-12-01055-t001" orientation="portrait" position="float">
viruses-12-01055-t001_Table 1<span class="label" tagx="label" title="label">Table 1</span> <p>Definitions and initial values for the populations represented by each compartment: the values here are the averages of the model values across all countries developed in a prior COVID-19 study [<a href="#B22-viruses-12-01055">22</a>].</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Symbols</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Value</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Units</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Definitions</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">S<sub>0</sub></td> <td align="center" valign="middle" rowspan="1" colspan="1">57.05 Ã— 10<sup>6</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">people</td> <td align="center" valign="middle" rowspan="1" colspan="1">Susceptible individuals</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">E<sub>0</sub></td> <td align="center" valign="middle" rowspan="1" colspan="1">66.50</td> <td align="center" valign="middle" rowspan="1" colspan="1">people</td> <td align="center" valign="middle" rowspan="1" colspan="1">Exposed individuals</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">A<sub>0</sub></td> <td align="center" valign="middle" rowspan="1" colspan="1">13.30</td> <td align="center" valign="middle" rowspan="1" colspan="1">people</td> <td align="center" valign="middle" rowspan="1" colspan="1">Asymptomatic individuals</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">I<sub>0</sub></td> <td align="center" valign="middle" rowspan="1" colspan="1">13.30</td> <td align="center" valign="middle" rowspan="1" colspan="1">people</td> <td align="center" valign="middle" rowspan="1" colspan="1">Symptomatic individuals</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">Rec<sub>0</sub></td> <td align="center" valign="middle" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" rowspan="1" colspan="1">people</td> <td align="center" valign="middle" rowspan="1" colspan="1">Recovered individuals</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">W<sub>0</sub></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1%</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">unitless</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">% of viral pathogen in environment</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="viruses-12-01055-t002" orientation="portrait" position="float">
viruses-12-01055-t002_Table 2<span class="label" tagx="label" title="label">Table 2</span> <p>Definitions for the nominal parameter values used in this study: parameter values were developed from empirical findings and country-level data, as discussed in another study [<a href="#B22-viruses-12-01055">22</a>].</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Symbols</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Values</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Units</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Definitions</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">ð�œ‡</td> <td align="center" valign="middle" rowspan="1" colspan="1">1/(80.3 Ã— 365)</td> <td align="center" valign="middle" rowspan="1" colspan="1">1/day</td> <td align="center" valign="middle" rowspan="1" colspan="1">Natural death rate (reciprocal of the upper bound of average human lifespan)</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">ð�œ‡<sub>I</sub></td> <td align="center" valign="middle" rowspan="1" colspan="1">0.00159</td> <td align="center" valign="middle" rowspan="1" colspan="1">1/day</td> <td align="center" valign="middle" rowspan="1" colspan="1">Infected death rate (natural death rate + disease-induced death rate)</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">ð�œ‚ = (<i>â�µ</i> âˆ’ <i>Îµ<sup>âˆ’</sup></i><sup>1</sup><i>)</i></td> <td align="center" valign="middle" rowspan="1" colspan="1">5.5</td> <td align="center" valign="middle" rowspan="1" colspan="1">days</td> <td align="center" valign="middle" rowspan="1" colspan="1">SARS-CoV-2 incubation period</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">1/<i>â�µ</i></td> <td align="center" valign="middle" rowspan="1" colspan="1">ð�œ‚ âˆ’ <i>Îµ</i><sup>âˆ’1</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">days</td> <td align="center" valign="middle" rowspan="1" colspan="1">Expected time in the asymptomatic state</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>Î½</i></td> <td align="center" valign="middle" rowspan="1" colspan="1">0.0305</td> <td align="center" valign="middle" rowspan="1" colspan="1">1/day</td> <td align="center" valign="middle" rowspan="1" colspan="1">Recovery rate (average of 3 to 6 weeks)</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>p</i></td> <td align="center" valign="middle" rowspan="1" colspan="1">95.6%</td> <td align="center" valign="middle" rowspan="1" colspan="1">percent</td> <td align="center" valign="middle" rowspan="1" colspan="1">Percent that moves along the â€œmildâ€� recovery track</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>k</i></td> <td align="center" valign="middle" rowspan="1" colspan="1">0.649</td> <td align="center" valign="middle" rowspan="1" colspan="1">1/day</td> <td align="center" valign="middle" rowspan="1" colspan="1">Waning virus rate in the environment (using the average of all material values, wood, steal, cardboard, and plastic)</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>Î²<sub>a</sub></i></td> <td align="center" valign="middle" rowspan="1" colspan="1">0.550</td> <td align="center" valign="middle" rowspan="1" colspan="1">1/day</td> <td align="center" valign="middle" rowspan="1" colspan="1">Contact rate of people with people Ã— transmission probability of people to people by A-person</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>Î²<sub>I</sub></i></td> <td align="center" valign="middle" rowspan="1" colspan="1">0.491</td> <td align="center" valign="middle" rowspan="1" colspan="1">1/day</td> <td align="center" valign="middle" rowspan="1" colspan="1">Contact rate of people with people Ã— transmission probability of people to people by I-person</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>Î²<sub>W</sub></i></td> <td align="center" valign="middle" rowspan="1" colspan="1">0.031</td> <td align="center" valign="middle" rowspan="1" colspan="1">1/day</td> <td align="center" valign="middle" rowspan="1" colspan="1">Contact rate of person with environment Ã— transmission probability of environment to people</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>ð�œŽ<sub>a</sub></i></td> <td align="center" valign="middle" rowspan="1" colspan="1">3.404</td> <td align="center" valign="middle" rowspan="1" colspan="1">1/day</td> <td align="center" valign="middle" rowspan="1" colspan="1">Contact rate of person with environment Ã— probability of shedding by A-people to environment</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>ð�œŽ<sub>I</sub></i></td> <td align="center" valign="middle" rowspan="1" colspan="1">13.492</td> <td align="center" valign="middle" rowspan="1" colspan="1">1/day</td> <td align="center" valign="middle" rowspan="1" colspan="1">Contact rate of person with environment) Ã— probability of shedding by I-people to environment</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><i>1/Îµ</i></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2.478</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">days</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Average number of days before infectiousness</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="viruses-12-01055-t003" orientation="portrait" position="float">
viruses-12-01055-t003_Table 3<span class="label" tagx="label" title="label">Table 3</span> <p>Virulence parameters: this is a list of uniformly modulated parameters and the direction in which they change when virulence is increased or decreased. When virulence changes, an up arrow (â†‘) indicates the parameter is increased (by an equivalent percent) and a down arrow (â†") indicates the parameter is decreased (by an equivalent percent).</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Symbols</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Definition</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Virulence Increased</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Virulence Decreased</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">ð�œ‡<sub>I</sub></td> <td align="center" valign="middle" rowspan="1" colspan="1">Infected death rate (natural death rate + disease induced death rate)</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†‘</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†"</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">ð�œ‚ = (<i>â�µ</i> âˆ’ <i>Îµ</i><sup>âˆ’1</sup><i>)</i></td> <td align="center" valign="middle" rowspan="1" colspan="1">SARS-CoV-2 incubation period</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†"</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†‘</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1">1/<i>â�µ</i></td> <td align="center" valign="middle" rowspan="1" colspan="1">Expected time in the asymptomatic state</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†‘</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†"</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>Î½</i></td> <td align="center" valign="middle" rowspan="1" colspan="1">Recovery rate (average of 3 to 6 weeks)</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†"</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†‘</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>p</i></td> <td align="center" valign="middle" rowspan="1" colspan="1">Percent that moves along the â€œmildâ€� recovery track</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†"</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†‘</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>Î²<sub>A</sub></i></td> <td align="center" valign="middle" rowspan="1" colspan="1">Contact rate of people with people Ã— transmission probability of people to people by A-person</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†‘</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†"</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>Î²<sub>I</sub></i></td> <td align="center" valign="middle" rowspan="1" colspan="1">Contact rate of people with people Ã— transmission probability of people to people by I-person</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†‘</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†"</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>ð�œŽ<sub>A</sub></i></td> <td align="center" valign="middle" rowspan="1" colspan="1">Contact rate of person with environment) Ã— (probability of shedding by A-people to environment</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†‘</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†"</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>ð�œŽ<sub>I</sub></i></td> <td align="center" valign="middle" rowspan="1" colspan="1">Contact rate of person with environment Ã— probability of shedding by I-people to environment</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†‘</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†"</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><i>1/Îµ</i></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Average number of days before infectious</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">â†"</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">â†‘</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="viruses-12-01055-t004" orientation="portrait" position="float">
viruses-12-01055-t004_Table 4<span class="label" tagx="label" title="label">Table 4</span> <p>Survival parameters: this is a list of parameters that are uniformly modulated and the direction in which they change when survival is increased or decreased. An up arrow (â†‘) indicates the parameter is increased by some percent when the equivalent (percent) change in survival is applied. A down arrow (â†") indicates the parameter is decreased by some percent when the equivalent (percent) change in survival is applied.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Symbols</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Definition</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Survival Increased</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Survival Decreased</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><i>k</i></td> <td align="center" valign="middle" rowspan="1" colspan="1">Waning virus rate in environment (using the average of all material values, wood, steal, cardboard, and plastic)</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†"</td> <td align="center" valign="middle" rowspan="1" colspan="1">â†‘</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><i>Î²<sub>W</sub></i></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Contact rate of person with environment Ã— transmission probability of environment to people</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">â†‘</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">â†"</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="viruses-12-01055-t005" orientation="portrait" position="float">
viruses-12-01055-t005_Table 5<span class="label" tagx="label" title="label">Table 5</span> <p>Positive correlation scenario: comparing epidemic metrics under low survival/low virulence versus high survival/high virulence scenarios (as in the positive correlation scenario). For each metric analyzed, these are the heatmap values for the bottom left (at â€œcoordinatesâ€� (Vir, Sur) â†’ (âˆ’5%, âˆ’5%)) and top right (at â€œcoordinatesâ€� (Vir, Sur) â†’ (+5%, +5%)) corners.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Epidemic Metric</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">min Virulence, min Survival</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">max Virulence, max Survival</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">% Difference between min Survival and max Survival </th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Peak total infected (people)</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">5.68 Ã— 10<sup>6</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">7.64 Ã— 10<sup>6</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">+34.51%</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>t<sub>max</sub><sup>âˆ’1</sup> (days<sup>âˆ’1</sup>)</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.99 Ã— 10<sup>âˆ’2</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">2.23 Ã— 10<sup>âˆ’2</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">+12.06%</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Total after 30 days (people)</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">8.18 Ã— 10<sup>7</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.62 Ã— 10<sup>8</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">+98.04%</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>Basic reproductive ratio (<i>R</i><sub>0</sub>)</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1.95</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">3.78</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">+93.84%</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="viruses-12-01055-t006" orientation="portrait" position="float">
viruses-12-01055-t006_Table 6<span class="label" tagx="label" title="label">Table 6</span> <p>Negative correlation scenario: comparing epidemic metrics under low survival/low virulence versus high survival/high virulence scenarios (as in the negative correlation scenario). For each metric analyzed, these are the global heatmap values for the top left (at â€œcoordinatesâ€� (Vir, Sur) â†’ (+5%, âˆ’5%)) and bottom right corners (at â€œcoordinatesâ€� (Vir, Sur) â†’ (âˆ’5%, +5%)).</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Epidemic Metric</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">max Virulence, min Survival</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">min Virulence, max Survival</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">% Difference between min Survival and max Survival </th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Peak total infected (people)</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">7.39 Ã— 10<sup>6</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">5.99 Ã— 10<sup>6</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’23.47%</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>t<sub>max</sub><sup>âˆ’1</sup> (days<sup>âˆ’1</sup>)</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">2.10 Ã— 10<sup>âˆ’2</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">2.23 Ã— 10<sup>âˆ’2</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’0.15%</td> </tr> <tr> <td align="center" valign="middle" rowspan="1" colspan="1"><b>Total infected after 30 days (people)</b></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.16 Ã— 10<sup>8</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">1.13 Ã— 10<sup>8</sup></td> <td align="center" valign="middle" rowspan="1" colspan="1">âˆ’2.68%</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>Basic reproductive ratio (<i>R</i><sub>0</sub>)</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">3.67</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1.99</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">âˆ’84.39%</td> </tr> </tbody> </table> </div> </div>  </body></html>