<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Nucleic Acids Res</span><span class="iso-abbrev" title="iso-abbrev">Nucleic Acids Res</span><span class="publisher-id" title="publisher-id">nar</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Nucleic Acids Research</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0305-1048</span><span class="issn-epub" tagx="issn" title="issn-epub">1362-4962</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/nar/gkaa413">10.1093/nar/gkaa413</a></span><span class="publisher-id" title="publisher-id">gkaa413</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">AcademicSubjects/SCI00010</span>: <span class="subject" title="subject">Web Server Issue</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
NanoSPC: a scalable, portable, cloud compatible viral nanopore metagenomic data processing pipeline</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-1969-2484</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Yifei</span></span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">Nuffield Department of Medicine, University of Oxford</span><span class="country" tagx="country" title="country">UK</span></span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">NIHR Oxford Biomedical Research Centre, University of Oxford</span><span class="country" tagx="country" title="country">UK</span></span><a href="#FN1" /><a href="#COR1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang-Turner</span><span class="given-names" tagx="given-names" title="given-names">Fan</span></span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">Nuffield Department of Medicine, University of Oxford</span><span class="country" tagx="country" title="country">UK</span></span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">NIHR Oxford Biomedical Research Centre, University of Oxford</span><span class="country" tagx="country" title="country">UK</span></span><a href="#FN1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volk</span><span class="given-names" tagx="given-names" title="given-names">Denis</span></span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">Nuffield Department of Medicine, University of Oxford</span><span class="country" tagx="country" title="country">UK</span></span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">NIHR Oxford Biomedical Research Centre, University of Oxford</span><span class="country" tagx="country" title="country">UK</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crook</span><span class="given-names" tagx="given-names" title="given-names">Derrick</span></span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">Nuffield Department of Medicine, University of Oxford</span><span class="country" tagx="country" title="country">UK</span></span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">NIHR Oxford Biomedical Research Centre, University of Oxford</span><span class="country" tagx="country" title="country">UK</span></span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
To whom correspondence should be addressed. Email: <span class="email" tagx="email" title="email">yifei.xu@ndm.ox.ac.uk</span></div> <div class="fn-type-" title=""> <p>The authors wish it to be known that, in their opinion, the first two authors should be regarded as Joint First Authors.</p> </div> </div><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020-7-7</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-5-5</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-5-5</span></span><span class="volume" tagx="volume" title="volume">48</span><span class="issue" tagx="issue" title="issue">W1</span><span class="fpage" tagx="fpage" title="fpage">W366</span><span class="lpage" tagx="lpage" title="lpage">W371</span><span class="history" title="history"><span class="accepted" title="accepted">accepted: 2020-5-11</span><span class="rev-recd" title="rev-recd">rev-recd: 2020-4-22</span><span class="received" title="received">received: 2020-3-05</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="gkaa413.pdf" title="self-uri">gkaa413.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Metagenomic sequencing combined with Oxford Nanopore Technology has the potential to become a point-of-care test for infectious disease in public health and clinical settings, providing rapid diagnosis of infection, guiding individual patient management and treatment strategies, and informing infection prevention and control practices. However, publicly available, streamlined, and reproducible pipelines for analyzing Nanopore metagenomic sequencing data are still lacking. Here we introduce NanoSPC, a scalable, portable and cloud compatible pipeline for analyzing Nanopore sequencing data. NanoSPC can identify potentially pathogenic viruses and bacteria simultaneously to provide comprehensive characterization of individual samples. The pipeline can also detect single nucleotide variants and assemble high quality complete consensus genome sequences, permitting high-resolution inference of transmission. We implement NanoSPC using Nextflow manager within Docker images to allow reproducibility and portability of the analysis. Moreover, we deploy NanoSPC to our scalable pathogen pipeline platform, enabling elastic computing for high throughput Nanopore data on HPC cluster as well as multiple cloud platforms, such as Google Cloud, Amazon Elastic Computing Cloud, Microsoft Azure and OpenStack. Users could either access our web interface (<a href="https://nanospc.mmmoxford.uk">https://nanospc.mmmoxford.uk</a>) to run cloud-based analysis, monitor process, and visualize results, as well as download Docker images and run command line to analyse data locally.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">NIHR Oxford Biomedical Research Centre</span><span class="institution-id" title="institution-id">10.13039/501100013373</span></div> </div> </div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
INTRODUCTION</div> <p>Oxford Nanopore Technology (ONT) is a third generation sequencing technology with the advantage of generating real-time, long read data with highly portable devices. Nanopore sequencing has been successfully applied in a broad range of research areas, including human genetics, cancer, microbiology, plant, and infectious diseases (<a href="#B1">1â€"5</a>). On-site sequencing with ONT has enabled real-time surveillance and tracking of Ebola, Zika and Lassa epidemics (<a href="#B6">6â€"8</a>).</p> <p>Metagenomic sequencing has the capacity to detect all potential pathogens from individual clinical samples, and provide genomic information for comprehensive characterization of the pathogens, microbiome analyses, and investigation of epidemiology and transmission (<a href="#B9">9â€"11</a>). Metagenomic sequencing with ONT has the potential to become a point-of-care test for infectious diseases in clinical and public health settings, providing rapid diagnosis of infection, guide individual patient management and treatment strategies, and informing infection prevention and control practices (<a href="#B12">12â€"14</a>). Nanopore metagenomic sequencing of the novel coronavirus disease 2019 (COVID-19) that causes the ongoing pandemic in the world has provided critical and timely evidence for human-to-human transmission of this virus (<a href="#B15">15</a>).</p> <p>Nanopore sequencing data is characterized by high error rates (approximately 10% with state of art chemistry and basecalling algorithms) compared to next generation sequencing (approximately 0.1%) (<a href="#B16">16</a>). A variety of bioinformatic tools have been developed to overcome such high error rates and improve data quality for each step associated with the analysis, including basecalling, reads mapping, variants calling, and genome assembly (<a href="#B17">17â€"21</a>). However, publicly available streamlined and reproducible pipelines for analyzing Nanopore metagenomic sequencing data are still lacking, which consequently could impede its application, especially for users with minimum bioinformatics knowledge.</p> <p>Here, we introduce NanoSPC, a scalable, portable and cloud compatible pipeline for analyzing Nanopore sequencing data. NanoSPC can identify potentially pathogenic viruses and bacteria simultaneously to provide comprehensive characterization of individual samples. The pipeline can also detect single nucleotide variants and assemble high quality complete consensus genome sequences, which permits inference of transmission with high resolution. We implement NanoSPC using Nextflow pipeline manager within Docker images to allow reproducibility and portability of the analysis. We deploy NanoSPC to our scalable pathogen pipeline platform, enabling elastic computing for high throughput data via HPC cluster as well as multiple commercial cloud platforms.</p> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
METHODS AND FUNCTIONALITIES</div> <div class="datainputandqualitycontrol" title="sec"> <div class="title" tagx="title" title="title">
Data input and quality control</div> <p>NanoSPC takes Nanopore sequencing reads of the fastq format and, optionally, raw signal data of the fast5 format as input (Figure <a href="#F1">1</a>). We have tested the pipeline with fastq reads produced by the R9.4 and R9.4.1 version flow cells, native and rapid barcoding kits. The acceptable multi-read fast5 format should contain data pertaining to multiple reads in each fast5 file. While NanoSPC is designed to analyze Nanopore sequencing data, some of its modules could be used to analyze sequencing data from other platforms, such as PacBio or Illumina. The pipeline investigates the quality of the sequencing reads using NanoPlot (<a href="#B22">22</a>). A comprehensive statistical summary is produced to report the overall data quality, including number of reads, total nucleotide bases, mean and median read length, and quality scores. In addition, a variety of informative graphs are generated to display multiple aspects of the data, such as cumulative yield plot showing efficiency of the flow cell against time, heat map of the physical layout of the flow cell comparing the efficiency of each channel, and violin plots showing base call quality against time.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>A schematic overview of NanoSPC. NanoSPC is a scalable, portable, and cloud compatible pipeline that can analyse the Nanopore metagenomic sequencing data. NanoSPC can identify potentially pathogenic viruses and bacteria simultaneously to provide comprehensive characterization of individual samples. The pipeline can also detect single nucleotide variants and assemble high quality complete consensus genome sequences. NanoSPC uses Nextflow pipeline manager and packs all the software dependencies within Docker images (red). NanoSPC can be deployed into the scalable pathogen pipeline platform, enabling elastic computing for high throughput Nanopore data on multiple cloud platforms (green). NanoSPC can be accessed via a web interface to run cloud-based analysis as well as Docker images to analyze data locally (blue).</p>   </div> </div> <div class="identificationofspecies" title="sec"> <div class="title" tagx="title" title="title">
Identification of species</div> <p>Metagenomic sequencing data are generally associated with a high level background. In order to identify potentially pathogenic viruses and bacteria with high sensitivity and specificity, we implement a method that combines taxonomic classification, reference based mapping, and filtering.</p> <p>NanoSPC first applies Centrifuge v1.0.3 (<a href="#B23">23</a>) to classify sequencing reads to a taxonomic identifier in the centrifuge reference database (p_compressed+h+v) that comprises 20 174 complete bacterial, archaeal, viral, and human genomes corresponding to 11 539 taxonomic IDs. One primary assignment with a score &amp;gt;150 is reported for individual reads. Based on the centrifuge report, the pipeline selects a draft reference genome for each viral species, and maps sequencing reads to the draft reference using Minimap2 (<a href="#B24">24</a>). In order to optimize the reference sequence for viral species, a preliminary consensus sequence for each viral species is generated using a simple majority voting method that selects the most abundant base at each genomic position. The preliminary consensus sequences are then BLASTed against a customized viral reference database to determine more optimal reference sequences. Finally, we map the reads to these references using Minimap2 for a second time. The customized viral reference database comprises of &amp;gt;86 000 complete genomes of viral pathogens downloaded from NIAID Virus Pathogen Database and Analysis Resource (ViPR) (<a href="#B25">25</a>) in March 2020. We aim to update this viral reference database every four months or when significant novel pathogen species are discovered.</p> <p>To distinguish species present in the data and artifacts, the pipeline implements the following filtering of the mapping results: (I) retain mapped reads with a mapping quality &amp;gt;50; (II) retain reads that have &amp;gt;80% of the bases mapped to the reference sequence (i.e. if the length of a read is 1000Â bp, &amp;gt;800Â bp are required to be mapped). Viral species with â‰¥2 mapped reads or one mapped read longer than 400Â bp are reported.</p> </div> <div class="genomeassemblyandvariantcalling" title="sec"> <div class="title" tagx="title" title="title">
Genome assembly and variant calling</div> <p>Whole genome sequencing can provide high-resolution investigation of transmission and characterization of the spatiotemporal spread of outbreaks. While low accuracy on the sequencing read level (âˆ¼90%) is a major limitation of nanopore sequencing, genome assembly can generate high-accuracy consensus sequence with adequate sequencing depth. The full genome consensus sequences generated by NanoSPC have been used to delineate nosocomial transmission of influenza A virus and human metapneumovirus, contributing to improvement of infection prevention and control practices (<a href="#B26">26</a>).</p> <p>NanoSPC enables three variant calling modes, namely nanopolish, medaka and clair. With the nanopolish mode, NanoSPC takes both sequencing reads of the fastq format and raw signal data of the fast5 format as inputs, and applies Nanopolish (<a href="#B19">19</a>) to detect single nucleotide variants. The pipeline considers candidate variants from the aligned reads when the variant frequency is &amp;gt;10% and the mapping depth is &amp;gt;10. To distinguish true variants and artifacts, NanoSPC keeps variants that the number of signal reads used to call the variant are &amp;gt;10 and the fraction of signal reads that support the variant are &amp;gt;75%. Finally, reads are mapped against the consensus sequence and only positions that are supported by &amp;gt;70% of mapped reads are kept.</p> <p>While the nanopolish mode can generate consensus sequence with high accuracy, it is computationally intensive and time consuming, due to the large size of fast5 signal data. Therefore, signal based variant calling methods present a challenge to data storage, transfer, and analysis. Fortunately, with the Medaka and clair mode, NanoSPC applies Medaka (ONT) and clair (<a href="#B27">27</a>) to detect single nucleotide variants. The advantages of these two modes is that they only take fastq format sequencing reads, and can be competitive with the nanopolish mode and being much faster.</p> <p>The parameter values that we describe in the methods and functionalities section are selected based on publications (<a href="#B6">6</a>,<a href="#B28">28</a>) and our recent studies (<a href="#B26">26</a>,<a href="#B29">29</a>).</p> </div> <div class="influenzadrugresistanceanalysis" title="sec"> <div class="title" tagx="title" title="title">
Influenza drug resistance analysis</div> <p>If influenza viruses are identified from the sequencing data, the pipeline determines the type of influenza virus (A or B) and the HA and NA subtype of influenza A virus based on the reference sequences. The pipeline analyzes assembled consensus sequences of Neuraminidase and Matrix 2 genes to identify mutations that confer resistance to antiviral agents, including oseltamivir, zanamivir, and amantadine. Drug resistance mutations are listed in <a href="#sup1">Supplementary Table S1</a>.</p> </div> <div class="barcodedemultiplexing" title="sec"> <div class="title" tagx="title" title="title">
Barcode demultiplexing</div> <p>The continuous improved data yield from each ONT flow cell permits users to make more efficient use and reduce cost by multiplexing several samples in a single sequencing run. Our previous work has shown that barcode contamination reads could account for &amp;lt;1% of the total reads from a multiplexed sequencing run (<a href="#B30">30</a>), highlighting the need for careful barcode demultiplexing and the trade-off between sensitivity and specificity that applies to the demultiplexing methods. NanoSPC enables three demultiplexing modes, namely default-porechop, strict-porechop, and qcat. The default-porechop mode requires a barcode sequence to be present at either end of each read using porechop (<a href="https://github.com/rrwick/Porechop">https://github.com/rrwick/Porechop</a>), thus maximizing the number of classified reads for downstream analysis. The strict-porechop mode requires the same barcode sequence to be present at both ends of each read, minimizing the number of misclassified reads. The qcat mode employs the demultiplexer offered by ONT and works similar to the default-porechop mode, due to the fact that porechop has been deprecated.</p> </div> </div> <div class="implementation" title="sec"> <div class="title" tagx="title" title="title">
IMPLEMENTATION</div> <div class="scalableandelasticcomputing" title="sec"> <div class="title" tagx="title" title="title">
Scalable and elastic computing</div> <p>In order to perform reproducible and portable analysis of sequencing data, we implement NanoSPC using Nextflow pipeline manager (<a href="#B31">31</a>) that enables parallel processing of multiple datasets (Figure <a href="#F1">1</a>). Moreover, all the software dependencies of the pipeline are packed within Docker images that can be executed on a wide variety of computing infrastructures.</p> <p>Scalable Pathogon Pipeline Platform (SP<sup>3</sup>) is an open source web-based platform that we developed to host container-centric bioinformatic pipelines (<a href="#B32">32</a>). We deploy NanoSPC to the SP<sup>3</sup> platform to allow the pipeline to be run at high-performance computing (HPC) clusters and major commercial cloud platforms, such as Google Cloud, Amazon Elastic Computing Cloud (Amazon EC2), Microsoft Azure and OpenStack Cloud. SP<sup>3</sup> has a cloud operation layer that provides interface to cloud platforms. A cloud agnostic scaler and scheduler allocate compute nodes based on CPU and memory requirement of the submitted jobs, and deallocate computational resources after the completion of the jobs. All SP<sup>3</sup> software are deployed to a head node equipped with Ubuntu 18.04 and Nextflow. Compute nodes are only being built and used when tasks are sent to the head node. SP<sup>3</sup> platform manages pipelines via a series of Web APIs, providing a range of functionalities, such as fetching data from European Nucleotide Archive (ENA) and other data sources, and excuating data analysis.</p> </div> <div class="webinterfaceandstand-aloneapplication" title="sec"> <div class="title" tagx="title" title="title">
Web interface and stand-alone application</div> <p>We build a web interface for users to interact with cloud platforms via SP<sup>3</sup> and perform data analysis (Figure <a href="#F1">1</a>). An authorised user can login to the SP<sup>3</sup> system to upload their Nanopore sequencing data and execute the pipeline. Users can monitor the progress of the submitted job in real-time. The interface displays detailed logs, running commands, output files being generated, processing time, CPU usage, and memory usage for each individual process in the pipeline. Upon job completion, a complete run report displays the analysis results for each dataset. Users can choose to download the result files in bulk or only files that are of interest through running a command provided in the interface. The interface is served by the Nginx web server, configured to authenticate against web APIs, allowing user access control via web LDAP authentication. The web interface is written in python.</p> <p>For users wishing to use NanoSPC to analyze data locally, we build Docker images that wrap the entire pipeline into a single environment. Users can download the Docker images and run command lines to perform the analysis.</p> </div> <div class="availability" title="sec"> <div class="title" tagx="title" title="title">
Availability</div> <p>Details of web interface for cloud-based analysis and stand-alone application are available at <a href="https://nanospc.mmmoxford.uk">https://nanospc.mmmoxford.uk</a>.</p> </div> <div class="usage" title="sec"> <div class="title" tagx="title" title="title">
Usage</div> <p>We illustrate the usage of NanoSPC with an exemplar Nanopore metagenomic sequencing dataset. The dataset contains Nanopore metagenomic sequencing reads generated from a clinical respiratory sample. The sample has been tested positive for human metapneumovirus (hmpv) in the clinical diagnostic laboratory as described in (<a href="#B26">26</a>). Web interfaces for submitting the jobs to cloud-based analysis and real-time monitoring of the progress are shown in Figure <a href="#F2">2A</a> andÂ <a href="#F2">B</a>. Analysis of the data shows that Nanopore sequencing generated 428 914 reads with mean length of 615 bp (Figure <a href="#F2">2C</a>). Taxonomic classification of the sequencing reads are displayed using the Krona plots (Figure <a href="#F2">2D</a>). In this case, bacterial and viral reads accounted for 88% and 5% of the total sequencing reads. Among the viral reads, 67% and 33% are identified as hmpv and human parainfluenza virus type 3 (hpiv-3) reads, exemplifying simultaneous identification of multiple potentially pathogenic species from individual metagenomic sequencing dataset. Mapping to reference sequences showed that 14 821 hmpv reads cover the complete genome at mean coverage of 886, and 7323 hpiv-3 reads cover the complete genome at mean coverage of 370 (Figure <a href="#F2">2E</a>). The medaka mode is employed for calling variants, the total execution time is 13 mins (Figure <a href="#F2">2F</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Example showing cloud-based analysis of Nanopore metagenomic sequencing data via NanoSPC. (<b>A</b>) and (<b>B</b>) Web interfaces for executing data analysis and real-time monitoring of the progress. (<b>C</b>) Statistical summary of the data quality. (<b>D</b>) Taxonomic assignment of sequencing reads, percentage of bacterial and viral reads. (<b>E</b>) Genome coverage by mapping sequencing reads to reference sequences. (<b>F</b>) Execution time for each process in the pipeline.</p>   </div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
CONCLUSION</div> <p>We report NanoSPC, a scalable, portable, and cloud compatibilable pipeline for analyzing metagenomic sequencing data generated using ONT. NanoSPC differs from other pipelines, such as NanoPipe (<a href="#B33">33</a>), by analyzing metagenomic sequencing data and identifying a range of organisms without a priori knowledge of species contained in the data. The implementation of cloud computing enables NanoSPC to increase the ease and efficiency of analysis of high throughput Nanopore metagenomic data. The analysis results can potentially be used in multiple clinical and public health applications.</p> <p>We have applied NanoSPC to identify and assemble genomes of a range of pathogen species, including influenza viruses and human metapneumovirus, from Nanopore metagenomic sequencing data of clinical samples (<a href="#B29">29</a>). The generated sequences have been used to investigate drug resistance and genetic diversity of influenza viruses from a UK hospital during the 2018/19 influenza season. The sequences also provided high resolution characterization of nosocomial transmission of influenza A virus and human metapneumovirus, contributing to improvement of infection prevention and control practices (<a href="#B26">26</a>).</p> <p>Our future step is to include prediction of antiviral drug resistance for viral species, such as influenza A virus, and phylogenetic analysis in the pipeline. Other features including discovery of novel pathogen species, particularly viruses, are also under investigation. ONT is undergoing constant improvement in sequencing chemistry and library preparation methods. ONT has released a new sequencing chemistry (R10 version flow cell) that provides improved accuracy. Our pipeline should be able to process R10 data but validating tests have not been conducted due to data availability. We are committed to continuously update our pipeline with the development of bioinformatics tools to enhance the accuracy and efficiency of the data analysis.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">gkaa413_Supplemental_File</span><div class="media" title="media"><a href="gkaa413_supplemental_file.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="supplementarydata" title="sec"> <div class="title" tagx="title" title="title">
SUPPLEMENTARY DATA</div> <p><a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkaa413#supplementary-data">Supplementary Data</a> are available at NAR Online.</p> </div> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
FUNDING</div> <p>NIHR Oxford Biomedical Research Centre. Funding for open access charge:Â NIHR Oxford Biomedical Research Centre.</p> <p><i>Conflict of interest statement</i>. None declared.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="B1" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koren</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miga</span><span class="given-names" tagx="given-names" title="given-names">K.H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quick</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rand</span><span class="given-names" tagx="given-names" title="given-names">A.C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasani</span><span class="given-names" tagx="given-names" title="given-names">T.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tyson</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beggs</span><span class="given-names" tagx="given-names" title="given-names">A.D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dilthey</span><span class="given-names" tagx="given-names" title="given-names">A.T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fiddes</span><span class="given-names" tagx="given-names" title="given-names">I.T.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Nanopore sequencing and assembly of a human genome with ultra-long reads</span>. <span class="source" tagx="source" title="source">Nat. Biotechnol.</span><span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">36</span>:<span class="fpage" tagx="fpage" title="fpage">338</span>â€"<span class="lpage" tagx="lpage" title="lpage">345</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29431738">29431738</a></span></span></li> <li tag="ref"><a name="B2" /><span class="label" tagx="label" title="label">2.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakamoto</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sereewattanawoot</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A new era of long-read sequencing for cancer genomics</span>. <span class="source" tagx="source" title="source">J Hum Genet</span>. <span class="year" tagx="year" title="year">2020</span>; <span class="volume" tagx="volume" title="volume">65</span>:<span class="fpage" tagx="fpage" title="fpage">3</span>â€"<span class="lpage" tagx="lpage" title="lpage">10</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31474751">31474751</a></span></span></li> <li tag="ref"><a name="B3" /><span class="label" tagx="label" title="label">3.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wouter</span><span class="given-names" tagx="given-names" title="given-names">D.C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arne</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeÂ Pooter</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Svenn</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peter</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mojca</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sleegers</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christine</span><span class="given-names" tagx="given-names" title="given-names">V.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural variants identified by Oxford Nanopore PromethION sequencing of the human genome</span>. <span class="source" tagx="source" title="source">Genome Research</span>. <span class="year" tagx="year" title="year">(2019)</span>; <span class="volume" tagx="volume" title="volume">29</span>:<span class="fpage" tagx="fpage" title="fpage">1178</span>â€"<span class="lpage" tagx="lpage" title="lpage">1187</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31186302">31186302</a></span></span></li> <li tag="ref"><a name="B4" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olsen</span><span class="given-names" tagx="given-names" title="given-names">H.E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paten</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akeson</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Oxford Nanopore MinION: delivery of nanopore sequencing to the genomics community</span>. <span class="source" tagx="source" title="source">Genome Biol.</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">17</span>:<span class="fpage" tagx="fpage" title="fpage">239</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27887629">27887629</a></span></span></li> <li tag="ref"><a name="B5" /><span class="label" tagx="label" title="label">5.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belser</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Istace</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Denis</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dubarry</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baurens</span><span class="given-names" tagx="given-names" title="given-names">F.-C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falentin</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Genete</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berrabah</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ChÃ¨vre</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delourme</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Chromosome-scale assemblies of plant genomes using nanopore long reads and optical maps</span>. <span class="source" tagx="source" title="source">Na.t Plants</span>. <span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">879</span>â€"<span class="lpage" tagx="lpage" title="lpage">887</span>.</span></li> <li tag="ref"><a name="B6" /><span class="label" tagx="label" title="label">6.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kafetzopoulou</span><span class="given-names" tagx="given-names" title="given-names">L.E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pullan</span><span class="given-names" tagx="given-names" title="given-names">S.T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ehichioya</span><span class="given-names" tagx="given-names" title="given-names">D.U.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pahlmann</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thielebein</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hinzmann</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oestereich</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wozniak</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Metagenomic sequencing at the epicenter of the Nigeria 2018 Lassa fever outbreak</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2019</span>; <span class="volume" tagx="volume" title="volume">363</span>:<span class="fpage" tagx="fpage" title="fpage">74</span>â€"<span class="lpage" tagx="lpage" title="lpage">77</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30606844">30606844</a></span></span></li> <li tag="ref"><a name="B7" /><span class="label" tagx="label" title="label">7.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quick</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loman</span><span class="given-names" tagx="given-names" title="given-names">N.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duraffour</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simpson</span><span class="given-names" tagx="given-names" title="given-names">J.T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Severi</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cowley</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bore</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koundouno</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudas</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mikhail</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Real-time, portable genome sequencing for Ebola surveillance</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">530</span>:<span class="fpage" tagx="fpage" title="fpage">228</span>â€"<span class="lpage" tagx="lpage" title="lpage">232</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26840485">26840485</a></span></span></li> <li tag="ref"><a name="B8" /><span class="label" tagx="label" title="label">8.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quick</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grubaugh</span><span class="given-names" tagx="given-names" title="given-names">N.D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pullan</span><span class="given-names" tagx="given-names" title="given-names">S.T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Claro</span><span class="given-names" tagx="given-names" title="given-names">I.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">A.D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gangavarapu</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliveira</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robles-Sikisaka</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rogers</span><span class="given-names" tagx="given-names" title="given-names">T.F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beutler</span><span class="given-names" tagx="given-names" title="given-names">N.A.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Multiplex PCR method for MinION and Illumina sequencing of Zika and other virus genomes directly from clinical samples</span>. <span class="source" tagx="source" title="source">Nat. Protoc.</span><span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">12</span>:<span class="fpage" tagx="fpage" title="fpage">1261</span>â€"<span class="lpage" tagx="lpage" title="lpage">1276</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28538739">28538739</a></span></span></li> <li tag="ref"><a name="B9" /><span class="label" tagx="label" title="label">9.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiu</span><span class="given-names" tagx="given-names" title="given-names">C.Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Clinical metagenomics</span>. <span class="source" tagx="source" title="source">Nat. Rev. Genet.</span><span class="year" tagx="year" title="year">2019</span>; <span class="volume" tagx="volume" title="volume">20</span>:<span class="fpage" tagx="fpage" title="fpage">341</span>â€"<span class="lpage" tagx="lpage" title="lpage">355</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30918369">30918369</a></span></span></li> <li tag="ref"><a name="B10" /><span class="label" tagx="label" title="label">10.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlaberg</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiu</span><span class="given-names" tagx="given-names" title="given-names">C.Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Procop</span><span class="given-names" tagx="given-names" title="given-names">G.W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weinstock</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="collab'">collab: Professional Practice Committee and Committee on Laboratory Practices of the American Society for Microbiology</span><span class="collab'">collab: Microbiology Resource Committee of the College of American Pathologists</span></span><span class="mixed-article-title" title="mixed-article-title">Validation of metagenomic next-generation sequencing tests for universal pathogen detection</span>. <span class="source" tagx="source" title="source">Arch. Pathol. Lab. Med.</span><span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">141</span>:<span class="fpage" tagx="fpage" title="fpage">776</span>â€"<span class="lpage" tagx="lpage" title="lpage">786</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28169558">28169558</a></span></span></li> <li tag="ref"><a name="B11" /><span class="label" tagx="label" title="label">11.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">M.R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naccache</span><span class="given-names" tagx="given-names" title="given-names">S.N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samayoa</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biagtan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bashir</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salamat</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somasekar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Federman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Actionable diagnosis of neuroleptospirosis by next-generation sequencing</span>. <span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">370</span>:<span class="fpage" tagx="fpage" title="fpage">2408</span>â€"<span class="lpage" tagx="lpage" title="lpage">2417</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24896819">24896819</a></span></span></li> <li tag="ref"><a name="B12" /><span class="label" tagx="label" title="label">12.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charalampous</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kay</span><span class="given-names" tagx="given-names" title="given-names">G.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aydin</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldan</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeanes</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rae</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grundy</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wain</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Nanopore metagenomics enables rapid clinical diagnosis of bacterial lower respiratory infection</span>. <span class="source" tagx="source" title="source">Nat. Biotechnol.</span><span class="year" tagx="year" title="year">2019</span>; <span class="volume" tagx="volume" title="volume">37</span>:<span class="fpage" tagx="fpage" title="fpage">783</span>â€"<span class="lpage" tagx="lpage" title="lpage">792</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31235920">31235920</a></span></span></li> <li tag="ref"><a name="B13" /><span class="label" tagx="label" title="label">13.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greninger</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naccache</span><span class="given-names" tagx="given-names" title="given-names">S.N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Federman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mbala</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bres</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stryke</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouquet</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somasekar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linnen</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Rapid metagenomic identification of viral pathogens in clinical samples by real-time nanopore sequencing analysis</span>. <span class="source" tagx="source" title="source">Genome Med.</span><span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">99</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26416663">26416663</a></span></span></li> <li tag="ref"><a name="B14" /><span class="label" tagx="label" title="label">14.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kafetzopoulou</span><span class="given-names" tagx="given-names" title="given-names">L.E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Efthymiadis</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewandowski</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crook</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carter</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osborne</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aarons</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hewson</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hiscox</span><span class="given-names" tagx="given-names" title="given-names">JA.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carroll</span><span class="given-names" tagx="given-names" title="given-names">MW.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Assessment of metagenomic Nanopore and Illumina sequencing for recovering whole genome sequences of chikungunya and dengue viruses directly from clinical samples</span>. <span class="source" tagx="source" title="source">Euro Surveill.</span><span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">23</span>:<span class="fpage" tagx="fpage" title="fpage">1800228</span>.</span></li> <li tag="ref"><a name="B15" /><span class="label" tagx="label" title="label">15.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">J.F.-W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kok</span><span class="given-names" tagx="given-names" title="given-names">K.-H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">To</span><span class="given-names" tagx="given-names" title="given-names">K.K.-.W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>. <span class="source" tagx="source" title="source">Lancet</span>. <span class="year" tagx="year" title="year">2020</span>; <span class="volume" tagx="volume" title="volume">395</span>:<span class="fpage" tagx="fpage" title="fpage">514</span>â€"<span class="lpage" tagx="lpage" title="lpage">523</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31986261">31986261</a></span></span></li> <li tag="ref"><a name="B16" /><span class="label" tagx="label" title="label">16.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rang</span><span class="given-names" tagx="given-names" title="given-names">F.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kloosterman</span><span class="given-names" tagx="given-names" title="given-names">W.P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">deÂ Ridder</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">From squiggle to basepair: computational approaches for improving nanopore sequencing read accuracy</span>. <span class="source" tagx="source" title="source">Genome Biol.</span><span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">90</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30005597">30005597</a></span></span></li> <li tag="ref"><a name="B17" /><span class="label" tagx="label" title="label">17.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Minimap and miniasm: fast mapping and de novo assembly for noisy long sequences</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">32</span>:<span class="fpage" tagx="fpage" title="fpage">2103</span>â€"<span class="lpage" tagx="lpage" title="lpage">2110</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27153593">27153593</a></span></span></li> <li tag="ref"><a name="B18" /><span class="label" tagx="label" title="label">18.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koren</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walenz</span><span class="given-names" tagx="given-names" title="given-names">B.P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berlin</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bergman</span><span class="given-names" tagx="given-names" title="given-names">N.H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillippy</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation</span>. <span class="source" tagx="source" title="source">Genome Research</span>. <span class="year" tagx="year" title="year">2017)</span>; <span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">722</span>â€"<span class="lpage" tagx="lpage" title="lpage">736</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28298431">28298431</a></span></span></li> <li tag="ref"><a name="B19" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loman</span><span class="given-names" tagx="given-names" title="given-names">N.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quick</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simpson</span><span class="given-names" tagx="given-names" title="given-names">J.T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A complete bacterial genome assembled de novo using only nanopore sequencing data</span>. <span class="source" tagx="source" title="source">Nat. Methods</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">12</span>:<span class="fpage" tagx="fpage" title="fpage">733</span>â€"<span class="lpage" tagx="lpage" title="lpage">735</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26076426">26076426</a></span></span></li> <li tag="ref"><a name="B20" /><span class="label" tagx="label" title="label">20.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaser</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SoviÄ‡</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagarajan</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Å ikiÄ‡</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fast and accurate de novo genome assembly from long uncorrected reads</span>. <span class="source" tagx="source" title="source">Genome Res</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">737</span>â€"<span class="lpage" tagx="lpage" title="lpage">746</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28100585">28100585</a></span></span></li> <li tag="ref"><a name="B21" /><span class="label" tagx="label" title="label">21.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruan</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fast and accurate long-read assembly with wtdbg2</span>. <span class="source" tagx="source" title="source">Nat. Methods</span>. <span class="year" tagx="year" title="year">2020</span>; <span class="volume" tagx="volume" title="volume">17</span>:<span class="fpage" tagx="fpage" title="fpage">155</span>â€"<span class="lpage" tagx="lpage" title="lpage">158</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31819265">31819265</a></span></span></li> <li tag="ref"><a name="B22" /><span class="label" tagx="label" title="label">22.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeÂ Coster</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dâ€™Hert</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schultz</span><span class="given-names" tagx="given-names" title="given-names">D.T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cruts</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VanÂ Broeckhoven</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">NanoPack: visualizing and processing long-read sequencing data</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">34</span>:<span class="fpage" tagx="fpage" title="fpage">2666</span>â€"<span class="lpage" tagx="lpage" title="lpage">2669</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29547981">29547981</a></span></span></li> <li tag="ref"><a name="B23" /><span class="label" tagx="label" title="label">23.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breitwieser</span><span class="given-names" tagx="given-names" title="given-names">F.P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzberg</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Centrifuge: rapid and sensitive classification of metagenomic sequences</span>. <span class="source" tagx="source" title="source">Genome Res.</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">26</span>:<span class="fpage" tagx="fpage" title="fpage">1721</span>â€"<span class="lpage" tagx="lpage" title="lpage">1729</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27852649">27852649</a></span></span></li> <li tag="ref"><a name="B24" /><span class="label" tagx="label" title="label">24.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Minimap2: pairwise alignment for nucleotide sequences</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">34</span>:<span class="fpage" tagx="fpage" title="fpage">3094</span>â€"<span class="lpage" tagx="lpage" title="lpage">3100</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29750242">29750242</a></span></span></li> <li tag="ref"><a name="B25" /><span class="label" tagx="label" title="label">25.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pickett</span><span class="given-names" tagx="given-names" title="given-names">B.E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sadat</span><span class="given-names" tagx="given-names" title="given-names">E.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noronha</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Squires</span><span class="given-names" tagx="given-names" title="given-names">R.B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunt</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaremba</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">ViPR: an open bioinformatics database and analysis resource for virology research</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">40</span>:<span class="fpage" tagx="fpage" title="fpage">D593</span>â€"<span class="lpage" tagx="lpage" title="lpage">D598</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22006842">22006842</a></span></span></li> <li tag="ref"><a name="B26" /><span class="label" tagx="label" title="label">26.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewandowski</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeffery</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Downs</span><span class="given-names" tagx="given-names" title="given-names">L.O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foster</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanderson</span><span class="given-names" tagx="given-names" title="given-names">N.D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kavanagh</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaughan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salvagno</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vipond</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Nanopore metagenomic sequencing to investigate nosocomial transmission of human metapneumovirus from a unique genetic group among haematology patients in the United Kingdom</span>. <span class="source" tagx="source" title="source">J. Infect.</span><span class="year" tagx="year" title="year">2020</span>; <span class="volume" tagx="volume" title="volume">80</span>:<span class="fpage" tagx="fpage" title="fpage">571</span>â€"<span class="lpage" tagx="lpage" title="lpage">577</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32092386">32092386</a></span></span></li> <li tag="ref"><a name="B27" /><span class="label" tagx="label" title="label">27.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">C.-L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">Y.-S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">C.-I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">C.-M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">C.-M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">T.-W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Exploring the limit of using a deep neural network on pileup data for germline variant calling</span>. <span class="source" tagx="source" title="source">Nature Machine Intelligence</span>. <span class="year" tagx="year" title="year">2020</span>; <span class="volume" tagx="volume" title="volume">2</span>:<span class="fpage" tagx="fpage" title="fpage">220</span>â€"<span class="lpage" tagx="lpage" title="lpage">227</span>.</span></li> <li tag="ref"><a name="B28" /><span class="label" tagx="label" title="label">28.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanderson</span><span class="given-names" tagx="given-names" title="given-names">N.D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Street</span><span class="given-names" tagx="given-names" title="given-names">T.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foster</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swann</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atkins</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brent</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McNally</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oakley</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peto</span><span class="given-names" tagx="given-names" title="given-names">T.E.A.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Real-time analysis of nanopore-based metagenomic sequencing from infected orthopaedic devices</span>. <span class="source" tagx="source" title="source">BMC Genomics</span>. <span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">714</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30261842">30261842</a></span></span></li> <li tag="ref"><a name="B29" /><span class="label" tagx="label" title="label">29.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewandowski</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steven</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pullan</span><span class="given-names" tagx="given-names" title="given-names">S.T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lumley</span><span class="given-names" tagx="given-names" title="given-names">S.F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dona</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicholas</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alison</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morgan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bright</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Metagenomic nanopore sequencing of influenza virus direct from clinical respiratory samples</span>. <span class="source" tagx="source" title="source">J. Clin. Microbiol.</span><span class="year" tagx="year" title="year">2019</span>; <span class="volume" tagx="volume" title="volume">58</span>:<span class="fpage" tagx="fpage" title="fpage">e00963-19</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31666364">31666364</a></span></span></li> <li tag="ref"><a name="B30" /><span class="label" tagx="label" title="label">30.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewandowski</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lumley</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pullan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vipond</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carroll</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foster</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matthews</span><span class="given-names" tagx="given-names" title="given-names">P.C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peto</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crook</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span>.</span><span class="mixed-article-title" title="mixed-article-title">Detection of viral pathogens with multiplex nanopore MinION sequencing: be careful with cross-talk</span>. <span class="source" tagx="source" title="source">Front Microbiol</span>. <span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">2225</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30283430">30283430</a></span></span></li> <li tag="ref"><a name="B31" /><span class="label" tagx="label" title="label">31.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DiÂ Tommaso</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chatzou</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Floden</span><span class="given-names" tagx="given-names" title="given-names">E.W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barja</span><span class="given-names" tagx="given-names" title="given-names">P.P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palumbo</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Notredame</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nextflow enables reproducible computational workflows</span>. <span class="source" tagx="source" title="source">Nat Biotechnol</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">35</span>:<span class="fpage" tagx="fpage" title="fpage">316</span>â€"<span class="lpage" tagx="lpage" title="lpage">319</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28398311">28398311</a></span></span></li> <li tag="ref"><a name="B32" /><span class="label" tagx="label" title="label">32.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang-Turner</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volk</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fowler</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swann</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bull</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoosdally</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Connor</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peto</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crook</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Scalable Pathogen Pipeline Platform (SPâˆ§3): enabling unified genomic data analysis with elastic cloud computing</span>. <span class="source" tagx="source" title="source">2019 IEEE 12th International Conference on Cloud Computing (CLOUD)</span>. <span class="year" tagx="year" title="year">2019</span>; <span class="fpage" tagx="fpage" title="fpage">478</span>â€"<span class="lpage" tagx="lpage" title="lpage">480</span>.</span></li> <li tag="ref"><a name="B33" /><span class="label" tagx="label" title="label">33.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shabardina</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kischka</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manske</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grundmann</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frith</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MakaÅ‚owski</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">NanoPipeâ€"a web server for nanopore MinION sequencing data analysis</span>. <span class="source" tagx="source" title="source">GigaScience</span>. <span class="year" tagx="year" title="year">2019</span>; <span class="volume" tagx="volume" title="volume">8</span>:<span class="comment">doi:10.1093/gigascience/giy169</span>.</span></li> </ul> </div> </div>  </body></html>