<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Biomol Struct Dyn</span><span class="iso-abbrev" title="iso-abbrev">J. Biomol. Struct. Dyn</span><span class="publisher-id" title="publisher-id">TBSD</span><span class="publisher-id" title="publisher-id">tbsd20</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Biomolecular Structure &amp;amp; Dynamics</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0739-1102</span><span class="issn-epub" tagx="issn" title="issn-epub">1538-0254</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor &amp;amp; Francis</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1080/07391102.2020.1753577">10.1080/07391102.2020.1753577</a></span><span class="publisher-id" title="publisher-id">1753577</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Targeting SARS-CoV-2: a systematic drug repurposing approach to identify promising inhibitors against 3C-like proteinase and 2â€²-O-ribose methyltransferase</div> <div class="alt-title" title="alt-title">
R. J. Khan et al.</div> <div class="alt-title" title="alt-title">
Journal of Biomolecular Structure and Dynamics </div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">Rameez Jabeer</span></span><a href="#AF0001"><sup>a</sup></a><a href="#EN0001"><sup>*</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jha</span><span class="given-names" tagx="given-names" title="given-names">Rajat Kumar</span></span><a href="#AF0001"><sup>a</sup></a><a href="#EN0001"><sup>*</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amera</span><span class="given-names" tagx="given-names" title="given-names">Gizachew Muluneh</span></span><a href="#AF0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">Monika</span></span><a href="#AF0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">Ekampreet</span></span><a href="#AF0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pathak</span><span class="given-names" tagx="given-names" title="given-names">Amita</span></span><a href="#AF0002"><sup>b</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">Rashmi Prabha</span></span><a href="#AF0003"><sup>c</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muthukumaran</span><span class="given-names" tagx="given-names" title="given-names">Jayaraman</span></span><a href="#AF0001"><sup>a</sup></a><a href="#AN0001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">Amit Kumar</span></span><a href="#AF0001"><sup>a</sup></a><a href="#AN0001" /></span><span class="citation_author_institution" id="AF0001">[a], <span class="institution" title="institution">Department of Biotechnology, School of Engineering and Technology, Sharda University</span><span class="country" tagx="country" title="country">India</span></span><span class="citation_author_institution" id="AF0002">[b], <span class="institution" title="institution">Department of Chemistry, Indian Institute of Technology</span><span class="country" tagx="country" title="country">India</span></span><span class="citation_author_institution" id="AF0003">[c], <span class="institution" title="institution">Department of Biotechnology, IILM College of Engineering &amp;amp; Technology</span><span class="country" tagx="country" title="country">India</span></span></div> <div class="author-notes" title="author-notes"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">*</span><p>Authors Contributed equally.</p> </div> <div class="corresp" title="corresp"><b>CONTACT</b> Jayaraman Muthukumaran <span class="email" tagx="email" title="email">j.muthukumaran@sharda.ac.in</span>; Amit Kumar Singh <span class="email" tagx="email" title="email">amitk.singh@sharda.ac.in</span></div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="history" title="history"><span class="received" title="received">received: 2020-2-17</span><span class="accepted" title="accepted">accepted: 2020-3-30</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Informa UK Limited, trading as Taylor &amp;amp; Francis Group</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the COVID-19 pandemic or until permissions are revoked in writing. Upon expiration of these permissions, PMC is granted a perpetual license to make this article available via PMC and Europe PMC, consistent with existing copyright protections.</span></span></div><a class="self-uri" href="TBSD_0_1753577.pdf" title="self-uri">TBSD_0_1753577.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>The recent pandemic associated with SARS-CoV-2, a virus of the Coronaviridae family, has resulted in an unprecedented number of infected people. The highly contagious nature of this virus makes it imperative for us to identify promising inhibitors from pre-existing antiviral drugs. Two druggable targets, namely 3C-like proteinase (3CLpro) and 2â€²-O-ribose methyltransferase (2â€²-O-MTase) were selected in this study due to their indispensable nature in the viral life cycle. 3CLpro is a cysteine protease responsible for the proteolysis of replicase polyproteins resulting in the formation of various functional proteins, whereas 2â€²-O-MTase methylates the ribose 2â€²-O position of the first and second nucleotide of viral mRNA, which sequesters it from the host immune system. The selected drug target proteins were screened against an in-house library of 123 antiviral drugs. Two promising drug molecules were identified for each protein based on their estimated free energy of binding (Î"G), the orientation of drug molecules in the active site and the interacting residues. The selected protein-drug complexes were then subjected to MD simulation, which consists of various structural parameters to equivalently reflect their physiological state. From the virtual screening results, two drug molecules were selected for each drug target protein [Paritaprevir (Î"G = âˆ’9.8â€‰kcal/mol) &amp;amp; Raltegravir (Î"G = âˆ’7.8â€‰kcal/mol) for 3CLpro and Dolutegravir (Î"G = âˆ’9.4â€‰kcal/mol) and Bictegravir (Î"G = âˆ’8.4â€‰kcal/mol) for 2â€²-OMTase]. After the extensive computational analysis, we proposed that Raltegravir, Paritaprevir, Bictegravir and Dolutegravir are excellent lead candidates for these crucial proteins and they could become potential therapeutic drugs against SARS-CoV-2. </p> <p>Communicated by Ramaswamy H. Sarma</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Drug repurposing</div> <div class="kwd" title="kwd">
SARS-CoV-2</div> <div class="kwd" title="kwd">
3C-like proteinase</div> <div class="kwd" title="kwd">
2â€²-O-ribose methyltransferase</div> <div class="kwd" title="kwd">
docking</div> <div class="kwd" title="kwd">
MD simulation</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1.</span><div class="title" tagx="title" title="title">
Introduction</div> <p>Members of the family Coronaviridae are enveloped and maintain a single-strand, positive-sense RNA genome ranging from 26 to 32â€‰kb in length (Su etÂ al., <a href="#CIT0040">2016</a>). They can be classified into four genera: alpha, beta, delta and gamma, out of which alpha and beta coronaviruses (CoVs) are known to infect humans (de Wilde etÂ al., <a href="#CIT0017">2018</a>). They are circulated among humans, other mammals, and birds and can cause respiratory, enteric, hepatic and neurologic diseases (Weiss &amp;amp; Leibowitz, <a href="#CIT0046">2011</a>; Zhu etÂ al., <a href="#CIT0056">2020</a>). Even though the majority of human coronavirus infections are mild, the epidemics of two betacoronaviruses (Î²CoV), namely Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) and Middle East Respiratory Syndrome coronavirus (MERS-CoV), has caused more than 10,000 cumulative cases in the past two decades, with mortality rates of 10% for SARS-CoV and 37% for MERS-CoV (World Health Organization, 2019). In late December 2019, several local health facilities reported groups of patients with pneumonia of unknown cause, and they were found to be epidemiologically connected with the Huanan wholesale seafood market in Wuhan, Hubei province, China (Huang etÂ al., <a href="#CIT0021">2020</a>). Next-generation sequencing analysis from bronchoalveolar lavage fluid samples and cultured isolates from the patients indicated a novel human-infecting coronavirus (Huang etÂ al., <a href="#CIT0021">2020</a>), officially named Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) (World Health Organization, <a href="#CIT0050">2020a</a>). Also, the World Health Organisation (WHO) has officially named the disease caused by SARS-CoV-2 as coronavirus disease 2019 (COVID-19) (World Health Organization, <a href="#CIT0050">2020a</a>).</p> <p>As of 29 March 2020, there have been more than 652,000 confirmed cases of COVID-19 across 177 countries/regions, with 30,300 fatalities. China, Italy and Spain have been the most affected countries with combined fatalities of more than 19,100 patients (Johns Hopkins CSSE, <a href="#CIT0022">2020</a>). On 11 March, the WHO has officially declared COVID-19 as a pandemic (World Health Organization, <a href="#CIT0052">2020c</a>). Earlier, on 30 January 2020, the World Health Organisation (WHO) declared SARS-CoV-2 pandemic as a Public Health Emergency of International Concern (World Health Organization, <a href="#CIT0054">2020e</a>). According to the Centers for Disease Control and Prevention (CDC) and the WHO, the global death rate has been 3.4% (World Health Organization, <a href="#CIT0051">2020b</a>), but it varies significantly with the age of the patients (Centers for Disease Control and Prevention, <a href="#CIT0012">2020</a>). In order to control the menace caused by SARS-CoV-2, we suggest two of its indispensable druggable targets, namely 3C-like proteinase and 2â€²-O-ribose methyltransferase (Benkert etÂ al., <a href="#CIT0005">2011</a>; Yang etÂ al., <a href="#CIT0055">2003</a>). 3C-like proteinase (3CLpro) is a cysteine protease present in the Coronavirus replicase polyprotein. This protease plays a central role in viral replication and transcription functions through extensive proteolysis of two replicase polyproteins, pp1a and pp1ab (Yang etÂ al., <a href="#CIT0055">2003</a>). It cleaves at least 11 inter-domain sites on the pp1a and pp1ab polyproteins to generate individual functional proteins, including an RNA-directed RNA polymerase, a helicase, an exoribonuclease, an endoribonuclease and a 2â€²-O-ribose methyltransferase. Also, it is excised from polyproteins by its proteolytic activity and forms a homodimer with one active site per subunit (Yang etÂ al., <a href="#CIT0055">2003</a>). All of these features of 3CLpro makes it a principal drug target for SARS-CoV-2.</p> <p>Another crucial protein responsible for viral replication and expression in host cells is non-structural protein 16 (nsp16) or 2â€²-OMTase. Most viral mRNAs possess a 5â€²-terminal cap structure (m7GpppN) which is essential for efficient splicing, nuclear export, translation and stability (Chen etÂ al., <a href="#CIT0014">2011</a>). This structure undergoes methylation catalysed by 2â€²-O-ribose methyltransferase at the ribose 2â€²-O position of the first and second nucleotide of the mRNA(Chen etÂ al., <a href="#CIT0014">2011</a>; Lugari etÂ al., <a href="#CIT0028">2010</a>). Nsp 16 encodes the 2â€²-OMTase which provides the viral mRNA with the ability to camouflage and obscure itself from the host cell, thus preventing recognition and activation of the host immune response which is essential for successful viral infection (Menachery etÂ al., <a href="#CIT0033">2014</a>). This protein can, therefore, act as another potential drug target for the SARS-CoV-2.</p> <p>The present study utilises the systematic drug repurposing approach to identify antiviral drugs which can act as promising inhibitors against 3CLpro and 2â€²-OMTase and also understand their inhibitory mechanism through extensive <i>in silico</i> approaches. We have employed molecular docking and molecular dynamics (MD) simulation studies to calculate various structural parameters including the estimated binding free energy (Î"G) of the drugs, their estimated inhibition constant (Ki), Root Mean Square Deviation (RMSD), Root Mean Square Fluctuation (RMSF), Radius of Gyration (Rg), Solvent Accessible Surface Area (SASA), Principal Component Analysis (PCA) and the intermolecular hydrogen bonds (H-bonds) for free and drug bounded protein molecules.</p> </div> <div class="materialandmethods" title="sec"><span class="label" tagx="label" title="label">2.</span><div class="title" tagx="title" title="title">
Material and methods</div> <div class="homologymodellingandmodelevaluation" title="sec"><span class="label" tagx="label" title="label">2.1.</span><div class="title" tagx="title" title="title">
Homology modelling and model evaluation</div> <p>The amino acid sequence of SARS-CoV-2 orf1ab polyprotein was retrieved from National Centre for Biotechnological Information (NCBI) (Tax id: 2697049, Accession No: QHD43415). It was then subjected to pairwise sequence alignment against 3CLpro and 2â€²-OMTase from its close relative SARS-CoV (tax id: 694009) to figure out their amino acid sequence in SARS-CoV-2. Homology modelling of these proteins was performed using SWISS-MODEL (Waterhouse etÂ al., <a href="#CIT0045">2018</a>) with the user template option. Template structures were selected from Protein BLAST (Camacho etÂ al., <a href="#CIT0010">2009</a>) against Protein Data Bank (PDB) (Berman etÂ al., <a href="#CIT0007">2000</a>) based on query coverage, percentage of sequence identity and crystal structure resolution. Energy minimisation of modelled proteins was achieved using YASARA (Krieger etÂ al., <a href="#CIT0025">2009</a>) and GalaxyWEB (Ko etÂ al., <a href="#CIT0024">2012</a>) web servers. Model evaluation was performed before and after minimisation using Structure Analysis and Verification Server (SAVES) v5.0 meta server (UCLA MBI), which includes ERRAT (Colovos &amp;amp; Yeates, <a href="#CIT0016">1993</a>), Verify-3D (Luthy etÂ al., <a href="#CIT0029">1992</a>) and PROCHECK (Laskowski etÂ al., <a href="#CIT0027">1993</a>) programmes.</p> </div> <div class="screeningandselectionofantiviraldrugsagainst3clproand2â€²-omtase" title="sec"><span class="label" tagx="label" title="label">2.2.</span><div class="title" tagx="title" title="title">
Screening and selection of antiviral drugs against 3CLpro and 2â€²-OMTase</div> <p>PyRx virtual screening software v0.8 (<a href="http://pyrx.sourceforge.net/downloads">http://pyrx.sourceforge.net/downloads</a>) was used for the screening of antiviral drugs against 3CLpro and 2â€²-OMTase proteins. One hundred twenty-three entries belonging to the category of antiviral drugs were downloaded from DrugBank database (Wishart etÂ al., <a href="#CIT0047">2018</a>) and were subsequently converted into a single structure data file (SDF) library (see <a href="https://doi.org/10.1080/07391102.2020.1753577">supplementary data</a>, <a href="#t0003">Table 3</a>). AutoDockVina (Trott &amp;amp; Olson, <a href="#CIT0042">2009</a>) docking wizard inbuilt in PyRx was utilised for all of the docking calculations. In PyRx, blind docking with the exhaustiveness 32 option was used to carry out the virtual screening and docking of antiviral drugs against 3CLpro (Grid Box: center_<i>x</i>â€‰=â€‰75.98, center_y = âˆ’15.53, center_<i>z</i>â€‰=â€‰18.01, size_<i>x</i>â€‰=â€‰70.46, size_<i>y</i>â€‰=â€‰90.53, size_<i>z</i>â€‰=â€‰61.34) and 2â€²-OMTase (Grid Box: center_<i>x</i>â€‰=â€‰64.12, center_<i>y</i>â€‰=â€‰70.35, center_<i>z</i>â€‰=â€‰71.46, size_<i>x</i>â€‰=â€‰67.11, size_<i>y</i>â€‰=â€‰66.13, size_<i>z</i>â€‰=â€‰68.36). Once we got the results obtained from virtual screening, we selected two drug molecules for each protein based on the estimated binding free energy (Î"G), estimated inhibition constant (Ki), their orientation at the catalytic site and the interacting residues. The selected protein-drug complexes were further subjected to MD simulations for understanding the structural stability of protein-drug complexes at the long-interval.</p> <div class="table-wrap_UNKNOWN" id="t0003" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3.</span> <p>Interaction details of Paritaprevir and Raltegravir with 3CLpro.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">S/No</th> <th align="center">DrugBank ID</th> <th align="center">HB</th> <th align="center">D (Ã…)</th> <th align="center">Pi-SR</th> <th align="center">D (Ã…)</th> <th align="center">vdWISR</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">1.</td> <td align="left">Paritaprevir (DB09297)</td> <td align="left">Ser46</td> <td align="left">2.33</td> <td align="left">Met49 Leu50 Cys145 Pro168</td> <td align="left">5.27 3.85, 4.61 4.60 3.64</td> <td align="left">Thr24, Thr25, Leu27, His41, Cys44, Thr45, Glu47, Phe140, Gly143, Glu166, Gln189</td> </tr> <tr> <td align="left">2.</td> <td align="left">Raltegravir (DB06817)</td> <td align="left">Thr24 Ser46 Asn142 Gly143 Glu166</td> <td align="left">2.53 2.47 2.39 2.11 2.59</td> <td align="left">Thr25 Met49 Met165</td> <td align="left">3.93 4.87 5.14</td> <td align="left">Thr26, Leu27, His41, Cys44, Thr45, Phe140, Ser144, Cys145</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Hydrogen Bond residues (HB), Distance (D), Pi-Interaction Sharing Residues (Pi-SR), and Van der Waals Interaction Sharing Residues (vdWISR).</p> </div> </div> </div> </div> <div class="moleculardynamics(md)simulations" title="sec"><span class="label" tagx="label" title="label">2.3.</span><div class="title" tagx="title" title="title">
Molecular dynamics (MD) simulations</div> <p>To make the docking results mimic the physiological state of protein molecules, the free proteins and the best protein-drug complexes were subjected to MD simulation in an explicit solvent model. MD simulation was performed using GROMACS 2019 (Abraham etÂ al.) with GROMOS96 43a1 force field parameters (Chiu etÂ al., <a href="#CIT0015">2009</a>). The topology of the drug molecules was created using PRODRG webserver (Schuttelkopf &amp;amp; van Aalten, <a href="#CIT0039">2004</a>). The simulation of all of the proteins and protein-drug complexes were run for a period of 100â€‰ns. To make the system electrostatically neutral, counter ions were added to the protein-drug complexes. The complexes were solvated within 10â€‰Ã… SPC/E water cube (Berendsen etÂ al., <a href="#CIT0006">1987</a>). The protein-drug complexes were minimised in multiple steps using steepest descent method, where minimisation of the whole system, water cube and non-heavy atoms of the complexes were accomplished. The entire systems were then progressively heated up to 300â€‰K on a time scale of 100â€‰ps. The equilibration steps were performed in two different phases, one with constant pressure and temperature <b>(</b>NPT<b>)</b> and the other with steady volume and temperature (NVT). Various structural parameters, like Root Mean Square Deviation (RMSD), Root Mean Square Fluctuations (RMSF), Radius of Gyration (Rg), Principal Component Analysis (PCA) based on essential dynamics (ED) approach, Inter-molecular Hydrogen Bonding (H-bonding) and Solvent-Accessible Surface Area (SASA) were calcualted as a function of time to explore the structural behaviour of the proteins and protein-drug complexes.</p> </div> </div> <div class="resultsanddiscussion" title="sec"><span class="label" tagx="label" title="label">3.</span><div class="title" tagx="title" title="title">
Results and discussion</div> <div class="homologymodelingandmodelevaluation" title="sec"><span class="label" tagx="label" title="label">3.1.</span><div class="title" tagx="title" title="title">
Homology modeling and model evaluation</div> <p>Homology modelling is one of the standard <i>in silico</i> procedures, which allows predicting the three-dimensional structure of a protein molecule based on experimentally solved homologous structures known as structural templates (Bordoli etÂ al., <a href="#CIT0008">2009</a>). Sequence identity and Query coverage are two crucial parameters to identify the suitable structural templates for homology modelling technique. Moreover, the 3D structure of the target protein is essential to explore its function (Micheletti, <a href="#CIT0035">2013</a>). The 3D models of 3CLpro and 2â€²-OMTase were predicted using SWISS-MODEL (Waterhouse etÂ al., <a href="#CIT0045">2018</a>). For the best results, we used the user-defined template option present in SWISS-MODEL. Protein BLAST (Camacho etÂ al., <a href="#CIT0010">2009</a>) results were taken into account for the selection of suitable template structures. The X-ray crystal structure of 3CLpro from Human SARS coronavirus (SARS-CoV) solved at 1.9â€‰Ã… (PDB ID: 1UJ1) resolution was selected as the template for SARS-CoV-2 3CLpro. For 2â€²-OMTase, we used a 2â€‰Ã… resolution X-ray crystal structure of nsp10/nsp16 complex (PDB ID: 3R24) of SARS-CoV. The template (PDB ID: 1UJ1) was selected because it has 96.08% sequence identity and 100% query coverage towards SARS-CoV-2 3CLpro. Also, the template (PDB ID: 3R24) has a sequence identity of 93.29% and query coverage of 100% against SARS-CoV-2. In SWISS-MODEl, Global Model Quality Estimation (GMQE) (Waterhouse etÂ al., <a href="#CIT0045">2018</a>) score of 0.99 out of 1 and QMEAN (Benkert etÂ al., <a href="#CIT0005">2011</a>) score of 0.09 was achieved for 3CLpro, whereas for 2â€²-OMTase, GMQE score was 0.97 out of 1, and QMEAN score was âˆ’2.57. These quality estimations depict the reliability of the modelled protein structure, targetâ€"template alignment and the template search method. It is also worth noting that 3CLpro makes a homodimeric structure, but since only one protomer is active in each dimer, we only modelled the single protomer, and further used it for <i>in silico</i> interaction or molecular docking and MD simulation studies (Chen etÂ al., <a href="#CIT0013">2006</a>).</p> <p>Once the models were obtained, they were evaluated using SAVES v5.0 metaserver based on ERRAT, Verify 3D and PROCHECK. The models were further refined and minimised using YASARA Energy Minimization Server and GalaxyWEB servers. The final model of 3CLpro had a Verify 3D score of 100% and ERRAT quality factor of 98.19. In the case of PROCHECK, 93.9% residues were present in the most favoured regions of Ramachandran Plot (Ramachandran etÂ al., <a href="#CIT0037">1963</a>), and more importantly, only one residue (Tyr154) was present in the disallowed regions. In the case of 2â€²-OMTase, the final model had a Verify 3D score of 87.33%, and ERRAT quality factor of 94.50. According to PROCHECK, 90% of the residues were in the most favoured regions of the Ramachandran plot, and notably, no residues were present in the disallowed regions. We used PDBsum (Laskowski etÂ al., <a href="#CIT0026">2018</a>) to calculate the overall average G-Factor score, which is 0.12 for both 3CLpro and 2â€²-OMTase, which also reflected the accuracy of the refined modelled structure (see <a href="https://doi.org/10.1080/07391102.2020.1753577">supplementary data</a>, <a href="#t0001">Tables 1</a> and <a href="#t0002">2</a>).</p> <div class="table-wrap_UNKNOWN" id="t0001" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <p>Selected drug molecules obtained from virtual screening of 3CLpro against the in-house library of anti-viral drugs.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">S/No</th> <th align="center">Drug Bank ID</th> <th align="center">EBE (kcal/mol)</th> <th align="center">EIC (M)</th> <th align="center">Molecular formula</th> <th align="center">Chemical scheme</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">1</td> <td align="center">Paritaprevir (DB09297)</td> <td align="center">âˆ’9.8</td> <td align="center">65.52â€‰nM</td> <td align="center">C<sub>40</sub>H<sub>43</sub>N<sub>7</sub>O<sub>7</sub>S</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> </tr> <tr> <td align="left">2</td> <td align="center">Raltegravir (DB06817)</td> <td align="center">âˆ’7.8</td> <td align="center">1.916 ÂµM</td> <td align="center">C<sub>20</sub>H<sub>21</sub>FN<sub>6</sub>O<sub>5</sub></td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>EBE: Estimated Binding Free Energy; EIC: Estimated Inhibition Constant.</p> </div> </div> </div> <div class="table-wrap_UNKNOWN" id="t0002" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2.</span> <p>Selected drug molecules obtained from virtual screening of 2â€²-OMTase against the in-house library of anti-viral drugs.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">S/No</th> <th align="center">Drug Bank ID</th> <th align="center">EBE (kcal/mol)</th> <th align="center">EIC(M)</th> <th align="center">Molecular formula</th> <th align="center">Chemical scheme</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">1</td> <td align="center">Dolutegravir (DB08930)</td> <td align="center">âˆ’9.4</td> <td align="center">128.7â€‰nM</td> <td align="center">C<sub>20</sub>H<sub>19</sub>F<sub>2</sub>N<sub>3</sub>O<sub>5</sub></td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> </tr> <tr> <td align="left">2</td> <td align="center">Bictegravir (DB11799)</td> <td align="center">âˆ’8.4</td> <td align="center">0.696 ÂµM</td> <td align="center">C<sub>21</sub>H<sub>18</sub>F<sub>3</sub>N<sub>3</sub>O<sub>5</sub></td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>EBE: Estimated Binding Free Energy; EIC: Estimated Inhibition Constant.</p> </div> </div> </div> </div> <div class="screeningofanti-viraldrugsagainst3clproand2â€²-omtase" title="sec"><span class="label" tagx="label" title="label">3.2.</span><div class="title" tagx="title" title="title">
Screening of anti-viral drugs against 3CLpro and 2â€²-OMTase</div> <p>Virtual screening is one of the essential tools in drug designing and discovery process as it allows the prediction of the conformation and orientation of a ligand or drug molecule within the binding site of a target molecule (Kitchen etÂ al., <a href="#CIT0023">2004</a>). Also, the virtual screening tools employ scoring functions to predict the strength of the protein-ligand interactions, generating a score for each docked pose. Subsequently, the docked poses are ranked based on the score provided by the scoring function to identify the pose that most closely resembles the actual binding mode of the ligand (Gimeno etÂ al., <a href="#CIT0019">2019</a>). PyRx was used for the virtual screening of antiviral drugs against 3CLpro and 2â€²-OMTase proteins. One hundred twenty-three compounds belonging to the category of antiviral drugs were downloaded from DrugBank (Wishart etÂ al., <a href="#CIT0047">2018</a>) and were subsequently converted into a single SDF library. This in-house SDF library was used to screen both the proteins. Two drugs were selected against each protein based on their estimated free energy of binding (Î"G), binding site orientation and the protein residues with which they were found to be interacting with drug molecules. In the case of 3CLpro (see <a href="#t0001">Table 1</a>), we chose two molecules, namely Paritaprevir (DB09297) and Raltegravir (DB06817). Similarly, for 2â€²-OMTase (see <a href="#t0002">Table 2</a>), the best two drugs were found to be Dolutegravir (DB08930) and Bictegravir (DB11799).</p> </div> <div class="bindingpatternofdrugmoleculesagainst3clproand2â€²-omtase" title="sec"><span class="label" tagx="label" title="label">3.3.</span><div class="title" tagx="title" title="title">
Binding pattern of drug molecules against 3CLpro and 2â€²-OMTase</div> <div class="3clproofsars-cov-2" title="sec"><span class="label" tagx="label" title="label">3.3.1.</span><div class="title" tagx="title" title="title">
3CLpro of SARS-CoV-2</div> <p>As SARS-CoV 3CLpro (PDB ID: 1UJ1), the predicted model also forms a homo-dimer with both protomers (signified as â€˜Aâ€™ and â€˜Bâ€™) oriented nearly at right angles to each other (see <a href="https://doi.org/10.1080/07391102.2020.1753577">supplementary data</a>, <a href="#F0001">Figure 1</a>). Each monomer of the modelled structure is composed of three distinct globular domains (see <a href="#F0001">Figure 1</a>). Domains I comprises residues 8â€"101, domain II includes residues 102â€"184 and domain III consists of residues 201â€"303 and is attached with domain II through an extended loop region (residues 185â€"200<b>)</b>. Domain I and Domain II form a chymotrypsin fold, whereas domain III is required for the homodimer assembly. The substrate-binding site is positioned in the cleft of domain I and domain II and is composed of a Cys145-His41 catalytic dyad. The substrate-binding pocket consists of His41, Phe140, Asn142, Gly143, Ser144, Cys145, Tyr161, His163, Glu166 and His172 residues (Yang etÂ al., <a href="#CIT0055">2003</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Interaction of drugs with 3CLpro (Domain I-Red, Domain II-Purple, Domain III-Cyan, Extended loop-Yellow). (a) Three dimensional representation of 3CLpro active site residues interacting with Paritaprevir (DB09297) (Green). (b) Two dimensional representation of 3CLpro active site residues interacting with Paritaprevir (Green) via Van der Waals interactions (slightly green colour), hydrogen bonds (dark green colour), and pi-interactions (light pink colour). (c) Three dimensional representation of 3CLpro active site residues interacting with Raltegravir (DB06817) (Orange). (d) Two dimensional representation of 3CLpro active site residues interacting with Raltegravir(Orange).</p>   </div> <p>The estimated binding free energy (Î"G) of Paritaprevir against 3CLpro was observed to be âˆ’9.8â€‰kcal/mol, and the estimated inhibitory constant was 65.52â€‰nM. On the other hand, the binding affinity (Î"G) of Raltegravir was measured to be âˆ’7.8â€‰kcal/mol, and the inhibitory constant was 1.916â€‰ÂµM. Both Paritaprevir and Raltegravir were found to be interacting with the active site residues (including the Cys145-His41 catalytic dyad) (see <a href="#t0003">Table 3</a>) and interacting with the substrate-binding pocket (see <a href="#F0001">Figure 1</a>). The binding free energy of 3CLpro with other drug molecules has been given in <a href="https://doi.org/10.1080/07391102.2020.1753577">supplementary information</a>.</p> </div> <div class="2â€²-omtaseofsars-cov-2" title="sec"><span class="label" tagx="label" title="label">3.3.2.</span><div class="title" tagx="title" title="title">
2â€²-OMTase of SARS-CoV-2</div> <p>The predicted model of 2â€²-O methyltransferase (2â€²-OMTase) (see <a href="#F0002">Figure 2</a>) comprises of a core MTase domain, S-Adenosyl methionine (SAM) binding region and a cap-binding groove. The core MTase domain is found to comprise residues 30â€"209 of nsp16 as found in SARS-CoV (Chen etÂ al., <a href="#CIT0014">2011</a>). 2â€²-OMTase catalyses the transfer of SAM methyl group by relying on a conserved K-D-K-E tetrad within the substrate-binding pocket for activity (Menachery etÂ al., <a href="#CIT0033">2014</a>). The surface provided by K-D-K-E (Lys-46, Asp-130, Lys-170 and Glu-203) motif present at the bottom of the central groove might bind the first adenine nucleotide conserved as the acceptor of methyl group during methylation (Chen etÂ al., <a href="#CIT0014">2011</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Interaction of drugs with 2â€²-OMTase (MTase Domain-Purple). (a) Three dimensional representation of 2â€²-OMTase active site residues interacting with Dolutegravir (DB08930) (Sand colour). (b) Two dimensional representation of 2â€²-OMTase active site residues interacting with Dolutegravir (Sand colour) via Van der Waals interactions (slightly green colour), hydrogen bonds (dark green colour), and pi-interactions (light pink colour, orange colour). (c) Three dimensional representation of 2â€²-OMTase active site residues interacting with Bictegravir (DB11799) (Yellow). (d) Two dimensional representation of 2â€²-OMTase active site residues interacting withBictegravir (Yellow).</p>   </div> <p>In the case of 2â€²-OMTase, Dolutegravir showed an estimated binding free energy (Î"G) of âˆ’9.4â€‰kcal/mol, and the inhibition constant was 128.7â€‰nM. Besides, Bictegravir conferred an estimated binding free energy (Î"G) of âˆ’8.4â€‰kcal/mol against 2â€²-OMTase, and the estimated inhibition constant was 0.696â€‰ÂµM. Moreover, both Dolutegravir and Bictegravir were found to be actively interacting with the catalytic site residues (see <a href="#t0004">Table 4</a>) and interacting with the substrate-binding pocket (see <a href="#F0002">Figure 2</a>). The binding free energy of 2â€²-OMTase with other drug molecules has been given in <a href="https://doi.org/10.1080/07391102.2020.1753577">supplementary information</a>.</p> <div class="table-wrap_UNKNOWN" id="t0004" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 4.</span> <p>Interaction details of Dolutegravir and Raltegravir with 2â€²-OMTase.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">S/No</th> <th align="center">DrugBank ID</th> <th align="center">HB</th> <th align="center">D (Ã…)</th> <th align="center">Pi-SR</th> <th align="center">D (Ã…)</th> <th align="center">vdWISR</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">1</td> <td align="left">Dolutegravir (DB08930)</td> <td align="left">Asp130 Tyr132 Lys170</td> <td align="left">2.45 2.94 1.95, 2.55</td> <td align="left">Met42 Leu100 Met131 Pro134</td> <td align="left">5.45 3.49 5.83 4.73</td> <td align="left">Asn43, Lys46, Gly71, Gly73, Ser74, Asp75, Asp99, Asn101, Glu203</td> </tr> <tr> <td align="left">2</td> <td align="left">Bictegravir (DB11799)</td> <td align="left">Asp130 Lys170 Asn198</td> <td align="left">2.43 2.28 2.15</td> <td align="left">Met42 Leu100 Met131 Pro134</td> <td align="left">4.34 4.46, 5.02 4.73 5.12, 5.17</td> <td align="left">Asn43, Lys46, Gly71, Asp99, Asn101, Tyr132, Asp133</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Hydrogen Bond residues (HB), Distance (D), Pi-Interaction Sharing Residues (Pi-SR), and Van der Waals Interaction Sharing Residues (vdWISR).</p> </div> </div> </div> <p>According to the results obtained from electrostatic surface potential, all of the drug molecules were found interacting mostly with the negatively charged residues of both 3CLpro (see <a href="#F0003">Figure 3</a>) and 2â€²-OMTase (see <a href="#F0004">Figure 4</a>), which revealed that these molecules were oriented in the active site of both the proteins.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>The electrostatic surface potential interaction of Paritaprevir (DB09297) (Green) and Raltegravir (DB06817) (Orange) bound to 3CLpro. The zoomed view is representing the active site cleft.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>The electrostatic surface potential interaction of Dolutegravir (DB08930) (Sand) and Bictegravir (DB11799) (Yellow) bound to 2â€²-OMTase. The zoomed view is representing the active site cleft.</p>   </div> </div> </div> <div class="moleculardynamics(md)simulation" title="sec"><span class="label" tagx="label" title="label">3.4.</span><div class="title" tagx="title" title="title">
Molecular dynamics (MD) simulation</div> <p>Molecular dynamics simulation is used to study the physical transitions of atoms as well as entire macromolecules and are effectively adopted to comprehend the structure-to-function relevance of a given macromolecule (Ambure etÂ al., <a href="#CIT0002">2019</a>; Amera etÂ al., <a href="#CIT0003">2019</a>; Hospital etÂ al., <a href="#CIT0020">2015</a>). Moreover, it can also be employed to illustrate the strength, pattern, and properties of protein-drug interactions and the dynamic conformational changes of macromolecule experiences under various physiological conditions (Amera etÂ al., <a href="#CIT0003">2019</a>). Different structural parameters, including RMSD, RMSF, Rg, PCA (based on ED approach), Intermolecular H-bondsand SASA, were evaluated as a function of time. To elucidate the conformational stability, dynamics, structural stability, folding properties and compactness of protein-drug complexes, MD simulations were performed for a period of 100â€‰ns.</p> <div class="rootmeansquaredeviationanalysis" title="sec"><span class="label" tagx="label" title="label">3.4.1.</span><div class="title" tagx="title" title="title">
Root Mean Square Deviation analysis</div> <p>RMSD is essential to quantify the structural stability of a protein or protein-drug complexes within a regular time frame. RMSD analysis depicted that unbound 3CLpro started stabilising after 15â€‰ns, and it maintained stability until 90â€‰ns. The RMSD pattern varied slightly during the final 10â€‰ns. Besides, the 3CLpro-Raltegravir complex also attained stability at around 10â€‰ns, and it maintained it until 90â€‰ns. The RMSD again decreased during the final 10â€‰ns (90â€"100â€‰ns). In contrast, 3CLpro-Paritaprevir complex gained stability at around 20â€‰ns, retained that stability until 70â€‰ns except for one minor dip at about 50â€‰ns. After 70â€‰ns, the average RMSD declined until the end of the simulation cycle (see <a href="#F0005">Figure 5a</a>). An average RMSD value of 0.295â€‰nm, 0.361â€‰nm and 0.401â€‰nm was obtained for unbound 3CLpro, 3CLpro-Raltegravir and 3CLpro-Paritaprevir complexes, respectively. Overall results explained that these two drug molecules did not significantly influence the structural stability of 3CLpro, although they nearly maintained the structural integrity, particularly Raltegravir. Based on the average RMSD values obtained from this analysis, the 3CLpro-Raltegravir complex was found to be closer to the drug-free form of 3CLpro.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>Analysis of Molecular Dynamics Simulation results of free 3CLpro (Black), 3CLpro-Raltegravir complex (Red) and 3CLpro-Paritaprevir complex (turquoise). (a) Root Mean Square Deviation (RMSD), (b) Root Mean Square Fluctuation (RMSF), (c) Radius of Gyration (Rg), (d) Solvent Accessible Surface Area (SASA).</p>   </div> <p>RMSD analysis of the free form of 2â€²-OMTase depicted that the protein molecule achieved stability at around 40â€‰ns, and it almost maintained its stability until 100â€‰ns. The maximum RMSD fluctuations were recorded in the first 20â€‰ns only. There were minor variations at around 60â€‰ns, but apart from that, the protein was quite stable until the end of the simulation cycle. The 2â€²-OMTase-Dolutegravir complex attained stability at around 20â€‰ns and maintained that stability until 90â€‰ns, except a minor drop at approximately 70â€‰ns. The average fluctuations spiked a little bit in the final 10â€‰ns (90â€"100â€‰ns) of this simulation. The 2â€²-OMTase-Bictegravir complex also took 40â€‰ns to attain stability, but it also maintained it until the end of the simulation cycle (40â€"100â€‰ns). It showed a minor dip at 60â€‰ns, but it was able to gain its stability back at around 65â€‰ns. As the free protein, the maximum fluctuations of protein-drug complexes were recorded in the first 20â€‰ns only, after that, all of them took a steady path (see <a href="#F0006">Figure 6a</a>) and maintained convergence state until the end of the simulation period. However, in the case 2â€²-OMTase-Dolutegravir complex, slight changes were observed at 90â€"100â€‰ns. An average RMSD value of 0.299â€‰nm, 0.393â€‰nm and 0.410â€‰nm was obtained for unbound 2â€²-OMTase, 2â€²-OMTase-Dolutegravir complex and 2â€²-OMTase-Bictegravir complex, respectively. As evident from the average RMSD values, both of the drug complexes were very comparable to each other, but the 2â€²-OMTase-Dolutegravir complex was found closer to the free protein molecule.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <p>Analysis of Molecular Dynamics Simulation results of free 2â€²-OMTase (Blue), 2â€²-OMTase-Bictegravir complex (Orange) and 2â€²-OMTase-Dolutegravir complex (Dark Green). (a) Root Mean Square Deviation (RMSD), (b) Root Mean Square Fluctuation (RMSF), (c) Radius of Gyration (Rg), (d) Solvent Accessible Surface Area (SASA).</p>   </div> </div> <div class="rootmeansquarefluctuationanalysis" title="sec"><span class="label" tagx="label" title="label">3.4.2.</span><div class="title" tagx="title" title="title">
Root Mean Square Fluctuation analysis</div> <p>RMSF is an essential structural parameter to identify the flexible and rigid regions of the protein structure. It is a measure of deviations of a particle from its original position. Moreover, it can also be used to identify the flexible residues in the protein, and thus it enables us to explore the conformational flexibility of the protein structure. The RMSF analysis was estimated for C-alpha atom of each residue representing the average displacement of each atom.</p> <p>In the case of 3CLpro, we observed the highest fluctuations in loop heavy domain II and domain III. The loop regions are very flexible elements of protein molecules, and their flexibility is essential to accommodate the drug at the binding site appropriately. Besides, the 3CLpro-Raltegravir complex was found to be showing more significant fluctuations in comparison to 3CLpro-Paritaprevir complex (see <a href="#F0005">Figure 5b</a>). The average RMSF values of 3CLpro, 3CLpro-Raltegravir complex and 3CLpro-Paritaprevir complex were found to be 0.170â€‰nm, 0.182â€‰nm and 0.184â€‰nm, respectively. Additionally, the majority of the protein residues were stable with RMSF values smaller than 0.3â€‰nm. Also, the catalytic dyad (His41 and Cys145) participating in interactions with the drug molecules remained highly stable throughout the MD simulation.</p> <p>In the case of 2â€²-OMTase, the fluctuations were observed throughout the entire protein. 2â€²-OMTase also has an extensively looped structure, and as previously explained, we did expect to see significant fluctuations across the molecule. Moreover, both protein-drug complexes were also highly flexible (see <a href="#F0006">Figure 6b</a>). The average RMSF values of 2â€²-OMTase, 2â€²-OMTase-Dolutegravir complex and 2â€²-OMTase-Bictegravir complex were found to be 0.155â€‰nm, 0.161â€‰nm and 0.165â€‰nm, respectively. Even though the average fluctuations of both protein-drug complexes were very similar, the 2â€²-OMTase-Dolutegravir complex was consistently found to be showing the most significant fluctuations. Besides, the conserved K-D-K-E (Lys-46, Asp-130, Lys-170 and Glu-203) remained remarkably stable throughout the MD simulation.</p> </div> <div class="radiusofgyration(rg)analysis" title="sec"><span class="label" tagx="label" title="label">3.4.3.</span><div class="title" tagx="title" title="title">
Radius of gyration (Rg) analysis</div> <p>The radius of gyration is a valuable tool to understand the folding properties and compactness of protein and protein-drug complexes. It can also be used to elucidate the influence a drug molecule exerts leading to conformational changes in protein structure. A relatively high Rg value indicates a protein molecule with loose packing, whereas a smaller Rg value indicates a protein structure with tight packing. This analysis is also used to understand whether the drug molecules maintained the folding behaviour of protein or not.</p> <p>In the case of 3CLpro, both of the protein-drug complexes were found to be more compact than the unbound protein molecule (see <a href="#F0005">Figure 5c</a>). The average Rg value of 3CLpro, 3CLpro-Raltegravir complex and 3CLpro-Paritaprevir complex was found to be 2.165â€‰nm, 2.116â€‰nm and 2.144â€‰nm, respectively. We also observed a few minor variations in Rg, which could be attributed to the conformational shifts that resulted from the changes in the secondary structures of protein during the MD simulation.</p> <p>In the case of 2â€²-OMTase, both the protein-drug complexes were slightly less compact in comparison to the free 2â€²-OMTase. The free protein, as well as the protein-drug complexes, attained stability at 30â€‰ns and remained stable until the end (see <a href="#F0006">Figure 6c</a>). The average Rg values of 2â€²-OMTase, 2â€²-OMTase-Dolutegravir complex and 2â€²-OMTase-Bictegravir complex were found to be 1.796â€‰nm, 1.799â€‰nm and 1.805â€‰nm, respectively. Most of the fluctuations in the Rg values were recorded before 20â€‰ns which again supported that these drug molecules stayed strongly bound to the active site and they helped to maintain the stability and compactness of the protein structure.</p> </div> <div class="intermolecularh-bonding" title="sec"><span class="label" tagx="label" title="label">3.4.4.</span><div class="title" tagx="title" title="title">
Intermolecular H-bonding</div> <p>Hydrogen bonding is among the most crucial parameters to understand the binding affinity of drug molecules towards a protein molecule. Formation or deformation of H-bonds is a necessary consideration during MD simulations. A large number of H-bonds present in between protein and drug molecule signifies a strong binding affinity.</p> <p>In the case of 3CLpro, we observed the maximum number of hydrogen bonds in the 3CLpro-Paritaprevir complex (six intermolecular H-bonds) followed by 3CLpro-Raltegravir complex (five intermolecular H-bonds). The average value of intermolecular H-bonds were 2 for both 3CLpro-Paritaprevir complex and 3CLpro-Raltegravir complex, respectively (see <a href="#F0007">Figure 7</a>).This results explained that both the drug molecules interacted effectively towards the active site of 3CLpro with a significant number of hydrogen bonds.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7.</span></a></div>   <p>Intermolecular hydrogen bonds between the drugs and 3CLpro; 3CLpro-Paritaprevir complex (turquoise), 3CLpro-Raltegravir complex (Red).</p>   </div> <p>In the case of 2â€²-OMTase, we observed a maximum number of hydrogen bonds in the 2â€²-OMTase-Dolutegravir complex (six intermolecular H-bonds) followed by 2â€²-OMTase-Bictegravir complex (four intermolecular H-bonds). The average value of intermolecular H-bonds were 2 and 1 for 2â€²-OMTase-Dolutegravir complex and 2â€²-OMTase-Bictegravir complex, respectively (see <a href="#F0008">Figure 8</a>). From the results of intermolecular hydrogen bonding, Dolutegravir was found to be interacting with a higher number of hydrogen bonds towards 2â€²-OMTase than the other drug molecule.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8.</span></a></div>   <p>Intermolecular hydrogen bonds between the drugs and 2â€²-OMTase; 2â€²-OMTase-Bictegravir complex (Orange), 2â€²-OMTase-Dolutegravir complex (Dark Green).</p>   </div> </div> <div class="solventaccessiblesurfacearea(sasa)" title="sec"><span class="label" tagx="label" title="label">3.4.5.</span><div class="title" tagx="title" title="title">
Solvent accessible surface area (SASA)</div> <p>Additionally, we also performed a SASA analysis of all of the proteins and protein-drug complexes. This analysis is useful in understanding the solvent behaviour (Hydrophilic or Hydrophobic) of a protein molecule as well as protein-drug complexes.</p> <p>No major differences were observed in the SASA profiles of 3CLpro and its protein-drug complexes (see <a href="#F0005">Figure 5d</a>). The average values of SASA for the free 3CLpro, 3CLpro-Paritaprevir complex and 3CLpro-Raltegravir complex were 133.89â€‰nm<sup>2</sup>, 131.79â€‰nm<sup>2</sup> and 131.26â€‰nm<sup>2</sup>, respectively.</p> <p>The average SASA of 2â€²-OMTase, 2â€²-OMTase-Dolutegravir complex and 2â€²-OMTase-Bictegravir complex were found to be 126.77â€‰nm<sup>2</sup>, 130.116â€‰nm<sup>2</sup> and 131.03â€‰nm<sup>2</sup>, respectively. The results suggested that both of the proteins-drug complexes were impressively stable after the binding of drug molecules to their active sites (see <a href="#F0006">Figure 6d</a>).</p> </div> <div class="principalcomponentanalysis(pca)" title="sec"><span class="label" tagx="label" title="label">3.4.6.</span><div class="title" tagx="title" title="title">
Principal component analysis (PCA)</div> <p>PCA of MD simulations is a standard technique widely used to identify the essential motion of protein molecules. The PCA was employed to reveal the biologically significant and relevant motions from the global trajectories of unbound proteins as well as the protein-drug complexes using Essential Dynamics (ED) approach. ED analyses the combined fluctuations of the most unsteady regions of protein molecules into two variables, namely Principal Component 1 (PC1) and Principal Component 2 (PC2), which represents the majority of fluctuations observed in MD simulation.</p> <p>PCA graph of unbound 3CLpro, 3CLpro-Raltegravir complex and 3CLpro-Paritaprevir complex revealed that the protein molecule was slightly more flexible in the presence of both drug molecules (see <a href="#F0009">Figure 9</a>). The trace of covariance matrix for 3CLpro, 3CLpro-Raltegravir complex and 3CLpro-Paritaprevir complex was found to be 115.482â€‰nm<sup>2</sup>, 136.366â€‰nm<sup>2</sup> and 132.059â€‰nm<sup>2</sup>, respectively. The 3CLpro-Paritaprevir complex was found to be occupying lesser conformational space than the 3CLpro-Raltegravir complex. This result, along with previous analyses such as RMSD, RMSF, Rg and SASA additionally supported that Raltegravir and also Paritaprevir could be the most promising candidates which might be inhibiting the function of 3CLpro.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9.</span></a></div>   <p>Principal Component Analysis (PCA) of free 3CLpro (Black), 3CLpro-Paritaprevir complex (turquoise) and 3CLpro-Raltegravir complex (Red).</p>   </div> <p>PCA of free 2â€²-OMTase, 2â€²-OMTase-Dolutegravir complex and 2â€²-OMTase-Bictegravir complex depicted that the protein, as well as the protein-drug complexes, were occupying almost equal conformational space (see <a href="#F0010">Figure 10</a>). Trace of the covariance matrix for 2â€²-OMTase, 2â€²-OMTase-Dolutegravir complex and 2â€²-OMTase-Bictegravir complex was found to be 106.463â€‰nm<sup>2</sup>, 123.878â€‰nm<sup>2</sup> and 117.187â€‰nm<sup>2</sup>, respectively. Together with all of the MD simulation analysis, PCA supported that Dolutegravir and Bictegravir could be the most promising drugs capable of inhibiting the function of 2â€²-OMTase.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 10.</span></a></div>   <p>Principal Component Analysis (PCA) of free 2â€²-OMTase (Blue), 2â€²-OMTase-Dolutegravir complex (Dark Green) and 2â€²-OMTase-Bictegravir complex (Orange).</p>   </div> <p>The average values of the structural parameters including RMSD, RMSF, Rg, SASA, PCA and Intermolecular H-bonding has been described in <a href="#t0005">Table 5</a> (3CLpro) and <a href="#t0006">Table 6</a> (2â€²-OMTase).</p> <div class="table-wrap_UNKNOWN" id="t0005" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 5.</span> <p>Time averaged structural properties obtained from MD simulation of 3CLpro.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Analysis</th> <th align="center">3CLpro</th> <th align="center">3CLpro-Raltegrevir</th> <th align="center">3CLpro-Paritaprevir</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">Average RMSD (nm)</td> <td align="center">0.295</td> <td align="center">0.361</td> <td align="center">0.401</td> </tr> <tr> <td align="left">Average RMSF (nm)</td> <td align="center">0.170</td> <td align="center">0.182</td> <td align="center">0.184</td> </tr> <tr> <td align="left">Average Rg (nm)</td> <td align="center">2.165</td> <td align="center">2.116</td> <td align="center">2.144</td> </tr> <tr> <td align="left">PCA (Trace of Covariance Matrix nm<sup>2</sup>)</td> <td align="center">115.482</td> <td align="center">136.366</td> <td align="center">132.059</td> </tr> <tr> <td align="left">Average Intermolecular Hydrogen Bonds (Protein-Ligand)</td> <td align="center">â€"</td> <td align="center">2</td> <td align="center">2</td> </tr> <tr> <td align="left">Average SASA (nm<sup>2</sup>)</td> <td align="center">133.89</td> <td align="center">131.26</td> <td align="center">131.79</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="t0006" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 6.</span> <p>Time averaged structural properties obtained from MD simulation of 2â€²-OMTase.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Analysis</th> <th align="center">2â€²-OMTase</th> <th align="center">2â€²-OMTase-Dolutegravir</th> <th align="center">2â€²-OMTase-Bictegravir</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">Average RMSD (nm)</td> <td align="center">0.299</td> <td align="center">0.393</td> <td align="center">0.410</td> </tr> <tr> <td align="left">Average RMSF (nm)</td> <td align="center">0.155</td> <td align="center">0.161</td> <td align="center">0.165</td> </tr> <tr> <td align="left">Average Rg (nm)</td> <td align="center">1.796</td> <td align="center">1.799</td> <td align="center">1.805</td> </tr> <tr> <td align="left">PCA (Trace of Covariance Matrix nm<sup>2</sup>)</td> <td align="center">106.463</td> <td align="center">123.878</td> <td align="center">117.187</td> </tr> <tr> <td align="left">Average Intermolecular Hydrogen Bonds (Protein-Ligand)</td> <td align="center">â€"</td> <td align="center">2</td> <td align="center">1</td> </tr> <tr> <td align="left">Average SASA (nm<sup>2</sup>)</td> <td align="center">126.77</td> <td align="center">130.116</td> <td align="center">131.03</td> </tr> </tbody> </table> </div> </div> <div class="comparativeanalysisofpreandpostmdsimulation" title="sec"><span class="label" tagx="label" title="label">3.4.7.</span><div class="title" tagx="title" title="title">
Comparative analysis of pre and post MD simulation</div> <p>Structural superposition before and after the MD simulation of drug-free 3CLpro structures showed an RMSD of 2.41â€‰Ã…. It indicates that the MD simulated 3CLpro structure was consistent and geometrically well optimised, which did not deviate, in comparison to the initial protein structure. Moreover, we also performed the structural superimposition analysis of drug bounded 3CLpro structures which explained that the RMSD before and after MD simulation of 3CLpro-Raltegravir complex and 3CLpro-Paritaprevir complex was reliable and stable with the values of 2.15â€‰Ã… and 2.47â€‰Ã…, respectively (see <a href="#F0011">Figure 11</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 11.</span></a></div>   <p>Structural superimposition of the initial and MD simulated free 3CLpro, 3CLpro-Raltegravir complex and 3CLpro-Paritaprevir complex.</p>   </div> <p>Structural superposition of pre and post MD simulation of drug-free 2â€²-OMTase structures showed an RMSD of 2.115â€‰Ã…. Besides, we also carried out the same analysis for both drug bounded structures of 2â€²-OMTase. Also, we performed the same structural superimposition analysis of drug bounded structures 2â€²-OMTase, which explained that the RMSD before and after MD simulation of 2â€²-OMTase-Dolutegravir complex and 2â€²-OMTase-Bictegravir complex was reliable and stable with the values of 1.767â€‰Ã… and 2.568â€‰Ã…, respectively (see <a href="#F0012">Figure 12</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 12.</span></a></div>   <p>Structural superimposition of the initial and MD simulated free 2â€²-OMTase, 2â€²-OMTase-Bictegravir complex and 2â€²-OMTase-Dolutegravir complex.</p>   </div> <p>This is additionally supported that these two drugs were oriented in the active site of both proteins throughout the 100â€‰ns MD simulations.</p> </div> </div> </div> <div class="conclusion" title="sec"><span class="label" tagx="label" title="label">4.</span><div class="title" tagx="title" title="title">
Conclusion</div> <p>The current study aimed to screen pre-existing antiviral drugs and come up with potential antidotes against SARS-CoV-2. The advantages of drug repurposing (also known as drug repositioning, drug reprofiling and drug retasking) include: the risk of the drug failing the safety and toxicity tests is much reduced, reduced costs during clinical trials and finally, significantly reduced time scale required for drug development (Pushpakom etÂ al., <a href="#CIT0036">2019</a>). Developing novel drugs or vaccines against SARS-CoV-2 will demand a lot of time, which we cannot afford at present. SARS-CoV-2 is spreading at a rapid rate; therefore, it is rational to look for pre-existing drugs and formulations, which can control this highly contagious virus. The precautionary steps taken by governments all over the world include social distancing, quarantines and lockdowns, which themselves have adverse psychological and economic repercussions (Madhav etÂ al., <a href="#CIT0030">2017</a>). The WHO also recognises the vast potential of drug repurposing approach, and on 18 March 2020, they announced â€˜SOLIDARITYâ€™ (World Health Organization, <a href="#CIT0053">2020d</a>) global mega trial of the four most promising COVID-19 treatments. In the present study, we have identified two drugs against 3CLpro (Raltegravir and Paritaprevir) and two drugs against 2â€²-OMTase (Bictegravir and Dolutegravir), which are supposed to act as potential inhibitors against COVID-19. The proposed drug candidates obtained from this computational study could be efficacious in developing a much-needed threauptics against SARS-CoV-2, and it will further strengthen the concept of drug repurposing. However, being not able to replicate the physiological conditions entirely is one of the main drawbacks of computational biology (Gimeno etÂ al., <a href="#CIT0019">2019</a>; Marklund &amp;amp; Benesch, <a href="#CIT0031">2019</a>) and therefore, further <i>inÂ vivo/inÂ vitro</i> validations are required to confirm the findings of this study. Raltegravir (Markowitz etÂ al., 2007), Bictegravir (Sax etÂ al., 2017) and Dolutegravir (Cahn etÂ al., 2013; Walmsley etÂ al., <a href="#CIT0044">2013</a>) belong to the family of integrase strand transfer inhibitors (INSTI). They are already being used in antiretroviral therapy (ART) against the Human Immuno-deficiency Virus (HIV) infections. The final drug, Paritaprevir(Feld etÂ al., 2016; Sulkowski etÂ al., 2015), is being used to treat chronic Hepatitis C Virus (HCV) infections. It suppresses viral replication by inhibiting the non-structural 3/4â€‰A protease of HCV (Carrion etÂ al., <a href="#CIT0011">2014</a>). Finally, the findings of this study might help develop antiviral therapeutics against SARS-CoV-2, expediting medical countermeasure advancement and should serve to persuade researchers and physicians to strengthen their efforts on discovering novel uses for existing medications.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental data for this article can be accessed online at <a href="https://doi.org/10.1080/07391102.2020.1753577">https://doi.org/10.1080/07391102.2020.1753577</a>.</div>  <div class="media" title="media"><a href="TBSD_A_1753577_SM1882.docx">LINK</a></div> </div> </div> </div> <div class="back" title="back"> <div class="acknowledgements" title="sec"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Dr. Amit Kumar Singh thanks the Department of Science and Technology (DST) and Indian National Science Academy (INSA), New Delhi, India. Gizachew Muluneh Amera thanks the College of Natural Science, Wollo University, Dessie, Ethiopia for the sponsorship. The authors also thank to Supercomputing Facility for Bioinformatics &amp;amp; Computational Biology, IIT Delhi.</p> </div> <div class="disclosurestatement" title="sec"> <div class="title" tagx="title" title="title">
Disclosure statement</div> <p>No potential conflict of interest was reported by the authors.</p> </div> <div class="ethicalstandards" title="sec"> <div class="title" tagx="title" title="title">
Ethical standards</div> <p>Ethical standards are compulsory for studies relating to human and animal subjects.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CIT0001" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abraham</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindahl</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Spoel</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hess</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, &amp;amp; <span class="collab'">collab: the GROMACS development team</span></span> (<span class="year" tagx="year" title="year">2014</span>). GROMACS User Manual version, <i>5</i>(2), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">298</span>.</span></li> <li tag="ref"><a name="CIT0002" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ambure</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhat</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puzyn</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Identifying natural compounds as multi-target-directed ligands against Alzheimerâ€™s disease: An in silico approach</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="volume" tagx="volume" title="volume">37</span>(<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">1282</span>â€"<span class="lpage" tagx="lpage" title="lpage">1306</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2018.1456975">10.1080/07391102.2018.1456975</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29578387">29578387</a></span></span></li> <li tag="ref"><a name="CIT0003" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amera</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pathak</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jha</span><span class="given-names" tagx="given-names" title="given-names">R. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muthukumaran</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">A. K.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Screening of promising molecules against MurG as drug target in multi-drug-resistant-<i>Acinetobacter baumannii</i> â€" Insights from comparative protein modeling, molecular docking and molecular dynamics simulation</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">23</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2019.1700167">10.1080/07391102.2019.1700167</a></span></span></li> <li tag="ref"><a name="CIT0004" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amera</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pathak</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">A. K.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Structure based in-silico study on UDP-N-acetylmuramoyl-L-alanyl-D-glutamate-2,6-diaminopimelate ligase (MurE) from <i>Acinetobacter baumannii</i> as a drug target against nosocomial infections</span>. <span class="source" tagx="source" title="source"><i>Informatics in Medicine Unlocked</i></span>, <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">100216</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.imu.2019.100216">10.1016/j.imu.2019.100216</a></span></span></li> <li tag="ref"><a name="CIT0005" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benkert</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biasini</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwede</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Toward the estimation of the absolute quality of individual protein structure models</span>. <span class="source" tagx="source" title="source"><i>Bioinformatics</i></span>, <span class="volume" tagx="volume" title="volume">27</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">343</span>â€"<span class="lpage" tagx="lpage" title="lpage">350</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btq662">10.1093/bioinformatics/btq662</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21134891">21134891</a></span></span></li> <li tag="ref"><a name="CIT0006" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berendsen</span><span class="given-names" tagx="given-names" title="given-names">H. J. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grigera</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Straatsma</span><span class="given-names" tagx="given-names" title="given-names">T. P.</span></span></span> (<span class="year" tagx="year" title="year">1987</span>). <span class="mixed-article-title" title="mixed-article-title">The missing term in effective pair potentials</span>. <span class="source" tagx="source" title="source"><i>The Journal of Physical Chemistry</i></span>, <span class="volume" tagx="volume" title="volume">91</span>(<span class="issue" tagx="issue" title="issue">24</span>), <span class="fpage" tagx="fpage" title="fpage">6269</span>â€"<span class="lpage" tagx="lpage" title="lpage">6271</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1021/j100308a038">10.1021/j100308a038</a></span></span></li> <li tag="ref"><a name="CIT0007" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berman</span><span class="given-names" tagx="given-names" title="given-names">H. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Westbrook</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilliland</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhat</span><span class="given-names" tagx="given-names" title="given-names">T. N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weissig</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shindyalov</span><span class="given-names" tagx="given-names" title="given-names">I. N.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourne</span><span class="given-names" tagx="given-names" title="given-names">P. E.</span></span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">The Protein Data Bank</span>. <span class="source" tagx="source" title="source"><i>Nucleic Acids Research</i></span>, <span class="volume" tagx="volume" title="volume">28</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">235</span>â€"<span class="lpage" tagx="lpage" title="lpage">242</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/28.1.235">10.1093/nar/28.1.235</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592235">10592235</a></span></span></li> <li tag="ref"><a name="CIT0008" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bordoli</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiefer</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnold</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benkert</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Battey</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwede</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Protein structure homology modeling using SWISS-MODEL workspace</span>. <span class="source" tagx="source" title="source"><i>Nature Protocols</i></span>, <span class="volume" tagx="volume" title="volume">4</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">13</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1038/nprot.2008.197">10.1038/nprot.2008.197</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19131951">19131951</a></span></span></li> <li tag="ref"><a name="CIT0009" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cahn</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pozniak</span><span class="given-names" tagx="given-names" title="given-names">A. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mingrone</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shuldyakov</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brites</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andrade-Villanueva</span><span class="given-names" tagx="given-names" title="given-names">J. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richmond</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buendia</span><span class="given-names" tagx="given-names" title="given-names">C. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fourie</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramgopal</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hagins</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felizarta</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madruga</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reuter</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newman</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Small</span><span class="given-names" tagx="given-names" title="given-names">C. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lombaard</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grinsztejn</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dorey</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, â€¦ <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Min</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Dolutegravir versus raltegravir in antiretroviral-experienced, integrase-inhibitor-naive adults with HIV: Week 48 results from the randomised, double-blind, non-inferiority SAILING study</span>. <span class="source" tagx="source" title="source"><i>The Lancet</i></span>, <span class="volume" tagx="volume" title="volume">382</span>(<span class="issue" tagx="issue" title="issue">9893</span>), <span class="fpage" tagx="fpage" title="fpage">700</span>â€"<span class="lpage" tagx="lpage" title="lpage">708</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(13)61221-0">10.1016/S0140-6736(13)61221-0</a></span></span></li> <li tag="ref"><a name="CIT0010" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Camacho</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coulouris</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avagyan</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papadopoulos</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bealer</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madden</span><span class="given-names" tagx="given-names" title="given-names">T. L.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">BLAST+: Architecture and applications</span>. <span class="source" tagx="source" title="source"><i>BMC Bioinformatics</i></span>, <span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">421</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-10-421">10.1186/1471-2105-10-421</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20003500">20003500</a></span></span></li> <li tag="ref"><a name="CIT0011" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrion</span><span class="given-names" tagx="given-names" title="given-names">A. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gutierrez</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">New antiviral agents for the treatment of hepatitis C: ABT-450</span>. <span class="source" tagx="source" title="source"><i>Expert Opinion on Pharmacotherapy</i></span>, <span class="volume" tagx="volume" title="volume">15</span>(<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">711</span>â€"<span class="lpage" tagx="lpage" title="lpage">716</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1517/14656566.2014.889116">10.1517/14656566.2014.889116</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24517400">24517400</a></span></span></li> <li tag="ref"><a name="CIT0012" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: Centers for Disease Control and Prevention.</span></span> (<span class="year" tagx="year" title="year">2020</span>). Severe outcomes among patients with coronavirus disease 2019 (COVID-19) â€" United States, February 12â€"March 16, 2020.</span></li> <li tag="ref"><a name="CIT0013" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Only one protomer is active in the dimer of SARS 3C-like proteinase</span>. <span class="source" tagx="source" title="source"><i>Journal of Biological Chemistry</i></span>, <span class="volume" tagx="volume" title="volume">281</span>(<span class="issue" tagx="issue" title="issue">20</span>), <span class="fpage" tagx="fpage" title="fpage">13894</span>â€"<span class="lpage" tagx="lpage" title="lpage">13898</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M510745200">10.1074/jbc.M510745200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16565086">16565086</a></span></span></li> <li tag="ref"><a name="CIT0014" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ke</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tien</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahola</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Biochemical and structural insights into the mechanisms of SARS coronavirus RNA ribose 2â€™-O-methylation by nsp16/nsp10 protein complex</span>. <span class="source" tagx="source" title="source"><i>PLoS Pathogens</i></span>, <span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">10</span>), <span class="fpage" tagx="fpage" title="fpage">e1002294</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1002294">10.1371/journal.ppat.1002294</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22022266">22022266</a></span></span></li> <li tag="ref"><a name="CIT0015" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiu</span><span class="given-names" tagx="given-names" title="given-names">S.-W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandit</span><span class="given-names" tagx="given-names" title="given-names">S. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">H. L.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jakobsson</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">An improved united atom force field for simulation of mixed lipid bilayers</span>. <span class="source" tagx="source" title="source"><i>The Journal of Physical Chemistry B</i></span>, <span class="volume" tagx="volume" title="volume">113</span>(<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">2748</span>â€"<span class="lpage" tagx="lpage" title="lpage">2763</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1021/jp807056c">10.1021/jp807056c</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19708111">19708111</a></span></span></li> <li tag="ref"><a name="CIT0016" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colovos</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeates</span><span class="given-names" tagx="given-names" title="given-names">T. O.</span></span></span> (<span class="year" tagx="year" title="year">1993</span>). <span class="mixed-article-title" title="mixed-article-title">Verification of protein structures: Patterns of nonbonded atomic interactions</span>. <span class="source" tagx="source" title="source"><i>Protein Science</i></span>, <span class="volume" tagx="volume" title="volume">2</span>(<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">1511</span>â€"<span class="lpage" tagx="lpage" title="lpage">1519</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1002/pro.5560020916">10.1002/pro.5560020916</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8401235">8401235</a></span></span></li> <li tag="ref"><a name="CIT0017" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Wilde</span><span class="given-names" tagx="given-names" title="given-names">A. H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snijder</span><span class="given-names" tagx="given-names" title="given-names">E. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kikkert</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Hemert</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Host factors in coronavirus replication</span>. <span class="source" tagx="source" title="source"><i>Current Topics in Microbiology and Immunology</i></span>, <span class="volume" tagx="volume" title="volume">419</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">42</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1007/82_2017_25">10.1007/82_2017_25</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28643204">28643204</a></span></span></li> <li tag="ref"><a name="CIT0018" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feld</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moreno</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trinh</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tam</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourgeois</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horsmans</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elkhashab</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernstein</span><span class="given-names" tagx="given-names" title="given-names">D. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Younes</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reindollar</span><span class="given-names" tagx="given-names" title="given-names">R. W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larsen</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howieson</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Polepally</span><span class="given-names" tagx="given-names" title="given-names">A. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pangerl</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shulman</span><span class="given-names" tagx="given-names" title="given-names">N. S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poordad</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Sustained virologic response of 100% in HCV genotype 1b patients with cirrhosis receiving ombitasvir/paritaprevir/r and dasabuvir for 12weeks</span>. <span class="source" tagx="source" title="source"><i>Journal of Hepatology</i></span>, <span class="volume" tagx="volume" title="volume">64</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">301</span>â€"<span class="lpage" tagx="lpage" title="lpage">307</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jhep.2015.10.005">10.1016/j.jhep.2015.10.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26476290">26476290</a></span></span></li> <li tag="ref"><a name="CIT0019" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gimeno</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ojeda-Montes</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TomÃ¡s-HernÃ¡ndez</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cereto-MassaguÃ©</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BeltrÃ¡n-DebÃ³n</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mulero</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pujadas</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia-VallvÃ©</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">The light and dark sides of virtual screening: What is there to know?</span><span class="source" tagx="source" title="source"><i>International Journal of Molecular Sciences</i></span>, <span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">6</span>), <span class="fpage" tagx="fpage" title="fpage">1375</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms20061375">10.3390/ijms20061375</a></span></span></li> <li tag="ref"><a name="CIT0020" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hospital</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GoÃ±i</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orozco</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GelpÃ­</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular dynamics simulations: Advances and applications</span>. <span class="source" tagx="source" title="source"><i>Advances and Applications in Bioinformatics and Chemistry</i></span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">37</span>â€"<span class="lpage" tagx="lpage" title="lpage">47</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.2147/AABC.S70333">10.2147/AABC.S70333</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26604800">26604800</a></span></span></li> <li tag="ref"><a name="CIT0021" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, â€¦ <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <span class="source" tagx="source" title="source"><i>The Lancet</i></span>, <span class="volume" tagx="volume" title="volume">395</span>(<span class="issue" tagx="issue" title="issue">10223</span>), <span class="fpage" tagx="fpage" title="fpage">497</span>â€"<span class="lpage" tagx="lpage" title="lpage">506</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30183-5">10.1016/S0140-6736(20)30183-5</a></span></span></li> <li tag="ref"><a name="CIT0022" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: Johns Hopkins CSSE.</span></span> (<span class="year" tagx="year" title="year">2020</span>). <i>Coronavirus COVID-19 global cases by Johns Hopkins CSSE</i> Retrieved March 28, 2020 from <a href="https://gisanddata.maps.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6">https://gisanddata.maps.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6</a></span></li> <li tag="ref"><a name="CIT0023" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitchen</span><span class="given-names" tagx="given-names" title="given-names">D. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Decornez</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furr</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bajorath</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Docking and scoring in virtual screening for drug discovery: Methods and applications</span>. <span class="source" tagx="source" title="source"><i>Nature Reviews Drug Discovery</i></span>, <span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">11</span>), <span class="fpage" tagx="fpage" title="fpage">935</span>â€"<span class="lpage" tagx="lpage" title="lpage">949</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1038/nrd1549">10.1038/nrd1549</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15520816">15520816</a></span></span></li> <li tag="ref"><a name="CIT0024" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ko</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heo</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seok</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">GalaxyWEB server for protein structure prediction and refinement</span>. <span class="source" tagx="source" title="source"><i>Nucleic Acids Research</i></span>, <span class="volume" tagx="volume" title="volume">40</span>(<span class="issue" tagx="issue" title="issue">W1</span>), <span class="fpage" tagx="fpage" title="fpage">W294</span>â€"<span class="lpage" tagx="lpage" title="lpage">W297</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gks493">10.1093/nar/gks493</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22649060">22649060</a></span></span></li> <li tag="ref"><a name="CIT0025" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krieger</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joo</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tyka</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karplus</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Improving physical realism, stereochemistry, and side-chain accuracy in homology modeling: Four approaches that performed well in CASP8</span>. <span class="source" tagx="source" title="source"><i>Proteins: Structure, Function, and Bioinformatics</i></span>, <span class="volume" tagx="volume" title="volume">77</span>(<span class="issue" tagx="issue" title="issue">S9</span>), <span class="fpage" tagx="fpage" title="fpage">114</span>â€"<span class="lpage" tagx="lpage" title="lpage">122</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1002/prot.22570">10.1002/prot.22570</a></span></span></li> <li tag="ref"><a name="CIT0026" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laskowski</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">JabÅ‚oÅ"ska</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pravda</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VaÅ™ekovÃ¡</span><span class="given-names" tagx="given-names" title="given-names">R. S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thornton</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">PDBsum: Structural summaries of PDB entries</span>. <span class="source" tagx="source" title="source"><i>Protein Science.</i></span>, <span class="volume" tagx="volume" title="volume">27</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">129</span>â€"<span class="lpage" tagx="lpage" title="lpage">134</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1002/pro.3289">10.1002/pro.3289</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28875543">28875543</a></span></span></li> <li tag="ref"><a name="CIT0027" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laskowski</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacArthur</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moss</span><span class="given-names" tagx="given-names" title="given-names">D. S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thornton</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span></span> (<span class="year" tagx="year" title="year">1993</span>). <span class="mixed-article-title" title="mixed-article-title">PROCHECK: A program to check the stereochemical quality of protein structures</span>. <span class="source" tagx="source" title="source"><i>Journal of Applied Crystallography</i></span>, <span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">283</span>â€"<span class="lpage" tagx="lpage" title="lpage">291</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1107/S0021889892009944">10.1107/S0021889892009944</a></span></span></li> <li tag="ref"><a name="CIT0028" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lugari</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Betzi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Decroly</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonnaud</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hermant</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guillemot</span><span class="given-names" tagx="given-names" title="given-names">J.-C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debarnot</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borg</span><span class="given-names" tagx="given-names" title="given-names">J.-P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouvet</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Canard</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morelli</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LÃ©cine</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular mapping of the RNA cap 2â€™-O-methyltransferase activation interface between severe acute respiratory syndrome coronavirus nsp10 and nsp16</span>. <span class="source" tagx="source" title="source"><i>Journal of Biological Chemistry</i></span>, <span class="volume" tagx="volume" title="volume">285</span>(<span class="issue" tagx="issue" title="issue">43</span>), <span class="fpage" tagx="fpage" title="fpage">33230</span>â€"<span class="lpage" tagx="lpage" title="lpage">33241</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M110.120014">10.1074/jbc.M110.120014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20699222">20699222</a></span></span></li> <li tag="ref"><a name="CIT0029" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luthy</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowie</span><span class="given-names" tagx="given-names" title="given-names">J. U.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eisenberg</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">1992</span>). <span class="mixed-article-title" title="mixed-article-title">Assessment of protein models with three-dimensional profiles</span>. <span class="source" tagx="source" title="source"><i>Nature</i></span>, <span class="volume" tagx="volume" title="volume">356</span>(<span class="issue" tagx="issue" title="issue">6364</span>), <span class="fpage" tagx="fpage" title="fpage">83</span>â€"<span class="lpage" tagx="lpage" title="lpage">85</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1038/356083a0">10.1038/356083a0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1538787">1538787</a></span></span></li> <li tag="ref"><a name="CIT0030" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madhav</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oppenheim</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gallivan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="source" tagx="source" title="source"><i>Disease control priorities: Improving health and reducing poverty</i></span> (<span class="edition" tagx="edition" title="edition">3rd ed</span>). <span class="publisher-name" tagx="publisher-name" title="publisher-name">The International Bank for Reconstruction and Development</span>.</span></li> <li tag="ref"><a name="CIT0031" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marklund</span><span class="given-names" tagx="given-names" title="given-names">E. G.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benesch</span><span class="given-names" tagx="given-names" title="given-names">J. L. P.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Weighing-up protein dynamics: The combination of native mass spectrometry and molecular dynamics simulations</span>. <span class="source" tagx="source" title="source"><i>Current Opinion in Structural Biology</i></span>, <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">50</span>â€"<span class="lpage" tagx="lpage" title="lpage">58</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.sbi.2018.12.011">10.1016/j.sbi.2018.12.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30743182">30743182</a></span></span></li> <li tag="ref"><a name="CIT0032" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markowitz</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">B.-Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gotuzzo</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mendo</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratanasuwan</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kovacs</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prada</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morales-Ramirez</span><span class="given-names" tagx="given-names" title="given-names">J. O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crumpacker</span><span class="given-names" tagx="given-names" title="given-names">C. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Isaacs</span><span class="given-names" tagx="given-names" title="given-names">R. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilde</span><span class="given-names" tagx="given-names" title="given-names">L. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">M. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wenning</span><span class="given-names" tagx="given-names" title="given-names">L. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teppler</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Rapid and durable antiretroviral effect of the HIV-1 integrase inhibitor raltegravir as part of combination therapy in treatment-naive patients with HIV-1 infection: Results of a 48-week controlled study</span>. <span class="source" tagx="source" title="source"><i>JAIDS Journal of Acquired Immune Deficiency Syndromes</i></span>, <span class="volume" tagx="volume" title="volume">46</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">125</span>â€"<span class="lpage" tagx="lpage" title="lpage">133</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1097/QAI.0b013e318157131c">10.1097/QAI.0b013e318157131c</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17721395">17721395</a></span></span></li> <li tag="ref"><a name="CIT0033" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menachery</span><span class="given-names" tagx="given-names" title="given-names">V. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debbink</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baric</span><span class="given-names" tagx="given-names" title="given-names">R. S.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Coronavirus non-structural protein 16: Evasion, attenuation, and possible treatments</span>. <span class="source" tagx="source" title="source"><i>Virus Research</i></span>, <span class="volume" tagx="volume" title="volume">194</span>, <span class="fpage" tagx="fpage" title="fpage">191</span>â€"<span class="lpage" tagx="lpage" title="lpage">199</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2014.09.009">10.1016/j.virusres.2014.09.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25278144">25278144</a></span></span></li> <li tag="ref"><a name="CIT0034" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menachery</span><span class="given-names" tagx="given-names" title="given-names">V. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yount</span><span class="given-names" tagx="given-names" title="given-names">B. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Josset</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gralinski</span><span class="given-names" tagx="given-names" title="given-names">L. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scobey</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agnihothram</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katze</span><span class="given-names" tagx="given-names" title="given-names">M. G.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baric</span><span class="given-names" tagx="given-names" title="given-names">R. S.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Attenuation and restoration of severe acute respiratory syndrome coronavirus mutant lacking 2â€™-o-methyltransferase activity</span>. <span class="source" tagx="source" title="source"><i>Journal of Virology</i></span>, <span class="volume" tagx="volume" title="volume">88</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">4251</span>â€"<span class="lpage" tagx="lpage" title="lpage">4264</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.03571-13">10.1128/JVI.03571-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24478444">24478444</a></span></span></li> <li tag="ref"><a name="CIT0035" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Micheletti</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Comparing proteins by their internal dynamics: Exploring structure-function relationships beyond static structural alignments</span>. <span class="source" tagx="source" title="source"><i>Physics of Life Reviews</i></span>, <span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">26</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.plrev.2012.10.009">10.1016/j.plrev.2012.10.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23199577">23199577</a></span></span></li> <li tag="ref"><a name="CIT0036" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pushpakom</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iorio</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eyers</span><span class="given-names" tagx="given-names" title="given-names">P. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Escott</span><span class="given-names" tagx="given-names" title="given-names">K. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hopper</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wells</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doig</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guilliams</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Latimer</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McNamee</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Norris</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanseau</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cavalla</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pirmohamed</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Drug repurposing: Progress, challenges and recommendations</span>. <span class="source" tagx="source" title="source"><i>Nature Reviews Drug Discovery</i></span>, <span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">41</span>â€"<span class="lpage" tagx="lpage" title="lpage">58</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1038/nrd.2018.168">10.1038/nrd.2018.168</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30310233">30310233</a></span></span></li> <li tag="ref"><a name="CIT0037" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramachandran</span><span class="given-names" tagx="given-names" title="given-names">G. N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramakrishnan</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasisekharan</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">1963</span>). <span class="mixed-article-title" title="mixed-article-title">Stereochemistry of polypeptide chain configurations</span>. <span class="source" tagx="source" title="source"><i>Journal of Molecular Biology</i></span>, <span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">95</span>â€"<span class="lpage" tagx="lpage" title="lpage">99</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/S0022-2836(63)80023-6">10.1016/S0022-2836(63)80023-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13990617">13990617</a></span></span></li> <li tag="ref"><a name="CIT0038" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sax</span><span class="given-names" tagx="given-names" title="given-names">P. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pozniak</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montes</span><span class="given-names" tagx="given-names" title="given-names">M. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koenig</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeJesus</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stellbrink</span><span class="given-names" tagx="given-names" title="given-names">H.-J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antinori</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Workowski</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slim</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reynes</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garner</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Custodio</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SenGupta</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quirk</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Coformulated bictegravir, emtricitabine, and tenofovir alafenamide versus dolutegravir with emtricitabine and tenofovir alafenamide, for initial treatment of HIV-1 infection (GS-US-380â€"1490): A randomised, double-blind, multicentre, phase 3, non-inferiority trial</span>. <span class="source" tagx="source" title="source"><i>The Lancet</i></span>, <span class="volume" tagx="volume" title="volume">390</span>(<span class="issue" tagx="issue" title="issue">10107</span>), <span class="fpage" tagx="fpage" title="fpage">2073</span>â€"<span class="lpage" tagx="lpage" title="lpage">2082</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(17)32340-1">10.1016/S0140-6736(17)32340-1</a></span></span></li> <li tag="ref"><a name="CIT0039" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schuttelkopf</span><span class="given-names" tagx="given-names" title="given-names">A. W.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Aalten</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">PRODRG: A tool for high-throughput crystallography of protein-ligand complexes</span>. <span class="source" tagx="source" title="source"><i>Acta Crystallographica Section D Biological Crystallography</i></span>, <span class="volume" tagx="volume" title="volume">60</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">1355</span>â€"<span class="lpage" tagx="lpage" title="lpage">1363</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1107/S0907444904011679">10.1107/S0907444904011679</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15272157">15272157</a></span></span></li> <li tag="ref"><a name="CIT0040" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">A. C. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">G. F.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Epidemiology, genetic recombination, and pathogenesis of coronaviruses</span>. <span class="source" tagx="source" title="source"><i>Trends in Microbiology.</i></span>, <span class="volume" tagx="volume" title="volume">24</span>(<span class="issue" tagx="issue" title="issue">6</span>), <span class="fpage" tagx="fpage" title="fpage">490</span>â€"<span class="lpage" tagx="lpage" title="lpage">502</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2016.03.003">10.1016/j.tim.2016.03.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27012512">27012512</a></span></span></li> <li tag="ref"><a name="CIT0041" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sulkowski</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eron</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wyles</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trinh</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lalezari</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slim</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhatti</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gathe</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruane</span><span class="given-names" tagx="given-names" title="given-names">P. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elion</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bredeek</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brennan</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blick</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khatri</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbons</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fredrick</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnell</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, â€¦ <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Podsadecki</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Ombitasvir, paritaprevir co-dosed with ritonavir, dasabuvir, and ribavirin for hepatitis C in patients co-infected with HIV-1: A randomized trial</span>. <span class="source" tagx="source" title="source"><i>JAMA</i></span>, <span class="volume" tagx="volume" title="volume">313</span>(<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">1223</span>â€"<span class="lpage" tagx="lpage" title="lpage">1231</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1001/jama.2015.1328">10.1001/jama.2015.1328</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25706092">25706092</a></span></span></li> <li tag="ref"><a name="CIT0042" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trott</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olson</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span>. <span class="source" tagx="source" title="source"><i>Journal of Computational Chemistry</i></span>, <span class="volume" tagx="volume" title="volume">31</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">455</span>â€"<span class="lpage" tagx="lpage" title="lpage">461</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.21334">10.1002/jcc.21334</a></span></span></li> <li tag="ref"><a name="CIT0043" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: UCLA MBI</span></span><i>The structure analysis and verification server</i> Retrieved from <a href="https://servicesn.mbi.ucla.edu/SAVES/">https://servicesn.mbi.ucla.edu/SAVES/</a></span></li> <li tag="ref"><a name="CIT0044" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walmsley</span><span class="given-names" tagx="given-names" title="given-names">S. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antela</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clumeck</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duiculescu</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eberhard</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GutiÃ©rrez</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hocqueloux</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maggiolo</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sandkovsky</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Granier</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pappa</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wynne</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Min</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nichols</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Dolutegravir plus abacavirâ€"lamivudine for the treatment of HIV-1 infection</span>. <span class="source" tagx="source" title="source"><i>New England Journal of Medicine</i></span>, <span class="volume" tagx="volume" title="volume">369</span>(<span class="issue" tagx="issue" title="issue">19</span>), <span class="fpage" tagx="fpage" title="fpage">1807</span>â€"<span class="lpage" tagx="lpage" title="lpage">1818</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMoa1215541">10.1056/NEJMoa1215541</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24195548">24195548</a></span></span></li> <li tag="ref"><a name="CIT0045" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waterhouse</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertoni</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bienert</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Studer</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tauriello</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gumienny</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heer</span><span class="given-names" tagx="given-names" title="given-names">F. T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Beer</span><span class="given-names" tagx="given-names" title="given-names">T. A. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rempfer</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bordoli</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lepore</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwede</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">SWISS-MODEL: Homology modelling of protein structures and complexes</span>. <span class="source" tagx="source" title="source"><i>Nucleic Acids Research</i></span>, <span class="volume" tagx="volume" title="volume">46</span>(<span class="issue" tagx="issue" title="issue">W1</span>), <span class="fpage" tagx="fpage" title="fpage">W296</span>â€"<span class="lpage" tagx="lpage" title="lpage">W303</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gky427">10.1093/nar/gky427</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29788355">29788355</a></span></span></li> <li tag="ref"><a name="CIT0046" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiss</span><span class="given-names" tagx="given-names" title="given-names">S. R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leibowitz</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Coronavirus pathogenesis</span>. <span class="source" tagx="source" title="source"><i>Advances in Virus Research</i></span>, <span class="volume" tagx="volume" title="volume">81</span>, <span class="fpage" tagx="fpage" title="fpage">85</span>â€"<span class="lpage" tagx="lpage" title="lpage">164</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/b978-0-12-385885-6.00009-2">10.1016/b978-0-12-385885-6.00009-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22094080">22094080</a></span></span></li> <li tag="ref"><a name="CIT0047" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wishart</span><span class="given-names" tagx="given-names" title="given-names">D. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feunang</span><span class="given-names" tagx="given-names" title="given-names">Y. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">A. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lo</span><span class="given-names" tagx="given-names" title="given-names">E. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marcu</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grant</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sajed</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sayeeda</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Assempour</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iynkkaran</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maciejewski</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gale</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chin</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cummings</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, â€¦ <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">DrugBank 5.0: A major update to the DrugBank database for 2018</span>. <span class="source" tagx="source" title="source"><i>Nucleic Acids Research</i></span>, <span class="volume" tagx="volume" title="volume">46</span>(<span class="issue" tagx="issue" title="issue">D1</span>), <span class="fpage" tagx="fpage" title="fpage">D1074</span>â€"<span class="lpage" tagx="lpage" title="lpage">D1082</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkx1037">10.1093/nar/gkx1037</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29126136">29126136</a></span></span></li> <li tag="ref"><a name="CIT0048" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: World Health Organization</span></span> (<span class="year" tagx="year" title="year">2003</span>, December). <i>Summary of probable SARS cases with onset of illness from 1 November 2002 to 31 July 2003</i> Retrieved March 28, 2020 from <a href="http://www.who.int/csr/sars/country/table2004_04_21/en/index.html">http://www.who.int/csr/sars/country/table2004_04_21/en/index.html</a></span></li> <li tag="ref"><a name="CIT0049" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: World Health Organization. (</span></span><span class="year" tagx="year" title="year">2019</span>, November). <i>Middle East respiratory syndrome coronavirus (MERS-CoV)</i> Retrieved March 28, 2020 from <a href="https://www.who.int/emergencies/mers-cov/en/">https://www.who.int/emergencies/mers-cov/en/</a></span></li> <li tag="ref"><a name="CIT0050" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: World Health Organization</span></span> (<span class="year" tagx="year" title="year">2020a</span>). <i>Naming the coronavirus disease (COVID-19) and the virus that causes it</i> Retrieved March 28, 2020 from <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/naming-the-coronavirus-disease-(covid-2019)-and-the-virus-that-causes-it">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/naming-the-coronavirus-disease-(covid-2019)-and-the-virus-that-causes-it</a></span></li> <li tag="ref"><a name="CIT0051" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: World Health Organization</span></span> (<span class="year" tagx="year" title="year">2020b</span>). <i>WHO Director-Generalâ€™s opening remarks at the media briefing on COVID-19 â€" 3 March 2020</i> Retrieved March 28, 2020 from <a href="https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19%E2%80%943-march-2020">https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19â€"3-march-2020</a></span></li> <li tag="ref"><a name="CIT0052" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: World Health Organization.</span></span> (<span class="year" tagx="year" title="year">2020c</span>). <span class="mixed-article-title" title="mixed-article-title">WHO Director-Generalâ€™s opening remarks at the media briefing on COVID-19 â€" 11 March 2020</span>. Retrieved March 28, 2020 from <a href="https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19%E2%80%9411-march-2020">https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19â€"11-march-2020</a></span></li> <li tag="ref"><a name="CIT0053" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: World Health Organization.</span></span> (<span class="year" tagx="year" title="year">2020</span>d). WHO Director-Generalâ€™s opening remarks at the media briefing on COVID-19 â€" 18 March 2020. Retrieved March 28, 2020.</span></li> <li tag="ref"><a name="CIT0054" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: World Health Organization</span></span> (<span class="year" tagx="year" title="year">2020</span>e). <i>Statement on the second meeting of the International Health Regulations (2005) Emergency Committee regarding the outbreak of novel coronavirus (2019-nCoV)</i> Retrieved March 28, 2020 from <a href="https://www.who.int/news-room/detail/30-01-2020-statement-on-the-second-meeting-of-the-international-health-regulations-(2005)-emergency-committee-regarding-the-outbreak-of-novel-coronavirus-(2019-ncov">https://www.who.int/news-room/detail/30-01-2020-statement-on-the-second-meeting-of-the-international-health-regulations-(2005)-emergency-committee-regarding-the-outbreak-of-novel-coronavirus-(2019-ncov</a>)</span></li> <li tag="ref"><a name="CIT0055" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lou</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mo</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">G. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anand</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartlam</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilgenfeld</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">The crystal structures of severe acute respiratory syndrome virus main protease and its complex with an inhibitor</span>. <span class="source" tagx="source" title="source"><i>Proceedings of the National Academy of Sciences of Sciences</i></span>, <span class="volume" tagx="volume" title="volume">100</span>(<span class="issue" tagx="issue" title="issue">23</span>), <span class="fpage" tagx="fpage" title="fpage">13190</span>â€"<span class="lpage" tagx="lpage" title="lpage">13195</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1835675100">10.1073/pnas.1835675100</a></span></span></li> <li tag="ref"><a name="CIT0056" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhan</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">G. F.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">A novel coronavirus from patients with pneumonia in China, 2019</span>. <span class="source" tagx="source" title="source"><i>New England Journal of Medicine</i></span>, <span class="volume" tagx="volume" title="volume">382</span>, <span class="fpage" tagx="fpage" title="fpage">727</span>â€"<span class="lpage" tagx="lpage" title="lpage">733</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMoa2001017">10.1056/NEJMoa2001017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31978945">31978945</a></span></span></li> </ul> </div> <div class="glossary" title="glossary"> <div class="def-list" title="def-list"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-item" title="def-item"> <div class="term" title="term">
3CLpro</div> <div class="def" title="def"> <p>3C-like proteinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
2â€²-O-MTase</div> <div class="def" title="def"> <p>2â€²-O-ribose methyltransferase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SARS-CoV-2</div> </div> </div> </div> </div>  </body></html>