<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Biomol Struct Dyn</span><span class="iso-abbrev" title="iso-abbrev">J. Biomol. Struct. Dyn</span><span class="publisher-id" title="publisher-id">TBSD</span><span class="publisher-id" title="publisher-id">tbsd20</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Biomolecular Structure &amp;amp; Dynamics</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0739-1102</span><span class="issn-epub" tagx="issn" title="issn-epub">1538-0254</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Taylor &amp;amp; Francis</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1080/07391102.2020.1778536">10.1080/07391102.2020.1778536</a></span><span class="publisher-id" title="publisher-id">1778536</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Virtual screening and dynamics of potential inhibitors targeting RNA binding domain of nucleocapsid phosphoprotein from SARS-CoV-2</div> <div class="alt-title" title="alt-title">
R. Yadav et al.</div> <div class="alt-title" title="alt-title">
Journal of Biomolecular Structure and Dynamics</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yadav</span><span class="given-names" tagx="given-names" title="given-names">Rohitash</span></span><a href="#AF0001"><sup>a</sup></a><a href="#FN0002"><sup>#</sup></a><a href="#AN0001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imran</span><span class="given-names" tagx="given-names" title="given-names">Mohammed</span></span><a href="#AF0002"><sup>b</sup></a><a href="#FN0002"><sup>#</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dhamija</span><span class="given-names" tagx="given-names" title="given-names">Puneet</span></span><a href="#AF0001"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchal</span><span class="given-names" tagx="given-names" title="given-names">Kapil</span></span><a href="#AF0003"><sup>c</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Handu</span><span class="given-names" tagx="given-names" title="given-names">Shailendra</span></span><a href="#AF0001"><sup>a</sup></a></span><span class="citation_author_institution" id="AF0001">[a], <span class="institution" title="institution">Department of Pharmacology, All India Institute of Medical Sciences</span><span class="country" tagx="country" title="country">India</span></span><span class="citation_author_institution" id="AF0002">[b], <span class="institution" title="institution">Department of Pharmacology, College of Medicine, Shaqra University</span><span class="country" tagx="country" title="country">Kingdom of Saudi Arabia</span></span><span class="citation_author_institution" id="AF0003">[c], <span class="institution" title="institution">School of Medicine, Johns Hopkins University</span><span class="country" tagx="country" title="country">USA</span></span></div> <div class="author-notes" title="author-notes"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">#</span><p>These authors contributed equally to this work.</p> </div> <div class="fn-type-" title=""> <p>Supplemental data for this article can be accessed online at <a href="https://doi.org/10.1080/07391102.2020.1778536">https://doi.org/10.1080/07391102.2020.1778536</a>.</p> </div> <div class="corresp" title="corresp">
CONTACT Rohitash Yadav <span class="email" tagx="email" title="email">rohitashyadav1@gmail.com</span><span class="institution" title="institution">Department of Pharmacology, All India Institute of Medical Sciences</span>, Rishikesh, <span class="country" tagx="country" title="country">India</span></div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-6-6</span></span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">16</span><span class="history" title="history"><span class="received" title="received">received: 2020-5-08</span><span class="accepted" title="accepted">accepted: 2020-5-29</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Informa UK Limited, trading as Taylor &amp;amp; Francis Group</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span></div><a class="self-uri" href="TBSD_A_1778536.pdf" title="self-uri">TBSD_A_1778536.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>The emergence of the coronavirus disease-2019 pandemic has led to an outbreak in the world. The SARS-CoV-2 is seventh and latest in coronavirus family with unique exonucleases for repairing any mismatches in newly transcribed genetic material. Therefore, drugs with novel additional mechanisms are required to simultaneously target and eliminate the virus. Thus, a newly deciphered N protein is taken as a target that belongs to SARS-CoV-2. They play a vital role in RNA transcription, viral replication and new virion formation. This study used virtual screening, molecular modeling and docking of the 8987 ligands from Asinex and PubChem databases against this novel target protein. Three hotspot sites having DScore â‰¥1 (Site 1, Site 2 and Site 3) for ligand binding were selected. Subsequently, high throughput screening, standard precision and extra precision docking process and molecular dynamics concluded three best drugs from two libraries. Two antiviral moieties from Asinex databases (5817 and 6799) have docking scores of âˆ’10.29 and âˆ’10.156; along with their respective free binding energies (Î"<i>G</i> bind) of âˆ’51.96 and âˆ’64.36 on Site 3. The third drug, Zidovudine, is from PubChem database with docking scores of âˆ’9.75 with its binding free energies (Î"<i>G</i> bind) of âˆ’59.43 on Site 3. The RMSD and RMSF were calculated for all the three drugs through molecular dynamics simulation studies for 50â€‰ns. Zidovudine shows a very stable interaction with fluctuation starting at 2.4â€‰Ã… on 2â€‰ns and remained stable at 3â€‰Ã… from 13 to 50â€‰ns. Thus, paving the way for further biological validation as a potential treatment.</p> <p>Communicated by Ramaswamy H. Sarma</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
COVID-19</div> <div class="kwd" title="kwd">
docking</div> <div class="kwd" title="kwd">
molecular dynamics</div> <div class="kwd" title="kwd">
nucleocapsid phosphoprotein</div> <div class="kwd" title="kwd">
SARS-CoV-2</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1.</span><div class="title" tagx="title" title="title">
Introduction</div> <p>Coronaviruses (CoV) are the positively single stranded RNA enveloped viruses having spherical or pleomorphic shape with club shaped glycoproteins on their outer surface (Fehr &amp;amp; Perlman, <a href="#CIT0017">2015</a>). Thus, deriving its name from a Latin word meaning â€˜crown or haloâ€™ (Paules etÂ al., 2020). There are seven recognized strains of these viruses such as 229E, NL63, OC43, HKU1, MERS-CoV, SARS-CoV-1, and the latest and novel SARS-CoV-2 which is responsible for the current outbreak (Zhu etÂ al., <a href="#CIT0042">2020</a>). The last two epidemics due to SARS-CoV-1 and MERS-CoV were associated with 10% and 37% mortality, respectively (Huang etÂ al., <a href="#CIT0025">2020</a>). The SARS-CoV-2 is causing the COVID-19 disease with variable mortality in different countries of almost all continents. It has been declared as the pandemic by World Health Organization (WHO) (Eurosurveillance, <a href="#CIT0040">2020</a>). It directly affects the respiratory tract leading to the Acute Respiratory Distress Syndrome (ARDS) and death in the want of oxygen (Chen etÂ al., <a href="#CIT0008">2020</a>).</p> <p>There are four structural proteins in CoV, including trimeric spike (S glycoprotein) responsible for attachment, fusion and entry with the help of angiotensin converting enzyme-2 (ACE2) on the human cell (Hasan etÂ al., <a href="#CIT0024">2020</a>). A pentameric small envelope protein (E protein) regulates ion channel activity (Gupta etÂ al., <a href="#CIT0021">2020</a>). The others are matrix protein (M protein) and nucleocapsid protein (N protein) for virion formation and replication (Brian &amp;amp; Baric, <a href="#CIT0005">2005</a>). Along with these some main proteases (M<sup>pro</sup>) are also present (Boopathi etÂ al., <a href="#CIT0003">2020</a>). In addition to that, unlike other RNA viruses, it has exonucleases to repair mismatches in newly transcribed double stranded RNA and provide resistance to many of the therapies (Shannon etÂ al., <a href="#CIT0039">2020</a>). So drugs with additional mechanisms are required to target the several viral proteins simultaneously. The nucleocapsid N proteins are present in large amount as structural protein and they are the least variable and highly conserved structure of CoV (Lin etÂ al., <a href="#CIT0028">2014</a>). The N-terminal Domain (NTD), the C-Terminal Domains (CTD) and a central Ser/Arg (SR) rich linker are three main domains of N proteins which are capable of binding to the RNA, oligomerization and phosphorylation, respectively (Chang etÂ al., <a href="#CIT0007">2013</a>; Lo etÂ al., <a href="#CIT0030">2013</a>). These bind with the RNA genome to form ribonucleoprotein complex necessary to produce virion core and RNA synthesis for viral replication (Cong etÂ al., <a href="#CIT0009">2019</a>; McBride etÂ al., 2014). Furthermore, studies have suggested their role in regulating hostâ€"pathogen interactions such as reorganizations of actin filaments, progression of the host cell cycle, and finally, apoptosis of their cells (Du etÂ al., <a href="#CIT0012">2008</a>; Surjit etÂ al., <a href="#CIT0041">2006</a>).</p> <p>The structure of nucleocapsid phosphoproteins is just like an overall right hand like fold with an extended central loop divided into an area called palm and fingers (e.g. 6M3M and 6VYO) (Kang etÂ al., <a href="#CIT0027">2020</a>). One of the fingers is having more basic amino acid residues and therefore called as basic finger. So instead of fewer location based on NTD and CTD, the protein binds with the basic amino acid residue moieties significantly with (A50, T57, H59, R92, I94, S105, R107, R149, Y172) which are located in the basic finger and its junction with the palm. The studies suggest that the suitable target is the hot spot located at the interface of basic finger and the palm (Dinesh etÂ al., <a href="#CIT0011">2020</a>).</p> <p>All the SARS-CoV-2 proteins are potential drug targets thus inhibiting N-proteins will impair the viral replication machinery (Gordon etÂ al., <a href="#CIT0020">2020</a>). Many studies have been conducted with different targets of this virus such as RNA-dependent RNA polymerase (Elfiky, <a href="#CIT0014">2020b</a>), Spike protein/ACE2 (Hasan etÂ al., <a href="#CIT0024">2020</a>) and main proteases (Elmezayen etÂ al., <a href="#CIT0015">2020</a>; Pant etÂ al., <a href="#CIT0035">2020</a>). Others have been conducted to find the potential treatments like phytochemicals (Aanouz etÂ al., <a href="#CIT0001">2020</a>; Elfiky, <a href="#CIT0013">2020a</a>; Enmozhi etÂ al., <a href="#CIT0016">2020</a>; Islam etÂ al., <a href="#CIT0026">2020</a>). However, because of the lack of safety and toxicity studies, they will take time for further validation and clinical trials. Therefore, designing and repurposing the drugs from FDA approved drug for inhibiting N protein binding with RNA domains will lead to the inhibition of viral replication and subsequent virion formation of SARS-CoV-2.</p> </div> <div class="materialandmethods" title="sec"><span class="label" tagx="label" title="label">2.</span><div class="title" tagx="title" title="title">
Material and methods</div> <div class="targetselectionandmolecularlibrarypreparation" title="sec"><span class="label" tagx="label" title="label">2.1.</span><div class="title" tagx="title" title="title">
Target selection and molecular library preparation</div> <p>Our objective was to design an inhibitor against RNA binding domain (RBD) of structural N protein of SARS-CoV-2. The three dimensional structure of N protein 6VYO of SARS-CoV-2 was retrieved from the Protein Data Bank (PDB) (<a href="https://www.rcsb.org/">https://www.rcsb.org/</a>) for RBD. The resolution of the selected target protein structure is 1.70â€‰Ã… and the structure was identified by the X-ray diffraction method. It was selected because of its high resolution value and the latest structure of nucleocapsid phosphoprotein of SARS-CoV-2 during the study.</p> <p>The five active sites were obtained which were designated as Site 1, Site 2, Site 3, Site 4 and Site 5. Out of these, three sites were selected for further analysis based on the DScore of more than one (DScore of Sites 1, 2 and 3 were1.039, 0.99 and 1.045, respectively). The 2D structure of ligands were downloaded from Asinex and PubChem open chemistry databases. Thus, a total of 8987 (eight thousands nine hundred and eighty-seven) ligands were grouped into two libraries (Lib A and Lib B). The Lib A included 8722 antiviral compounds downloaded from Asinex Elite molecular database (<a href="http://www.asinex.com/antiviral/">http://www.asinex.com/antiviral/</a>), whereas, Lib-B had 265 FDA approved moieties categorized as drug for infectious disease that were downloaded from NCBI PubChem (<a href="https://pubchem.ncbi.nlm.nih.gov/">https://pubchem.ncbi.nlm.nih.gov/</a>) (<a href="#F0001">Figure 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Virtual screening workflow targeting N protein of SARS CoV-2 (PDB ID: 6VYO) with Asinex and PubChem database moieties.</p>   </div> </div> <div class="proteinpreparation" title="sec"><span class="label" tagx="label" title="label">2.2.</span><div class="title" tagx="title" title="title">
Protein preparation</div> <p>The Schrodingerâ€™s Protein Preparation Wizard was used for the preparation of three-dimensional structure of target protein (PDB ID: 6VYO). It was done by correcting bond orders, the addition of hydrogen bonds (HBs), creating zero-order bonds to metal atoms, creating disulphide bonds, conversion of selenomethionine to methionine, filling in missing side chains prime, deleting waters from het groups and generating het states using EpikPh 7 to Â±2. The optimization of protein HBs was done through renovation by the overlying and minimizing hydrogen using OPLS3e force field (Madhavi Sastry etÂ al., <a href="#CIT0031">2013</a>).</p> </div> <div class="receptorgridgenerationandactivesiteprediction" title="sec"><span class="label" tagx="label" title="label">2.3.</span><div class="title" tagx="title" title="title">
Receptor grid generation and active site prediction</div> <p>The recognition of the active site or the binding site of the target structure is a key step in drug designing through computational docking of the novel chemical moieties. The active site of 6VYO target was found to be a pocket like region containing a mix of hydrophilic and hydrophobic regions with a variety of hydrogen donors and acceptors. The active sites for receptorâ€"ligand binding interaction of 6VYO proteins were predicted using sitemap modules in Maestro 12.0 Schrodinger suite (Halgren, <a href="#CIT0022">2009</a>). The receptor grid was generated using the generation module in Maestro 12.0 Schrodinger suite by preferring active site residues after their prediction. The size of grid box was centroid of selected residues of the protein receptor. The atoms of protein were fixed within the default parameters of the radii of Vander Waalâ€™s scaling factor of 1â€‰Ã… with partial charge cut-off of 0.25â€‰Ã… using OPLS3e force field and 20â€‰Ã… docked ligand length. The dimensions of the grid box and receptor setup were <i>x</i>â€‰=â€‰20â€‰Ã…, <i>y</i>â€‰=â€‰20â€‰Ã…, <i>z</i>â€‰=â€‰20â€‰Ã… and <i>x</i>â€‰=â€‰10â€‰Ã…, <i>y</i>â€‰=â€‰10â€‰Ã…, <i>z</i>â€‰=â€‰10â€‰Ã… during docking study, respectively, with a grid space of 1â€‰Ã….</p> </div> <div class="ligandspreparation" title="sec"><span class="label" tagx="label" title="label">2.4.</span><div class="title" tagx="title" title="title">
Ligands preparation</div> <p>The 8722 antiviral drugs and 265 FDA approved drugs used for infectious diseases were selected for virtual screening and molecular docking study against SARS-CoV-2 target proteins (6VYO) from Asinex Elite molecular database (<a href="http://www.asinex.com/antiviral/">http://www.asinex.com/antiviral/</a>) and NCBI PubChem database (<a href="https://pubchem.ncbi.nlm.nih.gov/">https://pubchem.ncbi.nlm.nih.gov/</a>), respectively. The ligands were prepared by using the ligand preparation module of molecular modeling package with suitable parameters like optimization, ring conformation, 2D to 3D conversion, determination of promoters, tautomers and ionization states at pH 7.0 along with partial atomic charges using OPLS3e force field A.</p> </div> <div class="virtualscreeningandmoleculardocking" title="sec"><span class="label" tagx="label" title="label">2.5.</span><div class="title" tagx="title" title="title">
Virtual screening and molecular docking</div> <p>Docking study of selected ligands (8722 antiviral drugs, 265 FDA approved drugs for infectious diseases) for high throughput virtual screening (HTVS) against a target protein (PDB ID: 6YVO) was done. The HTVS with flexible docking was performed on Schrodingerâ€™s ligand docking module between ligands and active Site 3 of target protein (6YVO). After HTVS, standard precision (SP) using ligand-docking process filtered out 500 ligands from Asinex database and 200 ligands from PubChem database. Out of those, 100 ligands from each library were selected which had best configuration with highest docking score of virtual screening following extra precision (XP) with flexible docking (Friesner etÂ al., <a href="#CIT0018">2006</a>; Halgren etÂ al., <a href="#CIT0023">2004</a>). These 100 ligands from each library were docked with Site 1 and Site 2 as well. Finally, the top 10 ligands from both libraries (Lib A and Lib B) were selected which were common against all the three sites for further analysis (<a href="#F0001">Figure 1</a>).</p> </div> <div class="calculationofbindingfreeenergiesbyprimemm-gbsaapproach" title="sec"><span class="label" tagx="label" title="label">2.6.</span><div class="title" tagx="title" title="title">
Calculation of binding free energies by prime MM-GBSA approach</div> <p>The ligands and receptor complexes were analyzed for their binding free energies by prime MM-GBSA (Molecular Mechanics Generalized Born Surface Area) module of Schrodinger suite with the OPLS3e force field (Genheden &amp;amp; Ryde, 2015). The prime MM-GBSA approach is used to calculate Î"<i>G</i> bind of each ligand which is based on the docking complex interactions. However, the prioritization of lead compound was based on rescoring function of MM-GBSA approach and its binding free energies. These calculations along with docking scores were considered to optimize seven final identified ligand molecules.</p> </div> <div class="admetscreening" title="sec"><span class="label" tagx="label" title="label">2.7.</span><div class="title" tagx="title" title="title">
ADMET screening</div> <p>The seven inhibitors were analyzed for their drug likeliness and pharmacokinetic properties such as ADME/T (Absorption, Distribution, Metabolism, Elimination or Toxicity) (Butina etÂ al., <a href="#CIT0006">2002</a>). The QikProp module implemented in the â€˜Schrodinger Suite 2019â€™ was used to estimate the drug likeliness of the inhibitors using pharmacokinetic parameters such as probable H-bonding atoms, human oral absorption (&amp;lt;25%: poor; &amp;gt;above 80%: good) and the IC<sub>50</sub> values for blockage of human Ether-a-go-go-Related Gene (hERG) Kâ€‰+â€‰channels(<div class="â€‰âˆ’5:_UNKNOWN" href="#CIT0029">
2004).</div></p> </div> <div class="moleculardynamicssimulationstudies" title="sec"><span class="label" tagx="label" title="label">2.8.</span><div class="title" tagx="title" title="title">
Molecular dynamics simulation studies</div> <p>The molecular dynamics (MD) simulation for proteinâ€"ligand interaction was done by using â€˜Desmond moduleâ€™ for studying the conformational change of protein in a solvent (<a href="https://www.schrodinger.com/desmond">https://www.schrodinger.com/desmond</a>). The influence and structural stability of water molecules was investigated by running the protocol for 50â€‰ns after following the MD protocol of minimization, heating, equilibrium and production run. The default protocol with OPLS3e force field of Desmond was used to saturate, minimizing the energy and determination of partial charges (Bowers etÂ al., <a href="#CIT0004">2006</a>).</p> <p>The complexes were immersed in orthorhombic box (10â€‰Ã—â€‰10â€‰Ã—â€‰10â€‰Ã…) of TIP3P solvent with 15,472 (ID 5817), 15,459 (ID 6799) and 15,467 (72187) water molecules, respectively, buffered at a distance of 10â€‰Ã… after the determination of topology and atomic coordinates of protein with the edge of the box. A constant temperature of 300â€‰K along with isobaric isothermal ensemble (NPT equilibrium) was used by Noseâ€"Hoover chain thermostat along with Martynâ€"Tobaisâ€"Klein barostat for maintaining pressure at one bar for 50â€‰ns for MD simulations. The resulting trajectories of proteinâ€"ligand interactions were recorded at every 50.0â€‰ps interval for analysis. The simulations interaction diagrams, protein structure details, proteinâ€"ligand RMSD and individual RMSF figures were generated.</p> </div> </div> <div class="resultsanddiscussions" title="sec"><span class="label" tagx="label" title="label">3.</span><div class="title" tagx="title" title="title">
Results and discussions</div> <div class="proteinstructurereliabilityandactivesiteselection" title="sec"><span class="label" tagx="label" title="label">3.1.</span><div class="title" tagx="title" title="title">
Protein structure reliability and active site selection</div> <p>Quality and reliability of three-dimensional structure of target protein is the important parameter for drug designing. The PROCHECK server presents Ramachandran plot displaying allowed and the disallowed regions about backbone dihedrals of protein residues. More than 90% score of most favored regions ensures a model of good quality. The Ramachandran plot (<a href="#F0002">Figure 2(A)</a>) of 6VYO proteins structures reveals that 100% of residues fall within most allowed region, which indicates that the three-dimensional protein structures are reliable (<a href="#t0001">Table 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>(A) Ramachandran plots of 6VYO proteins structure describe favored and the disallowed regions of the residues. (B) Various predicted ligand-binding sites of target protein structure (6VYO) with their respective DScore by sitemap module of SchrÃ¶dingerâ€™s software.</p>   </div> <div class="table-wrap_UNKNOWN" id="t0001" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> <p>Ramachandran plot statistics showing residues present infavoured and disallowed regions of 6VYO protein structure.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Properties</th> <th align="center">Residues</th> <th align="center">Percentage (%)</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">The Most favored regions</td> <td align="center">358</td> <td align="center">92.5</td> </tr> <tr> <td align="left">Additional allowed regions</td> <td align="center">29</td> <td align="center">7.5</td> </tr> <tr> <td align="left">Generously allowed regions</td> <td align="center">0</td> <td align="center">0.00</td> </tr> <tr> <td align="left">Disallowed regions</td> <td align="center">0</td> <td align="center">0.00</td> </tr> <tr> <td align="left">The end residues (excluding Glycine and Proline)</td> <td align="center">8</td> <td align="center">Â </td> </tr> <tr> <td align="left">Proline</td> <td align="center">64</td> <td align="center">Â </td> </tr> <tr> <td align="left">Glycine</td> <td align="center">40</td> <td align="center">Â </td> </tr> <tr> <td align="left">Total number of residues</td> <td align="center">499</td> <td align="center">100</td> </tr> </tbody> </table> </div> <p>This shows minimum steric interactions due to the forbidden psi and the phi angles thereby advocating good worth of stereo chemical quality of the 3D protein structure (Madhavi Sastry etÂ al., <a href="#CIT0031">2013</a>). After selection of 3D structure of target protein, active site was predicated for molecular docking. Five active sites were predicted using sitemap modules of Schrodingerâ€™s software (<a href="#F0002">Figure 2(B)</a>).</p> <p><a href="#t0002">Table 2</a> shows the associations of Sites 1, 2, 3, 4 and 5 with their respective amino acid residues grouped into chain A, B, C and D, docking score, size and site score. Out of the five-hotspot sites, the best three sites were selected based on their DScore. Site 3 having 1.045 DScore was selected as main target site for the study. The DScore of Sites 1 and 2 are 1.039 and 0.99, respectively. The site with DScore of more than one was taken as predictive for better study of interaction. The amino acids residues of Sites 1, 2 and 3 are associated with chain A &amp;amp; B, A &amp;amp;D and C &amp;amp; D, respectively. The docking results also showed the better docking scores with Sites 1 and 3 as compared to Site 2 (<a href="#F0003">Figures 3</a> and <a href="#F0004">4</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>Selected four lead molecules from Asinex database and their comparative docking score with Sites 1, 2 and 3 of target protein (6VYO).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>Selected three potential lead molecules from PubChem database and their comparative docking score with Sites 1, 2 and 3 of target protein (6VYO).</p>   </div> <div class="table-wrap_UNKNOWN" id="t0002" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2.</span> <p>Various active sites of target protein (PDB ID: 6VYO) with their respective DScore, size and site score and associated residues predicated by sitemap module of SchrÃ¶dingerâ€™s software.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">S/no.</th> <th align="center">Active site</th> <th align="center">DScore</th> <th align="center">Size</th> <th align="center">Site score</th> <th align="center">Residues</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">1</td> <td align="left">Site 1</td> <td align="char" char=".">1.039</td> <td align="char" char=".">216</td> <td align="char" char=".">1.009</td> <td align="left">Chain A: 50ALA, 51SER, 53PHE, 54THR, 55ALA, 56LEU, 57THR, 59HIP, 92ARG, 105SER, 107ARG, 109TYR, 149ARG, 156ALA, 157ILE, 158VAL, 160GLN, 161LEU, 162PRO, 163GLN, 164GLY, 165THR, 167LEU, 171PHE, 172TYR, 173ALA Chain B: 52TRP, 53PHE, 73PRO, 74ILE, 75ASN, 76THR, 77ASN, 78SER, 82ASP, 145hie, 146ile, 148thr, 149arg, 150asn, 153ASN, 154ASN, 155ALA, 158VAL, 159LEU, 160GLN, 157ILE</td> </tr> <tr> <td align="left">2</td> <td align="left">Site 2</td> <td align="char" char=".">0.99</td> <td align="char" char=".">196</td> <td align="char" char=".">0.98</td> <td align="left">Chain A: 52TPR, 53PHE, 73PRO, 74ILE, 75ASN 76THR, 77ASN, 112TYR, 114GLY, 115THR, 116GLY, 119ALA, 144ASP, 145HIE, 146ILE, 147GLY, 148THR, 149ARG, 150ASN, 153ASN, 154ASN, 157ILE, 158VAL, 159LEU, 160GLN Chain D: 54THR, 55ALA, 57THR, 59HIE, 90ALA, 91THR, 92ARG, 94ILE, 96GLY, 97GLY, 98ASP, 102LYS, 104LEU, 105SER, 106PRO, 107ARG, 109TYR, 158VAL, 159LEU, 160GLN, 161LEU, 163GLN, 165THR, 166THR, 167LEU, 173ALA</td> </tr> <tr> <td align="left">3</td> <td align="left">Site 3</td> <td align="char" char=".">1.045</td> <td align="char" char=".">183</td> <td align="char" char=".">1.039</td> <td align="left">Chain C: 50ALA, 51SER, 53PHE, 54THR, 55ALA, 56LEU, 57THR, 59HIP, 92ARG,107ARG, 109TYR, 149ARG, 156ALA, 157ILE, 158VAL Chain D: 53TRP, 73PRO, 74ILE, 75ASN, 76THR, 77ASN, 78SER, 82ASP, 145HIE, 146ILE, 147GLY, 148THR, 149ARG, 150ASN, 153ASN, 154ASN, 155ALA, 157ILE, 158VAL, 159LEU, 160GLN</td> </tr> <tr> <td align="left">4</td> <td align="left">Site 4</td> <td align="char" char=".">0.925</td> <td align="char" char=".">105</td> <td align="char" char=".">0.955</td> <td align="left">â€"</td> </tr> <tr> <td align="left">5</td> <td align="left">Site 5</td> <td align="char" char=".">0.981</td> <td align="char" char=".">90</td> <td align="char" char=".">0.968</td> <td align="left">â€"</td> </tr> </tbody> </table> </div> </div> <div class="virtualscreening&amp;moleculardocking" title="sec"><span class="label" tagx="label" title="label">3.2.</span><div class="title" tagx="title" title="title">
Virtual screening &amp;amp;molecular docking</div> <p>The selection of most probable drug candidates acquired from virtual screening and molecular docking of Asinex and PubChem datasets was obtained by analyzing scoring functions and the binding free energies-MM/GBSA. The top-scored molecules from both the databases were prioritized based on docking score. Out of 100 XP docking molecules, best top 10 docked ligands were selected from each library at Sites 1, 2 and 3. <a href="#t0003">Tables 3</a>(A-1 to A-10), 4 (C-1 to C-10) and 5 (E-1 to E-10) show the results of binding free energies Î"<i>G</i> bind (kcal/mol), docking scores, number of HBs and HB interactive residues of the ligands from the Lib-A on Site 1, Site 2 and Site 3, respectively.</p> <div class="table-wrap_UNKNOWN" id="t0003" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3.</span> <p>A-1 to A-10: Top 10 ligands from Lib-A; B-1 to B-10: top 10 ligands from Lib-B and their prime MM-GBSA, docking score, no of H bonds and H bond interactive residues with Site 1 (DScore â€" 1.39) of target protein (6VYO).</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th rowspan="2" align="left">S/no.</th> <th rowspan="2" align="center">Ligand ID</th> <th rowspan="2" align="center">2D structures</th> <th rowspan="2" align="center">Î"<i>G</i> bind (kcal/mol)</th> <th rowspan="2" align="center">Docking score</th> <th rowspan="2" align="center">No./H-bond</th> <th colspan="2" align="center">H-bond interactive residues <hr /> </th> </tr> <tr> <th align="center">Chain A</th> <th align="center">Chain B</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">A-1</td> <td align="center">5817</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’38.79</td> <td align="center">âˆ’10.61</td> <td align="center">5</td> <td align="left">ALA55(2)</td> <td align="left">ASN77 ASN154 ASN153</td> </tr> <tr> <td align="left">A-2</td> <td align="center">6799</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’44.61</td> <td align="center">âˆ’8.82</td> <td align="center">4</td> <td align="left">ALA55</td> <td align="left">ASN75 ASN154 ASN153</td> </tr> <tr> <td align="left">A-3</td> <td align="center">6797</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’40.21</td> <td align="center">âˆ’8.61</td> <td align="center">4</td> <td align="left">THR57</td> <td align="left">ASN75 ASN154 ASN153</td> </tr> <tr> <td align="left">A-4</td> <td align="center">8517</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’44.09</td> <td align="center">âˆ’8.28</td> <td align="center">3</td> <td align="left">THR57 ALA55</td> <td align="left">ASN75</td> </tr> <tr> <td align="left">A-5</td> <td align="center">7059</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’49.82</td> <td align="center">âˆ’8.25</td> <td align="center">4</td> <td align="left">THR57 ALA55</td> <td align="left">ASN77 ASN75</td> </tr> <tr> <td align="left">A-6</td> <td align="center">8503</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’43.45</td> <td align="center">âˆ’8.22</td> <td align="center">3</td> <td align="left">THR57 ALA55</td> <td align="left">ASN75</td> </tr> <tr> <td align="left">A-7</td> <td align="center">7025</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’33.96</td> <td align="center">âˆ’8.00</td> <td align="center">3</td> <td align="left">ASN75 THR76 ASN77</td> <td align="left">Â </td> </tr> <tr> <td align="left">A-8</td> <td align="center">7064</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’37.93</td> <td align="center">âˆ’7.97</td> <td align="center">5</td> <td align="left">THR57 ALA55</td> <td align="left">ASN77 THR76 ASN75</td> </tr> <tr> <td align="left">A-9</td> <td align="center">5782</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’38.66</td> <td align="center">âˆ’7.90</td> <td align="center">6</td> <td align="left">THR57 ALA55 ARG107</td> <td align="left">ASN77 ASN154 ASN153</td> </tr> <tr> <td align="left">A-10</td> <td align="center">6801</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’30.21</td> <td align="center">âˆ’7.77</td> <td align="center">4</td> <td align="left">THR57 ALA55</td> <td align="left">ASN154 ASN75</td> </tr> <tr> <td align="left">B-1</td> <td align="center">72187</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’23.50</td> <td align="center">âˆ’11.08</td> <td align="center">5</td> <td align="left">ARG107(2)</td> <td align="left">ASN77 ASN150(2)</td> </tr> <tr> <td align="left">B-2</td> <td align="center">37542</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’40.92</td> <td align="center">âˆ’8.58</td> <td align="center">5</td> <td align="left">ALA55(2)</td> <td align="left">ASN77 ASN154 ASN155</td> </tr> <tr> <td align="left">B-3</td> <td align="center">135413534</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’49.88</td> <td align="center">âˆ’8.10</td> <td align="center">5</td> <td align="left">ARG107</td> <td align="left">ASN75(2) ASN153 HIE145</td> </tr> <tr> <td align="left">B-4</td> <td align="center">2082</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’43.47</td> <td align="center">âˆ’7.13</td> <td align="center">2</td> <td align="left">ALA55(2)</td> <td align="left">Â </td> </tr> <tr> <td align="left">B-5</td> <td align="center">71616</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’43.00</td> <td align="center">âˆ’6.76</td> <td align="center">1</td> <td align="left">ALA55</td> <td align="left">Â </td> </tr> <tr> <td align="left">B-6</td> <td align="center">23665637</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’33.66</td> <td align="center">âˆ’6.75</td> <td align="center">4</td> <td align="left">ARG107</td> <td align="left">ASN77(2) ASN75</td> </tr> <tr> <td align="left">B-7</td> <td align="center">159269</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’37.04</td> <td align="center">âˆ’6.73</td> <td align="center">5</td> <td align="left">ARG107 THR57(2)</td> <td align="left">ASN75(2)</td> </tr> <tr> <td align="left">B-8</td> <td align="center">16124688</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’52.05</td> <td align="center">âˆ’6.61</td> <td align="center">3</td> <td align="left">Â </td> <td align="left">ASN77 ASN75(2)</td> </tr> <tr> <td align="left">B-9</td> <td align="center">131801472</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’14.73</td> <td align="center">âˆ’6.46</td> <td align="center">1</td> <td align="left">Â </td> <td align="left">ASN75</td> </tr> <tr> <td align="left">B-10</td> <td align="center">60877</td> <td align="center"><span class="inline-graphic" title="inline-graphic" /></td> <td align="center">âˆ’38.34</td> <td align="center">âˆ’6.38</td> <td align="center">3</td> <td align="left">THR57 ALA55</td> <td align="left">ASN75</td> </tr> </tbody> </table> </div> <p>While <a href="#t0003">Tables 3</a>(B-1 to B-10), 4(D-1 to D-10) and 5(F-1 to F-10) show the interaction of top 10 ligands from Lib-B on all the three sites (Site 1, Site 2 and Site 3) with their respective binding free energy Î"<i>G</i> bind (kcal/mol), docking scores, number of HBs and HB interactive residues in chain A, B, C and D.</p> <p>It has been observed that many drugs in each category are common with all three sites. Therefore, those drugs, which are common among top 10 with interactions on all three sites, were taken out for the conclusion. Out of top 10 from each target site, four drug-like-moieties having antiviral activities from the Asinex databases and three antiviral drugs from PubChem are found to have great affinity on the three selected hotspot sites based on their docking score. Thus, seven drugs are chosen from the Asinex and PubChem database (<a href="#t0006">Table 6</a>) based on their docking score on all the three sites.</p> <div class="table-wrap_UNKNOWN" id="t0006" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 6.</span> <p>G-1 to G-4: Four selected molecules of Lib-A from top 10 molecules against Sites 1, 2 and 3; H-1 to H-3: three selected molecules of Lib-B from top 10 molecules against Sites 1, 2 and 3.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">Â </th> <th align="center">Â </th> <th colspan="2" align="center">Binding Site 1<br /><br /><i>D</i> score (1.039) <hr /> </th> <th colspan="2" align="center">Binding Site 2<br /><br /><i>D</i> score (0.99) <hr /> </th> <th colspan="2" align="center">Binding Site 3<br /><br /><i>D</i> score (1.045) <hr /> </th> </tr> <tr> <th align="left">S/no.</th> <th align="center">Ligand ID</th> <th align="center">Docking score</th> <th align="center">MM-GBSA (kcal/mol)</th> <th align="center">Docking score</th> <th align="center">MM-GBSA (kcal/mol)</th> <th align="center">Docking score</th> <th align="center">MM-GBSA (kcal/mol)</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">G-1</td> <td align="center">5817</td> <td align="center">âˆ’10.19</td> <td align="center">âˆ’38.79</td> <td align="center">âˆ’6.752</td> <td align="center">âˆ’47.93</td> <td align="center">âˆ’10.288</td> <td align="center">âˆ’45.23</td> </tr> <tr> <td align="left">G-2</td> <td align="center">6799</td> <td align="center">âˆ’8.826</td> <td align="center">âˆ’44.61</td> <td align="center">âˆ’6113</td> <td align="center">âˆ’29.47</td> <td align="center">âˆ’10.156</td> <td align="center">âˆ’46.49</td> </tr> <tr> <td align="left">G-3</td> <td align="center">6797</td> <td align="center">âˆ’8.613</td> <td align="center">âˆ’40.21</td> <td align="center">âˆ’6.922</td> <td align="center">âˆ’35.18</td> <td align="center">âˆ’8.42</td> <td align="center">âˆ’46.54</td> </tr> <tr> <td align="left">G-4</td> <td align="center">7025</td> <td align="center">âˆ’8.28</td> <td align="center">âˆ’33.96</td> <td align="center">âˆ’6.94</td> <td align="center">âˆ’51.99</td> <td align="center">âˆ’8.92</td> <td align="center">âˆ’61.45</td> </tr> <tr> <td align="left">H-1</td> <td align="center">72187</td> <td align="center">âˆ’11.084</td> <td align="center">âˆ’23.50</td> <td align="center">âˆ’9.091</td> <td align="center">âˆ’16.37</td> <td align="center">âˆ’9.753</td> <td align="center">âˆ’44.64</td> </tr> <tr> <td align="left">H-2</td> <td align="center">135413534</td> <td align="center">âˆ’8.102</td> <td align="center">âˆ’49.88</td> <td align="center">âˆ’5.403</td> <td align="center">âˆ’39.44</td> <td align="center">âˆ’8.298</td> <td align="center">âˆ’48.05</td> </tr> <tr> <td align="left">H-3</td> <td align="center">37542</td> <td align="center">âˆ’8.588</td> <td align="center">âˆ’40.92</td> <td align="center">âˆ’6.377</td> <td align="center">âˆ’47.82</td> <td align="center">âˆ’7.943</td> <td align="center">âˆ’44.96</td> </tr> </tbody> </table> </div> <p>Thus, the ligands of Asinex databases with their docking scores on each sites (Site 1, Site 2, Site 3) are 5817 (âˆ’10.19, âˆ’6.75, âˆ’10.29); 6799 (âˆ’8.83, âˆ’6.11, âˆ’10.156); 6797 (âˆ’8.61, âˆ’6.92, âˆ’8.42) and 7025 (âˆ’8.28, âˆ’6.94, âˆ’8.92). While three drugs concluded from PubChem databases and their docking score on each sites are Zidovudine (âˆ’11.08, âˆ’9.09, âˆ’9.75), Valganciclovir (âˆ’8.102, âˆ’5.40, âˆ’8.30) and Ribavirin (âˆ’8.59, âˆ’6.38, âˆ’7.94). Further ranking of these molecules was done by rescoring MM-GBSA functions. It indicates change in free energies of binding of proteinâ€‰âˆ’â€‰receptor complex and corroborates binding affinities and docking accuracies.</p> </div> <div class="leadoptimizationthroughbindingfreeenergy,hbinteractionandadmetproperties" title="sec"><span class="label" tagx="label" title="label">3.3.</span><div class="title" tagx="title" title="title">
Lead optimization through binding free energy, HB interaction and ADMET properties</div> <p>The binding free energies (Î"<i>G</i> bind) are also in the acceptable range between âˆ’33.96 and âˆ’61.45â€‰kcal/mol of all four ligands from Lib A. The range of selected three ligands from Lib B is between âˆ’16.37 and âˆ’49.88 and each ligand molecule interacted with all three sites through different amino acid residues by H-bond interactions as well as other chemical interactions.</p> <p>Thus, the four ligands are from Asinex databases and their binding free energies (Î"<i>G</i> bind) on each sites (Site 1, Site 2, Site 3) are 5817 (âˆ’38.79, âˆ’47.93, âˆ’45.23); 6799 (âˆ’44.61, âˆ’29.47, âˆ’46.49); 6797(âˆ’40.21, âˆ’35.18, âˆ’46.54) and 7025 (âˆ’33.96, âˆ’51.99, âˆ’61.45). However, three drugs accomplished from PubChem databases and their binding free energies (Î"<i>G</i> bind) on each sites (Site 1, Site 2, Site 3) are Zidovudine (âˆ’23.50, âˆ’16.37, âˆ’44.64), Valganciclovir (âˆ’49.88, âˆ’39.44, âˆ’48.05) and Ribavirin (âˆ’40.92, âˆ’47.82, âˆ’44.96) (<a href="#t0006">Table 6</a>). Further ranking of these molecules is done by rescoring MM-GBSA functions. It indicates change in free energies of binding of proteinâ€"receptor complex and corroborates binding affinities and docking accuracies. The number of HBs interaction, docking score and binding free energies with that site (<a href="#t0003 t0004 t0005">Tables 3â€"5</a>) depict the binding capacity of a ligand to the target.</p> <div class="table-wrap_UNKNOWN" id="t0004" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 4.</span> <p>C-1 to C-10: Top 10 ligands from Lib-A; D-1 to D-10: top 10 ligands from Lib-B and their Prime MM-GBSA, docking score, no. of H bonds and H bond interactive residues with Site 2 (DScore â€" 0.99) of target protein (6VYO).</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th rowspan="2" align="left">S/no.</th> <th rowspan="2" align="center">Asinex ID</th> <th rowspan="2" align="center">2D structures</th> <th rowspan="2" align="center">Î"<i>G</i> bind (kcal/mol)</th> <th rowspan="2" align="center">Docking score</th> <th rowspan="2" align="center">No./H-bond</th> <th colspan="2" align="center">H-bond interactive residues <hr /> </th> </tr> <tr> <th align="center">Chain A</th> <th align="center">Chain D</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">C-1</td> <td align="left">6065</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’27.76</td> <td align="char" char=".">âˆ’8.41</td> <td align="left">6</td> <td align="left">ASN77 ASN75 ASN154 ALA155</td> <td align="left">ARG92 ALA173</td> </tr> <tr> <td align="left">C-2</td> <td align="left">3686</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’26.39</td> <td align="char" char=".">âˆ’7.96</td> <td align="left">2</td> <td align="left">ASN154</td> <td align="left">ARG92</td> </tr> <tr> <td align="left">C-3</td> <td align="left">6797</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>A-3</td> <td align="char" char=".">âˆ’35.18</td> <td align="char" char=".">âˆ’6.92</td> <td align="left">3</td> <td align="left">ASN75(2) ASN154</td> <td align="left">Â </td> </tr> <tr> <td align="left">C-4</td> <td align="left">5817</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>A-1</td> <td align="char" char=".">âˆ’47.93</td> <td align="char" char=".">âˆ’6.75</td> <td align="left">5</td> <td align="left">ASN77 ASN75(2)</td> <td align="left">ALA55, ARG149</td> </tr> <tr> <td align="left">C-5</td> <td align="left">7093</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’49.36</td> <td align="char" char=".">âˆ’6.63</td> <td align="left">4</td> <td align="left">ASN77(2) ASN154</td> <td align="left">ALA55</td> </tr> <tr> <td align="left">C-6</td> <td align="left">7025</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>A-7</td> <td align="char" char=".">âˆ’51.99</td> <td align="char" char=".">âˆ’6.39</td> <td align="left">3</td> <td align="left">ASN77 ASN75 THR76</td> <td align="left">Â </td> </tr> <tr> <td align="left">C-7</td> <td align="left">5584</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’33.13</td> <td align="char" char=".">âˆ’6.37</td> <td align="left">1</td> <td align="left">ARG92</td> <td align="left">Â </td> </tr> <tr> <td align="left">C-8</td> <td align="left">6806</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’33.00</td> <td align="char" char=".">âˆ’6.21</td> <td align="left">5</td> <td align="left">ASN77 ASN154</td> <td align="left">ALA173 ALA55(2)</td> </tr> <tr> <td align="left">C-9</td> <td align="left">6799</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>A-2</td> <td align="char" char=".">âˆ’29.47</td> <td align="char" char=".">âˆ’6.11</td> <td align="left">4</td> <td align="left">ALA155(2) ASN77</td> <td align="left">ARG107</td> </tr> <tr> <td align="left">C-10</td> <td align="left">7064</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>A-8</td> <td align="char" char=".">âˆ’53.01</td> <td align="char" char=".">âˆ’6.09</td> <td align="left">4</td> <td align="left">ASN77(2) ALA155</td> <td align="left">ARG92</td> </tr> <tr> <td align="left">D-1</td> <td align="left">72187</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>B-1</td> <td align="char" char=".">âˆ’16.37</td> <td align="char" char=".">âˆ’9.09</td> <td align="left">7</td> <td align="left">ASN154 ASN150 ARG107(2)</td> <td align="left">ALA55 ARG92(2)</td> </tr> <tr> <td align="left">D-2</td> <td align="left">6437075</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’23.45</td> <td align="char" char=".">âˆ’6.55</td> <td align="left">4</td> <td align="left">ASN154(2)</td> <td align="left">ARG107 ARG92</td> </tr> <tr> <td align="left">D-3</td> <td align="left">37542</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>B-2</td> <td align="char" char=".">âˆ’47.82</td> <td align="char" char=".">âˆ’6.37</td> <td align="left">4</td> <td align="left">ASN77(2)</td> <td align="left">ARG92, ARG107</td> </tr> <tr> <td align="left">D-4</td> <td align="left">23674512</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’60.99</td> <td align="char" char=".">âˆ’5.86</td> <td align="left">4</td> <td align="left">ASN154 ILE146</td> <td align="left">ARG107 ARG92</td> </tr> <tr> <td align="left">D-5</td> <td align="left">159269</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>B-7</td> <td align="char" char=".">âˆ’36.31</td> <td align="char" char=".">âˆ’5.56</td> <td align="left">4</td> <td align="left">ASN75(2) HIE145</td> <td align="left">ARG92</td> </tr> <tr> <td align="left">D-6</td> <td align="left">18283</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’43.15</td> <td align="char" char=".">âˆ’5.49</td> <td align="left">3</td> <td align="left">ASN75</td> <td align="left">ARG92 SER105</td> </tr> <tr> <td align="left">D-7</td> <td align="left">135413534</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>B-3</td> <td align="char" char=".">âˆ’39.44</td> <td align="char" char=".">âˆ’5.40</td> <td align="left">4</td> <td align="left">ASN75 ASN77</td> <td align="left">ALA173 ARG92</td> </tr> <tr> <td align="left">D-8</td> <td align="left">6256</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’30.87</td> <td align="char" char=".">âˆ’5.37</td> <td align="left">3</td> <td align="left">ASN75 HIE145</td> <td align="left">ARG92</td> </tr> <tr> <td align="left">D-9</td> <td align="left">6321416</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’57.63</td> <td align="char" char=".">âˆ’5.37</td> <td align="left">4</td> <td align="left">ASN77 ALA155</td> <td align="left">ARG107 ALA55</td> </tr> <tr> <td align="left">D-10</td> <td align="left">3672</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’28.95</td> <td align="char" char=".">âˆ’5.17</td> <td align="left">1</td> <td align="left">Â </td> <td align="left">ARG92</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="t0005" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 5.</span> <p>E-1 to E-10: Top 10 ligands from Lib-A; F-1 to F-10: top 10 ligands from Lib-B and their prime MM-GBSA, docking score, no. of H bonds and H bond interactive residues with Site 3 (DScore â€" 1.045) of target protein (6VYO).</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th rowspan="2" align="left">S/no.</th> <th rowspan="2" align="center">Asinex ID</th> <th rowspan="2" align="center">2D structures</th> <th rowspan="2" align="center">Î"<i>G</i> bind (kcal/mol)</th> <th rowspan="2" align="center">Docking bnjkop[score</th> <th rowspan="2" align="center">No./H-bond</th> <th colspan="2" align="center">H-bond interactive residues <hr /> </th> </tr> <tr> <th align="center">Chain C</th> <th align="center">Chain D</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">E-1</td> <td align="left">5817</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>A-1</td> <td align="char" char=".">âˆ’45.23</td> <td align="char" char=".">âˆ’10.28</td> <td align="left">4</td> <td align="left">ALA55(2)</td> <td align="left">ASN154 ASN154</td> </tr> <tr> <td align="left">E-2</td> <td align="left">6799</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>A-2</td> <td align="char" char=".">âˆ’46.49</td> <td align="char" char=".">âˆ’10.15</td> <td align="left">5</td> <td align="left">ALA55</td> <td align="left">ASN75(2) ASN153 ASN154</td> </tr> <tr> <td align="left">E-3</td> <td align="left">7066</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’42.79</td> <td align="char" char=".">âˆ’9.96</td> <td align="left">4</td> <td align="left">ALA55 THR57 ALA156</td> <td align="left">ASN75</td> </tr> <tr> <td align="left">E-4</td> <td align="left">6993</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’32.06</td> <td align="char" char=".">âˆ’9.29</td> <td align="left">3</td> <td align="left">ARG149 SER51</td> <td align="left">ASN154</td> </tr> <tr> <td align="left">E-5</td> <td align="left">6061</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’43.53</td> <td align="char" char=".">âˆ’9.17</td> <td align="left">6</td> <td align="left">ALA55(2) ARG107 THR57</td> <td align="left">ASN15 ASN154</td> </tr> <tr> <td align="left">E-6</td> <td align="left">7025</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>A-7</td> <td align="char" char=".">âˆ’61.45</td> <td align="char" char=".">âˆ’8.92</td> <td align="left">5</td> <td align="left">ALA55 ALA156 ARG107</td> <td align="left">ASN154 ASN75</td> </tr> <tr> <td align="left">E-7</td> <td align="left">6797</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>A-3</td> <td align="char" char=".">âˆ’46.54</td> <td align="char" char=".">âˆ’8.24</td> <td align="left">5</td> <td align="left">ALA55, THR57</td> <td align="left">ASN153 ASN154 ASN75</td> </tr> <tr> <td align="left">E-8</td> <td align="left">5802</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’50.28</td> <td align="char" char=".">âˆ’8.03</td> <td align="left">4</td> <td align="left">ALA55(2), THR57</td> <td align="left">ASN75</td> </tr> <tr> <td align="left">E-9</td> <td align="left">8503</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’35.38</td> <td align="char" char=".">âˆ’7.70</td> <td align="left">3</td> <td align="left">ALA55, THR57</td> <td align="left">ASN75</td> </tr> <tr> <td align="left">E-10</td> <td align="left">5584</td> <td align="left">Mentioned in <a href="#t0004">Table 4</a>C-7</td> <td align="char" char=".">âˆ’63.61</td> <td align="char" char=".">âˆ’7.66</td> <td align="left">1</td> <td align="left">Â </td> <td align="left">ASN153</td> </tr> <tr> <td align="left">F-1</td> <td align="left">72187</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>B-1</td> <td align="char" char=".">âˆ’44.64</td> <td align="char" char=".">âˆ’9.75</td> <td align="left">6</td> <td align="left">ALA55, ARG107</td> <td align="left">ASN75, ASN153, ASN150(2)</td> </tr> <tr> <td align="left">F-2</td> <td align="left">135413534</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>B-3</td> <td align="char" char=".">âˆ’48.05</td> <td align="char" char=".">âˆ’8.29</td> <td align="left">6</td> <td align="left">ALA55</td> <td align="left">ASN75(2), ASN77, ASN154</td> </tr> <tr> <td align="left">F-3</td> <td align="left">37542</td> <td align="left">Mentioned in <a href="#t0003">Table 3</a>B-2</td> <td align="char" char=".">âˆ’44.96</td> <td align="char" char=".">âˆ’7.94</td> <td align="left">7</td> <td align="left">ARG107, THR57, TYR109</td> <td align="left">ASN77, HIS145, ASN154(2)</td> </tr> <tr> <td align="left">F-4</td> <td align="left">60871</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’47.81</td> <td align="char" char=".">âˆ’7.60</td> <td align="left">3</td> <td align="left">ARG107, TYR111</td> <td align="left">ASN154</td> </tr> <tr> <td align="left">F-5</td> <td align="left">464205</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’38.84</td> <td align="char" char=".">âˆ’6.96</td> <td align="left">6</td> <td align="left">ARG107, ALA55, TYR109</td> <td align="left">ASN75(2), ASN154</td> </tr> <tr> <td align="left">F-6</td> <td align="left">71616</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’45.34</td> <td align="char" char=".">âˆ’6.86</td> <td align="left">1</td> <td align="left">Â </td> <td align="left">ASN154</td> </tr> <tr> <td align="left">F-7</td> <td align="left">135526609</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’44.02</td> <td align="char" char=".">âˆ’6.86</td> <td align="left">3</td> <td align="left">THR57, ALA55, ARG107</td> <td align="left">Â </td> </tr> <tr> <td align="left">F-8</td> <td align="left">6256</td> <td align="left">Mentioned in <a href="#t0004">Table 4</a>D-8</td> <td align="char" char=".">âˆ’42.51</td> <td align="char" char=".">âˆ’6.83</td> <td align="left">3</td> <td align="left">ARG107</td> <td align="left">HIS145, ASN75</td> </tr> <tr> <td align="left">F-9</td> <td align="left">90643431</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’46.43</td> <td align="char" char=".">âˆ’6.72</td> <td align="left">4</td> <td align="left">SER51, ARG107(2)</td> <td align="left">ASN153</td> </tr> <tr> <td align="left">F-10</td> <td align="left">16124688</td> <td align="left"><span class="inline-graphic" title="inline-graphic" /></td> <td align="char" char=".">âˆ’52.76</td> <td align="char" char=".">âˆ’6.46</td> <td align="left">3</td> <td align="left">ALA55, TYR109</td> <td align="left">ASN154</td> </tr> </tbody> </table> </div> <p>The chemical interaction of the top four ligands (Asinex ID: 5817; 6799, 6797 and 7025) from Lib A is shown in Supplementary Figures S1â€"S3(Aâ€"D) with Sites 1, 2 and 3. Asinex Id: 5817 interacts with all the three sites with variable number of HB interactions like 5, 4 and 4, respectively. Their interactive residues are five amino acids (Chain A: ALA55 (2); Chain B: ASN77, ASN154, ASN153) with Site 1; 4AA (Chain A: ASN77, ASN75 (2); Chain D: ALA55, ARG149) with Site 2 and four amino acids (Chain C: ALA55 (2); Chain D: ASN153, ASN154) with Site 3.</p> <p>The HB interaction of Asinex ID: 6799 and their interactive residues are four HB and 4AA (Chain A: ALA55; Chain B: ASN75, ASN154, ASN153) with Site 1; 4HB and 4AA (Chain C: ALA155 (2), ASN (77); Chain D: ARG107) with Site 2 and 5HB and 5AA (Chain C: ALA55 and Chain D: ASN75 (2), ASN153, ASN154) with Site 3.</p> <p>Ligands 6797 and their HB interaction with Sites 1, 2 and 3 are four HB, 4AA (Chain A: ASN57 and Chain B: ASN75, ASN153, ASN154); 3HB, 3AA (Chain A: ASN75 (2), ASN154) and 5HB and 5AA (Chain C: ALA55, THR57 and Chain D: ASN153, ASN154, ASN75), respectively.</p> <p>Ligands 7025 and their HB interaction with Sites 1, 2 and 3 are three HB, 3AA (Chain B: ASN75, ASN76, ASN77); 3HB, 3AA (Chain A: ASN75, ASN76, ASN77) and 5HB and 5AA (Chain C: ALA55, ALA156, ARG107 and Chain D: ASN154, ASN75), respectively.</p> <p>Supplementary Figures S4â€"S6(Aâ€"C) depict the chemical interaction of top three ligands (PubChem IDs: 72187, 135413534, 37542) from the PubChem database with Sites 1, 2 and 3 of N proteins. However PubChem ID: 72187 interacts with all the three sites with variable number of HB interactions numbering 5HBs with Site 1, 7HBs with Site 2 and 6HBs with site 3. The interaction of this ligand with H bond interactive amino acids are ARG107(2) in Chain A and ASN77, ASN150(2) in Chain B at Site 1, ASN154, ASN150 in Chain A and ALA55, ARG92(2), ARG(2) in Chain D with Site 2 and ALA55, ARG107 in Chain C &amp;amp; ASN75, ASN153, ASN150(2) in Chain D with Site 3.</p> <p>In addition, PubChem ID: 135413534 also has activity on all the three sites. The number of HB interactions on Site 1, Site 2 and Site 3 are 5, 4 and 6, respectively. The HB interactive amino acid with Site 1, Site 2 and Site 3 corresponds to five amino acid (ARG107 in Chain A and ASN75 (2), ASN153, HIE145 in Chain B), four AA (ASN75, ASN77 in Chain A and ALA173, ARG92 in Chain B) and six AA (ALA55 in Chain C and ASN75 (2), ASN154(2), ASN77), respectively.</p> <p>The ADME/T property values of all the four molecules from Asinex database molecules satisfied the acceptable ranges in term of Lipinskiâ€™s rule of five 5 (Lipinski, <a href="#CIT0029">2004</a>) and other pharmacokinetic parameters such as probable H-bonding atoms, human oral absorption (&amp;lt;25%: poor; &amp;gt;above 80%: good) and the IC<sub>50</sub> values for blockage of human Ether-a-go-go-Related Gene (hERG) Kâ€‰+â€‰channels (<div class="â€‰âˆ’5_UNKNOWN" href="#CIT0038">
2018). Finally, four lead optimized molecules are predicted to have degree of drug likeliness. <a href="#t0007">Table 7A</a> shows that molecular weights of the selected compounds are in the range from 325 to 420â€‰D, hydrogen bond acceptance from 9 to 10 and hydrogen bond donor between 2 and 4. It also shows that the compounds have rotatable bonds in the range of 4â€"8 while their polar surface areas ranges from 144.664 to 209.137. Furthermore, <a href="#t0007">Table 7A</a> mentions the detailed results including Lipinski rule of 5, Veberâ€™s rule and other pharmacokinetics parameters for these four compounds. Whereas, there is no need of ADMET prediction for three FDA approved drug molecules as they are already approved as anti-HIV as well as anti-Herpes drugs (<a href="#t0008">Table 7B</a>).</div></p> <div class="table-wrap_UNKNOWN" id="t0007" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 7A.</span> <p>The drug likeliness properties of the antiviral small molecules predicted by QikProp module.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th rowspan="2" align="left">S/no.</th> <th rowspan="2" align="center">Ligand ID</th> <th rowspan="2" align="center">Mol./weight</th> <th rowspan="2" align="center">H-bond acceptor</th> <th rowspan="2" align="center">H-bond donor</th> <th rowspan="2" align="center">Oral absorption</th> <th rowspan="2" align="center">% Human oral absorption</th> <th rowspan="2" align="center">QP Log HERG</th> <th rowspan="2" align="center">Lipinski rule of 5</th> <th colspan="2" align="center">Veber's rule <hr /> </th> </tr> <tr> <th align="center">Rotatable bond count</th> <th align="center">Polar surface area (PSA)</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">1</td> <td align="left">5817</td> <td align="left">383.4</td> <td align="char" char=".">10.95</td> <td align="char" char=".">4</td> <td align="left">2</td> <td align="left">34.355</td> <td align="left">âˆ’0.458</td> <td align="char" char=".">0</td> <td align="char" char=".">8</td> <td align="char" char=".">184.047</td> </tr> <tr> <td align="left">2</td> <td align="left">6799</td> <td align="left">383.4</td> <td align="char" char=".">9.25</td> <td align="char" char=".">3</td> <td align="left">2</td> <td align="left">34.546</td> <td align="left">âˆ’0.391</td> <td align="char" char=".">0</td> <td align="char" char=".">4</td> <td align="char" char=".">166.824</td> </tr> <tr> <td align="left">3</td> <td align="left">6797</td> <td align="left">325.321</td> <td align="char" char=".">9.25</td> <td align="char" char=".">3</td> <td align="left">2</td> <td align="left">36.852</td> <td align="left">âˆ’0.27</td> <td align="char" char=".">0</td> <td align="char" char=".">4</td> <td align="char" char=".">209.137</td> </tr> <tr> <td align="left">4</td> <td align="left">7025</td> <td align="left">419.436</td> <td align="char" char=".">10.45</td> <td align="char" char=".">2.25</td> <td align="left">1</td> <td align="left">17.254</td> <td align="left">âˆ’3.868</td> <td align="char" char=".">1</td> <td align="char" char=".">7</td> <td align="char" char=".">144.664</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="t0008" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 7B.</span> <p>Potential FDA approved drug molecules, their indication and mechanism of action.</p>  <table frame="hsides" rules="groups"> <colgroup> <col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup> <colgroup><col width="1*" /></colgroup>  <thead> <tr> <th align="left">S/no.</th> <th align="center">Name of drug molecules</th> <th align="center">PubChem Id</th> <th align="center">Indication</th> <th align="center">Mechanism of action</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left">1</td> <td align="left">Zidovudine</td> <td align="left">72187</td> <td align="left">Antiviral drug used for the treatment of human immunodeficiency virus (HIV) infections</td> <td align="left">Nucleoside reverse transcriptase inhibitor (NRTI) with activity against Human Immunodeficiency Virus Type 1 (HIV-1) (<a href="https://www.drugbank.ca/drugs/DB00495">https://www.drugbank.ca/drugs/DB00495</a>)</td> </tr> <tr> <td align="left">2</td> <td align="left">Valganciclovir</td> <td align="left">135413534</td> <td align="left">Antiviral drug used for the treatment of cytomegalovirus infections</td> <td align="left">Inhibition of viral DNA synthesis by incorporation with viral DNA&amp;amp; inhibits viral DNA polymerases (<a href="https://www.drugbank.ca/drugs/DB01610">https://www.drugbank.ca/drugs/DB01610</a>)</td> </tr> <tr> <td align="left">3</td> <td align="left">Ribavirin</td> <td align="left">37542</td> <td align="left">Used for the treatment of chronic Hepatitis C virus (HCV) infection</td> <td align="left">Inhibits of viral RNA and protein synthesis by inhibition of the enzyme RNA dependent RNA polymerase (<a href="https://www.drugbank.ca/drugs/DB00811">https://www.drugbank.ca/drugs/DB00811</a>)</td> </tr> </tbody> </table> </div> </div> <div class="mdsimulation" title="sec"><span class="label" tagx="label" title="label">3.4.</span><div class="title" tagx="title" title="title">
MD simulation</div> <p>Three best docked compounds Asinex 5817, 6799 and PubChem 72187 (Zidovudine) were subjected to MD simulations for 50â€‰ns. The RMSD and RMSF values were used to observe the stability of proteinâ€"ligand complexes as compared to unbound protein structure. The RMSD plot of 5817, 6799 and 72187 are depicted in <a href="#F0005">Figure 5(A</a>, B &amp;amp; C), respectively. Figure 5(A) elaborates the interaction of protein with the ligand Asinex 5817. It shows main bimodal fluctuations at 3.4â€‰Ã… during 12â€"18â€‰ns interval and another at 3.4â€‰Ã… for 30â€"36â€‰ns. However, <a href="#F0006">Figure 6</a>A, B and C show the favorable residual interactions which remained for more than 30% of the time. The hydrogen bonds with more than 70% duration of interactions were considered as strong bonds. The interaction with 40â€"70% was considered as medium while less than 40% was clubbed into weak hydrogen interactions. The medium strength of interaction of ligands were seen with D: ASN75 (66%) and C: TYR109 (45%). It also has D: TRP 52 linked to the center of the benzene ring with Piâ€"Pi interaction stocking at 72%. Other residues such as C: ARG107, C: THR57, D: ASP82 and D: ALA155 showed water bridges links.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>RMSD trajectories of Site-3 of protein-ligand complexes during 50<b>â€‰</b>ns. (A) 5817, (B) 6799 and (C) 72187 (Zidovudine).</p>   </div> <div class="fig-group" title="fig-group"><span class="label" tagx="label" title="label">Figure 6.</span> <p>(A) <b>(</b>A-1) The bar charts of proteinâ€"ligand interaction fraction over the course of trajectory. (A-2) Protein residue interactions with ligand 5817. (B) (B-1) The bar charts of proteinâ€"ligand interaction fraction over the course of trajectory. (B-2) Protein residue interactions with ligand 6799. (C) (C-1) The bar charts of proteinâ€"ligand interaction fraction over the course of trajectory. (C-2) Protein residue interactions with ligand 72187 (Zidovudine).</p>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/" /></div>  </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/" /></div>  </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/" /></div>  </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7.</span></a></div>   <p>Binding free energy between proteinâ€"ligand (5817, 6799, 72187) complex was predicted by using MM-GBSA method. The values of binding free energy was calculated per 5<b>â€‰</b>ns.</p>   </div> <p>While <a href="#F0005">Figure 5(B)</a> demonstrates the fluctuations in the region 3.5â€‰Ã… from 6 to 10â€‰ns and 4.8â€‰Ã… from 28 to 50â€‰ns. This interaction remained stable and sustained for a longer period. The ligand atom interaction with strong hydrogen bonds were seen in D: ASN75 (86% on NH and 99% on O), C: ALA55 (95%), medium interaction in C: TYR109 (44%) and weak interactions in C: ARG107 (36%) and C: ARG92 (38%). While residues D: ASN77 (38%) and C: ARG 149 (32%) showed water bridges (<a href="#F0006">Figure 6(B)</a>). <a href="#F0005">Figure 5(C)</a> shows the interaction of Zidovudine with the target protein. It shows a very stable interactions and minimal fluctuation. Its increase started to 2.4â€‰Ã… on 2â€‰ns and remained stable up to 3â€‰Ã… from 13 to 50â€‰ns. Its interactions are also complex and multiple bond formations. Majority of the residues form the hydrogen bonds and maximum interactions were seen with the residue ASN. Only three residues (D: THR148, D: ASN153, D: ASN77) exclusively forms the water bridges and one residue form the Van der Waals interaction (C: ARG107). All other residues show hydrogen bond interactions with long period of time including three strong, two medium and two weak hydrogen interactions such as C: ARG107 having variable percentage with three oxygen atom and one water linkage, C: ALA55 (71% with N), C: TYR109 (33%), D: ASN75 (95% O and 45% NH), D: ASN 154 (83%, 33% for two different O atom and one water linkage) and D: ASN150 (more than 50% with two O and two water molecules) (<a href="#F0006">Figure 6(C)</a>).</p> <p>The RMSF fluctuation is also very low in case of Zidovudine (0.8 minimum to 4.3) as compared to two other ligands [5817 (minimum 0.8â€"5.8) and 6799 (minimum 1.2â€"6.2). Figure 7 shows comparative binding free energy (Î"<i>G</i> bind) which were calculated at every 5â€‰ns of all three ligands. The Î"<i>G</i> bind value for the three ligands 5817, 6799 and 72187 calculated over 50â€‰ns are âˆ’51.96, âˆ’64.36 and âˆ’59.43, respectively. The binding free energy (Î"<i>G</i> bind) are very significant in all the three ligands. It can be seen (<a href="#F0006">Figure 6(A, B and C)</a>) that compound 5817, 6799 and 72187 formed more stable hydrogen bonds with the residues of binding Site 3 of target protein.</p> </div> </div> <div class="conclusion" title="sec"><span class="label" tagx="label" title="label">4.</span><div class="title" tagx="title" title="title">
Conclusion</div> <p>We identified a total of seven hits in the first step after molecular docking, four from Asinex and three from PubChem database, which can inhibit the structural N proteins of SARS-CoV-2. Many studies are targeting different proteins including main protease (Mahanta etÂ al., <a href="#CIT0032">2020</a>; Mittal etÂ al., <a href="#CIT0034">2020</a>) and spike proteins (Basit etÂ al., <a href="#CIT0002">2020</a>; de Oliveira etÂ al., <a href="#CIT0010">2020</a>). However, the homologous proteins of NTD has been used in one earlier study that has shown theophylline and pyrimidine groups of drugs as promisable interactions (Sarma etÂ al., <a href="#CIT0037">2020</a>). However, our study is the first study that targeted binding potential of known drugs and drug like moieties on the crystallized novel proteins structure 6VYO. There are 4â€"5 significant hydrogen bond interactions, on an average, associated with each drug molecule. It has interestingly been found that most of the hydrogen bonds interactions of all the seven drugs mainly are associated with ASN residues.</p> <p>Thus, four Asinex ligands (5817, 6799, 6797, 7025) and three FDA approved drug molecules (PubChem ID: 72187, 135413534, 37542) were found to have good docking score on each site (Site 1, Site 2 and Site 3). The drug like moieties from Asinex have antiviral activities and drugs from PubChem databases are Zidovudine, Valganciclovir and Ribavirin. Thus, anti-HIV drugs as well as anti-Herpes drugs are active on this protein. Furthermore, second-step analysis was done by MD simulation to find final three drugs. Out of those, Zidovudine has stronger and more stable interaction with this protein structure. Therefore, the drugs especially Zidovudine can be further explored in the clinical trial as the potential treatment against the SARS CoV-2.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplemental Material</div>  <div class="media" title="media"><a href="TBSD_A_1778536_SM9051.docx">LINK</a></div> </div> </div> </div> <div class="back" title="back"> <div class="acknowledgements" title="sec"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Special thanks to the Schrodinger corporations for their help.</p> </div> <div class="disclosurestatement" title="sec"> <div class="title" tagx="title" title="title">
Disclosure statement</div> <p>No potential conflict of interest was reported by the authors.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CIT0001" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aanouz</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belhassan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El-Khatabi</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lakhlifi</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El-Ldrissi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouachrine</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Moroccan medicinal plants as inhibitors against SARS-CoV-2 main protease: Computational investigations</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1758790">10.1080/07391102.2020.1758790</a></span></span></li> <li tag="ref"><a name="CIT0002" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basit</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rehman</span><span class="given-names" tagx="given-names" title="given-names">S. U.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Truncated human Angiotensin Converting Enzyme 2; A potential inhibitor of SARS-CoV-2 spike glycoprotein and potent COVID-19 therapeutic agent</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">17</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1768150">10.1080/07391102.2020.1768150</a></span></span></li> <li tag="ref"><a name="CIT0003" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boopathi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poma</span><span class="given-names" tagx="given-names" title="given-names">A. B.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolandaivel</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Novel 2019 coronavirus structure, mechanism of action, antiviral drug promises and rule out against its treatment</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">14</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1758788">10.1080/07391102.2020.1758788</a></span></span></li> <li tag="ref"><a name="CIT0004" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowers</span><span class="given-names" tagx="given-names" title="given-names">K. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chow</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dror</span><span class="given-names" tagx="given-names" title="given-names">R. O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eastwood</span><span class="given-names" tagx="given-names" title="given-names">M. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gregersen</span><span class="given-names" tagx="given-names" title="given-names">B. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klepeis</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KolossvÃ¡ry</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moraes</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sacerdoti</span><span class="given-names" tagx="given-names" title="given-names">F. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salmon</span><span class="given-names" tagx="given-names" title="given-names">J. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, &amp;amp;<span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaw</span><span class="given-names" tagx="given-names" title="given-names">D. E.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). Scalable algorithms for molecular dynamics simulations on commodity clusters [Paper presentation]. Proceedings of the 2006 ACM/IEEE Conference on Supercomputing, SCâ€™06, New York, New York, USA (p. 84). ACM Press. <span class="pub-id"><a href="https://dx.doi.org/10.1145/1188455.1188544">10.1145/1188455.1188544</a></span></span></li> <li tag="ref"><a name="CIT0005" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brian</span><span class="given-names" tagx="given-names" title="given-names">D. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baric</span><span class="given-names" tagx="given-names" title="given-names">R. S.</span></span></span> (<span class="year" tagx="year" title="year">2005</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Coronavirus genome structure and replication</span> In <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enjuanes</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span> (Ed.), <span class="source" tagx="source" title="source"><i>Coronavirus replication and reverse genetics.</i> Current <i>topics in microbiology and immunology</i></span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/3-540-26765-4_1">10.1007/3-540-26765-4_1</a></span></span></li> <li tag="ref"><a name="CIT0006" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Butina</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Segall</span><span class="given-names" tagx="given-names" title="given-names">M. D.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frankcombe</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Predicting ADME properties in silico: Methods and models</span>. <span class="source" tagx="source" title="source"><i>Drug Discovery Today</i></span>, <span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">11</span>), <span class="fpage" tagx="fpage" title="fpage">S83</span>â€"<span class="lpage" tagx="lpage" title="lpage">S88</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S1359-6446(02)02288-2">10.1016/S1359-6446(02)02288-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12047885">12047885</a></span></span></li> <li tag="ref"><a name="CIT0007" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">C.-K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michael Chen</span><span class="given-names" tagx="given-names" title="given-names">C.-M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiang</span><span class="given-names" tagx="given-names" title="given-names">M.-H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu</span><span class="given-names" tagx="given-names" title="given-names">Y.-I.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">T.-H.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Transient oligomerization of the SARS-CoV N protein â€" Implication for virus ribonucleoprotein packaging</span>. <span class="source" tagx="source" title="source"><i>PLoS One</i></span>, <span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">e65045</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0065045">10.1371/journal.pone.0065045</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23717688">23717688</a></span></span></li> <li tag="ref"><a name="CIT0008" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ning</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Clinical characteristics of 113 deceased patients with coronavirus disease 2019: Retrospective study</span>. <span class="source" tagx="source" title="source"><i>BMJ (Clinical Research ed.)</i></span>, <span class="volume" tagx="volume" title="volume">368</span>, <span class="fpage" tagx="fpage" title="fpage">m1091</span><span class="pub-id"><a href="https://dx.doi.org/10.1136/bmj.m1091">10.1136/bmj.m1091</a></span></span></li> <li tag="ref"><a name="CIT0009" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cong</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ulasli</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schepers</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mauthe</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vâ€™kovski</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kriegenburg</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thiel</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Haan</span><span class="given-names" tagx="given-names" title="given-names">C. A. M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reggiori</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Nucleocapsid protein recruitment to replicationâ€"transcription complexes plays a crucial role in coronaviral life cycle</span>. <span class="source" tagx="source" title="source"><i>Journal of Virology</i></span>, <span class="volume" tagx="volume" title="volume">94</span>(<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">e01925</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01925-19">10.1128/JVI.01925-19</a></span></span></li> <li tag="ref"><a name="CIT0010" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Oliveira</span><span class="given-names" tagx="given-names" title="given-names">O. V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rocha</span><span class="given-names" tagx="given-names" title="given-names">G. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paluch</span><span class="given-names" tagx="given-names" title="given-names">A. S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Costa</span><span class="given-names" tagx="given-names" title="given-names">L. T.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Repurposing approved drugs as inhibitors of SARS-CoV-2S-protein from molecular modeling and virtual screening</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure &amp;amp; Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">14</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1772885">10.1080/07391102.2020.1772885</a></span></span></li> <li tag="ref"><a name="CIT0011" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dinesh</span><span class="given-names" tagx="given-names" title="given-names">D. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chalupska</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silhan</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veverka</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boura</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Structural basis of RNA recognition by the SARS-CoV-2 nucleocapsid phosphoprotein</span>. <span class="source" tagx="source" title="source"><i>BioRxiv</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.04.02.022194">10.1101/2020.04.02.022194</a></span></span></li> <li tag="ref"><a name="CIT0012" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">D.-Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">K.-Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">B.-J.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Priming with rAAV encoding RBD of SARS-CoV S protein and boosting with RBD-specific peptides for T cell epitopes elevated humoral and cellular immune responses against SARS-CoV infection</span>. <span class="source" tagx="source" title="source"><i>Vaccine</i></span>, <span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">13</span>), <span class="fpage" tagx="fpage" title="fpage">1644</span>â€"<span class="lpage" tagx="lpage" title="lpage">1651</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vaccine.2008.01.025">10.1016/j.vaccine.2008.01.025</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18289745">18289745</a></span></span></li> <li tag="ref"><a name="CIT0013" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>a). <span class="mixed-article-title" title="mixed-article-title">Natural products may interfere with SARS-CoV-2 attachment to the host cell</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">10</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1761881">10.1080/07391102.2020.1761881</a></span></span></li> <li tag="ref"><a name="CIT0014" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elfiky</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>b). <span class="mixed-article-title" title="mixed-article-title">SARS-CoV-2 RNA dependent RNA polymerase (RdRp) targeting: An in silico perspective</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure &amp;amp; Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">15</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1761882">10.1080/07391102.2020.1761882</a></span></span></li> <li tag="ref"><a name="CIT0015" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elmezayen</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Obaidi</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Åžahin</span><span class="given-names" tagx="given-names" title="given-names">A. T.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">YelekÃ§i</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Drug repurposing for coronavirus (COVID-19): <i>In silico</i> screening of known drugs against coronavirus 3CL hydrolase and protease enzymes</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1758791">10.1080/07391102.2020.1758791</a></span></span></li> <li tag="ref"><a name="CIT0016" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enmozhi</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raja</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sebastine</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joseph</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Andrographolide as a potential inhibitor of SARS-CoV-2 main protease: An in silico approach</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">10</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1760136">10.1080/07391102.2020.1760136</a></span></span></li> <li tag="ref"><a name="CIT0017" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fehr</span><span class="given-names" tagx="given-names" title="given-names">A. R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="chapter-title" tagx="chapter-title" title="chapter-title">Coronaviruses: An overview of their replication and pathogenesis</span> In <span class="source" tagx="source" title="source"><i>Coronaviruses: Methods and Protocols.</i></span> (Vol. <span class="volume" tagx="volume" title="volume">1282</span>, pp. <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">23</span>). <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-4939-2438-7_1">10.1007/978-1-4939-2438-7_1</a></span></span></li> <li tag="ref"><a name="CIT0040" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Eurosurveillance Editorial Team. (2020). Note from the editor: World Health Organization declares novel coronavirus (2019-nCoV) sixth public health emergency of international concern. <i>Euro Surveill</i>, <i>25</i>(5), 200131e. doi:<span class="pub-id"><a href="https://dx.doi.org/10.2807/1560-7917.ES.2020.25.5.200131e">10.2807/1560-7917.ES.2020.25.5.200131e</a></span>.</span></li> <li tag="ref"><a name="CIT0018" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friesner</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">R. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Repasky</span><span class="given-names" tagx="given-names" title="given-names">M. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frye</span><span class="given-names" tagx="given-names" title="given-names">L. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenwood</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halgren</span><span class="given-names" tagx="given-names" title="given-names">T. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanschagrin</span><span class="given-names" tagx="given-names" title="given-names">P. C.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mainz</span><span class="given-names" tagx="given-names" title="given-names">D. T.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Extra precision glide: Docking and scoring incorporating a model of hydrophobic enclosure for proteinâ€"ligand complexes</span><span class="source" tagx="source" title="source"><i>. <i>Journal of Medicinal Chemistry</i></i></span>, <span class="volume" tagx="volume" title="volume">49</span>(<span class="issue" tagx="issue" title="issue">21</span>), <span class="fpage" tagx="fpage" title="fpage">6177</span>â€"<span class="lpage" tagx="lpage" title="lpage">6196</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jm051256o">10.1021/jm051256o</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17034125">17034125</a></span></span></li> <li tag="ref"><a name="CIT0019" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Genheden</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryde</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities</span>. <span class="source" tagx="source" title="source"><i>Expert Opinion on Drug Discovery</i></span>, <span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">449</span>â€"<span class="lpage" tagx="lpage" title="lpage">461</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1517/17460441.2015.1032936">10.1517/17460441.2015.1032936</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25835573">25835573</a></span></span></li> <li tag="ref"><a name="CIT0020" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gordon</span><span class="given-names" tagx="given-names" title="given-names">D. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jang</span><span class="given-names" tagx="given-names" title="given-names">G. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouhaddou</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obernier</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oâ€™Meara</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">J. Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swaney</span><span class="given-names" tagx="given-names" title="given-names">D. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tummino</span><span class="given-names" tagx="given-names" title="given-names">T. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huettenhain</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaake</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richards</span><span class="given-names" tagx="given-names" title="given-names">A. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tutuncuoglu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foussard</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Batra</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Modak</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krogan</span><span class="given-names" tagx="given-names" title="given-names">N. J.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">A SARS-CoV-2-human proteinâ€"protein interaction map reveals drug targets and potential drug-repurposing</span>. <span class="source" tagx="source" title="source"><i>BioRxiv</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.03.22.002386">10.1101/2020.03.22.002386</a></span></span></li> <li tag="ref"><a name="CIT0021" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">M. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vemula</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donde</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouda</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Behera</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vadde</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">In-silico approaches to detect inhibitors of the human severe acute respiratory syndrome coronavirus envelope protein ion channel</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">11</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1751300">10.1080/07391102.2020.1751300</a></span></span></li> <li tag="ref"><a name="CIT0022" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halgren</span><span class="given-names" tagx="given-names" title="given-names">T. A.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Identifying and characterizing binding sites and assessing druggability</span>. <span class="source" tagx="source" title="source"><i>Journal of Chemical Information and Modeling</i></span>, <span class="volume" tagx="volume" title="volume">49</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">377</span>â€"<span class="lpage" tagx="lpage" title="lpage">389</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/ci800324m">10.1021/ci800324m</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19434839">19434839</a></span></span></li> <li tag="ref"><a name="CIT0023" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halgren</span><span class="given-names" tagx="given-names" title="given-names">T. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">R. B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friesner</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beard</span><span class="given-names" tagx="given-names" title="given-names">H. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frye</span><span class="given-names" tagx="given-names" title="given-names">L. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pollard</span><span class="given-names" tagx="given-names" title="given-names">W. T.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banks</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span>. <span class="source" tagx="source" title="source"><i>Journal of Medicinal Chemistry</i></span>, <span class="volume" tagx="volume" title="volume">47</span>(<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">1750</span>â€"<span class="lpage" tagx="lpage" title="lpage">1759</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jm030644s">10.1021/jm030644s</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15027866">15027866</a></span></span></li> <li tag="ref"><a name="CIT0024" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paray</span><span class="given-names" tagx="given-names" title="given-names">B. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hussain</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qadir</span><span class="given-names" tagx="given-names" title="given-names">F. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Attar</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aziz</span><span class="given-names" tagx="given-names" title="given-names">F. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharifi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Derakhshankhah</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasti</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mehrabi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shahpasand</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saboury</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falahati</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">A review on the cleavage priming of the spike protein on coronavirus by angiotensin-converting enzyme-2 and furin</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1754293">10.1080/07391102.2020.1754293</a></span></span></li> <li tag="ref"><a name="CIT0025" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, â€¦ <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <span class="source" tagx="source" title="source"><i>The Lancet</i></span>, <span class="volume" tagx="volume" title="volume">395</span>(<span class="issue" tagx="issue" title="issue">10223</span>), <span class="fpage" tagx="fpage" title="fpage">497</span>â€"<span class="lpage" tagx="lpage" title="lpage">506</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30183-5">10.1016/S0140-6736(20)30183-5</a></span></span></li> <li tag="ref"><a name="CIT0026" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parves</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">A. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uddin</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rahman</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mamun</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossain</span><span class="given-names" tagx="given-names" title="given-names">M. N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halim</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">A molecular modeling approach to identify effective antiviral phytochemicals against the main protease of SARS-CoV-2</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure &amp;amp; Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1761883">10.1080/07391102.2020.1761883</a></span></span></li> <li tag="ref"><a name="CIT0027" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shan</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Crystal structure of SARS-CoV-2 nucleocapsid protein RNA binding domain reveals potential unique drug targeting sites</span>. <span class="source" tagx="source" title="source"><i>BioRxiv</i></span>. <span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.03.06.977876">10.1101/2020.03.06.977876</a></span></span></li> <li tag="ref"><a name="CIT0028" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">S. Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">C. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">Y. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">M. H.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Structural basis for the identification of the N-terminal domain of coronavirus nucleocapsid protein as an antiviral target</span>. <span class="source" tagx="source" title="source"><i>Journal of Medicinal Chemistry</i></span>, <span class="volume" tagx="volume" title="volume">57</span>(<span class="issue" tagx="issue" title="issue">6</span>), <span class="fpage" tagx="fpage" title="fpage">2247</span>â€"<span class="lpage" tagx="lpage" title="lpage">2257</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1021/jm500089r">10.1021/jm500089r</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24564608">24564608</a></span></span></li> <li tag="ref"><a name="CIT0029" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipinski</span><span class="given-names" tagx="given-names" title="given-names">C. A.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Lead- and drug-like compounds: The rule-of-five revolution</span>. <span class="source" tagx="source" title="source"><i>Drug Discovery Today: Technologies</i></span>, <span class="volume" tagx="volume" title="volume">1</span>(<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">337</span>â€"<span class="lpage" tagx="lpage" title="lpage">341</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ddtec.2004.11.007">10.1016/j.ddtec.2004.11.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24981612">24981612</a></span></span></li> <li tag="ref"><a name="CIT0030" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lo</span><span class="given-names" tagx="given-names" title="given-names">Y. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">S. Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">S. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C. T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiu</span><span class="given-names" tagx="given-names" title="given-names">Y. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">T. H.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">M. H.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Oligomerization of the carboxyl terminal domain of the human coronavirus 229E nucleocapsid protein</span>. <span class="source" tagx="source" title="source"><i>FEBS Letters</i></span>, <span class="volume" tagx="volume" title="volume">587</span>(<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">120</span>â€"<span class="lpage" tagx="lpage" title="lpage">127</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.febslet.2012.11.016">10.1016/j.febslet.2012.11.016</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23178926">23178926</a></span></span></li> <li tag="ref"><a name="CIT0031" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madhavi Sastry</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adzhigirey</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Annabhimoju</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sherman</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Protein and ligand preparation: Parameters, protocols, and influence on virtual screening enrichments</span>. <span class="source" tagx="source" title="source"><i>Journal of Computer-Aided Molecular Design</i></span>, <span class="volume" tagx="volume" title="volume">27</span>(<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">221</span>â€"<span class="lpage" tagx="lpage" title="lpage">234</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s10822-013-9644-8">10.1007/s10822-013-9644-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23579614">23579614</a></span></span></li> <li tag="ref"><a name="CIT0032" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahanta</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chowdhury</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gogoi</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goswami</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borah</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chetia</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borah</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buragohain</span><span class="given-names" tagx="given-names" title="given-names">A. K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gogoi</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Potential anti-viral activity of approved repurposed drug against main protease of SARS-CoV-2: An in silico based approach</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">15</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1768902">10.1080/07391102.2020.1768902</a></span></span></li> <li tag="ref"><a name="CIT0033" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McBride</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Zyl</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fielding</span><span class="given-names" tagx="given-names" title="given-names">B. C.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">The coronavirus nucleocapsid is a multifunctional protein</span>. <span class="source" tagx="source" title="source"><i>Viruses</i></span>, <span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">2991</span>â€"<span class="lpage" tagx="lpage" title="lpage">3018</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3390/v6082991">10.3390/v6082991</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25105276">25105276</a></span></span></li> <li tag="ref"><a name="CIT0034" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mittal</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumari</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srivastava</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asthana</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of potential molecules against COVID-19 main protease through structure-guided virtual screening approach</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">26</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1768151">10.1080/07391102.2020.1768151</a></span></span></li> <li tag="ref"><a name="CIT0035" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pant</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ravichandiran</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murty</span><span class="given-names" tagx="given-names" title="given-names">U. S. N.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srivastava</span><span class="given-names" tagx="given-names" title="given-names">H. K.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Peptide-like and small-molecule inhibitors against Covid-19</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure and Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">15</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1757510">10.1080/07391102.2020.1757510</a></span></span></li> <li tag="ref"><a name="CIT0036" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paules</span><span class="given-names" tagx="given-names" title="given-names">C. I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marston</span><span class="given-names" tagx="given-names" title="given-names">H. D.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fauci</span><span class="given-names" tagx="given-names" title="given-names">A. S.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Coronavirus infections-more than just the common cold</span>. <span class="source" tagx="source" title="source"><i>JAMA â€" Journal of the American Medical Association</i></span>, <span class="volume" tagx="volume" title="volume">323</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">707</span><span class="pub-id"><a href="https://dx.doi.org/10.1001/jama.2020.0757">10.1001/jama.2020.0757</a></span></span></li> <li tag="ref"><a name="CIT0037" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarma</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sekhar</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prajapat</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avti</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaur</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prakash</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dhibar</span><span class="given-names" tagx="given-names" title="given-names">D. P.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medhi</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">In silico homology assisted identification of inhibitor of RNA binding against 2019-nCoV N-protein (N terminal domain)</span>. <span class="source" tagx="source" title="source"><i>Journal of Biomolecular Structure &amp;amp; Dynamics</i></span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">11</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/07391102.2020.1753580">10.1080/07391102.2020.1753580</a></span></span></li> <li tag="ref"><a name="CIT0038" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuki</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ogura</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Honma</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Construction of an integrated database for hERG blocking small molecules</span>. <span class="source" tagx="source" title="source"><i>PLoS One</i></span>, <span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">e0199348</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0199348">10.1371/journal.pone.0199348</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29979714">29979714</a></span></span></li> <li tag="ref"><a name="CIT0039" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shannon</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le</span><span class="given-names" tagx="given-names" title="given-names">N. T.-T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selisko</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eydoux</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alvarez</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guillemot</span><span class="given-names" tagx="given-names" title="given-names">J.-C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Decroly</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peersen</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferron</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Canard</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">Remdesivir and SARS-CoV-2: Structural requirements at both nsp12 RdRp and nsp14 exonuclease active-sites</span>. <span class="source" tagx="source" title="source"><i>Antiviral Research</i></span>, <span class="volume" tagx="volume" title="volume">178</span>, <span class="fpage" tagx="fpage" title="fpage">104793</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2020.104793">10.1016/j.antiviral.2020.104793</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32283108">32283108</a></span></span></li> <li tag="ref"><a name="CIT0041" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Surjit</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chow</span><span class="given-names" tagx="given-names" title="given-names">V. T. K.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lal</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span></span> (<span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">The nucleocapsid protein of severe acute respiratory syndrome-coronavirus inhibits the activity of cyclinâ€"cyclin-dependent kinase complex and blocks S phase progression in mammalian cells</span><span class="source" tagx="source" title="source"><i>. <i>The Journal of Biological Chemistry</i></i></span>, <span class="volume" tagx="volume" title="volume">281</span>(<span class="issue" tagx="issue" title="issue">16</span>), <span class="fpage" tagx="fpage" title="fpage">10669</span>â€"<span class="lpage" tagx="lpage" title="lpage">10681</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M509233200">10.1074/jbc.M509233200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16431923">16431923</a></span></span></li> <li tag="ref"><a name="CIT0042" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhan</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">G. F.</span></span>, &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span> (<span class="year" tagx="year" title="year">2020</span>). <span class="mixed-article-title" title="mixed-article-title">A novel coronavirus from patients with pneumonia in China, 2019</span>. <span class="source" tagx="source" title="source"><i>The New England Journal of Medicine</i></span>, <span class="volume" tagx="volume" title="volume">382</span>(<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">727</span>â€"<span class="lpage" tagx="lpage" title="lpage">733</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMoa2001017">10.1056/NEJMoa2001017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31978945">31978945</a></span></span></li> </ul> </div> <div class="glossary" title="glossary"> <div class="def-list" title="def-list"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-item" title="def-item"> <div class="term" title="term">
CoV</div> <div class="def" title="def"> <p>coronavirus</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
COVID-19</div> <div class="def" title="def"> <p>coronavirus disease 2019</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
FDA</div> <div class="def" title="def"> <p>food and drug administration</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HB</div> <div class="def" title="def"> <p>hydrogen bond</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HTVS</div> <div class="def" title="def"> <p>high throughput virtual screening</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MD</div> <div class="def" title="def"> <p>molecular dynamics</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MERS</div> <div class="def" title="def"> <p>Middle East Respiratory Syndrome</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PDB</div> <div class="def" title="def"> <p>Protein Data Bank</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RNA</div> <div class="def" title="def"> <p>ribonucleic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SARS</div> <div class="def" title="def"> <p>Sudden Acute Respiratory Syndrome</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SP</div> <div class="def" title="def"> <p>standard precision</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
WHO</div> <div class="def" title="def"> <p>World Health Organization</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
XP</div> <div class="def" title="def"> <p>extra precision</p> </div> </div> </div> </div> </div>  </body></html>