<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Int J Mol Sci</span><span class="iso-abbrev" title="iso-abbrev">Int J Mol Sci</span><span class="publisher-id" title="publisher-id">ijms</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">International Journal of Molecular Sciences</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1422-0067</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/ijms21093099">10.3390/ijms21093099</a></span><span class="publisher-id" title="publisher-id">ijms-21-03099</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Structural and Evolutionary Analysis Indicate That the SARS-CoV-2 Mpro Is a Challenging Target for Small-Molecule Inhibitor Design</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-6802-8753</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BzÃ³wka</span><span class="given-names" tagx="given-names" title="given-names">Maria</span></span><a href="#af1-ijms-21-03099">1</a><a href="#fn1-ijms-21-03099">â€ </a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-0183-0845</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MitusiÅ"ska</span><span class="given-names" tagx="given-names" title="given-names">Karolina</span></span><a href="#af1-ijms-21-03099">1</a><a href="#fn1-ijms-21-03099">â€ </a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RaczyÅ"ska</span><span class="given-names" tagx="given-names" title="given-names">Agata</span></span><a href="#af1-ijms-21-03099">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Samol</span><span class="given-names" tagx="given-names" title="given-names">Aleksandra</span></span><a href="#af1-ijms-21-03099">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-9976-0429</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TuszyÅ"ski</span><span class="given-names" tagx="given-names" title="given-names">Jack A.</span></span><a href="#af2-ijms-21-03099">2</a><a href="#af3-ijms-21-03099">3</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-2530-6957</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ³ra</span><span class="given-names" tagx="given-names" title="given-names">Artur</span></span><a href="#af1-ijms-21-03099">1</a><a href="#c1-ijms-21-03099">*</a></span></div><span class="citation_author_institution" id="af1-ijms-21-03099">[1], </span><span class="citation_author_institution" id="af2-ijms-21-03099">[2], </span><span class="citation_author_institution" id="af3-ijms-21-03099">[3], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">a.gora@tunnelinggroup.pl</span>; Tel.: +48-32-237-16-59</div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">â€ </span><p>These authors contributed equally to this work.</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-5</span></span><span class="volume" tagx="volume" title="volume">21</span><span class="issue" tagx="issue" title="issue">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">3099</span><span class="history" title="history"><span class="received" title="received">received: 2020-3-28</span><span class="accepted" title="accepted">accepted: 2020-4-26</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The novel coronavirus whose outbreak took place in December 2019 continues to spread at a rapid rate worldwide. In the absence of an effective vaccine, inhibitor repurposing or de novo drug design may offer a longer-term strategy to combat this and future infections due to similar viruses. Here, we report on detailed classical and mixed-solvent molecular dynamics simulations of the main protease (Mpro) enriched by evolutionary and stability analysis of the protein. The results were compared with those for a highly similar severe acute respiratory syndrome (SARS) Mpro protein. In spite of a high level of sequence similarity, the active sites in both proteins showed major differences in both shape and size, indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, analysis of the binding siteâ€™s conformational changes during the simulation time indicated its flexibility and plasticity, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the protein with respect to flexible loop mutations indicated that the virusâ€™ mutability will pose a further challenge to the rational design of small-molecule inhibitors. However, few residues contribute significantly to the protein stability and thus can be considered as key anchoring residues for Mpro inhibitor design.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
coronavirus</div> <div class="kwd" title="kwd">
SARS-CoV</div> <div class="kwd" title="kwd">
SARS-CoV-2</div> <div class="kwd" title="kwd">
COVID-19</div> <div class="kwd" title="kwd">
molecular dynamics simulations</div> <div class="kwd" title="kwd">
ligand tracking approach</div> <div class="kwd" title="kwd">
drug design</div> <div class="kwd" title="kwd">
small-molecule inhibitors</div> <div class="kwd" title="kwd">
evolutionary analysis</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>In early December 2019, the first atypical pneumonia outbreak associated with the novel coronavirus of zoonotic origin (SARS-CoV-2) appeared in Wuhan City, Hubei Province, China [<a href="#B1-ijms-21-03099">1</a>,<a href="#B2-ijms-21-03099">2</a>]. In general, coronaviruses (CoVs) are classified into four major genera: Alphacoronavirus, Betacoronavirus (which primarily infect mammals), Gammacoronavirus, and Deltacoronavirus (which primarily infect birds) [<a href="#B3-ijms-21-03099">3</a>,<a href="#B4-ijms-21-03099">4</a>,<a href="#B5-ijms-21-03099">5</a>]. In humans, coronaviruses usually cause mild to moderate upper-respiratory tract illnesses, such as the common cold, however, the rarer forms of CoVs can be lethal. By the end of 2019, six kinds of human CoV have been identified: HCoV-NL63; HCoV-229E, belonging to Alphacoronavirus genera; HCoV-OC43; HCoV-HKU1; severe acute respiratory syndrome (SARS-CoV); and Middle East respiratory syndrome (MERS-CoV), belonging to Betacoronavirus genera [<a href="#B4-ijms-21-03099">4</a>]. Of the aforementioned CoVs, the latter two are the most dangerous and have been associated with the outbreak of two epidemics at the beginning of the 21st century [<a href="#B6-ijms-21-03099">6</a>]. In January 2020, SARS-CoV-2 was isolated and announced as a new, seventh type of human coronavirus. It was classified as a Betacoronavirus [<a href="#B2-ijms-21-03099">2</a>]. On the basis of the phylogenetic analysis of the genomic data of SARS-CoV-2, Zhang et al. indicated that SARS-CoV-2 is most closely related to two SARS-CoV sequences isolated from bats in 2015 and 2017. This suggests that the bat CoV and SARS-CoV-2 share a common ancestor, and the new virus can be considered as a SARS-like virus [<a href="#B7-ijms-21-03099">7</a>].</p> <p>The genome of coronaviruses typically contains a positive-sense, single-stranded RNA but it differs in size ranging between â‰ˆ26 and â‰ˆ32 kb. It also includes a variable number of open reading frames (ORFs), from 6 to 11. The first ORF is the largest, encoding nearly 70% of the entire genome and 16 non-structural proteins (nsps) [<a href="#B3-ijms-21-03099">3</a>,<a href="#B8-ijms-21-03099">8</a>]. Of the nsps, the main protease (Mpro, also known as a chymotrypsin-like cysteine protease, 3CLpro), encoded by nsp5, has been found to play a fundamental role in viral gene expression and replication, and thus it is an attractive target for anti-CoV drug design [<a href="#B9-ijms-21-03099">9</a>]. The remaining ORFs encode accessory and structural proteins, including spike surface glycoprotein (S), small envelope protein (E), matrix protein (M), and nucleocapsid protein (N).</p> <p>On the basis of the three sequenced genomes of SARS-CoV-2 (Wuhan/IVDC-HB-01/2019, Wuhan/IVDC-HB-04/2019, and Wuhan/IVDC-HB-05/2019, provided by the National Institute for Viral Disease Control and Prevention, CDC, China), Wu et al. performed a detailed genome annotation. The results were further compared to related coronavirusesâ€"1008 human SARS-CoV, 338 bat SARS-like CoV, and 3131 human MERS-CoV, indicating that the three strains of SARS-CoV-2 have almost identical genomes with 14 ORFs, encoding 27 proteins including 15 non-structural proteins (nsp1â€"10 and nsp12â€"16), 4 structural proteins (S, E, M, N), and 8 accessory proteins (3a, 3b, p6, 7a, 7b, 8b, 9b, and orf14). The only identified difference in the genome consisting of â‰ˆ29.8 kb nucleotides consisted of five nucleotides. The genome annotation revealed that SARS-CoV-2 is fairly similar to SARS-CoV at the amino acid level, however, there are some differences in the occurrence of accessory proteins, such as the fact that the 8a accessory protein, present in SARS-CoV, is absent in SARS-CoV-2, as well as the fact that the lengths of 8b and 3b proteins do not match. The phylogenetic analysis of SARS-CoV-2 showed it to be most closely related to SARS-like bat viruses, but no strain of SARS-like bat virus was found to cover all equivalent proteins of SARS-CoV-2 [<a href="#B10-ijms-21-03099">10</a>].</p> <p>As previously mentioned, the main protease is one of the key enzymes in the viral life cycle. Together with other non-structural proteins (papain-like protease, helicase, RNA-dependent RNA polymerase) and the spike glycoprotein structural protein, it is essential for interactions between the virus and host cell receptor during viral entry [<a href="#B11-ijms-21-03099">11</a>]. Initial analyses of genomic sequences of the four nsps mentioned above indicate that those enzymes are highly conserved, sharing more than 90% sequence similarity with the corresponding SARS-CoV enzymes [<a href="#B12-ijms-21-03099">12</a>].</p> <p>The first released crystal structure of the Mpro of SARS-CoV-2 (PDB) ID: 6lu7) was obtained by Prof. Yangâ€™s group from ShanghaiTech by co-crystallisation with a peptide-like inhibitor N-[(5 methylisoxazol-3-yl)carbonyl]alanyl-L-valyl-N~1-((1R,2Z)-4-(benzyloxy)-4-oxo-1-{[(3R)-2-oxopyrrolidin-3-yl]methyl}but-2-enyl)-L-leucinamide (N3 or PRD_002214) [<a href="#B13-ijms-21-03099">13</a>]. The same inhibitor was co-crystallised with other human coronaviruses, such as HCoV-NL63 (PDB ID: 5gwy), HCoV-KU1 (PDB ID: 3d23), or SARS-CoV (PDB ID: 2amq). This enzyme naturally forms a dimer, each of whose monomer consists of the N-terminal catalytic region and a C-terminal region [<a href="#B14-ijms-21-03099">14</a>]. Although 12 residues differ between both CoVs, only one, namely, S46 in SARS-CoV-2 (A46 in SARS-CoV), is located in the proximity of the entrance to the active site. However, such a small structural change would typically be not expected to substantially affect the binding of small molecules [<a href="#B12-ijms-21-03099">12</a>]. Such an assumption would routinely involve the generation of a library of derivatives and analogous on the basis of the scaffold of a drug that inhibits the corresponding protein in the SARS-CoV case. As shown in the present paper, regrettably, this strategy is not likely to succeed with SARS-CoV-2 for Mpro as a molecular target.</p> <p>In this study, we investigated how only 12 different residues, located mostly on the proteinâ€™s surface, may affect the behaviour of the active site pocket of the SARS-CoV-2 Mpro protein. To this end, we performed classical molecular dynamics simulations (cMD) of both SARS and SARS-CoV-2 Mpros, as well as mixed-solvents molecular dynamics simulations (MixMD) combined with small moleculesâ€™ tracking approach to analyse the conformational changes in the binding site. The experiment setup and methodology workflow is presented in <a href="#app1-ijms-21-03099">Figure S1</a>. Despite the structural differences in the active sites of both Mpro proteins, major issues involving plasticity and flexibility of the binding site could result in significant difficulties in inhibitor design for this molecular target. Indeed, an in silico attempt has already been made involving a massive virtual screening for Mpro inhibitors of SARS-CoV-2 using Deep Docking [<a href="#B15-ijms-21-03099">15</a>]. Other recent attempts focused on virtual screening for putative inhibitors of the same main protease of SARS-CoV-2 on the basis of the clinically approved drugs [<a href="#B16-ijms-21-03099">16</a>,<a href="#B17-ijms-21-03099">17</a>,<a href="#B18-ijms-21-03099">18</a>,<a href="#B19-ijms-21-03099">19</a>,<a href="#B20-ijms-21-03099">20</a>,<a href="#B21-ijms-21-03099">21</a>], and also on the basis of the compounds from different databases or libraries [<a href="#B22-ijms-21-03099">22</a>,<a href="#B23-ijms-21-03099">23</a>,<a href="#B24-ijms-21-03099">24</a>]. However, none of such attempts is likely to lead to clinical advances in the fight against SARS-CoV-2 for reasons we elaborate below.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
2. Results</div> <div class="2.1.crystalstructurecomparison,andlocationofthereplacedaminoacidsdistaltotheactivesite" title="sec"> <div class="title" tagx="title" title="title">
2.1. Crystal Structure Comparison, and Location of the Replaced Amino Acids Distal to the Active Site</div> <p>The first SARS-CoV-2 main proteaseâ€™s crystallographic structure was made publicly available through the Protein Data Bank (PDB) [<a href="#B25-ijms-21-03099">25</a>] as a complex with an N3 inhibitor (PDB ID: 6lu7) [<a href="#B13-ijms-21-03099">13</a>]. Next, the structure without the inhibitor was also made available (PDB ID: 6y2e) [<a href="#B26-ijms-21-03099">26</a>]. We refer to these structures as SARS-CoV-2 Mpro<sup>N3</sup> and SARS-CoV-2 Mpro, respectively. We also used two structures of the SARS-CoV main protease: one, referred to as SARS-CoV Mpro<sup>N3</sup> (PDB ID: 2amq), co-crystallised with the same N3 inhibitor, and the other without an inhibitor (PDB ID: 1q2w), which we refer to as SARS-CoV Mpro. The SARS-CoV-2 Mpro and SARS-CoV Mpro structures differ by only 12 amino acids located mostly on the proteinsâ€™ surface (<a href="#ijms-21-03099-f001">Figure 1</a>A, <a href="#app1-ijms-21-03099">Table S1</a>). Both enzymes share the same structural composition; they comprise three domains: domains I (residues 1â€"101) and II (residues 102â€"184) consist of an antiparallel Î²-barrel, and the Î±-helical domain III (residues 201â€"301) is required for the enzymatic activity [<a href="#B27-ijms-21-03099">27</a>]. Both enzymes resemble the structure of cysteine proteases, although their active site is lacking the third catalytic residue [<a href="#B28-ijms-21-03099">28</a>]; their active site comprises a catalytic dyad, namely, H41 and C145, and a particularly stable water molecule forms at least three hydrogen bond interactions with surrounding residues, including the catalytic histidine, which corresponds to the position of a third catalytic member (<a href="#ijms-21-03099-f001">Figure 1</a>B). It should be also noted that one of the differing amino acids in SARS-CoV-2 Mpro, namely, S46, is located on a C44-P52 loop, which is flanking the active site cavity.</p> </div> <div class="2.2.plasticityofthebindingcavities" title="sec"> <div class="title" tagx="title" title="title">
2.2. Plasticity of the Binding Cavities</div> <p>A total of 2 Âµs classical molecular dynamics (cMD) simulations of both SARS-CoV-2 and SARS-CoV Mpros with different starting points were run to examine the plasticity of their binding cavities. As different starting points we used (i) SARS-CoV-2 Mpro apo structure; (ii) SARS-CoV-2 Mpro with an N3 ligand, which was removed before starting the simulation; (iii) SARS-CoV Mpro apo structure; and (iv) SARS-CoV Mpro with the same N3 ligand, which was also removed before starting the simulation. A total of 10 replicas of 50 ns classical molecular dynamics simulations were performed for each protein. To improve conformation sampling, the starting geometry for each system was kept but the initial vectors were randomly assigned. A combination of the cMD approach with water molecules used as molecular probes is assumed to provide a highly detailed picture of the proteinâ€™s interior dynamics [<a href="#B29-ijms-21-03099">29</a>]. The small molecules tracking approach was used to determine the accessibility of the active site pocket in both SARS-CoV and SARS-CoV-2 Mpros, and a local distribution approach was used to provide information about an overall distribution of solvent in the proteinsâ€™ interior. To properly examine the flexibility of both active site cavities, we used the time-window mode implemented in AQUA-DUCT software [<a href="#B30-ijms-21-03099">30</a>] to analyse the water moleculesâ€™ flow through the cavity in a 10 ns time step and combined that with the outer pocket calculations to examine the plasticity and maximal accessible volume (MAV) of the binding cavity.</p> <p>Surprisingly, despite their high similarity, the binding cavities of SARS-CoV and SARS-CoV-2 Mpros showed significantly different MAV (Wald test, z = 2597, <i>p</i> &amp;lt; 0.05). Both proteins reduced their MAV upon inhibitor binding by approximately 20%, but the maximal volume of SARS-CoV was over 50% larger than those of SARS-CoV-2 (<a href="#ijms-21-03099-f002">Figure 2</a> and <a href="#app1-ijms-21-03099">Figure S2</a>).</p> </div> <div class="2.3.flexibilityoftheactivesiteentrance" title="sec"> <div class="title" tagx="title" title="title">
2.3. Flexibility of the Active Site Entrance</div> <p>To further examine the plasticity and flexibility of the main proteases binding cavities, we focused on the movements of loops surrounding their entrances and regulating the active sitesâ€™ accessibility. We found that one of the analysed loops of the SARS-CoV Mpro, namely, C44-P52 loop, was more flexible than the corresponding loops of SARS-CoV-2 Mpro structure, whereas the adjacent loops were mildly flexible (<a href="#ijms-21-03099-f003">Figure 3</a>). This could be indirectly assumed from the absence of the C44-P52 loop in the crystallographic structure of SARS-CoV Mpro structure. On the other hand, such flexibility could suggest that the presence of an inhibitor might stabilise the loops surrounding the active site. The analysis of B-factors of all deposited Mpro crystal structures fully confirmed these statements (<a href="#app1-ijms-21-03099">Figure S3</a>). It is worth adding that this loop was carrying the unique SARS-CoV-2 Mpro residue S46.</p> </div> <div class="2.4.cosolventhot-spotsanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.4. Cosolvent Hot-Spots Analysis</div> <p>The mixed-solvent MD simulations were run with six cosolvents: acetonitrile (ACN), benzene (BNZ), dimethylsulfoxide (DMSO), methanol (MEO), phenol (PHN), and urea (URE). Cosolvents were used as specific molecular probes, representing different chemical properties and functional groups that would complement the different regions of the binding site and the protein itself. Using small molecules tracking approach, we analysed the flow through the Mpros structures and identified the regions in which those molecules were being trapped and/or caged, located within the protein itself (global hot-spots; <a href="#app1-ijms-21-03099">Figures S4 and S5</a>) and inside the binding cavity (local hot-spots; <a href="#ijms-21-03099-f004">Figure 4</a> and <a href="#app1-ijms-21-03099">Figure S6</a>). The size and location of both types of hot-spots differed and provided complementary information. The global hot-spots identified potential binding/interacting sites in the whole protein structure and additionally provided information about regions attracting particular types of molecules, whereas local hot-spots described the actual available binding space of a specific cavity.</p> <p>The general distribution of the global hot-spots from particular cosolvents was quite similar and verified specific interactions with the particular regions of the analysed proteins. A notable number of hot-spots were located around the amino acids that varied between the SARS-CoV-2 and SARS-CoV Mpros (<a href="#app1-ijms-21-03099">Figures S4 and S5</a>). The largest number and the densest hot-spots are located within the binding cavity and the region essential for Mpro dimerisation [<a href="#B31-ijms-21-03099">31</a>], between the II and III domains. The binding cavity is particularly occupied by urea, benzene, and phenol hot-spots, which is especially interesting because these solvents exhibit different chemical properties. In addition, the analysis performed by PARS server [<a href="#B32-ijms-21-03099">32</a>] detected three cavities located between domain II and III that could contribute significantly to the protein flexibility; however, none of them was found as conserved and therefore were not considered as regulatory sites.</p> <p>A close inspection of the binding site cavity provided further details of cosolvent distribution. The benzene hot-spots for the SARS-CoV-2 Mpro structure are localised deep inside the active site cavity, whereas SARS-CoV Mpro features mostly benzene hot-spots at the cavity entrance (<a href="#ijms-21-03099-f005">Figure 5</a>). This is interesting because, in the absence of cosolvent molecules, the water accessible volume for SARS-CoV-2 Mpro was 50% smaller than in the case of SARS-CoV Mpro, underlining huge plasticity of the binding cavity and suggesting large conformational changes induced by interaction with a potential ligand. It is also interesting that both global and local hot-spots of the SARS-CoV Mpro structure are located in the proximity of the C44-P52 loop, which potentially regulates the access to the active site.</p> </div> <div class="2.5.potentialmutabilityofsars-cov-2" title="sec"> <div class="title" tagx="title" title="title">
2.5. Potential Mutability of SARS-CoV-2</div> <p>In general, all the above-mentioned findings indicate potential difficulties in the identification of specific inhibitors toward Mpro proteins. First, the binding site itself is characterised by large plasticity (over 20% change of the MAV upon ligand binding) and probably even distant to active site mutations modify Mpro binding properties. Secondly, the C44-P52 loop regulates access to the active site and can contribute to the discrimination of potential inhibitors. Therefore, additional mutations in the above-mentioned regions, which could appear during further SARS-CoV-2 evolution, can significantly change the affinity between Mpro and its ligands. To verify potential threat of further mutability of the Mpro protein, we performed (i) correlated mutation analysis (CMA) on multiple sequence alignment, (ii) the analysis of the contribution of already identified differences between the SARS-CoV and SARS-CoV-2 Mpros to protein stability, and (iii) prediction of further possible mutations caused by the most probable mutations, the substitution of single nucleotides in the mRNA sequence of Mpro.</p> <p>Indeed, the analysis performed with Comulator software [<a href="#B33-ijms-21-03099">33</a>] showed that within viral Mpros, evolutionary-correlated residues are dispersed throughout the structure. This indirectly supports our previous findings that distant amino acid mutation can contribute substantially to the binding site plasticity. It is worth adding that among evolutionary-correlated residues, we identified also those that differ between SARS-CoV-2 and SARS-CoV Mpros, located on the C44-P52 loop (<a href="#ijms-21-03099-f005">Figure 5</a>) and the F185-T201 linker loop. The CMA analysis indicated that particular residues in both loops are evolutionary-correlated. The Q189 from the linker loop correlates with residues from the C44-P52 loop, whereas R188, A191, and A194 correlate with selected residues from all domains, but not with the C44-P52 loop. As was shown, the mutation of amino acids distant from active site residues, which are evolutionary correlated, is most likely to modify the active site accessibility [<a href="#B34-ijms-21-03099">34</a>].</p> <p>In the interest of examining the energetical effect of the 12 amino acid replacement in the SARS-CoV-2 Mpro structure, we calculated their energetic contributions to the proteinâ€™s stability using FoldX [<a href="#B35-ijms-21-03099">35</a>]. As expected, the differences in total energies of the SARS-CoV Mpro and variants with introduced mutation from SARS-CoV-2 Mpro residue did not represent a significant energy change (<a href="#app1-ijms-21-03099">Table S1</a>). The biggest energy reduction was found for mutation H134F (âˆ’0.85 kcal/mol) and mutations R88K, S94A, T285A, and I286L only slightly reduced the total energy (<a href="#app1-ijms-21-03099">Table S1</a>).</p> <p>To investigate further possible mutations of SARS-CoV-2 Mpro, single nucleotide substitutions were introduced to the SARS-CoV-2 main protease gene. If a substitution of a single nucleotide caused translation to a different amino acid compared to the corresponding residue in the wild-type structure, an appropriate mutation was proposed with FoldX calculations. The most energetically favourable potential mutations were chosen on the basis of a âˆ’1.5 kcal/mol threshold (<a href="#ijms-21-03099-f006">Figure 6</a>A, <a href="#app1-ijms-21-03099">Table S2</a>). Most of the energetically favourable potential mutations include amino acids that are solvent-exposed on the proteinâ€™s surface, according to NetSurfP [<a href="#B36-ijms-21-03099">36</a>] results. These results show that in general, exposed amino acids are more likely to mutate.</p> <p>Additionally, the potential mutability of the binding cavity was investigated. Residues belonging to the binding cavity were found within 7 â"« from the N3 inhibitor. Then, we calculated the differences in the Gibbs free energy of protein folding with respect to the wild-type protein (<a href="#app1-ijms-21-03099">Table S3</a>) and presented the results as a heat map. The most energetically favourable potential mutations are shown as green, neutral as white, and unfavourable as red (<a href="#ijms-21-03099-f006">Figure 6</a>B). Interestingly, residues forming the catalytic dyad, namely, H41 and C145, are also prone to mutate. However, probably the most important message comes from the analysis of the potential mutability of the C44-P52 loop. Mutation of four of them has a stabilising effect for the protein, and is near-neutral for the rest the effect. These results indicate that the future evolution of the Mpro protein can significantly reduce the potential use of this protein as a molecular target for coronavirus treatment due to a highly probable development of drug resistance of this virus through mutations.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
3. Discussion</div> <p>The analysis of water moleculesâ€™ distribution and trajectories can be used for the analysis of proteinsâ€™ structural features and biochemical properties. It also provides additional support to the drug design and investigation of protein interior [<a href="#B29-ijms-21-03099">29</a>,<a href="#B37-ijms-21-03099">37</a>,<a href="#B38-ijms-21-03099">38</a>]. As we have shown in previous research, tracking of water molecules in the binding cavity combined with the local distribution approach can identify catalytic water positions [<a href="#B39-ijms-21-03099">39</a>]. Indeed, despite differences in the size and dynamics of the binding cavities of SARS-CoV and SARS-CoV-2 Mpros, the main identified water was always found in a position next to the H41 residue (<a href="#ijms-21-03099-f002">Figure 2</a>), and this location is assumed to indicate catalytic water of Mpro replacing the missing third catalytic site amino acid [<a href="#B28-ijms-21-03099">28</a>]. That was the first quality check of our methodology that approved our approach, and has initiated further investigations.</p> <p>As reported in the previous research, the overall plasticity of Mpro is required for proper enzyme functioning [<a href="#B26-ijms-21-03099">26</a>,<a href="#B40-ijms-21-03099">40</a>]. In the case of SARS-CoV the truncation of the linker loop (F185-T201) gave rise to a significant reduction in protein activity and confirmed that the proper orientation of the linker allows the shift between dimeric and monomeric forms [<a href="#B41-ijms-21-03099">41</a>]. Dimerisation of the enzyme is necessary for its catalytic activity, and the proper conformation of the seven N-terminal residues (N-finger) is required [<a href="#B42-ijms-21-03099">42</a>]. In SARS-CoV-2 Mpro, the T285 is replaced by alanine, and the I286 by leucine. It has been shown that replacing S284, T285, and I286 by alanine residues in SARS-CoV Mpro leads to a 3.6-fold enhancement of the catalytic activity of the enzyme. This is accompanied by changes in the structural dynamics of the enzyme that transmit the effect of the mutation to the catalytic centre. Indeed, the T285A replacement observed in the SARS-CoV-2 Mpro allows the two domains III to approach each other a little closer [<a href="#B43-ijms-21-03099">43</a>].</p> <p>The comparison of MD simulations of both main proteases initiated from different starting conformations (with and without N3 inhibitor) suggests that besides plasticity of the whole protein, there can be large differences between the accessibility to the binding cavity and/or the accommodation of the shape of the cavity in response to the inhibitor that can be bound. There are also differences in the outer pocketsâ€™ maximal accessible volumes between the two structures of SARS-CoV main proteases; the apo SARS-CoV Mpro structure used as a starting point of MD simulations has shown the largest MAV of all the analysed systems. These results suggest that the SARS-CoV main proteasesâ€™ binding cavity is highly flexible and changes both in volume and shape, significantly altering the ligand binding. This finding indicates a serious obstacle for a classical virtual screening approach and drug design in general. Numerous novel compounds that are considered as potential inhibitors of SARS-CoV have not reached the stage of clinical trials. The lack of success might be related to the above-mentioned plasticity of the binding cavity. Some of these compounds have been used for docking and virtual screening research aimed not only at SARS-CoV [<a href="#B44-ijms-21-03099">44</a>,<a href="#B45-ijms-21-03099">45</a>] but also at the novel SARS-CoV-2 [<a href="#B15-ijms-21-03099">15</a>,<a href="#B16-ijms-21-03099">16</a>,<a href="#B17-ijms-21-03099">17</a>,<a href="#B18-ijms-21-03099">18</a>,<a href="#B19-ijms-21-03099">19</a>,<a href="#B20-ijms-21-03099">20</a>,<a href="#B21-ijms-21-03099">21</a>]. Such an approach focuses mostly on the structural similarity between the binding pockets, but ignores the fact that the actual available binding space differs significantly. In general, a rational drug design can be a very successful tool in the identification of possible inhibitors in cases where the atomic resolution structure of the target protein is known. For a new target, when a highly homologous structure is available, a very logical strategy would be seeking chemically similar compounds or creating derivatives of this inhibitor, as well as finding those compounds that are predicted to have a higher affinity for the new target structure than the original one. This would be expected to work for SARS-CoV-2 proteins (such as Mpro) using SARS-CoV proteins as templates. However, our in-depth analysis indicates a very different situation taking place, with major shape and size differences emerging due to the binding site flexibility. Therefore, repurposing SARS drugs against COVID-19 may not be successful due to major shape and size differences, and despite docking methods, the enhanced sampling should be considered.</p> <p>The continuous effort of Diamond Light Source group [<a href="#B46-ijms-21-03099">46</a>] performing massive XChem crystallographic fragment screen against Mpro has resulted in 22 non-covalent hits in the active site and 44 covalent hits in the active site (March 17th). Interestingly, two hits were identified on the dimer interface. The positions of the hits inside the active site overlap with the position of the maximal accessible volume calculated from MD simulations and supports our finding on large binding site flexibility (<a href="#app1-ijms-21-03099">Figure S7</a>).</p> <p>The analysis of the water hot-spots shows the catalytic water hot-spot dominated water distribution inside the binding cavity. The remaining water hot-spots corresponded to a much lower water density level and were on the borders of the binding cavity, which suggests a rather hydrophobic or neutral interior of the binding cavity. The MixMD simulations performed with various cosolvents further confirmed these observations. The largest number and the densest hot-spots were located within the binding cavity and the region essential for Mpro dimerisation [<a href="#B31-ijms-21-03099">31</a>], between the II and III domains. The deep insight into the local hot-spot distribution of the various cosolvents underlines the large differences in binding sites plasticity. The smaller binding cavity of the SARS-CoV-2 enlarged significantly in the presence of a highly hydrophobic cosolvent. The benzene hot-spots were detected deep inside the cavity, and also near the C44-P52 loop. In contrast, in the case of SARS-CoV, benzene hot-spots were located only in the vicinity of the C44-P52 loop. Such a conclusion may also imply that a sufficiently potent inhibitor of SARS-CoV and/or SARS-CoV-2 Mpros needs to be able to open its way to the active site before it can successfully bind to its cavity. These results support the regulatory role of the C44-P52 loop and again alert against unwarranted use of simplified approaches for drug repositioning or docking.</p> <p>The difficulties in targeting the active site of the Mpros are also explained by evolutionary study and potential mutability analysis. As already pointed out, the C44-P52 loop is likely to regulate the access to the active site by enabling entrance of favourable small molecules and blocking the entry of unfavourable ones. The second important loop, F185-T201, which starts in the vicinity of the binding site and links I and II domains with the III domain contributes significantly to Mpro dimerisation [<a href="#B41-ijms-21-03099">41</a>].</p> <p>The initial analysis of the effect of the 12 amino acid replacements in SARS-CoV Mpro on the SARS-CoV-2 Mpro structure stability was expected to provide neutral or stabilising contribution to protein folding. Indeed, all replacements were found to stabilise the proteinâ€™s folding (e.g., H134F: âˆ’0.85 kcal/mol) or have an almost neutral character (e.g., R88K, S94A, T285A, I286L). The analysis of the potential risk of further Mpro structure evolution within the binding cavity suggests that mutations of residues that contribute to ligand binding or access to the active site are energetically favourable, and are likely to occur. Some of the residues that are prone to mutate would provide the inactive enzyme (e.g., the residues forming the catalytic dyad) and therefore could be considered as a blind alley in enzyme evolution, but others (e.g., amino acids from the C44-P52 loop, T45, S46, E47, L50) could significantly modify the inhibitors binding mode of Mpro. The locations of residues on the regulatory loop, which are prone to mutate puts in question the efforts to design inhibitors of the MPro active site as a viable long-term strategy. However, our results also indicate residues that are energetically unfavourable to mutate (e.g., P39, R40, P52, G143, G146, or L167), which could provide an anchor for successful drug design that can outlast coronavirus Mpro variability in future. Alternatively, we would suggest targeting the region between II and III domains, which contributes to the dimer formation.</p> </div> <div class="4.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
4. Materials and Methods</div> <div class="4.1.classicalmdsimulations" title="sec"> <div class="title" tagx="title" title="title">
4.1. Classical MD Simulations</div> <p>The H++ server [<a href="#B47-ijms-21-03099">47</a>] was used to protonate the SARS-CoV-2 (PDB IDs: 6lu7, and 6y2e) and SARS-CoV main proteasesâ€™ structures (PDB IDs: 2amq, and 1q2w) using standard parameters at pH 7.4. The missing 4-amino-acids-long loop of the 1q2w model was added using the corresponding loop of the 6lu7 model, and the quality of the loop refinement was confirmed by comparison with 2h2z structure of SARS-CoV (<a href="#app1-ijms-21-03099">Figure S8</a>). Additionally, 4 and 3 Na<sup>+</sup> ions were added to the SARS-CoV-2 and the SARS-CoV, respectively. Water molecules were placed using the combination of 3D-RISM [<a href="#B48-ijms-21-03099">48</a>] and the Placevent algorithm [<a href="#B49-ijms-21-03099">49</a>]. The AMBER 18 LEaP [<a href="#B50-ijms-21-03099">50</a>] was used to immerse models in a truncated octahedral box with 12 Ã… radius of TIP3P water molecules and prepare the systems for simulation using the ff14SB force field. PMEMD CUDA package of AMBER 18 software [<a href="#B50-ijms-21-03099">50</a>] was used to run a total of 2 Î¼s (10 replicas of 50 ns for each system) simulations of both SARS-CoV-2 and SARS-CoV Mpros systems using apo structures and structure with co-crystalised N3 inhibitor (removed before starting the simulations) to provide more starting points for simulations. To improve conformation sampling, the starting geometry for each system was kept but the initial vectors were randomly assigned. The minimisation procedure consisted of 2000 steps, involving 1000 steepest descent steps followed by 1000 steps of conjugate gradient energy minimisation, with decreasing constraints on the protein backbone (500, 125, and 25 kcal Ã— mol<sup>âˆ’1</sup> Ã— Ã…<sup>âˆ’2</sup>) and a final minimisation with no constraints of conjugate gradient energy minimization. Next, gradual heating was performed from 0 K to 300 K over 20 ps using a Langevin thermostat with a collision frequency of 1.0 ps<sup>âˆ’1</sup> in periodic boundary conditions with constant volume. Equilibration stage was run using the periodic boundary conditions with constant pressure for 1 ns with 1 fs step using Langevin dynamics with a frequency collision of 1 ps<sup>âˆ’1</sup> to maintain temperature. Production stage was run for 50 ns with a 2 fs time step using Langevin dynamics with a collision frequency of 1 ps<sup>âˆ’1</sup> to maintain constant temperature. Long-range electrostatic interactions were modelled using the particle mesh Ewald method with a non-bonded cut-off of 10 Ã… and the SHAKE algorithm. The coordinates were saved at an interval of 1 ps. The number of added water molecules is shown in <a href="#app1-ijms-21-03099">Table S4</a>.</p> <p>Because our analysis was focused on the binding site that is surrounded by short loops only, to keep a reasonable combination of the number and length of simulations, the single simulation length was set to 50 ns. As has been shown elsewhere [<a href="#B51-ijms-21-03099">51</a>,<a href="#B52-ijms-21-03099">52</a>], longer simulations do not provide additional information and could even have a tendency to move away from the native-like structures. This hypothesis was verified on 200 ns long simulations where all observed changes were combined with the movement of the III domain (<a href="#app1-ijms-21-03099">Figure S9</a>).</p> <p>Normal mode analysis for each system was conducted using cpptraj from AmberTools 18. Only heavy atoms of the protein were included for analysis.</p> </div> <div class="4.2.mixed-solventmdsimulationsâ€" title="sec"> <div class="title" tagx="title" title="title">
4.2. Mixed-Solvent MD Simulationsâ€"Cosolvent Preparation</div> <p>Six different cosolvents: acetonitrile (ACN), benzene (BNZ), dimethylsulfoxide (DMSO), methanol (MEO), phenol (PHN), and urea (URE) were selected to perform the mixed-solvent MD simulations. The chemical structures of cosolvent molecules were downloaded from the ChemSpider database [<a href="#B53-ijms-21-03099">53</a>], and a dedicated set of parameters was prepared. Parameters for ACN were adopted from the work by Nikitin and Lyubartsev [<a href="#B54-ijms-21-03099">54</a>], and parameters for URE were modified using the 8Mureabox force field to obtain parameters for a single molecule. For the rest of the co-solvent molecules, parameters were prepared using Antechamber [<a href="#B55-ijms-21-03099">55</a>] with Gasteiger charges [<a href="#B56-ijms-21-03099">56</a>]. The number of added water and cosolvents molecules is shown in <a href="#app1-ijms-21-03099">Table S5</a>, and the parameters for cosolvents are available in <a href="#app1-ijms-21-03099">Tables S6â€"S11</a>.</p> </div> <div class="4.3.mixed-solventmdsimulationsâ€" title="sec"> <div class="title" tagx="title" title="title">
4.3. Mixed-Solvent MD Simulationsâ€"Initial Configuration</div> <p>The Packmol software [<a href="#B57-ijms-21-03099">57</a>] was used to build the initial systems, consisting of protein (protonated according to the previously described procedure), water, and particular cosolvent molecules. We added 4 and 3 Na<sup>+</sup> ions to the SARS-CoV-2 Mpro and the SARS-CoV Mpro, respectively. It was assumed that the percentage concentration of the cosolvent should not exceed 5% (in the case of ACN, DMSO, MEO, and URE), or should be about 1% in the case of BNZ and PHN phenol (see <a href="#app1-ijms-21-03099">Table S5</a>). The mixed-solvent MD simulation procedures (minimization, equilibration, and production) carried out using the AMBER 18 package were identical to the classical MD simulations. Only the heating stage differedâ€"it was extended up to 40 ps. PMEMD CUDA package of AMBER 18 software [<a href="#B50-ijms-21-03099">50</a>] was used to run two replicas of 50 ns for each cosolvent of both SARS-CoV-2 and SARS-CoV Mpro systems using apo structures and structure with co-crystalised N3 inhibitor (removed before starting the simulations), thus providing a total of 2.4 Î¼s of MixMD simulations.</p> </div> <div class="4.4.waterandcosolventmoleculetracking" title="sec"> <div class="title" tagx="title" title="title">
4.4. Water and Cosolvent Molecule Tracking</div> <p>The AQUA-DUCT 1.0 software [<a href="#B30-ijms-21-03099">30</a>] was used to track water and cosolvent molecules. Molecules of interest, which entered the so-called <i>Object</i>, defined as a 5 Ã… sphere around the centre of geometry of active site residues, namely, H41, C145, H164, and D187, were traced within the <i>Scope</i> region, defined as the interior of a convex hull of both COVID-19 Mpro and SARS Mpro CÎ± atoms. All visualisations were made in PyMol [<a href="#B58-ijms-21-03099">58</a>].</p> <p>AQUA-DUCT was used to analyse maximal accessible volume (MAV), defined as the <i>outer</i> pocket [<a href="#B30-ijms-21-03099">30</a>]. Pockets were calculated by analysis of paths found during molecules tracking step in AQUA-DUCT. A regular grid was constructed, spanning all paths. The grid size was 1 Ã…. For each grid cell, the density of tracked molecules was calculated. Grid cells with nonzero density were used for pocket detection; the <i>outer</i> pocket represented the maximal possible space that could be explored by tracked molecules.</p> <p>To analyse the significance of the changes of the maximal accessible volume between systems, we used generalized linear models with a Poisson distribution based on AIC comparisons and model fit. Wald tests were used to test the significance of the variables. All tests were performed in Statistica (StatSoft 2019).</p> </div> <div class="4.5.hot-spotidentificationandselection" title="sec"> <div class="title" tagx="title" title="title">
4.5. Hot-Spot Identification and Selection</div> <p>AQUA-DUCT [<a href="#B30-ijms-21-03099">30</a>] was used to detect regions occupied by molecules of interest, as well as to identify the densest sites using a local solvent distribution approach. Those so-called hot-spots could be calculated as local and/or global, on the basis of the distribution of tracked molecules that visited the <i>Object</i> (local) or just the <i>Scope</i> without visiting the <i>Object</i> (global); here, they were considered as potential binding sites. For clarity, the size of each sphere representing a particular hot-spot was changed to reflect its occupation level. The selection of the most significant hot-spots consisted of indicating points showing the highest density in particular regions. From the set of points in the space, small groups of hot-spots were determined. Groups were further defined by distance (radius) from each other. Any point found within a distance shorter than the determined radius (3 Ã…) from any other point being part of a given group was counted toward the group. For each so designated group of points, one showing the highest density was chosen as representing the place.</p> </div> <div class="4.6.obtainingsars-cov-2mprogenesequence" title="sec"> <div class="title" tagx="title" title="title">
4.6. Obtaining SARS-CoV-2 Mpro Gene Sequence</div> <p>SARS-CoV-2 Mpro was downloaded from the PDB as a complex with an N3 inhibitor (PDB ID: 6lu7). Tblastn [<a href="#B59-ijms-21-03099">59</a>] was run on the basis of the protein amino acid sequence. We obtained 100% identity with 10055â€"10972 region of SARS-CoV-2 Mpro complete genome (Sequence ID: <i>MN985262.1</i>). Blastx [<a href="#B60-ijms-21-03099">60</a>] calculations were run with the selected region, and orf1a polyprotein (NCBI reference sequence: <i>YP_009725295.1</i>) amino acid sequence, identical with the previously downloaded SARS-CoV-2 Mpro, was received.</p> </div> <div class="4.7.energeticeffectofaminoacidsubstitutions" title="sec"> <div class="title" tagx="title" title="title">
4.7. Energetic Effect of Amino Acid Substitutions</div> <p>FoldX software [<a href="#B35-ijms-21-03099">35</a>] was used to insert substitutions into the structures of SARS-CoV and SARS-CoV-2 Mpros. To analyse the changes in energetic contribution to the protein stability of the two structures, 12 single-point mutations were introduced to the SARS structure using the BuildModel module. The BuildModel module introduces substitution(s) of selected amino acid(s), optimizes the structure of a new variant, and calculates the difference in the Gibbs free energy of protein folding between the wild-type and mutant variant in kilocalories per mole. The lower the difference in energetical terms, the more stable the mutant variant should be. Each of the residues in SARS-CoV Mpro was mutated to the respective SARS-CoV-2 Mpro residue, and the difference in total energies between the wild-type SARS-CoV-2 Mpro and the mutant structures were calculated. Then, to investigate further possible mutations of SARS-CoV-2 Mpro, single nucleotide substitutions were introduced to the SARS-CoV-2 main protease gene. If a substitution of a single nucleotide caused translation to an amino acid different than the corresponding residue in the wild-type structure, an appropriate mutation was proposed with FoldX software.</p> </div> <div class="4.8.comulatorcalculationsofcorrelationbetweenaminoacids" title="sec"> <div class="title" tagx="title" title="title">
4.8. Comulator Calculations of Correlation Between Amino Acids</div> <p>SARS-CoV Mpro was downloaded from the PDB (PDB ID: 1q2w). Blast [<a href="#B61-ijms-21-03099">61</a>] was run on the basis of the amino acid sequence. As a result, 2643 sequences of viral main proteases similar to chain A SARS-CoV Mpro were obtained. Clustal Omega [<a href="#B62-ijms-21-03099">62</a>] was used to prepare an alignment of those sequences. Comulator [<a href="#B33-ijms-21-03099">33</a>] was then employed to calculate the correlation between amino acids and, on the basis of the results, groups of positions in SARS-CoV Mpro sequence were selected whose amino acid occurrences strongly depended on each other.</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
5. Conclusions</div> <p>In this paper, we reported on molecular dynamics simulations of the main protease (Mpro), whose crystal structures have been released. We compared the Mpro for SARS-CoV-2 with a highly similar SARS-CoV protein. In spite of a high level of sequence similarity between these two homologous proteins, their active sites showed major differences in both shape and size, indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, a detailed analysis of the binding pocketâ€™s conformational changes during simulation time indicated its flexibility and plasticity, which dashes hope for rapid and reliable drug design. Moreover, our findings show the presence of a flexible C44-P52 loop regulating the access to the binding site pocket. A successful inhibitor may need to have an ability to relocate the loop from the entrance to bind to the catalytic pocket. However, mutations leading to changes in the amino acid sequence of the C44-P52 loop, although not affecting the folding of the protein, may result in the putative inhibitorsâ€™ inability to access the binding pocket and provide a probable development of drug resistance. To avoid this situation in which the future evolution of the Mpros can undermine all our efforts, we should focus on key functional residues or those whose further mutation will destabilise the protein (e.g., P39, R40, P52, G143, G146, or L167). Alternatively, we would suggest targeting the region between II and III domains, which contributes to the dimer formation. Our results provide the basis for drug design efforts aimed at this important protein target as part of the multifaceted global effort to eradicate COVID-19. In view of the presented challenges to finding a potent drug targeting Mpro, in our opinion, the most successful strategy would be to screen a large database of compounds with diverse structures involving or designing inhibitors de novo using a fragment-based approach. Both of these strategies, unfortunately, take much longer than a currently preferred approach based on repurposing existing FDA-approved compounds and hence should be pursued as a long-term plan of preparedness for future outbreaks of COVID epidemics involving this and other strains of the virus.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>The authors would like to thank Tomasz Skalski for his helpful advice in statistical analysis of data.</p> </div> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <p>Supplementary materials can be found at <span class="uri" title="uri"><a href="https://www.mdpi.com/1422-0067/21/9/3099/s1">https://www.mdpi.com/1422-0067/21/9/3099/s1</a></span>.</p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="ijms-21-03099-s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>Conceptualization, A.G.; methodology, M.B., K.M., A.R., and A.S.; software, M.B., K.M., A.R., and A.S.; validation, M.B., K.M., and A.G.; formal analysis, M.B., K.M., A.R., and A.S.; investigation, M.B., K.M., A.R., and A.S.; resources, M.B. and K.M.; data curation, M.B. and K.M.; writingâ€"original draft preparation, M.B. and K.M.; writingâ€"review and editing, J.A.T. and A.G.; visualization, M.B., K.M., A.R., and A.G; supervision, A.G.; project administration, A.G.; funding acquisition, J.A.T. and A.G. All authors have read and agreed to the published version of the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>K.M., M.B., A.R., A.S., and A.G.â€™s work was supported by the National Science Centre, Poland (grant no. DEC-2013/10/E/NZ1/00649 and DEC-2015/18/M/NZ1/00427). J.A.T. expresses gratitude for research support for this project received from IBM CAS and NSERC (Canada).</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest.</p> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div><span class="array" tagx="array" title="array">   Mpro Main protease   SARS Severe acute respiratory syndrome   COVID-19 Coronavirus disease 2019   SARS-CoV-2 Severe acute respiratory syndrome coronavirus 2   CoVs Coronaviruses   HCoV-NL63 Human coronavirus NL63   HCoV-229E Human coronavirus 229E   HCoV-OC43 Human coronavirus OC43   HCoV-HKU1 Human coronavirus HKU1   SARS-CoV Severe acute respiratory syndrome coronavirus   MERS-CoV Middle East respiratory syndrome coronavirus   ORFs Open reading frames   3CLpro Chymotrypsin-like cysteine protease   S Spike surface glycoprotein   E Small envelope protein   M Matrix protein   N Nucleocapsid protein   N3(PRD_002214) N-[(5 methylisoxazol-3-yl)carbonyl] alanyl-L-valyl-N~1-((1R,2Z)-4-(benzyloxy)-4-oxo-1-{[(3R)-2-oxopyrrolidin-3-yl]methyl}but-2-enyl)-L-leucinamide   cMD Classical molecular dynamics simulations   MixMD Mixed-solvent molecular dynamics simulations   PDB Protein Data Bank   MAV Maximal accessible volume   ACN Acetonitrile   BNZ Benzene   DMSO Dimethylsulfoxide   MEO Methanol   PHN Phenol   URE Urea   CMA Correlated mutation analysis   FDA Food and Drug Administration  </span></div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-ijms-21-03099" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">395</span><span class="fpage" tagx="fpage" title="fpage">497</span><span class="lpage" tagx="lpage" title="lpage">506</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30183-5">10.1016/S0140-6736(20)30183-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31986264">31986264</a></span></span></li> <li tag="ref"><a name="B2-ijms-21-03099" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span><span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">382</span><span class="fpage" tagx="fpage" title="fpage">727</span><span class="lpage" tagx="lpage" title="lpage">733</span><span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMoa2001017">10.1056/NEJMoa2001017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31978945">31978945</a></span></span></li> <li tag="ref"><a name="B3-ijms-21-03099" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woo</span><span class="given-names" tagx="given-names" title="given-names">P.C.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lau</span><span class="given-names" tagx="given-names" title="given-names">S.K.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">K.-Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronavirus Genomics and Bioinformatics Analysis</span><span class="source" tagx="source" title="source">Viruses</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">1804</span><span class="lpage" tagx="lpage" title="lpage">1820</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v2081803">10.3390/v2081803</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21994708">21994708</a></span></span></li> <li tag="ref"><a name="B4-ijms-21-03099" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">X.-Q.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inferring the hosts of coronavirus using dual statistical models based on nucleotide composition</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">17155</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep17155">10.1038/srep17155</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26607834">26607834</a></span></span></li> <li tag="ref"><a name="B5-ijms-21-03099" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Z.-L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Origin and evolution of pathogenic coronaviruses</span><span class="source" tagx="source" title="source">Nat. Rev. Microbiol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">181</span><span class="lpage" tagx="lpage" title="lpage">192</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41579-018-0118-9">10.1038/s41579-018-0118-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30531947">30531947</a></span></span></li> <li tag="ref"><a name="B6-ijms-21-03099" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fehr</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronaviruses: An Overview of Their Replication and Pathogenesis</span><span class="source" tagx="source" title="source">Methods Mol. Biol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">1282</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">23</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25720466">25720466</a></span></span></li> <li tag="ref"><a name="B7-ijms-21-03099" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Origin and evolution of the 2019 novel coronavirus</span><span class="source" tagx="source" title="source">Clin. Infect. Dis.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/cid/ciaa112">10.1093/cid/ciaa112</a></span></span></li> <li tag="ref"><a name="B8-ijms-21-03099" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bao</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">From SARS to MERS, Thrusting Coronaviruses into the Spotlight</span><span class="source" tagx="source" title="source">Viruses</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">59</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v11010059">10.3390/v11010059</a></span></span></li> <li tag="ref"><a name="B9-ijms-21-03099" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structures of Two Coronavirus Main Proteases: Implications for Substrate Binding and Antiviral Drug Design</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">82</span><span class="fpage" tagx="fpage" title="fpage">2515</span><span class="lpage" tagx="lpage" title="lpage">2527</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02114-07">10.1128/JVI.02114-07</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18094151">18094151</a></span></span></li> <li tag="ref"><a name="B10-ijms-21-03099" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China</span><span class="source" tagx="source" title="source">Cell Host Microbe</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">325</span><span class="lpage" tagx="lpage" title="lpage">328</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chom.2020.02.001">10.1016/j.chom.2020.02.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32035028">32035028</a></span></span></li> <li tag="ref"><a name="B11-ijms-21-03099" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zumla</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">J.F.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azhar</span><span class="given-names" tagx="given-names" title="given-names">E.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">D.S.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">K.-Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronavirusesâ€"Drug discovery and therapeutic options</span><span class="source" tagx="source" title="source">Nat. Rev. Drug Discov.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">327</span><span class="lpage" tagx="lpage" title="lpage">347</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrd.2015.37">10.1038/nrd.2015.37</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26868298">26868298</a></span></span></li> <li tag="ref"><a name="B12-ijms-21-03099" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morse</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lalonde</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Learning from the Past: Possible Urgent Prevention and Treatment Options for Severe Acute Respiratory Infections Caused by 2019-nCoV</span><span class="source" tagx="source" title="source">ChemBioChem</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">730</span><span class="lpage" tagx="lpage" title="lpage">738</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32022370">32022370</a></span></span></li> <li tag="ref"><a name="B13-ijms-21-03099" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structure of Mpro from COVID-19 virus and discovery of its inhibitors</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-020-2223-y">10.1038/s41586-020-2223-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32272481">32272481</a></span></span></li> <li tag="ref"><a name="B14-ijms-21-03099" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhong</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Without Its N-Finger, the Main Protease of Severe Acute Respiratory Syndrome Coronavirus Can Form a Novel Dimer through Its C-Terminal Domain</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">82</span><span class="fpage" tagx="fpage" title="fpage">4227</span><span class="lpage" tagx="lpage" title="lpage">4234</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02612-07">10.1128/JVI.02612-07</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18305043">18305043</a></span></span></li> <li tag="ref"><a name="B15-ijms-21-03099" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ton</span><span class="given-names" tagx="given-names" title="given-names">A.-T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gentile</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsing</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ban</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cherkasov</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rapid Identification of Potential Inhibitors of SARS-CoV-2 Main Protease by Deep Docking of 1.3 Billion Compounds</span><span class="source" tagx="source" title="source">Mol. Inform.</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/minf.202000028">10.1002/minf.202000028</a></span></span></li> <li tag="ref"><a name="B16-ijms-21-03099" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mu</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nelfinavir was predicted to be a potential inhibitor of 2019-nCov main protease by an integrative approach combining homology modelling, molecular docking and binding free energy calculation</span><span class="source" tagx="source" title="source">bioRxiv</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.01.27.921627">10.1101/2020.01.27.921627</a></span></span></li> <li tag="ref"><a name="B17-ijms-21-03099" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.-J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines</span><span class="source" tagx="source" title="source">J. Genet. Genomics</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">119</span><span class="lpage" tagx="lpage" title="lpage">121</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jgg.2020.02.001">10.1016/j.jgg.2020.02.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32173287">32173287</a></span></span></li> <li tag="ref"><a name="B18-ijms-21-03099" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hao</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Therapeutic Drugs Targeting 2019-nCoV Main Protease by High-Throughput Screening</span><span class="source" tagx="source" title="source">bioRxiv</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.01.28.922922">10.1101/2020.01.28.922922</a></span></span></li> <li tag="ref"><a name="B19-ijms-21-03099" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">D.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">G.-W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Potentially highly potent drugs for 2019-nCoV</span><span class="source" tagx="source" title="source">bioRxiv</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/2020.02.05.936013">10.1101/2020.02.05.936013</a></span></span></li> <li tag="ref"><a name="B20-ijms-21-03099" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Talluri</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Virtual High Throughput Screening Based Prediction of Potential Drugs for COVID-19</span><span class="source" tagx="source" title="source">Preprints</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.20944/preprints202002.0418.v2">10.20944/preprints202002.0418.v2</a></span></span></li> <li tag="ref"><a name="B21-ijms-21-03099" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yiu</span><span class="given-names" tagx="given-names" title="given-names">C.-P.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">K.-Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Prediction of the SARS-CoV-2 (2019-nCoV) 3C-like protease (3CLpro) structure: virtual screening reveals velpatasvir, ledipasvir, and other drug repurposing candidates</span><span class="source" tagx="source" title="source">F1000Research</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">129</span><span class="pub-id"><a href="https://dx.doi.org/10.12688/f1000research.22457.1">10.12688/f1000research.22457.1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32194944">32194944</a></span></span></li> <li tag="ref"><a name="B22-ijms-21-03099" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fischer</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sellner</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neranjan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lill</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SmieÅ¡ko</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Potential Inhibitors for Novel Coronavirus Protease Identified by Virtual Screening of 606 Million Compounds</span><span class="source" tagx="source" title="source">ChemRxiv</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.26434/chemrxiv.11923239">10.26434/chemrxiv.11923239</a></span></span></li> <li tag="ref"><a name="B23-ijms-21-03099" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gentile</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patamia</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scala</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sciortino</span><span class="given-names" tagx="given-names" title="given-names">M.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piperno</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rescifina</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibitors of SARS-CoV-2 Main Protease from a Library of Marine Natural Products: A Virtual Screening and Molecular Modeling Study</span><span class="source" tagx="source" title="source">Mar. Drugs</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">18</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">225</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/md18040225">10.3390/md18040225</a></span></span></li> <li tag="ref"><a name="B24-ijms-21-03099" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adem</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eyupoglu</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarfraz</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasul</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of potent COVID-19 main protease (Mpro) inhibitors from natural polyphenols: An in silico strategy unveils a hope against CORONA</span><span class="source" tagx="source" title="source">Preprints</span><span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.20944/preprints202003.0333.v1">10.20944/preprints202003.0333.v1</a></span></span></li> <li tag="ref"><a name="B25-ijms-21-03099" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berman</span><span class="given-names" tagx="given-names" title="given-names">H.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Protein Data Bank</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">235</span><span class="lpage" tagx="lpage" title="lpage">242</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/28.1.235">10.1093/nar/28.1.235</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592235">10592235</a></span></span></li> <li tag="ref"><a name="B26-ijms-21-03099" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Curth</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauerhering</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Becker</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rox</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilgenfeld</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved Î±-ketoamide inhibitors</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">368</span><span class="fpage" tagx="fpage" title="fpage">409</span><span class="lpage" tagx="lpage" title="lpage">412</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.abb3405">10.1126/science.abb3405</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32198291">32198291</a></span></span></li> <li tag="ref"><a name="B27-ijms-21-03099" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bacha</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrila</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Velazquez-Campoy</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leavitt</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freire</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of Novel Inhibitors of the SARS Coronavirus Main Protease 3CL pro <sup>â€ </sup></span><span class="source" tagx="source" title="source">Biochemistry</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">4906</span><span class="lpage" tagx="lpage" title="lpage">4912</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/bi0361766">10.1021/bi0361766</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15109248">15109248</a></span></span></li> <li tag="ref"><a name="B28-ijms-21-03099" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anand</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronavirus Main Proteinase (3CLpro) Structure: Basis for Design of Anti-SARS Drugs</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">300</span><span class="fpage" tagx="fpage" title="fpage">1763</span><span class="lpage" tagx="lpage" title="lpage">1767</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1085658">10.1126/science.1085658</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12746549">12746549</a></span></span></li> <li tag="ref"><a name="B29-ijms-21-03099" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MitusiÅ"ska</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RaczyÅ"ska</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BzÃ³wka</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bagrowska</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ³ra</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Applications of water molecules for analysis of macromolecule properties</span><span class="source" tagx="source" title="source">Comput. Struct. Biotechnol. J.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">355</span><span class="lpage" tagx="lpage" title="lpage">365</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.csbj.2020.02.001">10.1016/j.csbj.2020.02.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32123557">32123557</a></span></span></li> <li tag="ref"><a name="B30-ijms-21-03099" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magdziarz</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MitusiÅ"ska</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BzÃ³wka</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RaczyÅ"ska</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">StaÅ"czak</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banas</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bagrowska</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ³ra</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">AQUA-DUCT 1.0: Structural and functional analysis of macromolecules from an intramolecular voids perspective</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2019</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btz946">10.1093/bioinformatics/btz946</a></span></span></li> <li tag="ref"><a name="B31-ijms-21-03099" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teng</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Maturation Mechanism of Severe Acute Respiratory Syndrome (SARS) Coronavirus 3C-like Proteinase</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">285</span><span class="fpage" tagx="fpage" title="fpage">28134</span><span class="lpage" tagx="lpage" title="lpage">28140</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M109.095851">10.1074/jbc.M109.095851</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20489209">20489209</a></span></span></li> <li tag="ref"><a name="B32-ijms-21-03099" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panjkovich</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daura</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span></span><span class="mixed-article-title" title="mixed-article-title">PARS: A web server for the prediction of Protein Allosteric and Regulatory Sites</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">1314</span><span class="lpage" tagx="lpage" title="lpage">1315</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu002">10.1093/bioinformatics/btu002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24413526">24413526</a></span></span></li> <li tag="ref"><a name="B33-ijms-21-03099" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuipers</span><span class="given-names" tagx="given-names" title="given-names">R.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joosten</span><span class="given-names" tagx="given-names" title="given-names">H.-J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Berkel</span><span class="given-names" tagx="given-names" title="given-names">W.J.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leferink</span><span class="given-names" tagx="given-names" title="given-names">N.G.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rooijen</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ittmann</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Zimmeren</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jochens</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bornscheuer</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vriend</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">3DM: Systematic analysis of heterogeneous superfamily data to discover protein functionalities</span><span class="source" tagx="source" title="source">Proteins Struct. Funct. Bioinform.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">2101</span><span class="lpage" tagx="lpage" title="lpage">2113</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/prot.22725">10.1002/prot.22725</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20455266">20455266</a></span></span></li> <li tag="ref"><a name="B34-ijms-21-03099" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subramanian</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MitusiÅ"ska</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raedts</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almourfi</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joosten</span><span class="given-names" tagx="given-names" title="given-names">H.-J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hendriks</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sedelnikova</span><span class="given-names" tagx="given-names" title="given-names">S.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kengen</span><span class="given-names" tagx="given-names" title="given-names">S.W.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hagen</span><span class="given-names" tagx="given-names" title="given-names">W.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ³ra</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Distant Non-Obvious Mutations Influence the Activity of a Hyperthermophilic Pyrococcus furiosus Phosphoglucose Isomerase</span><span class="source" tagx="source" title="source">Biomolecules</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">212</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/biom9060212">10.3390/biom9060212</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31159273">31159273</a></span></span></li> <li tag="ref"><a name="B35-ijms-21-03099" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schymkowitz</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borg</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stricher</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nys</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rousseau</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Serrano</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The FoldX web server: An online force field</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">W382</span><span class="lpage" tagx="lpage" title="lpage">W388</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gki387">10.1093/nar/gki387</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980494">15980494</a></span></span></li> <li tag="ref"><a name="B36-ijms-21-03099" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klausen</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jespersen</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">K.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jurtz</span><span class="given-names" tagx="given-names" title="given-names">V.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SÃ¸nderby</span><span class="given-names" tagx="given-names" title="given-names">C.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sommer</span><span class="given-names" tagx="given-names" title="given-names">M.O.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winther</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petersen</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">NetSurfP-2.0: Improved prediction of protein structural features by integrated deep learning</span><span class="source" tagx="source" title="source">Proteins Struct. Funct. Bioinform.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">520</span><span class="lpage" tagx="lpage" title="lpage">527</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/prot.25674">10.1002/prot.25674</a></span></span></li> <li tag="ref"><a name="B37-ijms-21-03099" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spyrakis</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">M.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayder</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cozzini</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mozzarelli</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kellogg</span><span class="given-names" tagx="given-names" title="given-names">G.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Roles of Water in the Protein Matrix: A Largely Untapped Resource for Drug Discovery</span><span class="source" tagx="source" title="source">J. Med. Chem.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">6781</span><span class="lpage" tagx="lpage" title="lpage">6827</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acs.jmedchem.7b00057">10.1021/acs.jmedchem.7b00057</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28475332">28475332</a></span></span></li> <li tag="ref"><a name="B38-ijms-21-03099" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Beer</span><span class="given-names" tagx="given-names" title="given-names">S.B.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vermeulen</span><span class="given-names" tagx="given-names" title="given-names">N.P.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oostenbrink</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Role of Water Molecules in Computational Drug Design</span><span class="source" tagx="source" title="source">Curr. Top. Med. Chem.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">55</span><span class="lpage" tagx="lpage" title="lpage">66</span><span class="pub-id"><a href="https://dx.doi.org/10.2174/156802610790232288">10.2174/156802610790232288</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19929830">19929830</a></span></span></li> <li tag="ref"><a name="B39-ijms-21-03099" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MitusiÅ"ska</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magdziarz</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BzÃ³wka</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">StaÅ"czak</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gora</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Exploring Solanum tuberosum Epoxide Hydrolase Internal Architecture by Water Molecules Tracking</span><span class="source" tagx="source" title="source">Biomolecules</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">143</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/biom8040143">10.3390/biom8040143</a></span></span></li> <li tag="ref"><a name="B40-ijms-21-03099" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Needle</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lountos</span><span class="given-names" tagx="given-names" title="given-names">G.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waugh</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structures of the Middle East respiratory syndrome coronavirus 3C-like protease reveal insights into substrate specificity</span><span class="source" tagx="source" title="source">Acta Crystallogr. Sect. D Biol. Crystallogr.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">1102</span><span class="lpage" tagx="lpage" title="lpage">1111</span><span class="pub-id"><a href="https://dx.doi.org/10.1107/S1399004715003521">10.1107/S1399004715003521</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25945576">25945576</a></span></span></li> <li tag="ref"><a name="B41-ijms-21-03099" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">M.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">W.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">J.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">L.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">G.-G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">H.-P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Essential covalent linkage between the chymotrypsin-like domain and the extra domain of the SARS-CoV main protease</span><span class="source" tagx="source" title="source">J. Biochem.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">148</span><span class="fpage" tagx="fpage" title="fpage">349</span><span class="lpage" tagx="lpage" title="lpage">358</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jb/mvq071">10.1093/jb/mvq071</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20587646">20587646</a></span></span></li> <li tag="ref"><a name="B42-ijms-21-03099" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anand</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra alpha-helical domain</span><span class="source" tagx="source" title="source">EMBO J.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">3213</span><span class="lpage" tagx="lpage" title="lpage">3224</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/emboj/cdf327">10.1093/emboj/cdf327</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12093723">12093723</a></span></span></li> <li tag="ref"><a name="B43-ijms-21-03099" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dynamically-Driven Enhancement of the Catalytic Machinery of the SARS 3C-Like Protease by the S284-T285-I286/A Mutations on the Extra Domain</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e101941</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0101941">10.1371/journal.pone.0101941</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25036652">25036652</a></span></span></li> <li tag="ref"><a name="B44-ijms-21-03099" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeyachandran</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">N.-J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">C.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">S.-Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.-S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">Y.-M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">M.-H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structure-based virtual screening and experimental validation of the discovery of inhibitors targeted towards the human coronavirus nucleocapsid protein</span><span class="source" tagx="source" title="source">Mol. Biosyst.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">59</span><span class="lpage" tagx="lpage" title="lpage">66</span><span class="pub-id"><a href="https://dx.doi.org/10.1039/C5MB00582E">10.1039/C5MB00582E</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26542199">26542199</a></span></span></li> <li tag="ref"><a name="B45-ijms-21-03099" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dayer</span><span class="given-names" tagx="given-names" title="given-names">M.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taleb-Gassabi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dayer</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lopinavir; A Potent Drug against Coronavirus Infection: Insight from Molecular Docking Study</span><span class="source" tagx="source" title="source">Arch. Clin. Infect. Dis.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">12</span><span class="pub-id"><a href="https://dx.doi.org/10.5812/archcid.13823">10.5812/archcid.13823</a></span></span></li> <li tag="ref"><a name="B46-ijms-21-03099" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Resnick</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bradley</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gan</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douangamath</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krojer</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sethi</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geurink</span><span class="given-names" tagx="given-names" title="given-names">P.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aimon</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amitai</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bellini</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Rapid Covalent-Probe Discovery by Electrophile-Fragment Screening</span><span class="source" tagx="source" title="source">J. Am. Chem. Soc.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">141</span><span class="fpage" tagx="fpage" title="fpage">8951</span><span class="lpage" tagx="lpage" title="lpage">8968</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/jacs.9b02822">10.1021/jacs.9b02822</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31060360">31060360</a></span></span></li> <li tag="ref"><a name="B47-ijms-21-03099" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anandakrishnan</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aguilar</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Onufriev</span><span class="given-names" tagx="given-names" title="given-names">A.V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">H++ 3.0: Automating pK prediction and the preparation of biomolecular structures for atomistic molecular modeling and simulations</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">W537</span><span class="lpage" tagx="lpage" title="lpage">W541</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gks375">10.1093/nar/gks375</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22570416">22570416</a></span></span></li> <li tag="ref"><a name="B48-ijms-21-03099" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luchko</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gusarov</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roe</span><span class="given-names" tagx="given-names" title="given-names">D.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmerling</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuszynski</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kovalenko</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Three-Dimensional Molecular Theory of Solvation Coupled with Molecular Dynamics in Amber</span><span class="source" tagx="source" title="source">J. Chem. Theory Comput.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">607</span><span class="lpage" tagx="lpage" title="lpage">624</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/ct900460m">10.1021/ct900460m</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20440377">20440377</a></span></span></li> <li tag="ref"><a name="B49-ijms-21-03099" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sindhikara</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoshida</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirata</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Placevent: An algorithm for prediction of explicit solvent atom distribution-Application to HIV-1 protease and F-ATP synthase</span><span class="source" tagx="source" title="source">J. Comput. Chem.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">1536</span><span class="lpage" tagx="lpage" title="lpage">1543</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.22984">10.1002/jcc.22984</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22522665">22522665</a></span></span></li> <li tag="ref"><a name="B50-ijms-21-03099" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ben-Shalom</span><span class="given-names" tagx="given-names" title="given-names">I.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brozell</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cerutti</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheatham III</span><span class="given-names" tagx="given-names" title="given-names">T.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cruzeiro</span><span class="given-names" tagx="given-names" title="given-names">V.W.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darden</span><span class="given-names" tagx="given-names" title="given-names">T.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duke</span><span class="given-names" tagx="given-names" title="given-names">R.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghoreishi</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilson</span><span class="given-names" tagx="given-names" title="given-names">M.K.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="source" tagx="source" title="source">AMBER 2018</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">University of California</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA, USA</span><span class="year" tagx="year" title="year">2018</span></span></li> <li tag="ref"><a name="B51-ijms-21-03099" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heo</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feig</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">What makes it difficult to refine protein models further via molecular dynamics simulations?</span><span class="source" tagx="source" title="source">Proteins Struct. Funct. Bioinform.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">177</span><span class="lpage" tagx="lpage" title="lpage">188</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/prot.25393">10.1002/prot.25393</a></span></span></li> <li tag="ref"><a name="B52-ijms-21-03099" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MitusiÅ"ska</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skalski</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GÃ³ra</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Simple selection procedure to distinguish between static and flexible loops</span><span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">21</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">2293</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms21072293">10.3390/ijms21072293</a></span></span></li> <li tag="ref"><a name="B53-ijms-21-03099" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pence</span><span class="given-names" tagx="given-names" title="given-names">H.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">ChemSpider: An Online Chemical Information Resource</span><span class="source" tagx="source" title="source">J. Chem. Educ.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">1123</span><span class="lpage" tagx="lpage" title="lpage">1124</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/ed100697w">10.1021/ed100697w</a></span></span></li> <li tag="ref"><a name="B54-ijms-21-03099" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikitin</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyubartsev</span><span class="given-names" tagx="given-names" title="given-names">A.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">New six-site acetonitrile model for simulations of liquid acetonitrile and its aqueous mixtures</span><span class="source" tagx="source" title="source">J. Comput. Chem.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">2020</span><span class="lpage" tagx="lpage" title="lpage">2026</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.20721">10.1002/jcc.20721</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17450554">17450554</a></span></span></li> <li tag="ref"><a name="B55-ijms-21-03099" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kollman</span><span class="given-names" tagx="given-names" title="given-names">P.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Case</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span><span class="source" tagx="source" title="source">J. Mol. Graph. Model.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">247</span><span class="lpage" tagx="lpage" title="lpage">260</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jmgm.2005.12.005">10.1016/j.jmgm.2005.12.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16458552">16458552</a></span></span></li> <li tag="ref"><a name="B56-ijms-21-03099" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gasteiger</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marsili</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Iterative partial equalization of orbital electronegativityâ€"A rapid access to atomic charges</span><span class="source" tagx="source" title="source">Tetrahedron</span><span class="year" tagx="year" title="year">1980</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">3219</span><span class="lpage" tagx="lpage" title="lpage">3228</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0040-4020(80)80168-2">10.1016/0040-4020(80)80168-2</a></span></span></li> <li tag="ref"><a name="B57-ijms-21-03099" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MartÃ­nez</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andrade</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birgin</span><span class="given-names" tagx="given-names" title="given-names">E.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MartÃ­nez</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">PACKMOL: A package for building initial configurations for molecular dynamics simulations</span><span class="source" tagx="source" title="source">J. Comput. Chem.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">2157</span><span class="lpage" tagx="lpage" title="lpage">2164</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jcc.21224">10.1002/jcc.21224</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19229944">19229944</a></span></span></li> <li tag="ref"><a name="B58-ijms-21-03099" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="source" tagx="source" title="source">The PyMOL Molecular Graphics System, Version 2.0 SchrÃ¶dinger</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">SchrÃ¶dinger LLC.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY, USA</span></span></li> <li tag="ref"><a name="B59-ijms-21-03099" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gertz</span><span class="given-names" tagx="given-names" title="given-names">E.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">Y.-K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agarwala</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchÃ¤ffer</span><span class="given-names" tagx="given-names" title="given-names">A.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altschul</span><span class="given-names" tagx="given-names" title="given-names">S.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Composition-based statistics and translated nucleotide searches: Improving the TBLASTN module of BLAST</span><span class="source" tagx="source" title="source">BMC Biol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">41</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1741-7007-4-41">10.1186/1741-7007-4-41</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17156431">17156431</a></span></span></li> <li tag="ref"><a name="B60-ijms-21-03099" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Camacho</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coulouris</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avagyan</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papadopoulos</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bealer</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madden</span><span class="given-names" tagx="given-names" title="given-names">T.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">BLAST+: Architecture and applications</span><span class="source" tagx="source" title="source">BMC Bioinform.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">421</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-10-421">10.1186/1471-2105-10-421</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20003500">20003500</a></span></span></li> <li tag="ref"><a name="B61-ijms-21-03099" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altschul</span><span class="given-names" tagx="given-names" title="given-names">S.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gish</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myers</span><span class="given-names" tagx="given-names" title="given-names">E.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipman</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Basic local alignment search tool</span><span class="source" tagx="source" title="source">J. Mol. Biol.</span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">215</span><span class="fpage" tagx="fpage" title="fpage">403</span><span class="lpage" tagx="lpage" title="lpage">410</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0022-2836(05)80360-2">10.1016/S0022-2836(05)80360-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712">2231712</a></span></span></li> <li tag="ref"><a name="B62-ijms-21-03099" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sievers</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgins</span><span class="given-names" tagx="given-names" title="given-names">D.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Clustal Omega, Accurate Alignment of Very Large Numbers of Sequences</span><span class="source" tagx="source" title="source">Methods Mol. Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">1079</span><span class="fpage" tagx="fpage" title="fpage">105</span><span class="lpage" tagx="lpage" title="lpage">106</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-62703-646-7_6">10.1007/978-1-62703-646-7_6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24170397">24170397</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>The differences between the severe acute respiratory syndrome coronavirus main protease (SARS-CoV Mpro) and SARS-CoV-2 Mpro structures. (<b>A</b>) The overall structure of both SARS-CoV and SARS-CoV-2 Mpros with differing amino acids are marked as black (SARS-CoV Mpro) and blue (SARS-CoV-2 Mpro). (<b>B</b>) Close-up of the active site cavity and bound N3 inhibitor into SARS-CoV (black sticks) and SARS-CoV-2 (blue sticks) Mpros. The catalytic water molecule that resembles the position of the third member of the catalytic triad adopted from the cysteine proteases is shown for both SARS-CoV (black sphere) and SARS-CoV-2 (blue sphere) Mpros. The active site residues are shown as red sticks and the proteinsâ€™ structures are shown in surface representation. The differing residues in position 46 located near the entrance to the active site are marked with an asterisk (*) on the (<b>A</b>) and as blue and black lines on the (<b>B</b>) panel.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>The differences between the maximal accessible volume of the binding cavities calculated during molecular dynamics (MD) simulations of both apo structures of Mpros (SARS-CoV and SARS-CoV-2) and structures with co-crystallised N3 inhibitor (SARS-CoV<sup>N3</sup> and SARS-CoV-2<sup>N3</sup>) used as different starting points for 10 replicas of 50 ns per structure. The position of the blue sphere (hot-spot with highest density) in each structure reflects the position of the catalytic water molecule.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Flexibility of loops surrounding the entrance to the binding cavity of (<b>A</b>) SARS-CoV-2 Mpro, (<b>B</b>) SARS-CoV Mpro, (<b>C</b>) SARS-CoV Mpro<sup>N3</sup>, and (<b>D</b>) SARS-CoV Mpro<sup>N3</sup>. For the picture clarity, only residues creating loops were shown. Upper row: RMSF data. The active site residues are shown as red sticks, and the A46S replacement between SARS-CoV and SARS-CoV-2 main proteases is shown as light blue sticks. The width and colour of the shown residues reflect the level of loop flexibility. The wider and darker residues are more flexible. Lower row: the results of normal mode analysis as a superposition of active site surroundings; structures are coloured whiteâ€"initial conformation, blackâ€"final conformation, grayâ€"transient conformation.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Localisation of the local hot-spots identified in the binding site cavities in SARS-CoV-2 and SARS-CoV main proteases. Hot-spots of individual cosolvents are represented by spheres, and their size reflects the hot-spot density. The colour coding is as follows: purpleâ€"urea, greenâ€"dimethylsulfoxide, yellowâ€"methanol, orangeâ€"acetonitrile, pinkâ€"phenol, redâ€"benzene. The active site residues are shown as red sticks, and the proteinsâ€™ structures are shown in cartoon representation; loop 44â€"52 is grey. The proteinsâ€™ structures come from the MD simulation snapshots (first frame of the production stage).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Localisation of the evolutionary-correlated residues of Mpros (black sticks). The correlated mutation analysis (CMA) analysis provided four groups of evolutionary-correlated residues. The SARS-CoV-2 Mpro structure is presented as a cartoon, the active site residues are shown as red sticks, the unique residues of the SARS-CoV-2 Mpro as blue sticks, and the asterisk (*) indicates the residue belonging to the evolutionary-correlated residues, unique for SARS-CoV Mpro. The loop C44-P52 is coloured black and the F185-T201 loop is orange. Please note that within one of the correlated groups (upper left), the residues from C44-P52 loop are correlated with Q189 from the linker loop and with residue from III domain.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Potential mutability of SARS-CoV-2 Mpro. (<b>A</b>) Structure of SARS-CoV-2 Mpro with the most energetically favourable potential mutations of amino acids marked as green surface. Positions of amino acids that differ from the ones in SARS-CoV Mpro structure marked as blue sticks. Catalytic dyad marked as red. (<b>B</b>) The catalytic site of SARS-CoV-2 Mpro is shown as surface with the most energetically favourable potential mutations shown as green, neutral as white, and unfavourable as red. The C44-P52 loop is shown as black mesh.</p>   </div> </div>  </body></html>