<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7508890">7508890</a></span><span class="publisher-id" title="publisher-id">72368</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-020-72368-x">10.1038/s41598-020-72368-x</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Divalent cation-induced conformational changes of influenza virus hemagglutinin</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seok</span><span class="given-names" tagx="given-names" title="given-names">Jong Hyeon</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Hyojin</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">Dan Bi</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">An</span><span class="given-names" tagx="given-names" title="given-names">Jeong Suk</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Eun Jeong</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">Ji-Hye</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">Mi Sook</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Kyung Hyun</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">khkim@korea.ac.kr</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.222754.4</span><span class="institution-id" title="institution-id">0000 0001 0840 2678</span><span class="institution" title="institution">Department of Biotechnology and Bioinformatics, </span><span class="institution" title="institution">Korea University, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.410884.1</span><span class="institution-id" title="institution-id">0000 0004 0532 6173</span><span class="institution" title="institution">Department of Food and Nutrition, </span><span class="institution" title="institution">Duksung Womenâ€™s University, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-9-9</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">15457</span><span class="history" title="history"><span class="received" title="received">received: 2020-1-28</span><span class="accepted" title="accepted">accepted: 2020-7-15</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Divalent cations Cu<sup>2+</sup> and Zn<sup>2+</sup> can prevent the viral growth in mammalian cells during influenza infection, and viral titers decrease significantly on a copper surface. The underlying mechanisms include DNA damage by radicals, modulation of viral protease, M1 or neuraminidase, and morphological changes in viral particles. However, the molecular mechanisms underlying divalent cation-mediated antiviral activities are unclear. An unexpected observation of this study was that a Zn<sup>2+</sup> ion is bound by Glu68 and His137 residues at the head regions of two neighboring trimers in the crystal structure of hemagglutinin (HA) derived from A/Thailand/CU44/2006. The binding of Zn<sup>2+</sup> at high concentrations induced multimerization of HA and decreased its acid stability. The acid-induced conformational change of HA occurred even at neutral pH in the presence of Zn<sup>2+</sup>. The fusion of viral and host endosomal membranes requires substantial conformational changes in HA upon exposure to acidic pH. Therefore, our results suggest that binding of Zn<sup>2+</sup> may facilitate the conformational changes of HA, analogous to that induced by acidic pH.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Influenza virus</div> <div class="kwd" title="kwd">
X-ray crystallography</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">strategy</span></div><span class="award-id" tagx="award-id" title="award-id">NRF-2016R1E1A1A01942558</span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100004080</span><span class="institution" title="institution">Korea Drug Development Fund</span></div> </div><span class="award-id" tagx="award-id" title="award-id">KDDF-201712-14</span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Influenza viruses belonging to the family <i>Orthomyxoviridae</i> comprise four antigenic types: A, B, C, and, D. Influenza A and B viruses co-circulate and cause seasonal epidemics, and influenza C virus causes severe disease in children<sup><a href="#CR1">1</a>,<a href="#CR2">2</a></sup>. Based on the antigenicity of hemagglutinin (HA) and neuraminidase (NA), 18 HA and 11 NA subtypes of influenza A viruses have been identified in wild waterfowl, and subtypes H17N10 and H18N11 have been discovered in bats <sup><a href="#CR3">3</a>,<a href="#CR4">4</a></sup>. The various subtypes are divided into two groups, group 1 (including H1, H2 and H5) and group 2 (including H3 and H7). Thus far, three HA (H1, H2, and H3) and two NA (N1 and N2) subtypes have adapted to humans to produce pandemic (pdm) and seasonal influenza A viruses<sup><a href="#CR5">5</a></sup>. The dynamic nature of influenza viral infections, particularly due to the variation of HA, remains a big challenge in minimizing the mismatch between circulating seasonal strains and vaccines.</p> <p id="Par3">Progression of viral particles from early to late endosomes results in large conformational changes of the active form of HA that is derived from the proteolytic cleavage of HA0 precursor into HA1 and HA2<sup><a href="#CR6">6</a>,<a href="#CR7">7</a></sup>. The HA structure has two distinct regions. The head region is composed of the receptor binding and vestigial esterase domains of HA1, whereas the stem region includes a long central helix of HA2. The HA plays a critical role for acidic pH-induced membrane fusion in viral fitness<sup><a href="#CR8">8</a>,<a href="#CR9">9</a></sup> and involves the dissociation of the head regions and the extension of central coiled-coil domains in the stem regions. In this context, the stability of HA is an important attribute of influenza viral fitness, which regulates and enhances viral growth and pathogenesis<sup><a href="#CR10">10</a>,<a href="#CR11">11</a></sup>. The stability of HA is changed significantly within and among the subtypes<sup><a href="#CR10">10</a>,<a href="#CR12">12</a></sup>, and the recent A(H1N1) 2009pdm viruses evolved to exhibit significantly more stable HA as they became seasonal strains circulating globally<sup><a href="#CR13">13</a>,<a href="#CR14">14</a></sup>. Mutations in the coiled coil region of HA2 and at the interface between HA1 and HA2 subunits influence the stability of HA<sup><a href="#CR15">15</a>â€"<a href="#CR18">18</a></sup>. We also demonstrated that the recombinant HA derived from A/Korea/01/2009, a 2009pdm isolate, is a monomer in solution and crystals, where HA molecules are arranged in a head-to-head manner<sup><a href="#CR19">19</a></sup>. However, the monomeric HA has a significantly lower stability than the trimer.</p> <p id="Par4">The influenza A viruses contain a significant number of Zn<sup>2+</sup> ion<sup><a href="#CR20">20</a></sup>. The Zn<sup>2+</sup> is shown to bind to M1 protein with a tetrahedral coordination with two Cys and two His residues, where the Zn<sup>2+</sup> ion may play a structural role<sup><a href="#CR21">21</a></sup>. Cu<sup>2+</sup> and Zn<sup>2+</sup> ions, albeit less effective in the latter, significantly reduced the influenza viral titers in MDCK cells and on a copper surface<sup><a href="#CR22">22</a>,<a href="#CR23">23</a></sup>. Zn<sup>2+</sup> is also an important cofactor of many viral proteins, including nucleocapsid protein 7 of HIV-1, NS3 of hepatitis C virus, VP30 of Ebola virus, E7 of human papilloma virus, VP6 of rotavirus, and E protein of Semliki Forest virus<sup><a href="#CR24">24</a>â€"<a href="#CR28">28</a></sup>, mainly showing a structural role in zinc binding motifs. However, our understanding of the molecular mechanisms of divalent cation-mediated antiviral activities is far from complete. In this study, solving the crystal structure of HA derived from A/Thailand/CU44/2006 (CU44) led to a surprising finding of a divalent cation Zn<sup>2+</sup> bound to the head regions of two neighboring trimers. The divalent cation in the crystal structure appeared to enhance the stability of HA, but significantly decreased the melting temperature T<sub>m</sub> of the CU44 HA, depending on the Zn<sup>2+</sup> concentrations. The acid-induced conformational change of HA occurred even at neutral pH in the presence of Zn<sup>2+</sup>. Therefore, our results suggest that binding of Zn<sup>2+</sup> may facilitate the conformational change of HA, analogous to that induced by acidic pH. Here, we report the structure of HA bound to Zn<sup>2+</sup> ions and the effects of the divalent cation on pH-dependent conformational changes of HA.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="structureofcu44hawithzn2+ion" title="sec"> <div class="title" tagx="title" title="title">
Structure of Cu44 HA with Zn<sup>2+</sup> ion</div> <p id="Par5">We originally aimed to improve the stability of HA in solution and crystals to allow high resolution analyses of how pH-induced conformational changes affect its biological functions. A total of nine mutants of the CU44 HA wild-type were constructed: L69S, I71S, L156S, I184S, F260A, and A276T of HA1 in the head region, and L73S, M77I, and V91I of HA2 in the stem region (based on H3 numbering). The mutations largely accounted for reduced exposure of hydrophobic amino acid residues on the surface and increased hydrophobic residues at the monomerâ€"monomer interface within the trimer. They were expressed in insect cell culture, purified, and characterized by differential scanning fluorimetry (DSF). No improvement in protein stability was seen in terms of melting temperature (T<sub>m</sub>).</p> <p id="Par6">We attempted to determine the crystal structure of the CU44 HA wild type as a control, by molecular replacement using PDB 4EDB as a template. The structure in space group C2 (aâ€‰=â€‰215.7Â Ã…, bâ€‰=â€‰124.3Â Ã…, câ€‰=â€‰214.5Â Ã…, bâ€‰=â€‰102.9Â°) was refined at 3.2Â Ã… resolution with an R/R<sub>free</sub> of 26.3/29.5 and exhibited good overall geometry (Table <a href="#Tab1">1</a>). The HA trimers adopt an alternating up-and-down arrangement with head-to-head interactions (Fig.Â <a href="#Fig1">1</a>A). Surprisingly, anomalous difference Fourier maps calculated with phases from a model showed a pronounced peak at the head-to-head interface between two neighboring HA trimers, clearly visible at the 7 Ïƒ level (Fig. <a href="#MOESM1">S1</a>). Fluorescence scans around the anomalous edges using the HA crystals strongly suggested that Zn<sup>2+</sup> ion could contribute to the strong anomalous signal at the absorption edge near at 1.284Â Ã… (Fig. <a href="#MOESM1">S2</a>). Despite the lack of Zn<sup>2+</sup> in the crystallization medium, Zn<sup>2+</sup> ion was tentatively refined with full occupancy and B-factor of 41â€"47Â Ã…<sup>2</sup> (three Zn<sup>2+</sup> ions in the asymmetric unit). The divalent ion binds to the side chains of Glu68 and His137 from two adjacent trimers (Fig.Â <a href="#Fig1">1</a>B). In the final 2|F<sub>o</sub>|Â âˆ’Â |F<sub>c</sub>| and |F<sub>o</sub>|Â âˆ’Â |F<sub>c</sub>| difference Fourier maps, the divalent cation adopts an octahedral molecular geometry, mediated by the coordination with Glu68 and His137. They likely act as bi- and monodentate ligands, respectively, to the Zn<sup>2+</sup> ion with the coordination bond lengths of 1.9â€"2.3Â Ã…. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Data collection and refinement statistics.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" /> <th align="left">HA-Zn<sup>2+</sup> complex</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="2"><b>Data collection</b></td> </tr> <tr> <td align="left">Resolution range (Ã…)<sup>a</sup></td> <td align="left">48.48â€"3.24 (3.36â€"3.24)</td> </tr> <tr> <td align="left">Space group</td> <td align="left">C 1 2 1</td> </tr> <tr> <td align="left">Cell dimensions: a, b, c (Ã…)</td> <td align="left">215.71 124.27 214.53</td> </tr> <tr> <td align="left">Î±, Î², Î³ (Â°)</td> <td align="left">Î±â€‰=â€‰90.00, Î²â€‰=â€‰102.91, Î³â€‰=â€‰90.00</td> </tr> <tr> <td align="left">Unique reflections</td> <td align="left">87,995</td> </tr> <tr> <td align="left">Completeness (%)</td> <td align="left">99.91 (100)</td> </tr> <tr> <td align="left">I/Ïƒ(I)</td> <td align="left">7.73 (3.23)</td> </tr> <tr> <td align="left">R-pim (%)</td> <td align="left">7.89 (20.64)</td> </tr> <tr> <td align="left">Multiplicity</td> <td align="left">13.3 (12.3)</td> </tr> <tr> <td align="left">CC<sub>1/2</sub></td> <td align="left">0.99 (0.92)</td> </tr> <tr> <td align="left">Wilson B factor (Ã…<sup>2</sup>)</td> <td align="left">54.83</td> </tr> <tr> <td align="left" colspan="2"><b>Refinement</b></td> </tr> <tr> <td align="left" rowspan="2">Resolution (Ã…)</td> <td align="left">48.75â€"3.24</td> </tr> <tr> <td align="left">(3.356â€"3.24)</td> </tr> <tr> <td align="left">Reflections used in refinement</td> <td align="left">87,955 (8830)</td> </tr> <tr> <td align="left">R<sub>work</sub>/R<sub>free (%)</sub></td> <td align="left">26.3/29.5</td> </tr> <tr> <td align="left">Number of non-hydrogen atoms</td> <td align="left">24,413</td> </tr> <tr> <td align="left">Â Protein</td> <td align="left">23,024</td> </tr> <tr> <td align="left">Â Ligands</td> <td align="left">961</td> </tr> <tr> <td align="left">Â Solvent</td> <td align="left">428</td> </tr> <tr> <td align="left">RMS deviation (bonds, Ã…)</td> <td align="left">0.015</td> </tr> <tr> <td align="left">Â (Angles, Â°)</td> <td align="left">1.80</td> </tr> <tr> <td align="left">Ramachandran favored (%)</td> <td align="left">90.4</td> </tr> <tr> <td align="left">Â Allowed (%)</td> <td align="left">9.0</td> </tr> <tr> <td align="left">Â Outliers (%)</td> <td align="left">0</td> </tr> <tr> <td align="left">Average B-factors (Ã…<sup>2</sup>)</td> <td align="left">74.7</td> </tr> <tr> <td align="left">Protein</td> <td align="left">75.4</td> </tr> <tr> <td align="left">Chromophore</td> <td align="left">85.8</td> </tr> <tr> <td align="left">Solvent</td> <td align="left">16.8</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup>Values in parentheses are for the highest-resolution shell.</p> </div>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Overview of the structure of the CU44 HA. (<b>A</b>) Top and side views along the C axis of the crystal packing of HA. The HA trimers adopt an alternating up and down arrangement with head-to-head interactions, which are shown in different colors. Divalent cations are shown as red spheres. A magnified view of the HA trimers, which are shown in secondary structures in different colors (blue, pink, and lime green) for each monomer. (<b>B</b>) Zn<sup>2+</sup> ion binding site showing an octahedral coordination with two Glu68 and two His137 residues, from two adjacent HA trimers, in the 2|F<sub>o</sub>|Â âˆ’Â |F<sub>c</sub>| electron density contoured at 1.2 Ïƒ. The figures were created with PyMOL (<a href="https://www.pymol.org/">https://www.pymol.org/</a>).</p>   </div> <p /> </div> <div class="effectsofdivalentcationsonhaproteins" title="sec"> <div class="title" tagx="title" title="title">
Effects of divalent cations on HA proteins</div> <p id="Par7">In parallel, we carried out gel shift experiments involving native polyacrylamide gel electrophoresis of the CU44 HA in the presence of various divalent cations. The Cu<sup>2+</sup> and Zn<sup>2+</sup> cations promoted robust multimerization of HA (Fig.Â <a href="#Fig2">2</a>A, upper panel). To assess whether the presence of the divalent cations can impact the stability of HA, DSF was used to measure the changes in T<sub>m</sub>. Many divalent cation had no effect on the T<sub>m</sub> values of the HA, whereas the T<sub>m</sub> values in the presence of Ni<sup>2+</sup>, Zn<sup>2+</sup> and Cu<sup>2+</sup> were significantly reduced by 8Â Â°C and 18Â Â°C and could not be obtained, respectively (Fig.Â <a href="#Fig2">2</a>A, lower panel). When HA was treated with 0.1â€"1Â mM Zn<sup>2+</sup>, a significant shift in the size exclusion chromatography (SEC) profile was observed. As the concentration of Zn<sup>2+</sup> increased, multimeric forms of the HA were eluted at the void volume (Fig.Â <a href="#Fig2">2</a>B, upper panel). The HA protein sample was filtered through a 0.22Â Î¼m membrane prior to chromatography and the peak at the void volume in the SEC profile became lower as the Zn<sup>2+</sup>concentration increased, suggesting that larger aggregates of HA formed at high concentrations of Zn<sup>2+</sup>. In contrast, the HA protein also induced multimerization at acidic pH, but not to the extent of formation of the larger aggregates induced by Zn<sup>2+</sup> (Fig.Â <a href="#Fig2">2</a>B, lower panel). Additionally, the multimerization of the HA by Zn<sup>2+</sup> was not abolished by the treatment of ethylenediaminetetraacetic acid (EDTA) at concentrations of 1â€"10Â mM (Fig. <a href="#MOESM1">S3</a>). The conformational changes and multimeric associations of HA induced by Zn<sup>+2</sup> or acidic pH have been examined using SEC-MALS and binding of antibodies (Fig.Â <a href="#Fig2">2</a>C, D). The molecular weights of HA multimers induced by Zn<sup>+2</sup> or acidic pH were 1785 or 2039Â kDa, respectively, corresponding to the associations of approximately 9 HA trimers, and the HA multimers induced by Zn<sup>+2</sup> showed a relatively polydisperse population. In addition, the conformations induced by Zn<sup>+2</sup> or acidic pH were not recognized by CR9114, but by the D2 interface antibody that binds to the head domain (Fig.Â <a href="#Fig2">2</a>D)<sup><a href="#CR29">29</a></sup>. The results indicate that the stem conformations of HA induced by Zn<sup>+2</sup> or acidic pH are not recognized by CR9114 and do not represent the native prefusion conformations. Our results thus suggest that the treatment with Zn<sup>2+</sup> induces the conformations of HA, similar to those induced by acidic pH. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Characterization of HA proteins in the presence of divalent cations. (<b>A</b>) Native polyacrylamide gel electrophoresis results showing the band shifts in the presence of Zn<sup>2+</sup> and Cu<sup>2+</sup> (upper panel). The CU44 HA protein was isolated as a trimer, whereas the two cations induced multimerization or aggregation of the HA molecules. Differential scanning fluorimetry transition curves of HA proteins at different divalent cations (lower panel). HA protein was incubated at 25Â Â°C, and then the temperature was increased by 0.5Â Â°C every 30Â s for 50Â min. (<b>B</b>) The shift in elution volume of the CU44 HA in the presence of Zn<sup>2+</sup> (upper panel) and acidic pH conditions (lower panel) in SEC. The experiment was performed using a Superdex 200 GL column equilibrated with 50Â mM Tris (pH 8.0) and 100Â mM NaCl, and the shifts were monitored as the concentration of Zn<sup>2+</sup> increased from 0.1 to 1.0Â mM. (<b>C</b>) The SEC-MALS results of CU44 HA. The HA was incubated with 1Â mM Zn<sup>2+</sup> or 100Â mM MES (pH 5.5) at 24Â Â°C for 30Â min and analyzed using SEC on a Superdex 200 GL column equilibrated with PBS at a flow rate of 0.5Â ml/min. (<b>D</b>) Antibody recognition of HA conformations, CR9114 (upper panel) and D2 H1-1/H3-1 (lower panel). CU44 HA was incubated with 1Â mM Zn<sup>2+</sup> or 100Â mM MES (pH 5.5) for ELISA at 4Â Â°C overnight. The stem-specific CR9114 or head-specific D2 H1-1/H3-1 at concentrations of 400, 80, 1, 3, 0.64, 0.128, and 0.026Â ng per well was used and detected by anti-human IgG1-HRP as the secondary antibody. Absorbance at 450Â nm was monitored.</p>   </div> <p /> <p id="Par8">Comparison of the amino acid sequences of HA from different subtypes of influenza viruses revealed the conservation of the Zn<sup>2+</sup> binding residues, Glu68 and His137, in H1 strains, but not in other subtypes (Table <a href="#Tab2">2</a>). In some strains, including the 1918pdm and the 1933â€"1936 seasonal strains, the corresponding amino acid residues of HA were substituted with Asp or Tyr, respectively. Nevertheless, the unique binding of Zn<sup>2+</sup> to HA in a broad range of H1 subtype influenza A viruses reveal a subtype-specific interaction of HA with Zn<sup>2+</sup>. <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>Amino acid residue variations of HA among H1 strains and subtypes.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2" /> <th align="left" colspan="2">Residue number</th> </tr> <tr> <th align="left">68</th> <th align="left">137</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="3"><b>Strains</b></td> </tr> <tr> <td align="left">A/Brevig_Mission/1/1918</td> <td align="left">D</td> <td align="left">Y</td> </tr> <tr> <td align="left">A/Puerto Rico/8/1934</td> <td align="left">D</td> <td align="left">H</td> </tr> <tr> <td align="left">A/Albany/4835/1948</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left">A/Malaysia/1954</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left">A/California/45/1978</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left">A/Victoria/36/1988</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left">A/Texas/36/1991</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left">A/Hong_Kong/1035/1998</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left">A/Taiwan/141/2002</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left">A/Thailand/CU44/2006</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left">A/Florida/12/2007</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left">A/Hawaii/02/2008</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left">A/California/07/2009</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left">A/Pennsylvania/27/2016</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left" colspan="3"><b>Subtypes</b></td> </tr> <tr> <td align="left">H1</td> <td align="left">E</td> <td align="left">H</td> </tr> <tr> <td align="left">H2</td> <td align="left">D</td> <td align="left">V</td> </tr> <tr> <td align="left">H3</td> <td align="left">D</td> <td align="left">R</td> </tr> <tr> <td align="left">H4</td> <td align="left">D</td> <td align="left">R</td> </tr> <tr> <td align="left">H5</td> <td align="left">D</td> <td align="left">Y</td> </tr> <tr> <td align="left">H6</td> <td align="left">D</td> <td align="left">Y</td> </tr> <tr> <td align="left">H7</td> <td align="left">D</td> <td align="left">R</td> </tr> <tr> <td align="left">H8</td> <td align="left">D</td> <td align="left">A</td> </tr> <tr> <td align="left">H9</td> <td align="left">D</td> <td align="left">D</td> </tr> <tr> <td align="left">H10</td> <td align="left">D</td> <td align="left">R</td> </tr> <tr> <td align="left">H11</td> <td align="left">D</td> <td align="left">F</td> </tr> <tr> <td align="left">H12</td> <td align="left">D</td> <td align="left">N</td> </tr> <tr> <td align="left">H13</td> <td align="left">T</td> <td align="left">D</td> </tr> <tr> <td align="left">H14</td> <td align="left">D</td> <td align="left">R</td> </tr> <tr> <td align="left">H15</td> <td align="left">D</td> <td align="left">R</td> </tr> <tr> <td align="left">H16</td> <td align="left">A</td> <td align="left">D</td> </tr> <tr> <td align="left">H17</td> <td align="left">D</td> <td align="left">F</td> </tr> </tbody> </table>  <p /> </div> <div class="ph-dependentconformationalchangesofcu44ha" title="sec"> <div class="title" tagx="title" title="title">
pH-dependent conformational changes of CU44 HA</div> <p id="Par9">Acidic pH in the endosome induces a large conformational change of HA whose stability to an activation pH is necessary for influenza virus pathogenicity and transmissibility<sup><a href="#CR6">6</a>â€"<a href="#CR9">9</a>,<a href="#CR14">14</a></sup>. In order to examine the pH-dependent stability of the CU44 HA, DSF experiments were carried out by incubating the HA at different pH conditions followed by rapid neutralization. Two distinct T<sub>m</sub> values were obtained for the CU44 HA, 58Â Â°C at pH 5.9â€"7.0 and 56Â Â°C at pH 5.0â€"5.7 (Fig.Â <a href="#Fig3">3</a>A, upper panel). In comparison, the conformational changes started to occur at pH 6.0 upon a 10-min incubation without neutralization, with the changes occurring at pHâ€‰<div class="â€‰6.0_UNKNOWN" href="#Fig3">
3A, middle panel). The T<sub>m</sub> values continuously decreased from 52Â Â°C at pH 5.9â€"5.6 toâ€‰~â€‰40Â Â°C at pH 5.0. Upon a 24-h incubation, two distinct conformations were apparent, with T<sub>m</sub> values at 58Â Â°C at pH 7.0 and 48Â Â°C at pH below 6.0 (Fig.Â <a href="#Fig3">3</a>A, lower panel). We postulate that these two distinct conformations correspond to pre- and postfusion states of the CU44 HA and its transition from a reversible to irreversible conformations occurs at pH 5.6â€"5.8. </div></p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Differential scanning fluorimetry transition curves of HA proteins at different pH conditions. (<b>A</b>) The T<sub>m</sub> values of the CU44 HA were measured after it was incubated at different pH from 7.0 to 5.0 for 10Â min followed by rapid neutralization (upper panel) and without neutralization (middle panel) at room temperature, and 24Â h incubation without neutralization (lower panel). HA protein was incubated at 25Â Â°C, and the temperature was increased by 0.5Â Â°C every 30Â s for 50Â min. (<b>B</b>) pH-dependent aggregation using native polyacrylamide gel electrophoresis, (<b>C</b>) differential scanning fluorimetry, and (<b>D</b>) Proteolytic cleavage of the CU44 HA. HA aggregated at pH 5.4 in the absence of Zn<sup>2+</sup>, whereas it aggregated readily at pH 7.4 in the presence of Zn<sup>2+</sup>, which was also dependent on the concentrations of Zn<sup>2+</sup>. The T<sub>m</sub> values of the HA were measured at concentrations of 0, 0.125, 0.25, 0.5, and 1Â mM of Zn<sup>2+</sup> at different pH conditions (pH 7.4, 6.0, 5.9, 5.8, and 5.4) followed by rapid neutralization at room temperature. The T<sub>m</sub> values for the CU44 HA significantly decreased from 58 to 54Â Â°C. For the proteolytic cleavage, after the HA was mixed with 10Â Âµg of TPCK-trypsin at 37Â Â°C for 10Â min, it was stopped by adding 5X sample buffer and heating at 95Â Â°C for 5Â min, and analysed by SDS-PAGE.</p>   </div> <p /> </div> <div class="effectsofzn2+onconformationalchangesandproteolyticactivationofha" title="sec"> <div class="title" tagx="title" title="title">
Effects of Zn<sup>2+</sup> on conformational changes and proteolytic activation of HA</div> <p id="Par10">To study the effect of Zn<sup>2+</sup> on the pH-dependent stability of the CU44 HA, the HA protein was incubated with Zn<sup>2+</sup> ion for 10Â min followed by rapid neutralization at different pH conditions. Stepwise multimerization of HA proteins was observed as the pH decreased, reaching complete multimerization at below pH 5.4, whereas it occurred readily in the presence of Zn<sup>2+</sup> even at neutral pH conditions (Fig.Â <a href="#Fig3">3</a>B). More pronounced multimerization was evident at higher concentrations of Zn<sup>2+</sup>. In addition, the increased concentration of Zn<sup>2+</sup> reduced the T<sub>m</sub> values of the HA even at neutral pH in a concentration-dependent manner (Fig.Â <a href="#Fig3">3</a>C). The T<sub>m</sub> values decreased from 58 to 54Â Â°C. Thus, the conformational changes of the HA induced by Zn<sup>2+</sup> were similar to that occurring upon acidic incubation without neutralization, which occurred more readily as the concentration of Zn<sup>2+</sup> increased.</p> <p id="Par11">Infectivity of influenza viruses depends on cleavage activation of the HA precursor into HA1 and HA2 by host proteases<sup><a href="#CR30">30</a></sup>. We characterized pH-dependent proteolytic activation of the CU44 HA using SDS-PAGE. The HA1 subunit was completely degraded at a pH below 5.8, as it was highly susceptible to trypsin cleavage after the acid-induced conformational change (Fig.Â <a href="#Fig3">3</a>D). At 1Â mM Zn<sup>2+</sup>, the CU44 HA exhibited greatly enhanced susceptibility to proteolytic degradation even at neutral pH, similar to that induced by acidic pH.</p> </div> <div class="effectsofzn2+onhawild-typeandmutants" title="sec"> <div class="title" tagx="title" title="title">
Effects of Zn<sup>2+</sup> on HA wild-type and mutants</div> <p id="Par12">Although the thermal stability of HA was significantly reduced by Zn<sup>2+</sup> and the crystal structure was determined at 3.2Â Ã… resolution, the N-linked glycan structure was exceptionally well-defined in the 2|F<sub>o</sub>|Â âˆ’Â |F<sub>c</sub>| electron density maps, which is unusual in HA structures. It showed a branched sugar chain of up to six monosaccharide units attached to Asn residues of the head region (Figs. <a href="#Fig4">4</a>, <a href="#MOESM1">S4</a>). The glycans attached to Asn54 and Asn87 are much closer to the Zn<sup>2+</sup> binding site than other glycans in the head region, suggesting fairly stable conformations. In addition, the multimerization of glycosylated or deglycosylated HA induced by Zn<sup>2+</sup> was not prevented by the treatment of EDTA (Fig. <a href="#MOESM1">S3</a>, upper and middle panels). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Glycan structures attached to Asn54 (upper panel) and Asn87 (lower panel) of the CU44 HA. The 2|F<sub>o</sub>|Â âˆ’Â |F<sub>c</sub>| electron density contoured at 1.0 Ïƒ and polder omit difference electron density contoured at 3.0 Ïƒ (in blue mesh) and at 3.5 Ïƒ (in brown mesh). The glycans are closer to the Zn<sup>2+</sup> binding site than other glycans in the head region. The monosaccharide units are shown in stick representations. The figures were created using PyMOL (<a href="https://www.pymol.org/">https://www.pymol.org/</a>).</p>   </div> <p /> <p id="Par13">In order to examine the effects of Zn<sup>2+</sup> on biophysical properties of HA, single and double mutations at the Zn<sup>2+</sup> binding site were introduced: the H137A single mutant displayed the essentially identical T<sub>m</sub> of the wild-type in the absence of Zn<sup>2+</sup> and the same changes in T<sub>m</sub> in the presence of Zn<sup>2+</sup> (Fig. <a href="#MOESM1">S5</a>). The incubation of Zn<sup>2+</sup> with the mutants induced pH-dependent conformational changes similar to those of the wild-type, where high concentrations of Zn<sup>2+</sup> equally promoted multimerization of the mutants, similar to those produced by the wild-type (Figs. <a href="#MOESM1">S3</a> and <a href="#MOESM1">S5</a>, lower panels for the E68A/H137A and H137A mutants, respectively). Therefore, the HA, already transformed into the postfusion conformation at high concentrations of Zn<sup>2+</sup>, is not blocked by the mutation or EDTA treatment. When the purified HA crystallized at 10â€"20Â mg/ml by mixing with an equal volume of precipitant solution, protein crystals were produced at less than 0.4Â mM Zn<sup>2+</sup>, whereas precipitates were found at higher than 0.4Â mM. No crystals or precipitates were found at 0Â mM Zn<sup>2+</sup> (data not shown). Taken together, our results suggest that Zn<sup>2+</sup> binding to HA has two phases: Zn<sup>+2</sup> binds specifically to the head regions of HA at low concentrations, whereas, at high concentrations, it binds nonspecifically to HA, causing conformational changes similar to those induced by acidic pH.</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par14">HA is a highly dynamic molecule capable of mediating a pH-dependent transition from a prefusion to postfusion state during virus entry. Induction of irreversible conformational changes upon acidification results in the formation of a stable postfusion state via a spring-loaded mechanism<sup><a href="#CR31">31</a></sup>. The prefusion conformation of HA is metastable, and the receptor binding head region acts as a clamp to keep the stem region in its metastable prefusion state at neutral pH. Acidification shifts the dynamic equilibrium of HA that favors the progression of membrane fusion<sup><a href="#CR31">31</a></sup>. We initially constructed nine mutants to improve the stability and quality of HA in solution and crystals, and the wild-type HA crystal structure derived from A/Thailand/CU44/2006 was determined as a control. Surprisingly, the highest peak in the anomalous difference electron density maps was located at the head domain at the inter-trimer interface, which was tentatively identified as Zn<sup>2+</sup>, based on native gel electrophoresis and absorption-edge fluorescence scan results.</p> <p id="Par15">The Zn<sup>2+</sup> divalent cation is essential for life, although excess Zn<sup>2+</sup> is toxic to the cells, possibly due to its inhibition of key enzymes<sup><a href="#CR32">32</a></sup>. Zn<sup>2+</sup> is involved in many biological processes including infection of a variety of microbial and viral pathogens. During the infection, the innate immune system controls the concentration of Zn<sup>2+</sup> to prevent the growth of pathogens<sup><a href="#CR33">33</a></sup>. Zn<sup>2+</sup> was present in influenza virions and binds to M1 protein<sup><a href="#CR20">20</a>,<a href="#CR21">21</a></sup>. In addition, the titers of influenza viruses were significantly reduced by adding Zn<sup>2+</sup> in MDCK cells<sup><a href="#CR22">22</a></sup>. The mode of inhibition by Zn<sup>2+</sup> against influenza viruses was investigated in the context of DNA damage by radicals, viral protease or neuraminidase inhibition, or morphological changes of virions<sup><a href="#CR21">21</a>,<a href="#CR22">22</a></sup>. However, the molecular mechanism of divalent ion-mediated inhibition of influenza viruses is not well understood.</p> <p id="Par16">In the crystal structure, the HA trimers line up in an alternating up and down orientation with head-to-head interactions, where Zn<sup>2+</sup> is bound between two adjacent HA trimers. This divalent cation may be acquired from the PEG 2000 used in crystallization or the medium of insect cells. In aqueous solution, Zn<sup>2+</sup> can be coordinated with up to six water molecules, but it is usually tetrahedrally coordinated in zinc finger proteins and enzymes<sup><a href="#CR34">34</a></sup>. The coordination geometry of four or five coordination to Zn<sup>2+</sup> is applied to catalysis with high Lewis acidity, whereas it is rarely hexacoordinated in catalytic binding sites<sup><a href="#CR35">35</a>,<a href="#CR36">36</a></sup>. In the HA structure, the carboxylic oxygen atoms of Glu68 and imidazole nitrogen atoms of His137 are directly coordinated to the divalent cation, which results in a distorted octahedral, six-coordination geometry, involving bidentate interactions of carboxylic groups. The Znâ€"O and Znâ€"N bond lengths are from 2.0 to 2.2Â Ã…. The mean Znâ€"O and Znâ€"N distances from octahedral structures in the Cambridge Structural Database and aqueous solution are 2.10â€"2.11 Ã…<sup><a href="#CR35">35</a></sup>.</p> <p id="Par17">The amino acid residues, Glu68 and His137, coordinated to Zn<sup>2+</sup> are highly conserved in a broad range of H1 subtypes, although they are substituted with Asp or Tyr, respectively, in the early 1918 pandemic and the seasonal 1933â€"1936 strains, and not conserved in other subtypes and type B (Table <a href="#Tab2">2</a>). It is likely that the Zn<sup>2+</sup> binding has evolved to play a structural role in the H1 subtype viruses for the past 80Â years. Moreover, it was previously suggested that the acid stability of HA during membrane fusion regulates interspecies adaptation of the influenza viruses<sup><a href="#CR37">37</a></sup>, in which the HA acid stability affects pathogenicity with pandemic capability. In this context, the arrangement of the HA trimers with head-to-head interactions in the HA-Zn<sup>2+</sup> complex is unique, and the Zn<sup>2+</sup>-induced conformational changes of HA equivalent to those upon acidification are very striking. Recent analyses revealed that acidification and receptor binding shifts the dynamic equilibrium of HA in favor of forward progression to membrane fusion, and either receptor binding or the presence of a target membrane promotes HA2 conversion to the postfusion state<sup><a href="#CR38">38</a></sup>. Similarly, enhanced interactions of the head regions by the presence of Zn<sup>2+</sup> at low concentrations can provide an efficient way of stabilization of HA, whereas high concentrations of Zn<sup>2+</sup> induces significant conformational changes of HA, which may result in clustering of the HA molecules, analogous to receptor-promoted formation of the postfusion conformations.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="cloningandbaculovirusproductionofha" title="sec"> <div class="title" tagx="title" title="title">
Cloning and baculovirus production of HA</div> <p id="Par18">The recombinant HA protein derived from A/Thailand/CU44/2006 (CU44) was produced in insect cells using recombinant baculovirus expression vectors, and the vector was constructed in previous work<sup><a href="#CR39">39</a></sup>. The synthesized cDNA of the CU44 wild-type HA after RNA extraction was amplified using polymerase chain reaction (PCR). The wild-type HA gene (1-503 of HA0 based on H3 numbering) was cloned into pFastBac HT A (INVITROGEN, Carlsbad, CA, USA) downstream of the gp67 secretion signal sequence of the transfer vector pAcGP67A (BD BIOSCIENCES, Woburn, MA, USA). The cloned construct had residues 18â€"344 (1-326) (HA1) and 345â€"520 (1-176) (HA2), with a thrombin cleavage site, foldon domain, and 6â€‰Ã—â€‰His-tag downstream of the HA gene sequence. The recombinant HA protein contained additional plasmid-encoded residues ADPG and RSLVPR at the N- and C-terminus, respectively. Mutant HA genes were prepared by site-directed mutagenesis using a pair of complementary oligonucleotides containing the H137A mutation in the gene (MACROGEN, Seoul, Korea). Briefly, ten cycles of PCR was performed with mutant oligonucleotides to amplify the mutant HA gene, which was treated with Dpn1 (NEB, MA, USA) to remove wild type HA genes for 1Â h at 37Â Â°C. Plasmids encoding the mutant HA gene (H137A) was transformed and amplified in <i>Escherichia coli</i> strain DH5Î± and the recombinant bacmid was generated according to the Bac-to-Bac expression system protocol (INVITROGEN). The sequences were confirmed by automated sequencing (MACROGEN, Seoul, Korea). The CU44 HA in this study showed three mutations different from the original CU44 HA: HA1 M116I and HA2 I91V and N169S, which was used as a wild-type. These mutations also exist in nature. <i>Spodoptera frugiperda</i> insect cells were transfected with the recombinant bacmid using Cellfectin II (INVITROGEN), which were harvested at 72Â h post-transfection and centrifuged at 2000Â rpm for 15Â min to obtain baculovirus, expressing recombinant HA in the supernatant. The transfection efficiency was confirmed by PCR after extraction of DNA from 400Â Î¼l of virus.</p> </div> <div class="haproteinpurification" title="sec"> <div class="title" tagx="title" title="title">
HA protein purification</div> <p id="Par19">Baculovirus containing the CU44 HA wild-type or mutant (H137A) gene was used to infect suspension cultures of Hi5 cells, as described previously<sup><a href="#CR39">39</a></sup>. After 3Â days at 28Â Â°C, the culture medium was harvested. After centrifugation at 4000Â rpm for 30Â min, the supernatant was applied to a nickelâ€"nitrilotriacetic acid (Niâ€"NTA) affinity column (QIAGEN, Hilden, Germany) equilibrated with 20Â mM Trisâ€"HCl, pH 8.0, and 100Â mM NaCl. The column was washed with buffer containing 30Â mM imidazole, and the precursor HA protein was eluted in an imidazole gradient (30â€"400Â mM). The eluted fractions were dialyzed against 20Â mM Trisâ€"HCl (pH 8.0) and 20Â mM NaCl, and hydrolyzed by TPCK-treated trypsin for 4â€"5Â h at 4Â Â°C for cleavage of HA0 into HA1 and HA2 and removal of the foldon and 6â€‰Ã—â€‰His-tag. The active form of HA was purified using Superdex 200 10/300 GL size exclusion chromatography in 50Â mM Trisâ€"HCl (pH 8.0) and 100Â mM NaCl, connected to an Ã"KTA FPLC system (GE Healthcare, Buckinghamshire, UK). It was concentrated in an Amicon 10,000 MWCO concentrator (Merck Millipore, Billerica, MA, USA) to 10Â mg/ml for characterization and crystallization.</p> </div> <div class="cloning,expression,andpurificationofendoh" title="sec"> <div class="title" tagx="title" title="title">
Cloning, expression, and purification of Endo H</div> <p id="Par20">The synthesized cDNA encoding for endo-Î²-N-acetylglucosaminidase H (Endo H), a glycohydrolase secreted by <i>Streptomyces plicatus</i><sup><a href="#CR40">40</a></sup>, was cloned into the PMAL-p2 plasmid and transformed into <i>Escherichia coli</i> BL21 (DE3) (INVITROGEN, Carlsbad, USA). The cells after 0.4Â mM IPTG induction were harvested by centrifugation, sonicated, and centrifuged at 12,000Ã—<i>g</i> for 20Â min. The supernatant containing the Endo H was purified by affinity and size exclusion chromatography using amylose (NEB, MA, USA) and Superdex 200 10/300 (GE HEALTHCARE, Uppsala, Sweden) columns, respectively. Using the purified Endo H, HA samples were treated at 0.5Â Î¼g/ml in phosphate buffer (pH 6.8) at 4Â Â°C for 30Â h and separated from the Endo H using a 10Â kDaÂ MW cutoff membrane (MERCK MILLIPORE, Billerica, MA, USA). Deglycosylation of HA by the Endo H was determined by gel-shift assays.</p> </div> <div class="nativepolyacrylamidegelelectrophoresis(page)" title="sec"> <div class="title" tagx="title" title="title">
Native polyacrylamide gel electrophoresis (PAGE)</div> <p id="Par21">The purified proteins were prepared under native condition (mixed with 5â€‰Ã—â€‰loading buffer, 312.5Â mM Trisâ€"HCl (pH 6.8), 50% glycerol, and 0.05% bromophenol blue, without Î²-mercaptoethanol and sodium dodecyl sulfate). The HA wild-type or mutant (2â€"5Â Âµg) was incubated in 100Â mM HEPES or MES and 100Â mM NaCl at different pH and 25Â Â°C for 1Â h. Electrophoresis was carried out in 25Â mM Trisâ€"HCl buffer (pH 8.8) and 192Â mM glycine at 200Â V and 80Â mA for 1Â h.</p> <p id="Par22">For a gel shift assay of divalent cations, the CU44 HA was mixed with 5Â mM Li<sup>+</sup>, NA<sup>+</sup>, K<sup>+</sup>, Mg<sup>2+</sup>, Ca<sup>2+</sup>, Fe<sup>2+</sup>, Co<sup>2+</sup>, Ni<sup>2+</sup>, Cu<sup>2+</sup>, and Zn<sup>2+</sup> ions in 20Â mM Trisâ€"HCl (pH 8.0) and 100Â mM NaCl overnight. For HA-Zn<sup>2+</sup> complexes, the CU44 HA was mixed with Zn<sup>2+</sup> in 0â€"1Â mM in 20Â mM Trisâ€"HCl (pH 8.0) and 100Â mM NaCl overnight, which was then incubated in 100Â mM MES-HEPES at pH 7.4, pH 6.0, pH 5.9, pH 5.8, or pH 5.4 at room temperature for 1Â h. It was either recovered to neutral pH by mixing with 2Â M Trisâ€"HCL (pH 8.5) after 10Â min, or incubated further for 24Â h. The samples were submitted for native PAGE. In the case of the EDTA treatment, the HA-Zn<sup>2+</sup> complex was incubated in 1Â mM or 10Â mM EDTA for 1Â h or overnight. The electrophoresis was carried out in 25Â mM Trisâ€"HCl buffer (pH 8.8) with 192Â mM glycine at 200Â V and 80Â mA for 1Â h. The gels were stained using Coomassie Brilliant Blue R-250 (SIGMA-ALDRICH, St. Louis, MO, USA).</p> </div> <div class="sizeexclusionchromatographyandmulti-anglelightscattering(sec-mals)analysis" title="sec"> <div class="title" tagx="title" title="title">
Size exclusion chromatography and multi-angle light scattering (SEC-MALS) analysis</div> <p id="Par23">The elution shift of the CU44 HA-Zn<sup>2+</sup> complex was examined by mixing the CU44 HA with 0â€"1Â mM Zn<sup>2+</sup> in 20Â mM Trisâ€"HCl (pH 8.0) and 100Â mM NaCl overnight. Each sample was applied to a Superdex 200 GL column (GE HEALTHCARE) equilibrated with 50Â mM Tris (pH 8.0) and 100Â mM NaCl at a flow rate of 0.5Â ml/min, connected to a UFLC system (SHIMADZU, Kyoto, Japan). Light scattering and refractive index were measured using in-line WYATT-787-TS miniDAWN TREOS (WYATT Technology, Santa Barbara, CA, USA), and the data were analysed by Astra 6 software.</p> </div> <div class="harecognitionbyantibodiesusingelisa" title="sec"> <div class="title" tagx="title" title="title">
HA recognition by antibodies using ELISA</div> <p id="Par24">CU44 HA was incubated with 1Â mM Zn<sup>2+</sup> or 100Â mM MES (pH 5.5) for ELISA at 4Â Â°C overnight. The 8-well strip was coated with the native, Zn<sup>2+</sup>, or acidic-pH incubated CU44 HA in a total volume of 100Â Âµl (2Â ng/Âµl) in microplates (Corning). Following blocking with 1% BSA and PBS washing, primary antibodies (the stem-specific CR9114 or head-specific D2 H1-1/H3-1) were applied at 400, 80, 1, 3, 0.64, 0.128, and 0.026Â ng per well at 37Â Â°C for 1Â h. They were then washed with PBS, and detected by anti-human IgG1-HRP (Abcam, Cambridge, UK) as the secondary antibody for 1Â h. 150Â Âµl of 3,3â€²,5,5â€²-tetramethylbenzidine (Sigma-Aldrich) was used as substrate and the reaction was stopped by the addition of 2Â N sulfuric acid, and the absorbance at 450Â nm was monitored.</p> </div> <div class="dsf" title="sec"> <div class="title" tagx="title" title="title">
DSF</div> <p id="Par25">DSF was used to measure the shift in transition temperature of HA at different pH conditions, using Stratagene MX3005P (AGILENT TECHNOLOGIES, Santa Clara, CA, USA). The reaction mixture (25Â Âµl) containing 5Â Âµg of the HA wild-type or mutant, 5X SYPRO Orange (INVITROGEN), 100Â mM HEPES or MES and 100Â mM NaCl was incubated at different pHs at 25Â Â°C for 1Â h. Temperature increment at the rate of 0.5Â Â°C every 30Â s for 50Â min and relative fluorescence units were recorded at the excitation and emission wavelengths of 492Â nm and 610Â nm, respectively. The transition temperature was calculated from the maxima of the first derivative of relative fluorescence units/temperature using MxPro QPCR Software.</p> </div> <div class="trypsindigestionforproteasesusceptibility" title="sec"> <div class="title" tagx="title" title="title">
Trypsin digestion for protease susceptibility</div> <p id="Par26">For proteolytic cleavage analysis, the CU44 HA was mixed with Zn<sup>2+</sup> in 0â€"1Â mM in 20Â mM Trisâ€"HCl (pH 8.0) and 100Â mM NaCl overnight, which was then incubated in 100Â mM MES-HEPES at pH 7.4, pH 6.0, pH 5.9, pH 5.8, or pH 5.4 at room temperature for 1Â h. It was recovered to neutral pH by mixing with 2Â M Trisâ€"HCL (pH 8.5) and mixed with 10Â Âµg of TPCK-Trypsin (THERMO FISHER SCIENTIFIC, Waltham, MA, USA) at 37Â Â°C for 10Â min. Each reaction was stopped by adding 5â€‰Ã—â€‰sample buffer and heating at 95Â Â°C for 5Â min. Aliquots were analyzed by SDS-PAGE.</p> </div> <div class="crystallizationanddatacollection" title="sec"> <div class="title" tagx="title" title="title">
Crystallization and data collection</div> <p id="Par27">The purified HA was concentrated to 10Â mg/ml in 50Â mM Trisâ€"HCl (pH 8.0) and 100Â mM NaCl, and crystallized by mixing with an equal volume of precipitant solution using the sitting drop vapor diffusion method at room temperature for 3â€"7Â days. The precipitant solution contained 0.1Â M Trisâ€"HCl, pH 7.25, 20% PEG 2000 and 2% dimethylsulfoxide used as additive. After a week, small crystals were obtained. Diffraction data were collected with the crystals flash-cooled at 100Â K in a stream of liquid N<sub>2</sub> in the mother liquor containing 10% ethylene glycerol using synchrotron radiation sources at beamlines BL-17A at photon factory (Tsukuba, Japan) and 11C at Pohang Light Source-II (PLS-II), Korea. The crystals of HA-Zn<sup>2+</sup> complex were diffracted to 3.4Â Ã… resolution, which belong to space group C2 with unit cell dimensions of aâ€‰=â€‰215.7Â Ã…, bâ€‰=â€‰124.2Â Ã…, câ€‰=â€‰214.5Â Ã…, aâ€‰=â€‰90Â°, bâ€‰=â€‰102.9Â°, and câ€‰=â€‰90Â° (Table <a href="#Tab1">1</a>). All data were processed and scaled using the XDS<sup><a href="#CR41">41</a></sup>.</p> </div> <div class="structuresolutionandrefinement" title="sec"> <div class="title" tagx="title" title="title">
Structure solution and refinement</div> <p id="Par28">The crystal structure of the CU44 HA-Zn<sup>2+</sup> complex was determined by molecular replacement using the CU44 HA structure (PDB ID 4EDB) as a template using PHENIX<sup><a href="#CR42">42</a></sup>. The initial solution was optimized by rigid body refinement, which produced interpretable electron density maps for the overall structure. Manual adjustment of the backbone and side chains was conducted and crystallographic refinement was carried out using the PHENIX program<sup><a href="#CR42">42</a></sup>. Difference Fourier maps, 2|F<sub>o</sub>|Â âˆ’Â |F<sub>c</sub>| and |F<sub>o</sub>|Â âˆ’Â |F<sub>c</sub>|, were used to model the active site or loop regions. After a few rounds of model rebuilding, water molecules were added using the |F<sub>o</sub>|Â âˆ’Â |F<sub>c</sub>| map peaks above 3.0 Ïƒ. The R<sub>free</sub> value was used as an indicator to validate the water picking and refinement procedure and to guard against possible overfitting of the data<sup><a href="#CR43">43</a></sup>. An anomalous difference Fourier map was generated to locate the position of Zn<sup>2+</sup> ion, and Polder omit maps for different regions were generated<sup><a href="#CR44">44</a></sup>. The final R factor and R<sub>free</sub> were 0.263 and 0.295, respectively. Data and refinement statistics for the HA-Zn<sup>2+</sup> complex are presented in Table <a href="#Tab1">1</a>. Stereochemical analysis of all refined structures using PHENIX<sup><a href="#CR42">42</a></sup> revealed 1.6% outliers in the Ramachandran plot with 81.2% favored region among the 2577 residues. These outliers were located in the disordered loop regions.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_72368_MOESM1_ESM.pdf">LINK</a> <p>Supplementary figures.</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p>is available for this paper at 10.1038/s41598-020-72368-x.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We wish to acknowledge the technical support from the staffs at beamline 11C at Pohang Light Source-II (PLS-II), Korea and BL-17A at the Photon factory, Japan, and thank Dr. J. Lee for providing an interface antibody gene (D2 H1-1/H3-1). This work was supported by grants from the strategy (NRF-2016R1E1A1A01942558) and by Korea Drug Development Fund funded by Ministry of Science and ICT, Ministry of Trade, Industry, and Energy, and Ministry of Health and Welfare (KDDF-201712-14).</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>K.H.K. and M.S.C. conceived and analyzed the HA experimental characterization, J.H.S., H.J.K., D.B.L., J.S.A., E.J.K., and J.-H. Lee conducted mutation and characterization experiments, and M.S.C. and K.H.K. wrote manuscript.</p> <div class="title" tagx="title" title="title">
Data availability</div> <p>The refined model has been deposited in the Protein Data Bank (PDB ID 6LKS).</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par29">The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsuzaki</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Clinical features of influenza C virus infection in children</span><span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">193</span><span class="fpage" tagx="fpage" title="fpage">1229</span><span class="lpage" tagx="lpage" title="lpage">1235</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/502973">10.1086/502973</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16586359">16586359</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krammer</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Influenza</span><span class="source" tagx="source" title="source">Nat. Rev. Dis. Prim.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">3</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41572-018-0002-y">10.1038/s41572-018-0002-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29955068">29955068</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">SX</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">New world bats harbor diverse influenza A viruses</span><span class="source" tagx="source" title="source">Plos Pathog.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e1003657</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1003657">10.1371/journal.ppat.1003657</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24130481">24130481</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Hemagglutinin homologue from H17N10 bat influenza virus exhibits divergent receptor-binding and pH-dependent fusion activities</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">110</span><span class="fpage" tagx="fpage" title="fpage">1458</span><span class="lpage" tagx="lpage" title="lpage">1463</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1218509110">10.1073/pnas.1218509110</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23297216">23297216</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schrauwen</span><span class="given-names" tagx="given-names" title="given-names">EJA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fouchier</span><span class="given-names" tagx="given-names" title="given-names">RAM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Host adaptation and transmission of influenza A viruses in mammals</span><span class="source" tagx="source" title="source">Emerg. Microbes Infect.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">e9</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/emi.2014.9">10.1038/emi.2014.9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26038511">26038511</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Godley</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Introduction of intersubunit disulfide bonds in the membrane-distal region of the influenza hemagglutinin abolishes membrane-fusion activity</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">68</span><span class="fpage" tagx="fpage" title="fpage">635</span><span class="lpage" tagx="lpage" title="lpage">645</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0092-8674(92)90140-8">10.1016/0092-8674(92)90140-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1739972">1739972</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kemble</span><span class="given-names" tagx="given-names" title="given-names">GW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danieli</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lipid-anchored influenza hemagglutinin promotes hemifusion, not complete fusion</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">76</span><span class="fpage" tagx="fpage" title="fpage">383</span><span class="lpage" tagx="lpage" title="lpage">391</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0092-8674(94)90344-1">10.1016/0092-8674(94)90344-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8293471">8293471</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bullough</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughson</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skehel</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiley</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structure of influenza haemagglutinin at the pH of membrane fusion</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">371</span><span class="fpage" tagx="fpage" title="fpage">37</span><span class="lpage" tagx="lpage" title="lpage">43</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/371037a0">10.1038/371037a0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8072525">8072525</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skehel</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiley</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span></span><span class="mixed-article-title" title="mixed-article-title">N- and C-terminal residues combine in the fusion-pH influenza hemagglutinin HA<sub>2</sub> subunit to form an N cap that terminates the triple-stranded coiled coil</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">96</span><span class="fpage" tagx="fpage" title="fpage">8967</span><span class="lpage" tagx="lpage" title="lpage">8972</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.96.16.8967">10.1073/pnas.96.16.8967</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10430879">10430879</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DuBois</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Acid stability of the hemagglutinin protein regulates H5N1 influenza virus pathogenicity</span><span class="source" tagx="source" title="source">Plos Pathog.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">e1002398</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1002398">10.1371/journal.ppat.1002398</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22144894">22144894</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaraket</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Increased acid stability of the hemagglutinin protein enhances H5N1 influenza virus growth in the upper respiratory tract but is insufficient for transmission in ferrets</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">9911</span><span class="lpage" tagx="lpage" title="lpage">9922</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/Jvi.01175-13">10.1128/Jvi.01175-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23824818">23824818</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galloway</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reed</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinhauer</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influenza HA subtypes demonstrate divergent phenotypes for cleavage activation and ph of fusion: implications for host range and adaptation</span><span class="source" tagx="source" title="source">Plos Pathog.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e1003151</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1003151">10.1371/journal.ppat.1003151</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23459660">23459660</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Structural stability of influenza A(H1N1)pdm09 virus</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">88</span><span class="fpage" tagx="fpage" title="fpage">4828</span><span class="lpage" tagx="lpage" title="lpage">4838</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/Jvi.02278-13">10.1128/Jvi.02278-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24522930">24522930</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russier</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular requirements for a pandemic influenza virus: an acid-stable hemagglutinin protein</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">113</span><span class="fpage" tagx="fpage" title="fpage">1636</span><span class="lpage" tagx="lpage" title="lpage">1641</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1524384113">10.1073/pnas.1524384113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26811446">26811446</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cotter</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">ZY</span></span></span><span class="mixed-article-title" title="mixed-article-title">A single amino acid in the stalk region of the H1N1pdm influenza virus HA protein affects viral fusion, stability and infectivity</span><span class="source" tagx="source" title="source">Plos Pathog.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e1003831</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1003831">10.1371/journal.ppat.1003831</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24391498">24391498</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Early steps of the conformational change of influenza virus hemagglutinin to a fusion active stateâ€"stability and energetics of the hemagglutinin</span><span class="source" tagx="source" title="source">BBA Biomembr.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">1614</span><span class="fpage" tagx="fpage" title="fpage">3</span><span class="lpage" tagx="lpage" title="lpage">13</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0005-2736(03)00158-5">10.1016/S0005-2736(03)00158-5</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinhauer</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wharton</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skehel</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiley</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Studies of the membrane-fusion activities of fusion peptide mutants of influenza-virus hemagglutinin</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">6643</span><span class="lpage" tagx="lpage" title="lpage">6651</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.69.11.6643-6651.1995">10.1128/JVI.69.11.6643-6651.1995</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7474073">7474073</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mair</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ludwig</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrmann</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sieben</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="mixed-article-title" title="mixed-article-title">Receptor binding and pH stabilityâ€"how influenza A virus hemagglutinin affects host-specific virus infection</span><span class="source" tagx="source" title="source">BBA Biomembr.</span><span class="year" tagx="year" title="year">1838</span><span class="volume" tagx="volume" title="volume">1153â€"1168</span><span class="fpage" tagx="fpage" title="fpage">2014</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbamem.2013.10.004">10.1016/j.bbamem.2013.10.004</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Insight into structural diversity of influenza virus haemagglutinin</span><span class="source" tagx="source" title="source">J. Gen. Virol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">1712</span><span class="lpage" tagx="lpage" title="lpage">1722</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.051136-0">10.1099/vir.0.051136-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23636824">23636824</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elster</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A small percentage of influenza-virus M1 protein contains zinc but zinc does not influence in-vitro M1-RNA interaction</span><span class="source" tagx="source" title="source">J. Gen. Virol.</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">37</span><span class="lpage" tagx="lpage" title="lpage">42</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/0022-1317-75-1-37">10.1099/0022-1317-75-1-37</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8113738">8113738</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okada</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miura</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takeuchi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Zinc- and pH-dependent conformational transition in a putative interdornain linker region of the influenza virus matrix protein M1</span><span class="source" tagx="source" title="source">Biochemistry</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">1978</span><span class="lpage" tagx="lpage" title="lpage">1984</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/bi027176t">10.1021/bi027176t</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12590584">12590584</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noyce</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michels</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keevil</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inactivation of influenza A virus on copper versus stainless steel surfaces</span><span class="source" tagx="source" title="source">Appl. Environ. Microb.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">73</span><span class="fpage" tagx="fpage" title="fpage">2748</span><span class="lpage" tagx="lpage" title="lpage">2750</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/Aem.01139-06">10.1128/Aem.01139-06</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horie</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inactivation and morphological changes of avian influenza virus by copper ions</span><span class="source" tagx="source" title="source">Arch. Virol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">153</span><span class="fpage" tagx="fpage" title="fpage">1467</span><span class="lpage" tagx="lpage" title="lpage">1472</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00705-008-0154-2">10.1007/s00705-008-0154-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18592130">18592130</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Zinc finger structures in the human immunodeficiency virus type 1 nucleocapsid protein facilitate efficient minus- and plus-strand transfer</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">74</span><span class="fpage" tagx="fpage" title="fpage">8980</span><span class="lpage" tagx="lpage" title="lpage">8988</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/Jvi.74.19.8980-8988.2000">10.1128/Jvi.74.19.8980-8988.2000</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10982342">10982342</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Urbani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The metal binding site of the hepatitis C virus NS3 proteaseâ€"a spectroscopic investigation</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">273</span><span class="fpage" tagx="fpage" title="fpage">18760</span><span class="lpage" tagx="lpage" title="lpage">18769</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.273.30.18760">10.1074/jbc.273.30.18760</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9668049">9668049</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Modrof</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Becker</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muhlberger</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ebola virus transcription activator VP30 is a zinc-binding protein</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">3334</span><span class="lpage" tagx="lpage" title="lpage">3338</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/Jvi.77.5.3334-3338.2003">10.1128/Jvi.77.5.3334-3338.2003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12584359">12584359</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roth</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Metal thiolate coordination in the E7 proteins of human papilloma virus-16 and cottontail rabbit papilloma-virus as expressed in <i>Escherichia-coli</i></span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">267</span><span class="fpage" tagx="fpage" title="fpage">16390</span><span class="lpage" tagx="lpage" title="lpage">16395</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1322900">1322900</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">CY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kielian</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of a specific region in the E1 fusion protein involved in zinc inhibition of Semliki Forest virus fusion</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">3588</span><span class="lpage" tagx="lpage" title="lpage">3594</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/Jvi.07115-11">10.1128/Jvi.07115-11</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22258261">22258261</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular-level analysis of the serum antibody repertoire in young adults before and after seasonal influenza vaccination</span><span class="source" tagx="source" title="source">Nat. Med.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">1456</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nm.4224">10.1038/nm.4224</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27820605">27820605</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zambon</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span></span><span class="mixed-article-title" title="mixed-article-title">The pathogenesis of influenza in humans</span><span class="source" tagx="source" title="source">Rev. Med. Virol.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">227</span><span class="lpage" tagx="lpage" title="lpage">241</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/rmv.319">10.1002/rmv.319</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11479929">11479929</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carr</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaudhry</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influenza hemagglutinin is spring-loaded by a metastable native conformation</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">14306</span><span class="lpage" tagx="lpage" title="lpage">14313</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.94.26.14306">10.1073/pnas.94.26.14306</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9405608">9405608</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fierke</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span></span><span class="mixed-article-title" title="mixed-article-title">The BaeSR regulon is involved in defense against zinc toxicity in <i>E. coli</i></span><span class="source" tagx="source" title="source">Metallomics</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">372</span><span class="lpage" tagx="lpage" title="lpage">383</span><span class="pub-id"><a href="https://dx.doi.org/10.1039/c3mt20217h">10.1039/c3mt20217h</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23446818">23446818</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Djoko</span><span class="given-names" tagx="given-names" title="given-names">KY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ong</span><span class="given-names" tagx="given-names" title="given-names">CLY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McEwan</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span></span><span class="mixed-article-title" title="mixed-article-title">The role of copper and zinc toxicity in innate immune defense against bacterial pathogens</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">290</span><span class="fpage" tagx="fpage" title="fpage">18954</span><span class="lpage" tagx="lpage" title="lpage">18961</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.R115.647099">10.1074/jbc.R115.647099</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26055706">26055706</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marcus</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ionic-radii in aqueous-solutions</span><span class="source" tagx="source" title="source">Chem. Rev.</span><span class="year" tagx="year" title="year">1988</span><span class="volume" tagx="volume" title="volume">88</span><span class="fpage" tagx="fpage" title="fpage">1475</span><span class="lpage" tagx="lpage" title="lpage">1498</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/cr00090a003">10.1021/cr00090a003</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alberts</span><span class="given-names" tagx="given-names" title="given-names">IL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nadassy</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wodak</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of zinc binding sites in protein crystal structures</span><span class="source" tagx="source" title="source">Protein Sci.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">1700</span><span class="lpage" tagx="lpage" title="lpage">1716</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/pro.5560070805">10.1002/pro.5560070805</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10082367">10082367</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">RJP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Energized (entatic) states of groups and of secondary structures in proteins and metalloproteins</span><span class="source" tagx="source" title="source">Eur. J. Biochem.</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">234</span><span class="fpage" tagx="fpage" title="fpage">363</span><span class="lpage" tagx="lpage" title="lpage">381</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1432-1033.1995.363_b.x">10.1111/j.1432-1033.1995.363_b.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8536678">8536678</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Insight into highly conserved H1 subtype-specific epitopes in influenza virus hemagglutinin</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e89803</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0089803">10.1371/journal.pone.0089803</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24587046">24587046</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">DK</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Direct visualization of the conformational dynamics of single influenza hemagglutinin trimers</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">174</span><span class="fpage" tagx="fpage" title="fpage">926</span><span class="lpage" tagx="lpage" title="lpage">937.e12</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2018.05.050">10.1016/j.cell.2018.05.050</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29961575">29961575</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seok</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Conformational modulation of influenza virus hemagglutinin: characterization and in vivo efficacy of monomeric form</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">7540</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-08021-x">10.1038/s41598-017-08021-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28790432">28790432</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Roey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mutations of endo-beta-N-acetylglucosaminidase H active site residues Asp130 and Glu132: activities and conformations</span><span class="source" tagx="source" title="source">Protein Sci.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">2338</span><span class="lpage" tagx="lpage" title="lpage">2346</span><span class="pub-id"><a href="https://dx.doi.org/10.1110/ps.8.11.2338">10.1110/ps.8.11.2338</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10595536">10595536</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kabsch</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Xds</span><span class="source" tagx="source" title="source">Acta Crystallogr. D</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">125</span><span class="lpage" tagx="lpage" title="lpage">132</span><span class="pub-id"><a href="https://dx.doi.org/10.1107/S0907444909047337">10.1107/S0907444909047337</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20124692">20124692</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span><span class="source" tagx="source" title="source">Acta Crystallogr. Sect. D Struct. Biol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">213</span><span class="lpage" tagx="lpage" title="lpage">221</span><span class="pub-id"><a href="https://dx.doi.org/10.1107/S0907444909052925">10.1107/S0907444909052925</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunger</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span></span><span class="mixed-article-title" title="mixed-article-title">Free R-valueâ€"a novel statistical quantity for assessing the accuracy of crystal-structures</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">1992</span><span class="volume" tagx="volume" title="volume">355</span><span class="fpage" tagx="fpage" title="fpage">472</span><span class="lpage" tagx="lpage" title="lpage">475</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/355472a0">10.1038/355472a0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18481394">18481394</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liebschner</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Polder maps: improving OMIT maps by excluding bulk solvent</span><span class="source" tagx="source" title="source">Acta Crystallogr. D Struct. Biol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">73</span><span class="fpage" tagx="fpage" title="fpage">148</span><span class="lpage" tagx="lpage" title="lpage">157</span><span class="pub-id"><a href="https://dx.doi.org/10.1107/S2059798316018210">10.1107/S2059798316018210</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28177311">28177311</a></span></span></li> </ul> </div> </div>  </body></html>